{
  "cells": [
    {
      "cell_type": "markdown",
      "metadata": {
        "id": "9CwaW0b0CZoU"
      },
      "source": [
        "<font face=\"B Mitra\" size=4>\n",
        "<div dir=rtl align=center>\n",
        "<br>\n",
        "<img src=\"https://aut.ac.ir/templates/tmpl_modern01/images/logo_fa.png\" alt=\"Amirkabir University Logo\" width=\"100\">\n",
        "<br>\n",
        "<font size=6>\n",
        "<b>پروژه سوم داده کاوی</b>\n",
        "<br>\n",
        "<b><font size=5> استاد درس: دکتر فاطمه شاکری</b>\n",
        "<hr>\n",
        "</div>\n",
        "</font>"
      ]
    },
    {
      "cell_type": "code",
      "execution_count": 1,
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/",
          "height": 0
        },
        "id": "wqiVDGR55Wfk",
        "outputId": "510beef4-2e90-4f5d-ada2-529c296583dc"
      },
      "outputs": [
        {
          "name": "stdout",
          "output_type": "stream",
          "text": [
            "Collecting langdetect\n",
            "  Downloading langdetect-1.0.9.tar.gz (981 kB)\n",
            "\u001b[2K     \u001b[90m━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━\u001b[0m \u001b[32m981.5/981.5 kB\u001b[0m \u001b[31m11.7 MB/s\u001b[0m eta \u001b[36m0:00:00\u001b[0m\n",
            "\u001b[?25h  Preparing metadata (setup.py) ... \u001b[?25l\u001b[?25hdone\n",
            "Requirement already satisfied: six in /usr/local/lib/python3.10/dist-packages (from langdetect) (1.16.0)\n",
            "Building wheels for collected packages: langdetect\n",
            "  Building wheel for langdetect (setup.py) ... \u001b[?25l\u001b[?25hdone\n",
            "  Created wheel for langdetect: filename=langdetect-1.0.9-py3-none-any.whl size=993227 sha256=4f2bb50dc003c54f58730e4fdb035400e73b620f86038d5809df9adb9b363d33\n",
            "  Stored in directory: /root/.cache/pip/wheels/95/03/7d/59ea870c70ce4e5a370638b5462a7711ab78fba2f655d05106\n",
            "Successfully built langdetect\n",
            "Installing collected packages: langdetect\n",
            "Successfully installed langdetect-1.0.9\n",
            "Collecting scispacy\n",
            "  Downloading scispacy-0.5.4-py3-none-any.whl (45 kB)\n",
            "\u001b[2K     \u001b[90m━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━\u001b[0m \u001b[32m45.4/45.4 kB\u001b[0m \u001b[31m1.1 MB/s\u001b[0m eta \u001b[36m0:00:00\u001b[0m\n",
            "\u001b[?25hRequirement already satisfied: spacy<3.8.0,>=3.7.0 in /usr/local/lib/python3.10/dist-packages (from scispacy) (3.7.4)\n",
            "Collecting scipy<1.11 (from scispacy)\n",
            "  Downloading scipy-1.10.1-cp310-cp310-manylinux_2_17_x86_64.manylinux2014_x86_64.whl (34.4 MB)\n",
            "\u001b[2K     \u001b[90m━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━\u001b[0m \u001b[32m34.4/34.4 MB\u001b[0m \u001b[31m26.0 MB/s\u001b[0m eta \u001b[36m0:00:00\u001b[0m\n",
            "\u001b[?25hRequirement already satisfied: requests<3.0.0,>=2.0.0 in /usr/local/lib/python3.10/dist-packages (from scispacy) (2.31.0)\n",
            "Collecting conllu (from scispacy)\n",
            "  Downloading conllu-4.5.3-py2.py3-none-any.whl (16 kB)\n",
            "Requirement already satisfied: numpy in /usr/local/lib/python3.10/dist-packages (from scispacy) (1.25.2)\n",
            "Requirement already satisfied: joblib in /usr/local/lib/python3.10/dist-packages (from scispacy) (1.4.2)\n",
            "Collecting nmslib>=1.7.3.6 (from scispacy)\n",
            "  Downloading nmslib-2.1.1.tar.gz (188 kB)\n",
            "\u001b[2K     \u001b[90m━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━\u001b[0m \u001b[32m188.7/188.7 kB\u001b[0m \u001b[31m12.9 MB/s\u001b[0m eta \u001b[36m0:00:00\u001b[0m\n",
            "\u001b[?25h  Preparing metadata (setup.py) ... \u001b[?25l\u001b[?25hdone\n",
            "Requirement already satisfied: scikit-learn>=0.20.3 in /usr/local/lib/python3.10/dist-packages (from scispacy) (1.2.2)\n",
            "Collecting pysbd (from scispacy)\n",
            "  Downloading pysbd-0.3.4-py3-none-any.whl (71 kB)\n",
            "\u001b[2K     \u001b[90m━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━\u001b[0m \u001b[32m71.1/71.1 kB\u001b[0m \u001b[31m10.2 MB/s\u001b[0m eta \u001b[36m0:00:00\u001b[0m\n",
            "\u001b[?25hCollecting pybind11<2.6.2 (from nmslib>=1.7.3.6->scispacy)\n",
            "  Using cached pybind11-2.6.1-py2.py3-none-any.whl (188 kB)\n",
            "Requirement already satisfied: psutil in /usr/local/lib/python3.10/dist-packages (from nmslib>=1.7.3.6->scispacy) (5.9.5)\n",
            "Requirement already satisfied: charset-normalizer<4,>=2 in /usr/local/lib/python3.10/dist-packages (from requests<3.0.0,>=2.0.0->scispacy) (3.3.2)\n",
            "Requirement already satisfied: idna<4,>=2.5 in /usr/local/lib/python3.10/dist-packages (from requests<3.0.0,>=2.0.0->scispacy) (3.7)\n",
            "Requirement already satisfied: urllib3<3,>=1.21.1 in /usr/local/lib/python3.10/dist-packages (from requests<3.0.0,>=2.0.0->scispacy) (2.0.7)\n",
            "Requirement already satisfied: certifi>=2017.4.17 in /usr/local/lib/python3.10/dist-packages (from requests<3.0.0,>=2.0.0->scispacy) (2024.6.2)\n",
            "Requirement already satisfied: threadpoolctl>=2.0.0 in /usr/local/lib/python3.10/dist-packages (from scikit-learn>=0.20.3->scispacy) (3.5.0)\n",
            "Requirement already satisfied: spacy-legacy<3.1.0,>=3.0.11 in /usr/local/lib/python3.10/dist-packages (from spacy<3.8.0,>=3.7.0->scispacy) (3.0.12)\n",
            "Requirement already satisfied: spacy-loggers<2.0.0,>=1.0.0 in /usr/local/lib/python3.10/dist-packages (from spacy<3.8.0,>=3.7.0->scispacy) (1.0.5)\n",
            "Requirement already satisfied: murmurhash<1.1.0,>=0.28.0 in /usr/local/lib/python3.10/dist-packages (from spacy<3.8.0,>=3.7.0->scispacy) (1.0.10)\n",
            "Requirement already satisfied: cymem<2.1.0,>=2.0.2 in /usr/local/lib/python3.10/dist-packages (from spacy<3.8.0,>=3.7.0->scispacy) (2.0.8)\n",
            "Requirement already satisfied: preshed<3.1.0,>=3.0.2 in /usr/local/lib/python3.10/dist-packages (from spacy<3.8.0,>=3.7.0->scispacy) (3.0.9)\n",
            "Requirement already satisfied: thinc<8.3.0,>=8.2.2 in /usr/local/lib/python3.10/dist-packages (from spacy<3.8.0,>=3.7.0->scispacy) (8.2.3)\n",
            "Requirement already satisfied: wasabi<1.2.0,>=0.9.1 in /usr/local/lib/python3.10/dist-packages (from spacy<3.8.0,>=3.7.0->scispacy) (1.1.3)\n",
            "Requirement already satisfied: srsly<3.0.0,>=2.4.3 in /usr/local/lib/python3.10/dist-packages (from spacy<3.8.0,>=3.7.0->scispacy) (2.4.8)\n",
            "Requirement already satisfied: catalogue<2.1.0,>=2.0.6 in /usr/local/lib/python3.10/dist-packages (from spacy<3.8.0,>=3.7.0->scispacy) (2.0.10)\n",
            "Requirement already satisfied: weasel<0.4.0,>=0.1.0 in /usr/local/lib/python3.10/dist-packages (from spacy<3.8.0,>=3.7.0->scispacy) (0.3.4)\n",
            "Requirement already satisfied: typer<0.10.0,>=0.3.0 in /usr/local/lib/python3.10/dist-packages (from spacy<3.8.0,>=3.7.0->scispacy) (0.9.4)\n",
            "Requirement already satisfied: smart-open<7.0.0,>=5.2.1 in /usr/local/lib/python3.10/dist-packages (from spacy<3.8.0,>=3.7.0->scispacy) (6.4.0)\n",
            "Requirement already satisfied: tqdm<5.0.0,>=4.38.0 in /usr/local/lib/python3.10/dist-packages (from spacy<3.8.0,>=3.7.0->scispacy) (4.66.4)\n",
            "Requirement already satisfied: pydantic!=1.8,!=1.8.1,<3.0.0,>=1.7.4 in /usr/local/lib/python3.10/dist-packages (from spacy<3.8.0,>=3.7.0->scispacy) (2.7.3)\n",
            "Requirement already satisfied: jinja2 in /usr/local/lib/python3.10/dist-packages (from spacy<3.8.0,>=3.7.0->scispacy) (3.1.4)\n",
            "Requirement already satisfied: setuptools in /usr/local/lib/python3.10/dist-packages (from spacy<3.8.0,>=3.7.0->scispacy) (67.7.2)\n",
            "Requirement already satisfied: packaging>=20.0 in /usr/local/lib/python3.10/dist-packages (from spacy<3.8.0,>=3.7.0->scispacy) (24.0)\n",
            "Requirement already satisfied: langcodes<4.0.0,>=3.2.0 in /usr/local/lib/python3.10/dist-packages (from spacy<3.8.0,>=3.7.0->scispacy) (3.4.0)\n",
            "Requirement already satisfied: language-data>=1.2 in /usr/local/lib/python3.10/dist-packages (from langcodes<4.0.0,>=3.2.0->spacy<3.8.0,>=3.7.0->scispacy) (1.2.0)\n",
            "Requirement already satisfied: annotated-types>=0.4.0 in /usr/local/lib/python3.10/dist-packages (from pydantic!=1.8,!=1.8.1,<3.0.0,>=1.7.4->spacy<3.8.0,>=3.7.0->scispacy) (0.7.0)\n",
            "Requirement already satisfied: pydantic-core==2.18.4 in /usr/local/lib/python3.10/dist-packages (from pydantic!=1.8,!=1.8.1,<3.0.0,>=1.7.4->spacy<3.8.0,>=3.7.0->scispacy) (2.18.4)\n",
            "Requirement already satisfied: typing-extensions>=4.6.1 in /usr/local/lib/python3.10/dist-packages (from pydantic!=1.8,!=1.8.1,<3.0.0,>=1.7.4->spacy<3.8.0,>=3.7.0->scispacy) (4.12.1)\n",
            "Requirement already satisfied: blis<0.8.0,>=0.7.8 in /usr/local/lib/python3.10/dist-packages (from thinc<8.3.0,>=8.2.2->spacy<3.8.0,>=3.7.0->scispacy) (0.7.11)\n",
            "Requirement already satisfied: confection<1.0.0,>=0.0.1 in /usr/local/lib/python3.10/dist-packages (from thinc<8.3.0,>=8.2.2->spacy<3.8.0,>=3.7.0->scispacy) (0.1.5)\n",
            "Requirement already satisfied: click<9.0.0,>=7.1.1 in /usr/local/lib/python3.10/dist-packages (from typer<0.10.0,>=0.3.0->spacy<3.8.0,>=3.7.0->scispacy) (8.1.7)\n",
            "Requirement already satisfied: cloudpathlib<0.17.0,>=0.7.0 in /usr/local/lib/python3.10/dist-packages (from weasel<0.4.0,>=0.1.0->spacy<3.8.0,>=3.7.0->scispacy) (0.16.0)\n",
            "Requirement already satisfied: MarkupSafe>=2.0 in /usr/local/lib/python3.10/dist-packages (from jinja2->spacy<3.8.0,>=3.7.0->scispacy) (2.1.5)\n",
            "Requirement already satisfied: marisa-trie>=0.7.7 in /usr/local/lib/python3.10/dist-packages (from language-data>=1.2->langcodes<4.0.0,>=3.2.0->spacy<3.8.0,>=3.7.0->scispacy) (1.1.1)\n",
            "Building wheels for collected packages: nmslib\n",
            "  Building wheel for nmslib (setup.py) ... \u001b[?25l\u001b[?25hcanceled\n",
            "\u001b[31mERROR: Operation cancelled by user\u001b[0m\u001b[31m\n",
            "\u001b[0mCollecting https://s3-us-west-2.amazonaws.com/ai2-s2-scispacy/releases/v0.5.4/en_core_sci_lg-0.5.4.tar.gz\n",
            "  Downloading https://s3-us-west-2.amazonaws.com/ai2-s2-scispacy/releases/v0.5.4/en_core_sci_lg-0.5.4.tar.gz (531.2 MB)\n",
            "\u001b[2K     \u001b[90m━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━\u001b[0m \u001b[32m531.2/531.2 MB\u001b[0m \u001b[31m1.9 MB/s\u001b[0m eta \u001b[36m0:00:00\u001b[0m\n",
            "\u001b[?25h  Preparing metadata (setup.py) ... \u001b[?25l\u001b[?25hdone\n",
            "Requirement already satisfied: spacy<3.8.0,>=3.7.4 in /usr/local/lib/python3.10/dist-packages (from en-core-sci-lg==0.5.4) (3.7.4)\n",
            "Requirement already satisfied: spacy-legacy<3.1.0,>=3.0.11 in /usr/local/lib/python3.10/dist-packages (from spacy<3.8.0,>=3.7.4->en-core-sci-lg==0.5.4) (3.0.12)\n",
            "Requirement already satisfied: spacy-loggers<2.0.0,>=1.0.0 in /usr/local/lib/python3.10/dist-packages (from spacy<3.8.0,>=3.7.4->en-core-sci-lg==0.5.4) (1.0.5)\n",
            "Requirement already satisfied: murmurhash<1.1.0,>=0.28.0 in /usr/local/lib/python3.10/dist-packages (from spacy<3.8.0,>=3.7.4->en-core-sci-lg==0.5.4) (1.0.10)\n",
            "Requirement already satisfied: cymem<2.1.0,>=2.0.2 in /usr/local/lib/python3.10/dist-packages (from spacy<3.8.0,>=3.7.4->en-core-sci-lg==0.5.4) (2.0.8)\n",
            "Requirement already satisfied: preshed<3.1.0,>=3.0.2 in /usr/local/lib/python3.10/dist-packages (from spacy<3.8.0,>=3.7.4->en-core-sci-lg==0.5.4) (3.0.9)\n",
            "Requirement already satisfied: thinc<8.3.0,>=8.2.2 in /usr/local/lib/python3.10/dist-packages (from spacy<3.8.0,>=3.7.4->en-core-sci-lg==0.5.4) (8.2.3)\n",
            "Requirement already satisfied: wasabi<1.2.0,>=0.9.1 in /usr/local/lib/python3.10/dist-packages (from spacy<3.8.0,>=3.7.4->en-core-sci-lg==0.5.4) (1.1.3)\n",
            "Requirement already satisfied: srsly<3.0.0,>=2.4.3 in /usr/local/lib/python3.10/dist-packages (from spacy<3.8.0,>=3.7.4->en-core-sci-lg==0.5.4) (2.4.8)\n",
            "Requirement already satisfied: catalogue<2.1.0,>=2.0.6 in /usr/local/lib/python3.10/dist-packages (from spacy<3.8.0,>=3.7.4->en-core-sci-lg==0.5.4) (2.0.10)\n",
            "Requirement already satisfied: weasel<0.4.0,>=0.1.0 in /usr/local/lib/python3.10/dist-packages (from spacy<3.8.0,>=3.7.4->en-core-sci-lg==0.5.4) (0.3.4)\n",
            "Requirement already satisfied: typer<0.10.0,>=0.3.0 in /usr/local/lib/python3.10/dist-packages (from spacy<3.8.0,>=3.7.4->en-core-sci-lg==0.5.4) (0.9.4)\n",
            "Requirement already satisfied: smart-open<7.0.0,>=5.2.1 in /usr/local/lib/python3.10/dist-packages (from spacy<3.8.0,>=3.7.4->en-core-sci-lg==0.5.4) (6.4.0)\n",
            "Requirement already satisfied: tqdm<5.0.0,>=4.38.0 in /usr/local/lib/python3.10/dist-packages (from spacy<3.8.0,>=3.7.4->en-core-sci-lg==0.5.4) (4.66.4)\n",
            "Requirement already satisfied: requests<3.0.0,>=2.13.0 in /usr/local/lib/python3.10/dist-packages (from spacy<3.8.0,>=3.7.4->en-core-sci-lg==0.5.4) (2.31.0)\n",
            "Requirement already satisfied: pydantic!=1.8,!=1.8.1,<3.0.0,>=1.7.4 in /usr/local/lib/python3.10/dist-packages (from spacy<3.8.0,>=3.7.4->en-core-sci-lg==0.5.4) (2.7.3)\n",
            "Requirement already satisfied: jinja2 in /usr/local/lib/python3.10/dist-packages (from spacy<3.8.0,>=3.7.4->en-core-sci-lg==0.5.4) (3.1.4)\n",
            "Requirement already satisfied: setuptools in /usr/local/lib/python3.10/dist-packages (from spacy<3.8.0,>=3.7.4->en-core-sci-lg==0.5.4) (67.7.2)\n",
            "Requirement already satisfied: packaging>=20.0 in /usr/local/lib/python3.10/dist-packages (from spacy<3.8.0,>=3.7.4->en-core-sci-lg==0.5.4) (24.0)\n",
            "Requirement already satisfied: langcodes<4.0.0,>=3.2.0 in /usr/local/lib/python3.10/dist-packages (from spacy<3.8.0,>=3.7.4->en-core-sci-lg==0.5.4) (3.4.0)\n",
            "Requirement already satisfied: numpy>=1.19.0 in /usr/local/lib/python3.10/dist-packages (from spacy<3.8.0,>=3.7.4->en-core-sci-lg==0.5.4) (1.25.2)\n",
            "Requirement already satisfied: language-data>=1.2 in /usr/local/lib/python3.10/dist-packages (from langcodes<4.0.0,>=3.2.0->spacy<3.8.0,>=3.7.4->en-core-sci-lg==0.5.4) (1.2.0)\n",
            "Requirement already satisfied: annotated-types>=0.4.0 in /usr/local/lib/python3.10/dist-packages (from pydantic!=1.8,!=1.8.1,<3.0.0,>=1.7.4->spacy<3.8.0,>=3.7.4->en-core-sci-lg==0.5.4) (0.7.0)\n",
            "Requirement already satisfied: pydantic-core==2.18.4 in /usr/local/lib/python3.10/dist-packages (from pydantic!=1.8,!=1.8.1,<3.0.0,>=1.7.4->spacy<3.8.0,>=3.7.4->en-core-sci-lg==0.5.4) (2.18.4)\n",
            "Requirement already satisfied: typing-extensions>=4.6.1 in /usr/local/lib/python3.10/dist-packages (from pydantic!=1.8,!=1.8.1,<3.0.0,>=1.7.4->spacy<3.8.0,>=3.7.4->en-core-sci-lg==0.5.4) (4.12.1)\n",
            "Requirement already satisfied: charset-normalizer<4,>=2 in /usr/local/lib/python3.10/dist-packages (from requests<3.0.0,>=2.13.0->spacy<3.8.0,>=3.7.4->en-core-sci-lg==0.5.4) (3.3.2)\n",
            "Requirement already satisfied: idna<4,>=2.5 in /usr/local/lib/python3.10/dist-packages (from requests<3.0.0,>=2.13.0->spacy<3.8.0,>=3.7.4->en-core-sci-lg==0.5.4) (3.7)\n",
            "Requirement already satisfied: urllib3<3,>=1.21.1 in /usr/local/lib/python3.10/dist-packages (from requests<3.0.0,>=2.13.0->spacy<3.8.0,>=3.7.4->en-core-sci-lg==0.5.4) (2.0.7)\n",
            "Requirement already satisfied: certifi>=2017.4.17 in /usr/local/lib/python3.10/dist-packages (from requests<3.0.0,>=2.13.0->spacy<3.8.0,>=3.7.4->en-core-sci-lg==0.5.4) (2024.6.2)\n",
            "Requirement already satisfied: blis<0.8.0,>=0.7.8 in /usr/local/lib/python3.10/dist-packages (from thinc<8.3.0,>=8.2.2->spacy<3.8.0,>=3.7.4->en-core-sci-lg==0.5.4) (0.7.11)\n",
            "Requirement already satisfied: confection<1.0.0,>=0.0.1 in /usr/local/lib/python3.10/dist-packages (from thinc<8.3.0,>=8.2.2->spacy<3.8.0,>=3.7.4->en-core-sci-lg==0.5.4) (0.1.5)\n",
            "Requirement already satisfied: click<9.0.0,>=7.1.1 in /usr/local/lib/python3.10/dist-packages (from typer<0.10.0,>=0.3.0->spacy<3.8.0,>=3.7.4->en-core-sci-lg==0.5.4) (8.1.7)\n",
            "Requirement already satisfied: cloudpathlib<0.17.0,>=0.7.0 in /usr/local/lib/python3.10/dist-packages (from weasel<0.4.0,>=0.1.0->spacy<3.8.0,>=3.7.4->en-core-sci-lg==0.5.4) (0.16.0)\n",
            "Requirement already satisfied: MarkupSafe>=2.0 in /usr/local/lib/python3.10/dist-packages (from jinja2->spacy<3.8.0,>=3.7.4->en-core-sci-lg==0.5.4) (2.1.5)\n",
            "Requirement already satisfied: marisa-trie>=0.7.7 in /usr/local/lib/python3.10/dist-packages (from language-data>=1.2->langcodes<4.0.0,>=3.2.0->spacy<3.8.0,>=3.7.4->en-core-sci-lg==0.5.4) (1.1.1)\n",
            "Building wheels for collected packages: en-core-sci-lg\n",
            "  Building wheel for en-core-sci-lg (setup.py) ... \u001b[?25l\u001b[?25hdone\n",
            "  Created wheel for en-core-sci-lg: filename=en_core_sci_lg-0.5.4-py3-none-any.whl size=531476539 sha256=f1810676dc6777e21cc1817f8c3be3e24e18f8e4342579d3d6dee798ce962ecd\n",
            "  Stored in directory: /root/.cache/pip/wheels/04/77/e7/614feae28ed7056dd89b009b32c30573bd85fb7d1ea3c7c227\n",
            "Successfully built en-core-sci-lg\n",
            "Installing collected packages: en-core-sci-lg\n",
            "Successfully installed en-core-sci-lg-0.5.4\n"
          ]
        }
      ],
      "source": [
        "!pip install langdetect\n",
        "!pip install scispacy\n",
        "!pip install https://s3-us-west-2.amazonaws.com/ai2-s2-scispacy/releases/v0.5.4/en_core_sci_lg-0.5.4.tar.gz"
      ]
    },
    {
      "cell_type": "code",
      "execution_count": 2,
      "metadata": {
        "id": "hF_lqFaDUMGY"
      },
      "outputs": [],
      "source": [
        "\n",
        "import numpy as np\n",
        "import pandas as pd\n",
        "\n",
        "# Plotting and visualization\n",
        "import matplotlib.pyplot as plt\n",
        "plt.style.use('ggplot')\n",
        "%config InlineBackend.figure_formats = ['svg']\n",
        "import seaborn as sns\n",
        "\n",
        "# Used to draw a progress bar for longer method calls\n",
        "from tqdm import tqdm\n",
        "\n",
        "# Used to detect language used in each document\n",
        "from langdetect import detect\n",
        "from langdetect import DetectorFactory\n",
        "DetectorFactory.seed = 0\n",
        "\n",
        "# Pre-trained natural language processing pipeline for biomedical use\n",
        "import spacy\n",
        "from spacy.lang.en.stop_words import STOP_WORDS\n",
        "import en_core_sci_lg\n",
        "\n",
        "# Used to import list of punctuations\n",
        "import string\n",
        "\n",
        "# Feature extraction (text vectorizers)\n",
        "from sklearn.feature_extraction.text import TfidfVectorizer\n",
        "from sklearn.feature_extraction.text import CountVectorizer\n",
        "\n",
        "# Dimensionality reduction\n",
        "from sklearn.decomposition import PCA\n",
        "from sklearn.manifold import TSNE\n",
        "\n",
        "# Clustering and evaluation\n",
        "from sklearn.cluster import KMeans\n",
        "from sklearn import metrics\n",
        "from scipy.spatial.distance import cdist\n",
        "\n",
        "# Topic modeling\n",
        "from sklearn.decomposition import LatentDirichletAllocation"
      ]
    },
    {
      "cell_type": "markdown",
      "metadata": {
        "id": "mY9evTyRQA82"
      },
      "source": [
        "!gdown 15E8FLX0C-6qpK-lDBEQJXw00Lsdcvjae"
      ]
    },
    {
      "cell_type": "code",
      "execution_count": 3,
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/",
          "height": 0
        },
        "id": "MMZnPe1TcsLa",
        "outputId": "c84f007b-7b8a-4cf5-fd75-63e19efee1a6"
      },
      "outputs": [
        {
          "name": "stdout",
          "output_type": "stream",
          "text": [
            "Downloading...\n",
            "From (original): https://drive.google.com/uc?id=15E8FLX0C-6qpK-lDBEQJXw00Lsdcvjae\n",
            "From (redirected): https://drive.google.com/uc?id=15E8FLX0C-6qpK-lDBEQJXw00Lsdcvjae&confirm=t&uuid=2d3d4a3b-5b0f-4cb8-a328-76ef7ccad3c6\n",
            "To: /content/10k_df.csv\n",
            "100% 341M/341M [00:07<00:00, 44.8MB/s]\n"
          ]
        }
      ],
      "source": [
        "!gdown 15E8FLX0C-6qpK-lDBEQJXw00Lsdcvjae"
      ]
    },
    {
      "cell_type": "code",
      "execution_count": 4,
      "metadata": {
        "id": "mG_ng9hsUMGc"
      },
      "outputs": [],
      "source": [
        "df_10k = pd.read_csv('10k_df.csv')"
      ]
    },
    {
      "cell_type": "code",
      "execution_count": 5,
      "metadata": {
        "id": "CppfUdF3Zyp8"
      },
      "outputs": [],
      "source": [
        "df_10k.fillna(value=\" \",inplace=True)"
      ]
    },
    {
      "cell_type": "code",
      "execution_count": 6,
      "metadata": {
        "id": "ehfaa_-eUMGd"
      },
      "outputs": [],
      "source": [
        "df = df_10k.sample(1500, random_state=42)\n",
        "del df_10k"
      ]
    },
    {
      "cell_type": "code",
      "execution_count": 7,
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/",
          "height": 501
        },
        "id": "YAyZvCT8c14n",
        "outputId": "9cd195ef-afd4-4e76-f04a-259bc6002bb5"
      },
      "outputs": [
        {
          "data": {
            "application/vnd.google.colaboratory.intrinsic+json": {
              "summary": "{\n  \"name\": \"df\",\n  \"rows\": 1500,\n  \"fields\": [\n    {\n      \"column\": \"paper_id\",\n      \"properties\": {\n        \"dtype\": \"string\",\n        \"num_unique_values\": 1500,\n        \"samples\": [\n          \"a3e99e6c5f921c7d0195bacf6e6a93d7d348a50d\",\n          \"b181570ef0a99eb223e537a174365cbcebf896fd\",\n          \"ec8af8cbd03cfd3d345d295c88d0900af3a6d6fd\"\n        ],\n        \"semantic_type\": \"\",\n        \"description\": \"\"\n      }\n    },\n    {\n      \"column\": \"doi\",\n      \"properties\": {\n        \"dtype\": \"string\",\n        \"num_unique_values\": 1457,\n        \"samples\": [\n          \"10.1016/j.pharma.2008.05.004\",\n          \"10.1111/j.1462-2920.2009.01962.x\",\n          \"10.3389/fimmu.2018.01397\"\n        ],\n        \"semantic_type\": \"\",\n        \"description\": \"\"\n      }\n    },\n    {\n      \"column\": \"abstract\",\n      \"properties\": {\n        \"dtype\": \"string\",\n        \"num_unique_values\": 1067,\n        \"samples\": [\n          \"Several viruses cause pulmonary infections due to their shared tropism with cells of the respiratory tract. These respiratory problems due to viral infection become a public health concern due to rapid transmission through air/aerosols or via direct-indirect contact with infected persons. In addition, the cross-species transmission causes alterations to viral genetic makeup thereby increasing the risk of emergence of pathogens with new and more potent infectivity. With the introduction of effective nucleic acid-based technologies, post translational gene silencing (PTGS) is being increasingly used to silence viral gene targets and has shown promising approach towards management of many viral infections. Since several host factors are also utilized by these viruses during various stages of infection, silencing these host factors can also serve as promising therapeutic tool. Several nucleic acid-based technologies such as short interfering RNAs (siRNA), antisense oligonucleotides, aptamers, deoxyribozymes (DNAzymes), and ribozymes have been studied and used against management of respiratory viruses. These therapeutic nucleic acids can be efficiently delivered through the airways. Studies have also shown efficacy of gene therapy in clinical trials against respiratory syncytial virus (RSV) as well as models of respiratory diseases including severe acute respiratory syndrome (SARS), measles and influenza. In this review, we have summarized some of the recent advancements made in the area of nucleic acid based therapeutics and highlighted the emerging roles of nucleic acids in the management of some of the severe respiratory viral infections. We have also focused on the methods of their delivery and associated challenges.\",\n          \"Background: Newcastle disease virus (NDV) is an enveloped RNA virus, bearing severe economic losses to the poultry industry worldwide. Previous virion proteomic studies have shown that enveloped viruses carry multiple host cellular proteins both internally and externally during their life cycle. To address whether it also occurred during NDV infection, we performed a comprehensive proteomic analysis of highly purified NDV La Sota strain particles.\\nResults: In addition to five viral structural proteins, we detected thirty cellular proteins associated with purified NDV La Sota particles. The identified cellular proteins comprised several functional categories, including cytoskeleton proteins, annexins, molecular chaperones, chromatin modifying proteins, enzymes-binding proteins, calcium-binding proteins and signal transduction-associated proteins. Among these, three host proteins have not been previously reported in virions of other virus families, including two signal transduction-associated proteins (syntenin and Ras small GTPase) and one tumor-associated protein (tumor protein D52). The presence of five selected cellular proteins (i.e., \\u03b2-actin, tubulin, annexin A2, heat shock protein Hsp90 and ezrin) associated with the purified NDV particles was validated by Western blot or immunogold labeling assays. Conclusions: The current study presented the first standard proteomic profile of NDV. The results demonstrated the incorporation of cellular proteins in NDV particles, which provides valuable information for elucidating viral infection and pathogenesis.\",\n          \"Recent studies have shown that gamma interferon (IFN-\\u03b3) synergizes with the innate IFNs (IFN-\\u03b1 and IFN-\\u03b2) to inhibit herpes simplex virus type 1 (HSV-1) replication in vitro. To determine whether this phenomenon is shared by other herpesviruses, we investigated the effects of IFNs on human cytomegalovirus (HCMV) replication.\\nWe have found that as with HSV-1, IFN-\\u03b3 synergizes with the innate IFNs (IFN-\\u03b1/\\u03b2) to potently inhibit HCMV replication in vitro. While pre-treatment of human foreskin fibroblasts (HFFs) with IFN-\\u03b1, IFN-\\u03b2 or IFN-\\u03b3 alone inhibited HCMV plaque formation by ~30 to 40-fold, treatment with IFN-\\u03b1 and IFN-\\u03b3 or IFN-\\u03b2 and IFN-\\u03b3 inhibited HCMV plaque formation by 163-and 662-fold, respectively. The generation of isobole plots verified that the observed inhibition of HCMV plaque formation and replication in HFFs by IFN-\\u03b1/\\u03b2 and IFN-\\u03b3 was a synergistic interaction. Additionally, real-time PCR analyses of the HCMV immediate early (IE) genes (IE1 and IE2) revealed that IE mRNA expression was profoundly decreased in cells stimulated with IFN-\\u03b1/\\u03b2 and IFN-\\u03b3 (~5-11-fold) as compared to vehicle-treated cells. Furthermore, decreased IE mRNA expression was accompanied by a decrease in IE protein expression, as demonstrated by western blotting and immunofluorescence.\\nConclusion: These findings suggest that IFN-\\u03b1/\\u03b2 and IFN-\\u03b3 synergistically inhibit HCMV replication through a mechanism that may involve the regulation of IE gene expression. We hypothesize that IFN-\\u03b3 produced by activated cells of the adaptive immune response may potentially synergize with endogenous type I IFNs to inhibit HCMV dissemination in vivo.\"\n        ],\n        \"semantic_type\": \"\",\n        \"description\": \"\"\n      }\n    },\n    {\n      \"column\": \"body_text\",\n      \"properties\": {\n        \"dtype\": \"string\",\n        \"num_unique_values\": 1499,\n        \"samples\": [\n          \"The Ebola epidemic in West Africa: Challenges, opportunities, and policy priority areas Introduction\\nIn December 2013, a reported unknown contagious and lethal illness started with a young boy in Gu\\u00e9 ck\\u00e9 dou, Guineada town well-known for the presence of West African traders from Guinea, Sierra Leone, Liberia, and C\\u00f4 te d'Ivoire. The disease, which caught health officials in the region off guard, was soon identified as the Ebola virus disease (EVD), a disease with a case fatality rate of 90% (Heymann, 2015) . The Ebola epidemic wreaking havoc in West Africa has led to a global ripple effect that has affected the United States and other Western countries. In the absence of a vaccine or treatment for Ebola, the disease has alarmed the global public health community and caused panic among some segments of the population. The fear associated with Ebola, a deadly disease that respects no borders or socioeconomic status, has captured the attention of the global health community (Baden et al., 2014) . Although Ebola has been around since 1976 (Piot, 2012) , with past outbreaks contained in Uganda and Congo, many people do not have the knowledge of how the disease is contracted and spread, and, thus, fears and myths continue unabated. At the time of this writing, according to the World Health Organization (WHO), worldwide a total of 17,145 cases of EVD have been reported in five affected countries (Guinea, Liberia, Mali, Sierra Leone, and the United States) and three previously affected countries (Nigeria, Senegal, and Spain) up to November 30, 2014 . Globally, there have been 6,070 reported deaths (WHO, 2014a) . Liberia, Sierra Leone, and Guinea remain the epicenter of the disease. As of November 30, 2014, WHO reports 2,164 cumulative cases and 1,327 deaths in Guinea; 7,635 cumulative cases and 3,145 deaths in Liberia; and 7,312 cumulative cases and 1,583 cumulative deaths in Sierra Leone. Among the three countries at the epicenter of the disease, 17,111 cumulative cases and 6,055 cumulative deaths have been reported (WHO, 2014a) . Although the number of new cases is reported to be stabilizing, the situation in these countries remains uncertain, with no treatment in sight. A map of countries at the epicenter of the Ebola epidemic in West Africa is displayed in Figure 1 .\\nEbola has tested the readiness of health care systems around the world. The disease has especially drawn attention to the inadequacies of sub-Saharan African countries' health care systems and the lack of health care infrastructures to handle complex health emergencies in low-resourced, developing country settings. Because Ebola is an unprecedented disease that threatens the global community, international health agencies and the private sector are in a race with time to develop appropriate medications and potential vaccines to treat and prevent the disease. Although the advancement of such solutions is paramount, it is important to look beyond the current epidemic by examining factors that have led to such inefficiencies in the affected countries' health care systems.\\nAll three affected countries are emerging from civil wars, which has resulted in low levels of the availability of health resources and dysfunctional health care systems, largely resulting from the lack of adequately trained and available health care workers. Sub-Saharan Africa accounts for 24% of the global burden of disease but with only 3% of the world's available health workforce (World Bank, 2013) . Even before the Ebola outbreak, as democracy was being reembraced in these countries, health care providers were overwhelmed with meeting system-wide needs and developing postwar capacities while economic growth and redevelopment were occurring. It is pertinent, then, to examine the challenges and opportunities for addressing the Ebola epidemic in the affected countries and propose strategies for strengthening these countries' health systems.\\nAlthough the governments of the affected countries have worked to improve the economy and health status of their populations, life expectancy across this region remains low. Physician and hospital bed capacities are dismally inadequate from a comparative global perspective, minimally meeting the basic health care needs of the general population. Infant and maternal death rates remain high. Because of high levels of illiteracy, it becomes difficult to disseminate uniform health messages. Economic and healthrelated information of the affected countries is displayed in Table 1 .\\nPublic health infrastructure in the region lacks even the rudimentary armaments to wage battle against an enemy such as Ebola (Forrester et al., 2014) . Because the dire necessity in West Africa is so immense, opportunities to make a difference are many for American health care professionals, including nurses, individually and collectively. A critical factor lacking in the fight to address the current Ebola outbreak in West Africa is the absence of better delineation of contextual factors that serve as challenges to mitigating the spread of the disease. What are the individual, structural, and community factors that impede efforts to address Ebola in the West African countries at the epicenter of the epidemic? Also, despite the devastating effect of the disease on populations, the Ebola epidemic should be viewed as an opportunity for reorienting health care delivery systems and building countries' capacities at a level such that they are capable of addressing new and future outbreaks of Ebola-like infectious diseases in sub-Saharan Africa.\\nThe purpose of this article is to (a) critically examine the sociopolitical and economic conditions that created the environment for the Ebola epidemic to occur, (b) identify challenges to and opportunities for the prevention and control of Ebola and future outbreaks, and (c) discuss policy recommendations and priority areas for addressing the Ebola epidemic and future outbreaks in West Africa.\\nOur approach is guided by the use of literature in peer-reviewed journals on disease burden and health system reforms in developing countries, specifically in sub-Saharan West African countries. Periodicals released by international organizations, including the United Nations Development Program, WHO, United States Agency for International Development, and the World Bank, on global health challenges were also relied on to critically examine the health, sociopolitical, and economic conditions and to identify the priority and policy areas discussed in this article. Building on the policy analysis work of de-Graft Aikins, Boynton, and Atanga (2010), the structural psychology work of Hepworth (2004) , and the antiretroviral adherence analysis work of Kagee et al. (2011) , we extrapolated from these prior works to critically examine the challenges and opportunities that, if understood and addressed, can effectively contribute to halting the spread of Ebola and potential infectious diseases specific to the West African setting. \\nIn this section, we organize and discuss findings from our analysis with a focus on personal-/individual-, structural-/organizational-, and community-level challenges that must be addressed in an effort to reduce the spread of the disease in West Africa. Although many view the disease primarily from a problem-based perspective, we contend that there are multiple opportunities for the prevention and control of Ebola and future infectious disease outbreaks in the West African setting. We maintain that, as with most controversial health and social issues, embracing a community-based, participatory approach in which key members of the public participate in decision making on how to mitigate the impact of Ebola on their communities would be a viable approach. Such approaches should also focus on the assets and resiliency of local health care workers and the affected populations.\\nTo fully accomplish this participatory approach, trust by the local people of their government officials and health care workers will be pivotal. Therefore, a combination of centralized \\\"top-down\\\" and decentralized \\\"bottom-up\\\" approaches is proposed, as is the need to include trust-building capacities in order to work effectively (Chowdhury, 2012) . In the case of the Ebola epidemic in West Africa, the use of the bottomup approach (i.e., an empowerment model that creates opportunities for public health frontline workers to engage leaders and members of local communities to discuss factors surrounding the disease) is essential. Within the context of the bottom-up approach, community leaders are regarded as important stakeholders and knowledge reservoirs in contributing positively to programs that embrace and represent the values of their community members. The bottom-up approach has been used in several developing countries in designing effective primary health care programs (Chowdhury, 2012; Magnussen, Ehiri, & Jolly, 2004) . Thus, combining an array of individual-, structural-, and community-level factors directed at curtailing the Ebola epidemic in West Africa is essential.\\nPersonal/Individual-Level Challenges Entrenched Poverty and Food Insecurity. Barriers at the personal level affect individuals; people and their families may or may not be able to exercise control over their actions. For instance, the number of people in sub-Saharan Africa who live on incomes between $1.25 U.S. dollars to $2.00 U.S. dollars per day is disproportionately high compared with other regions of the world (World Bank, 2013). This extreme poverty causes individual resource scarcity, overcrowding, and rampant sharing of the basic amenities needed for survival. Many in the affected African countries go without bare necessities including food, water, sanitation, and shelter. Thus, Ebola is bound to adversely affect the poorest of individualsdthose displaced during civil conflicts in these three countries and now living in urban slums.\\nFear, Perceptions of Risk, and Social Stigma. Infectious diseases such as Ebola set in motion psychological responses including fear and stigma (Kinsman, 2012) . Social stigma as a response to a disease is not a new phenomenon. Historically, the following infectious diseases are associated with a stigma: influenza, tuberculosis, HIV/AIDS, and severe acute respiratory syndrome (Barrett & Brown, 2008; Kinsman, 2012) .\\nLimited Access to Modern Health Care. Limited access to modern health care services is an important challenge. Even before the Ebola outbreak, access to basic primary health care services was a major challenge to good health in the affected African countries. Access to early supportive care is pivotal for individuals suspected of being infected with Ebola. With interventions from the U.S. military and other international organizations, the number of Ebola treatment units has increased in Liberia. Unfortunately, because of the lack of transportation and limited ambulance services, individuals in some affected areas may not have access to needed supportive caredcare that would increase their chance for survival.\\nFinally, how cultural practices and traditional African belief systems influence the prevention and control of Ebola must be considered in the fight against the Ebola epidemic in West Africa. Unlike the Western paradigm of disease, which is based on objectivity, the African concepts of disease are heavily influenced by mystical factors, the community, and the universe (Cheetham & Griffifths, 1982) . Beliefs in witchcraft, religion, and ancestral spirits as reasons for being ill are common among many Africans (Chukwuneke, Ezenonu, Onyire, & Ezenonu, 2012) . Western headlines have been filled with stories of sorcery, illusion, and curse as reasons for some families being decimated by Ebola.\\nStructural factors are those institutional and organizational health systems challenges that are caused in part by how health care in the countries are organized, financed, and delivered to the population. For instance, are the governments adequately funding health care in these countries? Although individual-level factors are essential for the prevention and containment of Ebola, structural-level factors are also critical.\\nCountries Emerging from Civil Conflicts. Structurally, all the countries at the epicenter of Ebola are emerging N u r s O u t l o o k 6 3 ( 2 0 1 5 ) 3 0 e 4 0 from civil conflicts with dysfunctional, fragile health systems. For instance, in Liberia, cultural, economic, educational, and health initiatives were all halted, with the exception of few international organizations providing health care to the general population (Buseh, 2008) . Rebels went on a rampage, looting and destroying clinics and hospitals. Many trained health care workers were killed and/or fled the country and have not returned. Health care systems in these countries remain fragile, and government spending on health care remains low. Ebola has exacerbated the problems of these fragile systems.\\nEndemic Multimorbidities and Competing Health Priorities. All the countries affected by Ebola also experience endemic multimorbidities with competing health priorities. In sub-Saharan Africa, infectious and noninfectious diseases (both emergent and re-emerging) continue to affect a large proportion of the population (Boutayeb, 2006) . The West African region is ripe for the coexistence of multiple health problems (e.g., very high infant and maternal mortality rates) as shown in Table 1 . Diseases that affect mostly the poor in many sub-Saharan African countriesdreferred to as neglected tropical diseases (parasitic worm infestations, schistosomiasis, river blindness, and so on)dcontinue to persist (Boutayeb, 2006) . The multimorbidity of diseases found in the region puts an extra strain on the health care systems of these countries with inadequate budgets.\\nPoor Public Health Infrastructure. Structurally, countries in the West African region lack the public health infrastructure for managing infectious disease outbreaks (Boozary et al., 2014) . In the affected countries, there is limited disease surveillance in place, and almost all health centers constantly face logistical issues. Ebola warrants full occupational protection, with the use of special equipment and clothing when caring for patients. However, even before the Ebola outbreak, there was a dearth of medical equipment and supplies in the affected countries. The lack of supplies and the reuse of equipment serve to increase the risk for medical staff, thus the unprecedented high number of cases and deaths from Ebola among health care workers in the affected countries.\\nAlthough individual-and structural-level factors are essential for the prevention and containment of the Ebola epidemic in West Africa, understanding the community-level factors is also essential. Individuals and institutions exist in communities. Many African societies and cultures pride themselves on a communal approach to health and survival. Thus, there is a synergistic effect between individuals and communities. When an increased number of individuals contract Ebola in a certain community, this can have an adverse effect on the well-being of that community at large. Several factors determined as pivotal community-level challenges to the prevention and control of Ebola are discussed later.\\nAfrican region, borders are porous and contiguous, and, thus, conditions are ripe for the disease to spread rapidly from one country to another. For many West Africans, the border is just an imaginary boundary; movements between countries are fluid and constant. For instance, it is very common for individuals living in Guinea to have family members living in Liberia or Sierra Leone and for people across the region to speak similar languages and/or belong to the same tribe.\\nMistrust of Government and Modern Health Care Services. Deep distrust of government and government health services exists in many sub-Saharan African countries. The mistrust of government in this region is especially pervasive in countries that have endured civil conflicts and for decades have suffered lies and deceit from their leaders. Trust is a critical component of public health because trust influences how people and a community respond to public health interventions and messages during complex emergencies (Meredith, Eisenman, Rhodes, Ryan, & Long, 2007) .\\nClash of Ethnomedicine with Biomedicine. The Ebola epidemic in West Africa has also revealed not only the inadequacies of modern health care systems in the affected countries but also the clash of biomedical systems with African traditional healing systems. In all sub-Saharan African countries, both systems exist in parallel and are not integrated. African traditional healing practices have existed for hundreds of years. For many individuals, especially in rural settings, their first line of health care is an African traditional healer. It is common for family members to seek health care from a traditional healer before finally seeking health care at a clinic or hospital. These ethnomedical practices are highly popular, have been used for centuries, and continue to be used in the treatment of both noninfectious and infectious diseases, including HIV/ AIDS (Chinsembu, 2009 ). In the absence of treatment or an efficacious vaccine for Ebola, it is understandable that many individuals in the region have turned to African traditional healers for help.\\nAlthough much of the information seen in the media has centered on the lethal nature of Ebola and its potential for rapid spread to Western countries, this type of narrative only reinforces existing global health inequities. Instead of viewing the Ebola epidemic in West Africa only from a problem-based perspective, it should be viewed through another lens, as is the case with the HIV/AIDS epidemic, which is now viewed as an opportunity for addressing global public health threats. At a minimum, in the N u r s O u t l o o k 6 3 ( 2 0 1 5 ) 3 0 e 4 0 age of international travel and commerce, diseases can spread very easily from one setting to another. There are several essential areas that can serve as opportunities for addressing the Ebola epidemic and future infectious disease outbreaks in sub-Saharan Africa.\\nEbola not only affects an individual, but it also affects the families of persons with the disease. Once a family member contracts Ebola, the chance of other family members contracting the disease increases. Thus, developing a family-centered approach to curtail the spread of the disease is essential. Lessons could be learned from the HIV/ADS epidemic in which embracing a family-centered approach to caring for individuals has been hailed as effective in the management and prevention of HIV infection (DeGennaro & Zeitz, 2009 ).\\nIt is particularly important to start the dialogue with community leaders and elders. African societies, especially in rural areas, are hierarchical in structure, and, thus, intervention aimed at finding lasting solutions to issues affecting communities at large must usually get the endorsement of the community leaders. In the prevention of Ebola, the use of good public health principles through incorporating community participation is essential. In a study exploring the value of community participation in disease surveillance in Niger, investigators found that the quality of surveillance on guinea worm control campaigns improved when community leaders were involved (Ndiaye, Quick, Sanda, & Niandou, 2003) . Community leaders will be more effective in mobilizing their people around the efforts being promoted by the ministries of health and international organizations.\\nThe Ebola epidemic is a public health problem that is difficult to address solely by one country. Governments in the affected countries do not have the capacity for delivering full and needed services to all people. Thus, the international community should assume some role in shoring up the affected countries' systems. Governments in the affected region constantly face competing needs and priorities and sometimes become easily distracted from daily governance and service provisions when large issues, such as an Ebola outbreak, can easily overwhelm the already fragile systems (Ross, 2013) . However, the sovereignty of the affected countries must continue to be respected even as the international community partners address the Ebola epidemic. The international community will do harm if the public perceives that their governments are inefficient in providing basic health services.\\nAlthough the Ebola outbreak in West Africa is unprecedented, this outbreak gives the international community an opportunity to work together to figure out ways to address infectious diseases in resource-poor settings, such as researching, developing, and testing possible vaccines and treatment. Thus far, there is no known drug or vaccine that can be used safely in human populations for the prevention and treatment of Ebola. Some promising agents are being developed by several companies, including GlaxoSmithKline. The National Institutes of Health is also working on an Ebola vaccine. Although there is an urgency to develop treatment and vaccines, clinical trials must be designed and conducted under straight ethical guidelines, balancing any potential harm against potential benefits accrued by the affected populations.\\nSeveral explanations are gleaned from the analyses of the Ebola epidemic in West Africa and reported in this article. First, Ebola is being presented as a fatal disease that spans global boundaries and has now spread to several Western countries, including the United States. Second, public health officials cannot afford to be complacent about the emerging and re-emerging infections. During the last 2 decades, the public health community has witnessed a series of emerging and reemerging infections (e.g., subacute respiratory tract infection, avian flu, H1N1, the Middle East Respiratory Syndrome, and now Ebola in West Africa), all of which have put the global public health community on high alert. A third perspective is that when infectious disease outbreaks occur in developing countries, these countries are unlikely to have adequately resourced and staffed health care facilities capable of addressing the spread of the disease and managing those who are already infected. International agencies should review existing pandemic plans in an effort to increase their efforts to control outbreaks in the future. Could the Ebola outbreak in West Africa be a prelude to bigger outbreaks to come, potentially a different type of virus, maybe a more lethal virus and one spread via the airborne route?\\nAlthough we used a problem-based perspective in presenting the challenges of addressing the Ebola epidemic in West Africa, we are in no way neglecting the strengths and resilience of the affected countries. Health care professionals in the United States, including nurses who have returned from working in the West African region, could be excellent sources of ideas about how nurses and other health care professionals in the United States can become more engaged in addressing the epidemic in West Africa. A good place to start would be for health care workers to understand that although these populations in West Africa are vulnerable to disease, poverty, and other adversities, they are also very resilient. Thus, in the following section, we briefly discuss the strengths of the general population and health care workers in the West African setting.\\nAlthough Ebola has been raging since March 2014 in West Africa, it was not until mid-August that the disease began to make headline news in the United States when two Americans working at a hospital in Liberia contracted the virus and had to be flown back to the United States. Since then, the news coming out of West Africa has continued to be bleak, almost entirely focused on the Ebola epidemic, with the potential for spreading and more deaths to occur. Sometimes the news seems sensationalized, presenting a doomsday scenario that entire populations will be decimated if nothing is done to halt the disease. Yet, the countries at the epicenter of the disease (Liberia, Guinea, and Sierra Leone) are no strangers to tragedies. For instance, in the 1990s, civil conflicts erupted in both Liberia and Sierra Lonedsenseless wars that led to hundreds of thousands of lives lost. Today the conflicts in these countries are over, and almost every Liberian, whether they were in Liberia during the active conflict or abroad, has some painful story to tell.\\nHistorically, although much research has stressed the emotional deficits and vulnerabilities of populations in sub-Saharan Africa, less research has illustrated their strengths, resources, and resilience (Almedom, 2009). Black people around the world and as a racial group have thrived in the wake of traumatic experiences (Bryant-Davis, 2005) . Although the media in the United States is fraught with news about Ebola coming to the United States and the panic calls from some politicians to close the borders and not allow West Africans from the three most affected countries to enter the United States, Africans in the affected countries continued to scramble for resources in an effort to survive daily. Generally speaking, Africans are a resilient people. As the international community continues to work on addressing the Ebola epidemic in West Africa, we must not lose sight of the strength and will of the affected populations to survive in these countries despite the meager economic and health resources available, in particular considering their histories of suffering colonial injustices and their experiences of racial bias.\\nEven when access to modern health care may be impossible, African communities use communal approaches in devising ways to address their own problems when tragedies emerge in their communities instead of waiting for people from the outside to rescue them. Although many Africans increasingly embrace the Western way of life, they are also highly spiritual, relying heavily on traditions, rituals, and the spiritual guidance of their ancestral spirits for day-to-day survival (Some, 1999) . Their ability to navigate both Western and African traditional healing systems is an asset that should be embraced.\\nLike most of sub-Saharan Africa, in the West African Ebola-affected countries, there are many factors that increase the population's vulnerabilities to disease, not the least of which is poverty. In a review of the literature on the construct of vulnerability, poverty, and health in sub-Saharan African populations, many of the studies focused mostly on the deficits of the populations, with limited or no attention paid to the assets and resiliency of local populations that have sustained them for centuries (Almedom, 2009) . However, in an issues brief examining health and social systems, Almedom (2009) argues that focusing on the observed weaknesses of systems of African countries overshadows the real and potential strengths that need to be harnessed for the development of community discourse around critical health and social issues. Because resilience is a dynamic process, it will be important to explore factors that lead to the endurance and resiliency of West Africans during complex emergencies, as is the case now of the Ebola epidemic. As Almedom (2009) sums up, \\\"Focusing on the strengths of formal and informal African institutions so far has served to understand the dynamics of adaptive processes that sustained lives and livelihoods\\\" (p. 6).\\nStill, the pestilence of Ebola, which has killed thousands, destroyed families, and created orphans, has crippled not only the health care systems of these countries but also their economies, in particular the educational and agricultural sectors. Schools remained closed, many marketplaces are closed, and farming activities have decreased or halted in some regions. Many are confused and fearful about what is happening. Some are angry that the international community was painfully slow to respond to the epidemic. Many people, however, are enduring the disease with the same resignation as they did when faced with challenges during the civil war and other adversities. Ebola too will end regardless of how long it takes to mitigate the spread of the disease. Populations in the affected countries are coming to terms with the Ebola epidemic and increasingly learning about its seriousness; they are doing everything they can to protect themselves and their families from contracting the disease. Although the epidemic is reported to be stabilizing, the disease is far from over.\\nLiberian Head of State and Nobel Peace Prize winner President Ellen Johnson Sirleaf explained the situation on October 19, 2014, in a \\\"letter to the world\\\" broadcast on the British Broadcasting Corporation News Hour.\\n. Ebola has managed to bring my country to a standstill. . There is no coincidence Ebola has taken hold in three fragile states e Liberia, Sierra Leone and Guinea e all battling to overcome the effects of interconnected wars . the war destroyed our public infrastructure, crushed our economy and led to an exodus of educated professionals. A country that had some 3,000 qualified doctors at the start of the war was dependent by its end on barely three dozen . Ebola is not just a health crisisd across West Africa, a generation of young people risk being lost to an economic catastrophe as harvests are missed, markets are shut and borders are closed . The time for talking or theorizing is over (Sirleaf, 2014) .\\nFrom the onset of the Ebola epidemic in West Africa, health care workers, including physicians and nurses struggling to defeat the epidemic have in many cases, also became frontline victims of the disease. In many ways, the advent of the Ebola epidemic in West Africa has intensified and broadened the challenges faced by health care professionals, including nurses, working on the frontlines in developing countries. The situation in West Africa is compounded by the lack of health care workers skilled in the prevention and treatment of infectious diseases. Despite the bravery and resilience of the nurses and the efforts of other health care workers in the affected countries, Ebola has threatened the productivity of nurses and led to increased attrition and lower morale. If these structural factors are not addressed, the quality of care for Ebola patients and patients suffering from other conditions could be even further threatened.\\nAccording to reports from WHO, the initial infection rates showed how health care workers at the very heart of West Africa's fight against Ebola were adversely affected, many of whom were poorly equipped, poorly paid, and insufficiently prepared and became some of Ebola's immediate causalities (WHO, 2014b) . Although the exact numbers of health care workers who have contracted Ebola and who have died from the disease is unknown and inaccurate, Ebola has taken the lives of prominent doctors and nurses in Sierra Leone, Liberia, and Guinea, depriving these countries not only of experienced and dedicated medical care but also of inspirational national heroes (WHO, 2014b) . According to WHO, as of the end of November 2014, in countries at the epicenter of the disease, a total of 622 health care workers are known to have been infected with Ebola; 346 of them have died (WHO, 2014a).\\nHealth care workers in the affected countries have cared for patients and their families in difficult health care environments, which in some cases increased their risk of contracting Ebola. These are courageous health care workers; despite not knowing what they were dealing with, most did not run from the challenges. Rather, they faced the EVD head on with bravery and resilience. Despite working in areas with severe workforce shortages, inadequate and dysfunctional facilities, and the propensity to be at increased personal risk for contracting Ebola, health care workers in the affected countries are working tirelessly to save the lives of their people while helping to reduce the spread of the disease. In Liberia, Dr. Samuel Brisbane, director of the emergency department at Monrovia's John F. Kennedy Memorial Medical Center, was the first Liberian doctor to die in the country's Ebola outbreak. Dr. Brisbane's colleagues, Dr. Josh Mugele from Indiana School of Medicine and Chad Priest from Indiana School of Nursing, paid tribute to him in an essay in the September 2014 edition of The New England Journal of Medicine and in an interview on National Public Radio, saying that Brisbane's death was a \\\"selfless act\\\" because he did not have to treat those patients (Mugele & Priest, 2014) .\\nWe believe our friend (Dr. Brisbane) died a good death d as did all the nurses and doctors who have sacrificed themselves caring for patients with this awful disease . He didn't have to be a doctor at that stage of his life. But he kept doing it even though he knew [Ebola] was very contagious and he had a high likelihood of getting it. Dying was a selfless act on his part . I think a good death is not about how you die. It's about the quality of the life you left behind. And that's not an end-of-life discussion. It's a beginning-of-life discussion. (Mugele & Priest, 2014) .\\nIn all three countries seriously affected by Ebola, there are many nurses and doctors who have done similar great work and would be considered heroes and heroines in sacrificing their lives in caring for patients with a highly infectious diseasedmany of whose stories have not been told. We highly applaud all the doctors and nurses in the affected countries who have been working and continue to deal with patient needs that so vastly exceed the resources available in these low-resourced countries. Whether or not we would consider the loss of the lives of hundreds of nurses and doctors fighting the Ebola disease in West Africa a \\\"good death,\\\" probably more relevant questions for nurses and other health care workers in the United States to ponder as we push for global health equity are the following: Why has it taken so long to discover a treatment and/or vaccine for Ebola even after it has been around for over three decades? Are the deaths of health care workers and the general populations in West Africa from Ebola preventable ones? Would a treatment and/or vaccine for Ebola have been discovered a long time ago if wealthy Western populations were affected with such a deadly pestilence?\\nAs for long-term prospects, Farmer (2014) insists that the formula for success against Ebola and threats like it is to \\\"invest in a comprehensive model of prevention and care for the poor\\\" (p. 38). Without intact public health and health care delivery systems in all countries, little can be done to control outbreaks or offer the chance of recovery to those stricken. Improved clinical care across the world is necessary. Facilities and personnel capable of offering early diagnosis and aggressive supportive care, including fluid resuscitation, electrolyte replacement, and blood products, may be able to increase the survival rate of those with the EVD to 90% (Farmer, 2014) .\\nWe provide some of these policy areas that will require efforts from the affected countries, regional West African organizations, and the international community.\\nAddressing the Socioeconomic Issues of Poverty, Food Security, and Livelihoods Including Improving Access to Clean Water and Sanitation Facilities People living in poverty are at an increased risk of experiencing deleterious effects from Ebola. Entrenched poverty makes it impractical for families to purchase their food and other supplies in advance and stockpile when quarantine orders are issued by the government. In rural settings, it is reported that famers are no longer engaged in communal farming on as large a scale as before. Without large farms, segments of the population are likely to experience hunger even when Ebola is halted. Providing seeds during the next farming season may be an important policy issue to consider helping address future hunger in the populations of affected countries. The United States Agency for International Development and the World Food Program are scaling up efforts in providing support for individuals receiving treatment in Ebola treatment units, Ebola survivors who are discharged from treatment centers, the families of people infected with Ebola, and communities with widespread transmission (United States Agency for International Development, 2014).\\nEbola is a disease that calls for the complete overhaul and transformation of African health systems. The countries affected by the epidemic are all emerging from civil conflicts that had an effect on their health care delivery systems. Although much effort has been made during last 2 decades to build the health care systems in sub-Saharan African countries, the Ebola epidemic in West Africa underscores the need for developed countries and international organizations to provide foreign aid support repairing and building the capacity of nonfunctioning health care systems. As these systems are restructured, they should include resources to strengthen both the provision of basic primary health care as well as to be able to handle infectious disease outbreaks.\\nViewed within a global context, the current Ebola outbreak in West Africa and future outbreaks will require coordination among the affected countries, governments, and health agencies as well as a myriad of partners dedicated to international health. In addressing this type of epidemic, there are many stakeholders and actors. Thus, coordination may not be clear, and it could be a logistical nightmare to unify resources in the fight against the disease. Addressing Ebola in the affected countries not only includes the involvement of doctors and nurses but also involves issues of transportation, food, medical supplies, and the development of telecommunications networks as well as addressing other logistics.\\nThe health care systems in sub-Saharan African countries face serious challenges, including the lack of the technologies seen in the developed countries to aid in the diagnosis and management of diseases. Information technology systems are purported to improve access to care, lower costs, and create efficiencies in the overall system (Cline & Luiz, 2013) . In all the countries affected by Ebola, road networks are poor, electricity is limited, and medical records are still kept in the old-fashioned way with papers and folders. As part of the scaling-up process in the fight against Ebola, we propose the development and integration of information technology systems in the affected countries' health systems. Technology can also be used to enhance communication between the central unit of command in charge of coordinating Ebola in capital cities and outlying health care centers in rural villages and towns.\\nIn many of the affected countries, nurses are frontline workers. Many are trusted and respected members of local communities and thus are considered reliable resources for health management in both formal and informal settings. West African nurses speak the local languages and, thus, can be tasked with providing factual and relevant information inside and outside of their communities about this alarming and disturbing disease. Nurses can help inform the public about what information is necessary for individuals suspected of being exposed to Ebola so they can receive the most prompt and appropriate health care. In the case of the affected countries, nurses can take to the lead to help dispel rumors and address social stigma and common misconceptions that a frightened population may be hearing through news outlets. Community forums can be held in which nurses could provide real-time information and hand out resources.\\nDeveloping multipronged research agendas aimed at addressing Ebola in the region is important. First, scientists' quests to increase our understanding of the epidemiology and pathogenesis of the Ebola virus will help us prevent and manage the disease. Along with such epidemiological research, policy makers should also conduct research aimed at understanding the barriers to addressing Ebola. Whether the current efforts being mounted to address the Ebola outbreak in West African are successful will depend heavily on engagement and collaboration with the people in the affected communities. As far as we know, there are no reports about local populations' beliefs and attitudes about Ebola and what they think about the response efforts from their national governments and international organizations relative to the Ebola outbreak. This is an opportunity to assess the perceptions of those who have the disease as well as those at risk of contracting it, including family members. This is especially important in the rush to develop effective treatment and a vaccine. In one of the few studies exploring the cultural contexts of a past Ebola outbreak in Northern Uganda, investigators found that indigenous epidemic control measures and some cultural practices (e.g., burial practices) amplified the outbreak (Hewlett & Amola, 2003) . Hewlett and Amola conclude that consideration must be given to the narratives of local populations relative to the Ebola outbreak. The population's cultural beliefs and local population characteristics must be taken into account in developing appropriate and accurate targeted messages in conjunction with local community leaders.\\nWe have discussed at length the need for the international community to increase investment in strengthening the health care systems of sub-Saharan African countries. This is especially important in countries that are emerging from civil conflicts. Without sustained funding, it will be difficult to bring Ebola under control in the West African region. Globally, much progress has been made by countries around the world in improving the health and well-being of their people in the 20th century. However, disparities continue to persist in the developing countries in sub-Saharan Africa. Ultimately, when viewed from a global health perspective, Ebola is suggestive of the increasing disparity in global health care between developed and developing countries. Only when we are able to have honest conversations and hold our leaders and policy makers accountable will global health issues be made a priority agenda.\\nIn this article, we identified and discussed some critical challenges and opportunities related to the current Ebola epidemic in West Africa. We also proposed specific policy agendas that should be undertaken by the affected countries and the international community. Among the lay public in West Africa and abroad as well as among scientists, uncertainties also persist, and myths about the disease continue to be widespread. It is evident that there are numerous issues to unravel about this disease. The fight to contain this virulent disease has taken on global urgency and is now being viewed through the lens of global public health. Absolute human power is what it will take to control this epidemic. How scientists, policy makers, and health care workers, including nurses, work at disentangling the multicomplex and unknown factors surrounding Ebola will be important in containing this disease as well as future outbreaks.\\nScientific advances are being made in the quest for treatment and vaccines. However, in pursuing these advances, it is important to encourage global public health research efforts that combine integrative, topdown and bottom-up approaches to disease prevention and care with consideration of the knowledge and cultural sensitivity of local populations. An integrative approach will go a long way in gaining the trust of those afflicted by Ebola. An integrative approach will also have greater potential for preventing the spread of Ebola to other sub-Saharan African countries. If policy makers become complacent and ignore the interconnected challenges and opportunities outlined and described in this article, they run the risk of seeing current efforts aimed at addressing the spread of the disease reversed. Today, it is Ebola in West Africa; tomorrow it could be another disease in another sub-Saharan African country, Latin America, Asia, or even the United States. How can we risk not preparing and developing measures aimed at addressing these emergencies in advance? More important, do wealthy countries with health resources and public health expertise have any obligation to help stem the tide of this pestilence that is ravaging poor and vulnerable populations in West Africa? Readers can ask these questions, but failure of the global community to jointly and vigorously address the current Ebola epidemic in West Africa by assisting the affected countries to rebuild their health care systems will only serve to contribute to the spread of emerging and re-emerging infectious diseases globally. \",\n          \"iNTRODUCTiON The adaptive immune system is composed of B and T cells that form a highly selective guard against evolving pathogens. The foundation of the adaptive immune response is based on the enormous diversity of T and B cell antigen receptors that can recognize epitopes from a near infinite number of different internal and external antigens. This profound diversity of T (TCRs) and B cell receptors (BCRs) is generated by V-D-J gene recombination of the TCR/BCR locus and subsequent somatic hypermutation and class-switching recombination of B cells after antigen stimulation. Thus, study of the immune repertoire, portrayed as the antigen-specific information within lymphocytes, has been a key to understanding the response of adaptive immunity during infection.\\nDespite extensive efforts using traditional techniques, analysis of the immune repertoire with high resolution has remained difficult. Several sequencing strategies, for example, Sanger sequencing, have been implemented to determine cDNA segments encoding variable regions of immunoglobulin (or TCRs) (1, 2) . However, these low-throughput techniques lack the power to provide a broad picture of the full immune repertoire. During the past two decades, however, technical advances in high-throughput sequencing (HTS), also known as next-generation sequencing (NGS), along with evolving bioinformatic and statistical tools, have provided a new approach capable of analyzing the immune repertoire at the single sequence level. These methods create an unprecedentedly high-resolution picture of the immune repertoire and also provide massive data that cover each lymphocyte from the sample, in theory, dispensing with limitation of sequencing quantity (3) .\\nConsidering the extremely important role of the adaptive immune system in defending against infectious agents, HTS has great potential to aid in the discovery novel infectious agents and also offers new approaches for antibody or vaccine development. In this review, we introduce the implementation of HTS to the study of the immune repertoire and review the associated bioinformatic tools required for data processing and analysis. We then focus on the success of this technology in facilitating the exploration of infection-related immune repertoires for clinical diagnosis, treatment, and prevention.\\nAmazing diversity makes the immune system the most effective system to fight against a broad scope of disease causing pathogens. This repertoire is generated by a complex series of genetic events (4) . For T cells, the variable region of each TCR chain consists of three complementary determining regions (CDRs) and four frame regions (FRs) . CDRs are the variable portion of the receptor and determine the antigen specificity. While CDR1 and CDR2 are formed by variable (V) gene, CDR3 is generated by random selection and recombination of variable (V), diversity (D), and joining (J) gene segments in the heavy chain (V and J region gene segments in light chain) (5, 6) (Figure 1) . Thus, CDR3 is the most diverse component of a receptor, which binds MHC molecules and (or) antigens. Construction of the TCR with an alpha chain and a beta chain is also a process that contributes to receptor diversity.\\nThe formation and revision of the T and B cell lymphocyte receptor repertoire is a highly dynamic process. The number of each lymphocyte clone changes dramatically and depends on cell specificity and the history of antigen exposure. When encountering exogenous antigens, T cells that express receptors capable of binding to a specifically compatible peptide-MHC (pMHC) complex will expand, resulting in a massive population of antigen-specific T cells that initiate the adaptive immune response (7) (8) (9) (10) (11) . This antigen-driven proliferation process of T cells is distinct between CD4+ and CD8+ T cells after initial antigenic stimulus. Although these two types of T cells show comparable protein expression, proliferation rate, and transcriptome features after 2 days of non-infective stimulation, subsequent division of T cells differently depends on continuous existence of self-pMHC complexes. CD4+ T cells proliferate in a limited pattern, and its subsequent response requires persistent stimulation from antigen-presenting cells. CD8+ T cell is \\\"programed\\\" to extensive expansion after short stimulation even when transferred into antigen-free hosts (12) .\\nThe post-antigen stimulation response of B cells is more complicated because it is accompanied by somatic hypermutation and class-switch recombination that offer additional diversification of the B cell repertoire (6) . Somatic hypermutation is the process of introducing point mutations at CDR1, CDR2, CDR3, and FR3 to produce B cells of higher affinity to target antigens. These higher affinity clones are then selected and expanded, which is called affinity maturation. Additionally, in class-switch recombination, the gene loci encoding the C region of BCRs are excised and replaced by a series of new constant gene segments, resulting in functional differences of IgG, IgE, or IgA that participate in different immune mechanisms during pathogen elimination.\\nTraditional Strategies for Studying the immune Repertoire Prior to HTS, many strategies were developed to explore postinfection immune repertoires (13). Immunoscope spectratyping has been used to investigate TCR/BCR repertoires since the 1990s (5, 14) . In this technique, using one (for B cell) or more (for T cell) V or J gene specific primer pairs, the length of CDR3 can be determined (15) . CDR3 length in healthy population shows a bellshaped pattern, indicating a polyclonal repertoire. However, the unusual peaks in infected patients imply a perturbed oligoclonal repertoire with clonal expansion. As such, CDR3 spectratyping provides robust information on the complexity and stability of circulating T/B cell repertoires and insights into the immune repertoire after infection (16) (17) (18) (19) (20) . Even if it is relatively easy and cheap, the nucleotide sequences of CDR3 remain obscure, and the extent of heterogeneity within a particular CDR3 length cannot be assessed.\\nDetailed nucleotide sequences of gene segments encoding the variable region can be determined by traditional DNA sequencing techniques such as Sanger sequencing (21, 22) . Flow cytometry and CDR3 spectratyping help to isolate T/B cells of interest, which complements weaknesses of Sanger sequencing in quantity limitation. Single-cell sequencing is able to identify sequences of several B cells that produce monoclonal antibodies specific to certain virus, which contributes greatly to analyzing genetic features of the antibodies in the process of antibody discovery (2, (22) (23) (24) . After collecting peripheral blood samples, the B cells or memory B cells are isolated and immortalized to produce antibodies. According to the HAI titers and neutralizing titers determined by ELISA, the virus-specific B cells can be identified, which helps to narrow B cell candidates for sequencing. These functional test-based antibody discovery strategies are successful but laborious. Despite this, these strategies are well designed for targeted antibody searching; however, it is insufficient for creating a high-resolution picture of the human immune repertoire.\\nHigh-throughput sequencing has recently become a novel and powerful tool to investigate the immune repertoire. The depth and comprehensiveness of high-throughput immune repertoire sequencing are greater than ever, and the enormous sequencing data of disease-specific TCR/BCR clones provide great potential for the revealing dynamic changes in clonality during infectious states.\\nEstablishing a lymphocyte repertoire database starts from sample collection from carefully selected populations and isolation of interested T cell or B cell subgroups. Due to the well-acknowledged heterogeneity of TCRs and BCRs between individuals, longitudinal studies tracking dynamic alterations in certain population help to reduce difficulties in data interpretation at unraveling the infection course. Classification of subgroups of T cell and B cells, e.g., naive and memory T/B cells, CD4+, and CD8+ T cells, is necessary if distinct behavior of these subgroups is considered in detail.\\nThe methodology of library preparation and amplification need careful design since it affects accuracy of the ultimate sequencing data. Due to the difference of V and J gene segments, a common primer does not apply to sequencing of CDR3. Multiplex PCR is capable to amplify multiple loci simultaneously and, however, is likely to introduce bias. It is because of non-specific amplification, primer-dimer formation, and uneven reaction conditions. More precise and quantitative multiplex PCR may be achieved through primer concentration adjustment and bias filtering using amplification bias among the templates as controls (23) . Another alternative PCR method is 5\\u2032RACE PCR, which provides a less biased PCR library using primers that bind downstream of the variable domain (24) .\\nSequencing techniques are evolving continuously to be deeper and more precise, and there are three prevalent HTS platforms available today. The comparison of mechanisms, sequencing depth, and other critical features of each platform is shown in Table 1 . The Illumina and Roche 454 platforms have been most commonly used during immune repertoire analysis. The outputs of each platform must be analyzed with caution because of notable platform-specific sequencing error (25) (26) (27) (28) . Insertions and deletions of nucleotides, resulting from imperfect interpretation of homopolymeric stretches, are considerable for the Roche 454 platform (29) , while substitution errors are predominant in Illumina platform (30, 31) . The overall error rate of Illumina platform is lowest while that of Ion Torrent is highest among the three (32) . In an attempt to correct sequencing errors, three algorithms are most commonly used including k-mer spectrum, multiple sequence alignment, and suffix tree (26) . Based on (27) . PCR and sequencing errors inevitably result in overestimate of repertoire diversity. The common statistical strategy for both PCR and sequencing error removal is eliminating low abundance and low-quality sequences (with low Phred score), but it leads to a great loss of sequencing information. To rescue these sequences, low-quality CDR3 sequences with no more than three low-quality nucleotides can be mapped to \\\"core clonotypes\\\" derived from high-quality sequences with allowed mismatches at low-quality position. Then, the low-abundance core clonotypes are merged with the high-abundance core clonotypes with less than three allowed mismatches at V (\\u22642 mismatches), D (\\u22641 mismatches), and J (\\u22642 mismatches) gene segments to correct PCR errors. This integrated algorithm based on sequence quality abundance could efficiently correct artificial errors and avoid information loss, thus providing more reliable estimation of repertoire diversity (28) . Unique molecular identifiers that label each starting molecule help to reduce both PCR and sequencing errors (24, 41) . Combined with this experimental strategy, molecular identifier groups-based error correction (MIGEC) corrects PCR and sequencing errors more efficiently than other quality-and frequency-based strategies (42) .\\nDetermination of the V-D-J gene segments from which the CDR3s are rearranged, as well as identification of point mutations, is often achieved using the ImMunoGeneTics database (http:// www.imgt.org) (43) , despite the controversies about its validity (44) (45) (46) . New V-D-J gene annotation tools based on various algorithms are reported, such as IgBLAST (47), iHMMune-align (48) , and Decombinator (49, 50) ( Table 2 ). In addition, many integrated bioinformatic tools (MiTCR, LymAnalyzer, Change-O, etc.) for data processing are developed recently (51-62) ( Table 2) , which provides various statistical approaches for diversity estimation, repertoire comparison, clustering analysis, and somatic hypermutation analysis ( Table 3) . Despite these tools, standardized bioinformatic analysis and visualization strategy is lacking, which remains the main obstacle for comparison of researches from different investigators.\\nHigh-throughput sequencing techniques have revolutionized the study of the immune repertoire. Utilizing HTS, many important insights into mechanisms of immune response have been gained. It is also the cornerstone for potential clinical applications of repertoire analysis, including identification of diagnostic biomarkers, design of therapeutic antibodies, and development of new vaccines.\\nEstimating the diversity of a TCR/BCR repertoire is necessary for estimating the theoretical size of the repertoire and for tracking changes in clonal populations during the clinical course of infection. Several different methods may be used to describe the diversity of lymphocyte repertoires at different levels -VDJ recombination diversity (90), Simpson diversity index (91, 92) , and some non-parametric methods. Decrease in the overall diversity of the immune repertoire have been observed after various antigen exposures, including HIV, influenza, and human herpes virus, which implies expansion of particular T/B cell clones (67, 88, 92, 93) . Our group compared changes in the diversity of the TCR beta chain and BCR heavy chain after H7N9 virus infection. Interestingly, these results show that the diversity of the BCR heavy chain starts to increase 2 weeks after H7N9 infection, while the TCR beta chain repertoire continues to contract. In addition, a more diverse BCR repertoire and a less diverse TCR beta chain repertoire in convalescent phase correlate with improved prognoses, implying differences in the response process of humoral and cellular immunity.\\nThe immune response to vaccination has been used as an ideal model for antibody repertoire research due to the convenience drawing blood samples at well-defined time points (94) . Studies using vaccines, such as influenza and TT, have revealed dynamic changes in the size and diversity of antibody repertoires before and after antigen stimulation (41, 90, 95, 96) . Comparison of post-vaccination responses suggests divergent repertoire properties among individuals, different age groups, and successive immunization of the same individual with different influenza vaccines (TIV and LAIV) (66, 90, 95) . The maximum clonal response has been found to occur 7 days after vaccination, but the magnitude of response varies between individuals despite an identical immune challenge, which may be influenced by previous exposure, age, and other concurrent immune responses.\\nIn addition, study of B cell memory can be achieved by repeated sequencing of samples taken from the same individual in separate immune responses to an antigen. Vollmers and colleagues identified a group of B-cell clones as a recall response to two substantially different vaccine compositions, implying the possibility of identifying cross-specific antibody using repertoire analysis (41) . Immune responses recorded by sequencing data have also been useful in testing the role of adjuvants in eliciting broad-spectrum antibodies. Wiley and co-workers tested the immune response of mice immunized with malaria vaccine by analyzing IgG repertoires. They found that TLR agonist used as adjuvant increases the diversity of IgG variable region, which is related to improved ability of the antibodies to recognize a broad spectrum of epitopes (97) . These studies exemplify a new level of details assessing vaccine response and pioneered HTS implications in vaccine design.\\nIn infected patients, antigen-specific T/B cell repertoires form in response to antigen exposure in both circulation and peripheral tissues. Immune repertoire sequencing provides broad information including crucial antigen-specific clones, which have the potential to halt the spread of pathogens (98) . Diagnostic marker discovery using sequencing data relies on these antigen-specific clones with stereotyped features in the post-infection repertoires. These features are assessed at different levels such as gene rearrangement, identical or similar CDR3 sequence overlap, and certain CDR3s length. After influenza H1N1 vaccination, the dominant clonotype of Ig heavy chains has the same V-J gene rearrangement, CDR3 length, and somatic mutation position in CDR1 and CDR3 with previously reported influenza antibodies (66) . However, in this study, the convergent dominant sequence is only found in one individual. Further researches in a broader population including non-dominant sequences are needed. A more successful example is reported in Ig repertoires related to dengue virus infection. Using cross validation and other approaches, stereotyped CDR3 sequences or CDR3 lengths that have high prevalence in the acute dengue samples are found to be specific to acute dengue infection, which are absent or of low prevalence in healthy and post-convalescent population (88) .\\nIdentification of pathogen-specific sequences also helps in differential diagnosis between infectious and non-infectious diseases. According to Dziubianau et al., comparing PBMCs-derived T cell clonotypes specific to a given virus with T cells from different origins (allograft-derived and urine-derived lymphocytes) provides a new methodology for differential diagnosis of two post-transplant complications -BKV-associated nephropathy and acute cellular rejection, which shows a glimpse of applications of T cell sequencing in diagnosis (99) . In addition, a recent study searched sequencing data for CDR3 amino acid motifs that have been reported to be specific for a particular pathogen and succeeded in identifying CDR3 sequences identical or similar to these motifs in post-vaccination volunteers (100) . According to these results, interestingly, it is low frequency sequences that possess the probability of becoming promising biomarkers instead of the dominant ones.\\nHowever, immune responses show dramatic differences in CDR3 sequences responding to same pathogens across individuals and age groups. This intrinsic divergence between individuals is the major obstacle in finding \\\"public\\\" sequences as optimal biomarkers. Nevertheless, we hold a promise for the application of HTS data in differential diagnosis because it provides a large number of candidate sequences for biomarker investigation. Instead of using single biomarkers such as PSA or AFP in diagnosis, a combination panel of selected sequences may establish a pathogen-specific sequence library for diagnosis, which holds the potential of unprecedented sensitivity and specificity.\\nRecombinant monoclonal antibodies have great potential in the treatment of specific infections. In recent years, several strategies, including phage display libraries, single B cell expression, and B cell immortalization, have been used to discover antibodies against specific antigens (101) . HTS of the antibody repertoire, combined with subsequent bioinformatic tools or traditional screening tools, has facilitated the identification of antigenspecific sequences (102) (103) (104) (105) .\\nThe methodology predicting antigen specificity completely from analysis of BCR sequences has not been possible yet, albeit it has considerable potential in immune repertoire studies. Nevertheless, there have been many efforts made in mining the HTS data for functional antibodies (106) . A relatively direct method for identifying such sequences is based on the similarity of amino acid sequence to previously reported antibodies. Researchers have successfully found sequences of high identity with the broadly neutralizing antibodies and strain-specific antibodies from established antibody repertoires of patients with influenza infection or vaccination (66) . Some of these sequences have proven to have neutralizing activity, validating the potential of deep sequencing-based antibody identification. Success of this work also suggests the possibility of monoclonal antibody synthesis without cell cloning for treatment (66) . Furthermore, another method using the frequency rank of heavy chain and light chain sequences to predict the function of antibody sequences has been reported successful in mouse models (107) .\\nExciting work by Kwong and coworkers demonstrates the feasibility of identifying neutralizing active clones through bioinformatic analysis from HIV patients (108, 109) . They established several steps for interrogating variants of known neutralizing antibody classes from HIV-infected patients, with or without previous knowledge if the patient had antibodies belonging to this family. First, the heavy or light chain sequences, derived from the germline IGHV or IGLV gene same as template antibody, are isolated from a new donor. Then, these sequences were compared with the germline gene for \\\"divergence\\\" and the template antibody sequence for \\\"identity, \\\" generating a contour plot called \\\"divergence/identity plot. \\\" The sequences segregate into clusters in the plot, from which the high divergence and high identity sequences were selected as candidates for neutralizing antibodies. This process is called \\\"Grid-based strategy. \\\" Then, the germline sequence, candidate sequences, and template antibody sequences of the same class are merged to build a phylogenetic tree rooted by the template sequence, which is called \\\"cross-donor analysis. \\\" The heavy chain sequences from new donor clustered in the subtree of the template sequences are then expressed with template light chain to generate antibodies. Most of these sequences have neutralizing activity to HIV-1. The outstanding efficiency of this method was also demonstrated when compared to sequence-based strategy and prevalence-based strategy (108, 109) . Lu et al. also validated this phylogenetic method in identifying functional anti-Staphylococcus aureus antibodies (110) . These strategies start from previously reported antibody sequences. However, such antibody sequences are not always available, especially during poorly characterized viral infections such as H7N9.\\nPairing the heavy and light chains as an integrated antibody has been another challenge for HTS-based immune repertoire analysis. In most cases, researchers only focus on the heavy chain, which causes a critical loss of antibody integrity and leads to problems in following synthesis of artificial monoclonal antibody. Two strategies have been reported to correctly pair the heavy chain and light chain sequences based on the frequency or evolution models. Reddy and colleagues (107) have pioneered pairing based on the frequency ranks, using plasma cells isolated from bone marrow of immunized mice and matching the two chains of similar rank order. Monoclonal antibodies expressed in this way did show antigen specificity. Due to the linkage of heavy chain and light chain as an integrated protein, their evolution undergoes the same enzymatic mutation process, and they evolve together to bind the same antigen with high affinity. Based on this theoretical foundation, phylogenic analysis has been used as another method to compare the evolutionary topography of the heavy chain and light chain after bioinformatic identification of transcripts related to a known HIV neutralizing antibody (109, 111) . Reconstituted novel antibodies consist of phylogenetically matched chains showing similar neutralizing function but less auto-reactivity compared to the mismatched ones.\\nSeveral groups have recently achieved advances in the technology of paired sequencing of antibodies. Single-cell PCR has been utilized to create a two-dimensional bar-coded primer matrix to link two chains of the BCR (112) . Using this technique, Busse and coworkers analyzed paired sequences of over 46,000 B cells in one experiment and accomplished subsequent antibody gene cloning and expression. At the same time, Turchaninova and coworkers performed pioneering research in emulsion-based technology for sequencing antibody repertoires of paired chains (113) . They used water-in-oil emulsions for cell-based overlap expansion RT-PCR, although its yield was relatively low yield. Another high-throughput paired sequencing method by DeKosky et al. used micro well plates to isolate B cells and magnetic beads to capture mRNAs (114) . Very recently, DeKosky's group combined and improved these previous techniques, and developed a costeffective and efficient methodology to establish a more precisely paired repertoire (115) .\\nPredicting T cell specificity based on TCR heterodimer sequence is more difficult than antibodies because of the highly variable nature of each of the components of the TCR-peptide-MHC complex (116) . Due to the challenges posed by the highly variable CDR3 loop of the TCR and the complexity of predicting protein-protein interactions (117, 118) , experimental functional tests for mining antigen-specific T cells might be a more fruitful approach (119).\\nRecent advances in HTS-based antibody sequencing may provide the biggest benefit for the field of vaccine development. Over the years, efforts to elicit protective immune responses to HIV by immunization have not been successful. During acute viral infections, high-affinity neutralizing antibodies develop in just weeks. However, generating effective broadly neutralizing antibodies during chronic infections, such as HIV, takes significantly longer time. Furthermore, the neutralizing power of these antibodies is often variable due to impairment of the host immune function, unusual features of Env, and co-evolution of the virus in response to the host antibody response (120, 121) .\\nDeep sequencing analysis has identified rare variants of known HIV-neutralizing antibodies and has elucidated the ontogeny of these neutralizing antibodies (108, 109, 111, 122) . These findings have cast a light on antibody guided vaccine development. In following studies, the HTS-based phylogenetic strategy greatly facilitated study in co-evolution of neutralizing antibodies and virus mutants (123) . Combined with long-term follow-up studies, these results illustrate how mutations in some sites allow the virus to escape some neutralizing antibodies, and how the virus, with the help of secondary neutralizing antibodies, becomes sensitive to the neutralizing antibody (98, 124, 125) . These studies suggest a promising pathway to elicit broadly neutralizing antibodies by sequential immunization with selected immunogens (123, 126) . Furthermore, structure studies of the neutralizing antibody family provide candidates for future vaccine designs (127) .\\nHigh-throughput sequencing has been a breakthrough technology for the study of the immune repertoire and has already had a profound effect on our knowledge of the immune systems physiology during health and disease. In particular, HTS has transformed our understanding of immune repertoire formation during infection, malignancy, and autoimmunity. Advances in this filed of research will rely on progress of similar laboratory techniques. Collection of more precise sequencing data can be anticipated with consistent improvement of sequencing techniques and error correction strategies. An increasing number of researchers have shown interest in this area in recent years, and this has provided vast quantities of data that could provide answers to important existing questions. These data repositories should be effectively utilized. Establishing a public database and collecting deep sequencing data for collective collaboration will facilitate information exchange and the investigation of the varieties of repertoires across gender, age, race, and healthy state. In addition, development of standardized bioinformatic tools will be indispensable for harnessing HTS output.\\nAlthough the expected potential of immune repertoire studies in clinical use is enormous, more work remains to be done to incorporate the observed dynamic changes and sequence signatures with clinical features and outcomes. Questions remain about how the severity of certain infections is related to alterations of the immune repertoire response and various manifestations of CDR3 sequences, and how to predict abundance of protective immunoglobulins or T cell from a given sequence library. In terms of therapeutic discoveries, identification and production of functional antibodies and T cells will promote the development of passive immune therapies and vaccines. Traditional and recently reported large-scale screening strategies may contribute greatly to this process. Advances in HTS of the immune repertoire during health and disease will provide comprehensive views of the adaptive immune response in very near future and will open the door to more rationale immunotherapy for infection.\\nDH was involved in study design, wrote the first draft of the manuscript, conducted the literature search, reviewed the abstracts, performed the analysis, and contributed to the final draft; CC and SC were involved in study design and reviewed the abstracts. ES revised the manuscript. YS designed and supervised the study, revised the final draft, and contributed to the analysis. All authors have read and approved the final manuscript. \",\n          \"Extracts and isolated diterpene glycosides are currently widely used in the food industry as sugar substitutes and sweeteners, because of their unique organoleptic properties, namely their sweetness and the virtual absence of bitterness and adventitious flavors, as well as their extremely low calorific value [1] .\\nAt the same time, there is great interest in published data on the antiviral and antibacterial activities of Stevia extracts. Thus, in in vitro experiments, Stevia extracts effectively suppressed the activity of human retroviruses (HRV) by blocking the binding of virus with susceptible cells. Another study presented data on the antibacterial activity of aqueous extracts of Stevia in relation to enterohemorrhagic Escherichia coli [2 -5] .\\nThe limited amount of information available on this question led us to study the antiviral properties of a dried extract of Stevia.\\nThe biological activity of a dried, purified extract of Stevia leaves was assessed using a modified method [6, 7] to test antiviral activity in collaboration with the All-Russian Science Research Institute of Veterinary Virology and Microbiology (VNIIBBiM).\\nTests were performed using the RNA-containing Teschen disease virus (porcine teschovirus) and the DNA-containing infectious rhinotracheitis (IRT) virus (bovine herpesvirus 1).\\nExperiments also used the RNA-containing human coronavirus (human coronavirus (Hco V-229E)).\\nStudies were performed using a model based on transformed green monkey kidney (Vero) cells. The virustatic and virucidal actions of the dried extract were tested in three repeats on monolayer cultures of Vero cells, which were incubated at 37\\u00b0C in an atmosphere of 5% CO 2 using 96-well plates (Costar, UK). The antiviral activity of the dried extract from Stevia leaves was assessed using standard methods in terms of the ability of the substance to prevent the cytopathic effect of the virus on cells as compared with controls after 72 h of incubation [8] . Virus titers were measured as lg TCD 50 (50% tissue cytopathic doses) values as described by Reid and Mench [9] .\\nControls consisted of cell cultures infected with virus at the experimental dose without addition of test substance (virus controls) and intact cell cultures supplemented with Eagle-M maintenance medium instead of test solution (cell controls).\\nVirustatic (inhibitory) actions were determined by infecting Vero cell cultures with virus-containing material at a multiplicity of infection of 0.0001 -0.001 TCD 50 /cell. After virus infection, cells were incubated at 37\\u00b0C for 1 -1. \\nThese studies showed that the dried, purified extract obtained from Stevia leaves at a dose of 2000 mg/ml inhibited the reproduction of Teschen disease virus by 0.5 lg TCD 50 , IRT virus by 0.25 lg TCD 50 , and coronavirus by 0.33 lg TCD 50 (Table 1) .\\nThe dried, purified extract at a dose of 4000 mg/ml inactivated Teschen disease virus by 0.75 lg TCD 50 , IRT virus by 0.5 lg TCD 50 , and coronavirus by 0.66 lg TCD 50 ( Table 2) .\\nThe commonly used antiviral agent remantadine is known [10] to have high virustatic action but virtually no virucidal activity, i.e., it does not suppress viruses in the intercellular space or in blood vessels. Unlike remantadine, solutions of the dried, purified Stevia extract had marked virustatic and virucidal properties. Until recently, dried Stevia extracts (steviosides) were regarded mainly as sweeteners. The demonstration of antiviral properties of Stevia extract makes it attractive for further investigation as a potential medicinal agent. \"\n        ],\n        \"semantic_type\": \"\",\n        \"description\": \"\"\n      }\n    },\n    {\n      \"column\": \"authors\",\n      \"properties\": {\n        \"dtype\": \"string\",\n        \"num_unique_values\": 1462,\n        \"samples\": [\n          \"Messner, Wolfgang.  Payson, Sarah E\",\n          \" Mohon, Abu Naser.  Elahi, Rubayet.  Khan, Wasif<br>A..  Haque, Rashidul.  Sullivan, David J..  Alam,<br>Mohammad Shafiul\",\n          \" Kedik, S. A..  Yartsev, E. I..  Stanishevskaya,<br>I. E.\"\n        ],\n        \"semantic_type\": \"\",\n        \"description\": \"\"\n      }\n    },\n    {\n      \"column\": \"title\",\n      \"properties\": {\n        \"dtype\": \"string\",\n        \"num_unique_values\": 1500,\n        \"samples\": [\n          \" The Ebola epidemic in West Africa: Challenges,<br>opportunities, and policy priority areas\",\n          \" Role of the golgi apparatus in cellular<br>pathology\",\n          \" Antiviral activity of dried extract of Stevia\"\n        ],\n        \"semantic_type\": \"\",\n        \"description\": \"\"\n      }\n    },\n    {\n      \"column\": \"journal\",\n      \"properties\": {\n        \"dtype\": \"string\",\n        \"num_unique_values\": 824,\n        \"samples\": [\n          \"Clinical Techniques in Small Animal Practice\",\n          \"Disaster Bioethics: Normative Issues When Nothing is Normal\",\n          \"J Electron Microsc Tech\"\n        ],\n        \"semantic_type\": \"\",\n        \"description\": \"\"\n      }\n    },\n    {\n      \"column\": \"abstract_summary\",\n      \"properties\": {\n        \"dtype\": \"string\",\n        \"num_unique_values\": 1067,\n        \"samples\": [\n          \" Several viruses cause pulmonary infections<br>due to their shared tropism with cells of the<br>respiratory tract. These respiratory problems due to viral<br>infection become a public health concern due to rapid<br>transmission through air/aerosols or via direct-indirect<br>contact with infected persons. In addition, the<br>cross-species transmission causes alterations to viral<br>genetic makeup thereby increasing the risk of<br>emergence of pathogens with new and more potent<br>infectivity. With the introduction of effective nucleic<br>acid-based technologies, post translational gene<br>silencing (PTGS) is being increasingly used to silence<br>viral gene targets and has shown promising approach<br>towards management of many viral infections. Since...\",\n          \" Background: Newcastle disease virus (NDV) is<br>an enveloped RNA virus, bearing severe economic<br>losses to the poultry industry worldwide. Previous<br>virion proteomic studies have shown that enveloped<br>viruses carry multiple host cellular proteins both<br>internally and externally during their life cycle. To<br>address whether it also occurred during NDV infection,<br>we performed a comprehensive proteomic analysis<br>of highly purified NDV La Sota strain<br>particles.\\nResults: In addition to five viral structural proteins,<br>we detected thirty cellular proteins associated<br>with purified NDV La Sota particles. The identified<br>cellular proteins comprised several functional<br>categories, including cytoskeleton proteins, annexins,<br>molecular chaperones, chromatin modifying proteins,...\",\n          \" Recent studies have shown that gamma<br>interferon (IFN-\\u03b3) synergizes with the innate IFNs (IFN-\\u03b1<br>and IFN-\\u03b2) to inhibit herpes simplex virus type 1<br>(HSV-1) replication in vitro. To determine whether<br>this phenomenon is shared by other herpesviruses,<br>we investigated the effects of IFNs on human<br>cytomegalovirus (HCMV) replication.\\nWe have found that as with<br>HSV-1, IFN-\\u03b3 synergizes with the innate IFNs<br>(IFN-\\u03b1/\\u03b2) to potently inhibit HCMV replication in vitro.<br>While pre-treatment of human foreskin fibroblasts<br>(HFFs) with IFN-\\u03b1, IFN-\\u03b2 or IFN-\\u03b3 alone inhibited HCMV<br>plaque formation by ~30 to 40-fold, treatment with<br>IFN-\\u03b1 and IFN-\\u03b3 or IFN-\\u03b2 and IFN-\\u03b3 inhibited HCMV...\"\n        ],\n        \"semantic_type\": \"\",\n        \"description\": \"\"\n      }\n    }\n  ]\n}",
              "type": "dataframe",
              "variable_name": "df"
            },
            "text/html": [
              "\n",
              "  <div id=\"df-5d0e9bc1-e618-4e1e-bd7e-8bb1b727891e\" class=\"colab-df-container\">\n",
              "    <div>\n",
              "<style scoped>\n",
              "    .dataframe tbody tr th:only-of-type {\n",
              "        vertical-align: middle;\n",
              "    }\n",
              "\n",
              "    .dataframe tbody tr th {\n",
              "        vertical-align: top;\n",
              "    }\n",
              "\n",
              "    .dataframe thead th {\n",
              "        text-align: right;\n",
              "    }\n",
              "</style>\n",
              "<table border=\"1\" class=\"dataframe\">\n",
              "  <thead>\n",
              "    <tr style=\"text-align: right;\">\n",
              "      <th></th>\n",
              "      <th>paper_id</th>\n",
              "      <th>doi</th>\n",
              "      <th>abstract</th>\n",
              "      <th>body_text</th>\n",
              "      <th>authors</th>\n",
              "      <th>title</th>\n",
              "      <th>journal</th>\n",
              "      <th>abstract_summary</th>\n",
              "    </tr>\n",
              "  </thead>\n",
              "  <tbody>\n",
              "    <tr>\n",
              "      <th>6252</th>\n",
              "      <td>fefd6cdcddcb00d1ba2ef1a0d3bba9f312010a1d</td>\n",
              "      <td>10.1111/j.1524-4733.2008.00379.x</td>\n",
              "      <td></td>\n",
              "      <td>China is the biggest country in the world that...</td>\n",
              "      <td>Li, Youing.  Sun, Xin.  Wang, Li</td>\n",
              "      <td>Evidence-Based Medicine in China</td>\n",
              "      <td>Value in Health</td>\n",
              "      <td>Not provided.</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>4684</th>\n",
              "      <td>2a3d080d5dcf6771db002366691f35e5d6f0e493</td>\n",
              "      <td>10.1007/s13238-010-0029-7</td>\n",
              "      <td></td>\n",
              "      <td>✉ Correspondence: zlshi@wh.iov.cn Bat, the onl...</td>\n",
              "      <td>Shi, Zhengli</td>\n",
              "      <td>Bat and virus</td>\n",
              "      <td>Protein &amp; Cell</td>\n",
              "      <td>Not provided.</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>1731</th>\n",
              "      <td>2de657aabab331e75657bfc1443542fd345b6f26</td>\n",
              "      <td>10.1016/j.ccc.2019.06.001</td>\n",
              "      <td>KEYWORDS\\nPandemic Influenza Disaster prepared...</td>\n",
              "      <td>Outbreaks of infectious disease pose unique ch...</td>\n",
              "      <td>Maves, Ryan C..  Jamros, Christina M..  Smith...</td>\n",
              "      <td>Intensive Care Unit Preparedness During&lt;br&gt;Pa...</td>\n",
              "      <td>Crit Care Clin</td>\n",
              "      <td>KEYWORDS\\nPandemic Influenza Disaster&lt;br&gt;prep...</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>4742</th>\n",
              "      <td>bc22758234283f024f16e2dbe660bc909b512b92</td>\n",
              "      <td>10.1016/j.vetpar.2007.11.020</td>\n",
              "      <td>Cryptosporidium parvum is recognized as one of...</td>\n",
              "      <td>Diarrhoeal diseases of infectious aethiology r...</td>\n",
              "      <td>Klein, Pavel.  Kleinová, Tereza.  Volek,&lt;br&gt;Z...</td>\n",
              "      <td>Effect of Cryptosporidium parvum infection on...</td>\n",
              "      <td>Veterinary Parasitology</td>\n",
              "      <td>Cryptosporidium parvum is recognized as one o...</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>4521</th>\n",
              "      <td>a204aafa38365dbcc0a26af3ca2c6d3313d7fab2</td>\n",
              "      <td>10.1111/irv.12518</td>\n",
              "      <td>, Brazil, in the period of Background: The res...</td>\n",
              "      <td>The inclusion criteria for asymptomatic adults...</td>\n",
              "      <td>Moreira, Luciana Peniche.  Watanabe, Aripuana...</td>\n",
              "      <td>Respiratory syncytial virus evaluation among&lt;...</td>\n",
              "      <td>Influenza Other Respir Viruses</td>\n",
              "      <td>, Brazil, in the period of Background: The&lt;br...</td>\n",
              "    </tr>\n",
              "  </tbody>\n",
              "</table>\n",
              "</div>\n",
              "    <div class=\"colab-df-buttons\">\n",
              "\n",
              "  <div class=\"colab-df-container\">\n",
              "    <button class=\"colab-df-convert\" onclick=\"convertToInteractive('df-5d0e9bc1-e618-4e1e-bd7e-8bb1b727891e')\"\n",
              "            title=\"Convert this dataframe to an interactive table.\"\n",
              "            style=\"display:none;\">\n",
              "\n",
              "  <svg xmlns=\"http://www.w3.org/2000/svg\" height=\"24px\" viewBox=\"0 -960 960 960\">\n",
              "    <path d=\"M120-120v-720h720v720H120Zm60-500h600v-160H180v160Zm220 220h160v-160H400v160Zm0 220h160v-160H400v160ZM180-400h160v-160H180v160Zm440 0h160v-160H620v160ZM180-180h160v-160H180v160Zm440 0h160v-160H620v160Z\"/>\n",
              "  </svg>\n",
              "    </button>\n",
              "\n",
              "  <style>\n",
              "    .colab-df-container {\n",
              "      display:flex;\n",
              "      gap: 12px;\n",
              "    }\n",
              "\n",
              "    .colab-df-convert {\n",
              "      background-color: #E8F0FE;\n",
              "      border: none;\n",
              "      border-radius: 50%;\n",
              "      cursor: pointer;\n",
              "      display: none;\n",
              "      fill: #1967D2;\n",
              "      height: 32px;\n",
              "      padding: 0 0 0 0;\n",
              "      width: 32px;\n",
              "    }\n",
              "\n",
              "    .colab-df-convert:hover {\n",
              "      background-color: #E2EBFA;\n",
              "      box-shadow: 0px 1px 2px rgba(60, 64, 67, 0.3), 0px 1px 3px 1px rgba(60, 64, 67, 0.15);\n",
              "      fill: #174EA6;\n",
              "    }\n",
              "\n",
              "    .colab-df-buttons div {\n",
              "      margin-bottom: 4px;\n",
              "    }\n",
              "\n",
              "    [theme=dark] .colab-df-convert {\n",
              "      background-color: #3B4455;\n",
              "      fill: #D2E3FC;\n",
              "    }\n",
              "\n",
              "    [theme=dark] .colab-df-convert:hover {\n",
              "      background-color: #434B5C;\n",
              "      box-shadow: 0px 1px 3px 1px rgba(0, 0, 0, 0.15);\n",
              "      filter: drop-shadow(0px 1px 2px rgba(0, 0, 0, 0.3));\n",
              "      fill: #FFFFFF;\n",
              "    }\n",
              "  </style>\n",
              "\n",
              "    <script>\n",
              "      const buttonEl =\n",
              "        document.querySelector('#df-5d0e9bc1-e618-4e1e-bd7e-8bb1b727891e button.colab-df-convert');\n",
              "      buttonEl.style.display =\n",
              "        google.colab.kernel.accessAllowed ? 'block' : 'none';\n",
              "\n",
              "      async function convertToInteractive(key) {\n",
              "        const element = document.querySelector('#df-5d0e9bc1-e618-4e1e-bd7e-8bb1b727891e');\n",
              "        const dataTable =\n",
              "          await google.colab.kernel.invokeFunction('convertToInteractive',\n",
              "                                                    [key], {});\n",
              "        if (!dataTable) return;\n",
              "\n",
              "        const docLinkHtml = 'Like what you see? Visit the ' +\n",
              "          '<a target=\"_blank\" href=https://colab.research.google.com/notebooks/data_table.ipynb>data table notebook</a>'\n",
              "          + ' to learn more about interactive tables.';\n",
              "        element.innerHTML = '';\n",
              "        dataTable['output_type'] = 'display_data';\n",
              "        await google.colab.output.renderOutput(dataTable, element);\n",
              "        const docLink = document.createElement('div');\n",
              "        docLink.innerHTML = docLinkHtml;\n",
              "        element.appendChild(docLink);\n",
              "      }\n",
              "    </script>\n",
              "  </div>\n",
              "\n",
              "\n",
              "<div id=\"df-0d5d17ae-134b-4117-af59-9706c3a4623f\">\n",
              "  <button class=\"colab-df-quickchart\" onclick=\"quickchart('df-0d5d17ae-134b-4117-af59-9706c3a4623f')\"\n",
              "            title=\"Suggest charts\"\n",
              "            style=\"display:none;\">\n",
              "\n",
              "<svg xmlns=\"http://www.w3.org/2000/svg\" height=\"24px\"viewBox=\"0 0 24 24\"\n",
              "     width=\"24px\">\n",
              "    <g>\n",
              "        <path d=\"M19 3H5c-1.1 0-2 .9-2 2v14c0 1.1.9 2 2 2h14c1.1 0 2-.9 2-2V5c0-1.1-.9-2-2-2zM9 17H7v-7h2v7zm4 0h-2V7h2v10zm4 0h-2v-4h2v4z\"/>\n",
              "    </g>\n",
              "</svg>\n",
              "  </button>\n",
              "\n",
              "<style>\n",
              "  .colab-df-quickchart {\n",
              "      --bg-color: #E8F0FE;\n",
              "      --fill-color: #1967D2;\n",
              "      --hover-bg-color: #E2EBFA;\n",
              "      --hover-fill-color: #174EA6;\n",
              "      --disabled-fill-color: #AAA;\n",
              "      --disabled-bg-color: #DDD;\n",
              "  }\n",
              "\n",
              "  [theme=dark] .colab-df-quickchart {\n",
              "      --bg-color: #3B4455;\n",
              "      --fill-color: #D2E3FC;\n",
              "      --hover-bg-color: #434B5C;\n",
              "      --hover-fill-color: #FFFFFF;\n",
              "      --disabled-bg-color: #3B4455;\n",
              "      --disabled-fill-color: #666;\n",
              "  }\n",
              "\n",
              "  .colab-df-quickchart {\n",
              "    background-color: var(--bg-color);\n",
              "    border: none;\n",
              "    border-radius: 50%;\n",
              "    cursor: pointer;\n",
              "    display: none;\n",
              "    fill: var(--fill-color);\n",
              "    height: 32px;\n",
              "    padding: 0;\n",
              "    width: 32px;\n",
              "  }\n",
              "\n",
              "  .colab-df-quickchart:hover {\n",
              "    background-color: var(--hover-bg-color);\n",
              "    box-shadow: 0 1px 2px rgba(60, 64, 67, 0.3), 0 1px 3px 1px rgba(60, 64, 67, 0.15);\n",
              "    fill: var(--button-hover-fill-color);\n",
              "  }\n",
              "\n",
              "  .colab-df-quickchart-complete:disabled,\n",
              "  .colab-df-quickchart-complete:disabled:hover {\n",
              "    background-color: var(--disabled-bg-color);\n",
              "    fill: var(--disabled-fill-color);\n",
              "    box-shadow: none;\n",
              "  }\n",
              "\n",
              "  .colab-df-spinner {\n",
              "    border: 2px solid var(--fill-color);\n",
              "    border-color: transparent;\n",
              "    border-bottom-color: var(--fill-color);\n",
              "    animation:\n",
              "      spin 1s steps(1) infinite;\n",
              "  }\n",
              "\n",
              "  @keyframes spin {\n",
              "    0% {\n",
              "      border-color: transparent;\n",
              "      border-bottom-color: var(--fill-color);\n",
              "      border-left-color: var(--fill-color);\n",
              "    }\n",
              "    20% {\n",
              "      border-color: transparent;\n",
              "      border-left-color: var(--fill-color);\n",
              "      border-top-color: var(--fill-color);\n",
              "    }\n",
              "    30% {\n",
              "      border-color: transparent;\n",
              "      border-left-color: var(--fill-color);\n",
              "      border-top-color: var(--fill-color);\n",
              "      border-right-color: var(--fill-color);\n",
              "    }\n",
              "    40% {\n",
              "      border-color: transparent;\n",
              "      border-right-color: var(--fill-color);\n",
              "      border-top-color: var(--fill-color);\n",
              "    }\n",
              "    60% {\n",
              "      border-color: transparent;\n",
              "      border-right-color: var(--fill-color);\n",
              "    }\n",
              "    80% {\n",
              "      border-color: transparent;\n",
              "      border-right-color: var(--fill-color);\n",
              "      border-bottom-color: var(--fill-color);\n",
              "    }\n",
              "    90% {\n",
              "      border-color: transparent;\n",
              "      border-bottom-color: var(--fill-color);\n",
              "    }\n",
              "  }\n",
              "</style>\n",
              "\n",
              "  <script>\n",
              "    async function quickchart(key) {\n",
              "      const quickchartButtonEl =\n",
              "        document.querySelector('#' + key + ' button');\n",
              "      quickchartButtonEl.disabled = true;  // To prevent multiple clicks.\n",
              "      quickchartButtonEl.classList.add('colab-df-spinner');\n",
              "      try {\n",
              "        const charts = await google.colab.kernel.invokeFunction(\n",
              "            'suggestCharts', [key], {});\n",
              "      } catch (error) {\n",
              "        console.error('Error during call to suggestCharts:', error);\n",
              "      }\n",
              "      quickchartButtonEl.classList.remove('colab-df-spinner');\n",
              "      quickchartButtonEl.classList.add('colab-df-quickchart-complete');\n",
              "    }\n",
              "    (() => {\n",
              "      let quickchartButtonEl =\n",
              "        document.querySelector('#df-0d5d17ae-134b-4117-af59-9706c3a4623f button');\n",
              "      quickchartButtonEl.style.display =\n",
              "        google.colab.kernel.accessAllowed ? 'block' : 'none';\n",
              "    })();\n",
              "  </script>\n",
              "</div>\n",
              "\n",
              "    </div>\n",
              "  </div>\n"
            ],
            "text/plain": [
              "                                      paper_id  \\\n",
              "6252  fefd6cdcddcb00d1ba2ef1a0d3bba9f312010a1d   \n",
              "4684  2a3d080d5dcf6771db002366691f35e5d6f0e493   \n",
              "1731  2de657aabab331e75657bfc1443542fd345b6f26   \n",
              "4742  bc22758234283f024f16e2dbe660bc909b512b92   \n",
              "4521  a204aafa38365dbcc0a26af3ca2c6d3313d7fab2   \n",
              "\n",
              "                                   doi  \\\n",
              "6252  10.1111/j.1524-4733.2008.00379.x   \n",
              "4684         10.1007/s13238-010-0029-7   \n",
              "1731         10.1016/j.ccc.2019.06.001   \n",
              "4742      10.1016/j.vetpar.2007.11.020   \n",
              "4521                 10.1111/irv.12518   \n",
              "\n",
              "                                               abstract  \\\n",
              "6252                                                      \n",
              "4684                                                      \n",
              "1731  KEYWORDS\\nPandemic Influenza Disaster prepared...   \n",
              "4742  Cryptosporidium parvum is recognized as one of...   \n",
              "4521  , Brazil, in the period of Background: The res...   \n",
              "\n",
              "                                              body_text  \\\n",
              "6252  China is the biggest country in the world that...   \n",
              "4684  ✉ Correspondence: zlshi@wh.iov.cn Bat, the onl...   \n",
              "1731  Outbreaks of infectious disease pose unique ch...   \n",
              "4742  Diarrhoeal diseases of infectious aethiology r...   \n",
              "4521  The inclusion criteria for asymptomatic adults...   \n",
              "\n",
              "                                                authors  \\\n",
              "6252                   Li, Youing.  Sun, Xin.  Wang, Li   \n",
              "4684                                       Shi, Zhengli   \n",
              "1731   Maves, Ryan C..  Jamros, Christina M..  Smith...   \n",
              "4742   Klein, Pavel.  Kleinová, Tereza.  Volek,<br>Z...   \n",
              "4521   Moreira, Luciana Peniche.  Watanabe, Aripuana...   \n",
              "\n",
              "                                                  title  \\\n",
              "6252                   Evidence-Based Medicine in China   \n",
              "4684                                      Bat and virus   \n",
              "1731   Intensive Care Unit Preparedness During<br>Pa...   \n",
              "4742   Effect of Cryptosporidium parvum infection on...   \n",
              "4521   Respiratory syncytial virus evaluation among<...   \n",
              "\n",
              "                             journal  \\\n",
              "6252                 Value in Health   \n",
              "4684                  Protein & Cell   \n",
              "1731                  Crit Care Clin   \n",
              "4742         Veterinary Parasitology   \n",
              "4521  Influenza Other Respir Viruses   \n",
              "\n",
              "                                       abstract_summary  \n",
              "6252                                      Not provided.  \n",
              "4684                                      Not provided.  \n",
              "1731   KEYWORDS\\nPandemic Influenza Disaster<br>prep...  \n",
              "4742   Cryptosporidium parvum is recognized as one o...  \n",
              "4521   , Brazil, in the period of Background: The<br...  "
            ]
          },
          "execution_count": 7,
          "metadata": {},
          "output_type": "execute_result"
        }
      ],
      "source": [
        "df.head()"
      ]
    },
    {
      "cell_type": "code",
      "execution_count": 8,
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/",
          "height": 0
        },
        "id": "5tnNnTvBl9eQ",
        "outputId": "a1731ec0-ae1b-4bf1-9b30-722ed2d6d2ef"
      },
      "outputs": [
        {
          "name": "stdout",
          "output_type": "stream",
          "text": [
            "language\n",
            "en    96.933333\n",
            "de     1.666667\n",
            "fr     0.533333\n",
            "es     0.533333\n",
            "it     0.133333\n",
            "nl     0.133333\n",
            "pt     0.066667\n",
            "Name: proportion, dtype: float64\n"
          ]
        }
      ],
      "source": [
        "def detect_language(text):\n",
        "    try:\n",
        "        return detect(text)\n",
        "    except:\n",
        "        return 'unknown'\n",
        "\n",
        "# Detect language for each article\n",
        "df['language'] = df['body_text'].apply(detect_language)\n",
        "language_percentage = df['language'].value_counts(normalize=True) * 100\n",
        "print(language_percentage)\n",
        "\n",
        "# Filter out non-English articles\n",
        "df = df[df['language'] == 'en']\n",
        "df = df.drop(columns=['language'])"
      ]
    },
    {
      "cell_type": "code",
      "execution_count": 9,
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/",
          "height": 0
        },
        "id": "S9pK9j7ilp7d",
        "outputId": "d138030b-cf34-4a11-b395-c1cea95c951f"
      },
      "outputs": [
        {
          "name": "stderr",
          "output_type": "stream",
          "text": [
            "/usr/local/lib/python3.10/dist-packages/spacy/language.py:2195: FutureWarning: Possible set union at position 6328\n",
            "  deserializers[\"tokenizer\"] = lambda p: self.tokenizer.from_disk(  # type: ignore[union-attr]\n",
            "/usr/local/lib/python3.10/dist-packages/spacy/pipeline/lemmatizer.py:211: UserWarning: [W108] The rule-based lemmatizer did not find POS annotation for one or more tokens. Check that your pipeline includes components that assign token.pos, typically 'tagger'+'attribute_ruler' or 'morphologizer'.\n",
            "  warnings.warn(Warnings.W108)\n"
          ]
        }
      ],
      "source": [
        "# Initialize the spaCy model\n",
        "parser = en_core_sci_lg.load(disable=[\"tagger\", \"ner\"])\n",
        "parser.max_length = 3000000\n",
        "\n",
        "def clean_text(text):\n",
        "    # Tokenize & Lemmatization\n",
        "    doc = parser(text)\n",
        "    tokens_lemma = [token.lemma_ for token in doc]\n",
        "\n",
        "    # Removing Stopwords\n",
        "    tokens_lemma_no_stopwords = [token for token in tokens_lemma if token.lower() not in STOP_WORDS]\n",
        "\n",
        "    # Removing Punctuation\n",
        "    tokens_lemma_no_stopwords_no_punctuation = [token for token in tokens_lemma_no_stopwords if token not in string.punctuation]\n",
        "\n",
        "    return tokens_lemma_no_stopwords_no_punctuation\n",
        "\n",
        "# Apply the cleaning function to the text columns\n",
        "df['cleaned_text'] = df['body_text'].apply(clean_text)"
      ]
    },
    {
      "cell_type": "code",
      "execution_count": 10,
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/",
          "height": 573
        },
        "id": "DLlX2XyI7QX8",
        "outputId": "05875918-5aa8-454c-af99-c58deaef2fd2"
      },
      "outputs": [
        {
          "data": {
            "application/vnd.google.colaboratory.intrinsic+json": {
              "summary": "{\n  \"name\": \"df\",\n  \"rows\": 1454,\n  \"fields\": [\n    {\n      \"column\": \"paper_id\",\n      \"properties\": {\n        \"dtype\": \"string\",\n        \"num_unique_values\": 1454,\n        \"samples\": [\n          \"f78956e70c8bd1756ffb24ae47c35c3217bed384\",\n          \"51ad0f064355c4678f18225f7d231f9865f44a11\",\n          \"7400110e77de9824ff53797dfade054fa9a9cf80\"\n        ],\n        \"semantic_type\": \"\",\n        \"description\": \"\"\n      }\n    },\n    {\n      \"column\": \"doi\",\n      \"properties\": {\n        \"dtype\": \"string\",\n        \"num_unique_values\": 1411,\n        \"samples\": [\n          \"10.3201/eid1601.091376\",\n          \"10.1016/j.jcv.2011.12.020\",\n          \"10.1101/2020.04.14.040204\"\n        ],\n        \"semantic_type\": \"\",\n        \"description\": \"\"\n      }\n    },\n    {\n      \"column\": \"abstract\",\n      \"properties\": {\n        \"dtype\": \"string\",\n        \"num_unique_values\": 1044,\n        \"samples\": [\n          \"InnoMetrics is the system that aims at collecting software development process metrics in a non-invasive way to access and optimize the development process and its efficiency. This paper demonstrates the development and analysis of energy consumption of MacOS systems based on the software process measurement data. It represents the experience of the development of MacOS system collector and Transfer, in addition to the user interface and early analysis of energy consumption metrics calculations.\",\n          \"During the global outbreak of severe acute respiratory syndrome (SARS) in 2003, treatment was empiric. We report the case history of the index patient in a hospital outbreak of SARS in Hong Kong. The patient recovered after conventional antimicrobial therapy. Further studies are needed to address treatment of SARS, which has high attack and death rates.\",\n          \"Intestinal transplantation (IT) is the least common form of organ transplantation; however, it has shown exceptional growth and improvement in graft survival rates over the past two decades mainly due to better outcomes achieved during the first year of transplantation (76 % at 1 year), due to improvement in surgical techniques and the development of better immunosupressive therapies as we understand more about the relationship between the recipient and host immune system. There are still ongoing issues with chronic rejection and long-term survival. Intestinal transplantation is still an acceptable therapy for patients with intestinal failure (IF), but it is generally reserved for patients who develop severe and life-threatening complications despite standard therapies, or those who are not able to maintain a good quality of life. The purpose of this review is to describe the current status, indications, outcomes and advances in the field of intestinal transplantation.\"\n        ],\n        \"semantic_type\": \"\",\n        \"description\": \"\"\n      }\n    },\n    {\n      \"column\": \"body_text\",\n      \"properties\": {\n        \"dtype\": \"string\",\n        \"num_unique_values\": 1453,\n        \"samples\": [\n          \"Respiratory tract infection is common in infants and young children and is a major public health problem in this age group [1] . It is mainly caused by infection of seasonal viruses, such as respiratory syncytial virus (RSV) and rhinovirus [2] [3] [4] . Most infections manifest as an upper respiratory infection, while 20-40 % develop lower respiratory infection, and a small number (2 -3 %) of infected infants require hospitalization [5, 6] . This is of particular concern as infants with lower respiratory viral infection are at increased risk of developing recurrent wheezing or childhood asthma [7] [8] [9] [10] [11] [12] [13] . Several risk factors have been identified for the development of severe lower respiratory viral infections, such as premature birth, young age, being born in relation to the RSV season, underlying chronic lung disease, congenital heart disease, high parity, young maternal age and poor socioeconomic factors [14] [15] [16] . Recently, genetic polymorphisms have been identified which are associated with severe viral infection [17, 18] . Many of the identified genetic variants associated with severe lower respiratory tract infections are also associated with and pathways common to asthma, including innate immune genes involved in cytokine and chemokine signaling, and epithelial cell function [13, 17, [19] [20] [21] [22] [23] .\\nAs the clinical hallmark of asthma is wheezing, and therapy includes beta agonists, a number of clinical studies over the past 2 decades have examined the relationship between genetic variants in \\u03b2 2 -Adrenergic receptor (ADRB2) and asthma [24] [25] [26] [27] [28] [29] [30] . The polymorphism that resulted in the Gly16 allele was associated with decreased bronchodilator responses in early clinical studies; however, larger clinical studies have provided equivocal results [31] [32] [33] . Other genetic variants in the promoter region of ADRB2 have also been associated with asthma severity [28, 34] . Despite the significant interest in ADRB2 polymorphisms related to asthma, the relative contribution of ADRB2 genetic variants to severity of respiratory viral infections has not been established. As wheezing is also the clinical hallmark of lower respiratory viral infections, and variable response to bronchodilators has been demonstrated in infants hospitalized with RSV lower respiratory viral infections, it is possible that ADRB2 genetic variants also contribute to the severity of acute respiratory viral infections [35, 36] .\\nThe aim of the present study was to explore associations between the genetic variants in ADRB2 that have been associated with a response to asthma therapy during an acute asthma exacerbation and the severity of infant acute respiratory viral infections. We examined the relationship between 5 common polymorphisms in the promoter region and coding block of the ADRB2 gene and infant severity of respiratory infections in a prospective cohort of Caucasian and African American infants.\\nIn order to examine the association of genetic variants in the regulatory regions of the ADRB2 gene and the severity of viral respiratory infections during infancy, we conducted an analysis of infants enrolled in the Tennessee Children's Respiratory Initiative (TCRI), a prospective cohort of mother-infant dyads enrolled during an infant's acute viral respiratory illness over four respiratory viral seasons, 2004-2008. Term (\\u226537 weeks) and non-low birth weight (\\u22652275 g) infants who were otherwise healthy with no significant co-morbidities or cardio-pulmonary disease were eligible, with oversampling for hospitalization infants. Eighty five percent of enrolled subjects had at least one respiratory virus identified by polymerase chain reaction (RSV, rhinovirus, influenza, parainfluenza, Human Metapneumovirus, and coronavirus); 56 % had labconfirmed RSV infection. The rationale, methods, and detailed inclusion and exclusion criteria have been reported previously [37] . Although 630 infants were enrolled in TCRI, this study was limited to the 261 Caucasian and 113 African American infants for whom DNA was available. The protocol was approved by the Institutional Review Board of Vanderbilt University, and mothers provided informed consent for their infants. All parents provided written informed consent for both their and their child's study participation.\\nInfants included in this study had either upper respiratory tract infections or lower respiratory tract infections. The severity of acute respiratory infection was determined by experienced research nurses through medical record chart review using an ordinal bronchiolitis severity score (BSS). The score is an aggregate of assigned values ranging of 0-3 in categories of respiratory rate, room air oxygen saturation, the presence and extent of wheezing, and the presence and extent of flaring and retractions [37] [38] [39] . The BSS ranges from 0 to 12, with higher scores indicating more severe illness and a difference of 0.5 being a clinically significant difference [40] . We measured the BSS at the end of the outpatient visit or after discharge from an inpatient admission, and the most severe recorded value was used for analysis.\\nDNA was extracted by Vanderbilt's DNA core laboratory from whole blood or saliva samples (Oragene by DNA Genotek) by standardized protocols. All oragene samples were processed immediately after collection, while blood samples were either processed immediately or frozen before processing. Genotyping at five pre-specified loci (-2387, -2274, -1343, +46, and +79) of ADRB2 gene was done using Taqman SM\\u00a9 Genotyping Assays (Applied Biosystems, Foster City, CA, USA) with predetermined primers. The five single-nucleotide polymorphisms (SNP)s were rs1432623, rs11168068, and rs2400707 at the promoter region, and rs1042713 SNP and rs1042714 SNP at the coding region of ADRB2 gene. Probes for the Applied Biosystems assays used were labeled at their 5\\u2032end with either VIC\\u00ae or 6-carboxy-fluorescine (FAM) reporter dyes. The reaction components include: 2.5 \\u03bcl Taqman Universal PCR Master Mix (Applied Biosystems), 0.125 \\u03bcl or 0.250 \\u03bcl assay mix (Applied Biosystems) and approximately 5 ng of genomic DNA in a total volume of 10 \\u03bcl per single tube reaction. PCR amplification was performed in a thermal cycler (Techne TC-412) with an initial step of 95\\u00b0C for 10 min followed by 50 cycles of 92\\u00b0C for 15 s and 60\\u00b0C for 1 min. After amplification, the fluorescence of each sample was read on the ABI 7900HT (DNA Resources Core at Vanderbilt University) and analyzed with the Sequence Detection Software (Applied Biosystems). Five percent of samples were run as blind duplicates with 100 % concordance. Primers and probes for all SNPs are listed in Table 1 .\\nThe demographics and characteristics of the 374 infants included, with either upper respiratory infection or lower respiratory tract infection, are presented as frequencies and proportions for specific categorical variables and medians and interquartile ranges were calculated for continuous variables. All analyses were stratified by race (Caucasian or African American), as a surrogate measure of European or African ancestry, to reduce the known impact of population substructure which can produce spurious associations [41] . A pairwise linkage disequilibrium (LD) was estimated for the SNPs in ADRB2 using the standardized summary statistics D' and r, calculated by the HaploView program (Whitehead Institute for Biomedical Research, Cambridge, MA, USA). SNPs were tested for deviations from Hardy Weinberg proportions, also using Haploview [42] . The promoter, coding and combined promoter and coding SNPs were assigned to blocks using D' confidence interval of Gabriel et al. [43] . Individual haplotypes were estimated using the program PHASE v2.1.1 [44, 45] . The haplotype frequencies for each block were estimated using the expectation maximization (EM) algorithm, again using the HaploView program. Non-parametric Kruskal-Wallis test were applied to compare the difference in BSS among infants with each SNP genotype, as well as each haplotype genotype. Haplotype genotypes with frequency less than 2 % were excluded from haplotype analysis. We used a proportional odds model for the ordinal BSS to test the additive effect of haplotypes. Covariates included in the regression model included infant age at enrollment, gender, daycare exposure, secondhand smoke exposure, any prior history of breastfeeding, any siblings at home, and enrollment season. All analyses were conducted separately according to race. All analyses were performed using R-software version 2.11.1 (www.r-project.org). We used a two-sided 5 % significance level for all statistical inferences.\\nThere were in total 374 infants, 261 Caucasians and 113 African Americans, enrolled in the TCRI cohort with available DNA and at least one ADRB2 polymorphism tested ( Table 3 lists the allele frequencies of 5 SNPs separately for Caucasian and African Americans. In both Caucasians and African Americans, infants had slightly more T, T, and G alleles for the three promoter SNP rs1432623, (Figs. 1 and 2) . This non statistical difference was consistent within both Caucasians and African Americans. Among African Americans, there was a decreasing trend of the severity of acute respiratory infection with an increasing number of copies of C, C, and A allele of the three promoter SNPs rs1432623, rs11168068, and rs2400707, respectively (Fig. 1) . However, analyses with additive models indicated no statistical significance (p > 0.05).\\nAmong 2 5 (32) possible haplotype combinations, only four haplotypes with more than 2 % of frequency were identified (Table 4 and Fig. 3 ). There were two promoter haplotypes and three coding block haplotypes. There were no statistically significant differences in the promoter and coding block haplotype frequencies between Caucasian and African Americans. However, when all 5 SNPs were combined, the distribution of haplotype frequency was significantly different between Caucasians and African Americans, with genotype CCAAC more common in African Americans and rare in Caucasians (p < 0.001).\\nPromoter haplotype CCA was associated with a decreased BSS in African Americans in a dose dependent manner (p = 0.037 in the univariate analysis) (Fig. 4) . Infants without the CCA haplotype had the highest BSS (median 5.5, interquartile range [IQR]: 2.0, 8.0). Infants with one copy of CCA haplotype had a lower bronchiolitis severity score (median 4.0, IQR: 1.0, 7.5). Infants with 2 copies of the CCA haplotype had the comparatively lowest BSS (median 3.0, IQR: 1.0, 4.0). This dose dependent relationship persisted after adjusting for infant age, gender, daycare exposure, secondhand smoke exposure, prior history of breastfeeding, siblings at home, and enrollment season (adjusted odds ratio: 0.59, 95 % confidence interval [CI]: 0.36, 0.98, p = 0.042). This protective effect of CCA haplotype was consistent when we categorized the acute respiratory infection into upper respiratory infection and lower respiratory tract infection. In a similar dose dependent manner, infants with more copies of CCA haplotype were less likely to have lower respiratory tract infection compared with infants with less copies of CCA haplotype (p = 0.05). On the other hand, promoter haplotype TTG was associated with a higher BSS in African Americans (Additional file 1).\\nThe protective effect of CCA haplotype in African Americans was identified in all combined haplotypes with CCA combination (Figs. 5 and 6 ). However, due to the sample size, the effect was not statistically significant. In addition, all African American infants with a coding block haplotype GG had CCA combination in their promoter region; therefore, there was a similar additive protective effect of GG in African Americans (Additional file 2). There was no relationship between BSS and the other two coding block haplotypes in African Americans (Additional files 3 and 4). This was also true for the combined haplotypes (Additional files 5 and 6).\\nIn both Caucasian and African American infants, there were no significant relationships between BSS and coding block haplotypes except for GG haplotype among African Americans (Additional file 2). This was also true for the combined haplotypes. In general, there was no significant relationship between BSS and any haplotypes in Caucasian infants. There was no similar promoter haplotype CCA relationship with BSS in Caucasians as observed in the African Americans (Fig. 4 ).\\nIn this study we have identified a haplotype in the promoter region of the ADRB2 gene which is associated with a decreased BSS in African American infants. This promoter haplotype of the ADRB2 gene was not associated with the severity of respiratory infections in Caucasian infants. The common coding block polymorphisms, corresponding to amino acid changes at codons 16 and 27, were not associated with a protective effect on severity of respiratory infections in Caucasian or African American infants. Our results agree with the studied conducted by Janssen and colleagues, who found no relationship between the SNP rs1042713 and susceptibility to RSV bronchiolitis in a European population [17] . This observation that the CCA promoter region haplotype is associated with a protective effect on respiratory viral infection severity is similar to our observation in a cohort of adults hospitalized for an asthma exacerbation, where this haplotype was associated with spirometric improvement in response to asthma-specific therapy that included beta-agonists and corticosteroids [34] . In that study, we noted spirometric improvement only in Caucasian patients, although the small number of African American non-responders in that cohort may have limited our ability to detect a similar effect in African Americans [34] .\\nThe mechanism by which a haplotype in the promoter region of the ADRB2 gene is associated with a decreased BSS is not known. However, we can speculate that [46] . Moore et al. demonstrated that ADRB2 genetic variants are associated with differences in ADRB2 expression in cultured human airway smooth muscle (HASM) cells, an in vitro system used as a model for changes in the human airway [47] These studies were performed prior to identification of the complex promoter haplotypes, and we are not aware that these haplotypes have been studied in HASM. Small differences in airway tone could thus be important in the setting of acute viral respiratory infections.\\nAlthough beyond the scope of the current observational study, we also speculate that ADRB2 genotype might also influence response to bronchiolitis-specific therapy. The primary goal of TCRI was to establish a longitudinal, prospective investigation of infants and their biological mothers on the acute and the long-term health consequences of varying viral respiratory tract infections on early childhood outcomes [37] . Although a BSS was assigned for each infant at acute care visit or hospital admission, we did not study or quantify the effects of different treatments, including beta-agonist therapy. A number of clinical studies have suggested that some infants with bronchiolitis appear to improve with bronchodilator therapy, while on average it is not efficacious [48] [49] [50] . As ADRB2 is the primary target of bronchodilator therapy, variation in response to beta-agonists might reflect genetic variants in ADRB2.\\nThe diversity of SNPs of the ADRB2 gene was explored by Drysdale et al., using immortalized lymphocytes from 23 Caucasians, 19 African-Americans, 20 Asians, and 15 Hispanic-Latinos [27] . This report from (-2387, -2274 , and -1343) and in the coding block SNPs discussed in this manuscript [28] . Such frequency distributions observed in our study are consistent with the allele frequencies reported in the reference population of the International HapMap Project [51] and Hawkins et al. [28] . Our stratified analyses by race therefore allow us to detect the association between severity of a respiratory illness and haplotype frequency within each racial group, and to minimize the spurious associations due to such population substructure. We recognize that the significance of our findings is limited by our small sample size and in classifying infants in this study by a severity score rather than by measuring some direct response to bronchodilator therapy. However, BSS correlated with the severity of disease (classified as upper respiratory and lower respiratory illness) well. Our sensitivity analyses with the severity of disease (upper versus lower respiratory illness) showed consistent results indicated that BSS is a valid surrogate of disease severity. Lastly, although a p value of 0.042 is at the threshold of statistical significance, this p value reflects sample size of 113 African Americans in our study. Whether the association between BSS and CCA haplotype in African Americans is powerful or not depends upon not only p value, but also upon the effect size such as odds ratio and the clinical significance of the findings. As a difference of 0.5 in BSS is considered as clinically significant, a 65 % reduced odds of developing a more severe disease (higher BSS) comparing 2 copies of CCA with 0 copy of CCA might also be clinically significant and meaningful.\\nWe hope that this hypothesis-generating study will encourage the incorporation of pharmacokinetic data in future studies of bronchiolitis. Although validation studies in other populations are needed, this study underscores that the development of personalized approaches to the treatment of bronchiolitis may benefit from the analysis of specific genetic variants.\\nIn summary, we have observed that promoter region haplotype of ADRB2 CCA is associated with a protective effect on respiratory illness severity in African American infants. In this era of personalized medicine, ADRB2 genotype may be predictive of the severity of respiratory viral infection and potential response to therapy for the most common acute infant illness and one for which we currently have no therapy. Next steps would be to validate these findings, as well as evaluate available clinical trial data for which DNA might be available. Fig. 6 BSS distribution stratified by copy number of combined haplotype CCAAC in African Americans infants. The scatter plots and the box-and-whisker plots of BSS across 0, 1, and 2 copies of combined haplotype CCAAC for African American infants. P values were obtained from multivariable regression model adjusted for infant age at enrollment, gender, daycare exposure, secondhand smoke exposure, any prior history of breastfeeding, any siblings at home, and enrollment season\",\n          \"Acute bronchiolitis, a viral infection of the lower respiratory tract, is one of the most substantial health burdens for infants and young children worldwide. 1 Respiratory syncytial virus is the most prevalent viral cause of bronchiolitis in infants. Estimates suggest that about 34 million new cases of lower respiratory infection due to respiratory syncytial virus occur globally in children younger than 5 years, with 3\\u00b74 million admissions to hospitals and about 199 000 deaths per year, predominantly in the developing world. 2 In developed countries such as the USA, bronchiolitis is the most common reason for admission to hospital in the fi rst 12 months of life, 3 accounting for approximately 100 000 infant admissions annually. Although admissions to hospital have declined from 2000 to 2010, emergency department visits have increased, in addition to increased use of mechanical ventilation and hospital charges. 4, 5 The clinical management remains challenging despite the frequency, global reach, economic cost, and morbidity and mortality associated with bronchiolitis. Several treatment strategies (including bronchodilators and corticosteroids) showed no eff ect in pooled metaanalyses, making supportive care the hallmark of current therapy. In this Seminar, we aim to summarise the current evidence for the epidemiology, pathophysiology, diagnostic approach, and management of acute viral bronchiolitis.\\nBronchiolitis is a seasonal infection, with the season typically beginning in late October in the temperate northern hemisphere, peaking in January or February, and ending in April. 6 Globally, independent of region, respiratory syncytial virus infection peaks consistently during annual or biannual epidemics. Although the peak and duration of these epidemics vary worldwide, they are consistent year-to-year within a country. 7 Some data suggest that climate might also be associated with prevalence of respiratory syncytial virus infection, with global surveillance suggesting that infection peaks during wet months in areas with high precipitation and during cooler months in hot regions. 7 Indoor crowding in population-dense areas during rainy seasons or cooler months might be one factor that facilitates viral transmission. 8 Additionally, weather-related factors, such as inhalation of cold and dry air that might impair ciliary function, the airway mucosa, and inhibition of temperature-dependent antiviral responses, might infl uence both disease transmission and severity. 9, 10 Altitude, climate, and meteorological conditions (such as wind speed and dew point) have been shown to have a modest association with bronchiolitis. 11, 12 Furthermore, air pollutants, such as ozone and traffi c pollutants, have been associated with exacerbations of respiratory infections in children younger than 5 years. [13] [14] [15] Environmental tobacco smoke has been associated with increased risk for respiratory syncytial virus-attributable admission to hospital and disease severity in those admitted. 16, 17 As with other respiratory viral infections, the risk of severe respiratory syncytial virus bronchiolitis might be greater in boys than in girls. 18, 19 This diff erence might be due to diff erences in lung and airway development, and by genetic factors. 8, 20 Pathophysiology Bronchiolitis is characterised by extensive infl ammation and oedema of the airways, increased mucus production, and necrosis of airway epithelial cells. 21 Respiratory syncytial virus binds to epithelial cells and replicates, resulting in epithelial necrosis and ciliary destruction. 21, 22 The cell destruction triggers an infl ammatory response with proliferation of polymorphonuclear cells and lymphocytes. The submucosa and adventitial tissues become oedematous with increased mucus secretion. 22 Plugs composed of cellular debris and mucus form in the bronchiole lumens leading to bronchiolar obstruction, air trapping, and diff erent degrees of lobar collapse. 21 Microbiology Molecular testing has led to an improved understanding of the viruses associated with bronchiolitis. Respiratory syncytial virus remains the most commonly identifi ed virus, detected in 41-83% of patients. [23] [24] [25] [26] [27] [28] Other viruses associated with bronchiolitis include rhinovirus, metapneumovirus, coronavirus, human bocavirus, infl uenza virus, adenovirus, and parainfl uenza virus. 24, 25, 27, 29, 30 Studies have investigated whether severity of illness, as measured by need for hospital admission, length of hospital stay, intensive care unit admission, repeated emergency department visits, and apnoea, is associated with specifi c viral infections or co-infections, but the evidence is confl icting. Data from some studies have shown that in infections involving a single virus, respiratory syncytial virus is associated with a more severe course compared with other viruses. 31, 32 Up to 30% of children with bronchiolitis are found to have co-infections with two other viruses, 29 with the combination of respiratory syncytial virus and rhinovirus being the most commonly reported. 29 Some evidence suggests that co-infection in bronchiolitis, particularly respiratory syncytial virus in combination with rhinovirus or metapneumovirus, could be associated with a more severe disease course compared with infection by a single virus. 25, 30, 31, 33, 34 However, other studies do not confi rm this association. 24, 26 Furthermore, although use of nucleic acid amplifi cation tests has greatly improved our ability to detect viruses present in respiratory infections, studies using these technologies have also found at least one respiratory virus in up to 30% of children younger than 6 years with no respiratory symptoms. [35] [36] [37] These viruses might be detected because of asymptomatic colonisation, incubation before clinical infection, or prolonged viral shedding post-infection. The confl icting evidence and high prevalence of respiratory viruses in asymptomatic children suggest no indication at this time that management should vary based on presumed viral cause and presence, or absence of viral co-infections.\\nThe diagnosis of bronchiolitis is clinical and thus requires a clinician to recognise signs and symptoms of viral lower respiratory tract infection in young children. Peak incidence occurs between 3 months and 6 months of age. 38 Since the early clinical defi nition by Court, and as noted in recent practice guidelines, the most specifi c defi nition is in infants. [38] [39] [40] Although the same physiology can occur in toddlers older than 12 months, many clinical trials have excluded these children or have included them as a small subgroup of patients. Bronchiolitis in toddlers can overlap with other conditions such as viral-induced wheezing and asthma, and application of evidence from trials predominantly assessing infants might not be appropriate. Further eff orts to focus the defi nition might assist eff orts to standardise care.\\nThe classic clinical presentation of bronchiolitis starts with symptoms of a viral upper respiratory infection, such as nasal discharge, that progress to the lower respiratory tract over several days (fi gure 1). Timing of symptom progression can vary, and young infants can present with apnoea. Lower respiratory tract symptoms of bronchiolitis include persistent cough, tachypnoea, and increased work of breathing, as shown by intercostal or supraclavicular retractions, use of abdominal muscles, grunting, or nasal fl aring. Auscultatory fi ndings include crackles and wheeze. A hallmark characteristic of bronchiolitis is the minute-to-minute variation in clinical fi ndings, as mucus and debris in the airways are cleared by coughing or as the child's state changes from sleep to agitation. This variation can confound assessment, and often requires several examinations over a period of observation. Nasal congestion can also confound the clinical assessment. Nasal suctioning might help to ascertain which fi ndings are truly from the lower airways. Fever can be present in about a third of infants with bronchiolitis, but it is usually present early in the illness with a temperature less than 39\\u00b0C. 38 The median duration of symptoms is about 2 weeks, with 10-20% of infants still having symptoms at 3 weeks after onset. 38, 41 Various clinical scores have been shown in studies and clinical protocols to correlate with disease severity and improvement. 42, 43 Although documentation of a score can be useful as an objective measure, individual scores are not highly predictive, and they should be repeated and combined with other measures of severity for a universal assessment to guide decision making. The diff erential diagnosis for bronchiolitis includes considerations of various infectious and non-infectious causes. Absence of upper respiratory symptoms should raise suspicion of other causes of respiratory distress in young infants, including cardiac disease, congenital airway abnormalities such as a vascular ring, or foreign body aspiration. Other infections can resemble or complicate bronchiolitis. Pertussis should be considered in infants with severe or paroxysmal cough, or with known exposure. Bacterial infections complicating viral bronchiolitis, including otitis media or pneumonia, might present as a new fever or worsening status later in the course of illness.\\nVarious risk factors have been associated with progression to severe bronchiolitis. Those supported by the strongest evidence include presence of chronic lung disease of prematurity and haemodynamically important congenital heart disease, with immunodefi ciency and neuromuscular disorders also considered as high risk in practice guidelines. 38, 39 Young infants (aged <2-3 months) and those with a history of premature birth (especially <32 weeks' gestation) are also at high risk for progression and can present with apnoea without other clinical fi ndings. Studies assessing the risk for further apnoea in hospital have found it to be limited to infants less than 1 month for full-term infants, 48 weeks postconceptional age for preterm infants, or those with apnoea observed before admission. 44\\nBronchiolitis is a clinical diagnosis based on history and physical examination, according to consensus across national guidelines. For infants with a typical presentation of bronchiolitis, routine imaging or laboratory testing is not recommended, as they increase costs without evidence for benefi t (table).\\nAppropriate use of pulse oximetry monitoring and initiation of oxygen for bronchiolitis have received increasing attention in studies and practice guidelines. Findings suggest that arbitrary thresholds for oxygen administration might drive hospital admissions and prolong hospital length of stay. These outcomes represent only part of the morbidity of bronchiolitis, but developing evidence suggests that intermittent hypoxaemia might occur commonly in otherwise stable infants with bronchiolitis and raises questions as to whether this factor should be used as a sole indication for admission to hospitals. A Canadian randomised trial found reduced admissions to hospital from the emergency department without any increase in revisits when pulse oximeters displayed values 3% higher than the actual value, suggesting that arbitrary pulse oximetry thresholds result in unnecessary admissions. 51 A similar trial in the UK in the hospital setting found that reduction of the oxygen threshold from 94% to 90% resulted in earlier discharge from the hospital without any evidence of adverse outcomes. 52 A US trial comparing intermittent versus continuous pulse oximetry in non-hypoxaemic infants in hospitals found similar outcomes between the groups. 53 This US trial and other evidence supports recommendations in US practice guidelines that clinicians use a threshold of 90% for initiation of oxygen, whereas UK guidelines recommend 92%. 38, 39 As the child improves, reduction in intensity of monitoring to intermittent checks is appropriate. A recent study using blinded oximetry at home showed that a substantial proportion of infants with bronchiolitis who are otherwise doing well have oxygen desaturations less than 90%, particularly during sleep, further calling into question arbitrary thresholds for hospital admissions and initiation of oxygen. 54 This evidence will probably lead to eff orts to reduce continuous monitoring in children without other indications for monitoring.\\nThe majority of children with bronchiolitis have either normal radiographs or radiographic fi ndings consistent with simple bronchiolitis, 55-58 including peribronchial thickening, hyperinfl ation, and atelectasis. One prospective study 58 of routine radiographs as part of the assessment for bronchiolitis in the emergency department reported airspace disease in 17 (7%) of 246 patients. Despite the low prevalence of radiographic pneumonia, this and other studies have reported an increase in antibiotic prescription after radiographs are performed, because of non-specifi c fi ndings that infl uence clinicians' decisions. 58, 59 Factors that have been associated with defi nite focal infi ltrates consistent with pneumonia include hypoxia (oxygen saturation <92%), [56] [57] [58] 60 grunting, persistently focal crackles, and fever (especially >39\\u00b0C). 38 Chest radiographs should only be considered in patients when the presentation is not classic for bronchiolitis. These situations include when another diagnosis (such as foreign body aspiration) is high on the diff erential diagnosis, when a child is severely ill and respiratory failure is imminent, and when symptoms are progressing or not resolving according to the typical disease course expected for bronchiolitis. Lung ultrasound is increasingly used to assess cardiopulmonary conditions in adults and children. Several studies have investigated the use of lung ultrasound in the diagnosis of bronchiolitis. Two small studies found that ultrasound fi ndings in infants with bronchiolitis correlate with clinical fi ndings, and might be more specifi c than chest radiography, 61,62 but further studies are needed to establish whether there is a role for ultrasound in diagnosis or assessment of severity.\\nWith the development of PCR to detect respiratory viruses in the nasopharynx, interest in the use of viral testing for causative diagnosis in bronchiolitis has increased. Virological testing, however, does not generally assist in management and is insuffi cient to predict outcomes. 30, 63 Many national guidelines therefore recommend against routine virological testing in bronchiolitis (table). Recent studies suggest that higher respiratory syncytial virus genomic load, measured using quantitative PCR, might be associated with increased length of stay, use of respiratory support, and need for intensive care, in addition to recurrent wheezing, compared with lower viral loads. 28, 31, 32, 64 Further study is warranted to confi rm this association and clarify whether viral load measurement improves understanding of disease pathophysiology and severity. Several guidelines recommend using respiratory syncytial virus testing to guide cohorting of patients; however, the viruses most likely to cause bronchiolitis are all transmitted in a similar fashion (close contact with large-particle aerosols or direct contact with contaminated fomites). [65] [66] [67] Thus, infection control might not be dependent on the identifi cation of specifi c viruses, but rather on following strict precautions including hand hygiene, separating infants in shared hospital rooms by more than 1 m, and other infection control procedures. 65 Additionally, given the sensitivity of PCR testing, results should be interpreted with caution. Certain viruses, such as rhinovirus, might be detected because of viral shedding from an unrelated illness or colonisation; whereas certain other viruses, such as respiratory syncytial virus and metapneumovirus, are almost always associated with an acute infection.\\nBlood and urine testing is not routinely recommended as part of standard practice in the diagnostic work-up of bronchiolitis (table) . A blood gas measurement should not be routinely obtained in infants with bronchiolitis, unless there are signs of impending respiratory failure or severe distress. Proportions of serious bacterial infections, especially bacteraemia and meningitis, are very low in infants with bronchiolitis. 68 Abnormal white blood cell count is rarely useful in predicting serious bacterial infections in children infected with respiratory syncytial virus. 69 Guidelines universally do not recommend complete blood counts in infants with bronchiolitis unless blood count is part of assessment for a fever in infants younger than 1-2 months. Similarly, given that bacteraemia is exceedingly rare (with cited proportions of <0\\u00b71% in the post-pneumococcal vaccine era), blood cultures should not be routinely performed, except in the septic work-up of infants younger than 1-2 months, 70, 71 or in those with severe illness and signs of sepsis. Hydration status is an important consideration in infants with bronchiolitis and should be determined by clinical examination. Routine measurement of serum electrolytes is of little value in the majority of infants.\\nUrinary tract infections in infants with bronchiolitis occur with greater frequency than do bacteraemia and meningitis, with proportions ranging from 1% to 7%. 68, 71, 72 It is reasonable to obtain a urinalysis and urine culture for infants aged less than 60 days with fever and for older febrile infants who have risk factors for urinary tract infections, 73 but urine should not be routinely obtained in all infants with bronchiolitis.\\nCurrent recommendations for management of bronchiolitis focus on agents to treat the patho physiological eff ects of viral lower respiratory infection (eg, bronchodilators and hypertonic saline). Specifi c antivirals such as ribavirin to treat respiratory syncytial virus infection are not recommended in practice guidelines for typical cases of bronchiolitis because of challenging delivery methods, high cost, and potential health risks to caregivers. Multiple new agents for prevention and treatment are under investigation and might become available in the future, including immunoglobulins, small interfering RNA interference, fusion inhibitors, and small molecules. 74 Numerous studies have assessed the role of bronchodilators for the treatment of bronchiolitis, and systematic reviews have found no consistent benefi t. A 2014 Cochrane Collaboration systematic review identifi ed 30 studies assessing bronchodilators, predominantly salbutamol and excluding epinephrine, and 21 studies that looked specifi ally at clinical scores found no evidence of benefi t in any outcomes for infants admitted to hospitals. 75 In outpatients, oxygen saturation, admission to hospital, or time to resolution of symptoms did not improve with bronchodilator usage compared with placebo. For outpatient studies assessing short-term change in pooled clinical scores, the reviewers found a small signifi cant diff erence in mean score (Z=2\\u00b726; p=0\\u00b7024) that was of small eff ect with minimal clinical importance (fi gure 2). Outpatient studies were heterogeneous (I\\u00b2=81%; p<0\\u00b700001), and those showing benefi t in scores tended to include older children and children with recurrent wheezing.\\nNebulised epinephrine was assessed in another Cochrane Collaboration systematic review. 76 This review found no benefi t for epinephrine compared with placebo for inpatients in hospital length of stay or other outcomes. A multicentre Scandinavian study published after this Cochrane review found that inpatients receiving standing doses of epinephrine had longer length of stay compared with inpatients receiving as-needed epinephrine or placebo. 77 For outpatients, the Cochrane review found a diff erence in the numbers of admissions associated with epinephrine treatment during the time of an emergency department visit, but not during the overall course of illness when assessed at 1 week. Clinical practice guidelines including those from the USA, UK, and Canada do not recommend treatment with bronchodilators for bronchiolitis because of this evidence (table) . 38 \\nNebulised hypertonic saline is thought to reduce airway oedema, decrease mucus plugging, improve mucociliary clearance, and rehydrate the airway surface liquid in infants with bronchiolitis. 78 These physiological changes are extrapolated from the cystic fi brosis literature, 79, 80 and the pathophysiological processes in acute bronchiolitis are diff erent. Therefore, the theoretical benefi ts of hypertonic saline seen in cystic fi brosis might not be present in infants with acute viral bronchiolitis. Although initial trials demonstrated some ability of hypertonic saline to decrease hospital length of stay and transiently improve clinical severity score, 81-83 more recent trials demonstrated confl icting results. [84] [85] [86] [87] [88] [89] [90] The trials that showed the largest benefi t were done in hospitals with lengths of stay more than 72 h; thus, hypertonic saline for infants in countries and institutions in which the length of stay approaches or exceeds 72 h might be benefi cial at reducing length of stay. The confl icting results are \\nTest for heterogeneity in the inpatient studies demonstrated low inconsistency between the nine studies (I\\u00b2=36%; p=0\\u00b713) and the summary eff ect was not signifi cant (Z=1\\u00b706; p=0\\u00b729), the outpatient studies demonstrated very high inconsistency between the 12 studies (I\\u00b2=81%; p<0\\u00b700001) and the summary eff ect was signifi cant (Z=2\\u00b726; p=0\\u00b7024), and the overall heterogeneity of the meta-analysis demonstrated high inconsistency between the 21 studies (I\\u00b2=73%; p<0\\u00b700001) and the overall summary eff ect was signifi cant (Z=2\\u00b74; p=0\\u00b7016). IPR=ipratropium. SAL=salbutamol. neb=nebulised. Reproduced from Gadomski and Scribani, 75 refl ected in the diff erences in recommendations across national guidelines (table), with some countries not recommending hypertonic saline, some recommending use for all inpatients, and some recommending use only in moderate to severe illness. The largest systematic review and meta-analysis, published in 2015, examined 24 trials and 3209 patients. 91 Infants receiving hypertonic saline had a signifi cant diff erence in hospital length of stay of -0\\u00b745 days (95% CI -0\\u00b782 to -0\\u00b708; p=0\\u00b701) compared with those receiving 0\\u00b79% saline or standard care. In seven trials, hypertonic saline reduced the risk of admission to hospital by 20% (risk ratio 0\\u00b78; 95% CI 0\\u00b767-0\\u00b796) compared with 0\\u00b79% saline. No substantial adverse eff ects of hypertonic saline were noted in this systematic review. Although the results of this meta-analysis were signifi cant, attention should be paid to the outer limits of the confi dence intervals, which approach no diff erence, and to the clinical relevance of these diff erences. A second 2015 meta-analysis of 15 trials and 1922 patients found a smaller, but signifi cant, decrease in length of stay by -0\\u00b736 days (95% CI -0\\u00b75 to -0\\u00b722; p<0\\u00b7001) in those who received hypertonic saline (fi gure 3). 92 This study found a discrepancy between the overall combined eff ect of all studies on length of stay and the negative results from the largest trials, allowing the authors to conclude that neither individual trials nor pooled estimates provide a strong evidence-based foundation for the use of hypertonic saline. Both meta-analyses showed substantial heterogeneity across studies (I\\u00b2=82\\u00b71%, p<0\\u00b7001; 91 and I\\u00b2=77\\u00b78%, p=0\\u00b7029 92 ). A recent reanalysis of the fi rst 2015 meta-analysis removed two outlying Chinese studies and accounted for imbalances in day of illness at presentation. 93 These analyses resolved the heterogeneity and found that hypertonic saline does not reduce length of stay in infants admitted to hospitals with bronchiolitis (mean diff erence in length of stay removing outliers -0\\u00b721 days; 95% CI -0\\u00b743 to 0\\u00b702; mean diff erence in length of stay accounting for day of illness imbalance 0\\u00b702 days; 95% CI -0\\u00b714 to 0\\u00b717). Large trials have not demonstrated benefi t for hypertonic saline and this meta-analysis found no eff ect of hypertonic saline on length of stay after adjustment for outliers and imbalances. The decision to undertake future trials is controversial given the positive results of some metaanalyses and negative results of others.\\nMultiple studies have examined the role of corticosteroids in the management of children with bronchiolitis. Data from two large multicentre trials have shown no benefi t to corticosteroids alone in reducing admissions to hospital, 94, 95 and a 2013 Cochrane Collaboration review supports the results of these studies. 96 This review included only studies that enrolled children younger than 24 months with a fi rst episode of wheezing and signs of a viral illness. Among the included eight outpatient studies (1824 participants) comparing corticosteroids with placebo there was no reduction in admission at day 1 (Z=1\\u00b705; p=0\\u00b73) or day 7 (Z=1\\u00b738; p=0\\u00b717) after enrolment (fi gure 4), clinical scores, length of stay in the emergency department, or length of time to resolution of symptoms. Among the nine inpatient studies (772 participants), length of hospital stay was not reduced. 96 On the basis of this evidence, multiple clinical practice guidelines recommend against the use of corticosteroids for infants with bronchiolitis (table) . Although clinicians report considering a family or personal history of atopy when deciding whether to treat infants with bronchiolitis with corticosteroids, 41 there is no evidence that such infants receive any benefi t from corticosteroid treatment. [94] [95] [96] Evidence for the presence or absence of respiratory syncytial virus infection in these infants being associated with a response to corticosteroids is also unavailable. 94, 95 Authors from a large study with a factorial design have suggested, in an unadjusted analysis, that high-dose corticosteroids in combination with nebulised epinephrine might reduce admissions for outpatients with bronchiolitis by day 7, 95 but these results are considered exploratory. 39, 45 High-fl ow oxygen and respiratory support\\nNon-invasive technologies to improve oxygenation and ventilation for bronchiolitis include humidifi ed high-fl ow nasal cannula oxygen and continuous positive airway pressure. 97 High-fl ow nasal cannula allows delivery of high fl ows (usually 1-2 L/kg per min) with humidifi cation and a cannula designed to improve patient tolerance. It has been used widely in premature infants, but the mechanisms of action are unclear, in particular whether it might deliver positive end-expiratory pressure in some conditions. Evidence for effi cacy of high-fl ow nasal cannula is predominantly observational, with studies documenting Corneli (2007) Goebel (2000) Kuyucu (2004) Plint (2009) improved respiratory parameters and reduced intubation rates after implementation. 97 One small randomised trial compared high-fl ow nasal cannula with hypertonic saline and found no diff erence in change in respiratory score. 98 Concerns about high-fl ow nasal cannula include the potential for rapid deterioration if the infant is not closely monitored and costs associated with overuse. Continuous positive airway pressure has been studied in intensive care settings in observational studies and several small trials, with some evidence of improved respiratory parameters. 97 The UK guidelines recommend considering continuous positive airway pressure in children with impending respiratory failure from bronchiolitis.\\nAntibiotic overuse in children with bronchiolitis probably occurs because of concerns about the presence of fever, the young age of aff ected patients, diffi culty diff erentiating atelectasis from infectious consolidation on chest radiograph, and concern for undetected secondary bacterial infection. Bronchiolitis, however, has a clear viral cause and the occurrence of secondary bacterial infections is low, with a risk of bacteraemia or meningitis of less than 1%. 99 A detailed review of randomised clinical trials found that routine use of antibiotics did not improve duration of symptoms, length of hospital stay, need for oxygen therapy, or hospital admission. 38 Overuse of antibiotics is known to result in unnecessary adverse eff ects on the patient, and the development of antimicrobial resistance. Routine use should be avoided unless there is clear evidence of a secondary bacterial infection (table) . Acute otitis media has been documented in up to 60% of infants with bronchiolitis. 100, 101 Antibiotic use for acute otitis media in bronchiolitis should follow established evidence and guidelines for acute otitis media. 102 Macrolide antibiotics have anti-infl ammatory properties that might have potential benefi t in mitigating the infl ammation present in bronchiolitis. Two randomised trials found that there was no diff erence between azithromycin and placebo in hospital length of stay, need for oxygen, or hospital re-admission. 103, 104 Another randomised trial found that azithromycin lowered nasal interleukin-8 concentrations, prolonged time to subsequent wheezing episodes, and resulted in fewer days with respiratory symptoms in the year following the bronchiolitis episode compared with placebo. 105 Finally, a US multicentre study found that in children aged 12-72 months with a history of recurrent severe lower respiratory tract infections, early administration of azithromycin during a lower respiratory tract infection reduced the likelihood of progression to a severe infection, but it is not clear whether the underlying disease in these children was bronchiolitis or some other disease process. 106 Given the current evidence, routine use of macrolides is not recommended in bronchiolitis and more research is needed to clarify any potential role it might have in the future.\\nHydration, suctioning, and chest physiotherapy have been suggested as supportive therapies. Infants with bronchiolitis might have diffi culty feeding because of nasal congestion and increased work of breathing; thus, hydration remains a cornerstone of therapy. A multicentre study of 759 infants younger than 12 months admitted to hospital with bronchiolitis showed no benefi t of intravenous fl uids compared with administration of fl uids by nasogastric tube in mean length of stay, admission to the intensive care unit, need for ventilatory support, and adverse events. This trial also found that a nasogastric tube might be easier to place than an intravenous line in these infants. 107 Most guidelines recommend either nasogastric or intravenous fl uids to maintain hydration, with the UK and Scottish guidelines preferring nasogastric or orogastric hydration in those that can tolerate it compared with intravenous hydration (table). If intravenous fl uids are used, isotonic fl uids are preferred to avoid risk of hyponatraemia. 38 Because infants are obligate nasal breathers, nasal suctioning has been suggested to help with clearing of the nares, improve the work of breathing, and improve feeding; however, suctioning might irritate the nasal mucosa and result in oedema. No randomised controlled trials have examined the role of nasal suctioning in bronchiolitis. The insuffi cient available evidence includes a retrospective cohort study of 740 infants 108 and a small observational study of 40 infants. 109 These studies suggest that deep suctioning might increase length of stay for inpatients, 108 infrequent suctioning is associated with an increased length of stay, 108 and oxygen saturation might increase after suctioning. 109 To draw conclusions about causality from these observational studies is diffi cult, because the potential for confouding by indication exists (eg, sicker children might be more likely to receive deep suctioning). Evidence suggests that oxygen saturation increases after nasal irrigation even without suctioning. 110 Current guidelines give diff ering recommendations with regard to suctioning; those that support its use recommend only superfi cial suctioning rather than deep suctioning (table) . 38, 39, 45 Chest physiotherapy use in bronchiolitis appears to vary by country. 111, 112 A recent Cochrane Collaboration review of 12 studies (1249 participants) demonstrated no evidence of benefi t to any type of chest physiotherapy among inpatients in length of stay, oxygen saturation, or respiratory parameters. 113 No published guidelines routinely recommend chest physiotherapy for the management of uncomplicated bronchiolitis in otherwise healthy children without respiratory comorbidities (table) . 38, 39, 45 Prognosis Much work has been published about the risk of developing recurrent wheezing and asthma following bronchiolitis in infancy. [114] [115] [116] Studies have followed birth cohorts to determine the risk of subsequent wheezing after lower respiratory tract infection in young childhood, and cohorts of children admitted to hospital with bronchiolitis. [114] [115] [116] [117] Overall, admission to hospital with bronchiolitis at a young age is associated with an increased risk of recurrent wheezing. Studies report that 17-60% of children with bronchiolitis might develop recurrent wheezing in the years following their initial admission to hospital. A large study from Taiwan that followed up 1981 children admitted with bronchiolitis before age 3 years found that by age 10 years, 351 (17\\u00b77%) of 1981 children with bronchiolitis had a diagnosis of asthma compared with 2159 (11\\u00b77%) of 18 527 controls (hazard ratio 1\\u00b758; 95% CI 1\\u00b741-1\\u00b771). 118 One small cohort study of 138 patients has suggested that 18 (39%) of 46 children admitted with bronchiolitis before 12 months have asthma by 18 years compared with eight (9%) of 92 controls. 115 However, another study that followed a birth cohort of 1246 children found that although lower respiratory tract infection in childhood was associated with an increased risk of recurrent wheezing, this association decreased with age and was not signifi cant by age 13 years. 116 Most children in this cohort had mild illness not requiring hospital admission, and severity of illness might be associated with the increased risk of asthma. 119 The question remains whether respiratory infection at a young age itself predisposes children to asthma through damage or alteration of lung function, or whether children with severe bronchiolitis might have individual risk factors (such as altered immune response or airway function) that predisposes them to both severe bronchiolitis and recurrent wheezing. 8\\nSubstantial knowledge gaps and controversies exist in the management of acute bronchiolitis. The role of nebulised hypertonic saline in acute management is not clear, resulting in confl icting recommendations across clinical guidelines (table) . Although meta-analyses suggest a small reduction of length of stay, these analyses are limited by heterogeneity, not accounting for duration of illness, and not considering the role of outlying study populations. 91 Two large multicentre trials do not support a clinically signifi cant diff erence in length of stay for inpatients and there is no clear evidence of cost benefi t. 4, [89] [90] [91] 120 While the same meta-analysis suggests a possible reduction in admissions for outpatients, the confi dence intervals were wide and the studies were also heterogeneous. 91 No multicentre studies of nebulised hypertonic saline have been completed in outpatients. Although the evidence is increasing that hypertonic saline has little role in meaningfully reducing the length of stay, its role in outpatients is less clear. A large outpatient multicentre study could clarify whether there is any benefi t.\\nEvidence also suggests that combined therapy with nebulised epinephrine and corticosteroids might reduce admissions to hospital. 95 Synergy between corticosteroids and \\u03b22 agonists is well documented in clinical trials of asthma management. [121] [122] [123] [124] Basic science literature also shows that \\u03b22 agonists and corticosteroids enhance each other's eff ectiveness, particularly with regard to antiinfl ammatory gene expression. 125, 126 Because of the economic burden of bronchiolitis and the plausible basic and clinical evidence for synergy, a large and multicentre trial is needed to ascertain whether combined therapy with epinephrine and corticosteroids is benefi cial.\\nOxygen saturation and the use of pulse oximetry play an important role in the decision to admit infants with bronchiolitis to hospital and in the length of their hospital stay. [51] [52] [53] [54] [55] [56] [57] [58] [59] Clinical practice guidelines also give confl icting guidance on the level of oxygen saturation at which admission should be considered. Furthermore, a substantial proportion of discharged infants have episodes of transient desaturation. 54 Again, in view of the large health-care costs associated with hospital admission in bronchiolitis, further research is needed to clarify the level of oxygen saturation requiring admission, the role of continuous and spot measurements of oxygen saturation, and the clinical importance of transient desaturations in otherwise stable young infants.\\nTAF contributed to the design and coordinated the writing of this manuscript. All authors contributed to the literature search and writing of this manuscript.\\nWe declare no competing interests. TAF, ACP, and JJZ have participated in trials of bronchiolitis funded by unrestricted public or academic institutional funding sources that had no infl uence on the conception and writing of this manuscript.\",\n          \"In March and April 2009, health professionals warned the Mexican Ministry of Health about the increase in incidence of acute respiratory illness in young adults [1] . In April, a new viral subtype of influenza A(H1N1)pdm09, a combination of swine, avian and human virus, was identified in the United States and in samples originated from Mexico [2, 3] . On April 25 th , the The patients were identified in the SINAN database, using the following variables for selecting the cases and controls: RT PCR (positive for influenza A(H1N1)pdm2009), final classification (confirmed), criteria (laboratory), evolution (recovered or death), hospitalization (yes), date of hospitalization (28/06/2009 to 29/08/2009), municipalities of residence and hospitals (metropolitan areas of Sao Paulo and Campinas).\\nAt the time the patients were selected for the study, the SINAN included 193 reported and confirmed deaths of patients with influenza A(H1N1)pdm2009 and SARI, all of which were assessed in this study (cases) . For each case of death, two control patients were selected among the 1988 patients hospitalized during the study period with reported and confirmed influenza A(H1N1)pdm2009 and SARI who recovered. The controls were time-paired randomly selected from those hospitalized in the same epidemiological week (EW) or, if necessary, one week later than the date of hospitalization of cases to prevent possible differences in opportunity of treatment and clinical management protocol.\\nData of the study were collected by trained health professionals using two standardized questionnaires. The hospital questionnaire collected information from medical charts, evaluated in 177 hospitals were patients were hospitalized. Of these, 56.5% belonged to the Unified Health System-SUS and 43.5% were private hospitals. The following data were collected: previous pathological history, occurrence of pregnancy, health care, symptomatology during admission, laboratory and radiological tests, need for ICU admission, treatment, complications and disease progression. The International Classification of Diseases, 10 th revision-ICD-10, was used for coding comorbidities indicated in the medical charts. The Ministry of Health provided the antiviral drug known as Oseltamivir for treatment during the pandemic. The protocol for clinical management established that the treatment was for cases hospitalized with SARI and ambulatory patients with risk conditions for severity. Treatment should be started within 48hours after symptom onset. All patients studied had laboratory confirmations of influenza A(H1N1)pdm09 from respiratory secretion samples analyzed using the RT-PCR method, performed by the Adolfo Lutz Institute [16] . The definition of SARI included: fever associated with cough and dyspnea; pneumonia; respiratory insufficiency; tachypnea; radiological changes compatible with pneumonia; oxygen therapy or mechanical ventilation. SARI was confirmed in all cases and controls selected for the study by data their on medical charts.\\nHousehold interviews were conducted in 161 cases (83.4%), with 14 refusals (7.3%) and 18 (9.3%) addresses not found. A total of 337 (85.1%) controls were interviewed, with 6 (1.5%) refusals and 43 (11.1%) addresses not found. The household questionnaire included socio-demographic data, history of previous care related to the episode that led to hospitalization and presence of influenza vaccination. Occupations were grouped according to the occupational pyramid of influenza risk [17] . For cases, interviews were conducted with the closest family members; for controls, with the patients themselves, excluding children, whose parents/legal guardians responded for them.\\nAfter data collection, questionnaires were assessed by supervisors to verify the consistency of information and the fulfillment of criteria to include patients in this study. Hospital and household questionnaires were input to the Sphynx database, version 5.\\nThe present study began in order to assist with decisions on surveillance and control actions. The use of data was approved by the Research Ethics Committee, School of Public Health, University of S\\u00e3o Paulo-FSP-USP (protocol 2283, OF.COEP/312/11). Data collection was performed in accordance with the recommendations of the National Health Council for Human Research, including the signing of an informed consent form by patients, family members or legal guardians.\\nClinical and demographic characteristics are shown as median and inter-quartile intervals or percentages, and comparisons were made using the Mann-Whitney U or chi-square tests, as required. Odds ratio-OR and 95% confidence intervals-95%CI were calculated to assess the factors associated with death. Variables with p-value < 0.20 in the univariate analysis and adjustment variables were selected for the multiple logistic regression, which used forward stepwise strategy. The model including all patients assessed the following dichotomous categorical variables: health insurance, previous care, vaccination against influenza in 2009, sex (reference category = male); and the following risk conditions: diabetes mellitus, obesity, chronic kidney disease, chronic liver disease, immunosuppression, chronic cardiovascular disease, chronic neurological disease and delay in development, using the absence of a condition as a reference category. The model also assessed: age group [<18 years (reference), 18 to 59 years, 60 years], use of antiviral therapy [no use (reference category), 48 hours after the onset of symptoms, > 48 and 72 hours, > 72 hours], high-risk individuals or those with any of the risk conditions listed above adapted from the Centers for Disease Control and Prevention-CDC [18] [no condition (reference category), one condition, and two or more conditions], and time between onset of symptoms and admission in days as the continuous variable. The model was controlled for epidemiologic week of admission at hospital (categorized as EW26-28 (reference), EW29-31 and EW31-34), and income. A second model was developed with patients aged 18 years and over, which also assessed level of education [without formal education or incomplete primary education, complete primary education, complete secondary education, complete higher education (reference category)] and occupational risk [very high and high, average and low (reference category)]. Patients aged 18 to 22 years with incomplete higher education were included in reference category of education. For the modelling, patients with no occupation were included in the reference category of occupational risk. Significance of variables was assessed by Wald test and adjustment of models by Hosmer-Lemeshow test.\\nThe analyses were performed using SPSS software, version 17.0. P-values < 0.05 were considered to be significant. Table 1 presents the socioeconomic characteristics of patients and information associated with the disease obtained from the household interviews. Among patients aged 18 years and over, there was a higher risk of death for those without formal education or with incomplete primary education, OR 2.33(95% CI 1. 13-4.89 ) and complete primary education, OR 3.06 (95%CI 1.41-6.82), when compared to those with higher education. Distribution of ethnic groups, family income and history of smoking were not associated with death. The history of previous vaccination against seasonal influenza in 2009 showed protection against death. Health professionals, considered to be at a high risk of developing influenza, showed protection against death, OR 0.15(95%CI 0.02-0.61). Patients who had sought care prior to hospitalization had a risk of death 3.70 times (95%CI 2.25-6.30) that of patients without previous care, with a predominance of emergency care (45.34%). Table 2 presents the distribution of cases and controls according to variables assessed in the hospital investigation. Median age of cases was higher than that of controls (p<0.001) and the 18-to-59-year age group had an OR for death of 4.03(95%CI 2.62-6.31), compared to those aged less than 18 years. Elderly individuals were also at a significant risk of death.\\nThere were no differences in sex distribution in either group. With regard to risk conditions for severity, 73.6% of cases and 38.1% of controls reported having at least one risk factor, with an OR of 4.49 (95%CI 3.09-6.60). When individuals with more than one of these conditions were assessed, the risk of death increased, OR 10.29 (95%CI 6.04-17.85). The following medical conditions were risk factors for death: obesity, diabetes mellitus, chronic cardiovascular diseases, chronic kidney diseases, neurological diseases and delay in development, chronic liver diseases and immunosuppressive diseases. The immunosuppression-related diseases were: neoplasms (n = 18), HIV/AIDS (n = 8), organ transplants (n = 6), autoimmune diseases (n = 5) and use of immunosuppressants (n = 3).\\nOf the 212 women of reproductive age (10 to 49 years) who were studied, 33.5% were pregnant and there were no differences in the frequency of pregnancy in either group.\\nThe use of antiviral treatment during hospitalization was a protective factor against death, with an OR of 0.67 (95%CI 0.45-0.98). When time between onset of symptoms and beginning of antiviral treatment was assessed, there was an 81% protection among individuals who received treatment within the first 48 hours, with an OR of 0.19 (95%CI 0.11-0.35), and 65% between 48 and 72 hours, with an OR of 0.35 (95%CI 0.16-0.74). After 72 hours, there was no significant protection. Median of days between date of onset of symptoms and date of hospitalization was 5.0 among cases and 2.0 among controls (p<0.001). Among patients with chronic pulmonary disease, median was 4.0 days among cases and 2.0 days among controls; among those with asthma, 4.5 days and 2.0 days; and among pregnant women, 4.0 days and 2.0 days, respectively. With regard to health professionals, the median between the date of onset of symptoms and that of hospitalization was three days for both cases and controls. Median between date of hospitalization and of death or hospital discharge was seven days among cases and four days among controls (Table 2) .\\nAccording to medical charts, fever and cough occurred in 87.8% of cases and 79.3% of controls; and the proportions of dyspnea were 87% and 62.7%, respectively. The definition of cases of Severe Acute Respiratory Syndrome -SARS (fever+cough+dyspnea) based on information obtained from medical charts occurred in 68.9% of cases and 56.5% of controls. Vomit and diarrhea were present in 8% of cases and 18% of controls. Treatment with antibiotics was used in 95.8% of cases and 73.1% of controls. The mean number of antibiotics used was 4.0 and 1.8 respectively. Among cases, there was a higher proportion of hospitalizations in intensive care units (83.4%x16.8%), use of mechanical ventilation (94.8%x10.8%) and dialysis (25.0%x2.3%). The proportion of complications was higher among cases than controls, with an emphasis on: respiratory distress syndrome (64.7%vs4.9%), shock (69.4%vs1.0%), sepsis (63.2%vs2.3%) and kidney complications (44.6%vs1.3%), respectively. Table 3 shows the laboratory findings of entry tests of patients in the hospital. Cases had lower numbers of leukocytes and platelets and higher values of creatine phosphokinase-CPK, lactic dehydrogenase-LDH, glutamic-oxaloacetic transaminase, glutamic-pyruvic transaminase, urea and creatinine than controls, with statistical significance.\\nIn sub-group analysis, previous conditions of risk for severity and obesity were risk factors for death in univariate analysis for children younger than 18 years and women of reproductive age (10 to 49 years). Among children, presence of immunosuppression, chronic cardiovascular disease and neurological disease/delay in development were also risk factors, as well as diabetes mellitus among women. Treatment with antiviral administered within 72 hours after the onset of symptoms was a protective factor for women of reproductive age. Tables 4 and 5 show variables associated with death in the final models of multiple logistic regression, in all patients studied and in sub-group analysis of adults (aged 18 years and over). In patients in general, risk factors for death were: being aged between 18 and 59 years (reference: until 18 years), having one or more risk conditions for the severity of the disease, obesity, immunosuppression and having sought care prior to hospitalization. Although chronic neurological diseases/delay in development and chronic liver disease were highly associated with death, these variables were excluded from the final model because they had estimates with large confidence intervals. Private health insurance and antiviral treatment administered in the first 72 hours after the onset of symptoms were protective factors against death. Among adults aged 18 years and over, the risk factors for death were having more than one risk factor for severity of the disease, obesity and care prior to hospitalization. The protective factors were: antiviral drug administered in the first 48 hours after the onset of symptoms and being a health professional or health unit worker.\\nSensitivity analysis was performed to assess the severity of patients admitted during the study period. The proportion of ICU admissions and use of mechanical ventilation was estimated for epidemiological week of hospitalization, categorized as EW 26 to 28, EW 29 to 31 and EW 32 to 34. The results showed higher proportions of ICU admissions (50.0% x 16.8% x 7.7%) and mechanical ventilation (45.4% x 10.8% x 2.6%) in the three first weeks, among the controls. The cases showed high proportions in all periods analyzed: 89.7% x 84.0% x 76.9% for need of ICU and 89.7% x 97.6% x 89.7% for need of mechanical ventilation, in the 3 periods, respectively. The opportunity for antiviral treatment showed lower proportion of treatment in the first period of the study, in both cases (37.9% x 74.4% x 74.3%) and controls (59.1% x 74.1% x 89.7%). Sensitivity analysis to assess the opportunity for treatment of children and adults showed similar proportions for the groups: 73.9% and 74%, respectively.\\nIn the present study, risk factors for death were analyzed in patients infected with influenza A (H1N1)pdm09 and hospitalized with SARI. There was an increased risk of death in patients aged 18 to 59 years; those with obesity, immunosuppression, neurological and developmental diseases; or those who had received care prior to hospitalization. Additionally, presence of at least one risk factor for aggravation of the disease was also associated with occurrence of death. Use of antiviral drugs, especially when administered in the first 72 hours after onset of symptoms was protective factor against death.\\nThe design of this case-control study, including all deaths reported and confirmed in the region during the study period and the time-paired random selection of controls among patients who survived after hospitalization with SARI, with hospital and household data collection, enabled the analysis of the factors associated with death to be extended, including socioeconomic variables, access to health services, and data on hospitalizations.\\nFew studies have assessed risk factors for death in patients with SARI [15, 19] . Some studies have analyzed risk factors for severity of the disease and deaths, comparing patients hospitalized in ICUs and in wards [13, 14, [20] [21] [22] [23] , whereas one study used the rate of mortality from influenza A(H1N1)pdm09 in two epidemic waves [24] . A study conducted in Australia analyzed risk factors for hospitalization due to influenza, comparing cases with controls without infection in the community [25] . Other studies have analyzed epidemiological surveillance data, including mortality rates and comparisons among deaths, hospitalizations, treatment and other factors [9, 10, 12, [26] [27] [28] [29] [30] [31] [32] [33] [34] [35] . Some of the factors associated with death identified in this study have been described by other authors, such as the presence of comorbidities [31, 35] , age18years [13] , obesity [15, 19, 20] , neurological and developmental diseases [23, 24, 31] and immunosuppression [15, 21, 24, 34] . Children showed lower risk of death, despite a similar proportion of treatment than adults. A similar result was observed in a study of systematic review and metaanalysis, where the risk of death was 0.28 (95% CI 0.19-0.141), when compared to young adults [36] . In contrast, other factors associated with severity of infection that have been reported by different researchers, such as diabetes [19, 34] ; other chronic pulmonary diseases, excluding asthma and chronic obstructive pulmonary disease [20] ; chronic pulmonary disease, including asthma [31] ; and chronic cardiovascular disease [21, 24, 34] , were not associated with mortality in the present study. The population assessed, which was restricted to hospitalized patients with severe influenza could at least partly explain these differences. Diabetes, chronic cardiovascular and kidney disease, associated with death in univariate analysis, did not remain risk factors after multiple analysis. The design of this study, with data collected on medical charts and household interviews, allowed for adjustments in the analysis that can also explain part of these differences.\\nDeaths occurred mainly in adults aged 18 to 59 years, with a small proportion of cases and controls among elderly individuals. A possible preexisting immunity against the new influenza A(H1N1)pdm09 virus in this age group has been suggested, probably due to previous exposure, through other infections or vaccinations, to an influenza A(H1N1) virus which is genetically and antigenically more associated with the new virus than contemporary strains of seasonal H1N1 viruses [37] .\\nAll patients in the present study, whether cases or controls, had a laboratory-confirmed infection of influenza A/H1N1pdm09, which was [38] .\\nIt is possible that the probability of being hospitalized varied throughout the study. Treatment was available for hospitalized patients and for ambulatory patients with risk conditions for severity. This could have influenced hospitalization, but not much in our study population, consisting of patients with SARI. During the pandemic, criteria for hospitalization changed as well as the availability of treatment, especially in the first 3 weeks of the epidemic, as shown in sensitivity analysis, confirming the need for controlling period of hospitalization in the study.\\nThe socioeconomic profile was similar in both groups assessed, with no difference in income or having private health insurance. There was greater protection against death among patients with an occupation with higher risk of exposure to infection, such as health professionals. A greater opportunity of access to treatment among these workers could explain such findings, as the period between the date of onset of symptoms and the date of admission was shorter in this group. The low proportion of deaths among patients categorized as having a higher risk of exposure has also been observed in other studies [30, 33] .\\nPregnancy was not a risk factor for death in this study. In contrast, frequency of pregnant women among those of reproductive age hospitalized with severe manifestation of influenza A (H1N1)pdm09 was high, suggesting that pregnancy could be associated with development of SARI. An association between pregnancy and use of mechanical ventilation during hospitalization due to influenza A(H1N1)pdm2009 [25] and a higher admission rate were reported when compared to the general population [32] . In 2009, in the state of S\\u00e3o Paulo, 27.4% of all confirmed cases of influenza A(H1N1)pdm09 in women of reproductive age (10 to 49 years), hospitalized and recorded in the SINAN database, were found in pregnant women. In Brazil, only 4.27% of women of reproductive age had live newborns in the reference year according to the 2010 Census [39] .\\nPatients who died presented important laboratory changes when compared to controls, with lower levels of leukocytes and platelets and an increase in CPK, LDH, urea and creatinine. Studies revealed a similar trend for the laboratory results of leukocytes [23] , CPK [26, 27, 30] , LDH [26] , platelets [22] and creatinine [27, 30, 34] .\\nAlthough most patients who died received antiviral treatment, only 9.3% were treated in the first 48 hours. Early treatment was a protective factor against death, especially when provided in the first 72 hours after onset of symptoms. Other studies have also shown a more favorable prognostic with antiviral treatment [13, 21, 23, 27, 28, 29, 34] .\\nHistory of care prior to hospitalization was a risk factor for death from influenza A(H1N1) pdm09, which could reflect the difficulty in accessing health services. Emergency services were primarily sought for care prior to hospitalization, which could have contributed to greater delay in diagnosis and beginning of treatment. A study conducted in Mexico showed a similar result with regard to delay in hospitalization [29] .\\nIn one study, which assessed the overall standardized mortality rate from influenza A (H1N1)pdm09, the mortality among patients with preexisting chronic respiratory diseases was lower than those with other chronic diseases 24 . There might have been problems with recording information about previous diseases in the medical charts in the presence of acute pulmonary disease, especially among those with chronic pulmonary disease.\\nThe present study had some limitations. Underreporting of SARI by health professionals is likely to have occurred, as reporting this disease was not compulsory in Brazil before the pandemic. However, during the pandemic, professionals were more sensitive to reporting it. The system for notification, which was mandatory during the epidemic, was facilitated to allow for rapid notifications. The availability of samples from autopsies of cases (deaths) could have resulted in greater access to the laboratory diagnosis of this infection. Household data collection could also have caused information bias, as data on cases were collected through interviews with family members, while data on controls were obtained from the patients themselves, when they were adults. The proportion of refusals of household interviews was higher among cases (7.3%) than controls (1.6%), p = 0.001. Another possible limitation was memory bias, as a result of the time between the hospitalization and interview. However, the information analyzed from household interviews was about socio-demographic characteristics, which is less vulnerable to information and memory bias. Data on preexisting diseases were obtained from medical charts in hospitals, where they are completed at the moment of hospitalization, in the presence of the patients. The quality of completion of medical charts can differ between hospitals. Nonetheless, information about the presence of chronic diseases, clinical management of patients, laboratory results, radiological tests and treatment was available in these charts. The difficulties encountered during data collection were minimized by the use of a standardized questionnaire.\\nIn conclusion, the risk conditions for the severity of the disease, particularly obesity, immunosuppression and neurological/developmental diseases, were also important risk factors for death, emphasizing the need for good vaccination coverage during influenza vaccination campaigns in all patients with chronic diseases belonging to more vulnerable groups. Another relevant result was the antiviral treatment within the first 72 hours of onset of symptoms as a protective factor, also emphasizing the need to warn the population about the importance of seeking early medical care when an influenza-like illness appears, especially among high-risk groups and those with signs of disease aggravation. Training of health professionals, especially physicians, is required for adequate clinical management of patients and early antiviral treatment.\"\n        ],\n        \"semantic_type\": \"\",\n        \"description\": \"\"\n      }\n    },\n    {\n      \"column\": \"authors\",\n      \"properties\": {\n        \"dtype\": \"string\",\n        \"num_unique_values\": 1422,\n        \"samples\": [\n          \" Chen, Junbo.  Zhu, Huachen.  Horby, Peter W.. <br>Wang, Qianli.  Zhou, Jiaxin.  Jiang, Hui.  Liu, Liwei. <br>Zhang, Tianchen.  Zhang, Yongli.  Chen, Xinhua.  Deng,<br>Xiaowei.  Nikolay, Birgit.  Wang, Wei.  Cauchemez, Simon. <br>Guan, Yi.  Uyeki, Timothy M..  Yu, Hongjie\",\n          \" Chen, Yu.  Dong, Yuanji.  Cai, Shaozhe.  Ye, Cong. <br>Dong, Lingli\",\n          \" Arens, Amanda.  Scott, Cheryl.  Osburn, Bennie\"\n        ],\n        \"semantic_type\": \"\",\n        \"description\": \"\"\n      }\n    },\n    {\n      \"column\": \"title\",\n      \"properties\": {\n        \"dtype\": \"string\",\n        \"num_unique_values\": 1454,\n        \"samples\": [\n          \" Peptide Vaccine: Progress and Challenges\",\n          \" The Impact of Ebola Virus Disease on Government<br>Expenditure in Sierra Leone\",\n          \" The clinical implication of dynamic<br>neutrophil to lymphocyte ratio and D-dimer in COVID-19: A<br>retrospective study in Suzhou China\"\n        ],\n        \"semantic_type\": \"\",\n        \"description\": \"\"\n      }\n    },\n    {\n      \"column\": \"journal\",\n      \"properties\": {\n        \"dtype\": \"string\",\n        \"num_unique_values\": 785,\n        \"samples\": [\n          \"AMB Express\",\n          \"Dail and Hammar&#x02019;s Pulmonary Pathology\",\n          \"Benumof's Airway Management\"\n        ],\n        \"semantic_type\": \"\",\n        \"description\": \"\"\n      }\n    },\n    {\n      \"column\": \"abstract_summary\",\n      \"properties\": {\n        \"dtype\": \"string\",\n        \"num_unique_values\": 1044,\n        \"samples\": [\n          \" InnoMetrics is the system that aims at<br>collecting software development process metrics in a<br>non-invasive way to access and optimize the development<br>process and its efficiency. This paper demonstrates<br>the development and analysis of energy<br>consumption of MacOS systems based on the software process<br>measurement data. It represents the experience of the<br>development of MacOS system collector and Transfer, in<br>addition to the user interface and early analysis of<br>energy consumption metrics calculations.\",\n          \" During the global outbreak of severe acute<br>respiratory syndrome (SARS) in 2003, treatment was<br>empiric. We report the case history of the index patient<br>in a hospital outbreak of SARS in Hong Kong. The<br>patient recovered after conventional antimicrobial<br>therapy. Further studies are needed to address<br>treatment of SARS, which has high attack and death rates.\",\n          \" Intestinal transplantation (IT) is the least<br>common form of organ transplantation; however, it has<br>shown exceptional growth and improvement in graft<br>survival rates over the past two decades mainly due to<br>better outcomes achieved during the first year of<br>transplantation (76 % at 1 year), due to improvement in surgical<br>techniques and the development of better<br>immunosupressive therapies as we understand more about the<br>relationship between the recipient and host immune system.<br>There are still ongoing issues with chronic<br>rejection and long-term survival. Intestinal<br>transplantation is still an acceptable therapy for patients<br>with intestinal failure (IF), but it is generally...\"\n        ],\n        \"semantic_type\": \"\",\n        \"description\": \"\"\n      }\n    },\n    {\n      \"column\": \"cleaned_text\",\n      \"properties\": {\n        \"dtype\": \"object\",\n        \"semantic_type\": \"\",\n        \"description\": \"\"\n      }\n    }\n  ]\n}",
              "type": "dataframe",
              "variable_name": "df"
            },
            "text/html": [
              "\n",
              "  <div id=\"df-2cbeb672-3bb4-4dfa-8e54-1d505eb2bbbc\" class=\"colab-df-container\">\n",
              "    <div>\n",
              "<style scoped>\n",
              "    .dataframe tbody tr th:only-of-type {\n",
              "        vertical-align: middle;\n",
              "    }\n",
              "\n",
              "    .dataframe tbody tr th {\n",
              "        vertical-align: top;\n",
              "    }\n",
              "\n",
              "    .dataframe thead th {\n",
              "        text-align: right;\n",
              "    }\n",
              "</style>\n",
              "<table border=\"1\" class=\"dataframe\">\n",
              "  <thead>\n",
              "    <tr style=\"text-align: right;\">\n",
              "      <th></th>\n",
              "      <th>paper_id</th>\n",
              "      <th>doi</th>\n",
              "      <th>abstract</th>\n",
              "      <th>body_text</th>\n",
              "      <th>authors</th>\n",
              "      <th>title</th>\n",
              "      <th>journal</th>\n",
              "      <th>abstract_summary</th>\n",
              "      <th>cleaned_text</th>\n",
              "    </tr>\n",
              "  </thead>\n",
              "  <tbody>\n",
              "    <tr>\n",
              "      <th>6252</th>\n",
              "      <td>fefd6cdcddcb00d1ba2ef1a0d3bba9f312010a1d</td>\n",
              "      <td>10.1111/j.1524-4733.2008.00379.x</td>\n",
              "      <td></td>\n",
              "      <td>China is the biggest country in the world that...</td>\n",
              "      <td>Li, Youing.  Sun, Xin.  Wang, Li</td>\n",
              "      <td>Evidence-Based Medicine in China</td>\n",
              "      <td>Value in Health</td>\n",
              "      <td>Not provided.</td>\n",
              "      <td>[china, biggest, country, world, 1.3, billion,...</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>4684</th>\n",
              "      <td>2a3d080d5dcf6771db002366691f35e5d6f0e493</td>\n",
              "      <td>10.1007/s13238-010-0029-7</td>\n",
              "      <td></td>\n",
              "      <td>✉ Correspondence: zlshi@wh.iov.cn Bat, the onl...</td>\n",
              "      <td>Shi, Zhengli</td>\n",
              "      <td>Bat and virus</td>\n",
              "      <td>Protein &amp; Cell</td>\n",
              "      <td>Not provided.</td>\n",
              "      <td>[✉, correspondence, zlshi@wh.iov.cn, bat, flyi...</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>1731</th>\n",
              "      <td>2de657aabab331e75657bfc1443542fd345b6f26</td>\n",
              "      <td>10.1016/j.ccc.2019.06.001</td>\n",
              "      <td>KEYWORDS\\nPandemic Influenza Disaster prepared...</td>\n",
              "      <td>Outbreaks of infectious disease pose unique ch...</td>\n",
              "      <td>Maves, Ryan C..  Jamros, Christina M..  Smith...</td>\n",
              "      <td>Intensive Care Unit Preparedness During&lt;br&gt;Pa...</td>\n",
              "      <td>Crit Care Clin</td>\n",
              "      <td>KEYWORDS\\nPandemic Influenza Disaster&lt;br&gt;prep...</td>\n",
              "      <td>[outbreaks, infectious, disease, pose, unique,...</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>4742</th>\n",
              "      <td>bc22758234283f024f16e2dbe660bc909b512b92</td>\n",
              "      <td>10.1016/j.vetpar.2007.11.020</td>\n",
              "      <td>Cryptosporidium parvum is recognized as one of...</td>\n",
              "      <td>Diarrhoeal diseases of infectious aethiology r...</td>\n",
              "      <td>Klein, Pavel.  Kleinová, Tereza.  Volek,&lt;br&gt;Z...</td>\n",
              "      <td>Effect of Cryptosporidium parvum infection on...</td>\n",
              "      <td>Veterinary Parasitology</td>\n",
              "      <td>Cryptosporidium parvum is recognized as one o...</td>\n",
              "      <td>[diarrhoeal, diseases, infectious, aethiology,...</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>4521</th>\n",
              "      <td>a204aafa38365dbcc0a26af3ca2c6d3313d7fab2</td>\n",
              "      <td>10.1111/irv.12518</td>\n",
              "      <td>, Brazil, in the period of Background: The res...</td>\n",
              "      <td>The inclusion criteria for asymptomatic adults...</td>\n",
              "      <td>Moreira, Luciana Peniche.  Watanabe, Aripuana...</td>\n",
              "      <td>Respiratory syncytial virus evaluation among&lt;...</td>\n",
              "      <td>Influenza Other Respir Viruses</td>\n",
              "      <td>, Brazil, in the period of Background: The&lt;br...</td>\n",
              "      <td>[inclusion, criteria, asymptomatic, adults, ab...</td>\n",
              "    </tr>\n",
              "  </tbody>\n",
              "</table>\n",
              "</div>\n",
              "    <div class=\"colab-df-buttons\">\n",
              "\n",
              "  <div class=\"colab-df-container\">\n",
              "    <button class=\"colab-df-convert\" onclick=\"convertToInteractive('df-2cbeb672-3bb4-4dfa-8e54-1d505eb2bbbc')\"\n",
              "            title=\"Convert this dataframe to an interactive table.\"\n",
              "            style=\"display:none;\">\n",
              "\n",
              "  <svg xmlns=\"http://www.w3.org/2000/svg\" height=\"24px\" viewBox=\"0 -960 960 960\">\n",
              "    <path d=\"M120-120v-720h720v720H120Zm60-500h600v-160H180v160Zm220 220h160v-160H400v160Zm0 220h160v-160H400v160ZM180-400h160v-160H180v160Zm440 0h160v-160H620v160ZM180-180h160v-160H180v160Zm440 0h160v-160H620v160Z\"/>\n",
              "  </svg>\n",
              "    </button>\n",
              "\n",
              "  <style>\n",
              "    .colab-df-container {\n",
              "      display:flex;\n",
              "      gap: 12px;\n",
              "    }\n",
              "\n",
              "    .colab-df-convert {\n",
              "      background-color: #E8F0FE;\n",
              "      border: none;\n",
              "      border-radius: 50%;\n",
              "      cursor: pointer;\n",
              "      display: none;\n",
              "      fill: #1967D2;\n",
              "      height: 32px;\n",
              "      padding: 0 0 0 0;\n",
              "      width: 32px;\n",
              "    }\n",
              "\n",
              "    .colab-df-convert:hover {\n",
              "      background-color: #E2EBFA;\n",
              "      box-shadow: 0px 1px 2px rgba(60, 64, 67, 0.3), 0px 1px 3px 1px rgba(60, 64, 67, 0.15);\n",
              "      fill: #174EA6;\n",
              "    }\n",
              "\n",
              "    .colab-df-buttons div {\n",
              "      margin-bottom: 4px;\n",
              "    }\n",
              "\n",
              "    [theme=dark] .colab-df-convert {\n",
              "      background-color: #3B4455;\n",
              "      fill: #D2E3FC;\n",
              "    }\n",
              "\n",
              "    [theme=dark] .colab-df-convert:hover {\n",
              "      background-color: #434B5C;\n",
              "      box-shadow: 0px 1px 3px 1px rgba(0, 0, 0, 0.15);\n",
              "      filter: drop-shadow(0px 1px 2px rgba(0, 0, 0, 0.3));\n",
              "      fill: #FFFFFF;\n",
              "    }\n",
              "  </style>\n",
              "\n",
              "    <script>\n",
              "      const buttonEl =\n",
              "        document.querySelector('#df-2cbeb672-3bb4-4dfa-8e54-1d505eb2bbbc button.colab-df-convert');\n",
              "      buttonEl.style.display =\n",
              "        google.colab.kernel.accessAllowed ? 'block' : 'none';\n",
              "\n",
              "      async function convertToInteractive(key) {\n",
              "        const element = document.querySelector('#df-2cbeb672-3bb4-4dfa-8e54-1d505eb2bbbc');\n",
              "        const dataTable =\n",
              "          await google.colab.kernel.invokeFunction('convertToInteractive',\n",
              "                                                    [key], {});\n",
              "        if (!dataTable) return;\n",
              "\n",
              "        const docLinkHtml = 'Like what you see? Visit the ' +\n",
              "          '<a target=\"_blank\" href=https://colab.research.google.com/notebooks/data_table.ipynb>data table notebook</a>'\n",
              "          + ' to learn more about interactive tables.';\n",
              "        element.innerHTML = '';\n",
              "        dataTable['output_type'] = 'display_data';\n",
              "        await google.colab.output.renderOutput(dataTable, element);\n",
              "        const docLink = document.createElement('div');\n",
              "        docLink.innerHTML = docLinkHtml;\n",
              "        element.appendChild(docLink);\n",
              "      }\n",
              "    </script>\n",
              "  </div>\n",
              "\n",
              "\n",
              "<div id=\"df-2c64418b-4df5-485d-9b65-bc7a3268e4e9\">\n",
              "  <button class=\"colab-df-quickchart\" onclick=\"quickchart('df-2c64418b-4df5-485d-9b65-bc7a3268e4e9')\"\n",
              "            title=\"Suggest charts\"\n",
              "            style=\"display:none;\">\n",
              "\n",
              "<svg xmlns=\"http://www.w3.org/2000/svg\" height=\"24px\"viewBox=\"0 0 24 24\"\n",
              "     width=\"24px\">\n",
              "    <g>\n",
              "        <path d=\"M19 3H5c-1.1 0-2 .9-2 2v14c0 1.1.9 2 2 2h14c1.1 0 2-.9 2-2V5c0-1.1-.9-2-2-2zM9 17H7v-7h2v7zm4 0h-2V7h2v10zm4 0h-2v-4h2v4z\"/>\n",
              "    </g>\n",
              "</svg>\n",
              "  </button>\n",
              "\n",
              "<style>\n",
              "  .colab-df-quickchart {\n",
              "      --bg-color: #E8F0FE;\n",
              "      --fill-color: #1967D2;\n",
              "      --hover-bg-color: #E2EBFA;\n",
              "      --hover-fill-color: #174EA6;\n",
              "      --disabled-fill-color: #AAA;\n",
              "      --disabled-bg-color: #DDD;\n",
              "  }\n",
              "\n",
              "  [theme=dark] .colab-df-quickchart {\n",
              "      --bg-color: #3B4455;\n",
              "      --fill-color: #D2E3FC;\n",
              "      --hover-bg-color: #434B5C;\n",
              "      --hover-fill-color: #FFFFFF;\n",
              "      --disabled-bg-color: #3B4455;\n",
              "      --disabled-fill-color: #666;\n",
              "  }\n",
              "\n",
              "  .colab-df-quickchart {\n",
              "    background-color: var(--bg-color);\n",
              "    border: none;\n",
              "    border-radius: 50%;\n",
              "    cursor: pointer;\n",
              "    display: none;\n",
              "    fill: var(--fill-color);\n",
              "    height: 32px;\n",
              "    padding: 0;\n",
              "    width: 32px;\n",
              "  }\n",
              "\n",
              "  .colab-df-quickchart:hover {\n",
              "    background-color: var(--hover-bg-color);\n",
              "    box-shadow: 0 1px 2px rgba(60, 64, 67, 0.3), 0 1px 3px 1px rgba(60, 64, 67, 0.15);\n",
              "    fill: var(--button-hover-fill-color);\n",
              "  }\n",
              "\n",
              "  .colab-df-quickchart-complete:disabled,\n",
              "  .colab-df-quickchart-complete:disabled:hover {\n",
              "    background-color: var(--disabled-bg-color);\n",
              "    fill: var(--disabled-fill-color);\n",
              "    box-shadow: none;\n",
              "  }\n",
              "\n",
              "  .colab-df-spinner {\n",
              "    border: 2px solid var(--fill-color);\n",
              "    border-color: transparent;\n",
              "    border-bottom-color: var(--fill-color);\n",
              "    animation:\n",
              "      spin 1s steps(1) infinite;\n",
              "  }\n",
              "\n",
              "  @keyframes spin {\n",
              "    0% {\n",
              "      border-color: transparent;\n",
              "      border-bottom-color: var(--fill-color);\n",
              "      border-left-color: var(--fill-color);\n",
              "    }\n",
              "    20% {\n",
              "      border-color: transparent;\n",
              "      border-left-color: var(--fill-color);\n",
              "      border-top-color: var(--fill-color);\n",
              "    }\n",
              "    30% {\n",
              "      border-color: transparent;\n",
              "      border-left-color: var(--fill-color);\n",
              "      border-top-color: var(--fill-color);\n",
              "      border-right-color: var(--fill-color);\n",
              "    }\n",
              "    40% {\n",
              "      border-color: transparent;\n",
              "      border-right-color: var(--fill-color);\n",
              "      border-top-color: var(--fill-color);\n",
              "    }\n",
              "    60% {\n",
              "      border-color: transparent;\n",
              "      border-right-color: var(--fill-color);\n",
              "    }\n",
              "    80% {\n",
              "      border-color: transparent;\n",
              "      border-right-color: var(--fill-color);\n",
              "      border-bottom-color: var(--fill-color);\n",
              "    }\n",
              "    90% {\n",
              "      border-color: transparent;\n",
              "      border-bottom-color: var(--fill-color);\n",
              "    }\n",
              "  }\n",
              "</style>\n",
              "\n",
              "  <script>\n",
              "    async function quickchart(key) {\n",
              "      const quickchartButtonEl =\n",
              "        document.querySelector('#' + key + ' button');\n",
              "      quickchartButtonEl.disabled = true;  // To prevent multiple clicks.\n",
              "      quickchartButtonEl.classList.add('colab-df-spinner');\n",
              "      try {\n",
              "        const charts = await google.colab.kernel.invokeFunction(\n",
              "            'suggestCharts', [key], {});\n",
              "      } catch (error) {\n",
              "        console.error('Error during call to suggestCharts:', error);\n",
              "      }\n",
              "      quickchartButtonEl.classList.remove('colab-df-spinner');\n",
              "      quickchartButtonEl.classList.add('colab-df-quickchart-complete');\n",
              "    }\n",
              "    (() => {\n",
              "      let quickchartButtonEl =\n",
              "        document.querySelector('#df-2c64418b-4df5-485d-9b65-bc7a3268e4e9 button');\n",
              "      quickchartButtonEl.style.display =\n",
              "        google.colab.kernel.accessAllowed ? 'block' : 'none';\n",
              "    })();\n",
              "  </script>\n",
              "</div>\n",
              "\n",
              "    </div>\n",
              "  </div>\n"
            ],
            "text/plain": [
              "                                      paper_id  \\\n",
              "6252  fefd6cdcddcb00d1ba2ef1a0d3bba9f312010a1d   \n",
              "4684  2a3d080d5dcf6771db002366691f35e5d6f0e493   \n",
              "1731  2de657aabab331e75657bfc1443542fd345b6f26   \n",
              "4742  bc22758234283f024f16e2dbe660bc909b512b92   \n",
              "4521  a204aafa38365dbcc0a26af3ca2c6d3313d7fab2   \n",
              "\n",
              "                                   doi  \\\n",
              "6252  10.1111/j.1524-4733.2008.00379.x   \n",
              "4684         10.1007/s13238-010-0029-7   \n",
              "1731         10.1016/j.ccc.2019.06.001   \n",
              "4742      10.1016/j.vetpar.2007.11.020   \n",
              "4521                 10.1111/irv.12518   \n",
              "\n",
              "                                               abstract  \\\n",
              "6252                                                      \n",
              "4684                                                      \n",
              "1731  KEYWORDS\\nPandemic Influenza Disaster prepared...   \n",
              "4742  Cryptosporidium parvum is recognized as one of...   \n",
              "4521  , Brazil, in the period of Background: The res...   \n",
              "\n",
              "                                              body_text  \\\n",
              "6252  China is the biggest country in the world that...   \n",
              "4684  ✉ Correspondence: zlshi@wh.iov.cn Bat, the onl...   \n",
              "1731  Outbreaks of infectious disease pose unique ch...   \n",
              "4742  Diarrhoeal diseases of infectious aethiology r...   \n",
              "4521  The inclusion criteria for asymptomatic adults...   \n",
              "\n",
              "                                                authors  \\\n",
              "6252                   Li, Youing.  Sun, Xin.  Wang, Li   \n",
              "4684                                       Shi, Zhengli   \n",
              "1731   Maves, Ryan C..  Jamros, Christina M..  Smith...   \n",
              "4742   Klein, Pavel.  Kleinová, Tereza.  Volek,<br>Z...   \n",
              "4521   Moreira, Luciana Peniche.  Watanabe, Aripuana...   \n",
              "\n",
              "                                                  title  \\\n",
              "6252                   Evidence-Based Medicine in China   \n",
              "4684                                      Bat and virus   \n",
              "1731   Intensive Care Unit Preparedness During<br>Pa...   \n",
              "4742   Effect of Cryptosporidium parvum infection on...   \n",
              "4521   Respiratory syncytial virus evaluation among<...   \n",
              "\n",
              "                             journal  \\\n",
              "6252                 Value in Health   \n",
              "4684                  Protein & Cell   \n",
              "1731                  Crit Care Clin   \n",
              "4742         Veterinary Parasitology   \n",
              "4521  Influenza Other Respir Viruses   \n",
              "\n",
              "                                       abstract_summary  \\\n",
              "6252                                      Not provided.   \n",
              "4684                                      Not provided.   \n",
              "1731   KEYWORDS\\nPandemic Influenza Disaster<br>prep...   \n",
              "4742   Cryptosporidium parvum is recognized as one o...   \n",
              "4521   , Brazil, in the period of Background: The<br...   \n",
              "\n",
              "                                           cleaned_text  \n",
              "6252  [china, biggest, country, world, 1.3, billion,...  \n",
              "4684  [✉, correspondence, zlshi@wh.iov.cn, bat, flyi...  \n",
              "1731  [outbreaks, infectious, disease, pose, unique,...  \n",
              "4742  [diarrhoeal, diseases, infectious, aethiology,...  \n",
              "4521  [inclusion, criteria, asymptomatic, adults, ab...  "
            ]
          },
          "execution_count": 10,
          "metadata": {},
          "output_type": "execute_result"
        }
      ],
      "source": [
        "df.head()"
      ]
    },
    {
      "cell_type": "code",
      "execution_count": 44,
      "metadata": {
        "id": "7f9mPxRwjci8"
      },
      "outputs": [],
      "source": [
        "# Import necessary library\n",
        "from nltk.corpus import stopwords\n",
        "\n",
        "# Define the function to remove stop words\n",
        "def remove_stopwords(tokens, custom_stopwords):\n",
        "    return [token for token in tokens if token.lower() not in custom_stopwords]\n",
        "\n",
        "# Define the standard stop words set\n",
        "STOP_WORDS = set(stopwords.words('english'))\n",
        "\n",
        "# Add additional domain-specific stop words\n",
        "DOMAIN_SPECIFIC_STOP_WORDS = {'et', 'al', 'rt', 'mr'}\n",
        "\n",
        "# Combine both sets of stop words\n",
        "ALL_STOP_WORDS = STOP_WORDS.union(DOMAIN_SPECIFIC_STOP_WORDS)\n",
        "\n",
        "# Apply the remove_stopwords function to the 'cleaned_text' column of the dataframe\n",
        "df['cleaned_text'] = df['cleaned_text'].apply(lambda x: remove_stopwords(x, ALL_STOP_WORDS))"
      ]
    },
    {
      "cell_type": "code",
      "execution_count": 45,
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/",
          "height": 573
        },
        "id": "Q7GDznusO9il",
        "outputId": "f5c09abf-80b3-47a1-9e99-85a7065d74a1"
      },
      "outputs": [
        {
          "data": {
            "application/vnd.google.colaboratory.intrinsic+json": {
              "summary": "{\n  \"name\": \"df\",\n  \"rows\": 1454,\n  \"fields\": [\n    {\n      \"column\": \"paper_id\",\n      \"properties\": {\n        \"dtype\": \"string\",\n        \"num_unique_values\": 1454,\n        \"samples\": [\n          \"f78956e70c8bd1756ffb24ae47c35c3217bed384\",\n          \"51ad0f064355c4678f18225f7d231f9865f44a11\",\n          \"7400110e77de9824ff53797dfade054fa9a9cf80\"\n        ],\n        \"semantic_type\": \"\",\n        \"description\": \"\"\n      }\n    },\n    {\n      \"column\": \"doi\",\n      \"properties\": {\n        \"dtype\": \"string\",\n        \"num_unique_values\": 1411,\n        \"samples\": [\n          \"10.3201/eid1601.091376\",\n          \"10.1016/j.jcv.2011.12.020\",\n          \"10.1101/2020.04.14.040204\"\n        ],\n        \"semantic_type\": \"\",\n        \"description\": \"\"\n      }\n    },\n    {\n      \"column\": \"abstract\",\n      \"properties\": {\n        \"dtype\": \"string\",\n        \"num_unique_values\": 1044,\n        \"samples\": [\n          \"InnoMetrics is the system that aims at collecting software development process metrics in a non-invasive way to access and optimize the development process and its efficiency. This paper demonstrates the development and analysis of energy consumption of MacOS systems based on the software process measurement data. It represents the experience of the development of MacOS system collector and Transfer, in addition to the user interface and early analysis of energy consumption metrics calculations.\",\n          \"During the global outbreak of severe acute respiratory syndrome (SARS) in 2003, treatment was empiric. We report the case history of the index patient in a hospital outbreak of SARS in Hong Kong. The patient recovered after conventional antimicrobial therapy. Further studies are needed to address treatment of SARS, which has high attack and death rates.\",\n          \"Intestinal transplantation (IT) is the least common form of organ transplantation; however, it has shown exceptional growth and improvement in graft survival rates over the past two decades mainly due to better outcomes achieved during the first year of transplantation (76 % at 1 year), due to improvement in surgical techniques and the development of better immunosupressive therapies as we understand more about the relationship between the recipient and host immune system. There are still ongoing issues with chronic rejection and long-term survival. Intestinal transplantation is still an acceptable therapy for patients with intestinal failure (IF), but it is generally reserved for patients who develop severe and life-threatening complications despite standard therapies, or those who are not able to maintain a good quality of life. The purpose of this review is to describe the current status, indications, outcomes and advances in the field of intestinal transplantation.\"\n        ],\n        \"semantic_type\": \"\",\n        \"description\": \"\"\n      }\n    },\n    {\n      \"column\": \"body_text\",\n      \"properties\": {\n        \"dtype\": \"string\",\n        \"num_unique_values\": 1453,\n        \"samples\": [\n          \"Respiratory tract infection is common in infants and young children and is a major public health problem in this age group [1] . It is mainly caused by infection of seasonal viruses, such as respiratory syncytial virus (RSV) and rhinovirus [2] [3] [4] . Most infections manifest as an upper respiratory infection, while 20-40 % develop lower respiratory infection, and a small number (2 -3 %) of infected infants require hospitalization [5, 6] . This is of particular concern as infants with lower respiratory viral infection are at increased risk of developing recurrent wheezing or childhood asthma [7] [8] [9] [10] [11] [12] [13] . Several risk factors have been identified for the development of severe lower respiratory viral infections, such as premature birth, young age, being born in relation to the RSV season, underlying chronic lung disease, congenital heart disease, high parity, young maternal age and poor socioeconomic factors [14] [15] [16] . Recently, genetic polymorphisms have been identified which are associated with severe viral infection [17, 18] . Many of the identified genetic variants associated with severe lower respiratory tract infections are also associated with and pathways common to asthma, including innate immune genes involved in cytokine and chemokine signaling, and epithelial cell function [13, 17, [19] [20] [21] [22] [23] .\\nAs the clinical hallmark of asthma is wheezing, and therapy includes beta agonists, a number of clinical studies over the past 2 decades have examined the relationship between genetic variants in \\u03b2 2 -Adrenergic receptor (ADRB2) and asthma [24] [25] [26] [27] [28] [29] [30] . The polymorphism that resulted in the Gly16 allele was associated with decreased bronchodilator responses in early clinical studies; however, larger clinical studies have provided equivocal results [31] [32] [33] . Other genetic variants in the promoter region of ADRB2 have also been associated with asthma severity [28, 34] . Despite the significant interest in ADRB2 polymorphisms related to asthma, the relative contribution of ADRB2 genetic variants to severity of respiratory viral infections has not been established. As wheezing is also the clinical hallmark of lower respiratory viral infections, and variable response to bronchodilators has been demonstrated in infants hospitalized with RSV lower respiratory viral infections, it is possible that ADRB2 genetic variants also contribute to the severity of acute respiratory viral infections [35, 36] .\\nThe aim of the present study was to explore associations between the genetic variants in ADRB2 that have been associated with a response to asthma therapy during an acute asthma exacerbation and the severity of infant acute respiratory viral infections. We examined the relationship between 5 common polymorphisms in the promoter region and coding block of the ADRB2 gene and infant severity of respiratory infections in a prospective cohort of Caucasian and African American infants.\\nIn order to examine the association of genetic variants in the regulatory regions of the ADRB2 gene and the severity of viral respiratory infections during infancy, we conducted an analysis of infants enrolled in the Tennessee Children's Respiratory Initiative (TCRI), a prospective cohort of mother-infant dyads enrolled during an infant's acute viral respiratory illness over four respiratory viral seasons, 2004-2008. Term (\\u226537 weeks) and non-low birth weight (\\u22652275 g) infants who were otherwise healthy with no significant co-morbidities or cardio-pulmonary disease were eligible, with oversampling for hospitalization infants. Eighty five percent of enrolled subjects had at least one respiratory virus identified by polymerase chain reaction (RSV, rhinovirus, influenza, parainfluenza, Human Metapneumovirus, and coronavirus); 56 % had labconfirmed RSV infection. The rationale, methods, and detailed inclusion and exclusion criteria have been reported previously [37] . Although 630 infants were enrolled in TCRI, this study was limited to the 261 Caucasian and 113 African American infants for whom DNA was available. The protocol was approved by the Institutional Review Board of Vanderbilt University, and mothers provided informed consent for their infants. All parents provided written informed consent for both their and their child's study participation.\\nInfants included in this study had either upper respiratory tract infections or lower respiratory tract infections. The severity of acute respiratory infection was determined by experienced research nurses through medical record chart review using an ordinal bronchiolitis severity score (BSS). The score is an aggregate of assigned values ranging of 0-3 in categories of respiratory rate, room air oxygen saturation, the presence and extent of wheezing, and the presence and extent of flaring and retractions [37] [38] [39] . The BSS ranges from 0 to 12, with higher scores indicating more severe illness and a difference of 0.5 being a clinically significant difference [40] . We measured the BSS at the end of the outpatient visit or after discharge from an inpatient admission, and the most severe recorded value was used for analysis.\\nDNA was extracted by Vanderbilt's DNA core laboratory from whole blood or saliva samples (Oragene by DNA Genotek) by standardized protocols. All oragene samples were processed immediately after collection, while blood samples were either processed immediately or frozen before processing. Genotyping at five pre-specified loci (-2387, -2274, -1343, +46, and +79) of ADRB2 gene was done using Taqman SM\\u00a9 Genotyping Assays (Applied Biosystems, Foster City, CA, USA) with predetermined primers. The five single-nucleotide polymorphisms (SNP)s were rs1432623, rs11168068, and rs2400707 at the promoter region, and rs1042713 SNP and rs1042714 SNP at the coding region of ADRB2 gene. Probes for the Applied Biosystems assays used were labeled at their 5\\u2032end with either VIC\\u00ae or 6-carboxy-fluorescine (FAM) reporter dyes. The reaction components include: 2.5 \\u03bcl Taqman Universal PCR Master Mix (Applied Biosystems), 0.125 \\u03bcl or 0.250 \\u03bcl assay mix (Applied Biosystems) and approximately 5 ng of genomic DNA in a total volume of 10 \\u03bcl per single tube reaction. PCR amplification was performed in a thermal cycler (Techne TC-412) with an initial step of 95\\u00b0C for 10 min followed by 50 cycles of 92\\u00b0C for 15 s and 60\\u00b0C for 1 min. After amplification, the fluorescence of each sample was read on the ABI 7900HT (DNA Resources Core at Vanderbilt University) and analyzed with the Sequence Detection Software (Applied Biosystems). Five percent of samples were run as blind duplicates with 100 % concordance. Primers and probes for all SNPs are listed in Table 1 .\\nThe demographics and characteristics of the 374 infants included, with either upper respiratory infection or lower respiratory tract infection, are presented as frequencies and proportions for specific categorical variables and medians and interquartile ranges were calculated for continuous variables. All analyses were stratified by race (Caucasian or African American), as a surrogate measure of European or African ancestry, to reduce the known impact of population substructure which can produce spurious associations [41] . A pairwise linkage disequilibrium (LD) was estimated for the SNPs in ADRB2 using the standardized summary statistics D' and r, calculated by the HaploView program (Whitehead Institute for Biomedical Research, Cambridge, MA, USA). SNPs were tested for deviations from Hardy Weinberg proportions, also using Haploview [42] . The promoter, coding and combined promoter and coding SNPs were assigned to blocks using D' confidence interval of Gabriel et al. [43] . Individual haplotypes were estimated using the program PHASE v2.1.1 [44, 45] . The haplotype frequencies for each block were estimated using the expectation maximization (EM) algorithm, again using the HaploView program. Non-parametric Kruskal-Wallis test were applied to compare the difference in BSS among infants with each SNP genotype, as well as each haplotype genotype. Haplotype genotypes with frequency less than 2 % were excluded from haplotype analysis. We used a proportional odds model for the ordinal BSS to test the additive effect of haplotypes. Covariates included in the regression model included infant age at enrollment, gender, daycare exposure, secondhand smoke exposure, any prior history of breastfeeding, any siblings at home, and enrollment season. All analyses were conducted separately according to race. All analyses were performed using R-software version 2.11.1 (www.r-project.org). We used a two-sided 5 % significance level for all statistical inferences.\\nThere were in total 374 infants, 261 Caucasians and 113 African Americans, enrolled in the TCRI cohort with available DNA and at least one ADRB2 polymorphism tested ( Table 3 lists the allele frequencies of 5 SNPs separately for Caucasian and African Americans. In both Caucasians and African Americans, infants had slightly more T, T, and G alleles for the three promoter SNP rs1432623, (Figs. 1 and 2) . This non statistical difference was consistent within both Caucasians and African Americans. Among African Americans, there was a decreasing trend of the severity of acute respiratory infection with an increasing number of copies of C, C, and A allele of the three promoter SNPs rs1432623, rs11168068, and rs2400707, respectively (Fig. 1) . However, analyses with additive models indicated no statistical significance (p > 0.05).\\nAmong 2 5 (32) possible haplotype combinations, only four haplotypes with more than 2 % of frequency were identified (Table 4 and Fig. 3 ). There were two promoter haplotypes and three coding block haplotypes. There were no statistically significant differences in the promoter and coding block haplotype frequencies between Caucasian and African Americans. However, when all 5 SNPs were combined, the distribution of haplotype frequency was significantly different between Caucasians and African Americans, with genotype CCAAC more common in African Americans and rare in Caucasians (p < 0.001).\\nPromoter haplotype CCA was associated with a decreased BSS in African Americans in a dose dependent manner (p = 0.037 in the univariate analysis) (Fig. 4) . Infants without the CCA haplotype had the highest BSS (median 5.5, interquartile range [IQR]: 2.0, 8.0). Infants with one copy of CCA haplotype had a lower bronchiolitis severity score (median 4.0, IQR: 1.0, 7.5). Infants with 2 copies of the CCA haplotype had the comparatively lowest BSS (median 3.0, IQR: 1.0, 4.0). This dose dependent relationship persisted after adjusting for infant age, gender, daycare exposure, secondhand smoke exposure, prior history of breastfeeding, siblings at home, and enrollment season (adjusted odds ratio: 0.59, 95 % confidence interval [CI]: 0.36, 0.98, p = 0.042). This protective effect of CCA haplotype was consistent when we categorized the acute respiratory infection into upper respiratory infection and lower respiratory tract infection. In a similar dose dependent manner, infants with more copies of CCA haplotype were less likely to have lower respiratory tract infection compared with infants with less copies of CCA haplotype (p = 0.05). On the other hand, promoter haplotype TTG was associated with a higher BSS in African Americans (Additional file 1).\\nThe protective effect of CCA haplotype in African Americans was identified in all combined haplotypes with CCA combination (Figs. 5 and 6 ). However, due to the sample size, the effect was not statistically significant. In addition, all African American infants with a coding block haplotype GG had CCA combination in their promoter region; therefore, there was a similar additive protective effect of GG in African Americans (Additional file 2). There was no relationship between BSS and the other two coding block haplotypes in African Americans (Additional files 3 and 4). This was also true for the combined haplotypes (Additional files 5 and 6).\\nIn both Caucasian and African American infants, there were no significant relationships between BSS and coding block haplotypes except for GG haplotype among African Americans (Additional file 2). This was also true for the combined haplotypes. In general, there was no significant relationship between BSS and any haplotypes in Caucasian infants. There was no similar promoter haplotype CCA relationship with BSS in Caucasians as observed in the African Americans (Fig. 4 ).\\nIn this study we have identified a haplotype in the promoter region of the ADRB2 gene which is associated with a decreased BSS in African American infants. This promoter haplotype of the ADRB2 gene was not associated with the severity of respiratory infections in Caucasian infants. The common coding block polymorphisms, corresponding to amino acid changes at codons 16 and 27, were not associated with a protective effect on severity of respiratory infections in Caucasian or African American infants. Our results agree with the studied conducted by Janssen and colleagues, who found no relationship between the SNP rs1042713 and susceptibility to RSV bronchiolitis in a European population [17] . This observation that the CCA promoter region haplotype is associated with a protective effect on respiratory viral infection severity is similar to our observation in a cohort of adults hospitalized for an asthma exacerbation, where this haplotype was associated with spirometric improvement in response to asthma-specific therapy that included beta-agonists and corticosteroids [34] . In that study, we noted spirometric improvement only in Caucasian patients, although the small number of African American non-responders in that cohort may have limited our ability to detect a similar effect in African Americans [34] .\\nThe mechanism by which a haplotype in the promoter region of the ADRB2 gene is associated with a decreased BSS is not known. However, we can speculate that [46] . Moore et al. demonstrated that ADRB2 genetic variants are associated with differences in ADRB2 expression in cultured human airway smooth muscle (HASM) cells, an in vitro system used as a model for changes in the human airway [47] These studies were performed prior to identification of the complex promoter haplotypes, and we are not aware that these haplotypes have been studied in HASM. Small differences in airway tone could thus be important in the setting of acute viral respiratory infections.\\nAlthough beyond the scope of the current observational study, we also speculate that ADRB2 genotype might also influence response to bronchiolitis-specific therapy. The primary goal of TCRI was to establish a longitudinal, prospective investigation of infants and their biological mothers on the acute and the long-term health consequences of varying viral respiratory tract infections on early childhood outcomes [37] . Although a BSS was assigned for each infant at acute care visit or hospital admission, we did not study or quantify the effects of different treatments, including beta-agonist therapy. A number of clinical studies have suggested that some infants with bronchiolitis appear to improve with bronchodilator therapy, while on average it is not efficacious [48] [49] [50] . As ADRB2 is the primary target of bronchodilator therapy, variation in response to beta-agonists might reflect genetic variants in ADRB2.\\nThe diversity of SNPs of the ADRB2 gene was explored by Drysdale et al., using immortalized lymphocytes from 23 Caucasians, 19 African-Americans, 20 Asians, and 15 Hispanic-Latinos [27] . This report from (-2387, -2274 , and -1343) and in the coding block SNPs discussed in this manuscript [28] . Such frequency distributions observed in our study are consistent with the allele frequencies reported in the reference population of the International HapMap Project [51] and Hawkins et al. [28] . Our stratified analyses by race therefore allow us to detect the association between severity of a respiratory illness and haplotype frequency within each racial group, and to minimize the spurious associations due to such population substructure. We recognize that the significance of our findings is limited by our small sample size and in classifying infants in this study by a severity score rather than by measuring some direct response to bronchodilator therapy. However, BSS correlated with the severity of disease (classified as upper respiratory and lower respiratory illness) well. Our sensitivity analyses with the severity of disease (upper versus lower respiratory illness) showed consistent results indicated that BSS is a valid surrogate of disease severity. Lastly, although a p value of 0.042 is at the threshold of statistical significance, this p value reflects sample size of 113 African Americans in our study. Whether the association between BSS and CCA haplotype in African Americans is powerful or not depends upon not only p value, but also upon the effect size such as odds ratio and the clinical significance of the findings. As a difference of 0.5 in BSS is considered as clinically significant, a 65 % reduced odds of developing a more severe disease (higher BSS) comparing 2 copies of CCA with 0 copy of CCA might also be clinically significant and meaningful.\\nWe hope that this hypothesis-generating study will encourage the incorporation of pharmacokinetic data in future studies of bronchiolitis. Although validation studies in other populations are needed, this study underscores that the development of personalized approaches to the treatment of bronchiolitis may benefit from the analysis of specific genetic variants.\\nIn summary, we have observed that promoter region haplotype of ADRB2 CCA is associated with a protective effect on respiratory illness severity in African American infants. In this era of personalized medicine, ADRB2 genotype may be predictive of the severity of respiratory viral infection and potential response to therapy for the most common acute infant illness and one for which we currently have no therapy. Next steps would be to validate these findings, as well as evaluate available clinical trial data for which DNA might be available. Fig. 6 BSS distribution stratified by copy number of combined haplotype CCAAC in African Americans infants. The scatter plots and the box-and-whisker plots of BSS across 0, 1, and 2 copies of combined haplotype CCAAC for African American infants. P values were obtained from multivariable regression model adjusted for infant age at enrollment, gender, daycare exposure, secondhand smoke exposure, any prior history of breastfeeding, any siblings at home, and enrollment season\",\n          \"Acute bronchiolitis, a viral infection of the lower respiratory tract, is one of the most substantial health burdens for infants and young children worldwide. 1 Respiratory syncytial virus is the most prevalent viral cause of bronchiolitis in infants. Estimates suggest that about 34 million new cases of lower respiratory infection due to respiratory syncytial virus occur globally in children younger than 5 years, with 3\\u00b74 million admissions to hospitals and about 199 000 deaths per year, predominantly in the developing world. 2 In developed countries such as the USA, bronchiolitis is the most common reason for admission to hospital in the fi rst 12 months of life, 3 accounting for approximately 100 000 infant admissions annually. Although admissions to hospital have declined from 2000 to 2010, emergency department visits have increased, in addition to increased use of mechanical ventilation and hospital charges. 4, 5 The clinical management remains challenging despite the frequency, global reach, economic cost, and morbidity and mortality associated with bronchiolitis. Several treatment strategies (including bronchodilators and corticosteroids) showed no eff ect in pooled metaanalyses, making supportive care the hallmark of current therapy. In this Seminar, we aim to summarise the current evidence for the epidemiology, pathophysiology, diagnostic approach, and management of acute viral bronchiolitis.\\nBronchiolitis is a seasonal infection, with the season typically beginning in late October in the temperate northern hemisphere, peaking in January or February, and ending in April. 6 Globally, independent of region, respiratory syncytial virus infection peaks consistently during annual or biannual epidemics. Although the peak and duration of these epidemics vary worldwide, they are consistent year-to-year within a country. 7 Some data suggest that climate might also be associated with prevalence of respiratory syncytial virus infection, with global surveillance suggesting that infection peaks during wet months in areas with high precipitation and during cooler months in hot regions. 7 Indoor crowding in population-dense areas during rainy seasons or cooler months might be one factor that facilitates viral transmission. 8 Additionally, weather-related factors, such as inhalation of cold and dry air that might impair ciliary function, the airway mucosa, and inhibition of temperature-dependent antiviral responses, might infl uence both disease transmission and severity. 9, 10 Altitude, climate, and meteorological conditions (such as wind speed and dew point) have been shown to have a modest association with bronchiolitis. 11, 12 Furthermore, air pollutants, such as ozone and traffi c pollutants, have been associated with exacerbations of respiratory infections in children younger than 5 years. [13] [14] [15] Environmental tobacco smoke has been associated with increased risk for respiratory syncytial virus-attributable admission to hospital and disease severity in those admitted. 16, 17 As with other respiratory viral infections, the risk of severe respiratory syncytial virus bronchiolitis might be greater in boys than in girls. 18, 19 This diff erence might be due to diff erences in lung and airway development, and by genetic factors. 8, 20 Pathophysiology Bronchiolitis is characterised by extensive infl ammation and oedema of the airways, increased mucus production, and necrosis of airway epithelial cells. 21 Respiratory syncytial virus binds to epithelial cells and replicates, resulting in epithelial necrosis and ciliary destruction. 21, 22 The cell destruction triggers an infl ammatory response with proliferation of polymorphonuclear cells and lymphocytes. The submucosa and adventitial tissues become oedematous with increased mucus secretion. 22 Plugs composed of cellular debris and mucus form in the bronchiole lumens leading to bronchiolar obstruction, air trapping, and diff erent degrees of lobar collapse. 21 Microbiology Molecular testing has led to an improved understanding of the viruses associated with bronchiolitis. Respiratory syncytial virus remains the most commonly identifi ed virus, detected in 41-83% of patients. [23] [24] [25] [26] [27] [28] Other viruses associated with bronchiolitis include rhinovirus, metapneumovirus, coronavirus, human bocavirus, infl uenza virus, adenovirus, and parainfl uenza virus. 24, 25, 27, 29, 30 Studies have investigated whether severity of illness, as measured by need for hospital admission, length of hospital stay, intensive care unit admission, repeated emergency department visits, and apnoea, is associated with specifi c viral infections or co-infections, but the evidence is confl icting. Data from some studies have shown that in infections involving a single virus, respiratory syncytial virus is associated with a more severe course compared with other viruses. 31, 32 Up to 30% of children with bronchiolitis are found to have co-infections with two other viruses, 29 with the combination of respiratory syncytial virus and rhinovirus being the most commonly reported. 29 Some evidence suggests that co-infection in bronchiolitis, particularly respiratory syncytial virus in combination with rhinovirus or metapneumovirus, could be associated with a more severe disease course compared with infection by a single virus. 25, 30, 31, 33, 34 However, other studies do not confi rm this association. 24, 26 Furthermore, although use of nucleic acid amplifi cation tests has greatly improved our ability to detect viruses present in respiratory infections, studies using these technologies have also found at least one respiratory virus in up to 30% of children younger than 6 years with no respiratory symptoms. [35] [36] [37] These viruses might be detected because of asymptomatic colonisation, incubation before clinical infection, or prolonged viral shedding post-infection. The confl icting evidence and high prevalence of respiratory viruses in asymptomatic children suggest no indication at this time that management should vary based on presumed viral cause and presence, or absence of viral co-infections.\\nThe diagnosis of bronchiolitis is clinical and thus requires a clinician to recognise signs and symptoms of viral lower respiratory tract infection in young children. Peak incidence occurs between 3 months and 6 months of age. 38 Since the early clinical defi nition by Court, and as noted in recent practice guidelines, the most specifi c defi nition is in infants. [38] [39] [40] Although the same physiology can occur in toddlers older than 12 months, many clinical trials have excluded these children or have included them as a small subgroup of patients. Bronchiolitis in toddlers can overlap with other conditions such as viral-induced wheezing and asthma, and application of evidence from trials predominantly assessing infants might not be appropriate. Further eff orts to focus the defi nition might assist eff orts to standardise care.\\nThe classic clinical presentation of bronchiolitis starts with symptoms of a viral upper respiratory infection, such as nasal discharge, that progress to the lower respiratory tract over several days (fi gure 1). Timing of symptom progression can vary, and young infants can present with apnoea. Lower respiratory tract symptoms of bronchiolitis include persistent cough, tachypnoea, and increased work of breathing, as shown by intercostal or supraclavicular retractions, use of abdominal muscles, grunting, or nasal fl aring. Auscultatory fi ndings include crackles and wheeze. A hallmark characteristic of bronchiolitis is the minute-to-minute variation in clinical fi ndings, as mucus and debris in the airways are cleared by coughing or as the child's state changes from sleep to agitation. This variation can confound assessment, and often requires several examinations over a period of observation. Nasal congestion can also confound the clinical assessment. Nasal suctioning might help to ascertain which fi ndings are truly from the lower airways. Fever can be present in about a third of infants with bronchiolitis, but it is usually present early in the illness with a temperature less than 39\\u00b0C. 38 The median duration of symptoms is about 2 weeks, with 10-20% of infants still having symptoms at 3 weeks after onset. 38, 41 Various clinical scores have been shown in studies and clinical protocols to correlate with disease severity and improvement. 42, 43 Although documentation of a score can be useful as an objective measure, individual scores are not highly predictive, and they should be repeated and combined with other measures of severity for a universal assessment to guide decision making. The diff erential diagnosis for bronchiolitis includes considerations of various infectious and non-infectious causes. Absence of upper respiratory symptoms should raise suspicion of other causes of respiratory distress in young infants, including cardiac disease, congenital airway abnormalities such as a vascular ring, or foreign body aspiration. Other infections can resemble or complicate bronchiolitis. Pertussis should be considered in infants with severe or paroxysmal cough, or with known exposure. Bacterial infections complicating viral bronchiolitis, including otitis media or pneumonia, might present as a new fever or worsening status later in the course of illness.\\nVarious risk factors have been associated with progression to severe bronchiolitis. Those supported by the strongest evidence include presence of chronic lung disease of prematurity and haemodynamically important congenital heart disease, with immunodefi ciency and neuromuscular disorders also considered as high risk in practice guidelines. 38, 39 Young infants (aged <2-3 months) and those with a history of premature birth (especially <32 weeks' gestation) are also at high risk for progression and can present with apnoea without other clinical fi ndings. Studies assessing the risk for further apnoea in hospital have found it to be limited to infants less than 1 month for full-term infants, 48 weeks postconceptional age for preterm infants, or those with apnoea observed before admission. 44\\nBronchiolitis is a clinical diagnosis based on history and physical examination, according to consensus across national guidelines. For infants with a typical presentation of bronchiolitis, routine imaging or laboratory testing is not recommended, as they increase costs without evidence for benefi t (table).\\nAppropriate use of pulse oximetry monitoring and initiation of oxygen for bronchiolitis have received increasing attention in studies and practice guidelines. Findings suggest that arbitrary thresholds for oxygen administration might drive hospital admissions and prolong hospital length of stay. These outcomes represent only part of the morbidity of bronchiolitis, but developing evidence suggests that intermittent hypoxaemia might occur commonly in otherwise stable infants with bronchiolitis and raises questions as to whether this factor should be used as a sole indication for admission to hospitals. A Canadian randomised trial found reduced admissions to hospital from the emergency department without any increase in revisits when pulse oximeters displayed values 3% higher than the actual value, suggesting that arbitrary pulse oximetry thresholds result in unnecessary admissions. 51 A similar trial in the UK in the hospital setting found that reduction of the oxygen threshold from 94% to 90% resulted in earlier discharge from the hospital without any evidence of adverse outcomes. 52 A US trial comparing intermittent versus continuous pulse oximetry in non-hypoxaemic infants in hospitals found similar outcomes between the groups. 53 This US trial and other evidence supports recommendations in US practice guidelines that clinicians use a threshold of 90% for initiation of oxygen, whereas UK guidelines recommend 92%. 38, 39 As the child improves, reduction in intensity of monitoring to intermittent checks is appropriate. A recent study using blinded oximetry at home showed that a substantial proportion of infants with bronchiolitis who are otherwise doing well have oxygen desaturations less than 90%, particularly during sleep, further calling into question arbitrary thresholds for hospital admissions and initiation of oxygen. 54 This evidence will probably lead to eff orts to reduce continuous monitoring in children without other indications for monitoring.\\nThe majority of children with bronchiolitis have either normal radiographs or radiographic fi ndings consistent with simple bronchiolitis, 55-58 including peribronchial thickening, hyperinfl ation, and atelectasis. One prospective study 58 of routine radiographs as part of the assessment for bronchiolitis in the emergency department reported airspace disease in 17 (7%) of 246 patients. Despite the low prevalence of radiographic pneumonia, this and other studies have reported an increase in antibiotic prescription after radiographs are performed, because of non-specifi c fi ndings that infl uence clinicians' decisions. 58, 59 Factors that have been associated with defi nite focal infi ltrates consistent with pneumonia include hypoxia (oxygen saturation <92%), [56] [57] [58] 60 grunting, persistently focal crackles, and fever (especially >39\\u00b0C). 38 Chest radiographs should only be considered in patients when the presentation is not classic for bronchiolitis. These situations include when another diagnosis (such as foreign body aspiration) is high on the diff erential diagnosis, when a child is severely ill and respiratory failure is imminent, and when symptoms are progressing or not resolving according to the typical disease course expected for bronchiolitis. Lung ultrasound is increasingly used to assess cardiopulmonary conditions in adults and children. Several studies have investigated the use of lung ultrasound in the diagnosis of bronchiolitis. Two small studies found that ultrasound fi ndings in infants with bronchiolitis correlate with clinical fi ndings, and might be more specifi c than chest radiography, 61,62 but further studies are needed to establish whether there is a role for ultrasound in diagnosis or assessment of severity.\\nWith the development of PCR to detect respiratory viruses in the nasopharynx, interest in the use of viral testing for causative diagnosis in bronchiolitis has increased. Virological testing, however, does not generally assist in management and is insuffi cient to predict outcomes. 30, 63 Many national guidelines therefore recommend against routine virological testing in bronchiolitis (table). Recent studies suggest that higher respiratory syncytial virus genomic load, measured using quantitative PCR, might be associated with increased length of stay, use of respiratory support, and need for intensive care, in addition to recurrent wheezing, compared with lower viral loads. 28, 31, 32, 64 Further study is warranted to confi rm this association and clarify whether viral load measurement improves understanding of disease pathophysiology and severity. Several guidelines recommend using respiratory syncytial virus testing to guide cohorting of patients; however, the viruses most likely to cause bronchiolitis are all transmitted in a similar fashion (close contact with large-particle aerosols or direct contact with contaminated fomites). [65] [66] [67] Thus, infection control might not be dependent on the identifi cation of specifi c viruses, but rather on following strict precautions including hand hygiene, separating infants in shared hospital rooms by more than 1 m, and other infection control procedures. 65 Additionally, given the sensitivity of PCR testing, results should be interpreted with caution. Certain viruses, such as rhinovirus, might be detected because of viral shedding from an unrelated illness or colonisation; whereas certain other viruses, such as respiratory syncytial virus and metapneumovirus, are almost always associated with an acute infection.\\nBlood and urine testing is not routinely recommended as part of standard practice in the diagnostic work-up of bronchiolitis (table) . A blood gas measurement should not be routinely obtained in infants with bronchiolitis, unless there are signs of impending respiratory failure or severe distress. Proportions of serious bacterial infections, especially bacteraemia and meningitis, are very low in infants with bronchiolitis. 68 Abnormal white blood cell count is rarely useful in predicting serious bacterial infections in children infected with respiratory syncytial virus. 69 Guidelines universally do not recommend complete blood counts in infants with bronchiolitis unless blood count is part of assessment for a fever in infants younger than 1-2 months. Similarly, given that bacteraemia is exceedingly rare (with cited proportions of <0\\u00b71% in the post-pneumococcal vaccine era), blood cultures should not be routinely performed, except in the septic work-up of infants younger than 1-2 months, 70, 71 or in those with severe illness and signs of sepsis. Hydration status is an important consideration in infants with bronchiolitis and should be determined by clinical examination. Routine measurement of serum electrolytes is of little value in the majority of infants.\\nUrinary tract infections in infants with bronchiolitis occur with greater frequency than do bacteraemia and meningitis, with proportions ranging from 1% to 7%. 68, 71, 72 It is reasonable to obtain a urinalysis and urine culture for infants aged less than 60 days with fever and for older febrile infants who have risk factors for urinary tract infections, 73 but urine should not be routinely obtained in all infants with bronchiolitis.\\nCurrent recommendations for management of bronchiolitis focus on agents to treat the patho physiological eff ects of viral lower respiratory infection (eg, bronchodilators and hypertonic saline). Specifi c antivirals such as ribavirin to treat respiratory syncytial virus infection are not recommended in practice guidelines for typical cases of bronchiolitis because of challenging delivery methods, high cost, and potential health risks to caregivers. Multiple new agents for prevention and treatment are under investigation and might become available in the future, including immunoglobulins, small interfering RNA interference, fusion inhibitors, and small molecules. 74 Numerous studies have assessed the role of bronchodilators for the treatment of bronchiolitis, and systematic reviews have found no consistent benefi t. A 2014 Cochrane Collaboration systematic review identifi ed 30 studies assessing bronchodilators, predominantly salbutamol and excluding epinephrine, and 21 studies that looked specifi ally at clinical scores found no evidence of benefi t in any outcomes for infants admitted to hospitals. 75 In outpatients, oxygen saturation, admission to hospital, or time to resolution of symptoms did not improve with bronchodilator usage compared with placebo. For outpatient studies assessing short-term change in pooled clinical scores, the reviewers found a small signifi cant diff erence in mean score (Z=2\\u00b726; p=0\\u00b7024) that was of small eff ect with minimal clinical importance (fi gure 2). Outpatient studies were heterogeneous (I\\u00b2=81%; p<0\\u00b700001), and those showing benefi t in scores tended to include older children and children with recurrent wheezing.\\nNebulised epinephrine was assessed in another Cochrane Collaboration systematic review. 76 This review found no benefi t for epinephrine compared with placebo for inpatients in hospital length of stay or other outcomes. A multicentre Scandinavian study published after this Cochrane review found that inpatients receiving standing doses of epinephrine had longer length of stay compared with inpatients receiving as-needed epinephrine or placebo. 77 For outpatients, the Cochrane review found a diff erence in the numbers of admissions associated with epinephrine treatment during the time of an emergency department visit, but not during the overall course of illness when assessed at 1 week. Clinical practice guidelines including those from the USA, UK, and Canada do not recommend treatment with bronchodilators for bronchiolitis because of this evidence (table) . 38 \\nNebulised hypertonic saline is thought to reduce airway oedema, decrease mucus plugging, improve mucociliary clearance, and rehydrate the airway surface liquid in infants with bronchiolitis. 78 These physiological changes are extrapolated from the cystic fi brosis literature, 79, 80 and the pathophysiological processes in acute bronchiolitis are diff erent. Therefore, the theoretical benefi ts of hypertonic saline seen in cystic fi brosis might not be present in infants with acute viral bronchiolitis. Although initial trials demonstrated some ability of hypertonic saline to decrease hospital length of stay and transiently improve clinical severity score, 81-83 more recent trials demonstrated confl icting results. [84] [85] [86] [87] [88] [89] [90] The trials that showed the largest benefi t were done in hospitals with lengths of stay more than 72 h; thus, hypertonic saline for infants in countries and institutions in which the length of stay approaches or exceeds 72 h might be benefi cial at reducing length of stay. The confl icting results are \\nTest for heterogeneity in the inpatient studies demonstrated low inconsistency between the nine studies (I\\u00b2=36%; p=0\\u00b713) and the summary eff ect was not signifi cant (Z=1\\u00b706; p=0\\u00b729), the outpatient studies demonstrated very high inconsistency between the 12 studies (I\\u00b2=81%; p<0\\u00b700001) and the summary eff ect was signifi cant (Z=2\\u00b726; p=0\\u00b7024), and the overall heterogeneity of the meta-analysis demonstrated high inconsistency between the 21 studies (I\\u00b2=73%; p<0\\u00b700001) and the overall summary eff ect was signifi cant (Z=2\\u00b74; p=0\\u00b7016). IPR=ipratropium. SAL=salbutamol. neb=nebulised. Reproduced from Gadomski and Scribani, 75 refl ected in the diff erences in recommendations across national guidelines (table), with some countries not recommending hypertonic saline, some recommending use for all inpatients, and some recommending use only in moderate to severe illness. The largest systematic review and meta-analysis, published in 2015, examined 24 trials and 3209 patients. 91 Infants receiving hypertonic saline had a signifi cant diff erence in hospital length of stay of -0\\u00b745 days (95% CI -0\\u00b782 to -0\\u00b708; p=0\\u00b701) compared with those receiving 0\\u00b79% saline or standard care. In seven trials, hypertonic saline reduced the risk of admission to hospital by 20% (risk ratio 0\\u00b78; 95% CI 0\\u00b767-0\\u00b796) compared with 0\\u00b79% saline. No substantial adverse eff ects of hypertonic saline were noted in this systematic review. Although the results of this meta-analysis were signifi cant, attention should be paid to the outer limits of the confi dence intervals, which approach no diff erence, and to the clinical relevance of these diff erences. A second 2015 meta-analysis of 15 trials and 1922 patients found a smaller, but signifi cant, decrease in length of stay by -0\\u00b736 days (95% CI -0\\u00b75 to -0\\u00b722; p<0\\u00b7001) in those who received hypertonic saline (fi gure 3). 92 This study found a discrepancy between the overall combined eff ect of all studies on length of stay and the negative results from the largest trials, allowing the authors to conclude that neither individual trials nor pooled estimates provide a strong evidence-based foundation for the use of hypertonic saline. Both meta-analyses showed substantial heterogeneity across studies (I\\u00b2=82\\u00b71%, p<0\\u00b7001; 91 and I\\u00b2=77\\u00b78%, p=0\\u00b7029 92 ). A recent reanalysis of the fi rst 2015 meta-analysis removed two outlying Chinese studies and accounted for imbalances in day of illness at presentation. 93 These analyses resolved the heterogeneity and found that hypertonic saline does not reduce length of stay in infants admitted to hospitals with bronchiolitis (mean diff erence in length of stay removing outliers -0\\u00b721 days; 95% CI -0\\u00b743 to 0\\u00b702; mean diff erence in length of stay accounting for day of illness imbalance 0\\u00b702 days; 95% CI -0\\u00b714 to 0\\u00b717). Large trials have not demonstrated benefi t for hypertonic saline and this meta-analysis found no eff ect of hypertonic saline on length of stay after adjustment for outliers and imbalances. The decision to undertake future trials is controversial given the positive results of some metaanalyses and negative results of others.\\nMultiple studies have examined the role of corticosteroids in the management of children with bronchiolitis. Data from two large multicentre trials have shown no benefi t to corticosteroids alone in reducing admissions to hospital, 94, 95 and a 2013 Cochrane Collaboration review supports the results of these studies. 96 This review included only studies that enrolled children younger than 24 months with a fi rst episode of wheezing and signs of a viral illness. Among the included eight outpatient studies (1824 participants) comparing corticosteroids with placebo there was no reduction in admission at day 1 (Z=1\\u00b705; p=0\\u00b73) or day 7 (Z=1\\u00b738; p=0\\u00b717) after enrolment (fi gure 4), clinical scores, length of stay in the emergency department, or length of time to resolution of symptoms. Among the nine inpatient studies (772 participants), length of hospital stay was not reduced. 96 On the basis of this evidence, multiple clinical practice guidelines recommend against the use of corticosteroids for infants with bronchiolitis (table) . Although clinicians report considering a family or personal history of atopy when deciding whether to treat infants with bronchiolitis with corticosteroids, 41 there is no evidence that such infants receive any benefi t from corticosteroid treatment. [94] [95] [96] Evidence for the presence or absence of respiratory syncytial virus infection in these infants being associated with a response to corticosteroids is also unavailable. 94, 95 Authors from a large study with a factorial design have suggested, in an unadjusted analysis, that high-dose corticosteroids in combination with nebulised epinephrine might reduce admissions for outpatients with bronchiolitis by day 7, 95 but these results are considered exploratory. 39, 45 High-fl ow oxygen and respiratory support\\nNon-invasive technologies to improve oxygenation and ventilation for bronchiolitis include humidifi ed high-fl ow nasal cannula oxygen and continuous positive airway pressure. 97 High-fl ow nasal cannula allows delivery of high fl ows (usually 1-2 L/kg per min) with humidifi cation and a cannula designed to improve patient tolerance. It has been used widely in premature infants, but the mechanisms of action are unclear, in particular whether it might deliver positive end-expiratory pressure in some conditions. Evidence for effi cacy of high-fl ow nasal cannula is predominantly observational, with studies documenting Corneli (2007) Goebel (2000) Kuyucu (2004) Plint (2009) improved respiratory parameters and reduced intubation rates after implementation. 97 One small randomised trial compared high-fl ow nasal cannula with hypertonic saline and found no diff erence in change in respiratory score. 98 Concerns about high-fl ow nasal cannula include the potential for rapid deterioration if the infant is not closely monitored and costs associated with overuse. Continuous positive airway pressure has been studied in intensive care settings in observational studies and several small trials, with some evidence of improved respiratory parameters. 97 The UK guidelines recommend considering continuous positive airway pressure in children with impending respiratory failure from bronchiolitis.\\nAntibiotic overuse in children with bronchiolitis probably occurs because of concerns about the presence of fever, the young age of aff ected patients, diffi culty diff erentiating atelectasis from infectious consolidation on chest radiograph, and concern for undetected secondary bacterial infection. Bronchiolitis, however, has a clear viral cause and the occurrence of secondary bacterial infections is low, with a risk of bacteraemia or meningitis of less than 1%. 99 A detailed review of randomised clinical trials found that routine use of antibiotics did not improve duration of symptoms, length of hospital stay, need for oxygen therapy, or hospital admission. 38 Overuse of antibiotics is known to result in unnecessary adverse eff ects on the patient, and the development of antimicrobial resistance. Routine use should be avoided unless there is clear evidence of a secondary bacterial infection (table) . Acute otitis media has been documented in up to 60% of infants with bronchiolitis. 100, 101 Antibiotic use for acute otitis media in bronchiolitis should follow established evidence and guidelines for acute otitis media. 102 Macrolide antibiotics have anti-infl ammatory properties that might have potential benefi t in mitigating the infl ammation present in bronchiolitis. Two randomised trials found that there was no diff erence between azithromycin and placebo in hospital length of stay, need for oxygen, or hospital re-admission. 103, 104 Another randomised trial found that azithromycin lowered nasal interleukin-8 concentrations, prolonged time to subsequent wheezing episodes, and resulted in fewer days with respiratory symptoms in the year following the bronchiolitis episode compared with placebo. 105 Finally, a US multicentre study found that in children aged 12-72 months with a history of recurrent severe lower respiratory tract infections, early administration of azithromycin during a lower respiratory tract infection reduced the likelihood of progression to a severe infection, but it is not clear whether the underlying disease in these children was bronchiolitis or some other disease process. 106 Given the current evidence, routine use of macrolides is not recommended in bronchiolitis and more research is needed to clarify any potential role it might have in the future.\\nHydration, suctioning, and chest physiotherapy have been suggested as supportive therapies. Infants with bronchiolitis might have diffi culty feeding because of nasal congestion and increased work of breathing; thus, hydration remains a cornerstone of therapy. A multicentre study of 759 infants younger than 12 months admitted to hospital with bronchiolitis showed no benefi t of intravenous fl uids compared with administration of fl uids by nasogastric tube in mean length of stay, admission to the intensive care unit, need for ventilatory support, and adverse events. This trial also found that a nasogastric tube might be easier to place than an intravenous line in these infants. 107 Most guidelines recommend either nasogastric or intravenous fl uids to maintain hydration, with the UK and Scottish guidelines preferring nasogastric or orogastric hydration in those that can tolerate it compared with intravenous hydration (table). If intravenous fl uids are used, isotonic fl uids are preferred to avoid risk of hyponatraemia. 38 Because infants are obligate nasal breathers, nasal suctioning has been suggested to help with clearing of the nares, improve the work of breathing, and improve feeding; however, suctioning might irritate the nasal mucosa and result in oedema. No randomised controlled trials have examined the role of nasal suctioning in bronchiolitis. The insuffi cient available evidence includes a retrospective cohort study of 740 infants 108 and a small observational study of 40 infants. 109 These studies suggest that deep suctioning might increase length of stay for inpatients, 108 infrequent suctioning is associated with an increased length of stay, 108 and oxygen saturation might increase after suctioning. 109 To draw conclusions about causality from these observational studies is diffi cult, because the potential for confouding by indication exists (eg, sicker children might be more likely to receive deep suctioning). Evidence suggests that oxygen saturation increases after nasal irrigation even without suctioning. 110 Current guidelines give diff ering recommendations with regard to suctioning; those that support its use recommend only superfi cial suctioning rather than deep suctioning (table) . 38, 39, 45 Chest physiotherapy use in bronchiolitis appears to vary by country. 111, 112 A recent Cochrane Collaboration review of 12 studies (1249 participants) demonstrated no evidence of benefi t to any type of chest physiotherapy among inpatients in length of stay, oxygen saturation, or respiratory parameters. 113 No published guidelines routinely recommend chest physiotherapy for the management of uncomplicated bronchiolitis in otherwise healthy children without respiratory comorbidities (table) . 38, 39, 45 Prognosis Much work has been published about the risk of developing recurrent wheezing and asthma following bronchiolitis in infancy. [114] [115] [116] Studies have followed birth cohorts to determine the risk of subsequent wheezing after lower respiratory tract infection in young childhood, and cohorts of children admitted to hospital with bronchiolitis. [114] [115] [116] [117] Overall, admission to hospital with bronchiolitis at a young age is associated with an increased risk of recurrent wheezing. Studies report that 17-60% of children with bronchiolitis might develop recurrent wheezing in the years following their initial admission to hospital. A large study from Taiwan that followed up 1981 children admitted with bronchiolitis before age 3 years found that by age 10 years, 351 (17\\u00b77%) of 1981 children with bronchiolitis had a diagnosis of asthma compared with 2159 (11\\u00b77%) of 18 527 controls (hazard ratio 1\\u00b758; 95% CI 1\\u00b741-1\\u00b771). 118 One small cohort study of 138 patients has suggested that 18 (39%) of 46 children admitted with bronchiolitis before 12 months have asthma by 18 years compared with eight (9%) of 92 controls. 115 However, another study that followed a birth cohort of 1246 children found that although lower respiratory tract infection in childhood was associated with an increased risk of recurrent wheezing, this association decreased with age and was not signifi cant by age 13 years. 116 Most children in this cohort had mild illness not requiring hospital admission, and severity of illness might be associated with the increased risk of asthma. 119 The question remains whether respiratory infection at a young age itself predisposes children to asthma through damage or alteration of lung function, or whether children with severe bronchiolitis might have individual risk factors (such as altered immune response or airway function) that predisposes them to both severe bronchiolitis and recurrent wheezing. 8\\nSubstantial knowledge gaps and controversies exist in the management of acute bronchiolitis. The role of nebulised hypertonic saline in acute management is not clear, resulting in confl icting recommendations across clinical guidelines (table) . Although meta-analyses suggest a small reduction of length of stay, these analyses are limited by heterogeneity, not accounting for duration of illness, and not considering the role of outlying study populations. 91 Two large multicentre trials do not support a clinically signifi cant diff erence in length of stay for inpatients and there is no clear evidence of cost benefi t. 4, [89] [90] [91] 120 While the same meta-analysis suggests a possible reduction in admissions for outpatients, the confi dence intervals were wide and the studies were also heterogeneous. 91 No multicentre studies of nebulised hypertonic saline have been completed in outpatients. Although the evidence is increasing that hypertonic saline has little role in meaningfully reducing the length of stay, its role in outpatients is less clear. A large outpatient multicentre study could clarify whether there is any benefi t.\\nEvidence also suggests that combined therapy with nebulised epinephrine and corticosteroids might reduce admissions to hospital. 95 Synergy between corticosteroids and \\u03b22 agonists is well documented in clinical trials of asthma management. [121] [122] [123] [124] Basic science literature also shows that \\u03b22 agonists and corticosteroids enhance each other's eff ectiveness, particularly with regard to antiinfl ammatory gene expression. 125, 126 Because of the economic burden of bronchiolitis and the plausible basic and clinical evidence for synergy, a large and multicentre trial is needed to ascertain whether combined therapy with epinephrine and corticosteroids is benefi cial.\\nOxygen saturation and the use of pulse oximetry play an important role in the decision to admit infants with bronchiolitis to hospital and in the length of their hospital stay. [51] [52] [53] [54] [55] [56] [57] [58] [59] Clinical practice guidelines also give confl icting guidance on the level of oxygen saturation at which admission should be considered. Furthermore, a substantial proportion of discharged infants have episodes of transient desaturation. 54 Again, in view of the large health-care costs associated with hospital admission in bronchiolitis, further research is needed to clarify the level of oxygen saturation requiring admission, the role of continuous and spot measurements of oxygen saturation, and the clinical importance of transient desaturations in otherwise stable young infants.\\nTAF contributed to the design and coordinated the writing of this manuscript. All authors contributed to the literature search and writing of this manuscript.\\nWe declare no competing interests. TAF, ACP, and JJZ have participated in trials of bronchiolitis funded by unrestricted public or academic institutional funding sources that had no infl uence on the conception and writing of this manuscript.\",\n          \"In March and April 2009, health professionals warned the Mexican Ministry of Health about the increase in incidence of acute respiratory illness in young adults [1] . In April, a new viral subtype of influenza A(H1N1)pdm09, a combination of swine, avian and human virus, was identified in the United States and in samples originated from Mexico [2, 3] . On April 25 th , the The patients were identified in the SINAN database, using the following variables for selecting the cases and controls: RT PCR (positive for influenza A(H1N1)pdm2009), final classification (confirmed), criteria (laboratory), evolution (recovered or death), hospitalization (yes), date of hospitalization (28/06/2009 to 29/08/2009), municipalities of residence and hospitals (metropolitan areas of Sao Paulo and Campinas).\\nAt the time the patients were selected for the study, the SINAN included 193 reported and confirmed deaths of patients with influenza A(H1N1)pdm2009 and SARI, all of which were assessed in this study (cases) . For each case of death, two control patients were selected among the 1988 patients hospitalized during the study period with reported and confirmed influenza A(H1N1)pdm2009 and SARI who recovered. The controls were time-paired randomly selected from those hospitalized in the same epidemiological week (EW) or, if necessary, one week later than the date of hospitalization of cases to prevent possible differences in opportunity of treatment and clinical management protocol.\\nData of the study were collected by trained health professionals using two standardized questionnaires. The hospital questionnaire collected information from medical charts, evaluated in 177 hospitals were patients were hospitalized. Of these, 56.5% belonged to the Unified Health System-SUS and 43.5% were private hospitals. The following data were collected: previous pathological history, occurrence of pregnancy, health care, symptomatology during admission, laboratory and radiological tests, need for ICU admission, treatment, complications and disease progression. The International Classification of Diseases, 10 th revision-ICD-10, was used for coding comorbidities indicated in the medical charts. The Ministry of Health provided the antiviral drug known as Oseltamivir for treatment during the pandemic. The protocol for clinical management established that the treatment was for cases hospitalized with SARI and ambulatory patients with risk conditions for severity. Treatment should be started within 48hours after symptom onset. All patients studied had laboratory confirmations of influenza A(H1N1)pdm09 from respiratory secretion samples analyzed using the RT-PCR method, performed by the Adolfo Lutz Institute [16] . The definition of SARI included: fever associated with cough and dyspnea; pneumonia; respiratory insufficiency; tachypnea; radiological changes compatible with pneumonia; oxygen therapy or mechanical ventilation. SARI was confirmed in all cases and controls selected for the study by data their on medical charts.\\nHousehold interviews were conducted in 161 cases (83.4%), with 14 refusals (7.3%) and 18 (9.3%) addresses not found. A total of 337 (85.1%) controls were interviewed, with 6 (1.5%) refusals and 43 (11.1%) addresses not found. The household questionnaire included socio-demographic data, history of previous care related to the episode that led to hospitalization and presence of influenza vaccination. Occupations were grouped according to the occupational pyramid of influenza risk [17] . For cases, interviews were conducted with the closest family members; for controls, with the patients themselves, excluding children, whose parents/legal guardians responded for them.\\nAfter data collection, questionnaires were assessed by supervisors to verify the consistency of information and the fulfillment of criteria to include patients in this study. Hospital and household questionnaires were input to the Sphynx database, version 5.\\nThe present study began in order to assist with decisions on surveillance and control actions. The use of data was approved by the Research Ethics Committee, School of Public Health, University of S\\u00e3o Paulo-FSP-USP (protocol 2283, OF.COEP/312/11). Data collection was performed in accordance with the recommendations of the National Health Council for Human Research, including the signing of an informed consent form by patients, family members or legal guardians.\\nClinical and demographic characteristics are shown as median and inter-quartile intervals or percentages, and comparisons were made using the Mann-Whitney U or chi-square tests, as required. Odds ratio-OR and 95% confidence intervals-95%CI were calculated to assess the factors associated with death. Variables with p-value < 0.20 in the univariate analysis and adjustment variables were selected for the multiple logistic regression, which used forward stepwise strategy. The model including all patients assessed the following dichotomous categorical variables: health insurance, previous care, vaccination against influenza in 2009, sex (reference category = male); and the following risk conditions: diabetes mellitus, obesity, chronic kidney disease, chronic liver disease, immunosuppression, chronic cardiovascular disease, chronic neurological disease and delay in development, using the absence of a condition as a reference category. The model also assessed: age group [<18 years (reference), 18 to 59 years, 60 years], use of antiviral therapy [no use (reference category), 48 hours after the onset of symptoms, > 48 and 72 hours, > 72 hours], high-risk individuals or those with any of the risk conditions listed above adapted from the Centers for Disease Control and Prevention-CDC [18] [no condition (reference category), one condition, and two or more conditions], and time between onset of symptoms and admission in days as the continuous variable. The model was controlled for epidemiologic week of admission at hospital (categorized as EW26-28 (reference), EW29-31 and EW31-34), and income. A second model was developed with patients aged 18 years and over, which also assessed level of education [without formal education or incomplete primary education, complete primary education, complete secondary education, complete higher education (reference category)] and occupational risk [very high and high, average and low (reference category)]. Patients aged 18 to 22 years with incomplete higher education were included in reference category of education. For the modelling, patients with no occupation were included in the reference category of occupational risk. Significance of variables was assessed by Wald test and adjustment of models by Hosmer-Lemeshow test.\\nThe analyses were performed using SPSS software, version 17.0. P-values < 0.05 were considered to be significant. Table 1 presents the socioeconomic characteristics of patients and information associated with the disease obtained from the household interviews. Among patients aged 18 years and over, there was a higher risk of death for those without formal education or with incomplete primary education, OR 2.33(95% CI 1. 13-4.89 ) and complete primary education, OR 3.06 (95%CI 1.41-6.82), when compared to those with higher education. Distribution of ethnic groups, family income and history of smoking were not associated with death. The history of previous vaccination against seasonal influenza in 2009 showed protection against death. Health professionals, considered to be at a high risk of developing influenza, showed protection against death, OR 0.15(95%CI 0.02-0.61). Patients who had sought care prior to hospitalization had a risk of death 3.70 times (95%CI 2.25-6.30) that of patients without previous care, with a predominance of emergency care (45.34%). Table 2 presents the distribution of cases and controls according to variables assessed in the hospital investigation. Median age of cases was higher than that of controls (p<0.001) and the 18-to-59-year age group had an OR for death of 4.03(95%CI 2.62-6.31), compared to those aged less than 18 years. Elderly individuals were also at a significant risk of death.\\nThere were no differences in sex distribution in either group. With regard to risk conditions for severity, 73.6% of cases and 38.1% of controls reported having at least one risk factor, with an OR of 4.49 (95%CI 3.09-6.60). When individuals with more than one of these conditions were assessed, the risk of death increased, OR 10.29 (95%CI 6.04-17.85). The following medical conditions were risk factors for death: obesity, diabetes mellitus, chronic cardiovascular diseases, chronic kidney diseases, neurological diseases and delay in development, chronic liver diseases and immunosuppressive diseases. The immunosuppression-related diseases were: neoplasms (n = 18), HIV/AIDS (n = 8), organ transplants (n = 6), autoimmune diseases (n = 5) and use of immunosuppressants (n = 3).\\nOf the 212 women of reproductive age (10 to 49 years) who were studied, 33.5% were pregnant and there were no differences in the frequency of pregnancy in either group.\\nThe use of antiviral treatment during hospitalization was a protective factor against death, with an OR of 0.67 (95%CI 0.45-0.98). When time between onset of symptoms and beginning of antiviral treatment was assessed, there was an 81% protection among individuals who received treatment within the first 48 hours, with an OR of 0.19 (95%CI 0.11-0.35), and 65% between 48 and 72 hours, with an OR of 0.35 (95%CI 0.16-0.74). After 72 hours, there was no significant protection. Median of days between date of onset of symptoms and date of hospitalization was 5.0 among cases and 2.0 among controls (p<0.001). Among patients with chronic pulmonary disease, median was 4.0 days among cases and 2.0 days among controls; among those with asthma, 4.5 days and 2.0 days; and among pregnant women, 4.0 days and 2.0 days, respectively. With regard to health professionals, the median between the date of onset of symptoms and that of hospitalization was three days for both cases and controls. Median between date of hospitalization and of death or hospital discharge was seven days among cases and four days among controls (Table 2) .\\nAccording to medical charts, fever and cough occurred in 87.8% of cases and 79.3% of controls; and the proportions of dyspnea were 87% and 62.7%, respectively. The definition of cases of Severe Acute Respiratory Syndrome -SARS (fever+cough+dyspnea) based on information obtained from medical charts occurred in 68.9% of cases and 56.5% of controls. Vomit and diarrhea were present in 8% of cases and 18% of controls. Treatment with antibiotics was used in 95.8% of cases and 73.1% of controls. The mean number of antibiotics used was 4.0 and 1.8 respectively. Among cases, there was a higher proportion of hospitalizations in intensive care units (83.4%x16.8%), use of mechanical ventilation (94.8%x10.8%) and dialysis (25.0%x2.3%). The proportion of complications was higher among cases than controls, with an emphasis on: respiratory distress syndrome (64.7%vs4.9%), shock (69.4%vs1.0%), sepsis (63.2%vs2.3%) and kidney complications (44.6%vs1.3%), respectively. Table 3 shows the laboratory findings of entry tests of patients in the hospital. Cases had lower numbers of leukocytes and platelets and higher values of creatine phosphokinase-CPK, lactic dehydrogenase-LDH, glutamic-oxaloacetic transaminase, glutamic-pyruvic transaminase, urea and creatinine than controls, with statistical significance.\\nIn sub-group analysis, previous conditions of risk for severity and obesity were risk factors for death in univariate analysis for children younger than 18 years and women of reproductive age (10 to 49 years). Among children, presence of immunosuppression, chronic cardiovascular disease and neurological disease/delay in development were also risk factors, as well as diabetes mellitus among women. Treatment with antiviral administered within 72 hours after the onset of symptoms was a protective factor for women of reproductive age. Tables 4 and 5 show variables associated with death in the final models of multiple logistic regression, in all patients studied and in sub-group analysis of adults (aged 18 years and over). In patients in general, risk factors for death were: being aged between 18 and 59 years (reference: until 18 years), having one or more risk conditions for the severity of the disease, obesity, immunosuppression and having sought care prior to hospitalization. Although chronic neurological diseases/delay in development and chronic liver disease were highly associated with death, these variables were excluded from the final model because they had estimates with large confidence intervals. Private health insurance and antiviral treatment administered in the first 72 hours after the onset of symptoms were protective factors against death. Among adults aged 18 years and over, the risk factors for death were having more than one risk factor for severity of the disease, obesity and care prior to hospitalization. The protective factors were: antiviral drug administered in the first 48 hours after the onset of symptoms and being a health professional or health unit worker.\\nSensitivity analysis was performed to assess the severity of patients admitted during the study period. The proportion of ICU admissions and use of mechanical ventilation was estimated for epidemiological week of hospitalization, categorized as EW 26 to 28, EW 29 to 31 and EW 32 to 34. The results showed higher proportions of ICU admissions (50.0% x 16.8% x 7.7%) and mechanical ventilation (45.4% x 10.8% x 2.6%) in the three first weeks, among the controls. The cases showed high proportions in all periods analyzed: 89.7% x 84.0% x 76.9% for need of ICU and 89.7% x 97.6% x 89.7% for need of mechanical ventilation, in the 3 periods, respectively. The opportunity for antiviral treatment showed lower proportion of treatment in the first period of the study, in both cases (37.9% x 74.4% x 74.3%) and controls (59.1% x 74.1% x 89.7%). Sensitivity analysis to assess the opportunity for treatment of children and adults showed similar proportions for the groups: 73.9% and 74%, respectively.\\nIn the present study, risk factors for death were analyzed in patients infected with influenza A (H1N1)pdm09 and hospitalized with SARI. There was an increased risk of death in patients aged 18 to 59 years; those with obesity, immunosuppression, neurological and developmental diseases; or those who had received care prior to hospitalization. Additionally, presence of at least one risk factor for aggravation of the disease was also associated with occurrence of death. Use of antiviral drugs, especially when administered in the first 72 hours after onset of symptoms was protective factor against death.\\nThe design of this case-control study, including all deaths reported and confirmed in the region during the study period and the time-paired random selection of controls among patients who survived after hospitalization with SARI, with hospital and household data collection, enabled the analysis of the factors associated with death to be extended, including socioeconomic variables, access to health services, and data on hospitalizations.\\nFew studies have assessed risk factors for death in patients with SARI [15, 19] . Some studies have analyzed risk factors for severity of the disease and deaths, comparing patients hospitalized in ICUs and in wards [13, 14, [20] [21] [22] [23] , whereas one study used the rate of mortality from influenza A(H1N1)pdm09 in two epidemic waves [24] . A study conducted in Australia analyzed risk factors for hospitalization due to influenza, comparing cases with controls without infection in the community [25] . Other studies have analyzed epidemiological surveillance data, including mortality rates and comparisons among deaths, hospitalizations, treatment and other factors [9, 10, 12, [26] [27] [28] [29] [30] [31] [32] [33] [34] [35] . Some of the factors associated with death identified in this study have been described by other authors, such as the presence of comorbidities [31, 35] , age18years [13] , obesity [15, 19, 20] , neurological and developmental diseases [23, 24, 31] and immunosuppression [15, 21, 24, 34] . Children showed lower risk of death, despite a similar proportion of treatment than adults. A similar result was observed in a study of systematic review and metaanalysis, where the risk of death was 0.28 (95% CI 0.19-0.141), when compared to young adults [36] . In contrast, other factors associated with severity of infection that have been reported by different researchers, such as diabetes [19, 34] ; other chronic pulmonary diseases, excluding asthma and chronic obstructive pulmonary disease [20] ; chronic pulmonary disease, including asthma [31] ; and chronic cardiovascular disease [21, 24, 34] , were not associated with mortality in the present study. The population assessed, which was restricted to hospitalized patients with severe influenza could at least partly explain these differences. Diabetes, chronic cardiovascular and kidney disease, associated with death in univariate analysis, did not remain risk factors after multiple analysis. The design of this study, with data collected on medical charts and household interviews, allowed for adjustments in the analysis that can also explain part of these differences.\\nDeaths occurred mainly in adults aged 18 to 59 years, with a small proportion of cases and controls among elderly individuals. A possible preexisting immunity against the new influenza A(H1N1)pdm09 virus in this age group has been suggested, probably due to previous exposure, through other infections or vaccinations, to an influenza A(H1N1) virus which is genetically and antigenically more associated with the new virus than contemporary strains of seasonal H1N1 viruses [37] .\\nAll patients in the present study, whether cases or controls, had a laboratory-confirmed infection of influenza A/H1N1pdm09, which was [38] .\\nIt is possible that the probability of being hospitalized varied throughout the study. Treatment was available for hospitalized patients and for ambulatory patients with risk conditions for severity. This could have influenced hospitalization, but not much in our study population, consisting of patients with SARI. During the pandemic, criteria for hospitalization changed as well as the availability of treatment, especially in the first 3 weeks of the epidemic, as shown in sensitivity analysis, confirming the need for controlling period of hospitalization in the study.\\nThe socioeconomic profile was similar in both groups assessed, with no difference in income or having private health insurance. There was greater protection against death among patients with an occupation with higher risk of exposure to infection, such as health professionals. A greater opportunity of access to treatment among these workers could explain such findings, as the period between the date of onset of symptoms and the date of admission was shorter in this group. The low proportion of deaths among patients categorized as having a higher risk of exposure has also been observed in other studies [30, 33] .\\nPregnancy was not a risk factor for death in this study. In contrast, frequency of pregnant women among those of reproductive age hospitalized with severe manifestation of influenza A (H1N1)pdm09 was high, suggesting that pregnancy could be associated with development of SARI. An association between pregnancy and use of mechanical ventilation during hospitalization due to influenza A(H1N1)pdm2009 [25] and a higher admission rate were reported when compared to the general population [32] . In 2009, in the state of S\\u00e3o Paulo, 27.4% of all confirmed cases of influenza A(H1N1)pdm09 in women of reproductive age (10 to 49 years), hospitalized and recorded in the SINAN database, were found in pregnant women. In Brazil, only 4.27% of women of reproductive age had live newborns in the reference year according to the 2010 Census [39] .\\nPatients who died presented important laboratory changes when compared to controls, with lower levels of leukocytes and platelets and an increase in CPK, LDH, urea and creatinine. Studies revealed a similar trend for the laboratory results of leukocytes [23] , CPK [26, 27, 30] , LDH [26] , platelets [22] and creatinine [27, 30, 34] .\\nAlthough most patients who died received antiviral treatment, only 9.3% were treated in the first 48 hours. Early treatment was a protective factor against death, especially when provided in the first 72 hours after onset of symptoms. Other studies have also shown a more favorable prognostic with antiviral treatment [13, 21, 23, 27, 28, 29, 34] .\\nHistory of care prior to hospitalization was a risk factor for death from influenza A(H1N1) pdm09, which could reflect the difficulty in accessing health services. Emergency services were primarily sought for care prior to hospitalization, which could have contributed to greater delay in diagnosis and beginning of treatment. A study conducted in Mexico showed a similar result with regard to delay in hospitalization [29] .\\nIn one study, which assessed the overall standardized mortality rate from influenza A (H1N1)pdm09, the mortality among patients with preexisting chronic respiratory diseases was lower than those with other chronic diseases 24 . There might have been problems with recording information about previous diseases in the medical charts in the presence of acute pulmonary disease, especially among those with chronic pulmonary disease.\\nThe present study had some limitations. Underreporting of SARI by health professionals is likely to have occurred, as reporting this disease was not compulsory in Brazil before the pandemic. However, during the pandemic, professionals were more sensitive to reporting it. The system for notification, which was mandatory during the epidemic, was facilitated to allow for rapid notifications. The availability of samples from autopsies of cases (deaths) could have resulted in greater access to the laboratory diagnosis of this infection. Household data collection could also have caused information bias, as data on cases were collected through interviews with family members, while data on controls were obtained from the patients themselves, when they were adults. The proportion of refusals of household interviews was higher among cases (7.3%) than controls (1.6%), p = 0.001. Another possible limitation was memory bias, as a result of the time between the hospitalization and interview. However, the information analyzed from household interviews was about socio-demographic characteristics, which is less vulnerable to information and memory bias. Data on preexisting diseases were obtained from medical charts in hospitals, where they are completed at the moment of hospitalization, in the presence of the patients. The quality of completion of medical charts can differ between hospitals. Nonetheless, information about the presence of chronic diseases, clinical management of patients, laboratory results, radiological tests and treatment was available in these charts. The difficulties encountered during data collection were minimized by the use of a standardized questionnaire.\\nIn conclusion, the risk conditions for the severity of the disease, particularly obesity, immunosuppression and neurological/developmental diseases, were also important risk factors for death, emphasizing the need for good vaccination coverage during influenza vaccination campaigns in all patients with chronic diseases belonging to more vulnerable groups. Another relevant result was the antiviral treatment within the first 72 hours of onset of symptoms as a protective factor, also emphasizing the need to warn the population about the importance of seeking early medical care when an influenza-like illness appears, especially among high-risk groups and those with signs of disease aggravation. Training of health professionals, especially physicians, is required for adequate clinical management of patients and early antiviral treatment.\"\n        ],\n        \"semantic_type\": \"\",\n        \"description\": \"\"\n      }\n    },\n    {\n      \"column\": \"authors\",\n      \"properties\": {\n        \"dtype\": \"string\",\n        \"num_unique_values\": 1422,\n        \"samples\": [\n          \" Chen, Junbo.  Zhu, Huachen.  Horby, Peter W.. <br>Wang, Qianli.  Zhou, Jiaxin.  Jiang, Hui.  Liu, Liwei. <br>Zhang, Tianchen.  Zhang, Yongli.  Chen, Xinhua.  Deng,<br>Xiaowei.  Nikolay, Birgit.  Wang, Wei.  Cauchemez, Simon. <br>Guan, Yi.  Uyeki, Timothy M..  Yu, Hongjie\",\n          \" Chen, Yu.  Dong, Yuanji.  Cai, Shaozhe.  Ye, Cong. <br>Dong, Lingli\",\n          \" Arens, Amanda.  Scott, Cheryl.  Osburn, Bennie\"\n        ],\n        \"semantic_type\": \"\",\n        \"description\": \"\"\n      }\n    },\n    {\n      \"column\": \"title\",\n      \"properties\": {\n        \"dtype\": \"string\",\n        \"num_unique_values\": 1454,\n        \"samples\": [\n          \" Peptide Vaccine: Progress and Challenges\",\n          \" The Impact of Ebola Virus Disease on Government<br>Expenditure in Sierra Leone\",\n          \" The clinical implication of dynamic<br>neutrophil to lymphocyte ratio and D-dimer in COVID-19: A<br>retrospective study in Suzhou China\"\n        ],\n        \"semantic_type\": \"\",\n        \"description\": \"\"\n      }\n    },\n    {\n      \"column\": \"journal\",\n      \"properties\": {\n        \"dtype\": \"string\",\n        \"num_unique_values\": 785,\n        \"samples\": [\n          \"AMB Express\",\n          \"Dail and Hammar&#x02019;s Pulmonary Pathology\",\n          \"Benumof's Airway Management\"\n        ],\n        \"semantic_type\": \"\",\n        \"description\": \"\"\n      }\n    },\n    {\n      \"column\": \"abstract_summary\",\n      \"properties\": {\n        \"dtype\": \"string\",\n        \"num_unique_values\": 1044,\n        \"samples\": [\n          \" InnoMetrics is the system that aims at<br>collecting software development process metrics in a<br>non-invasive way to access and optimize the development<br>process and its efficiency. This paper demonstrates<br>the development and analysis of energy<br>consumption of MacOS systems based on the software process<br>measurement data. It represents the experience of the<br>development of MacOS system collector and Transfer, in<br>addition to the user interface and early analysis of<br>energy consumption metrics calculations.\",\n          \" During the global outbreak of severe acute<br>respiratory syndrome (SARS) in 2003, treatment was<br>empiric. We report the case history of the index patient<br>in a hospital outbreak of SARS in Hong Kong. The<br>patient recovered after conventional antimicrobial<br>therapy. Further studies are needed to address<br>treatment of SARS, which has high attack and death rates.\",\n          \" Intestinal transplantation (IT) is the least<br>common form of organ transplantation; however, it has<br>shown exceptional growth and improvement in graft<br>survival rates over the past two decades mainly due to<br>better outcomes achieved during the first year of<br>transplantation (76 % at 1 year), due to improvement in surgical<br>techniques and the development of better<br>immunosupressive therapies as we understand more about the<br>relationship between the recipient and host immune system.<br>There are still ongoing issues with chronic<br>rejection and long-term survival. Intestinal<br>transplantation is still an acceptable therapy for patients<br>with intestinal failure (IF), but it is generally...\"\n        ],\n        \"semantic_type\": \"\",\n        \"description\": \"\"\n      }\n    },\n    {\n      \"column\": \"cleaned_text\",\n      \"properties\": {\n        \"dtype\": \"object\",\n        \"semantic_type\": \"\",\n        \"description\": \"\"\n      }\n    },\n    {\n      \"column\": \"cleaned_text_str\",\n      \"properties\": {\n        \"dtype\": \"string\",\n        \"num_unique_values\": 1453,\n        \"samples\": [\n          \"respiratory tract infection common infants young children major public health problem age group 1 mainly caused infection seasonal viruses respiratory syncytial virus rsv rhinovirus 2 3 4 infections manifest upper respiratory infection 20 40 develop lower respiratory infection small number 2 -3 infected infants require hospitalization 5 6 particular concern infants lower respiratory viral infection increased risk developing recurrent wheezing childhood asthma 7 8 9 10 11 12 13 risk factors identified development severe lower respiratory viral infections premature birth young age born relation rsv season underlying chronic lung disease congenital heart disease high parity young maternal age poor socioeconomic factors 14 15 16 recently genetic polymorphisms identified associated severe viral infection 17 18 identified genetic variants associated severe lower respiratory tract infections associated pathways common asthma including innate immune genes involved cytokine chemokine signaling epithelial cell function 13 17 19 20 21 22 23 \\n clinical hallmark asthma wheezing therapy includes beta agonists number clinical studies past 2 decades examined relationship genetic variants \\u03b2 2 -adrenergic receptor adrb2 asthma 24 25 26 27 28 29 30 polymorphism resulted gly16 allele associated decreased bronchodilator responses early clinical studies larger clinical studies provided equivocal results 31 32 33 genetic variants promoter region adrb2 associated asthma severity 28 34 despite significant interest adrb2 polymorphisms related asthma relative contribution adrb2 genetic variants severity respiratory viral infections established wheezing clinical hallmark lower respiratory viral infections variable response bronchodilators demonstrated infants hospitalized rsv lower respiratory viral infections possible adrb2 genetic variants contribute severity acute respiratory viral infections 35 36 \\n aim present study explore associations genetic variants adrb2 associated response asthma therapy acute asthma exacerbation severity infant acute respiratory viral infections examined relationship 5 common polymorphisms promoter region coding block adrb2 gene infant severity respiratory infections prospective cohort caucasian african american infants \\n order examine association genetic variants regulatory regions adrb2 gene severity viral respiratory infections infancy conducted analysis infants enrolled tennessee children respiratory initiative tcri prospective cohort mother-infant dyads enrolled infant acute viral respiratory illness respiratory viral seasons 2004 2008 term \\u226537 weeks non-low birth weight \\u22652275 g infants healthy significant co-morbidities cardio-pulmonary disease eligible oversampling hospitalization infants eighty percent enrolled subjects respiratory virus identified polymerase chain reaction rsv rhinovirus influenza parainfluenza human metapneumovirus coronavirus 56 labconfirmed rsv infection rationale methods detailed inclusion exclusion criteria reported previously 37 630 infants enrolled tcri study limited 261 caucasian 113 african american infants dna available protocol approved institutional review board vanderbilt university mothers provided informed consent infants parents provided written informed consent child study participation \\n infants included study upper respiratory tract infections lower respiratory tract infections severity acute respiratory infection determined experienced research nurses medical record chart review ordinal bronchiolitis severity score bss score aggregate assigned values ranging 0 3 categories respiratory rate room air oxygen saturation presence extent wheezing presence extent flaring retractions 37 38 39 bss ranges 0 12 higher scores indicating severe illness difference 0.5 clinically significant difference 40 measured bss end outpatient visit discharge inpatient admission severe recorded value analysis \\n dna extracted vanderbilt dna core laboratory blood saliva samples oragene dna genotek standardized protocols oragene samples processed immediately collection blood samples processed immediately frozen processing genotyping pre-specified loci -2387 -2274 -1343 46 79 adrb2 gene taqman sm \\u00a9 genotyping assays applied biosystems foster city usa predetermined primers single-nucleotide polymorphisms (snp)s rs1432623 rs11168068 rs2400707 promoter region rs1042713 snp rs1042714 snp coding region adrb2 gene probes applied biosystems assays labeled 5\\u2032end vic \\u00ae 6-carboxy-fluorescine fam reporter dyes reaction components include 2.5 \\u03bcl taqman universal pcr master mix applied biosystems 0.125 \\u03bcl 0.250 \\u03bcl assay mix applied biosystems approximately 5 ng genomic dna total volume 10 \\u03bcl single tube reaction pcr amplification performed thermal cycler techne tc-412 initial step 95 \\u00b0 c 10 min followed 50 cycles 92 \\u00b0 c 15 60 \\u00b0 c 1 min amplification fluorescence sample read abi 7900ht dna resources core vanderbilt university analyzed sequence detection software applied biosystems percent samples run blind duplicates 100 concordance primers probes snps listed table 1 \\n demographics characteristics 374 infants included upper respiratory infection lower respiratory tract infection presented frequencies proportions specific categorical variables medians interquartile ranges calculated continuous variables analyses stratified race caucasian african american surrogate measure european african ancestry reduce known impact population substructure produce spurious associations 41 pairwise linkage disequilibrium ld estimated snps adrb2 standardized summary statistics r calculated haploview program whitehead institute biomedical research cambridge usa snps tested deviations hardy weinberg proportions haploview 42 promoter coding combined promoter coding snps assigned blocks confidence interval gabriel al. 43 individual haplotypes estimated program phase v2.1.1 44 45 haplotype frequencies block estimated expectation maximization em algorithm haploview program non-parametric kruskal-wallis test applied compare difference bss infants snp genotype haplotype genotype haplotype genotypes frequency 2 excluded haplotype analysis proportional odds model ordinal bss test additive effect haplotypes covariates included regression model included infant age enrollment gender daycare exposure secondhand smoke exposure prior history breastfeeding siblings home enrollment season analyses conducted separately according race analyses performed r-software version 2.11.1 www.r-project.org two-sided 5 significance level statistical inferences \\n total 374 infants 261 caucasians 113 african americans enrolled tcri cohort available dna adrb2 polymorphism tested table 3 lists allele frequencies 5 snps separately caucasian african americans caucasians african americans infants slightly g alleles promoter snp rs1432623 figs. 1 2 non statistical difference consistent caucasians african americans african americans decreasing trend severity acute respiratory infection increasing number copies c c allele promoter snps rs1432623 rs11168068 rs2400707 respectively fig. 1 analyses additive models indicated statistical significance p 0.05 \\n 2 5 32 possible haplotype combinations haplotypes 2 frequency identified table 4 fig. 3 promoter haplotypes coding block haplotypes statistically significant differences promoter coding block haplotype frequencies caucasian african americans 5 snps combined distribution haplotype frequency significantly different caucasians african americans genotype ccaac common african americans rare caucasians p 0.001 \\n promoter haplotype cca associated decreased bss african americans dose dependent manner p 0.037 univariate analysis fig. 4 infants cca haplotype highest bss median 5.5 interquartile range iqr 2.0 8.0 infants copy cca haplotype lower bronchiolitis severity score median 4.0 iqr 1.0 7.5 infants 2 copies cca haplotype comparatively lowest bss median 3.0 iqr 1.0 4.0 dose dependent relationship persisted adjusting infant age gender daycare exposure secondhand smoke exposure prior history breastfeeding siblings home enrollment season adjusted odds ratio 0.59 95 confidence interval ci 0.36 0.98 p 0.042 protective effect cca haplotype consistent categorized acute respiratory infection upper respiratory infection lower respiratory tract infection similar dose dependent manner infants copies cca haplotype likely lower respiratory tract infection compared infants copies cca haplotype p 0.05 hand promoter haplotype ttg associated higher bss african americans additional file 1 \\n protective effect cca haplotype african americans identified combined haplotypes cca combination figs. 5 6 sample size effect statistically significant addition african american infants coding block haplotype gg cca combination promoter region similar additive protective effect gg african americans additional file 2 relationship bss coding block haplotypes african americans additional files 3 4 true combined haplotypes additional files 5 6 \\n caucasian african american infants significant relationships bss coding block haplotypes gg haplotype african americans additional file 2 true combined haplotypes general significant relationship bss haplotypes caucasian infants similar promoter haplotype cca relationship bss caucasians observed african americans fig. 4 \\n study identified haplotype promoter region adrb2 gene associated decreased bss african american infants promoter haplotype adrb2 gene associated severity respiratory infections caucasian infants common coding block polymorphisms corresponding amino acid changes codons 16 27 associated protective effect severity respiratory infections caucasian african american infants results agree studied conducted janssen colleagues found relationship snp rs1042713 susceptibility rsv bronchiolitis european population 17 observation cca promoter region haplotype associated protective effect respiratory viral infection severity similar observation cohort adults hospitalized asthma exacerbation haplotype associated spirometric improvement response asthma-specific therapy included beta-agonists corticosteroids 34 study noted spirometric improvement caucasian patients small number african american non-responders cohort limited ability detect similar effect african americans 34 \\n mechanism haplotype promoter region adrb2 gene associated decreased bss known speculate 46 moore al. demonstrated adrb2 genetic variants associated differences adrb2 expression cultured human airway smooth muscle hasm cells vitro system model changes human airway 47 studies performed prior identification complex promoter haplotypes aware haplotypes studied hasm small differences airway tone important setting acute viral respiratory infections \\n scope current observational study speculate adrb2 genotype influence response bronchiolitis-specific therapy primary goal tcri establish longitudinal prospective investigation infants biological mothers acute long-term health consequences varying viral respiratory tract infections early childhood outcomes 37 bss assigned infant acute care visit hospital admission study quantify effects different treatments including beta-agonist therapy number clinical studies suggested infants bronchiolitis appear improve bronchodilator therapy average efficacious 48 49 50 adrb2 primary target bronchodilator therapy variation response beta-agonists reflect genetic variants adrb2 \\n diversity snps adrb2 gene explored drysdale al. immortalized lymphocytes 23 caucasians 19 african-americans 20 asians 15 hispanic-latinos 27 report -2387 -2274 -1343 coding block snps discussed manuscript 28 frequency distributions observed study consistent allele frequencies reported reference population international hapmap project 51 hawkins al. 28 stratified analyses race allow detect association severity respiratory illness haplotype frequency racial group minimize spurious associations population substructure recognize significance findings limited small sample size classifying infants study severity score measuring direct response bronchodilator therapy bss correlated severity disease classified upper respiratory lower respiratory illness sensitivity analyses severity disease upper versus lower respiratory illness showed consistent results indicated bss valid surrogate disease severity lastly p value 0.042 threshold statistical significance p value reflects sample size 113 african americans study association bss cca haplotype african americans powerful depends p value effect size odds ratio clinical significance findings difference 0.5 bss considered clinically significant 65 reduced odds developing severe disease higher bss comparing 2 copies cca 0 copy cca clinically significant meaningful \\n hope hypothesis-generating study encourage incorporation pharmacokinetic data future studies bronchiolitis validation studies populations needed study underscores development personalized approaches treatment bronchiolitis benefit analysis specific genetic variants \\n summary observed promoter region haplotype adrb2 cca associated protective effect respiratory illness severity african american infants era personalized medicine adrb2 genotype predictive severity respiratory viral infection potential response therapy common acute infant illness currently therapy steps validate findings evaluate available clinical trial data dna available fig. 6 bss distribution stratified copy number combined haplotype ccaac african americans infants scatter plots box-and-whisker plots bss 0 1 2 copies combined haplotype ccaac african american infants p values obtained multivariable regression model adjusted infant age enrollment gender daycare exposure secondhand smoke exposure prior history breastfeeding siblings home enrollment season\",\n          \"acute bronchiolitis viral infection lower respiratory tract substantial health burdens infants young children worldwide 1 respiratory syncytial virus prevalent viral cause bronchiolitis infants estimates suggest 34 million new cases lower respiratory infection respiratory syncytial virus occur globally children younger 5 years 3\\u00b74 million admissions hospitals 199 000 deaths year predominantly developing world 2 developed countries usa bronchiolitis common reason admission hospital fi rst 12 months life 3 accounting approximately 100 000 infant admissions annually admissions hospital declined 2000 2010 emergency department visits increased addition increased use mechanical ventilation hospital charges 4 5 clinical management remains challenging despite frequency global reach economic cost morbidity mortality associated bronchiolitis treatment strategies including bronchodilators corticosteroids showed eff ect pooled metaanalyses making supportive care hallmark current therapy seminar aim summarise current evidence epidemiology pathophysiology diagnostic approach management acute viral bronchiolitis \\n bronchiolitis seasonal infection season typically beginning late october temperate northern hemisphere peaking january february ending april 6 globally independent region respiratory syncytial virus infection peaks consistently annual biannual epidemics peak duration epidemics vary worldwide consistent year-to-year country 7 data suggest climate associated prevalence respiratory syncytial virus infection global surveillance suggesting infection peaks wet months areas high precipitation cooler months hot regions 7 indoor crowding population-dense areas rainy seasons cooler months factor facilitates viral transmission 8 additionally weather-related factors inhalation cold dry air impair ciliary function airway mucosa inhibition temperature-dependent antiviral responses infl uence disease transmission severity 9 10 altitude climate meteorological conditions wind speed dew point shown modest association bronchiolitis 11 12 furthermore air pollutants ozone traffi c pollutants associated exacerbations respiratory infections children younger 5 years 13 14 15 environmental tobacco smoke associated increased risk respiratory syncytial virus-attributable admission hospital disease severity admitted 16 17 respiratory viral infections risk severe respiratory syncytial virus bronchiolitis greater boys girls 18 19 diff erence diff erences lung airway development genetic factors 8 20 pathophysiology bronchiolitis characterised extensive infl ammation oedema airways increased mucus production necrosis airway epithelial cells 21 respiratory syncytial virus binds epithelial cells replicates resulting epithelial necrosis ciliary destruction 21 22 cell destruction triggers infl ammatory response proliferation polymorphonuclear cells lymphocytes submucosa adventitial tissues oedematous increased mucus secretion 22 plugs composed cellular debris mucus form bronchiole lumens leading bronchiolar obstruction air trapping diff erent degrees lobar collapse 21 microbiology molecular testing led improved understanding viruses associated bronchiolitis respiratory syncytial virus remains commonly identifi ed virus detected 41 83 patients 23 24 25 26 27 28 viruses associated bronchiolitis include rhinovirus metapneumovirus coronavirus human bocavirus infl uenza virus adenovirus parainfl uenza virus 24 25 27 29 30 studies investigated severity illness measured need hospital admission length hospital stay intensive care unit admission repeated emergency department visits apnoea associated specifi c viral infections co-infections evidence confl icting data studies shown infections involving single virus respiratory syncytial virus associated severe course compared viruses 31 32 30 children bronchiolitis found co-infections viruses 29 combination respiratory syncytial virus rhinovirus commonly reported 29 evidence suggests co-infection bronchiolitis particularly respiratory syncytial virus combination rhinovirus metapneumovirus associated severe disease course compared infection single virus 25 30 31 33 34 studies confi rm association 24 26 furthermore use nucleic acid amplifi cation tests greatly improved ability detect viruses present respiratory infections studies technologies found respiratory virus 30 children younger 6 years respiratory symptoms 35 36 37 viruses detected asymptomatic colonisation incubation clinical infection prolonged viral shedding post-infection confl icting evidence high prevalence respiratory viruses asymptomatic children suggest indication time management vary based presumed viral cause presence absence viral co-infections \\n diagnosis bronchiolitis clinical requires clinician recognise signs symptoms viral lower respiratory tract infection young children peak incidence occurs 3 months 6 months age 38 early clinical defi nition court noted recent practice guidelines specifi c defi nition infants 38 39 40 physiology occur toddlers older 12 months clinical trials excluded children included small subgroup patients bronchiolitis toddlers overlap conditions viral-induced wheezing asthma application evidence trials predominantly assessing infants appropriate eff orts focus defi nition assist eff orts standardise care \\n classic clinical presentation bronchiolitis starts symptoms viral upper respiratory infection nasal discharge progress lower respiratory tract days fi gure 1 timing symptom progression vary young infants present apnoea lower respiratory tract symptoms bronchiolitis include persistent cough tachypnoea increased work breathing shown intercostal supraclavicular retractions use abdominal muscles grunting nasal fl aring auscultatory fi ndings include crackles wheeze hallmark characteristic bronchiolitis minute-to-minute variation clinical fi ndings mucus debris airways cleared coughing child state changes sleep agitation variation confound assessment requires examinations period observation nasal congestion confound clinical assessment nasal suctioning help ascertain fi ndings truly lower airways fever present infants bronchiolitis usually present early illness temperature 39 \\u00b0 c 38 median duration symptoms 2 weeks 10 20 infants symptoms 3 weeks onset 38 41 clinical scores shown studies clinical protocols correlate disease severity improvement 42 43 documentation score useful objective measure individual scores highly predictive repeated combined measures severity universal assessment guide decision making diff erential diagnosis bronchiolitis includes considerations infectious non-infectious causes absence upper respiratory symptoms raise suspicion causes respiratory distress young infants including cardiac disease congenital airway abnormalities vascular ring foreign body aspiration infections resemble complicate bronchiolitis pertussis considered infants severe paroxysmal cough known exposure bacterial infections complicating viral bronchiolitis including otitis media pneumonia present new fever worsening status later course illness \\n risk factors associated progression severe bronchiolitis supported strongest evidence include presence chronic lung disease prematurity haemodynamically important congenital heart disease immunodefi ciency neuromuscular disorders considered high risk practice guidelines 38 39 young infants aged 2 3 months history premature birth especially 32 weeks gestation high risk progression present apnoea clinical fi ndings studies assessing risk apnoea hospital found limited infants 1 month full-term infants 48 weeks postconceptional age preterm infants apnoea observed admission 44 \\n bronchiolitis clinical diagnosis based history physical examination according consensus national guidelines infants typical presentation bronchiolitis routine imaging laboratory testing recommended increase costs evidence benefi table \\n appropriate use pulse oximetry monitoring initiation oxygen bronchiolitis received increasing attention studies practice guidelines findings suggest arbitrary thresholds oxygen administration drive hospital admissions prolong hospital length stay outcomes represent morbidity bronchiolitis developing evidence suggests intermittent hypoxaemia occur commonly stable infants bronchiolitis raises questions factor sole indication admission hospitals canadian randomised trial found reduced admissions hospital emergency department increase revisits pulse oximeters displayed values 3 higher actual value suggesting arbitrary pulse oximetry thresholds result unnecessary admissions 51 similar trial uk hospital setting found reduction oxygen threshold 94 90 resulted earlier discharge hospital evidence adverse outcomes 52 trial comparing intermittent versus continuous pulse oximetry non-hypoxaemic infants hospitals found similar outcomes groups 53 trial evidence supports recommendations practice guidelines clinicians use threshold 90 initiation oxygen uk guidelines recommend 92 38 39 child improves reduction intensity monitoring intermittent checks appropriate recent study blinded oximetry home showed substantial proportion infants bronchiolitis oxygen desaturations 90 particularly sleep calling question arbitrary thresholds hospital admissions initiation oxygen 54 evidence probably lead eff orts reduce continuous monitoring children indications monitoring \\n majority children bronchiolitis normal radiographs radiographic fi ndings consistent simple bronchiolitis 55 58 including peribronchial thickening hyperinfl ation atelectasis prospective study 58 routine radiographs assessment bronchiolitis emergency department reported airspace disease 17 7 246 patients despite low prevalence radiographic pneumonia studies reported increase antibiotic prescription radiographs performed non-specifi c fi ndings infl uence clinicians decisions 58 59 factors associated defi nite focal infi ltrates consistent pneumonia include hypoxia oxygen saturation 92 56 57 58 60 grunting persistently focal crackles fever especially 39 \\u00b0 c 38 chest radiographs considered patients presentation classic bronchiolitis situations include diagnosis foreign body aspiration high diff erential diagnosis child severely ill respiratory failure imminent symptoms progressing resolving according typical disease course expected bronchiolitis lung ultrasound increasingly assess cardiopulmonary conditions adults children studies investigated use lung ultrasound diagnosis bronchiolitis small studies found ultrasound fi ndings infants bronchiolitis correlate clinical fi ndings specifi c chest radiography 61,62 studies needed establish role ultrasound diagnosis assessment severity \\n development pcr detect respiratory viruses nasopharynx interest use viral testing causative diagnosis bronchiolitis increased virological testing generally assist management insuffi cient predict outcomes 30 63 national guidelines recommend routine virological testing bronchiolitis table recent studies suggest higher respiratory syncytial virus genomic load measured quantitative pcr associated increased length stay use respiratory support need intensive care addition recurrent wheezing compared lower viral loads 28 31 32 64 study warranted confi rm association clarify viral load measurement improves understanding disease pathophysiology severity guidelines recommend respiratory syncytial virus testing guide cohorting patients viruses likely cause bronchiolitis transmitted similar fashion close contact large-particle aerosols direct contact contaminated fomites 65 66 67 infection control dependent identifi cation specifi c viruses following strict precautions including hand hygiene separating infants shared hospital rooms 1 infection control procedures 65 additionally given sensitivity pcr testing results interpreted caution certain viruses rhinovirus detected viral shedding unrelated illness colonisation certain viruses respiratory syncytial virus metapneumovirus associated acute infection \\n blood urine testing routinely recommended standard practice diagnostic work-up bronchiolitis table blood gas measurement routinely obtained infants bronchiolitis signs impending respiratory failure severe distress proportions bacterial infections especially bacteraemia meningitis low infants bronchiolitis 68 abnormal white blood cell count rarely useful predicting bacterial infections children infected respiratory syncytial virus 69 guidelines universally recommend complete blood counts infants bronchiolitis blood count assessment fever infants younger 1 2 months similarly given bacteraemia exceedingly rare cited proportions 0\\u00b71 post-pneumococcal vaccine era blood cultures routinely performed septic work-up infants younger 1 2 months 70 71 severe illness signs sepsis hydration status important consideration infants bronchiolitis determined clinical examination routine measurement serum electrolytes little value majority infants \\n urinary tract infections infants bronchiolitis occur greater frequency bacteraemia meningitis proportions ranging 1 7 68 71 72 reasonable obtain urinalysis urine culture infants aged 60 days fever older febrile infants risk factors urinary tract infections 73 urine routinely obtained infants bronchiolitis \\n current recommendations management bronchiolitis focus agents treat patho physiological eff ects viral lower respiratory infection eg bronchodilators hypertonic saline specifi c antivirals ribavirin treat respiratory syncytial virus infection recommended practice guidelines typical cases bronchiolitis challenging delivery methods high cost potential health risks caregivers multiple new agents prevention treatment investigation available future including immunoglobulins small interfering rna interference fusion inhibitors small molecules 74 numerous studies assessed role bronchodilators treatment bronchiolitis systematic reviews found consistent benefi t. 2014 cochrane collaboration systematic review identifi ed 30 studies assessing bronchodilators predominantly salbutamol excluding epinephrine 21 studies looked specifi ally clinical scores found evidence benefi outcomes infants admitted hospitals 75 outpatients oxygen saturation admission hospital time resolution symptoms improve bronchodilator usage compared placebo outpatient studies assessing short-term change pooled clinical scores reviewers found small signifi nt diff erence mean score z=2\\u00b726 p=0\\u00b7024 small eff ect minimal clinical importance fi gure 2 outpatient studies heterogeneous i\\u00b2=81 p<0\\u00b700001 showing benefi scores tended include older children children recurrent wheezing \\n nebulised epinephrine assessed cochrane collaboration systematic review 76 review found benefi epinephrine compared placebo inpatients hospital length stay outcomes multicentre scandinavian study published cochrane review found inpatients receiving standing doses epinephrine longer length stay compared inpatients receiving as-needed epinephrine placebo 77 outpatients cochrane review found diff erence numbers admissions associated epinephrine treatment time emergency department visit overall course illness assessed 1 week clinical practice guidelines including usa uk canada recommend treatment bronchodilators bronchiolitis evidence table 38 \\n nebulised hypertonic saline thought reduce airway oedema decrease mucus plugging improve mucociliary clearance rehydrate airway surface liquid infants bronchiolitis 78 physiological changes extrapolated cystic fi brosis literature 79 80 pathophysiological processes acute bronchiolitis diff erent theoretical benefi ts hypertonic saline seen cystic fi brosis present infants acute viral bronchiolitis initial trials demonstrated ability hypertonic saline decrease hospital length stay transiently improve clinical severity score 81 83 recent trials demonstrated confl icting results 84 85 86 87 88 89 90 trials showed largest benefi hospitals lengths stay 72 h hypertonic saline infants countries institutions length stay approaches exceeds 72 h benefi cial reducing length stay confl icting results \\n test heterogeneity inpatient studies demonstrated low inconsistency studies i\\u00b2=36 p=0\\u00b713 summary eff ect signifi nt z=1\\u00b706 p=0\\u00b729 outpatient studies demonstrated high inconsistency 12 studies i\\u00b2=81 p<0\\u00b700001 summary eff ect signifi nt z=2\\u00b726 p=0\\u00b7024 overall heterogeneity meta-analysis demonstrated high inconsistency 21 studies i\\u00b2=73 p<0\\u00b700001 overall summary eff ect signifi nt z=2\\u00b74 p=0\\u00b7016 ipr ipratropium sal salbutamol neb nebulised reproduced gadomski scribani 75 refl ected diff erences recommendations national guidelines table countries recommending hypertonic saline recommending use inpatients recommending use moderate severe illness largest systematic review meta-analysis published 2015 examined 24 trials 3209 patients 91 infants receiving hypertonic saline signifi nt diff erence hospital length stay -0\\u00b745 days 95 ci -0\\u00b782 -0\\u00b708 p=0\\u00b701 compared receiving 0\\u00b79 saline standard care seven trials hypertonic saline reduced risk admission hospital 20 risk ratio 0\\u00b78 95 ci 0\\u00b767 0\\u00b796 compared 0\\u00b79 saline substantial adverse eff ects hypertonic saline noted systematic review results meta-analysis signifi nt attention paid outer limits confi dence intervals approach diff erence clinical relevance diff erences second 2015 meta-analysis 15 trials 1922 patients found smaller signifi nt decrease length stay -0\\u00b736 days 95 ci -0\\u00b75 -0\\u00b722 p<0\\u00b7001 received hypertonic saline fi gure 3 92 study found discrepancy overall combined eff ect studies length stay negative results largest trials allowing authors conclude individual trials pooled estimates provide strong evidence-based foundation use hypertonic saline meta-analyses showed substantial heterogeneity studies i\\u00b2=82\\u00b71 p<0\\u00b7001 91 i\\u00b2=77\\u00b78 p=0\\u00b7029 92 recent reanalysis fi rst 2015 meta-analysis removed outlying chinese studies accounted imbalances day illness presentation 93 analyses resolved heterogeneity found hypertonic saline reduce length stay infants admitted hospitals bronchiolitis mean diff erence length stay removing outliers -0\\u00b721 days 95 ci -0\\u00b743 0\\u00b702 mean diff erence length stay accounting day illness imbalance 0\\u00b702 days 95 ci -0\\u00b714 0\\u00b717 large trials demonstrated benefi hypertonic saline meta-analysis found eff ect hypertonic saline length stay adjustment outliers imbalances decision undertake future trials controversial given positive results metaanalyses negative results \\n multiple studies examined role corticosteroids management children bronchiolitis data large multicentre trials shown benefi corticosteroids reducing admissions hospital 94 95 2013 cochrane collaboration review supports results studies 96 review included studies enrolled children younger 24 months fi rst episode wheezing signs viral illness included outpatient studies 1824 participants comparing corticosteroids placebo reduction admission day 1 z=1\\u00b705 p=0\\u00b73 day 7 z=1\\u00b738 p=0\\u00b717 enrolment fi gure 4 clinical scores length stay emergency department length time resolution symptoms inpatient studies 772 participants length hospital stay reduced 96 basis evidence multiple clinical practice guidelines recommend use corticosteroids infants bronchiolitis table clinicians report considering family personal history atopy deciding treat infants bronchiolitis corticosteroids 41 evidence infants receive benefi corticosteroid treatment 94 95 96 evidence presence absence respiratory syncytial virus infection infants associated response corticosteroids unavailable 94 95 authors large study factorial design suggested unadjusted analysis high-dose corticosteroids combination nebulised epinephrine reduce admissions outpatients bronchiolitis day 7 95 results considered exploratory 39 45 high-fl ow oxygen respiratory support \\n non-invasive technologies improve oxygenation ventilation bronchiolitis include humidifi ed high-fl ow nasal cannula oxygen continuous positive airway pressure 97 high-fl ow nasal cannula allows delivery high fl ows usually 1 2 l/kg min humidifi cation cannula designed improve patient tolerance widely premature infants mechanisms action unclear particular deliver positive end-expiratory pressure conditions evidence effi cacy high-fl ow nasal cannula predominantly observational studies documenting corneli 2007 goebel 2000 kuyucu 2004 plint 2009 improved respiratory parameters reduced intubation rates implementation 97 small randomised trial compared high-fl ow nasal cannula hypertonic saline found diff erence change respiratory score 98 concerns high-fl ow nasal cannula include potential rapid deterioration infant closely monitored costs associated overuse continuous positive airway pressure studied intensive care settings observational studies small trials evidence improved respiratory parameters 97 uk guidelines recommend considering continuous positive airway pressure children impending respiratory failure bronchiolitis \\n antibiotic overuse children bronchiolitis probably occurs concerns presence fever young age aff ected patients diffi culty diff erentiating atelectasis infectious consolidation chest radiograph concern undetected secondary bacterial infection bronchiolitis clear viral cause occurrence secondary bacterial infections low risk bacteraemia meningitis 1 99 detailed review randomised clinical trials found routine use antibiotics improve duration symptoms length hospital stay need oxygen therapy hospital admission 38 overuse antibiotics known result unnecessary adverse eff ects patient development antimicrobial resistance routine use avoided clear evidence secondary bacterial infection table acute otitis media documented 60 infants bronchiolitis 100 101 antibiotic use acute otitis media bronchiolitis follow established evidence guidelines acute otitis media 102 macrolide antibiotics anti-infl ammatory properties potential benefi mitigating infl ammation present bronchiolitis randomised trials found diff erence azithromycin placebo hospital length stay need oxygen hospital re-admission 103 104 randomised trial found azithromycin lowered nasal interleukin-8 concentrations prolonged time subsequent wheezing episodes resulted fewer days respiratory symptoms year following bronchiolitis episode compared placebo 105 finally multicentre study found children aged 12 72 months history recurrent severe lower respiratory tract infections early administration azithromycin lower respiratory tract infection reduced likelihood progression severe infection clear underlying disease children bronchiolitis disease process 106 given current evidence routine use macrolides recommended bronchiolitis research needed clarify potential role future \\n hydration suctioning chest physiotherapy suggested supportive therapies infants bronchiolitis diffi culty feeding nasal congestion increased work breathing hydration remains cornerstone therapy multicentre study 759 infants younger 12 months admitted hospital bronchiolitis showed benefi intravenous fl uids compared administration fl uids nasogastric tube mean length stay admission intensive care unit need ventilatory support adverse events trial found nasogastric tube easier place intravenous line infants 107 guidelines recommend nasogastric intravenous fl uids maintain hydration uk scottish guidelines preferring nasogastric orogastric hydration tolerate compared intravenous hydration table intravenous fl uids isotonic fl uids preferred avoid risk hyponatraemia 38 infants obligate nasal breathers nasal suctioning suggested help clearing nares improve work breathing improve feeding suctioning irritate nasal mucosa result oedema randomised controlled trials examined role nasal suctioning bronchiolitis insuffi cient available evidence includes retrospective cohort study 740 infants 108 small observational study 40 infants 109 studies suggest deep suctioning increase length stay inpatients 108 infrequent suctioning associated increased length stay 108 oxygen saturation increase suctioning 109 draw conclusions causality observational studies diffi cult potential confouding indication exists eg sicker children likely receive deep suctioning evidence suggests oxygen saturation increases nasal irrigation suctioning 110 current guidelines diff ering recommendations regard suctioning support use recommend superfi cial suctioning deep suctioning table 38 39 45 chest physiotherapy use bronchiolitis appears vary country 111 112 recent cochrane collaboration review 12 studies 1249 participants demonstrated evidence benefi type chest physiotherapy inpatients length stay oxygen saturation respiratory parameters 113 published guidelines routinely recommend chest physiotherapy management uncomplicated bronchiolitis healthy children respiratory comorbidities table 38 39 45 prognosis work published risk developing recurrent wheezing asthma following bronchiolitis infancy 114 115 116 studies followed birth cohorts determine risk subsequent wheezing lower respiratory tract infection young childhood cohorts children admitted hospital bronchiolitis 114 115 116 117 overall admission hospital bronchiolitis young age associated increased risk recurrent wheezing studies report 17 60 children bronchiolitis develop recurrent wheezing years following initial admission hospital large study taiwan followed 1981 children admitted bronchiolitis age 3 years found age 10 years 351 17\\u00b77 1981 children bronchiolitis diagnosis asthma compared 2159 11\\u00b77 18 527 controls hazard ratio 1\\u00b758 95 ci 1\\u00b741 1\\u00b771 118 small cohort study 138 patients suggested 18 39 46 children admitted bronchiolitis 12 months asthma 18 years compared 9 92 controls 115 study followed birth cohort 1246 children found lower respiratory tract infection childhood associated increased risk recurrent wheezing association decreased age signifi nt age 13 years 116 children cohort mild illness requiring hospital admission severity illness associated increased risk asthma 119 question remains respiratory infection young age predisposes children asthma damage alteration lung function children severe bronchiolitis individual risk factors altered immune response airway function predisposes severe bronchiolitis recurrent wheezing 8 \\n substantial knowledge gaps controversies exist management acute bronchiolitis role nebulised hypertonic saline acute management clear resulting confl icting recommendations clinical guidelines table meta-analyses suggest small reduction length stay analyses limited heterogeneity accounting duration illness considering role outlying study populations 91 large multicentre trials support clinically signifi nt diff erence length stay inpatients clear evidence cost benefi t. 4 89 90 91 120 meta-analysis suggests possible reduction admissions outpatients confi dence intervals wide studies heterogeneous 91 multicentre studies nebulised hypertonic saline completed outpatients evidence increasing hypertonic saline little role meaningfully reducing length stay role outpatients clear large outpatient multicentre study clarify benefi t. \\n evidence suggests combined therapy nebulised epinephrine corticosteroids reduce admissions hospital 95 synergy corticosteroids \\u03b22 agonists documented clinical trials asthma management 121 122 123 124 basic science literature shows \\u03b22 agonists corticosteroids enhance eff ectiveness particularly regard antiinfl ammatory gene expression 125 126 economic burden bronchiolitis plausible basic clinical evidence synergy large multicentre trial needed ascertain combined therapy epinephrine corticosteroids benefi cial \\n oxygen saturation use pulse oximetry play important role decision admit infants bronchiolitis hospital length hospital stay 51 52 53 54 55 56 57 58 59 clinical practice guidelines confl icting guidance level oxygen saturation admission considered furthermore substantial proportion discharged infants episodes transient desaturation 54 view large health-care costs associated hospital admission bronchiolitis research needed clarify level oxygen saturation requiring admission role continuous spot measurements oxygen saturation clinical importance transient desaturations stable young infants \\n taf contributed design coordinated writing manuscript authors contributed literature search writing manuscript \\n declare competing interests taf acp jjz participated trials bronchiolitis funded unrestricted public academic institutional funding sources infl uence conception writing manuscript\",\n          \"march april 2009 health professionals warned mexican ministry health increase incidence acute respiratory illness young adults 1 april new viral subtype influenza a(h1n1)pdm09 combination swine avian human virus identified united states samples originated mexico 2 3 april 25 th patients identified sinan database following variables selecting cases controls pcr positive influenza a(h1n1)pdm2009 final classification confirmed criteria laboratory evolution recovered death hospitalization yes date hospitalization 28/06/2009 29/08/2009 municipalities residence hospitals metropolitan areas sao paulo campinas \\n time patients selected study sinan included 193 reported confirmed deaths patients influenza a(h1n1)pdm2009 sari assessed study cases case death control patients selected 1988 patients hospitalized study period reported confirmed influenza a(h1n1)pdm2009 sari recovered controls time-paired randomly selected hospitalized epidemiological week ew necessary week later date hospitalization cases prevent possible differences opportunity treatment clinical management protocol \\n data study collected trained health professionals standardized questionnaires hospital questionnaire collected information medical charts evaluated 177 hospitals patients hospitalized 56.5 belonged unified health system-sus 43.5 private hospitals following data collected previous pathological history occurrence pregnancy health care symptomatology admission laboratory radiological tests need icu admission treatment complications disease progression international classification diseases 10 th revision-icd-10 coding comorbidities indicated medical charts ministry health provided antiviral drug known oseltamivir treatment pandemic protocol clinical management established treatment cases hospitalized sari ambulatory patients risk conditions severity treatment started 48hours symptom onset patients studied laboratory confirmations influenza a(h1n1)pdm09 respiratory secretion samples analyzed rt-pcr method performed adolfo lutz institute 16 definition sari included fever associated cough dyspnea pneumonia respiratory insufficiency tachypnea radiological changes compatible pneumonia oxygen therapy mechanical ventilation sari confirmed cases controls selected study data medical charts \\n household interviews conducted 161 cases 83.4 14 refusals 7.3 18 9.3 addresses found total 337 85.1 controls interviewed 6 1.5 refusals 43 11.1 addresses found household questionnaire included socio-demographic data history previous care related episode led hospitalization presence influenza vaccination occupations grouped according occupational pyramid influenza risk 17 cases interviews conducted closest family members controls patients excluding children parents/legal guardians responded \\n data collection questionnaires assessed supervisors verify consistency information fulfillment criteria include patients study hospital household questionnaires input sphynx database version 5 \\n present study began order assist decisions surveillance control actions use data approved research ethics committee school public health university s\\u00e3o paulo-fsp-usp protocol 2283 of.coep/312/11 data collection performed accordance recommendations national health council human research including signing informed consent form patients family members legal guardians \\n clinical demographic characteristics shown median inter-quartile intervals percentages comparisons mann-whitney u chi-square tests required odds ratio-or 95 confidence intervals-95%ci calculated assess factors associated death variables p-value 0.20 univariate analysis adjustment variables selected multiple logistic regression forward stepwise strategy model including patients assessed following dichotomous categorical variables health insurance previous care vaccination influenza 2009 sex reference category male following risk conditions diabetes mellitus obesity chronic kidney disease chronic liver disease immunosuppression chronic cardiovascular disease chronic neurological disease delay development absence condition reference category model assessed age group 18 years reference 18 59 years 60 years use antiviral therapy use reference category 48 hours onset symptoms 48 72 hours 72 hours high-risk individuals risk conditions listed adapted centers disease control prevention-cdc 18 condition reference category condition conditions time onset symptoms admission days continuous variable model controlled epidemiologic week admission hospital categorized ew26 28 reference ew29 31 ew31 34 income second model developed patients aged 18 years assessed level education formal education incomplete primary education complete primary education complete secondary education complete higher education reference category occupational risk high high average low reference category patients aged 18 22 years incomplete higher education included reference category education modelling patients occupation included reference category occupational risk significance variables assessed wald test adjustment models hosmer-lemeshow test \\n analyses performed spss software version 17.0 p-values 0.05 considered significant table 1 presents socioeconomic characteristics patients information associated disease obtained household interviews patients aged 18 years higher risk death formal education incomplete primary education 2.33(95 ci 1 13 4.89 complete primary education 3.06 95%ci 1.41 6.82 compared higher education distribution ethnic groups family income history smoking associated death history previous vaccination seasonal influenza 2009 showed protection death health professionals considered high risk developing influenza showed protection death 0.15(95%ci 0.02 0.61 patients sought care prior hospitalization risk death 3.70 times 95%ci 2.25 6.30 patients previous care predominance emergency care 45.34 table 2 presents distribution cases controls according variables assessed hospital investigation median age cases higher controls p<0.001 18-to-59-year age group death 4.03(95%ci 2.62 6.31 compared aged 18 years elderly individuals significant risk death \\n differences sex distribution group regard risk conditions severity 73.6 cases 38.1 controls reported risk factor 4.49 95%ci 3.09 6.60 individuals conditions assessed risk death increased 10.29 95%ci 6.04 17.85 following medical conditions risk factors death obesity diabetes mellitus chronic cardiovascular diseases chronic kidney diseases neurological diseases delay development chronic liver diseases immunosuppressive diseases immunosuppression-related diseases neoplasms n 18 hiv/aids n 8) organ transplants n 6 autoimmune diseases n 5 use immunosuppressants n 3 \\n 212 women reproductive age 10 49 years studied 33.5 pregnant differences frequency pregnancy group \\n use antiviral treatment hospitalization protective factor death 0.67 95%ci 0.45 0.98 time onset symptoms beginning antiviral treatment assessed 81 protection individuals received treatment 48 hours 0.19 95%ci 0.11 0.35 65 48 72 hours 0.35 95%ci 0.16 0.74 72 hours significant protection median days date onset symptoms date hospitalization 5.0 cases 2.0 controls p<0.001 patients chronic pulmonary disease median 4.0 days cases 2.0 days controls asthma 4.5 days 2.0 days pregnant women 4.0 days 2.0 days respectively regard health professionals median date onset symptoms hospitalization days cases controls median date hospitalization death hospital discharge seven days cases days controls table 2 \\n according medical charts fever cough occurred 87.8 cases 79.3 controls proportions dyspnea 87 62.7 respectively definition cases severe acute respiratory syndrome -sars fever+cough+dyspnea based information obtained medical charts occurred 68.9 cases 56.5 controls vomit diarrhea present 8 cases 18 controls treatment antibiotics 95.8 cases 73.1 controls mean number antibiotics 4.0 1.8 respectively cases higher proportion hospitalizations intensive care units 83.4%x16.8 use mechanical ventilation 94.8%x10.8 dialysis 25.0%x2.3 proportion complications higher cases controls emphasis respiratory distress syndrome 64.7%vs4.9 shock 69.4%vs1.0 sepsis 63.2%vs2.3 kidney complications 44.6%vs1.3 respectively table 3 shows laboratory findings entry tests patients hospital cases lower numbers leukocytes platelets higher values creatine phosphokinase-cpk lactic dehydrogenase-ldh glutamic-oxaloacetic transaminase glutamic-pyruvic transaminase urea creatinine controls statistical significance \\n sub-group analysis previous conditions risk severity obesity risk factors death univariate analysis children younger 18 years women reproductive age 10 49 years children presence immunosuppression chronic cardiovascular disease neurological disease/delay development risk factors diabetes mellitus women treatment antiviral administered 72 hours onset symptoms protective factor women reproductive age tables 4 5 variables associated death final models multiple logistic regression patients studied sub-group analysis adults aged 18 years patients general risk factors death aged 18 59 years reference 18 years risk conditions severity disease obesity immunosuppression sought care prior hospitalization chronic neurological diseases/delay development chronic liver disease highly associated death variables excluded final model estimates large confidence intervals private health insurance antiviral treatment administered 72 hours onset symptoms protective factors death adults aged 18 years risk factors death risk factor severity disease obesity care prior hospitalization protective factors antiviral drug administered 48 hours onset symptoms health professional health unit worker \\n sensitivity analysis performed assess severity patients admitted study period proportion icu admissions use mechanical ventilation estimated epidemiological week hospitalization categorized ew 26 28 ew 29 31 ew 32 34 results showed higher proportions icu admissions 50.0 x 16.8 x 7.7 mechanical ventilation 45.4 x 10.8 x 2.6 weeks controls cases showed high proportions periods analyzed 89.7 x 84.0 x 76.9 need icu 89.7 x 97.6 x 89.7 need mechanical ventilation 3 periods respectively opportunity antiviral treatment showed lower proportion treatment period study cases 37.9 x 74.4 x 74.3 controls 59.1 x 74.1 x 89.7 sensitivity analysis assess opportunity treatment children adults showed similar proportions groups 73.9 74 respectively \\n present study risk factors death analyzed patients infected influenza (h1n1)pdm09 hospitalized sari increased risk death patients aged 18 59 years obesity immunosuppression neurological developmental diseases received care prior hospitalization additionally presence risk factor aggravation disease associated occurrence death use antiviral drugs especially administered 72 hours onset symptoms protective factor death \\n design case-control study including deaths reported confirmed region study period time-paired random selection controls patients survived hospitalization sari hospital household data collection enabled analysis factors associated death extended including socioeconomic variables access health services data hospitalizations \\n studies assessed risk factors death patients sari 15 19 studies analyzed risk factors severity disease deaths comparing patients hospitalized icus wards 13 14 20 21 22 23 study rate mortality influenza a(h1n1)pdm09 epidemic waves 24 study conducted australia analyzed risk factors hospitalization influenza comparing cases controls infection community 25 studies analyzed epidemiological surveillance data including mortality rates comparisons deaths hospitalizations treatment factors 9 10 12 26 27 28 29 30 31 32 33 34 35 factors associated death identified study described authors presence comorbidities 31 35 age18years 13 obesity 15 19 20 neurological developmental diseases 23 24 31 immunosuppression 15 21 24 34 children showed lower risk death despite similar proportion treatment adults similar result observed study systematic review metaanalysis risk death 0.28 95 ci 0.19 0.141 compared young adults 36 contrast factors associated severity infection reported different researchers diabetes 19 34 chronic pulmonary diseases excluding asthma chronic obstructive pulmonary disease 20 chronic pulmonary disease including asthma 31 chronic cardiovascular disease 21 24 34 associated mortality present study population assessed restricted hospitalized patients severe influenza partly explain differences diabetes chronic cardiovascular kidney disease associated death univariate analysis remain risk factors multiple analysis design study data collected medical charts household interviews allowed adjustments analysis explain differences \\n deaths occurred mainly adults aged 18 59 years small proportion cases controls elderly individuals possible preexisting immunity new influenza a(h1n1)pdm09 virus age group suggested probably previous exposure infections vaccinations influenza a(h1n1 virus genetically antigenically associated new virus contemporary strains seasonal h1n1 viruses 37 \\n patients present study cases controls laboratory-confirmed infection influenza a/h1n1pdm09 38 \\n possible probability hospitalized varied study treatment available hospitalized patients ambulatory patients risk conditions severity influenced hospitalization study population consisting patients sari pandemic criteria hospitalization changed availability treatment especially 3 weeks epidemic shown sensitivity analysis confirming need controlling period hospitalization study \\n socioeconomic profile similar groups assessed difference income private health insurance greater protection death patients occupation higher risk exposure infection health professionals greater opportunity access treatment workers explain findings period date onset symptoms date admission shorter group low proportion deaths patients categorized higher risk exposure observed studies 30 33 \\n pregnancy risk factor death study contrast frequency pregnant women reproductive age hospitalized severe manifestation influenza (h1n1)pdm09 high suggesting pregnancy associated development sari association pregnancy use mechanical ventilation hospitalization influenza a(h1n1)pdm2009 25 higher admission rate reported compared general population 32 2009 state s\\u00e3o paulo 27.4 confirmed cases influenza a(h1n1)pdm09 women reproductive age 10 49 years hospitalized recorded sinan database found pregnant women brazil 4.27 women reproductive age live newborns reference year according 2010 census 39 \\n patients died presented important laboratory changes compared controls lower levels leukocytes platelets increase cpk ldh urea creatinine studies revealed similar trend laboratory results leukocytes 23 cpk 26 27 30 ldh 26 platelets 22 creatinine 27 30 34 \\n patients died received antiviral treatment 9.3 treated 48 hours early treatment protective factor death especially provided 72 hours onset symptoms studies shown favorable prognostic antiviral treatment 13 21 23 27 28 29 34 \\n history care prior hospitalization risk factor death influenza a(h1n1 pdm09 reflect difficulty accessing health services emergency services primarily sought care prior hospitalization contributed greater delay diagnosis beginning treatment study conducted mexico showed similar result regard delay hospitalization 29 \\n study assessed overall standardized mortality rate influenza (h1n1)pdm09 mortality patients preexisting chronic respiratory diseases lower chronic diseases 24 problems recording information previous diseases medical charts presence acute pulmonary disease especially chronic pulmonary disease \\n present study limitations underreporting sari health professionals likely occurred reporting disease compulsory brazil pandemic pandemic professionals sensitive reporting system notification mandatory epidemic facilitated allow rapid notifications availability samples autopsies cases deaths resulted greater access laboratory diagnosis infection household data collection caused information bias data cases collected interviews family members data controls obtained patients adults proportion refusals household interviews higher cases 7.3 controls 1.6 p 0.001 possible limitation memory bias result time hospitalization interview information analyzed household interviews socio-demographic characteristics vulnerable information memory bias data preexisting diseases obtained medical charts hospitals completed moment hospitalization presence patients quality completion medical charts differ hospitals nonetheless information presence chronic diseases clinical management patients laboratory results radiological tests treatment available charts difficulties encountered data collection minimized use standardized questionnaire \\n conclusion risk conditions severity disease particularly obesity immunosuppression neurological/developmental diseases important risk factors death emphasizing need good vaccination coverage influenza vaccination campaigns patients chronic diseases belonging vulnerable groups relevant result antiviral treatment 72 hours onset symptoms protective factor emphasizing need warn population importance seeking early medical care influenza-like illness appears especially high-risk groups signs disease aggravation training health professionals especially physicians required adequate clinical management patients early antiviral treatment\"\n        ],\n        \"semantic_type\": \"\",\n        \"description\": \"\"\n      }\n    },\n    {\n      \"column\": \"cluster\",\n      \"properties\": {\n        \"dtype\": \"int32\",\n        \"num_unique_values\": 11,\n        \"samples\": [\n          0,\n          5,\n          9\n        ],\n        \"semantic_type\": \"\",\n        \"description\": \"\"\n      }\n    },\n    {\n      \"column\": \"hierarchical_cluster\",\n      \"properties\": {\n        \"dtype\": \"number\",\n        \"std\": 2,\n        \"min\": 0,\n        \"max\": 11,\n        \"num_unique_values\": 12,\n        \"samples\": [\n          9,\n          3,\n          2\n        ],\n        \"semantic_type\": \"\",\n        \"description\": \"\"\n      }\n    },\n    {\n      \"column\": \"dbscan_cluster\",\n      \"properties\": {\n        \"dtype\": \"number\",\n        \"std\": 0,\n        \"min\": -1,\n        \"max\": 0,\n        \"num_unique_values\": 2,\n        \"samples\": [\n          0,\n          -1\n        ],\n        \"semantic_type\": \"\",\n        \"description\": \"\"\n      }\n    }\n  ]\n}",
              "type": "dataframe",
              "variable_name": "df"
            },
            "text/html": [
              "\n",
              "  <div id=\"df-6d30044f-5a0e-40d0-8443-c9333cbbeb34\" class=\"colab-df-container\">\n",
              "    <div>\n",
              "<style scoped>\n",
              "    .dataframe tbody tr th:only-of-type {\n",
              "        vertical-align: middle;\n",
              "    }\n",
              "\n",
              "    .dataframe tbody tr th {\n",
              "        vertical-align: top;\n",
              "    }\n",
              "\n",
              "    .dataframe thead th {\n",
              "        text-align: right;\n",
              "    }\n",
              "</style>\n",
              "<table border=\"1\" class=\"dataframe\">\n",
              "  <thead>\n",
              "    <tr style=\"text-align: right;\">\n",
              "      <th></th>\n",
              "      <th>paper_id</th>\n",
              "      <th>doi</th>\n",
              "      <th>abstract</th>\n",
              "      <th>body_text</th>\n",
              "      <th>authors</th>\n",
              "      <th>title</th>\n",
              "      <th>journal</th>\n",
              "      <th>abstract_summary</th>\n",
              "      <th>cleaned_text</th>\n",
              "      <th>cleaned_text_str</th>\n",
              "      <th>cluster</th>\n",
              "      <th>hierarchical_cluster</th>\n",
              "      <th>dbscan_cluster</th>\n",
              "    </tr>\n",
              "  </thead>\n",
              "  <tbody>\n",
              "    <tr>\n",
              "      <th>6252</th>\n",
              "      <td>fefd6cdcddcb00d1ba2ef1a0d3bba9f312010a1d</td>\n",
              "      <td>10.1111/j.1524-4733.2008.00379.x</td>\n",
              "      <td></td>\n",
              "      <td>China is the biggest country in the world that...</td>\n",
              "      <td>Li, Youing.  Sun, Xin.  Wang, Li</td>\n",
              "      <td>Evidence-Based Medicine in China</td>\n",
              "      <td>Value in Health</td>\n",
              "      <td>Not provided.</td>\n",
              "      <td>[china, biggest, country, world, 1.3, billion,...</td>\n",
              "      <td>china biggest country world 1.3 billion popula...</td>\n",
              "      <td>5</td>\n",
              "      <td>2</td>\n",
              "      <td>-1</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>4684</th>\n",
              "      <td>2a3d080d5dcf6771db002366691f35e5d6f0e493</td>\n",
              "      <td>10.1007/s13238-010-0029-7</td>\n",
              "      <td></td>\n",
              "      <td>✉ Correspondence: zlshi@wh.iov.cn Bat, the onl...</td>\n",
              "      <td>Shi, Zhengli</td>\n",
              "      <td>Bat and virus</td>\n",
              "      <td>Protein &amp; Cell</td>\n",
              "      <td>Not provided.</td>\n",
              "      <td>[✉, correspondence, zlshi@wh.iov.cn, bat, flyi...</td>\n",
              "      <td>✉ correspondence zlshi@wh.iov.cn bat flying ma...</td>\n",
              "      <td>3</td>\n",
              "      <td>10</td>\n",
              "      <td>-1</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>1731</th>\n",
              "      <td>2de657aabab331e75657bfc1443542fd345b6f26</td>\n",
              "      <td>10.1016/j.ccc.2019.06.001</td>\n",
              "      <td>KEYWORDS\\nPandemic Influenza Disaster prepared...</td>\n",
              "      <td>Outbreaks of infectious disease pose unique ch...</td>\n",
              "      <td>Maves, Ryan C..  Jamros, Christina M..  Smith...</td>\n",
              "      <td>Intensive Care Unit Preparedness During&lt;br&gt;Pa...</td>\n",
              "      <td>Crit Care Clin</td>\n",
              "      <td>KEYWORDS\\nPandemic Influenza Disaster&lt;br&gt;prep...</td>\n",
              "      <td>[outbreaks, infectious, disease, pose, unique,...</td>\n",
              "      <td>outbreaks infectious disease pose unique chall...</td>\n",
              "      <td>5</td>\n",
              "      <td>2</td>\n",
              "      <td>-1</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>4742</th>\n",
              "      <td>bc22758234283f024f16e2dbe660bc909b512b92</td>\n",
              "      <td>10.1016/j.vetpar.2007.11.020</td>\n",
              "      <td>Cryptosporidium parvum is recognized as one of...</td>\n",
              "      <td>Diarrhoeal diseases of infectious aethiology r...</td>\n",
              "      <td>Klein, Pavel.  Kleinová, Tereza.  Volek,&lt;br&gt;Z...</td>\n",
              "      <td>Effect of Cryptosporidium parvum infection on...</td>\n",
              "      <td>Veterinary Parasitology</td>\n",
              "      <td>Cryptosporidium parvum is recognized as one o...</td>\n",
              "      <td>[diarrhoeal, diseases, infectious, aethiology,...</td>\n",
              "      <td>diarrhoeal diseases infectious aethiology repr...</td>\n",
              "      <td>2</td>\n",
              "      <td>0</td>\n",
              "      <td>-1</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>4521</th>\n",
              "      <td>a204aafa38365dbcc0a26af3ca2c6d3313d7fab2</td>\n",
              "      <td>10.1111/irv.12518</td>\n",
              "      <td>, Brazil, in the period of Background: The res...</td>\n",
              "      <td>The inclusion criteria for asymptomatic adults...</td>\n",
              "      <td>Moreira, Luciana Peniche.  Watanabe, Aripuana...</td>\n",
              "      <td>Respiratory syncytial virus evaluation among&lt;...</td>\n",
              "      <td>Influenza Other Respir Viruses</td>\n",
              "      <td>, Brazil, in the period of Background: The&lt;br...</td>\n",
              "      <td>[inclusion, criteria, asymptomatic, adults, ab...</td>\n",
              "      <td>inclusion criteria asymptomatic adults absence...</td>\n",
              "      <td>10</td>\n",
              "      <td>6</td>\n",
              "      <td>-1</td>\n",
              "    </tr>\n",
              "  </tbody>\n",
              "</table>\n",
              "</div>\n",
              "    <div class=\"colab-df-buttons\">\n",
              "\n",
              "  <div class=\"colab-df-container\">\n",
              "    <button class=\"colab-df-convert\" onclick=\"convertToInteractive('df-6d30044f-5a0e-40d0-8443-c9333cbbeb34')\"\n",
              "            title=\"Convert this dataframe to an interactive table.\"\n",
              "            style=\"display:none;\">\n",
              "\n",
              "  <svg xmlns=\"http://www.w3.org/2000/svg\" height=\"24px\" viewBox=\"0 -960 960 960\">\n",
              "    <path d=\"M120-120v-720h720v720H120Zm60-500h600v-160H180v160Zm220 220h160v-160H400v160Zm0 220h160v-160H400v160ZM180-400h160v-160H180v160Zm440 0h160v-160H620v160ZM180-180h160v-160H180v160Zm440 0h160v-160H620v160Z\"/>\n",
              "  </svg>\n",
              "    </button>\n",
              "\n",
              "  <style>\n",
              "    .colab-df-container {\n",
              "      display:flex;\n",
              "      gap: 12px;\n",
              "    }\n",
              "\n",
              "    .colab-df-convert {\n",
              "      background-color: #E8F0FE;\n",
              "      border: none;\n",
              "      border-radius: 50%;\n",
              "      cursor: pointer;\n",
              "      display: none;\n",
              "      fill: #1967D2;\n",
              "      height: 32px;\n",
              "      padding: 0 0 0 0;\n",
              "      width: 32px;\n",
              "    }\n",
              "\n",
              "    .colab-df-convert:hover {\n",
              "      background-color: #E2EBFA;\n",
              "      box-shadow: 0px 1px 2px rgba(60, 64, 67, 0.3), 0px 1px 3px 1px rgba(60, 64, 67, 0.15);\n",
              "      fill: #174EA6;\n",
              "    }\n",
              "\n",
              "    .colab-df-buttons div {\n",
              "      margin-bottom: 4px;\n",
              "    }\n",
              "\n",
              "    [theme=dark] .colab-df-convert {\n",
              "      background-color: #3B4455;\n",
              "      fill: #D2E3FC;\n",
              "    }\n",
              "\n",
              "    [theme=dark] .colab-df-convert:hover {\n",
              "      background-color: #434B5C;\n",
              "      box-shadow: 0px 1px 3px 1px rgba(0, 0, 0, 0.15);\n",
              "      filter: drop-shadow(0px 1px 2px rgba(0, 0, 0, 0.3));\n",
              "      fill: #FFFFFF;\n",
              "    }\n",
              "  </style>\n",
              "\n",
              "    <script>\n",
              "      const buttonEl =\n",
              "        document.querySelector('#df-6d30044f-5a0e-40d0-8443-c9333cbbeb34 button.colab-df-convert');\n",
              "      buttonEl.style.display =\n",
              "        google.colab.kernel.accessAllowed ? 'block' : 'none';\n",
              "\n",
              "      async function convertToInteractive(key) {\n",
              "        const element = document.querySelector('#df-6d30044f-5a0e-40d0-8443-c9333cbbeb34');\n",
              "        const dataTable =\n",
              "          await google.colab.kernel.invokeFunction('convertToInteractive',\n",
              "                                                    [key], {});\n",
              "        if (!dataTable) return;\n",
              "\n",
              "        const docLinkHtml = 'Like what you see? Visit the ' +\n",
              "          '<a target=\"_blank\" href=https://colab.research.google.com/notebooks/data_table.ipynb>data table notebook</a>'\n",
              "          + ' to learn more about interactive tables.';\n",
              "        element.innerHTML = '';\n",
              "        dataTable['output_type'] = 'display_data';\n",
              "        await google.colab.output.renderOutput(dataTable, element);\n",
              "        const docLink = document.createElement('div');\n",
              "        docLink.innerHTML = docLinkHtml;\n",
              "        element.appendChild(docLink);\n",
              "      }\n",
              "    </script>\n",
              "  </div>\n",
              "\n",
              "\n",
              "<div id=\"df-2b2afbf1-f36b-4e38-bada-c33883d8128c\">\n",
              "  <button class=\"colab-df-quickchart\" onclick=\"quickchart('df-2b2afbf1-f36b-4e38-bada-c33883d8128c')\"\n",
              "            title=\"Suggest charts\"\n",
              "            style=\"display:none;\">\n",
              "\n",
              "<svg xmlns=\"http://www.w3.org/2000/svg\" height=\"24px\"viewBox=\"0 0 24 24\"\n",
              "     width=\"24px\">\n",
              "    <g>\n",
              "        <path d=\"M19 3H5c-1.1 0-2 .9-2 2v14c0 1.1.9 2 2 2h14c1.1 0 2-.9 2-2V5c0-1.1-.9-2-2-2zM9 17H7v-7h2v7zm4 0h-2V7h2v10zm4 0h-2v-4h2v4z\"/>\n",
              "    </g>\n",
              "</svg>\n",
              "  </button>\n",
              "\n",
              "<style>\n",
              "  .colab-df-quickchart {\n",
              "      --bg-color: #E8F0FE;\n",
              "      --fill-color: #1967D2;\n",
              "      --hover-bg-color: #E2EBFA;\n",
              "      --hover-fill-color: #174EA6;\n",
              "      --disabled-fill-color: #AAA;\n",
              "      --disabled-bg-color: #DDD;\n",
              "  }\n",
              "\n",
              "  [theme=dark] .colab-df-quickchart {\n",
              "      --bg-color: #3B4455;\n",
              "      --fill-color: #D2E3FC;\n",
              "      --hover-bg-color: #434B5C;\n",
              "      --hover-fill-color: #FFFFFF;\n",
              "      --disabled-bg-color: #3B4455;\n",
              "      --disabled-fill-color: #666;\n",
              "  }\n",
              "\n",
              "  .colab-df-quickchart {\n",
              "    background-color: var(--bg-color);\n",
              "    border: none;\n",
              "    border-radius: 50%;\n",
              "    cursor: pointer;\n",
              "    display: none;\n",
              "    fill: var(--fill-color);\n",
              "    height: 32px;\n",
              "    padding: 0;\n",
              "    width: 32px;\n",
              "  }\n",
              "\n",
              "  .colab-df-quickchart:hover {\n",
              "    background-color: var(--hover-bg-color);\n",
              "    box-shadow: 0 1px 2px rgba(60, 64, 67, 0.3), 0 1px 3px 1px rgba(60, 64, 67, 0.15);\n",
              "    fill: var(--button-hover-fill-color);\n",
              "  }\n",
              "\n",
              "  .colab-df-quickchart-complete:disabled,\n",
              "  .colab-df-quickchart-complete:disabled:hover {\n",
              "    background-color: var(--disabled-bg-color);\n",
              "    fill: var(--disabled-fill-color);\n",
              "    box-shadow: none;\n",
              "  }\n",
              "\n",
              "  .colab-df-spinner {\n",
              "    border: 2px solid var(--fill-color);\n",
              "    border-color: transparent;\n",
              "    border-bottom-color: var(--fill-color);\n",
              "    animation:\n",
              "      spin 1s steps(1) infinite;\n",
              "  }\n",
              "\n",
              "  @keyframes spin {\n",
              "    0% {\n",
              "      border-color: transparent;\n",
              "      border-bottom-color: var(--fill-color);\n",
              "      border-left-color: var(--fill-color);\n",
              "    }\n",
              "    20% {\n",
              "      border-color: transparent;\n",
              "      border-left-color: var(--fill-color);\n",
              "      border-top-color: var(--fill-color);\n",
              "    }\n",
              "    30% {\n",
              "      border-color: transparent;\n",
              "      border-left-color: var(--fill-color);\n",
              "      border-top-color: var(--fill-color);\n",
              "      border-right-color: var(--fill-color);\n",
              "    }\n",
              "    40% {\n",
              "      border-color: transparent;\n",
              "      border-right-color: var(--fill-color);\n",
              "      border-top-color: var(--fill-color);\n",
              "    }\n",
              "    60% {\n",
              "      border-color: transparent;\n",
              "      border-right-color: var(--fill-color);\n",
              "    }\n",
              "    80% {\n",
              "      border-color: transparent;\n",
              "      border-right-color: var(--fill-color);\n",
              "      border-bottom-color: var(--fill-color);\n",
              "    }\n",
              "    90% {\n",
              "      border-color: transparent;\n",
              "      border-bottom-color: var(--fill-color);\n",
              "    }\n",
              "  }\n",
              "</style>\n",
              "\n",
              "  <script>\n",
              "    async function quickchart(key) {\n",
              "      const quickchartButtonEl =\n",
              "        document.querySelector('#' + key + ' button');\n",
              "      quickchartButtonEl.disabled = true;  // To prevent multiple clicks.\n",
              "      quickchartButtonEl.classList.add('colab-df-spinner');\n",
              "      try {\n",
              "        const charts = await google.colab.kernel.invokeFunction(\n",
              "            'suggestCharts', [key], {});\n",
              "      } catch (error) {\n",
              "        console.error('Error during call to suggestCharts:', error);\n",
              "      }\n",
              "      quickchartButtonEl.classList.remove('colab-df-spinner');\n",
              "      quickchartButtonEl.classList.add('colab-df-quickchart-complete');\n",
              "    }\n",
              "    (() => {\n",
              "      let quickchartButtonEl =\n",
              "        document.querySelector('#df-2b2afbf1-f36b-4e38-bada-c33883d8128c button');\n",
              "      quickchartButtonEl.style.display =\n",
              "        google.colab.kernel.accessAllowed ? 'block' : 'none';\n",
              "    })();\n",
              "  </script>\n",
              "</div>\n",
              "\n",
              "    </div>\n",
              "  </div>\n"
            ],
            "text/plain": [
              "                                      paper_id  \\\n",
              "6252  fefd6cdcddcb00d1ba2ef1a0d3bba9f312010a1d   \n",
              "4684  2a3d080d5dcf6771db002366691f35e5d6f0e493   \n",
              "1731  2de657aabab331e75657bfc1443542fd345b6f26   \n",
              "4742  bc22758234283f024f16e2dbe660bc909b512b92   \n",
              "4521  a204aafa38365dbcc0a26af3ca2c6d3313d7fab2   \n",
              "\n",
              "                                   doi  \\\n",
              "6252  10.1111/j.1524-4733.2008.00379.x   \n",
              "4684         10.1007/s13238-010-0029-7   \n",
              "1731         10.1016/j.ccc.2019.06.001   \n",
              "4742      10.1016/j.vetpar.2007.11.020   \n",
              "4521                 10.1111/irv.12518   \n",
              "\n",
              "                                               abstract  \\\n",
              "6252                                                      \n",
              "4684                                                      \n",
              "1731  KEYWORDS\\nPandemic Influenza Disaster prepared...   \n",
              "4742  Cryptosporidium parvum is recognized as one of...   \n",
              "4521  , Brazil, in the period of Background: The res...   \n",
              "\n",
              "                                              body_text  \\\n",
              "6252  China is the biggest country in the world that...   \n",
              "4684  ✉ Correspondence: zlshi@wh.iov.cn Bat, the onl...   \n",
              "1731  Outbreaks of infectious disease pose unique ch...   \n",
              "4742  Diarrhoeal diseases of infectious aethiology r...   \n",
              "4521  The inclusion criteria for asymptomatic adults...   \n",
              "\n",
              "                                                authors  \\\n",
              "6252                   Li, Youing.  Sun, Xin.  Wang, Li   \n",
              "4684                                       Shi, Zhengli   \n",
              "1731   Maves, Ryan C..  Jamros, Christina M..  Smith...   \n",
              "4742   Klein, Pavel.  Kleinová, Tereza.  Volek,<br>Z...   \n",
              "4521   Moreira, Luciana Peniche.  Watanabe, Aripuana...   \n",
              "\n",
              "                                                  title  \\\n",
              "6252                   Evidence-Based Medicine in China   \n",
              "4684                                      Bat and virus   \n",
              "1731   Intensive Care Unit Preparedness During<br>Pa...   \n",
              "4742   Effect of Cryptosporidium parvum infection on...   \n",
              "4521   Respiratory syncytial virus evaluation among<...   \n",
              "\n",
              "                             journal  \\\n",
              "6252                 Value in Health   \n",
              "4684                  Protein & Cell   \n",
              "1731                  Crit Care Clin   \n",
              "4742         Veterinary Parasitology   \n",
              "4521  Influenza Other Respir Viruses   \n",
              "\n",
              "                                       abstract_summary  \\\n",
              "6252                                      Not provided.   \n",
              "4684                                      Not provided.   \n",
              "1731   KEYWORDS\\nPandemic Influenza Disaster<br>prep...   \n",
              "4742   Cryptosporidium parvum is recognized as one o...   \n",
              "4521   , Brazil, in the period of Background: The<br...   \n",
              "\n",
              "                                           cleaned_text  \\\n",
              "6252  [china, biggest, country, world, 1.3, billion,...   \n",
              "4684  [✉, correspondence, zlshi@wh.iov.cn, bat, flyi...   \n",
              "1731  [outbreaks, infectious, disease, pose, unique,...   \n",
              "4742  [diarrhoeal, diseases, infectious, aethiology,...   \n",
              "4521  [inclusion, criteria, asymptomatic, adults, ab...   \n",
              "\n",
              "                                       cleaned_text_str  cluster  \\\n",
              "6252  china biggest country world 1.3 billion popula...        5   \n",
              "4684  ✉ correspondence zlshi@wh.iov.cn bat flying ma...        3   \n",
              "1731  outbreaks infectious disease pose unique chall...        5   \n",
              "4742  diarrhoeal diseases infectious aethiology repr...        2   \n",
              "4521  inclusion criteria asymptomatic adults absence...       10   \n",
              "\n",
              "      hierarchical_cluster  dbscan_cluster  \n",
              "6252                     2              -1  \n",
              "4684                    10              -1  \n",
              "1731                     2              -1  \n",
              "4742                     0              -1  \n",
              "4521                     6              -1  "
            ]
          },
          "execution_count": 45,
          "metadata": {},
          "output_type": "execute_result"
        }
      ],
      "source": [
        "df.head()"
      ]
    },
    {
      "cell_type": "code",
      "execution_count": 46,
      "metadata": {
        "id": "bnzpP2cW7Q2z"
      },
      "outputs": [],
      "source": [
        "# To String\n",
        "df['cleaned_text_str'] = df['cleaned_text'].apply(lambda tokens: ' '.join(tokens))\n",
        "\n",
        "# Initialize the TF-IDF Vectorizer with a maximum of 4096 features\n",
        "tfidf_vectorizer = TfidfVectorizer(max_features=4096)\n",
        "\n",
        "# Fit and transform the cleaned text to extract TF-IDF features\n",
        "tfidf_matrix = tfidf_vectorizer.fit_transform(df['cleaned_text_str'])\n",
        "\n",
        "# Convert the tfidf_matrix to a DataFrame for easier inspection\n",
        "tfidf_df = pd.DataFrame(tfidf_matrix.toarray(), columns=tfidf_vectorizer.get_feature_names_out())"
      ]
    },
    {
      "cell_type": "code",
      "execution_count": 15,
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/",
          "height": 235
        },
        "id": "yPPC-emutLs4",
        "outputId": "b5e61d5e-964a-4fa6-f266-67b7a35ba455"
      },
      "outputs": [
        {
          "data": {
            "application/vnd.google.colaboratory.intrinsic+json": {
              "type": "dataframe",
              "variable_name": "tfidf_df"
            },
            "text/html": [
              "\n",
              "  <div id=\"df-ac0b2f9f-2c51-446c-81b9-47d336b7a90b\" class=\"colab-df-container\">\n",
              "    <div>\n",
              "<style scoped>\n",
              "    .dataframe tbody tr th:only-of-type {\n",
              "        vertical-align: middle;\n",
              "    }\n",
              "\n",
              "    .dataframe tbody tr th {\n",
              "        vertical-align: top;\n",
              "    }\n",
              "\n",
              "    .dataframe thead th {\n",
              "        text-align: right;\n",
              "    }\n",
              "</style>\n",
              "<table border=\"1\" class=\"dataframe\">\n",
              "  <thead>\n",
              "    <tr style=\"text-align: right;\">\n",
              "      <th></th>\n",
              "      <th>00</th>\n",
              "      <th>000</th>\n",
              "      <th>0001</th>\n",
              "      <th>001</th>\n",
              "      <th>01</th>\n",
              "      <th>02</th>\n",
              "      <th>03</th>\n",
              "      <th>04</th>\n",
              "      <th>05</th>\n",
              "      <th>06</th>\n",
              "      <th>...</th>\n",
              "      <th>zikv</th>\n",
              "      <th>zone</th>\n",
              "      <th>zoonotic</th>\n",
              "      <th>µg</th>\n",
              "      <th>µl</th>\n",
              "      <th>µm</th>\n",
              "      <th>à1</th>\n",
              "      <th>μg</th>\n",
              "      <th>μl</th>\n",
              "      <th>μm</th>\n",
              "    </tr>\n",
              "  </thead>\n",
              "  <tbody>\n",
              "    <tr>\n",
              "      <th>0</th>\n",
              "      <td>0.0</td>\n",
              "      <td>0.028805</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.0</td>\n",
              "      <td>0.00000</td>\n",
              "      <td>0.0</td>\n",
              "      <td>0.023852</td>\n",
              "      <td>0.0</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.0</td>\n",
              "      <td>...</td>\n",
              "      <td>0.0</td>\n",
              "      <td>0.0</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.0</td>\n",
              "      <td>0.0</td>\n",
              "      <td>0.0</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.0</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.0</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>1</th>\n",
              "      <td>0.0</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.0</td>\n",
              "      <td>0.00000</td>\n",
              "      <td>0.0</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.0</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.0</td>\n",
              "      <td>...</td>\n",
              "      <td>0.0</td>\n",
              "      <td>0.0</td>\n",
              "      <td>0.021187</td>\n",
              "      <td>0.0</td>\n",
              "      <td>0.0</td>\n",
              "      <td>0.0</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.0</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.0</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>2</th>\n",
              "      <td>0.0</td>\n",
              "      <td>0.043144</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.0</td>\n",
              "      <td>0.00000</td>\n",
              "      <td>0.0</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.0</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.0</td>\n",
              "      <td>...</td>\n",
              "      <td>0.0</td>\n",
              "      <td>0.0</td>\n",
              "      <td>0.012920</td>\n",
              "      <td>0.0</td>\n",
              "      <td>0.0</td>\n",
              "      <td>0.0</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.0</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.0</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>3</th>\n",
              "      <td>0.0</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.0</td>\n",
              "      <td>0.00852</td>\n",
              "      <td>0.0</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.0</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.0</td>\n",
              "      <td>...</td>\n",
              "      <td>0.0</td>\n",
              "      <td>0.0</td>\n",
              "      <td>0.011280</td>\n",
              "      <td>0.0</td>\n",
              "      <td>0.0</td>\n",
              "      <td>0.0</td>\n",
              "      <td>0.032265</td>\n",
              "      <td>0.0</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.0</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>4</th>\n",
              "      <td>0.0</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.072229</td>\n",
              "      <td>0.0</td>\n",
              "      <td>0.00000</td>\n",
              "      <td>0.0</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.0</td>\n",
              "      <td>0.013501</td>\n",
              "      <td>0.0</td>\n",
              "      <td>...</td>\n",
              "      <td>0.0</td>\n",
              "      <td>0.0</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.0</td>\n",
              "      <td>0.0</td>\n",
              "      <td>0.0</td>\n",
              "      <td>0.000000</td>\n",
              "      <td>0.0</td>\n",
              "      <td>0.021868</td>\n",
              "      <td>0.0</td>\n",
              "    </tr>\n",
              "  </tbody>\n",
              "</table>\n",
              "<p>5 rows × 4096 columns</p>\n",
              "</div>\n",
              "    <div class=\"colab-df-buttons\">\n",
              "\n",
              "  <div class=\"colab-df-container\">\n",
              "    <button class=\"colab-df-convert\" onclick=\"convertToInteractive('df-ac0b2f9f-2c51-446c-81b9-47d336b7a90b')\"\n",
              "            title=\"Convert this dataframe to an interactive table.\"\n",
              "            style=\"display:none;\">\n",
              "\n",
              "  <svg xmlns=\"http://www.w3.org/2000/svg\" height=\"24px\" viewBox=\"0 -960 960 960\">\n",
              "    <path d=\"M120-120v-720h720v720H120Zm60-500h600v-160H180v160Zm220 220h160v-160H400v160Zm0 220h160v-160H400v160ZM180-400h160v-160H180v160Zm440 0h160v-160H620v160ZM180-180h160v-160H180v160Zm440 0h160v-160H620v160Z\"/>\n",
              "  </svg>\n",
              "    </button>\n",
              "\n",
              "  <style>\n",
              "    .colab-df-container {\n",
              "      display:flex;\n",
              "      gap: 12px;\n",
              "    }\n",
              "\n",
              "    .colab-df-convert {\n",
              "      background-color: #E8F0FE;\n",
              "      border: none;\n",
              "      border-radius: 50%;\n",
              "      cursor: pointer;\n",
              "      display: none;\n",
              "      fill: #1967D2;\n",
              "      height: 32px;\n",
              "      padding: 0 0 0 0;\n",
              "      width: 32px;\n",
              "    }\n",
              "\n",
              "    .colab-df-convert:hover {\n",
              "      background-color: #E2EBFA;\n",
              "      box-shadow: 0px 1px 2px rgba(60, 64, 67, 0.3), 0px 1px 3px 1px rgba(60, 64, 67, 0.15);\n",
              "      fill: #174EA6;\n",
              "    }\n",
              "\n",
              "    .colab-df-buttons div {\n",
              "      margin-bottom: 4px;\n",
              "    }\n",
              "\n",
              "    [theme=dark] .colab-df-convert {\n",
              "      background-color: #3B4455;\n",
              "      fill: #D2E3FC;\n",
              "    }\n",
              "\n",
              "    [theme=dark] .colab-df-convert:hover {\n",
              "      background-color: #434B5C;\n",
              "      box-shadow: 0px 1px 3px 1px rgba(0, 0, 0, 0.15);\n",
              "      filter: drop-shadow(0px 1px 2px rgba(0, 0, 0, 0.3));\n",
              "      fill: #FFFFFF;\n",
              "    }\n",
              "  </style>\n",
              "\n",
              "    <script>\n",
              "      const buttonEl =\n",
              "        document.querySelector('#df-ac0b2f9f-2c51-446c-81b9-47d336b7a90b button.colab-df-convert');\n",
              "      buttonEl.style.display =\n",
              "        google.colab.kernel.accessAllowed ? 'block' : 'none';\n",
              "\n",
              "      async function convertToInteractive(key) {\n",
              "        const element = document.querySelector('#df-ac0b2f9f-2c51-446c-81b9-47d336b7a90b');\n",
              "        const dataTable =\n",
              "          await google.colab.kernel.invokeFunction('convertToInteractive',\n",
              "                                                    [key], {});\n",
              "        if (!dataTable) return;\n",
              "\n",
              "        const docLinkHtml = 'Like what you see? Visit the ' +\n",
              "          '<a target=\"_blank\" href=https://colab.research.google.com/notebooks/data_table.ipynb>data table notebook</a>'\n",
              "          + ' to learn more about interactive tables.';\n",
              "        element.innerHTML = '';\n",
              "        dataTable['output_type'] = 'display_data';\n",
              "        await google.colab.output.renderOutput(dataTable, element);\n",
              "        const docLink = document.createElement('div');\n",
              "        docLink.innerHTML = docLinkHtml;\n",
              "        element.appendChild(docLink);\n",
              "      }\n",
              "    </script>\n",
              "  </div>\n",
              "\n",
              "\n",
              "<div id=\"df-a8537031-deab-422f-a393-ec08be6f853b\">\n",
              "  <button class=\"colab-df-quickchart\" onclick=\"quickchart('df-a8537031-deab-422f-a393-ec08be6f853b')\"\n",
              "            title=\"Suggest charts\"\n",
              "            style=\"display:none;\">\n",
              "\n",
              "<svg xmlns=\"http://www.w3.org/2000/svg\" height=\"24px\"viewBox=\"0 0 24 24\"\n",
              "     width=\"24px\">\n",
              "    <g>\n",
              "        <path d=\"M19 3H5c-1.1 0-2 .9-2 2v14c0 1.1.9 2 2 2h14c1.1 0 2-.9 2-2V5c0-1.1-.9-2-2-2zM9 17H7v-7h2v7zm4 0h-2V7h2v10zm4 0h-2v-4h2v4z\"/>\n",
              "    </g>\n",
              "</svg>\n",
              "  </button>\n",
              "\n",
              "<style>\n",
              "  .colab-df-quickchart {\n",
              "      --bg-color: #E8F0FE;\n",
              "      --fill-color: #1967D2;\n",
              "      --hover-bg-color: #E2EBFA;\n",
              "      --hover-fill-color: #174EA6;\n",
              "      --disabled-fill-color: #AAA;\n",
              "      --disabled-bg-color: #DDD;\n",
              "  }\n",
              "\n",
              "  [theme=dark] .colab-df-quickchart {\n",
              "      --bg-color: #3B4455;\n",
              "      --fill-color: #D2E3FC;\n",
              "      --hover-bg-color: #434B5C;\n",
              "      --hover-fill-color: #FFFFFF;\n",
              "      --disabled-bg-color: #3B4455;\n",
              "      --disabled-fill-color: #666;\n",
              "  }\n",
              "\n",
              "  .colab-df-quickchart {\n",
              "    background-color: var(--bg-color);\n",
              "    border: none;\n",
              "    border-radius: 50%;\n",
              "    cursor: pointer;\n",
              "    display: none;\n",
              "    fill: var(--fill-color);\n",
              "    height: 32px;\n",
              "    padding: 0;\n",
              "    width: 32px;\n",
              "  }\n",
              "\n",
              "  .colab-df-quickchart:hover {\n",
              "    background-color: var(--hover-bg-color);\n",
              "    box-shadow: 0 1px 2px rgba(60, 64, 67, 0.3), 0 1px 3px 1px rgba(60, 64, 67, 0.15);\n",
              "    fill: var(--button-hover-fill-color);\n",
              "  }\n",
              "\n",
              "  .colab-df-quickchart-complete:disabled,\n",
              "  .colab-df-quickchart-complete:disabled:hover {\n",
              "    background-color: var(--disabled-bg-color);\n",
              "    fill: var(--disabled-fill-color);\n",
              "    box-shadow: none;\n",
              "  }\n",
              "\n",
              "  .colab-df-spinner {\n",
              "    border: 2px solid var(--fill-color);\n",
              "    border-color: transparent;\n",
              "    border-bottom-color: var(--fill-color);\n",
              "    animation:\n",
              "      spin 1s steps(1) infinite;\n",
              "  }\n",
              "\n",
              "  @keyframes spin {\n",
              "    0% {\n",
              "      border-color: transparent;\n",
              "      border-bottom-color: var(--fill-color);\n",
              "      border-left-color: var(--fill-color);\n",
              "    }\n",
              "    20% {\n",
              "      border-color: transparent;\n",
              "      border-left-color: var(--fill-color);\n",
              "      border-top-color: var(--fill-color);\n",
              "    }\n",
              "    30% {\n",
              "      border-color: transparent;\n",
              "      border-left-color: var(--fill-color);\n",
              "      border-top-color: var(--fill-color);\n",
              "      border-right-color: var(--fill-color);\n",
              "    }\n",
              "    40% {\n",
              "      border-color: transparent;\n",
              "      border-right-color: var(--fill-color);\n",
              "      border-top-color: var(--fill-color);\n",
              "    }\n",
              "    60% {\n",
              "      border-color: transparent;\n",
              "      border-right-color: var(--fill-color);\n",
              "    }\n",
              "    80% {\n",
              "      border-color: transparent;\n",
              "      border-right-color: var(--fill-color);\n",
              "      border-bottom-color: var(--fill-color);\n",
              "    }\n",
              "    90% {\n",
              "      border-color: transparent;\n",
              "      border-bottom-color: var(--fill-color);\n",
              "    }\n",
              "  }\n",
              "</style>\n",
              "\n",
              "  <script>\n",
              "    async function quickchart(key) {\n",
              "      const quickchartButtonEl =\n",
              "        document.querySelector('#' + key + ' button');\n",
              "      quickchartButtonEl.disabled = true;  // To prevent multiple clicks.\n",
              "      quickchartButtonEl.classList.add('colab-df-spinner');\n",
              "      try {\n",
              "        const charts = await google.colab.kernel.invokeFunction(\n",
              "            'suggestCharts', [key], {});\n",
              "      } catch (error) {\n",
              "        console.error('Error during call to suggestCharts:', error);\n",
              "      }\n",
              "      quickchartButtonEl.classList.remove('colab-df-spinner');\n",
              "      quickchartButtonEl.classList.add('colab-df-quickchart-complete');\n",
              "    }\n",
              "    (() => {\n",
              "      let quickchartButtonEl =\n",
              "        document.querySelector('#df-a8537031-deab-422f-a393-ec08be6f853b button');\n",
              "      quickchartButtonEl.style.display =\n",
              "        google.colab.kernel.accessAllowed ? 'block' : 'none';\n",
              "    })();\n",
              "  </script>\n",
              "</div>\n",
              "\n",
              "    </div>\n",
              "  </div>\n"
            ],
            "text/plain": [
              "    00       000      0001  001       01   02        03   04        05   06  \\\n",
              "0  0.0  0.028805  0.000000  0.0  0.00000  0.0  0.023852  0.0  0.000000  0.0   \n",
              "1  0.0  0.000000  0.000000  0.0  0.00000  0.0  0.000000  0.0  0.000000  0.0   \n",
              "2  0.0  0.043144  0.000000  0.0  0.00000  0.0  0.000000  0.0  0.000000  0.0   \n",
              "3  0.0  0.000000  0.000000  0.0  0.00852  0.0  0.000000  0.0  0.000000  0.0   \n",
              "4  0.0  0.000000  0.072229  0.0  0.00000  0.0  0.000000  0.0  0.013501  0.0   \n",
              "\n",
              "   ...  zikv  zone  zoonotic   µg   µl   µm        à1   μg        μl   μm  \n",
              "0  ...   0.0   0.0  0.000000  0.0  0.0  0.0  0.000000  0.0  0.000000  0.0  \n",
              "1  ...   0.0   0.0  0.021187  0.0  0.0  0.0  0.000000  0.0  0.000000  0.0  \n",
              "2  ...   0.0   0.0  0.012920  0.0  0.0  0.0  0.000000  0.0  0.000000  0.0  \n",
              "3  ...   0.0   0.0  0.011280  0.0  0.0  0.0  0.032265  0.0  0.000000  0.0  \n",
              "4  ...   0.0   0.0  0.000000  0.0  0.0  0.0  0.000000  0.0  0.021868  0.0  \n",
              "\n",
              "[5 rows x 4096 columns]"
            ]
          },
          "execution_count": 15,
          "metadata": {},
          "output_type": "execute_result"
        }
      ],
      "source": [
        "tfidf_df.head()"
      ]
    },
    {
      "cell_type": "code",
      "execution_count": 47,
      "metadata": {
        "id": "Nn7maPOiEen9"
      },
      "outputs": [],
      "source": [
        "# Initialize PCA to retain 95% of the variance\n",
        "pca = PCA(n_components=0.95, random_state=42)\n",
        "\n",
        "# Fit and transform the TF-IDF matrix\n",
        "tfidf_pca = pca.fit_transform(tfidf_matrix.toarray())\n",
        "\n",
        "# Create a DataFrame for the PCA-transformed features\n",
        "pca_df = pd.DataFrame(tfidf_pca)"
      ]
    },
    {
      "cell_type": "code",
      "execution_count": 48,
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/",
          "height": 0
        },
        "id": "YmVrIicitRBk",
        "outputId": "9acb9201-fe41-428d-f1a2-e13d5110aa28"
      },
      "outputs": [
        {
          "name": "stdout",
          "output_type": "stream",
          "text": [
            "Original number of features: 4096\n",
            "Reduced number of features: 953\n"
          ]
        }
      ],
      "source": [
        "print(f\"Original number of features: {tfidf_df.shape[1]}\")\n",
        "print(f\"Reduced number of features: {pca_df.shape[1]}\")"
      ]
    },
    {
      "cell_type": "code",
      "execution_count": 49,
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/",
          "height": 476
        },
        "id": "Spx1oT1etTAG",
        "outputId": "21e84afa-47b6-4c2f-94f0-86bbfac8727c"
      },
      "outputs": [
        {
          "data": {
            "image/svg+xml": [
              "<?xml version=\"1.0\" encoding=\"utf-8\" standalone=\"no\"?>\n",
              "<!DOCTYPE svg PUBLIC \"-//W3C//DTD SVG 1.1//EN\"\n",
              "  \"http://www.w3.org/Graphics/SVG/1.1/DTD/svg11.dtd\">\n",
              "<svg xmlns:xlink=\"http://www.w3.org/1999/xlink\" width=\"610.916875pt\" height=\"340.833625pt\" viewBox=\"0 0 610.916875 340.833625\" xmlns=\"http://www.w3.org/2000/svg\" version=\"1.1\">\n",
              " <metadata>\n",
              "  <rdf:RDF xmlns:dc=\"http://purl.org/dc/elements/1.1/\" xmlns:cc=\"http://creativecommons.org/ns#\" xmlns:rdf=\"http://www.w3.org/1999/02/22-rdf-syntax-ns#\">\n",
              "   <cc:Work>\n",
              "    <dc:type rdf:resource=\"http://purl.org/dc/dcmitype/StillImage\"/>\n",
              "    <dc:date>2024-06-05T18:03:06.108475</dc:date>\n",
              "    <dc:format>image/svg+xml</dc:format>\n",
              "    <dc:creator>\n",
              "     <cc:Agent>\n",
              "      <dc:title>Matplotlib v3.7.1, https://matplotlib.org/</dc:title>\n",
              "     </cc:Agent>\n",
              "    </dc:creator>\n",
              "   </cc:Work>\n",
              "  </rdf:RDF>\n",
              " </metadata>\n",
              " <defs>\n",
              "  <style type=\"text/css\">*{stroke-linejoin: round; stroke-linecap: butt}</style>\n",
              " </defs>\n",
              " <g id=\"figure_1\">\n",
              "  <g id=\"patch_1\">\n",
              "   <path d=\"M 0 340.833625 \n",
              "L 610.916875 340.833625 \n",
              "L 610.916875 0 \n",
              "L 0 0 \n",
              "z\n",
              "\" style=\"fill: #ffffff\"/>\n",
              "  </g>\n",
              "  <g id=\"axes_1\">\n",
              "   <g id=\"patch_2\">\n",
              "    <path d=\"M 45.716875 301.34175 \n",
              "L 603.716875 301.34175 \n",
              "L 603.716875 24.14175 \n",
              "L 45.716875 24.14175 \n",
              "z\n",
              "\" style=\"fill: #e5e5e5\"/>\n",
              "   </g>\n",
              "   <g id=\"matplotlib.axis_1\">\n",
              "    <g id=\"xtick_1\">\n",
              "     <g id=\"line2d_1\">\n",
              "      <defs>\n",
              "       <path id=\"m665e335ae6\" d=\"M 0 0 \n",
              "L 0 3.5 \n",
              "\" style=\"stroke: #555555; stroke-width: 0.8\"/>\n",
              "      </defs>\n",
              "      <g>\n",
              "       <use xlink:href=\"#m665e335ae6\" x=\"71.080511\" y=\"301.34175\" style=\"fill: #555555; stroke: #555555; stroke-width: 0.8\"/>\n",
              "      </g>\n",
              "     </g>\n",
              "     <g id=\"text_1\">\n",
              "      <!-- 0 -->\n",
              "      <g style=\"fill: #555555\" transform=\"translate(67.899261 315.940188) scale(0.1 -0.1)\">\n",
              "       <defs>\n",
              "        <path id=\"DejaVuSans-30\" d=\"M 2034 4250 \n",
              "Q 1547 4250 1301 3770 \n",
              "Q 1056 3291 1056 2328 \n",
              "Q 1056 1369 1301 889 \n",
              "Q 1547 409 2034 409 \n",
              "Q 2525 409 2770 889 \n",
              "Q 3016 1369 3016 2328 \n",
              "Q 3016 3291 2770 3770 \n",
              "Q 2525 4250 2034 4250 \n",
              "z\n",
              "M 2034 4750 \n",
              "Q 2819 4750 3233 4129 \n",
              "Q 3647 3509 3647 2328 \n",
              "Q 3647 1150 3233 529 \n",
              "Q 2819 -91 2034 -91 \n",
              "Q 1250 -91 836 529 \n",
              "Q 422 1150 422 2328 \n",
              "Q 422 3509 836 4129 \n",
              "Q 1250 4750 2034 4750 \n",
              "z\n",
              "\" transform=\"scale(0.015625)\"/>\n",
              "       </defs>\n",
              "       <use xlink:href=\"#DejaVuSans-30\"/>\n",
              "      </g>\n",
              "     </g>\n",
              "    </g>\n",
              "    <g id=\"xtick_2\">\n",
              "     <g id=\"line2d_2\">\n",
              "      <g>\n",
              "       <use xlink:href=\"#m665e335ae6\" x=\"177.650412\" y=\"301.34175\" style=\"fill: #555555; stroke: #555555; stroke-width: 0.8\"/>\n",
              "      </g>\n",
              "     </g>\n",
              "     <g id=\"text_2\">\n",
              "      <!-- 200 -->\n",
              "      <g style=\"fill: #555555\" transform=\"translate(168.106662 315.940188) scale(0.1 -0.1)\">\n",
              "       <defs>\n",
              "        <path id=\"DejaVuSans-32\" d=\"M 1228 531 \n",
              "L 3431 531 \n",
              "L 3431 0 \n",
              "L 469 0 \n",
              "L 469 531 \n",
              "Q 828 903 1448 1529 \n",
              "Q 2069 2156 2228 2338 \n",
              "Q 2531 2678 2651 2914 \n",
              "Q 2772 3150 2772 3378 \n",
              "Q 2772 3750 2511 3984 \n",
              "Q 2250 4219 1831 4219 \n",
              "Q 1534 4219 1204 4116 \n",
              "Q 875 4013 500 3803 \n",
              "L 500 4441 \n",
              "Q 881 4594 1212 4672 \n",
              "Q 1544 4750 1819 4750 \n",
              "Q 2544 4750 2975 4387 \n",
              "Q 3406 4025 3406 3419 \n",
              "Q 3406 3131 3298 2873 \n",
              "Q 3191 2616 2906 2266 \n",
              "Q 2828 2175 2409 1742 \n",
              "Q 1991 1309 1228 531 \n",
              "z\n",
              "\" transform=\"scale(0.015625)\"/>\n",
              "       </defs>\n",
              "       <use xlink:href=\"#DejaVuSans-32\"/>\n",
              "       <use xlink:href=\"#DejaVuSans-30\" x=\"63.623047\"/>\n",
              "       <use xlink:href=\"#DejaVuSans-30\" x=\"127.246094\"/>\n",
              "      </g>\n",
              "     </g>\n",
              "    </g>\n",
              "    <g id=\"xtick_3\">\n",
              "     <g id=\"line2d_3\">\n",
              "      <g>\n",
              "       <use xlink:href=\"#m665e335ae6\" x=\"284.220313\" y=\"301.34175\" style=\"fill: #555555; stroke: #555555; stroke-width: 0.8\"/>\n",
              "      </g>\n",
              "     </g>\n",
              "     <g id=\"text_3\">\n",
              "      <!-- 400 -->\n",
              "      <g style=\"fill: #555555\" transform=\"translate(274.676563 315.940188) scale(0.1 -0.1)\">\n",
              "       <defs>\n",
              "        <path id=\"DejaVuSans-34\" d=\"M 2419 4116 \n",
              "L 825 1625 \n",
              "L 2419 1625 \n",
              "L 2419 4116 \n",
              "z\n",
              "M 2253 4666 \n",
              "L 3047 4666 \n",
              "L 3047 1625 \n",
              "L 3713 1625 \n",
              "L 3713 1100 \n",
              "L 3047 1100 \n",
              "L 3047 0 \n",
              "L 2419 0 \n",
              "L 2419 1100 \n",
              "L 313 1100 \n",
              "L 313 1709 \n",
              "L 2253 4666 \n",
              "z\n",
              "\" transform=\"scale(0.015625)\"/>\n",
              "       </defs>\n",
              "       <use xlink:href=\"#DejaVuSans-34\"/>\n",
              "       <use xlink:href=\"#DejaVuSans-30\" x=\"63.623047\"/>\n",
              "       <use xlink:href=\"#DejaVuSans-30\" x=\"127.246094\"/>\n",
              "      </g>\n",
              "     </g>\n",
              "    </g>\n",
              "    <g id=\"xtick_4\">\n",
              "     <g id=\"line2d_4\">\n",
              "      <g>\n",
              "       <use xlink:href=\"#m665e335ae6\" x=\"390.790213\" y=\"301.34175\" style=\"fill: #555555; stroke: #555555; stroke-width: 0.8\"/>\n",
              "      </g>\n",
              "     </g>\n",
              "     <g id=\"text_4\">\n",
              "      <!-- 600 -->\n",
              "      <g style=\"fill: #555555\" transform=\"translate(381.246463 315.940188) scale(0.1 -0.1)\">\n",
              "       <defs>\n",
              "        <path id=\"DejaVuSans-36\" d=\"M 2113 2584 \n",
              "Q 1688 2584 1439 2293 \n",
              "Q 1191 2003 1191 1497 \n",
              "Q 1191 994 1439 701 \n",
              "Q 1688 409 2113 409 \n",
              "Q 2538 409 2786 701 \n",
              "Q 3034 994 3034 1497 \n",
              "Q 3034 2003 2786 2293 \n",
              "Q 2538 2584 2113 2584 \n",
              "z\n",
              "M 3366 4563 \n",
              "L 3366 3988 \n",
              "Q 3128 4100 2886 4159 \n",
              "Q 2644 4219 2406 4219 \n",
              "Q 1781 4219 1451 3797 \n",
              "Q 1122 3375 1075 2522 \n",
              "Q 1259 2794 1537 2939 \n",
              "Q 1816 3084 2150 3084 \n",
              "Q 2853 3084 3261 2657 \n",
              "Q 3669 2231 3669 1497 \n",
              "Q 3669 778 3244 343 \n",
              "Q 2819 -91 2113 -91 \n",
              "Q 1303 -91 875 529 \n",
              "Q 447 1150 447 2328 \n",
              "Q 447 3434 972 4092 \n",
              "Q 1497 4750 2381 4750 \n",
              "Q 2619 4750 2861 4703 \n",
              "Q 3103 4656 3366 4563 \n",
              "z\n",
              "\" transform=\"scale(0.015625)\"/>\n",
              "       </defs>\n",
              "       <use xlink:href=\"#DejaVuSans-36\"/>\n",
              "       <use xlink:href=\"#DejaVuSans-30\" x=\"63.623047\"/>\n",
              "       <use xlink:href=\"#DejaVuSans-30\" x=\"127.246094\"/>\n",
              "      </g>\n",
              "     </g>\n",
              "    </g>\n",
              "    <g id=\"xtick_5\">\n",
              "     <g id=\"line2d_5\">\n",
              "      <g>\n",
              "       <use xlink:href=\"#m665e335ae6\" x=\"497.360114\" y=\"301.34175\" style=\"fill: #555555; stroke: #555555; stroke-width: 0.8\"/>\n",
              "      </g>\n",
              "     </g>\n",
              "     <g id=\"text_5\">\n",
              "      <!-- 800 -->\n",
              "      <g style=\"fill: #555555\" transform=\"translate(487.816364 315.940188) scale(0.1 -0.1)\">\n",
              "       <defs>\n",
              "        <path id=\"DejaVuSans-38\" d=\"M 2034 2216 \n",
              "Q 1584 2216 1326 1975 \n",
              "Q 1069 1734 1069 1313 \n",
              "Q 1069 891 1326 650 \n",
              "Q 1584 409 2034 409 \n",
              "Q 2484 409 2743 651 \n",
              "Q 3003 894 3003 1313 \n",
              "Q 3003 1734 2745 1975 \n",
              "Q 2488 2216 2034 2216 \n",
              "z\n",
              "M 1403 2484 \n",
              "Q 997 2584 770 2862 \n",
              "Q 544 3141 544 3541 \n",
              "Q 544 4100 942 4425 \n",
              "Q 1341 4750 2034 4750 \n",
              "Q 2731 4750 3128 4425 \n",
              "Q 3525 4100 3525 3541 \n",
              "Q 3525 3141 3298 2862 \n",
              "Q 3072 2584 2669 2484 \n",
              "Q 3125 2378 3379 2068 \n",
              "Q 3634 1759 3634 1313 \n",
              "Q 3634 634 3220 271 \n",
              "Q 2806 -91 2034 -91 \n",
              "Q 1263 -91 848 271 \n",
              "Q 434 634 434 1313 \n",
              "Q 434 1759 690 2068 \n",
              "Q 947 2378 1403 2484 \n",
              "z\n",
              "M 1172 3481 \n",
              "Q 1172 3119 1398 2916 \n",
              "Q 1625 2713 2034 2713 \n",
              "Q 2441 2713 2670 2916 \n",
              "Q 2900 3119 2900 3481 \n",
              "Q 2900 3844 2670 4047 \n",
              "Q 2441 4250 2034 4250 \n",
              "Q 1625 4250 1398 4047 \n",
              "Q 1172 3844 1172 3481 \n",
              "z\n",
              "\" transform=\"scale(0.015625)\"/>\n",
              "       </defs>\n",
              "       <use xlink:href=\"#DejaVuSans-38\"/>\n",
              "       <use xlink:href=\"#DejaVuSans-30\" x=\"63.623047\"/>\n",
              "       <use xlink:href=\"#DejaVuSans-30\" x=\"127.246094\"/>\n",
              "      </g>\n",
              "     </g>\n",
              "    </g>\n",
              "    <g id=\"text_6\">\n",
              "     <!-- Number of Components -->\n",
              "     <g style=\"fill: #555555\" transform=\"translate(253.05625 331.138) scale(0.12 -0.12)\">\n",
              "      <defs>\n",
              "       <path id=\"DejaVuSans-4e\" d=\"M 628 4666 \n",
              "L 1478 4666 \n",
              "L 3547 763 \n",
              "L 3547 4666 \n",
              "L 4159 4666 \n",
              "L 4159 0 \n",
              "L 3309 0 \n",
              "L 1241 3903 \n",
              "L 1241 0 \n",
              "L 628 0 \n",
              "L 628 4666 \n",
              "z\n",
              "\" transform=\"scale(0.015625)\"/>\n",
              "       <path id=\"DejaVuSans-75\" d=\"M 544 1381 \n",
              "L 544 3500 \n",
              "L 1119 3500 \n",
              "L 1119 1403 \n",
              "Q 1119 906 1312 657 \n",
              "Q 1506 409 1894 409 \n",
              "Q 2359 409 2629 706 \n",
              "Q 2900 1003 2900 1516 \n",
              "L 2900 3500 \n",
              "L 3475 3500 \n",
              "L 3475 0 \n",
              "L 2900 0 \n",
              "L 2900 538 \n",
              "Q 2691 219 2414 64 \n",
              "Q 2138 -91 1772 -91 \n",
              "Q 1169 -91 856 284 \n",
              "Q 544 659 544 1381 \n",
              "z\n",
              "M 1991 3584 \n",
              "L 1991 3584 \n",
              "z\n",
              "\" transform=\"scale(0.015625)\"/>\n",
              "       <path id=\"DejaVuSans-6d\" d=\"M 3328 2828 \n",
              "Q 3544 3216 3844 3400 \n",
              "Q 4144 3584 4550 3584 \n",
              "Q 5097 3584 5394 3201 \n",
              "Q 5691 2819 5691 2113 \n",
              "L 5691 0 \n",
              "L 5113 0 \n",
              "L 5113 2094 \n",
              "Q 5113 2597 4934 2840 \n",
              "Q 4756 3084 4391 3084 \n",
              "Q 3944 3084 3684 2787 \n",
              "Q 3425 2491 3425 1978 \n",
              "L 3425 0 \n",
              "L 2847 0 \n",
              "L 2847 2094 \n",
              "Q 2847 2600 2669 2842 \n",
              "Q 2491 3084 2119 3084 \n",
              "Q 1678 3084 1418 2786 \n",
              "Q 1159 2488 1159 1978 \n",
              "L 1159 0 \n",
              "L 581 0 \n",
              "L 581 3500 \n",
              "L 1159 3500 \n",
              "L 1159 2956 \n",
              "Q 1356 3278 1631 3431 \n",
              "Q 1906 3584 2284 3584 \n",
              "Q 2666 3584 2933 3390 \n",
              "Q 3200 3197 3328 2828 \n",
              "z\n",
              "\" transform=\"scale(0.015625)\"/>\n",
              "       <path id=\"DejaVuSans-62\" d=\"M 3116 1747 \n",
              "Q 3116 2381 2855 2742 \n",
              "Q 2594 3103 2138 3103 \n",
              "Q 1681 3103 1420 2742 \n",
              "Q 1159 2381 1159 1747 \n",
              "Q 1159 1113 1420 752 \n",
              "Q 1681 391 2138 391 \n",
              "Q 2594 391 2855 752 \n",
              "Q 3116 1113 3116 1747 \n",
              "z\n",
              "M 1159 2969 \n",
              "Q 1341 3281 1617 3432 \n",
              "Q 1894 3584 2278 3584 \n",
              "Q 2916 3584 3314 3078 \n",
              "Q 3713 2572 3713 1747 \n",
              "Q 3713 922 3314 415 \n",
              "Q 2916 -91 2278 -91 \n",
              "Q 1894 -91 1617 61 \n",
              "Q 1341 213 1159 525 \n",
              "L 1159 0 \n",
              "L 581 0 \n",
              "L 581 4863 \n",
              "L 1159 4863 \n",
              "L 1159 2969 \n",
              "z\n",
              "\" transform=\"scale(0.015625)\"/>\n",
              "       <path id=\"DejaVuSans-65\" d=\"M 3597 1894 \n",
              "L 3597 1613 \n",
              "L 953 1613 \n",
              "Q 991 1019 1311 708 \n",
              "Q 1631 397 2203 397 \n",
              "Q 2534 397 2845 478 \n",
              "Q 3156 559 3463 722 \n",
              "L 3463 178 \n",
              "Q 3153 47 2828 -22 \n",
              "Q 2503 -91 2169 -91 \n",
              "Q 1331 -91 842 396 \n",
              "Q 353 884 353 1716 \n",
              "Q 353 2575 817 3079 \n",
              "Q 1281 3584 2069 3584 \n",
              "Q 2775 3584 3186 3129 \n",
              "Q 3597 2675 3597 1894 \n",
              "z\n",
              "M 3022 2063 \n",
              "Q 3016 2534 2758 2815 \n",
              "Q 2500 3097 2075 3097 \n",
              "Q 1594 3097 1305 2825 \n",
              "Q 1016 2553 972 2059 \n",
              "L 3022 2063 \n",
              "z\n",
              "\" transform=\"scale(0.015625)\"/>\n",
              "       <path id=\"DejaVuSans-72\" d=\"M 2631 2963 \n",
              "Q 2534 3019 2420 3045 \n",
              "Q 2306 3072 2169 3072 \n",
              "Q 1681 3072 1420 2755 \n",
              "Q 1159 2438 1159 1844 \n",
              "L 1159 0 \n",
              "L 581 0 \n",
              "L 581 3500 \n",
              "L 1159 3500 \n",
              "L 1159 2956 \n",
              "Q 1341 3275 1631 3429 \n",
              "Q 1922 3584 2338 3584 \n",
              "Q 2397 3584 2469 3576 \n",
              "Q 2541 3569 2628 3553 \n",
              "L 2631 2963 \n",
              "z\n",
              "\" transform=\"scale(0.015625)\"/>\n",
              "       <path id=\"DejaVuSans-20\" transform=\"scale(0.015625)\"/>\n",
              "       <path id=\"DejaVuSans-6f\" d=\"M 1959 3097 \n",
              "Q 1497 3097 1228 2736 \n",
              "Q 959 2375 959 1747 \n",
              "Q 959 1119 1226 758 \n",
              "Q 1494 397 1959 397 \n",
              "Q 2419 397 2687 759 \n",
              "Q 2956 1122 2956 1747 \n",
              "Q 2956 2369 2687 2733 \n",
              "Q 2419 3097 1959 3097 \n",
              "z\n",
              "M 1959 3584 \n",
              "Q 2709 3584 3137 3096 \n",
              "Q 3566 2609 3566 1747 \n",
              "Q 3566 888 3137 398 \n",
              "Q 2709 -91 1959 -91 \n",
              "Q 1206 -91 779 398 \n",
              "Q 353 888 353 1747 \n",
              "Q 353 2609 779 3096 \n",
              "Q 1206 3584 1959 3584 \n",
              "z\n",
              "\" transform=\"scale(0.015625)\"/>\n",
              "       <path id=\"DejaVuSans-66\" d=\"M 2375 4863 \n",
              "L 2375 4384 \n",
              "L 1825 4384 \n",
              "Q 1516 4384 1395 4259 \n",
              "Q 1275 4134 1275 3809 \n",
              "L 1275 3500 \n",
              "L 2222 3500 \n",
              "L 2222 3053 \n",
              "L 1275 3053 \n",
              "L 1275 0 \n",
              "L 697 0 \n",
              "L 697 3053 \n",
              "L 147 3053 \n",
              "L 147 3500 \n",
              "L 697 3500 \n",
              "L 697 3744 \n",
              "Q 697 4328 969 4595 \n",
              "Q 1241 4863 1831 4863 \n",
              "L 2375 4863 \n",
              "z\n",
              "\" transform=\"scale(0.015625)\"/>\n",
              "       <path id=\"DejaVuSans-43\" d=\"M 4122 4306 \n",
              "L 4122 3641 \n",
              "Q 3803 3938 3442 4084 \n",
              "Q 3081 4231 2675 4231 \n",
              "Q 1875 4231 1450 3742 \n",
              "Q 1025 3253 1025 2328 \n",
              "Q 1025 1406 1450 917 \n",
              "Q 1875 428 2675 428 \n",
              "Q 3081 428 3442 575 \n",
              "Q 3803 722 4122 1019 \n",
              "L 4122 359 \n",
              "Q 3791 134 3420 21 \n",
              "Q 3050 -91 2638 -91 \n",
              "Q 1578 -91 968 557 \n",
              "Q 359 1206 359 2328 \n",
              "Q 359 3453 968 4101 \n",
              "Q 1578 4750 2638 4750 \n",
              "Q 3056 4750 3426 4639 \n",
              "Q 3797 4528 4122 4306 \n",
              "z\n",
              "\" transform=\"scale(0.015625)\"/>\n",
              "       <path id=\"DejaVuSans-70\" d=\"M 1159 525 \n",
              "L 1159 -1331 \n",
              "L 581 -1331 \n",
              "L 581 3500 \n",
              "L 1159 3500 \n",
              "L 1159 2969 \n",
              "Q 1341 3281 1617 3432 \n",
              "Q 1894 3584 2278 3584 \n",
              "Q 2916 3584 3314 3078 \n",
              "Q 3713 2572 3713 1747 \n",
              "Q 3713 922 3314 415 \n",
              "Q 2916 -91 2278 -91 \n",
              "Q 1894 -91 1617 61 \n",
              "Q 1341 213 1159 525 \n",
              "z\n",
              "M 3116 1747 \n",
              "Q 3116 2381 2855 2742 \n",
              "Q 2594 3103 2138 3103 \n",
              "Q 1681 3103 1420 2742 \n",
              "Q 1159 2381 1159 1747 \n",
              "Q 1159 1113 1420 752 \n",
              "Q 1681 391 2138 391 \n",
              "Q 2594 391 2855 752 \n",
              "Q 3116 1113 3116 1747 \n",
              "z\n",
              "\" transform=\"scale(0.015625)\"/>\n",
              "       <path id=\"DejaVuSans-6e\" d=\"M 3513 2113 \n",
              "L 3513 0 \n",
              "L 2938 0 \n",
              "L 2938 2094 \n",
              "Q 2938 2591 2744 2837 \n",
              "Q 2550 3084 2163 3084 \n",
              "Q 1697 3084 1428 2787 \n",
              "Q 1159 2491 1159 1978 \n",
              "L 1159 0 \n",
              "L 581 0 \n",
              "L 581 3500 \n",
              "L 1159 3500 \n",
              "L 1159 2956 \n",
              "Q 1366 3272 1645 3428 \n",
              "Q 1925 3584 2291 3584 \n",
              "Q 2894 3584 3203 3211 \n",
              "Q 3513 2838 3513 2113 \n",
              "z\n",
              "\" transform=\"scale(0.015625)\"/>\n",
              "       <path id=\"DejaVuSans-74\" d=\"M 1172 4494 \n",
              "L 1172 3500 \n",
              "L 2356 3500 \n",
              "L 2356 3053 \n",
              "L 1172 3053 \n",
              "L 1172 1153 \n",
              "Q 1172 725 1289 603 \n",
              "Q 1406 481 1766 481 \n",
              "L 2356 481 \n",
              "L 2356 0 \n",
              "L 1766 0 \n",
              "Q 1100 0 847 248 \n",
              "Q 594 497 594 1153 \n",
              "L 594 3053 \n",
              "L 172 3053 \n",
              "L 172 3500 \n",
              "L 594 3500 \n",
              "L 594 4494 \n",
              "L 1172 4494 \n",
              "z\n",
              "\" transform=\"scale(0.015625)\"/>\n",
              "       <path id=\"DejaVuSans-73\" d=\"M 2834 3397 \n",
              "L 2834 2853 \n",
              "Q 2591 2978 2328 3040 \n",
              "Q 2066 3103 1784 3103 \n",
              "Q 1356 3103 1142 2972 \n",
              "Q 928 2841 928 2578 \n",
              "Q 928 2378 1081 2264 \n",
              "Q 1234 2150 1697 2047 \n",
              "L 1894 2003 \n",
              "Q 2506 1872 2764 1633 \n",
              "Q 3022 1394 3022 966 \n",
              "Q 3022 478 2636 193 \n",
              "Q 2250 -91 1575 -91 \n",
              "Q 1294 -91 989 -36 \n",
              "Q 684 19 347 128 \n",
              "L 347 722 \n",
              "Q 666 556 975 473 \n",
              "Q 1284 391 1588 391 \n",
              "Q 1994 391 2212 530 \n",
              "Q 2431 669 2431 922 \n",
              "Q 2431 1156 2273 1281 \n",
              "Q 2116 1406 1581 1522 \n",
              "L 1381 1569 \n",
              "Q 847 1681 609 1914 \n",
              "Q 372 2147 372 2553 \n",
              "Q 372 3047 722 3315 \n",
              "Q 1072 3584 1716 3584 \n",
              "Q 2034 3584 2315 3537 \n",
              "Q 2597 3491 2834 3397 \n",
              "z\n",
              "\" transform=\"scale(0.015625)\"/>\n",
              "      </defs>\n",
              "      <use xlink:href=\"#DejaVuSans-4e\"/>\n",
              "      <use xlink:href=\"#DejaVuSans-75\" x=\"74.804688\"/>\n",
              "      <use xlink:href=\"#DejaVuSans-6d\" x=\"138.183594\"/>\n",
              "      <use xlink:href=\"#DejaVuSans-62\" x=\"235.595703\"/>\n",
              "      <use xlink:href=\"#DejaVuSans-65\" x=\"299.072266\"/>\n",
              "      <use xlink:href=\"#DejaVuSans-72\" x=\"360.595703\"/>\n",
              "      <use xlink:href=\"#DejaVuSans-20\" x=\"401.708984\"/>\n",
              "      <use xlink:href=\"#DejaVuSans-6f\" x=\"433.496094\"/>\n",
              "      <use xlink:href=\"#DejaVuSans-66\" x=\"494.677734\"/>\n",
              "      <use xlink:href=\"#DejaVuSans-20\" x=\"529.882812\"/>\n",
              "      <use xlink:href=\"#DejaVuSans-43\" x=\"561.669922\"/>\n",
              "      <use xlink:href=\"#DejaVuSans-6f\" x=\"631.494141\"/>\n",
              "      <use xlink:href=\"#DejaVuSans-6d\" x=\"692.675781\"/>\n",
              "      <use xlink:href=\"#DejaVuSans-70\" x=\"790.087891\"/>\n",
              "      <use xlink:href=\"#DejaVuSans-6f\" x=\"853.564453\"/>\n",
              "      <use xlink:href=\"#DejaVuSans-6e\" x=\"914.746094\"/>\n",
              "      <use xlink:href=\"#DejaVuSans-65\" x=\"978.125\"/>\n",
              "      <use xlink:href=\"#DejaVuSans-6e\" x=\"1039.648438\"/>\n",
              "      <use xlink:href=\"#DejaVuSans-74\" x=\"1103.027344\"/>\n",
              "      <use xlink:href=\"#DejaVuSans-73\" x=\"1142.236328\"/>\n",
              "     </g>\n",
              "    </g>\n",
              "   </g>\n",
              "   <g id=\"matplotlib.axis_2\">\n",
              "    <g id=\"ytick_1\">\n",
              "     <g id=\"line2d_6\">\n",
              "      <defs>\n",
              "       <path id=\"m7e8ecd305f\" d=\"M 0 0 \n",
              "L -3.5 0 \n",
              "\" style=\"stroke: #555555; stroke-width: 0.8\"/>\n",
              "      </defs>\n",
              "      <g>\n",
              "       <use xlink:href=\"#m7e8ecd305f\" x=\"45.716875\" y=\"297.642961\" style=\"fill: #555555; stroke: #555555; stroke-width: 0.8\"/>\n",
              "      </g>\n",
              "     </g>\n",
              "     <g id=\"text_7\">\n",
              "      <!-- 0.0 -->\n",
              "      <g style=\"fill: #555555\" transform=\"translate(22.81375 301.44218) scale(0.1 -0.1)\">\n",
              "       <defs>\n",
              "        <path id=\"DejaVuSans-2e\" d=\"M 684 794 \n",
              "L 1344 794 \n",
              "L 1344 0 \n",
              "L 684 0 \n",
              "L 684 794 \n",
              "z\n",
              "\" transform=\"scale(0.015625)\"/>\n",
              "       </defs>\n",
              "       <use xlink:href=\"#DejaVuSans-30\"/>\n",
              "       <use xlink:href=\"#DejaVuSans-2e\" x=\"63.623047\"/>\n",
              "       <use xlink:href=\"#DejaVuSans-30\" x=\"95.410156\"/>\n",
              "      </g>\n",
              "     </g>\n",
              "    </g>\n",
              "    <g id=\"ytick_2\">\n",
              "     <g id=\"line2d_7\">\n",
              "      <g>\n",
              "       <use xlink:href=\"#m7e8ecd305f\" x=\"45.716875\" y=\"242.721516\" style=\"fill: #555555; stroke: #555555; stroke-width: 0.8\"/>\n",
              "      </g>\n",
              "     </g>\n",
              "     <g id=\"text_8\">\n",
              "      <!-- 0.2 -->\n",
              "      <g style=\"fill: #555555\" transform=\"translate(22.81375 246.520735) scale(0.1 -0.1)\">\n",
              "       <use xlink:href=\"#DejaVuSans-30\"/>\n",
              "       <use xlink:href=\"#DejaVuSans-2e\" x=\"63.623047\"/>\n",
              "       <use xlink:href=\"#DejaVuSans-32\" x=\"95.410156\"/>\n",
              "      </g>\n",
              "     </g>\n",
              "    </g>\n",
              "    <g id=\"ytick_3\">\n",
              "     <g id=\"line2d_8\">\n",
              "      <g>\n",
              "       <use xlink:href=\"#m7e8ecd305f\" x=\"45.716875\" y=\"187.800071\" style=\"fill: #555555; stroke: #555555; stroke-width: 0.8\"/>\n",
              "      </g>\n",
              "     </g>\n",
              "     <g id=\"text_9\">\n",
              "      <!-- 0.4 -->\n",
              "      <g style=\"fill: #555555\" transform=\"translate(22.81375 191.59929) scale(0.1 -0.1)\">\n",
              "       <use xlink:href=\"#DejaVuSans-30\"/>\n",
              "       <use xlink:href=\"#DejaVuSans-2e\" x=\"63.623047\"/>\n",
              "       <use xlink:href=\"#DejaVuSans-34\" x=\"95.410156\"/>\n",
              "      </g>\n",
              "     </g>\n",
              "    </g>\n",
              "    <g id=\"ytick_4\">\n",
              "     <g id=\"line2d_9\">\n",
              "      <g>\n",
              "       <use xlink:href=\"#m7e8ecd305f\" x=\"45.716875\" y=\"132.878626\" style=\"fill: #555555; stroke: #555555; stroke-width: 0.8\"/>\n",
              "      </g>\n",
              "     </g>\n",
              "     <g id=\"text_10\">\n",
              "      <!-- 0.6 -->\n",
              "      <g style=\"fill: #555555\" transform=\"translate(22.81375 136.677845) scale(0.1 -0.1)\">\n",
              "       <use xlink:href=\"#DejaVuSans-30\"/>\n",
              "       <use xlink:href=\"#DejaVuSans-2e\" x=\"63.623047\"/>\n",
              "       <use xlink:href=\"#DejaVuSans-36\" x=\"95.410156\"/>\n",
              "      </g>\n",
              "     </g>\n",
              "    </g>\n",
              "    <g id=\"ytick_5\">\n",
              "     <g id=\"line2d_10\">\n",
              "      <g>\n",
              "       <use xlink:href=\"#m7e8ecd305f\" x=\"45.716875\" y=\"77.957181\" style=\"fill: #555555; stroke: #555555; stroke-width: 0.8\"/>\n",
              "      </g>\n",
              "     </g>\n",
              "     <g id=\"text_11\">\n",
              "      <!-- 0.8 -->\n",
              "      <g style=\"fill: #555555\" transform=\"translate(22.81375 81.7564) scale(0.1 -0.1)\">\n",
              "       <use xlink:href=\"#DejaVuSans-30\"/>\n",
              "       <use xlink:href=\"#DejaVuSans-2e\" x=\"63.623047\"/>\n",
              "       <use xlink:href=\"#DejaVuSans-38\" x=\"95.410156\"/>\n",
              "      </g>\n",
              "     </g>\n",
              "    </g>\n",
              "    <g id=\"text_12\">\n",
              "     <!-- Cumulative Explained Variance -->\n",
              "     <g style=\"fill: #555555\" transform=\"translate(16.318125 256.186125) rotate(-90) scale(0.12 -0.12)\">\n",
              "      <defs>\n",
              "       <path id=\"DejaVuSans-6c\" d=\"M 603 4863 \n",
              "L 1178 4863 \n",
              "L 1178 0 \n",
              "L 603 0 \n",
              "L 603 4863 \n",
              "z\n",
              "\" transform=\"scale(0.015625)\"/>\n",
              "       <path id=\"DejaVuSans-61\" d=\"M 2194 1759 \n",
              "Q 1497 1759 1228 1600 \n",
              "Q 959 1441 959 1056 \n",
              "Q 959 750 1161 570 \n",
              "Q 1363 391 1709 391 \n",
              "Q 2188 391 2477 730 \n",
              "Q 2766 1069 2766 1631 \n",
              "L 2766 1759 \n",
              "L 2194 1759 \n",
              "z\n",
              "M 3341 1997 \n",
              "L 3341 0 \n",
              "L 2766 0 \n",
              "L 2766 531 \n",
              "Q 2569 213 2275 61 \n",
              "Q 1981 -91 1556 -91 \n",
              "Q 1019 -91 701 211 \n",
              "Q 384 513 384 1019 \n",
              "Q 384 1609 779 1909 \n",
              "Q 1175 2209 1959 2209 \n",
              "L 2766 2209 \n",
              "L 2766 2266 \n",
              "Q 2766 2663 2505 2880 \n",
              "Q 2244 3097 1772 3097 \n",
              "Q 1472 3097 1187 3025 \n",
              "Q 903 2953 641 2809 \n",
              "L 641 3341 \n",
              "Q 956 3463 1253 3523 \n",
              "Q 1550 3584 1831 3584 \n",
              "Q 2591 3584 2966 3190 \n",
              "Q 3341 2797 3341 1997 \n",
              "z\n",
              "\" transform=\"scale(0.015625)\"/>\n",
              "       <path id=\"DejaVuSans-69\" d=\"M 603 3500 \n",
              "L 1178 3500 \n",
              "L 1178 0 \n",
              "L 603 0 \n",
              "L 603 3500 \n",
              "z\n",
              "M 603 4863 \n",
              "L 1178 4863 \n",
              "L 1178 4134 \n",
              "L 603 4134 \n",
              "L 603 4863 \n",
              "z\n",
              "\" transform=\"scale(0.015625)\"/>\n",
              "       <path id=\"DejaVuSans-76\" d=\"M 191 3500 \n",
              "L 800 3500 \n",
              "L 1894 563 \n",
              "L 2988 3500 \n",
              "L 3597 3500 \n",
              "L 2284 0 \n",
              "L 1503 0 \n",
              "L 191 3500 \n",
              "z\n",
              "\" transform=\"scale(0.015625)\"/>\n",
              "       <path id=\"DejaVuSans-45\" d=\"M 628 4666 \n",
              "L 3578 4666 \n",
              "L 3578 4134 \n",
              "L 1259 4134 \n",
              "L 1259 2753 \n",
              "L 3481 2753 \n",
              "L 3481 2222 \n",
              "L 1259 2222 \n",
              "L 1259 531 \n",
              "L 3634 531 \n",
              "L 3634 0 \n",
              "L 628 0 \n",
              "L 628 4666 \n",
              "z\n",
              "\" transform=\"scale(0.015625)\"/>\n",
              "       <path id=\"DejaVuSans-78\" d=\"M 3513 3500 \n",
              "L 2247 1797 \n",
              "L 3578 0 \n",
              "L 2900 0 \n",
              "L 1881 1375 \n",
              "L 863 0 \n",
              "L 184 0 \n",
              "L 1544 1831 \n",
              "L 300 3500 \n",
              "L 978 3500 \n",
              "L 1906 2253 \n",
              "L 2834 3500 \n",
              "L 3513 3500 \n",
              "z\n",
              "\" transform=\"scale(0.015625)\"/>\n",
              "       <path id=\"DejaVuSans-64\" d=\"M 2906 2969 \n",
              "L 2906 4863 \n",
              "L 3481 4863 \n",
              "L 3481 0 \n",
              "L 2906 0 \n",
              "L 2906 525 \n",
              "Q 2725 213 2448 61 \n",
              "Q 2172 -91 1784 -91 \n",
              "Q 1150 -91 751 415 \n",
              "Q 353 922 353 1747 \n",
              "Q 353 2572 751 3078 \n",
              "Q 1150 3584 1784 3584 \n",
              "Q 2172 3584 2448 3432 \n",
              "Q 2725 3281 2906 2969 \n",
              "z\n",
              "M 947 1747 \n",
              "Q 947 1113 1208 752 \n",
              "Q 1469 391 1925 391 \n",
              "Q 2381 391 2643 752 \n",
              "Q 2906 1113 2906 1747 \n",
              "Q 2906 2381 2643 2742 \n",
              "Q 2381 3103 1925 3103 \n",
              "Q 1469 3103 1208 2742 \n",
              "Q 947 2381 947 1747 \n",
              "z\n",
              "\" transform=\"scale(0.015625)\"/>\n",
              "       <path id=\"DejaVuSans-56\" d=\"M 1831 0 \n",
              "L 50 4666 \n",
              "L 709 4666 \n",
              "L 2188 738 \n",
              "L 3669 4666 \n",
              "L 4325 4666 \n",
              "L 2547 0 \n",
              "L 1831 0 \n",
              "z\n",
              "\" transform=\"scale(0.015625)\"/>\n",
              "       <path id=\"DejaVuSans-63\" d=\"M 3122 3366 \n",
              "L 3122 2828 \n",
              "Q 2878 2963 2633 3030 \n",
              "Q 2388 3097 2138 3097 \n",
              "Q 1578 3097 1268 2742 \n",
              "Q 959 2388 959 1747 \n",
              "Q 959 1106 1268 751 \n",
              "Q 1578 397 2138 397 \n",
              "Q 2388 397 2633 464 \n",
              "Q 2878 531 3122 666 \n",
              "L 3122 134 \n",
              "Q 2881 22 2623 -34 \n",
              "Q 2366 -91 2075 -91 \n",
              "Q 1284 -91 818 406 \n",
              "Q 353 903 353 1747 \n",
              "Q 353 2603 823 3093 \n",
              "Q 1294 3584 2113 3584 \n",
              "Q 2378 3584 2631 3529 \n",
              "Q 2884 3475 3122 3366 \n",
              "z\n",
              "\" transform=\"scale(0.015625)\"/>\n",
              "      </defs>\n",
              "      <use xlink:href=\"#DejaVuSans-43\"/>\n",
              "      <use xlink:href=\"#DejaVuSans-75\" x=\"69.824219\"/>\n",
              "      <use xlink:href=\"#DejaVuSans-6d\" x=\"133.203125\"/>\n",
              "      <use xlink:href=\"#DejaVuSans-75\" x=\"230.615234\"/>\n",
              "      <use xlink:href=\"#DejaVuSans-6c\" x=\"293.994141\"/>\n",
              "      <use xlink:href=\"#DejaVuSans-61\" x=\"321.777344\"/>\n",
              "      <use xlink:href=\"#DejaVuSans-74\" x=\"383.056641\"/>\n",
              "      <use xlink:href=\"#DejaVuSans-69\" x=\"422.265625\"/>\n",
              "      <use xlink:href=\"#DejaVuSans-76\" x=\"450.048828\"/>\n",
              "      <use xlink:href=\"#DejaVuSans-65\" x=\"509.228516\"/>\n",
              "      <use xlink:href=\"#DejaVuSans-20\" x=\"570.751953\"/>\n",
              "      <use xlink:href=\"#DejaVuSans-45\" x=\"602.539062\"/>\n",
              "      <use xlink:href=\"#DejaVuSans-78\" x=\"665.722656\"/>\n",
              "      <use xlink:href=\"#DejaVuSans-70\" x=\"724.902344\"/>\n",
              "      <use xlink:href=\"#DejaVuSans-6c\" x=\"788.378906\"/>\n",
              "      <use xlink:href=\"#DejaVuSans-61\" x=\"816.162109\"/>\n",
              "      <use xlink:href=\"#DejaVuSans-69\" x=\"877.441406\"/>\n",
              "      <use xlink:href=\"#DejaVuSans-6e\" x=\"905.224609\"/>\n",
              "      <use xlink:href=\"#DejaVuSans-65\" x=\"968.603516\"/>\n",
              "      <use xlink:href=\"#DejaVuSans-64\" x=\"1030.126953\"/>\n",
              "      <use xlink:href=\"#DejaVuSans-20\" x=\"1093.603516\"/>\n",
              "      <use xlink:href=\"#DejaVuSans-56\" x=\"1125.390625\"/>\n",
              "      <use xlink:href=\"#DejaVuSans-61\" x=\"1186.048828\"/>\n",
              "      <use xlink:href=\"#DejaVuSans-72\" x=\"1247.328125\"/>\n",
              "      <use xlink:href=\"#DejaVuSans-69\" x=\"1288.441406\"/>\n",
              "      <use xlink:href=\"#DejaVuSans-61\" x=\"1316.224609\"/>\n",
              "      <use xlink:href=\"#DejaVuSans-6e\" x=\"1377.503906\"/>\n",
              "      <use xlink:href=\"#DejaVuSans-63\" x=\"1440.882812\"/>\n",
              "      <use xlink:href=\"#DejaVuSans-65\" x=\"1495.863281\"/>\n",
              "     </g>\n",
              "    </g>\n",
              "   </g>\n",
              "   <g id=\"line2d_11\">\n",
              "    <path d=\"M 71.080511 288.74175 \n",
              "L 72.14621 278.236303 \n",
              "L 73.211909 270.272956 \n",
              "L 74.277608 263.95903 \n",
              "L 75.343307 258.987869 \n",
              "L 76.941856 252.94146 \n",
              "L 78.540404 247.756268 \n",
              "L 80.671802 241.793264 \n",
              "L 82.8032 236.534317 \n",
              "L 84.934598 231.951481 \n",
              "L 87.065996 227.811098 \n",
              "L 89.730244 223.070492 \n",
              "L 92.927341 217.864608 \n",
              "L 96.124438 213.123251 \n",
              "L 99.321535 208.691587 \n",
              "L 103.051482 203.935182 \n",
              "L 106.781428 199.551729 \n",
              "L 111.044224 194.881454 \n",
              "L 116.372719 189.443434 \n",
              "L 121.168365 184.850631 \n",
              "L 127.029709 179.595922 \n",
              "L 132.891054 174.671343 \n",
              "L 139.818097 169.179278 \n",
              "L 146.745141 164.014545 \n",
              "L 153.672184 159.131655 \n",
              "L 161.132077 154.143038 \n",
              "L 168.59197 149.416582 \n",
              "L 177.117563 144.294661 \n",
              "L 185.643155 139.42098 \n",
              "L 194.701596 134.505731 \n",
              "L 204.825737 129.275922 \n",
              "L 214.417028 124.550986 \n",
              "L 225.074018 119.557298 \n",
              "L 235.731008 114.815179 \n",
              "L 246.387998 110.304962 \n",
              "L 257.577838 105.797911 \n",
              "L 269.833376 101.100138 \n",
              "L 282.088915 96.638371 \n",
              "L 294.877303 92.213396 \n",
              "L 307.665691 88.02515 \n",
              "L 320.454079 84.062929 \n",
              "L 334.308166 80.001047 \n",
              "L 348.695103 76.026072 \n",
              "L 363.614889 72.146308 \n",
              "L 378.001825 68.631914 \n",
              "L 393.454461 65.085245 \n",
              "L 409.439946 61.646286 \n",
              "L 425.425431 58.429471 \n",
              "L 441.943766 55.333598 \n",
              "L 459.527799 52.267539 \n",
              "L 477.644682 49.340635 \n",
              "L 495.761566 46.634503 \n",
              "L 514.944148 43.997582 \n",
              "L 534.659579 41.510893 \n",
              "L 555.44071 39.121805 \n",
              "L 577.28754 36.847044 \n",
              "L 578.353239 36.74175 \n",
              "L 578.353239 36.74175 \n",
              "\" clip-path=\"url(#p6c14437394)\" style=\"fill: none; stroke: #e24a33; stroke-width: 1.5; stroke-linecap: square\"/>\n",
              "   </g>\n",
              "   <g id=\"patch_3\">\n",
              "    <path d=\"M 45.716875 301.34175 \n",
              "L 45.716875 24.14175 \n",
              "\" style=\"fill: none; stroke: #ffffff; stroke-linejoin: miter; stroke-linecap: square\"/>\n",
              "   </g>\n",
              "   <g id=\"patch_4\">\n",
              "    <path d=\"M 603.716875 301.34175 \n",
              "L 603.716875 24.14175 \n",
              "\" style=\"fill: none; stroke: #ffffff; stroke-linejoin: miter; stroke-linecap: square\"/>\n",
              "   </g>\n",
              "   <g id=\"patch_5\">\n",
              "    <path d=\"M 45.716875 301.34175 \n",
              "L 603.716875 301.34175 \n",
              "\" style=\"fill: none; stroke: #ffffff; stroke-linejoin: miter; stroke-linecap: square\"/>\n",
              "   </g>\n",
              "   <g id=\"patch_6\">\n",
              "    <path d=\"M 45.716875 24.14175 \n",
              "L 603.716875 24.14175 \n",
              "\" style=\"fill: none; stroke: #ffffff; stroke-linejoin: miter; stroke-linecap: square\"/>\n",
              "   </g>\n",
              "   <g id=\"text_13\">\n",
              "    <!-- Explained Variance by PCA Components -->\n",
              "    <g transform=\"translate(180.422125 18.14175) scale(0.144 -0.144)\">\n",
              "     <defs>\n",
              "      <path id=\"DejaVuSans-79\" d=\"M 2059 -325 \n",
              "Q 1816 -950 1584 -1140 \n",
              "Q 1353 -1331 966 -1331 \n",
              "L 506 -1331 \n",
              "L 506 -850 \n",
              "L 844 -850 \n",
              "Q 1081 -850 1212 -737 \n",
              "Q 1344 -625 1503 -206 \n",
              "L 1606 56 \n",
              "L 191 3500 \n",
              "L 800 3500 \n",
              "L 1894 763 \n",
              "L 2988 3500 \n",
              "L 3597 3500 \n",
              "L 2059 -325 \n",
              "z\n",
              "\" transform=\"scale(0.015625)\"/>\n",
              "      <path id=\"DejaVuSans-50\" d=\"M 1259 4147 \n",
              "L 1259 2394 \n",
              "L 2053 2394 \n",
              "Q 2494 2394 2734 2622 \n",
              "Q 2975 2850 2975 3272 \n",
              "Q 2975 3691 2734 3919 \n",
              "Q 2494 4147 2053 4147 \n",
              "L 1259 4147 \n",
              "z\n",
              "M 628 4666 \n",
              "L 2053 4666 \n",
              "Q 2838 4666 3239 4311 \n",
              "Q 3641 3956 3641 3272 \n",
              "Q 3641 2581 3239 2228 \n",
              "Q 2838 1875 2053 1875 \n",
              "L 1259 1875 \n",
              "L 1259 0 \n",
              "L 628 0 \n",
              "L 628 4666 \n",
              "z\n",
              "\" transform=\"scale(0.015625)\"/>\n",
              "      <path id=\"DejaVuSans-41\" d=\"M 2188 4044 \n",
              "L 1331 1722 \n",
              "L 3047 1722 \n",
              "L 2188 4044 \n",
              "z\n",
              "M 1831 4666 \n",
              "L 2547 4666 \n",
              "L 4325 0 \n",
              "L 3669 0 \n",
              "L 3244 1197 \n",
              "L 1141 1197 \n",
              "L 716 0 \n",
              "L 50 0 \n",
              "L 1831 4666 \n",
              "z\n",
              "\" transform=\"scale(0.015625)\"/>\n",
              "     </defs>\n",
              "     <use xlink:href=\"#DejaVuSans-45\"/>\n",
              "     <use xlink:href=\"#DejaVuSans-78\" x=\"63.183594\"/>\n",
              "     <use xlink:href=\"#DejaVuSans-70\" x=\"122.363281\"/>\n",
              "     <use xlink:href=\"#DejaVuSans-6c\" x=\"185.839844\"/>\n",
              "     <use xlink:href=\"#DejaVuSans-61\" x=\"213.623047\"/>\n",
              "     <use xlink:href=\"#DejaVuSans-69\" x=\"274.902344\"/>\n",
              "     <use xlink:href=\"#DejaVuSans-6e\" x=\"302.685547\"/>\n",
              "     <use xlink:href=\"#DejaVuSans-65\" x=\"366.064453\"/>\n",
              "     <use xlink:href=\"#DejaVuSans-64\" x=\"427.587891\"/>\n",
              "     <use xlink:href=\"#DejaVuSans-20\" x=\"491.064453\"/>\n",
              "     <use xlink:href=\"#DejaVuSans-56\" x=\"522.851562\"/>\n",
              "     <use xlink:href=\"#DejaVuSans-61\" x=\"583.509766\"/>\n",
              "     <use xlink:href=\"#DejaVuSans-72\" x=\"644.789062\"/>\n",
              "     <use xlink:href=\"#DejaVuSans-69\" x=\"685.902344\"/>\n",
              "     <use xlink:href=\"#DejaVuSans-61\" x=\"713.685547\"/>\n",
              "     <use xlink:href=\"#DejaVuSans-6e\" x=\"774.964844\"/>\n",
              "     <use xlink:href=\"#DejaVuSans-63\" x=\"838.34375\"/>\n",
              "     <use xlink:href=\"#DejaVuSans-65\" x=\"893.324219\"/>\n",
              "     <use xlink:href=\"#DejaVuSans-20\" x=\"954.847656\"/>\n",
              "     <use xlink:href=\"#DejaVuSans-62\" x=\"986.634766\"/>\n",
              "     <use xlink:href=\"#DejaVuSans-79\" x=\"1050.111328\"/>\n",
              "     <use xlink:href=\"#DejaVuSans-20\" x=\"1109.291016\"/>\n",
              "     <use xlink:href=\"#DejaVuSans-50\" x=\"1141.078125\"/>\n",
              "     <use xlink:href=\"#DejaVuSans-43\" x=\"1201.380859\"/>\n",
              "     <use xlink:href=\"#DejaVuSans-41\" x=\"1271.205078\"/>\n",
              "     <use xlink:href=\"#DejaVuSans-20\" x=\"1339.613281\"/>\n",
              "     <use xlink:href=\"#DejaVuSans-43\" x=\"1371.400391\"/>\n",
              "     <use xlink:href=\"#DejaVuSans-6f\" x=\"1441.224609\"/>\n",
              "     <use xlink:href=\"#DejaVuSans-6d\" x=\"1502.40625\"/>\n",
              "     <use xlink:href=\"#DejaVuSans-70\" x=\"1599.818359\"/>\n",
              "     <use xlink:href=\"#DejaVuSans-6f\" x=\"1663.294922\"/>\n",
              "     <use xlink:href=\"#DejaVuSans-6e\" x=\"1724.476562\"/>\n",
              "     <use xlink:href=\"#DejaVuSans-65\" x=\"1787.855469\"/>\n",
              "     <use xlink:href=\"#DejaVuSans-6e\" x=\"1849.378906\"/>\n",
              "     <use xlink:href=\"#DejaVuSans-74\" x=\"1912.757812\"/>\n",
              "     <use xlink:href=\"#DejaVuSans-73\" x=\"1951.966797\"/>\n",
              "    </g>\n",
              "   </g>\n",
              "  </g>\n",
              " </g>\n",
              " <defs>\n",
              "  <clipPath id=\"p6c14437394\">\n",
              "   <rect x=\"45.716875\" y=\"24.14175\" width=\"558\" height=\"277.2\"/>\n",
              "  </clipPath>\n",
              " </defs>\n",
              "</svg>\n"
            ],
            "text/plain": [
              "<Figure size 1000x500 with 1 Axes>"
            ]
          },
          "metadata": {},
          "output_type": "display_data"
        }
      ],
      "source": [
        "# Optionally, plot the explained variance ratio to visualize how much variance is retained\n",
        "plt.figure(figsize=(10, 5))\n",
        "plt.plot(np.cumsum(pca.explained_variance_ratio_))\n",
        "plt.xlabel('Number of Components')\n",
        "plt.ylabel('Cumulative Explained Variance')\n",
        "plt.title('Explained Variance by PCA Components')\n",
        "plt.grid()\n",
        "plt.show()"
      ]
    },
    {
      "cell_type": "code",
      "execution_count": 50,
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/",
          "height": 0
        },
        "id": "yllwsZlaJKCD",
        "outputId": "7f956097-8fc3-4fed-a9f2-9413f18c119a"
      },
      "outputs": [
        {
          "name": "stderr",
          "output_type": "stream",
          "text": [
            "/usr/local/lib/python3.10/dist-packages/sklearn/cluster/_kmeans.py:870: FutureWarning: The default value of `n_init` will change from 10 to 'auto' in 1.4. Set the value of `n_init` explicitly to suppress the warning\n",
            "  warnings.warn(\n",
            "/usr/local/lib/python3.10/dist-packages/sklearn/cluster/_kmeans.py:870: FutureWarning: The default value of `n_init` will change from 10 to 'auto' in 1.4. Set the value of `n_init` explicitly to suppress the warning\n",
            "  warnings.warn(\n",
            "/usr/local/lib/python3.10/dist-packages/sklearn/cluster/_kmeans.py:870: FutureWarning: The default value of `n_init` will change from 10 to 'auto' in 1.4. Set the value of `n_init` explicitly to suppress the warning\n",
            "  warnings.warn(\n",
            "/usr/local/lib/python3.10/dist-packages/sklearn/cluster/_kmeans.py:870: FutureWarning: The default value of `n_init` will change from 10 to 'auto' in 1.4. Set the value of `n_init` explicitly to suppress the warning\n",
            "  warnings.warn(\n",
            "/usr/local/lib/python3.10/dist-packages/sklearn/cluster/_kmeans.py:870: FutureWarning: The default value of `n_init` will change from 10 to 'auto' in 1.4. Set the value of `n_init` explicitly to suppress the warning\n",
            "  warnings.warn(\n",
            "/usr/local/lib/python3.10/dist-packages/sklearn/cluster/_kmeans.py:870: FutureWarning: The default value of `n_init` will change from 10 to 'auto' in 1.4. Set the value of `n_init` explicitly to suppress the warning\n",
            "  warnings.warn(\n",
            "/usr/local/lib/python3.10/dist-packages/sklearn/cluster/_kmeans.py:870: FutureWarning: The default value of `n_init` will change from 10 to 'auto' in 1.4. Set the value of `n_init` explicitly to suppress the warning\n",
            "  warnings.warn(\n",
            "/usr/local/lib/python3.10/dist-packages/sklearn/cluster/_kmeans.py:870: FutureWarning: The default value of `n_init` will change from 10 to 'auto' in 1.4. Set the value of `n_init` explicitly to suppress the warning\n",
            "  warnings.warn(\n",
            "/usr/local/lib/python3.10/dist-packages/sklearn/cluster/_kmeans.py:870: FutureWarning: The default value of `n_init` will change from 10 to 'auto' in 1.4. Set the value of `n_init` explicitly to suppress the warning\n",
            "  warnings.warn(\n",
            "/usr/local/lib/python3.10/dist-packages/sklearn/cluster/_kmeans.py:870: FutureWarning: The default value of `n_init` will change from 10 to 'auto' in 1.4. Set the value of `n_init` explicitly to suppress the warning\n",
            "  warnings.warn(\n",
            "/usr/local/lib/python3.10/dist-packages/sklearn/cluster/_kmeans.py:870: FutureWarning: The default value of `n_init` will change from 10 to 'auto' in 1.4. Set the value of `n_init` explicitly to suppress the warning\n",
            "  warnings.warn(\n",
            "/usr/local/lib/python3.10/dist-packages/sklearn/cluster/_kmeans.py:870: FutureWarning: The default value of `n_init` will change from 10 to 'auto' in 1.4. Set the value of `n_init` explicitly to suppress the warning\n",
            "  warnings.warn(\n",
            "/usr/local/lib/python3.10/dist-packages/sklearn/cluster/_kmeans.py:870: FutureWarning: The default value of `n_init` will change from 10 to 'auto' in 1.4. Set the value of `n_init` explicitly to suppress the warning\n",
            "  warnings.warn(\n",
            "/usr/local/lib/python3.10/dist-packages/sklearn/cluster/_kmeans.py:870: FutureWarning: The default value of `n_init` will change from 10 to 'auto' in 1.4. Set the value of `n_init` explicitly to suppress the warning\n",
            "  warnings.warn(\n",
            "/usr/local/lib/python3.10/dist-packages/sklearn/cluster/_kmeans.py:870: FutureWarning: The default value of `n_init` will change from 10 to 'auto' in 1.4. Set the value of `n_init` explicitly to suppress the warning\n",
            "  warnings.warn(\n",
            "/usr/local/lib/python3.10/dist-packages/sklearn/cluster/_kmeans.py:870: FutureWarning: The default value of `n_init` will change from 10 to 'auto' in 1.4. Set the value of `n_init` explicitly to suppress the warning\n",
            "  warnings.warn(\n",
            "/usr/local/lib/python3.10/dist-packages/sklearn/cluster/_kmeans.py:870: FutureWarning: The default value of `n_init` will change from 10 to 'auto' in 1.4. Set the value of `n_init` explicitly to suppress the warning\n",
            "  warnings.warn(\n",
            "/usr/local/lib/python3.10/dist-packages/sklearn/cluster/_kmeans.py:870: FutureWarning: The default value of `n_init` will change from 10 to 'auto' in 1.4. Set the value of `n_init` explicitly to suppress the warning\n",
            "  warnings.warn(\n",
            "/usr/local/lib/python3.10/dist-packages/sklearn/cluster/_kmeans.py:870: FutureWarning: The default value of `n_init` will change from 10 to 'auto' in 1.4. Set the value of `n_init` explicitly to suppress the warning\n",
            "  warnings.warn(\n",
            "/usr/local/lib/python3.10/dist-packages/sklearn/cluster/_kmeans.py:870: FutureWarning: The default value of `n_init` will change from 10 to 'auto' in 1.4. Set the value of `n_init` explicitly to suppress the warning\n",
            "  warnings.warn(\n",
            "/usr/local/lib/python3.10/dist-packages/sklearn/cluster/_kmeans.py:870: FutureWarning: The default value of `n_init` will change from 10 to 'auto' in 1.4. Set the value of `n_init` explicitly to suppress the warning\n",
            "  warnings.warn(\n",
            "/usr/local/lib/python3.10/dist-packages/sklearn/cluster/_kmeans.py:870: FutureWarning: The default value of `n_init` will change from 10 to 'auto' in 1.4. Set the value of `n_init` explicitly to suppress the warning\n",
            "  warnings.warn(\n",
            "/usr/local/lib/python3.10/dist-packages/sklearn/cluster/_kmeans.py:870: FutureWarning: The default value of `n_init` will change from 10 to 'auto' in 1.4. Set the value of `n_init` explicitly to suppress the warning\n",
            "  warnings.warn(\n",
            "/usr/local/lib/python3.10/dist-packages/sklearn/cluster/_kmeans.py:870: FutureWarning: The default value of `n_init` will change from 10 to 'auto' in 1.4. Set the value of `n_init` explicitly to suppress the warning\n",
            "  warnings.warn(\n",
            "/usr/local/lib/python3.10/dist-packages/sklearn/cluster/_kmeans.py:870: FutureWarning: The default value of `n_init` will change from 10 to 'auto' in 1.4. Set the value of `n_init` explicitly to suppress the warning\n",
            "  warnings.warn(\n",
            "/usr/local/lib/python3.10/dist-packages/sklearn/cluster/_kmeans.py:870: FutureWarning: The default value of `n_init` will change from 10 to 'auto' in 1.4. Set the value of `n_init` explicitly to suppress the warning\n",
            "  warnings.warn(\n",
            "/usr/local/lib/python3.10/dist-packages/sklearn/cluster/_kmeans.py:870: FutureWarning: The default value of `n_init` will change from 10 to 'auto' in 1.4. Set the value of `n_init` explicitly to suppress the warning\n",
            "  warnings.warn(\n",
            "/usr/local/lib/python3.10/dist-packages/sklearn/cluster/_kmeans.py:870: FutureWarning: The default value of `n_init` will change from 10 to 'auto' in 1.4. Set the value of `n_init` explicitly to suppress the warning\n",
            "  warnings.warn(\n",
            "/usr/local/lib/python3.10/dist-packages/sklearn/cluster/_kmeans.py:870: FutureWarning: The default value of `n_init` will change from 10 to 'auto' in 1.4. Set the value of `n_init` explicitly to suppress the warning\n",
            "  warnings.warn(\n",
            "/usr/local/lib/python3.10/dist-packages/sklearn/cluster/_kmeans.py:870: FutureWarning: The default value of `n_init` will change from 10 to 'auto' in 1.4. Set the value of `n_init` explicitly to suppress the warning\n",
            "  warnings.warn(\n"
          ]
        }
      ],
      "source": [
        "# Determine the optimal number of clusters using the Elbow Method\n",
        "distortions = []\n",
        "K = range(1, 31)\n",
        "for k in K:\n",
        "    kmeans = KMeans(n_clusters=k, random_state=42)\n",
        "    kmeans.fit(tfidf_pca)\n",
        "    distortions.append(sum(np.min(cdist(tfidf_pca, kmeans.cluster_centers_, 'euclidean'), axis=1)) / tfidf_pca.shape[0])"
      ]
    },
    {
      "cell_type": "code",
      "execution_count": 51,
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/",
          "height": 550
        },
        "id": "SZqfUheztiG0",
        "outputId": "e18293ca-36bf-4940-cdab-ece8c3ec5590"
      },
      "outputs": [
        {
          "data": {
            "image/svg+xml": [
              "<?xml version=\"1.0\" encoding=\"utf-8\" standalone=\"no\"?>\n",
              "<!DOCTYPE svg PUBLIC \"-//W3C//DTD SVG 1.1//EN\"\n",
              "  \"http://www.w3.org/Graphics/SVG/1.1/DTD/svg11.dtd\">\n",
              "<svg xmlns:xlink=\"http://www.w3.org/1999/xlink\" width=\"617.279375pt\" height=\"396.273625pt\" viewBox=\"0 0 617.279375 396.273625\" xmlns=\"http://www.w3.org/2000/svg\" version=\"1.1\">\n",
              " <metadata>\n",
              "  <rdf:RDF xmlns:dc=\"http://purl.org/dc/elements/1.1/\" xmlns:cc=\"http://creativecommons.org/ns#\" xmlns:rdf=\"http://www.w3.org/1999/02/22-rdf-syntax-ns#\">\n",
              "   <cc:Work>\n",
              "    <dc:type rdf:resource=\"http://purl.org/dc/dcmitype/StillImage\"/>\n",
              "    <dc:date>2024-06-05T18:06:10.600033</dc:date>\n",
              "    <dc:format>image/svg+xml</dc:format>\n",
              "    <dc:creator>\n",
              "     <cc:Agent>\n",
              "      <dc:title>Matplotlib v3.7.1, https://matplotlib.org/</dc:title>\n",
              "     </cc:Agent>\n",
              "    </dc:creator>\n",
              "   </cc:Work>\n",
              "  </rdf:RDF>\n",
              " </metadata>\n",
              " <defs>\n",
              "  <style type=\"text/css\">*{stroke-linejoin: round; stroke-linecap: butt}</style>\n",
              " </defs>\n",
              " <g id=\"figure_1\">\n",
              "  <g id=\"patch_1\">\n",
              "   <path d=\"M 0 396.273625 \n",
              "L 617.279375 396.273625 \n",
              "L 617.279375 0 \n",
              "L 0 0 \n",
              "z\n",
              "\" style=\"fill: #ffffff\"/>\n",
              "  </g>\n",
              "  <g id=\"axes_1\">\n",
              "   <g id=\"patch_2\">\n",
              "    <path d=\"M 52.079375 356.78175 \n",
              "L 610.079375 356.78175 \n",
              "L 610.079375 24.14175 \n",
              "L 52.079375 24.14175 \n",
              "z\n",
              "\" style=\"fill: #e5e5e5\"/>\n",
              "   </g>\n",
              "   <g id=\"matplotlib.axis_1\">\n",
              "    <g id=\"xtick_1\">\n",
              "     <g id=\"line2d_1\">\n",
              "      <path d=\"M 59.950848 356.78175 \n",
              "L 59.950848 24.14175 \n",
              "\" clip-path=\"url(#p23288580e9)\" style=\"fill: none; stroke: #ffffff; stroke-width: 0.8; stroke-linecap: square\"/>\n",
              "     </g>\n",
              "     <g id=\"line2d_2\">\n",
              "      <defs>\n",
              "       <path id=\"m90818259d3\" d=\"M 0 0 \n",
              "L 0 3.5 \n",
              "\" style=\"stroke: #555555; stroke-width: 0.8\"/>\n",
              "      </defs>\n",
              "      <g>\n",
              "       <use xlink:href=\"#m90818259d3\" x=\"59.950848\" y=\"356.78175\" style=\"fill: #555555; stroke: #555555; stroke-width: 0.8\"/>\n",
              "      </g>\n",
              "     </g>\n",
              "     <g id=\"text_1\">\n",
              "      <!-- 0 -->\n",
              "      <g style=\"fill: #555555\" transform=\"translate(56.769598 371.380187) scale(0.1 -0.1)\">\n",
              "       <defs>\n",
              "        <path id=\"DejaVuSans-30\" d=\"M 2034 4250 \n",
              "Q 1547 4250 1301 3770 \n",
              "Q 1056 3291 1056 2328 \n",
              "Q 1056 1369 1301 889 \n",
              "Q 1547 409 2034 409 \n",
              "Q 2525 409 2770 889 \n",
              "Q 3016 1369 3016 2328 \n",
              "Q 3016 3291 2770 3770 \n",
              "Q 2525 4250 2034 4250 \n",
              "z\n",
              "M 2034 4750 \n",
              "Q 2819 4750 3233 4129 \n",
              "Q 3647 3509 3647 2328 \n",
              "Q 3647 1150 3233 529 \n",
              "Q 2819 -91 2034 -91 \n",
              "Q 1250 -91 836 529 \n",
              "Q 422 1150 422 2328 \n",
              "Q 422 3509 836 4129 \n",
              "Q 1250 4750 2034 4750 \n",
              "z\n",
              "\" transform=\"scale(0.015625)\"/>\n",
              "       </defs>\n",
              "       <use xlink:href=\"#DejaVuSans-30\"/>\n",
              "      </g>\n",
              "     </g>\n",
              "    </g>\n",
              "    <g id=\"xtick_2\">\n",
              "     <g id=\"line2d_3\">\n",
              "      <path d=\"M 147.411663 356.78175 \n",
              "L 147.411663 24.14175 \n",
              "\" clip-path=\"url(#p23288580e9)\" style=\"fill: none; stroke: #ffffff; stroke-width: 0.8; stroke-linecap: square\"/>\n",
              "     </g>\n",
              "     <g id=\"line2d_4\">\n",
              "      <g>\n",
              "       <use xlink:href=\"#m90818259d3\" x=\"147.411663\" y=\"356.78175\" style=\"fill: #555555; stroke: #555555; stroke-width: 0.8\"/>\n",
              "      </g>\n",
              "     </g>\n",
              "     <g id=\"text_2\">\n",
              "      <!-- 5 -->\n",
              "      <g style=\"fill: #555555\" transform=\"translate(144.230413 371.380187) scale(0.1 -0.1)\">\n",
              "       <defs>\n",
              "        <path id=\"DejaVuSans-35\" d=\"M 691 4666 \n",
              "L 3169 4666 \n",
              "L 3169 4134 \n",
              "L 1269 4134 \n",
              "L 1269 2991 \n",
              "Q 1406 3038 1543 3061 \n",
              "Q 1681 3084 1819 3084 \n",
              "Q 2600 3084 3056 2656 \n",
              "Q 3513 2228 3513 1497 \n",
              "Q 3513 744 3044 326 \n",
              "Q 2575 -91 1722 -91 \n",
              "Q 1428 -91 1123 -41 \n",
              "Q 819 9 494 109 \n",
              "L 494 744 \n",
              "Q 775 591 1075 516 \n",
              "Q 1375 441 1709 441 \n",
              "Q 2250 441 2565 725 \n",
              "Q 2881 1009 2881 1497 \n",
              "Q 2881 1984 2565 2268 \n",
              "Q 2250 2553 1709 2553 \n",
              "Q 1456 2553 1204 2497 \n",
              "Q 953 2441 691 2322 \n",
              "L 691 4666 \n",
              "z\n",
              "\" transform=\"scale(0.015625)\"/>\n",
              "       </defs>\n",
              "       <use xlink:href=\"#DejaVuSans-35\"/>\n",
              "      </g>\n",
              "     </g>\n",
              "    </g>\n",
              "    <g id=\"xtick_3\">\n",
              "     <g id=\"line2d_5\">\n",
              "      <path d=\"M 234.872478 356.78175 \n",
              "L 234.872478 24.14175 \n",
              "\" clip-path=\"url(#p23288580e9)\" style=\"fill: none; stroke: #ffffff; stroke-width: 0.8; stroke-linecap: square\"/>\n",
              "     </g>\n",
              "     <g id=\"line2d_6\">\n",
              "      <g>\n",
              "       <use xlink:href=\"#m90818259d3\" x=\"234.872478\" y=\"356.78175\" style=\"fill: #555555; stroke: #555555; stroke-width: 0.8\"/>\n",
              "      </g>\n",
              "     </g>\n",
              "     <g id=\"text_3\">\n",
              "      <!-- 10 -->\n",
              "      <g style=\"fill: #555555\" transform=\"translate(228.509978 371.380187) scale(0.1 -0.1)\">\n",
              "       <defs>\n",
              "        <path id=\"DejaVuSans-31\" d=\"M 794 531 \n",
              "L 1825 531 \n",
              "L 1825 4091 \n",
              "L 703 3866 \n",
              "L 703 4441 \n",
              "L 1819 4666 \n",
              "L 2450 4666 \n",
              "L 2450 531 \n",
              "L 3481 531 \n",
              "L 3481 0 \n",
              "L 794 0 \n",
              "L 794 531 \n",
              "z\n",
              "\" transform=\"scale(0.015625)\"/>\n",
              "       </defs>\n",
              "       <use xlink:href=\"#DejaVuSans-31\"/>\n",
              "       <use xlink:href=\"#DejaVuSans-30\" x=\"63.623047\"/>\n",
              "      </g>\n",
              "     </g>\n",
              "    </g>\n",
              "    <g id=\"xtick_4\">\n",
              "     <g id=\"line2d_7\">\n",
              "      <path d=\"M 322.333293 356.78175 \n",
              "L 322.333293 24.14175 \n",
              "\" clip-path=\"url(#p23288580e9)\" style=\"fill: none; stroke: #ffffff; stroke-width: 0.8; stroke-linecap: square\"/>\n",
              "     </g>\n",
              "     <g id=\"line2d_8\">\n",
              "      <g>\n",
              "       <use xlink:href=\"#m90818259d3\" x=\"322.333293\" y=\"356.78175\" style=\"fill: #555555; stroke: #555555; stroke-width: 0.8\"/>\n",
              "      </g>\n",
              "     </g>\n",
              "     <g id=\"text_4\">\n",
              "      <!-- 15 -->\n",
              "      <g style=\"fill: #555555\" transform=\"translate(315.970793 371.380187) scale(0.1 -0.1)\">\n",
              "       <use xlink:href=\"#DejaVuSans-31\"/>\n",
              "       <use xlink:href=\"#DejaVuSans-35\" x=\"63.623047\"/>\n",
              "      </g>\n",
              "     </g>\n",
              "    </g>\n",
              "    <g id=\"xtick_5\">\n",
              "     <g id=\"line2d_9\">\n",
              "      <path d=\"M 409.794109 356.78175 \n",
              "L 409.794109 24.14175 \n",
              "\" clip-path=\"url(#p23288580e9)\" style=\"fill: none; stroke: #ffffff; stroke-width: 0.8; stroke-linecap: square\"/>\n",
              "     </g>\n",
              "     <g id=\"line2d_10\">\n",
              "      <g>\n",
              "       <use xlink:href=\"#m90818259d3\" x=\"409.794109\" y=\"356.78175\" style=\"fill: #555555; stroke: #555555; stroke-width: 0.8\"/>\n",
              "      </g>\n",
              "     </g>\n",
              "     <g id=\"text_5\">\n",
              "      <!-- 20 -->\n",
              "      <g style=\"fill: #555555\" transform=\"translate(403.431609 371.380187) scale(0.1 -0.1)\">\n",
              "       <defs>\n",
              "        <path id=\"DejaVuSans-32\" d=\"M 1228 531 \n",
              "L 3431 531 \n",
              "L 3431 0 \n",
              "L 469 0 \n",
              "L 469 531 \n",
              "Q 828 903 1448 1529 \n",
              "Q 2069 2156 2228 2338 \n",
              "Q 2531 2678 2651 2914 \n",
              "Q 2772 3150 2772 3378 \n",
              "Q 2772 3750 2511 3984 \n",
              "Q 2250 4219 1831 4219 \n",
              "Q 1534 4219 1204 4116 \n",
              "Q 875 4013 500 3803 \n",
              "L 500 4441 \n",
              "Q 881 4594 1212 4672 \n",
              "Q 1544 4750 1819 4750 \n",
              "Q 2544 4750 2975 4387 \n",
              "Q 3406 4025 3406 3419 \n",
              "Q 3406 3131 3298 2873 \n",
              "Q 3191 2616 2906 2266 \n",
              "Q 2828 2175 2409 1742 \n",
              "Q 1991 1309 1228 531 \n",
              "z\n",
              "\" transform=\"scale(0.015625)\"/>\n",
              "       </defs>\n",
              "       <use xlink:href=\"#DejaVuSans-32\"/>\n",
              "       <use xlink:href=\"#DejaVuSans-30\" x=\"63.623047\"/>\n",
              "      </g>\n",
              "     </g>\n",
              "    </g>\n",
              "    <g id=\"xtick_6\">\n",
              "     <g id=\"line2d_11\">\n",
              "      <path d=\"M 497.254924 356.78175 \n",
              "L 497.254924 24.14175 \n",
              "\" clip-path=\"url(#p23288580e9)\" style=\"fill: none; stroke: #ffffff; stroke-width: 0.8; stroke-linecap: square\"/>\n",
              "     </g>\n",
              "     <g id=\"line2d_12\">\n",
              "      <g>\n",
              "       <use xlink:href=\"#m90818259d3\" x=\"497.254924\" y=\"356.78175\" style=\"fill: #555555; stroke: #555555; stroke-width: 0.8\"/>\n",
              "      </g>\n",
              "     </g>\n",
              "     <g id=\"text_6\">\n",
              "      <!-- 25 -->\n",
              "      <g style=\"fill: #555555\" transform=\"translate(490.892424 371.380187) scale(0.1 -0.1)\">\n",
              "       <use xlink:href=\"#DejaVuSans-32\"/>\n",
              "       <use xlink:href=\"#DejaVuSans-35\" x=\"63.623047\"/>\n",
              "      </g>\n",
              "     </g>\n",
              "    </g>\n",
              "    <g id=\"xtick_7\">\n",
              "     <g id=\"line2d_13\">\n",
              "      <path d=\"M 584.715739 356.78175 \n",
              "L 584.715739 24.14175 \n",
              "\" clip-path=\"url(#p23288580e9)\" style=\"fill: none; stroke: #ffffff; stroke-width: 0.8; stroke-linecap: square\"/>\n",
              "     </g>\n",
              "     <g id=\"line2d_14\">\n",
              "      <g>\n",
              "       <use xlink:href=\"#m90818259d3\" x=\"584.715739\" y=\"356.78175\" style=\"fill: #555555; stroke: #555555; stroke-width: 0.8\"/>\n",
              "      </g>\n",
              "     </g>\n",
              "     <g id=\"text_7\">\n",
              "      <!-- 30 -->\n",
              "      <g style=\"fill: #555555\" transform=\"translate(578.353239 371.380187) scale(0.1 -0.1)\">\n",
              "       <defs>\n",
              "        <path id=\"DejaVuSans-33\" d=\"M 2597 2516 \n",
              "Q 3050 2419 3304 2112 \n",
              "Q 3559 1806 3559 1356 \n",
              "Q 3559 666 3084 287 \n",
              "Q 2609 -91 1734 -91 \n",
              "Q 1441 -91 1130 -33 \n",
              "Q 819 25 488 141 \n",
              "L 488 750 \n",
              "Q 750 597 1062 519 \n",
              "Q 1375 441 1716 441 \n",
              "Q 2309 441 2620 675 \n",
              "Q 2931 909 2931 1356 \n",
              "Q 2931 1769 2642 2001 \n",
              "Q 2353 2234 1838 2234 \n",
              "L 1294 2234 \n",
              "L 1294 2753 \n",
              "L 1863 2753 \n",
              "Q 2328 2753 2575 2939 \n",
              "Q 2822 3125 2822 3475 \n",
              "Q 2822 3834 2567 4026 \n",
              "Q 2313 4219 1838 4219 \n",
              "Q 1578 4219 1281 4162 \n",
              "Q 984 4106 628 3988 \n",
              "L 628 4550 \n",
              "Q 988 4650 1302 4700 \n",
              "Q 1616 4750 1894 4750 \n",
              "Q 2613 4750 3031 4423 \n",
              "Q 3450 4097 3450 3541 \n",
              "Q 3450 3153 3228 2886 \n",
              "Q 3006 2619 2597 2516 \n",
              "z\n",
              "\" transform=\"scale(0.015625)\"/>\n",
              "       </defs>\n",
              "       <use xlink:href=\"#DejaVuSans-33\"/>\n",
              "       <use xlink:href=\"#DejaVuSans-30\" x=\"63.623047\"/>\n",
              "      </g>\n",
              "     </g>\n",
              "    </g>\n",
              "    <g id=\"text_8\">\n",
              "     <!-- Number of clusters -->\n",
              "     <g style=\"fill: #555555\" transform=\"translate(273.84875 386.578) scale(0.12 -0.12)\">\n",
              "      <defs>\n",
              "       <path id=\"DejaVuSans-4e\" d=\"M 628 4666 \n",
              "L 1478 4666 \n",
              "L 3547 763 \n",
              "L 3547 4666 \n",
              "L 4159 4666 \n",
              "L 4159 0 \n",
              "L 3309 0 \n",
              "L 1241 3903 \n",
              "L 1241 0 \n",
              "L 628 0 \n",
              "L 628 4666 \n",
              "z\n",
              "\" transform=\"scale(0.015625)\"/>\n",
              "       <path id=\"DejaVuSans-75\" d=\"M 544 1381 \n",
              "L 544 3500 \n",
              "L 1119 3500 \n",
              "L 1119 1403 \n",
              "Q 1119 906 1312 657 \n",
              "Q 1506 409 1894 409 \n",
              "Q 2359 409 2629 706 \n",
              "Q 2900 1003 2900 1516 \n",
              "L 2900 3500 \n",
              "L 3475 3500 \n",
              "L 3475 0 \n",
              "L 2900 0 \n",
              "L 2900 538 \n",
              "Q 2691 219 2414 64 \n",
              "Q 2138 -91 1772 -91 \n",
              "Q 1169 -91 856 284 \n",
              "Q 544 659 544 1381 \n",
              "z\n",
              "M 1991 3584 \n",
              "L 1991 3584 \n",
              "z\n",
              "\" transform=\"scale(0.015625)\"/>\n",
              "       <path id=\"DejaVuSans-6d\" d=\"M 3328 2828 \n",
              "Q 3544 3216 3844 3400 \n",
              "Q 4144 3584 4550 3584 \n",
              "Q 5097 3584 5394 3201 \n",
              "Q 5691 2819 5691 2113 \n",
              "L 5691 0 \n",
              "L 5113 0 \n",
              "L 5113 2094 \n",
              "Q 5113 2597 4934 2840 \n",
              "Q 4756 3084 4391 3084 \n",
              "Q 3944 3084 3684 2787 \n",
              "Q 3425 2491 3425 1978 \n",
              "L 3425 0 \n",
              "L 2847 0 \n",
              "L 2847 2094 \n",
              "Q 2847 2600 2669 2842 \n",
              "Q 2491 3084 2119 3084 \n",
              "Q 1678 3084 1418 2786 \n",
              "Q 1159 2488 1159 1978 \n",
              "L 1159 0 \n",
              "L 581 0 \n",
              "L 581 3500 \n",
              "L 1159 3500 \n",
              "L 1159 2956 \n",
              "Q 1356 3278 1631 3431 \n",
              "Q 1906 3584 2284 3584 \n",
              "Q 2666 3584 2933 3390 \n",
              "Q 3200 3197 3328 2828 \n",
              "z\n",
              "\" transform=\"scale(0.015625)\"/>\n",
              "       <path id=\"DejaVuSans-62\" d=\"M 3116 1747 \n",
              "Q 3116 2381 2855 2742 \n",
              "Q 2594 3103 2138 3103 \n",
              "Q 1681 3103 1420 2742 \n",
              "Q 1159 2381 1159 1747 \n",
              "Q 1159 1113 1420 752 \n",
              "Q 1681 391 2138 391 \n",
              "Q 2594 391 2855 752 \n",
              "Q 3116 1113 3116 1747 \n",
              "z\n",
              "M 1159 2969 \n",
              "Q 1341 3281 1617 3432 \n",
              "Q 1894 3584 2278 3584 \n",
              "Q 2916 3584 3314 3078 \n",
              "Q 3713 2572 3713 1747 \n",
              "Q 3713 922 3314 415 \n",
              "Q 2916 -91 2278 -91 \n",
              "Q 1894 -91 1617 61 \n",
              "Q 1341 213 1159 525 \n",
              "L 1159 0 \n",
              "L 581 0 \n",
              "L 581 4863 \n",
              "L 1159 4863 \n",
              "L 1159 2969 \n",
              "z\n",
              "\" transform=\"scale(0.015625)\"/>\n",
              "       <path id=\"DejaVuSans-65\" d=\"M 3597 1894 \n",
              "L 3597 1613 \n",
              "L 953 1613 \n",
              "Q 991 1019 1311 708 \n",
              "Q 1631 397 2203 397 \n",
              "Q 2534 397 2845 478 \n",
              "Q 3156 559 3463 722 \n",
              "L 3463 178 \n",
              "Q 3153 47 2828 -22 \n",
              "Q 2503 -91 2169 -91 \n",
              "Q 1331 -91 842 396 \n",
              "Q 353 884 353 1716 \n",
              "Q 353 2575 817 3079 \n",
              "Q 1281 3584 2069 3584 \n",
              "Q 2775 3584 3186 3129 \n",
              "Q 3597 2675 3597 1894 \n",
              "z\n",
              "M 3022 2063 \n",
              "Q 3016 2534 2758 2815 \n",
              "Q 2500 3097 2075 3097 \n",
              "Q 1594 3097 1305 2825 \n",
              "Q 1016 2553 972 2059 \n",
              "L 3022 2063 \n",
              "z\n",
              "\" transform=\"scale(0.015625)\"/>\n",
              "       <path id=\"DejaVuSans-72\" d=\"M 2631 2963 \n",
              "Q 2534 3019 2420 3045 \n",
              "Q 2306 3072 2169 3072 \n",
              "Q 1681 3072 1420 2755 \n",
              "Q 1159 2438 1159 1844 \n",
              "L 1159 0 \n",
              "L 581 0 \n",
              "L 581 3500 \n",
              "L 1159 3500 \n",
              "L 1159 2956 \n",
              "Q 1341 3275 1631 3429 \n",
              "Q 1922 3584 2338 3584 \n",
              "Q 2397 3584 2469 3576 \n",
              "Q 2541 3569 2628 3553 \n",
              "L 2631 2963 \n",
              "z\n",
              "\" transform=\"scale(0.015625)\"/>\n",
              "       <path id=\"DejaVuSans-20\" transform=\"scale(0.015625)\"/>\n",
              "       <path id=\"DejaVuSans-6f\" d=\"M 1959 3097 \n",
              "Q 1497 3097 1228 2736 \n",
              "Q 959 2375 959 1747 \n",
              "Q 959 1119 1226 758 \n",
              "Q 1494 397 1959 397 \n",
              "Q 2419 397 2687 759 \n",
              "Q 2956 1122 2956 1747 \n",
              "Q 2956 2369 2687 2733 \n",
              "Q 2419 3097 1959 3097 \n",
              "z\n",
              "M 1959 3584 \n",
              "Q 2709 3584 3137 3096 \n",
              "Q 3566 2609 3566 1747 \n",
              "Q 3566 888 3137 398 \n",
              "Q 2709 -91 1959 -91 \n",
              "Q 1206 -91 779 398 \n",
              "Q 353 888 353 1747 \n",
              "Q 353 2609 779 3096 \n",
              "Q 1206 3584 1959 3584 \n",
              "z\n",
              "\" transform=\"scale(0.015625)\"/>\n",
              "       <path id=\"DejaVuSans-66\" d=\"M 2375 4863 \n",
              "L 2375 4384 \n",
              "L 1825 4384 \n",
              "Q 1516 4384 1395 4259 \n",
              "Q 1275 4134 1275 3809 \n",
              "L 1275 3500 \n",
              "L 2222 3500 \n",
              "L 2222 3053 \n",
              "L 1275 3053 \n",
              "L 1275 0 \n",
              "L 697 0 \n",
              "L 697 3053 \n",
              "L 147 3053 \n",
              "L 147 3500 \n",
              "L 697 3500 \n",
              "L 697 3744 \n",
              "Q 697 4328 969 4595 \n",
              "Q 1241 4863 1831 4863 \n",
              "L 2375 4863 \n",
              "z\n",
              "\" transform=\"scale(0.015625)\"/>\n",
              "       <path id=\"DejaVuSans-63\" d=\"M 3122 3366 \n",
              "L 3122 2828 \n",
              "Q 2878 2963 2633 3030 \n",
              "Q 2388 3097 2138 3097 \n",
              "Q 1578 3097 1268 2742 \n",
              "Q 959 2388 959 1747 \n",
              "Q 959 1106 1268 751 \n",
              "Q 1578 397 2138 397 \n",
              "Q 2388 397 2633 464 \n",
              "Q 2878 531 3122 666 \n",
              "L 3122 134 \n",
              "Q 2881 22 2623 -34 \n",
              "Q 2366 -91 2075 -91 \n",
              "Q 1284 -91 818 406 \n",
              "Q 353 903 353 1747 \n",
              "Q 353 2603 823 3093 \n",
              "Q 1294 3584 2113 3584 \n",
              "Q 2378 3584 2631 3529 \n",
              "Q 2884 3475 3122 3366 \n",
              "z\n",
              "\" transform=\"scale(0.015625)\"/>\n",
              "       <path id=\"DejaVuSans-6c\" d=\"M 603 4863 \n",
              "L 1178 4863 \n",
              "L 1178 0 \n",
              "L 603 0 \n",
              "L 603 4863 \n",
              "z\n",
              "\" transform=\"scale(0.015625)\"/>\n",
              "       <path id=\"DejaVuSans-73\" d=\"M 2834 3397 \n",
              "L 2834 2853 \n",
              "Q 2591 2978 2328 3040 \n",
              "Q 2066 3103 1784 3103 \n",
              "Q 1356 3103 1142 2972 \n",
              "Q 928 2841 928 2578 \n",
              "Q 928 2378 1081 2264 \n",
              "Q 1234 2150 1697 2047 \n",
              "L 1894 2003 \n",
              "Q 2506 1872 2764 1633 \n",
              "Q 3022 1394 3022 966 \n",
              "Q 3022 478 2636 193 \n",
              "Q 2250 -91 1575 -91 \n",
              "Q 1294 -91 989 -36 \n",
              "Q 684 19 347 128 \n",
              "L 347 722 \n",
              "Q 666 556 975 473 \n",
              "Q 1284 391 1588 391 \n",
              "Q 1994 391 2212 530 \n",
              "Q 2431 669 2431 922 \n",
              "Q 2431 1156 2273 1281 \n",
              "Q 2116 1406 1581 1522 \n",
              "L 1381 1569 \n",
              "Q 847 1681 609 1914 \n",
              "Q 372 2147 372 2553 \n",
              "Q 372 3047 722 3315 \n",
              "Q 1072 3584 1716 3584 \n",
              "Q 2034 3584 2315 3537 \n",
              "Q 2597 3491 2834 3397 \n",
              "z\n",
              "\" transform=\"scale(0.015625)\"/>\n",
              "       <path id=\"DejaVuSans-74\" d=\"M 1172 4494 \n",
              "L 1172 3500 \n",
              "L 2356 3500 \n",
              "L 2356 3053 \n",
              "L 1172 3053 \n",
              "L 1172 1153 \n",
              "Q 1172 725 1289 603 \n",
              "Q 1406 481 1766 481 \n",
              "L 2356 481 \n",
              "L 2356 0 \n",
              "L 1766 0 \n",
              "Q 1100 0 847 248 \n",
              "Q 594 497 594 1153 \n",
              "L 594 3053 \n",
              "L 172 3053 \n",
              "L 172 3500 \n",
              "L 594 3500 \n",
              "L 594 4494 \n",
              "L 1172 4494 \n",
              "z\n",
              "\" transform=\"scale(0.015625)\"/>\n",
              "      </defs>\n",
              "      <use xlink:href=\"#DejaVuSans-4e\"/>\n",
              "      <use xlink:href=\"#DejaVuSans-75\" x=\"74.804688\"/>\n",
              "      <use xlink:href=\"#DejaVuSans-6d\" x=\"138.183594\"/>\n",
              "      <use xlink:href=\"#DejaVuSans-62\" x=\"235.595703\"/>\n",
              "      <use xlink:href=\"#DejaVuSans-65\" x=\"299.072266\"/>\n",
              "      <use xlink:href=\"#DejaVuSans-72\" x=\"360.595703\"/>\n",
              "      <use xlink:href=\"#DejaVuSans-20\" x=\"401.708984\"/>\n",
              "      <use xlink:href=\"#DejaVuSans-6f\" x=\"433.496094\"/>\n",
              "      <use xlink:href=\"#DejaVuSans-66\" x=\"494.677734\"/>\n",
              "      <use xlink:href=\"#DejaVuSans-20\" x=\"529.882812\"/>\n",
              "      <use xlink:href=\"#DejaVuSans-63\" x=\"561.669922\"/>\n",
              "      <use xlink:href=\"#DejaVuSans-6c\" x=\"616.650391\"/>\n",
              "      <use xlink:href=\"#DejaVuSans-75\" x=\"644.433594\"/>\n",
              "      <use xlink:href=\"#DejaVuSans-73\" x=\"707.8125\"/>\n",
              "      <use xlink:href=\"#DejaVuSans-74\" x=\"759.912109\"/>\n",
              "      <use xlink:href=\"#DejaVuSans-65\" x=\"799.121094\"/>\n",
              "      <use xlink:href=\"#DejaVuSans-72\" x=\"860.644531\"/>\n",
              "      <use xlink:href=\"#DejaVuSans-73\" x=\"901.757812\"/>\n",
              "     </g>\n",
              "    </g>\n",
              "   </g>\n",
              "   <g id=\"matplotlib.axis_2\">\n",
              "    <g id=\"ytick_1\">\n",
              "     <g id=\"line2d_15\">\n",
              "      <path d=\"M 52.079375 310.83932 \n",
              "L 610.079375 310.83932 \n",
              "\" clip-path=\"url(#p23288580e9)\" style=\"fill: none; stroke: #ffffff; stroke-width: 0.8; stroke-linecap: square\"/>\n",
              "     </g>\n",
              "     <g id=\"line2d_16\">\n",
              "      <defs>\n",
              "       <path id=\"mb661cf147f\" d=\"M 0 0 \n",
              "L -3.5 0 \n",
              "\" style=\"stroke: #555555; stroke-width: 0.8\"/>\n",
              "      </defs>\n",
              "      <g>\n",
              "       <use xlink:href=\"#mb661cf147f\" x=\"52.079375\" y=\"310.83932\" style=\"fill: #555555; stroke: #555555; stroke-width: 0.8\"/>\n",
              "      </g>\n",
              "     </g>\n",
              "     <g id=\"text_9\">\n",
              "      <!-- 0.86 -->\n",
              "      <g style=\"fill: #555555\" transform=\"translate(22.81375 314.638538) scale(0.1 -0.1)\">\n",
              "       <defs>\n",
              "        <path id=\"DejaVuSans-2e\" d=\"M 684 794 \n",
              "L 1344 794 \n",
              "L 1344 0 \n",
              "L 684 0 \n",
              "L 684 794 \n",
              "z\n",
              "\" transform=\"scale(0.015625)\"/>\n",
              "        <path id=\"DejaVuSans-38\" d=\"M 2034 2216 \n",
              "Q 1584 2216 1326 1975 \n",
              "Q 1069 1734 1069 1313 \n",
              "Q 1069 891 1326 650 \n",
              "Q 1584 409 2034 409 \n",
              "Q 2484 409 2743 651 \n",
              "Q 3003 894 3003 1313 \n",
              "Q 3003 1734 2745 1975 \n",
              "Q 2488 2216 2034 2216 \n",
              "z\n",
              "M 1403 2484 \n",
              "Q 997 2584 770 2862 \n",
              "Q 544 3141 544 3541 \n",
              "Q 544 4100 942 4425 \n",
              "Q 1341 4750 2034 4750 \n",
              "Q 2731 4750 3128 4425 \n",
              "Q 3525 4100 3525 3541 \n",
              "Q 3525 3141 3298 2862 \n",
              "Q 3072 2584 2669 2484 \n",
              "Q 3125 2378 3379 2068 \n",
              "Q 3634 1759 3634 1313 \n",
              "Q 3634 634 3220 271 \n",
              "Q 2806 -91 2034 -91 \n",
              "Q 1263 -91 848 271 \n",
              "Q 434 634 434 1313 \n",
              "Q 434 1759 690 2068 \n",
              "Q 947 2378 1403 2484 \n",
              "z\n",
              "M 1172 3481 \n",
              "Q 1172 3119 1398 2916 \n",
              "Q 1625 2713 2034 2713 \n",
              "Q 2441 2713 2670 2916 \n",
              "Q 2900 3119 2900 3481 \n",
              "Q 2900 3844 2670 4047 \n",
              "Q 2441 4250 2034 4250 \n",
              "Q 1625 4250 1398 4047 \n",
              "Q 1172 3844 1172 3481 \n",
              "z\n",
              "\" transform=\"scale(0.015625)\"/>\n",
              "        <path id=\"DejaVuSans-36\" d=\"M 2113 2584 \n",
              "Q 1688 2584 1439 2293 \n",
              "Q 1191 2003 1191 1497 \n",
              "Q 1191 994 1439 701 \n",
              "Q 1688 409 2113 409 \n",
              "Q 2538 409 2786 701 \n",
              "Q 3034 994 3034 1497 \n",
              "Q 3034 2003 2786 2293 \n",
              "Q 2538 2584 2113 2584 \n",
              "z\n",
              "M 3366 4563 \n",
              "L 3366 3988 \n",
              "Q 3128 4100 2886 4159 \n",
              "Q 2644 4219 2406 4219 \n",
              "Q 1781 4219 1451 3797 \n",
              "Q 1122 3375 1075 2522 \n",
              "Q 1259 2794 1537 2939 \n",
              "Q 1816 3084 2150 3084 \n",
              "Q 2853 3084 3261 2657 \n",
              "Q 3669 2231 3669 1497 \n",
              "Q 3669 778 3244 343 \n",
              "Q 2819 -91 2113 -91 \n",
              "Q 1303 -91 875 529 \n",
              "Q 447 1150 447 2328 \n",
              "Q 447 3434 972 4092 \n",
              "Q 1497 4750 2381 4750 \n",
              "Q 2619 4750 2861 4703 \n",
              "Q 3103 4656 3366 4563 \n",
              "z\n",
              "\" transform=\"scale(0.015625)\"/>\n",
              "       </defs>\n",
              "       <use xlink:href=\"#DejaVuSans-30\"/>\n",
              "       <use xlink:href=\"#DejaVuSans-2e\" x=\"63.623047\"/>\n",
              "       <use xlink:href=\"#DejaVuSans-38\" x=\"95.410156\"/>\n",
              "       <use xlink:href=\"#DejaVuSans-36\" x=\"159.033203\"/>\n",
              "      </g>\n",
              "     </g>\n",
              "    </g>\n",
              "    <g id=\"ytick_2\">\n",
              "     <g id=\"line2d_17\">\n",
              "      <path d=\"M 52.079375 240.844747 \n",
              "L 610.079375 240.844747 \n",
              "\" clip-path=\"url(#p23288580e9)\" style=\"fill: none; stroke: #ffffff; stroke-width: 0.8; stroke-linecap: square\"/>\n",
              "     </g>\n",
              "     <g id=\"line2d_18\">\n",
              "      <g>\n",
              "       <use xlink:href=\"#mb661cf147f\" x=\"52.079375\" y=\"240.844747\" style=\"fill: #555555; stroke: #555555; stroke-width: 0.8\"/>\n",
              "      </g>\n",
              "     </g>\n",
              "     <g id=\"text_10\">\n",
              "      <!-- 0.88 -->\n",
              "      <g style=\"fill: #555555\" transform=\"translate(22.81375 244.643965) scale(0.1 -0.1)\">\n",
              "       <use xlink:href=\"#DejaVuSans-30\"/>\n",
              "       <use xlink:href=\"#DejaVuSans-2e\" x=\"63.623047\"/>\n",
              "       <use xlink:href=\"#DejaVuSans-38\" x=\"95.410156\"/>\n",
              "       <use xlink:href=\"#DejaVuSans-38\" x=\"159.033203\"/>\n",
              "      </g>\n",
              "     </g>\n",
              "    </g>\n",
              "    <g id=\"ytick_3\">\n",
              "     <g id=\"line2d_19\">\n",
              "      <path d=\"M 52.079375 170.850174 \n",
              "L 610.079375 170.850174 \n",
              "\" clip-path=\"url(#p23288580e9)\" style=\"fill: none; stroke: #ffffff; stroke-width: 0.8; stroke-linecap: square\"/>\n",
              "     </g>\n",
              "     <g id=\"line2d_20\">\n",
              "      <g>\n",
              "       <use xlink:href=\"#mb661cf147f\" x=\"52.079375\" y=\"170.850174\" style=\"fill: #555555; stroke: #555555; stroke-width: 0.8\"/>\n",
              "      </g>\n",
              "     </g>\n",
              "     <g id=\"text_11\">\n",
              "      <!-- 0.90 -->\n",
              "      <g style=\"fill: #555555\" transform=\"translate(22.81375 174.649392) scale(0.1 -0.1)\">\n",
              "       <defs>\n",
              "        <path id=\"DejaVuSans-39\" d=\"M 703 97 \n",
              "L 703 672 \n",
              "Q 941 559 1184 500 \n",
              "Q 1428 441 1663 441 \n",
              "Q 2288 441 2617 861 \n",
              "Q 2947 1281 2994 2138 \n",
              "Q 2813 1869 2534 1725 \n",
              "Q 2256 1581 1919 1581 \n",
              "Q 1219 1581 811 2004 \n",
              "Q 403 2428 403 3163 \n",
              "Q 403 3881 828 4315 \n",
              "Q 1253 4750 1959 4750 \n",
              "Q 2769 4750 3195 4129 \n",
              "Q 3622 3509 3622 2328 \n",
              "Q 3622 1225 3098 567 \n",
              "Q 2575 -91 1691 -91 \n",
              "Q 1453 -91 1209 -44 \n",
              "Q 966 3 703 97 \n",
              "z\n",
              "M 1959 2075 \n",
              "Q 2384 2075 2632 2365 \n",
              "Q 2881 2656 2881 3163 \n",
              "Q 2881 3666 2632 3958 \n",
              "Q 2384 4250 1959 4250 \n",
              "Q 1534 4250 1286 3958 \n",
              "Q 1038 3666 1038 3163 \n",
              "Q 1038 2656 1286 2365 \n",
              "Q 1534 2075 1959 2075 \n",
              "z\n",
              "\" transform=\"scale(0.015625)\"/>\n",
              "       </defs>\n",
              "       <use xlink:href=\"#DejaVuSans-30\"/>\n",
              "       <use xlink:href=\"#DejaVuSans-2e\" x=\"63.623047\"/>\n",
              "       <use xlink:href=\"#DejaVuSans-39\" x=\"95.410156\"/>\n",
              "       <use xlink:href=\"#DejaVuSans-30\" x=\"159.033203\"/>\n",
              "      </g>\n",
              "     </g>\n",
              "    </g>\n",
              "    <g id=\"ytick_4\">\n",
              "     <g id=\"line2d_21\">\n",
              "      <path d=\"M 52.079375 100.855601 \n",
              "L 610.079375 100.855601 \n",
              "\" clip-path=\"url(#p23288580e9)\" style=\"fill: none; stroke: #ffffff; stroke-width: 0.8; stroke-linecap: square\"/>\n",
              "     </g>\n",
              "     <g id=\"line2d_22\">\n",
              "      <g>\n",
              "       <use xlink:href=\"#mb661cf147f\" x=\"52.079375\" y=\"100.855601\" style=\"fill: #555555; stroke: #555555; stroke-width: 0.8\"/>\n",
              "      </g>\n",
              "     </g>\n",
              "     <g id=\"text_12\">\n",
              "      <!-- 0.92 -->\n",
              "      <g style=\"fill: #555555\" transform=\"translate(22.81375 104.65482) scale(0.1 -0.1)\">\n",
              "       <use xlink:href=\"#DejaVuSans-30\"/>\n",
              "       <use xlink:href=\"#DejaVuSans-2e\" x=\"63.623047\"/>\n",
              "       <use xlink:href=\"#DejaVuSans-39\" x=\"95.410156\"/>\n",
              "       <use xlink:href=\"#DejaVuSans-32\" x=\"159.033203\"/>\n",
              "      </g>\n",
              "     </g>\n",
              "    </g>\n",
              "    <g id=\"ytick_5\">\n",
              "     <g id=\"line2d_23\">\n",
              "      <path d=\"M 52.079375 30.861028 \n",
              "L 610.079375 30.861028 \n",
              "\" clip-path=\"url(#p23288580e9)\" style=\"fill: none; stroke: #ffffff; stroke-width: 0.8; stroke-linecap: square\"/>\n",
              "     </g>\n",
              "     <g id=\"line2d_24\">\n",
              "      <g>\n",
              "       <use xlink:href=\"#mb661cf147f\" x=\"52.079375\" y=\"30.861028\" style=\"fill: #555555; stroke: #555555; stroke-width: 0.8\"/>\n",
              "      </g>\n",
              "     </g>\n",
              "     <g id=\"text_13\">\n",
              "      <!-- 0.94 -->\n",
              "      <g style=\"fill: #555555\" transform=\"translate(22.81375 34.660247) scale(0.1 -0.1)\">\n",
              "       <defs>\n",
              "        <path id=\"DejaVuSans-34\" d=\"M 2419 4116 \n",
              "L 825 1625 \n",
              "L 2419 1625 \n",
              "L 2419 4116 \n",
              "z\n",
              "M 2253 4666 \n",
              "L 3047 4666 \n",
              "L 3047 1625 \n",
              "L 3713 1625 \n",
              "L 3713 1100 \n",
              "L 3047 1100 \n",
              "L 3047 0 \n",
              "L 2419 0 \n",
              "L 2419 1100 \n",
              "L 313 1100 \n",
              "L 313 1709 \n",
              "L 2253 4666 \n",
              "z\n",
              "\" transform=\"scale(0.015625)\"/>\n",
              "       </defs>\n",
              "       <use xlink:href=\"#DejaVuSans-30\"/>\n",
              "       <use xlink:href=\"#DejaVuSans-2e\" x=\"63.623047\"/>\n",
              "       <use xlink:href=\"#DejaVuSans-39\" x=\"95.410156\"/>\n",
              "       <use xlink:href=\"#DejaVuSans-34\" x=\"159.033203\"/>\n",
              "      </g>\n",
              "     </g>\n",
              "    </g>\n",
              "    <g id=\"text_14\">\n",
              "     <!-- Distortion -->\n",
              "     <g style=\"fill: #555555\" transform=\"translate(16.318125 219.857062) rotate(-90) scale(0.12 -0.12)\">\n",
              "      <defs>\n",
              "       <path id=\"DejaVuSans-44\" d=\"M 1259 4147 \n",
              "L 1259 519 \n",
              "L 2022 519 \n",
              "Q 2988 519 3436 956 \n",
              "Q 3884 1394 3884 2338 \n",
              "Q 3884 3275 3436 3711 \n",
              "Q 2988 4147 2022 4147 \n",
              "L 1259 4147 \n",
              "z\n",
              "M 628 4666 \n",
              "L 1925 4666 \n",
              "Q 3281 4666 3915 4102 \n",
              "Q 4550 3538 4550 2338 \n",
              "Q 4550 1131 3912 565 \n",
              "Q 3275 0 1925 0 \n",
              "L 628 0 \n",
              "L 628 4666 \n",
              "z\n",
              "\" transform=\"scale(0.015625)\"/>\n",
              "       <path id=\"DejaVuSans-69\" d=\"M 603 3500 \n",
              "L 1178 3500 \n",
              "L 1178 0 \n",
              "L 603 0 \n",
              "L 603 3500 \n",
              "z\n",
              "M 603 4863 \n",
              "L 1178 4863 \n",
              "L 1178 4134 \n",
              "L 603 4134 \n",
              "L 603 4863 \n",
              "z\n",
              "\" transform=\"scale(0.015625)\"/>\n",
              "       <path id=\"DejaVuSans-6e\" d=\"M 3513 2113 \n",
              "L 3513 0 \n",
              "L 2938 0 \n",
              "L 2938 2094 \n",
              "Q 2938 2591 2744 2837 \n",
              "Q 2550 3084 2163 3084 \n",
              "Q 1697 3084 1428 2787 \n",
              "Q 1159 2491 1159 1978 \n",
              "L 1159 0 \n",
              "L 581 0 \n",
              "L 581 3500 \n",
              "L 1159 3500 \n",
              "L 1159 2956 \n",
              "Q 1366 3272 1645 3428 \n",
              "Q 1925 3584 2291 3584 \n",
              "Q 2894 3584 3203 3211 \n",
              "Q 3513 2838 3513 2113 \n",
              "z\n",
              "\" transform=\"scale(0.015625)\"/>\n",
              "      </defs>\n",
              "      <use xlink:href=\"#DejaVuSans-44\"/>\n",
              "      <use xlink:href=\"#DejaVuSans-69\" x=\"77.001953\"/>\n",
              "      <use xlink:href=\"#DejaVuSans-73\" x=\"104.785156\"/>\n",
              "      <use xlink:href=\"#DejaVuSans-74\" x=\"156.884766\"/>\n",
              "      <use xlink:href=\"#DejaVuSans-6f\" x=\"196.09375\"/>\n",
              "      <use xlink:href=\"#DejaVuSans-72\" x=\"257.275391\"/>\n",
              "      <use xlink:href=\"#DejaVuSans-74\" x=\"298.388672\"/>\n",
              "      <use xlink:href=\"#DejaVuSans-69\" x=\"337.597656\"/>\n",
              "      <use xlink:href=\"#DejaVuSans-6f\" x=\"365.380859\"/>\n",
              "      <use xlink:href=\"#DejaVuSans-6e\" x=\"426.5625\"/>\n",
              "     </g>\n",
              "    </g>\n",
              "   </g>\n",
              "   <g id=\"line2d_25\">\n",
              "    <path d=\"M 77.443011 39.26175 \n",
              "L 94.935174 81.507054 \n",
              "L 112.427337 118.633819 \n",
              "L 129.9195 129.956106 \n",
              "L 147.411663 161.611422 \n",
              "L 164.903826 177.743369 \n",
              "L 182.395989 194.40098 \n",
              "L 199.888152 206.258469 \n",
              "L 217.380315 213.993993 \n",
              "L 234.872478 223.298584 \n",
              "L 252.364641 234.637283 \n",
              "L 269.856804 239.785605 \n",
              "L 287.348967 247.372765 \n",
              "L 304.84113 254.51901 \n",
              "L 322.333293 259.79593 \n",
              "L 339.825457 270.459671 \n",
              "L 357.31762 273.207124 \n",
              "L 374.809783 278.781329 \n",
              "L 392.301946 284.895954 \n",
              "L 409.794109 295.366486 \n",
              "L 427.286272 298.951367 \n",
              "L 444.778435 302.850839 \n",
              "L 462.270598 306.854826 \n",
              "L 479.762761 312.880942 \n",
              "L 497.254924 320.31472 \n",
              "L 514.747087 324.557879 \n",
              "L 532.23925 327.309592 \n",
              "L 549.731413 336.030276 \n",
              "L 567.223576 336.868208 \n",
              "L 584.715739 341.66175 \n",
              "\" clip-path=\"url(#p23288580e9)\" style=\"fill: none; stroke: #0000ff; stroke-width: 1.5; stroke-linecap: square\"/>\n",
              "    <defs>\n",
              "     <path id=\"md9be95b891\" d=\"M -3 3 \n",
              "L 3 -3 \n",
              "M -3 -3 \n",
              "L 3 3 \n",
              "\" style=\"stroke: #0000ff\"/>\n",
              "    </defs>\n",
              "    <g clip-path=\"url(#p23288580e9)\">\n",
              "     <use xlink:href=\"#md9be95b891\" x=\"77.443011\" y=\"39.26175\" style=\"fill: #0000ff; stroke: #0000ff\"/>\n",
              "     <use xlink:href=\"#md9be95b891\" x=\"94.935174\" y=\"81.507054\" style=\"fill: #0000ff; stroke: #0000ff\"/>\n",
              "     <use xlink:href=\"#md9be95b891\" x=\"112.427337\" y=\"118.633819\" style=\"fill: #0000ff; stroke: #0000ff\"/>\n",
              "     <use xlink:href=\"#md9be95b891\" x=\"129.9195\" y=\"129.956106\" style=\"fill: #0000ff; stroke: #0000ff\"/>\n",
              "     <use xlink:href=\"#md9be95b891\" x=\"147.411663\" y=\"161.611422\" style=\"fill: #0000ff; stroke: #0000ff\"/>\n",
              "     <use xlink:href=\"#md9be95b891\" x=\"164.903826\" y=\"177.743369\" style=\"fill: #0000ff; stroke: #0000ff\"/>\n",
              "     <use xlink:href=\"#md9be95b891\" x=\"182.395989\" y=\"194.40098\" style=\"fill: #0000ff; stroke: #0000ff\"/>\n",
              "     <use xlink:href=\"#md9be95b891\" x=\"199.888152\" y=\"206.258469\" style=\"fill: #0000ff; stroke: #0000ff\"/>\n",
              "     <use xlink:href=\"#md9be95b891\" x=\"217.380315\" y=\"213.993993\" style=\"fill: #0000ff; stroke: #0000ff\"/>\n",
              "     <use xlink:href=\"#md9be95b891\" x=\"234.872478\" y=\"223.298584\" style=\"fill: #0000ff; stroke: #0000ff\"/>\n",
              "     <use xlink:href=\"#md9be95b891\" x=\"252.364641\" y=\"234.637283\" style=\"fill: #0000ff; stroke: #0000ff\"/>\n",
              "     <use xlink:href=\"#md9be95b891\" x=\"269.856804\" y=\"239.785605\" style=\"fill: #0000ff; stroke: #0000ff\"/>\n",
              "     <use xlink:href=\"#md9be95b891\" x=\"287.348967\" y=\"247.372765\" style=\"fill: #0000ff; stroke: #0000ff\"/>\n",
              "     <use xlink:href=\"#md9be95b891\" x=\"304.84113\" y=\"254.51901\" style=\"fill: #0000ff; stroke: #0000ff\"/>\n",
              "     <use xlink:href=\"#md9be95b891\" x=\"322.333293\" y=\"259.79593\" style=\"fill: #0000ff; stroke: #0000ff\"/>\n",
              "     <use xlink:href=\"#md9be95b891\" x=\"339.825457\" y=\"270.459671\" style=\"fill: #0000ff; stroke: #0000ff\"/>\n",
              "     <use xlink:href=\"#md9be95b891\" x=\"357.31762\" y=\"273.207124\" style=\"fill: #0000ff; stroke: #0000ff\"/>\n",
              "     <use xlink:href=\"#md9be95b891\" x=\"374.809783\" y=\"278.781329\" style=\"fill: #0000ff; stroke: #0000ff\"/>\n",
              "     <use xlink:href=\"#md9be95b891\" x=\"392.301946\" y=\"284.895954\" style=\"fill: #0000ff; stroke: #0000ff\"/>\n",
              "     <use xlink:href=\"#md9be95b891\" x=\"409.794109\" y=\"295.366486\" style=\"fill: #0000ff; stroke: #0000ff\"/>\n",
              "     <use xlink:href=\"#md9be95b891\" x=\"427.286272\" y=\"298.951367\" style=\"fill: #0000ff; stroke: #0000ff\"/>\n",
              "     <use xlink:href=\"#md9be95b891\" x=\"444.778435\" y=\"302.850839\" style=\"fill: #0000ff; stroke: #0000ff\"/>\n",
              "     <use xlink:href=\"#md9be95b891\" x=\"462.270598\" y=\"306.854826\" style=\"fill: #0000ff; stroke: #0000ff\"/>\n",
              "     <use xlink:href=\"#md9be95b891\" x=\"479.762761\" y=\"312.880942\" style=\"fill: #0000ff; stroke: #0000ff\"/>\n",
              "     <use xlink:href=\"#md9be95b891\" x=\"497.254924\" y=\"320.31472\" style=\"fill: #0000ff; stroke: #0000ff\"/>\n",
              "     <use xlink:href=\"#md9be95b891\" x=\"514.747087\" y=\"324.557879\" style=\"fill: #0000ff; stroke: #0000ff\"/>\n",
              "     <use xlink:href=\"#md9be95b891\" x=\"532.23925\" y=\"327.309592\" style=\"fill: #0000ff; stroke: #0000ff\"/>\n",
              "     <use xlink:href=\"#md9be95b891\" x=\"549.731413\" y=\"336.030276\" style=\"fill: #0000ff; stroke: #0000ff\"/>\n",
              "     <use xlink:href=\"#md9be95b891\" x=\"567.223576\" y=\"336.868208\" style=\"fill: #0000ff; stroke: #0000ff\"/>\n",
              "     <use xlink:href=\"#md9be95b891\" x=\"584.715739\" y=\"341.66175\" style=\"fill: #0000ff; stroke: #0000ff\"/>\n",
              "    </g>\n",
              "   </g>\n",
              "   <g id=\"patch_3\">\n",
              "    <path d=\"M 52.079375 356.78175 \n",
              "L 52.079375 24.14175 \n",
              "\" style=\"fill: none; stroke: #ffffff; stroke-linejoin: miter; stroke-linecap: square\"/>\n",
              "   </g>\n",
              "   <g id=\"patch_4\">\n",
              "    <path d=\"M 610.079375 356.78175 \n",
              "L 610.079375 24.14175 \n",
              "\" style=\"fill: none; stroke: #ffffff; stroke-linejoin: miter; stroke-linecap: square\"/>\n",
              "   </g>\n",
              "   <g id=\"patch_5\">\n",
              "    <path d=\"M 52.079375 356.78175 \n",
              "L 610.079375 356.78175 \n",
              "\" style=\"fill: none; stroke: #ffffff; stroke-linejoin: miter; stroke-linecap: square\"/>\n",
              "   </g>\n",
              "   <g id=\"patch_6\">\n",
              "    <path d=\"M 52.079375 24.14175 \n",
              "L 610.079375 24.14175 \n",
              "\" style=\"fill: none; stroke: #ffffff; stroke-linejoin: miter; stroke-linecap: square\"/>\n",
              "   </g>\n",
              "   <g id=\"text_15\">\n",
              "    <!-- The Elbow Method showing the optimal number of clusters -->\n",
              "    <g transform=\"translate(118.904375 18.14175) scale(0.144 -0.144)\">\n",
              "     <defs>\n",
              "      <path id=\"DejaVuSans-54\" d=\"M -19 4666 \n",
              "L 3928 4666 \n",
              "L 3928 4134 \n",
              "L 2272 4134 \n",
              "L 2272 0 \n",
              "L 1638 0 \n",
              "L 1638 4134 \n",
              "L -19 4134 \n",
              "L -19 4666 \n",
              "z\n",
              "\" transform=\"scale(0.015625)\"/>\n",
              "      <path id=\"DejaVuSans-68\" d=\"M 3513 2113 \n",
              "L 3513 0 \n",
              "L 2938 0 \n",
              "L 2938 2094 \n",
              "Q 2938 2591 2744 2837 \n",
              "Q 2550 3084 2163 3084 \n",
              "Q 1697 3084 1428 2787 \n",
              "Q 1159 2491 1159 1978 \n",
              "L 1159 0 \n",
              "L 581 0 \n",
              "L 581 4863 \n",
              "L 1159 4863 \n",
              "L 1159 2956 \n",
              "Q 1366 3272 1645 3428 \n",
              "Q 1925 3584 2291 3584 \n",
              "Q 2894 3584 3203 3211 \n",
              "Q 3513 2838 3513 2113 \n",
              "z\n",
              "\" transform=\"scale(0.015625)\"/>\n",
              "      <path id=\"DejaVuSans-45\" d=\"M 628 4666 \n",
              "L 3578 4666 \n",
              "L 3578 4134 \n",
              "L 1259 4134 \n",
              "L 1259 2753 \n",
              "L 3481 2753 \n",
              "L 3481 2222 \n",
              "L 1259 2222 \n",
              "L 1259 531 \n",
              "L 3634 531 \n",
              "L 3634 0 \n",
              "L 628 0 \n",
              "L 628 4666 \n",
              "z\n",
              "\" transform=\"scale(0.015625)\"/>\n",
              "      <path id=\"DejaVuSans-77\" d=\"M 269 3500 \n",
              "L 844 3500 \n",
              "L 1563 769 \n",
              "L 2278 3500 \n",
              "L 2956 3500 \n",
              "L 3675 769 \n",
              "L 4391 3500 \n",
              "L 4966 3500 \n",
              "L 4050 0 \n",
              "L 3372 0 \n",
              "L 2619 2869 \n",
              "L 1863 0 \n",
              "L 1184 0 \n",
              "L 269 3500 \n",
              "z\n",
              "\" transform=\"scale(0.015625)\"/>\n",
              "      <path id=\"DejaVuSans-4d\" d=\"M 628 4666 \n",
              "L 1569 4666 \n",
              "L 2759 1491 \n",
              "L 3956 4666 \n",
              "L 4897 4666 \n",
              "L 4897 0 \n",
              "L 4281 0 \n",
              "L 4281 4097 \n",
              "L 3078 897 \n",
              "L 2444 897 \n",
              "L 1241 4097 \n",
              "L 1241 0 \n",
              "L 628 0 \n",
              "L 628 4666 \n",
              "z\n",
              "\" transform=\"scale(0.015625)\"/>\n",
              "      <path id=\"DejaVuSans-64\" d=\"M 2906 2969 \n",
              "L 2906 4863 \n",
              "L 3481 4863 \n",
              "L 3481 0 \n",
              "L 2906 0 \n",
              "L 2906 525 \n",
              "Q 2725 213 2448 61 \n",
              "Q 2172 -91 1784 -91 \n",
              "Q 1150 -91 751 415 \n",
              "Q 353 922 353 1747 \n",
              "Q 353 2572 751 3078 \n",
              "Q 1150 3584 1784 3584 \n",
              "Q 2172 3584 2448 3432 \n",
              "Q 2725 3281 2906 2969 \n",
              "z\n",
              "M 947 1747 \n",
              "Q 947 1113 1208 752 \n",
              "Q 1469 391 1925 391 \n",
              "Q 2381 391 2643 752 \n",
              "Q 2906 1113 2906 1747 \n",
              "Q 2906 2381 2643 2742 \n",
              "Q 2381 3103 1925 3103 \n",
              "Q 1469 3103 1208 2742 \n",
              "Q 947 2381 947 1747 \n",
              "z\n",
              "\" transform=\"scale(0.015625)\"/>\n",
              "      <path id=\"DejaVuSans-67\" d=\"M 2906 1791 \n",
              "Q 2906 2416 2648 2759 \n",
              "Q 2391 3103 1925 3103 \n",
              "Q 1463 3103 1205 2759 \n",
              "Q 947 2416 947 1791 \n",
              "Q 947 1169 1205 825 \n",
              "Q 1463 481 1925 481 \n",
              "Q 2391 481 2648 825 \n",
              "Q 2906 1169 2906 1791 \n",
              "z\n",
              "M 3481 434 \n",
              "Q 3481 -459 3084 -895 \n",
              "Q 2688 -1331 1869 -1331 \n",
              "Q 1566 -1331 1297 -1286 \n",
              "Q 1028 -1241 775 -1147 \n",
              "L 775 -588 \n",
              "Q 1028 -725 1275 -790 \n",
              "Q 1522 -856 1778 -856 \n",
              "Q 2344 -856 2625 -561 \n",
              "Q 2906 -266 2906 331 \n",
              "L 2906 616 \n",
              "Q 2728 306 2450 153 \n",
              "Q 2172 0 1784 0 \n",
              "Q 1141 0 747 490 \n",
              "Q 353 981 353 1791 \n",
              "Q 353 2603 747 3093 \n",
              "Q 1141 3584 1784 3584 \n",
              "Q 2172 3584 2450 3431 \n",
              "Q 2728 3278 2906 2969 \n",
              "L 2906 3500 \n",
              "L 3481 3500 \n",
              "L 3481 434 \n",
              "z\n",
              "\" transform=\"scale(0.015625)\"/>\n",
              "      <path id=\"DejaVuSans-70\" d=\"M 1159 525 \n",
              "L 1159 -1331 \n",
              "L 581 -1331 \n",
              "L 581 3500 \n",
              "L 1159 3500 \n",
              "L 1159 2969 \n",
              "Q 1341 3281 1617 3432 \n",
              "Q 1894 3584 2278 3584 \n",
              "Q 2916 3584 3314 3078 \n",
              "Q 3713 2572 3713 1747 \n",
              "Q 3713 922 3314 415 \n",
              "Q 2916 -91 2278 -91 \n",
              "Q 1894 -91 1617 61 \n",
              "Q 1341 213 1159 525 \n",
              "z\n",
              "M 3116 1747 \n",
              "Q 3116 2381 2855 2742 \n",
              "Q 2594 3103 2138 3103 \n",
              "Q 1681 3103 1420 2742 \n",
              "Q 1159 2381 1159 1747 \n",
              "Q 1159 1113 1420 752 \n",
              "Q 1681 391 2138 391 \n",
              "Q 2594 391 2855 752 \n",
              "Q 3116 1113 3116 1747 \n",
              "z\n",
              "\" transform=\"scale(0.015625)\"/>\n",
              "      <path id=\"DejaVuSans-61\" d=\"M 2194 1759 \n",
              "Q 1497 1759 1228 1600 \n",
              "Q 959 1441 959 1056 \n",
              "Q 959 750 1161 570 \n",
              "Q 1363 391 1709 391 \n",
              "Q 2188 391 2477 730 \n",
              "Q 2766 1069 2766 1631 \n",
              "L 2766 1759 \n",
              "L 2194 1759 \n",
              "z\n",
              "M 3341 1997 \n",
              "L 3341 0 \n",
              "L 2766 0 \n",
              "L 2766 531 \n",
              "Q 2569 213 2275 61 \n",
              "Q 1981 -91 1556 -91 \n",
              "Q 1019 -91 701 211 \n",
              "Q 384 513 384 1019 \n",
              "Q 384 1609 779 1909 \n",
              "Q 1175 2209 1959 2209 \n",
              "L 2766 2209 \n",
              "L 2766 2266 \n",
              "Q 2766 2663 2505 2880 \n",
              "Q 2244 3097 1772 3097 \n",
              "Q 1472 3097 1187 3025 \n",
              "Q 903 2953 641 2809 \n",
              "L 641 3341 \n",
              "Q 956 3463 1253 3523 \n",
              "Q 1550 3584 1831 3584 \n",
              "Q 2591 3584 2966 3190 \n",
              "Q 3341 2797 3341 1997 \n",
              "z\n",
              "\" transform=\"scale(0.015625)\"/>\n",
              "     </defs>\n",
              "     <use xlink:href=\"#DejaVuSans-54\"/>\n",
              "     <use xlink:href=\"#DejaVuSans-68\" x=\"61.083984\"/>\n",
              "     <use xlink:href=\"#DejaVuSans-65\" x=\"124.462891\"/>\n",
              "     <use xlink:href=\"#DejaVuSans-20\" x=\"185.986328\"/>\n",
              "     <use xlink:href=\"#DejaVuSans-45\" x=\"217.773438\"/>\n",
              "     <use xlink:href=\"#DejaVuSans-6c\" x=\"280.957031\"/>\n",
              "     <use xlink:href=\"#DejaVuSans-62\" x=\"308.740234\"/>\n",
              "     <use xlink:href=\"#DejaVuSans-6f\" x=\"372.216797\"/>\n",
              "     <use xlink:href=\"#DejaVuSans-77\" x=\"433.398438\"/>\n",
              "     <use xlink:href=\"#DejaVuSans-20\" x=\"515.185547\"/>\n",
              "     <use xlink:href=\"#DejaVuSans-4d\" x=\"546.972656\"/>\n",
              "     <use xlink:href=\"#DejaVuSans-65\" x=\"633.251953\"/>\n",
              "     <use xlink:href=\"#DejaVuSans-74\" x=\"694.775391\"/>\n",
              "     <use xlink:href=\"#DejaVuSans-68\" x=\"733.984375\"/>\n",
              "     <use xlink:href=\"#DejaVuSans-6f\" x=\"797.363281\"/>\n",
              "     <use xlink:href=\"#DejaVuSans-64\" x=\"858.544922\"/>\n",
              "     <use xlink:href=\"#DejaVuSans-20\" x=\"922.021484\"/>\n",
              "     <use xlink:href=\"#DejaVuSans-73\" x=\"953.808594\"/>\n",
              "     <use xlink:href=\"#DejaVuSans-68\" x=\"1005.908203\"/>\n",
              "     <use xlink:href=\"#DejaVuSans-6f\" x=\"1069.287109\"/>\n",
              "     <use xlink:href=\"#DejaVuSans-77\" x=\"1130.46875\"/>\n",
              "     <use xlink:href=\"#DejaVuSans-69\" x=\"1212.255859\"/>\n",
              "     <use xlink:href=\"#DejaVuSans-6e\" x=\"1240.039062\"/>\n",
              "     <use xlink:href=\"#DejaVuSans-67\" x=\"1303.417969\"/>\n",
              "     <use xlink:href=\"#DejaVuSans-20\" x=\"1366.894531\"/>\n",
              "     <use xlink:href=\"#DejaVuSans-74\" x=\"1398.681641\"/>\n",
              "     <use xlink:href=\"#DejaVuSans-68\" x=\"1437.890625\"/>\n",
              "     <use xlink:href=\"#DejaVuSans-65\" x=\"1501.269531\"/>\n",
              "     <use xlink:href=\"#DejaVuSans-20\" x=\"1562.792969\"/>\n",
              "     <use xlink:href=\"#DejaVuSans-6f\" x=\"1594.580078\"/>\n",
              "     <use xlink:href=\"#DejaVuSans-70\" x=\"1655.761719\"/>\n",
              "     <use xlink:href=\"#DejaVuSans-74\" x=\"1719.238281\"/>\n",
              "     <use xlink:href=\"#DejaVuSans-69\" x=\"1758.447266\"/>\n",
              "     <use xlink:href=\"#DejaVuSans-6d\" x=\"1786.230469\"/>\n",
              "     <use xlink:href=\"#DejaVuSans-61\" x=\"1883.642578\"/>\n",
              "     <use xlink:href=\"#DejaVuSans-6c\" x=\"1944.921875\"/>\n",
              "     <use xlink:href=\"#DejaVuSans-20\" x=\"1972.705078\"/>\n",
              "     <use xlink:href=\"#DejaVuSans-6e\" x=\"2004.492188\"/>\n",
              "     <use xlink:href=\"#DejaVuSans-75\" x=\"2067.871094\"/>\n",
              "     <use xlink:href=\"#DejaVuSans-6d\" x=\"2131.25\"/>\n",
              "     <use xlink:href=\"#DejaVuSans-62\" x=\"2228.662109\"/>\n",
              "     <use xlink:href=\"#DejaVuSans-65\" x=\"2292.138672\"/>\n",
              "     <use xlink:href=\"#DejaVuSans-72\" x=\"2353.662109\"/>\n",
              "     <use xlink:href=\"#DejaVuSans-20\" x=\"2394.775391\"/>\n",
              "     <use xlink:href=\"#DejaVuSans-6f\" x=\"2426.5625\"/>\n",
              "     <use xlink:href=\"#DejaVuSans-66\" x=\"2487.744141\"/>\n",
              "     <use xlink:href=\"#DejaVuSans-20\" x=\"2522.949219\"/>\n",
              "     <use xlink:href=\"#DejaVuSans-63\" x=\"2554.736328\"/>\n",
              "     <use xlink:href=\"#DejaVuSans-6c\" x=\"2609.716797\"/>\n",
              "     <use xlink:href=\"#DejaVuSans-75\" x=\"2637.5\"/>\n",
              "     <use xlink:href=\"#DejaVuSans-73\" x=\"2700.878906\"/>\n",
              "     <use xlink:href=\"#DejaVuSans-74\" x=\"2752.978516\"/>\n",
              "     <use xlink:href=\"#DejaVuSans-65\" x=\"2792.1875\"/>\n",
              "     <use xlink:href=\"#DejaVuSans-72\" x=\"2853.710938\"/>\n",
              "     <use xlink:href=\"#DejaVuSans-73\" x=\"2894.824219\"/>\n",
              "    </g>\n",
              "   </g>\n",
              "  </g>\n",
              " </g>\n",
              " <defs>\n",
              "  <clipPath id=\"p23288580e9\">\n",
              "   <rect x=\"52.079375\" y=\"24.14175\" width=\"558\" height=\"332.64\"/>\n",
              "  </clipPath>\n",
              " </defs>\n",
              "</svg>\n"
            ],
            "text/plain": [
              "<Figure size 1000x600 with 1 Axes>"
            ]
          },
          "metadata": {},
          "output_type": "display_data"
        }
      ],
      "source": [
        "# Plot the elbow\n",
        "plt.figure(figsize=(10, 6))\n",
        "plt.plot(K, distortions, 'bx-')\n",
        "plt.xlabel('Number of clusters')\n",
        "plt.ylabel('Distortion')\n",
        "plt.title('The Elbow Method showing the optimal number of clusters')\n",
        "plt.grid(True)\n",
        "plt.show()"
      ]
    },
    {
      "cell_type": "code",
      "execution_count": 78,
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/",
          "height": 608
        },
        "id": "meqpsBOxtkX-",
        "outputId": "e4935823-fe22-4c31-96f5-8f8af955dfc2"
      },
      "outputs": [
        {
          "name": "stderr",
          "output_type": "stream",
          "text": [
            "/usr/local/lib/python3.10/dist-packages/sklearn/cluster/_kmeans.py:870: FutureWarning: The default value of `n_init` will change from 10 to 'auto' in 1.4. Set the value of `n_init` explicitly to suppress the warning\n",
            "  warnings.warn(\n"
          ]
        },
        {
          "data": {
            "application/vnd.google.colaboratory.intrinsic+json": {
              "summary": "{\n  \"name\": \"df\",\n  \"rows\": 1454,\n  \"fields\": [\n    {\n      \"column\": \"paper_id\",\n      \"properties\": {\n        \"dtype\": \"string\",\n        \"num_unique_values\": 1454,\n        \"samples\": [\n          \"f78956e70c8bd1756ffb24ae47c35c3217bed384\",\n          \"51ad0f064355c4678f18225f7d231f9865f44a11\",\n          \"7400110e77de9824ff53797dfade054fa9a9cf80\"\n        ],\n        \"semantic_type\": \"\",\n        \"description\": \"\"\n      }\n    },\n    {\n      \"column\": \"doi\",\n      \"properties\": {\n        \"dtype\": \"string\",\n        \"num_unique_values\": 1411,\n        \"samples\": [\n          \"10.3201/eid1601.091376\",\n          \"10.1016/j.jcv.2011.12.020\",\n          \"10.1101/2020.04.14.040204\"\n        ],\n        \"semantic_type\": \"\",\n        \"description\": \"\"\n      }\n    },\n    {\n      \"column\": \"abstract\",\n      \"properties\": {\n        \"dtype\": \"string\",\n        \"num_unique_values\": 1044,\n        \"samples\": [\n          \"InnoMetrics is the system that aims at collecting software development process metrics in a non-invasive way to access and optimize the development process and its efficiency. This paper demonstrates the development and analysis of energy consumption of MacOS systems based on the software process measurement data. It represents the experience of the development of MacOS system collector and Transfer, in addition to the user interface and early analysis of energy consumption metrics calculations.\",\n          \"During the global outbreak of severe acute respiratory syndrome (SARS) in 2003, treatment was empiric. We report the case history of the index patient in a hospital outbreak of SARS in Hong Kong. The patient recovered after conventional antimicrobial therapy. Further studies are needed to address treatment of SARS, which has high attack and death rates.\",\n          \"Intestinal transplantation (IT) is the least common form of organ transplantation; however, it has shown exceptional growth and improvement in graft survival rates over the past two decades mainly due to better outcomes achieved during the first year of transplantation (76 % at 1 year), due to improvement in surgical techniques and the development of better immunosupressive therapies as we understand more about the relationship between the recipient and host immune system. There are still ongoing issues with chronic rejection and long-term survival. Intestinal transplantation is still an acceptable therapy for patients with intestinal failure (IF), but it is generally reserved for patients who develop severe and life-threatening complications despite standard therapies, or those who are not able to maintain a good quality of life. The purpose of this review is to describe the current status, indications, outcomes and advances in the field of intestinal transplantation.\"\n        ],\n        \"semantic_type\": \"\",\n        \"description\": \"\"\n      }\n    },\n    {\n      \"column\": \"body_text\",\n      \"properties\": {\n        \"dtype\": \"string\",\n        \"num_unique_values\": 1453,\n        \"samples\": [\n          \"Respiratory tract infection is common in infants and young children and is a major public health problem in this age group [1] . It is mainly caused by infection of seasonal viruses, such as respiratory syncytial virus (RSV) and rhinovirus [2] [3] [4] . Most infections manifest as an upper respiratory infection, while 20-40 % develop lower respiratory infection, and a small number (2 -3 %) of infected infants require hospitalization [5, 6] . This is of particular concern as infants with lower respiratory viral infection are at increased risk of developing recurrent wheezing or childhood asthma [7] [8] [9] [10] [11] [12] [13] . Several risk factors have been identified for the development of severe lower respiratory viral infections, such as premature birth, young age, being born in relation to the RSV season, underlying chronic lung disease, congenital heart disease, high parity, young maternal age and poor socioeconomic factors [14] [15] [16] . Recently, genetic polymorphisms have been identified which are associated with severe viral infection [17, 18] . Many of the identified genetic variants associated with severe lower respiratory tract infections are also associated with and pathways common to asthma, including innate immune genes involved in cytokine and chemokine signaling, and epithelial cell function [13, 17, [19] [20] [21] [22] [23] .\\nAs the clinical hallmark of asthma is wheezing, and therapy includes beta agonists, a number of clinical studies over the past 2 decades have examined the relationship between genetic variants in \\u03b2 2 -Adrenergic receptor (ADRB2) and asthma [24] [25] [26] [27] [28] [29] [30] . The polymorphism that resulted in the Gly16 allele was associated with decreased bronchodilator responses in early clinical studies; however, larger clinical studies have provided equivocal results [31] [32] [33] . Other genetic variants in the promoter region of ADRB2 have also been associated with asthma severity [28, 34] . Despite the significant interest in ADRB2 polymorphisms related to asthma, the relative contribution of ADRB2 genetic variants to severity of respiratory viral infections has not been established. As wheezing is also the clinical hallmark of lower respiratory viral infections, and variable response to bronchodilators has been demonstrated in infants hospitalized with RSV lower respiratory viral infections, it is possible that ADRB2 genetic variants also contribute to the severity of acute respiratory viral infections [35, 36] .\\nThe aim of the present study was to explore associations between the genetic variants in ADRB2 that have been associated with a response to asthma therapy during an acute asthma exacerbation and the severity of infant acute respiratory viral infections. We examined the relationship between 5 common polymorphisms in the promoter region and coding block of the ADRB2 gene and infant severity of respiratory infections in a prospective cohort of Caucasian and African American infants.\\nIn order to examine the association of genetic variants in the regulatory regions of the ADRB2 gene and the severity of viral respiratory infections during infancy, we conducted an analysis of infants enrolled in the Tennessee Children's Respiratory Initiative (TCRI), a prospective cohort of mother-infant dyads enrolled during an infant's acute viral respiratory illness over four respiratory viral seasons, 2004-2008. Term (\\u226537 weeks) and non-low birth weight (\\u22652275 g) infants who were otherwise healthy with no significant co-morbidities or cardio-pulmonary disease were eligible, with oversampling for hospitalization infants. Eighty five percent of enrolled subjects had at least one respiratory virus identified by polymerase chain reaction (RSV, rhinovirus, influenza, parainfluenza, Human Metapneumovirus, and coronavirus); 56 % had labconfirmed RSV infection. The rationale, methods, and detailed inclusion and exclusion criteria have been reported previously [37] . Although 630 infants were enrolled in TCRI, this study was limited to the 261 Caucasian and 113 African American infants for whom DNA was available. The protocol was approved by the Institutional Review Board of Vanderbilt University, and mothers provided informed consent for their infants. All parents provided written informed consent for both their and their child's study participation.\\nInfants included in this study had either upper respiratory tract infections or lower respiratory tract infections. The severity of acute respiratory infection was determined by experienced research nurses through medical record chart review using an ordinal bronchiolitis severity score (BSS). The score is an aggregate of assigned values ranging of 0-3 in categories of respiratory rate, room air oxygen saturation, the presence and extent of wheezing, and the presence and extent of flaring and retractions [37] [38] [39] . The BSS ranges from 0 to 12, with higher scores indicating more severe illness and a difference of 0.5 being a clinically significant difference [40] . We measured the BSS at the end of the outpatient visit or after discharge from an inpatient admission, and the most severe recorded value was used for analysis.\\nDNA was extracted by Vanderbilt's DNA core laboratory from whole blood or saliva samples (Oragene by DNA Genotek) by standardized protocols. All oragene samples were processed immediately after collection, while blood samples were either processed immediately or frozen before processing. Genotyping at five pre-specified loci (-2387, -2274, -1343, +46, and +79) of ADRB2 gene was done using Taqman SM\\u00a9 Genotyping Assays (Applied Biosystems, Foster City, CA, USA) with predetermined primers. The five single-nucleotide polymorphisms (SNP)s were rs1432623, rs11168068, and rs2400707 at the promoter region, and rs1042713 SNP and rs1042714 SNP at the coding region of ADRB2 gene. Probes for the Applied Biosystems assays used were labeled at their 5\\u2032end with either VIC\\u00ae or 6-carboxy-fluorescine (FAM) reporter dyes. The reaction components include: 2.5 \\u03bcl Taqman Universal PCR Master Mix (Applied Biosystems), 0.125 \\u03bcl or 0.250 \\u03bcl assay mix (Applied Biosystems) and approximately 5 ng of genomic DNA in a total volume of 10 \\u03bcl per single tube reaction. PCR amplification was performed in a thermal cycler (Techne TC-412) with an initial step of 95\\u00b0C for 10 min followed by 50 cycles of 92\\u00b0C for 15 s and 60\\u00b0C for 1 min. After amplification, the fluorescence of each sample was read on the ABI 7900HT (DNA Resources Core at Vanderbilt University) and analyzed with the Sequence Detection Software (Applied Biosystems). Five percent of samples were run as blind duplicates with 100 % concordance. Primers and probes for all SNPs are listed in Table 1 .\\nThe demographics and characteristics of the 374 infants included, with either upper respiratory infection or lower respiratory tract infection, are presented as frequencies and proportions for specific categorical variables and medians and interquartile ranges were calculated for continuous variables. All analyses were stratified by race (Caucasian or African American), as a surrogate measure of European or African ancestry, to reduce the known impact of population substructure which can produce spurious associations [41] . A pairwise linkage disequilibrium (LD) was estimated for the SNPs in ADRB2 using the standardized summary statistics D' and r, calculated by the HaploView program (Whitehead Institute for Biomedical Research, Cambridge, MA, USA). SNPs were tested for deviations from Hardy Weinberg proportions, also using Haploview [42] . The promoter, coding and combined promoter and coding SNPs were assigned to blocks using D' confidence interval of Gabriel et al. [43] . Individual haplotypes were estimated using the program PHASE v2.1.1 [44, 45] . The haplotype frequencies for each block were estimated using the expectation maximization (EM) algorithm, again using the HaploView program. Non-parametric Kruskal-Wallis test were applied to compare the difference in BSS among infants with each SNP genotype, as well as each haplotype genotype. Haplotype genotypes with frequency less than 2 % were excluded from haplotype analysis. We used a proportional odds model for the ordinal BSS to test the additive effect of haplotypes. Covariates included in the regression model included infant age at enrollment, gender, daycare exposure, secondhand smoke exposure, any prior history of breastfeeding, any siblings at home, and enrollment season. All analyses were conducted separately according to race. All analyses were performed using R-software version 2.11.1 (www.r-project.org). We used a two-sided 5 % significance level for all statistical inferences.\\nThere were in total 374 infants, 261 Caucasians and 113 African Americans, enrolled in the TCRI cohort with available DNA and at least one ADRB2 polymorphism tested ( Table 3 lists the allele frequencies of 5 SNPs separately for Caucasian and African Americans. In both Caucasians and African Americans, infants had slightly more T, T, and G alleles for the three promoter SNP rs1432623, (Figs. 1 and 2) . This non statistical difference was consistent within both Caucasians and African Americans. Among African Americans, there was a decreasing trend of the severity of acute respiratory infection with an increasing number of copies of C, C, and A allele of the three promoter SNPs rs1432623, rs11168068, and rs2400707, respectively (Fig. 1) . However, analyses with additive models indicated no statistical significance (p > 0.05).\\nAmong 2 5 (32) possible haplotype combinations, only four haplotypes with more than 2 % of frequency were identified (Table 4 and Fig. 3 ). There were two promoter haplotypes and three coding block haplotypes. There were no statistically significant differences in the promoter and coding block haplotype frequencies between Caucasian and African Americans. However, when all 5 SNPs were combined, the distribution of haplotype frequency was significantly different between Caucasians and African Americans, with genotype CCAAC more common in African Americans and rare in Caucasians (p < 0.001).\\nPromoter haplotype CCA was associated with a decreased BSS in African Americans in a dose dependent manner (p = 0.037 in the univariate analysis) (Fig. 4) . Infants without the CCA haplotype had the highest BSS (median 5.5, interquartile range [IQR]: 2.0, 8.0). Infants with one copy of CCA haplotype had a lower bronchiolitis severity score (median 4.0, IQR: 1.0, 7.5). Infants with 2 copies of the CCA haplotype had the comparatively lowest BSS (median 3.0, IQR: 1.0, 4.0). This dose dependent relationship persisted after adjusting for infant age, gender, daycare exposure, secondhand smoke exposure, prior history of breastfeeding, siblings at home, and enrollment season (adjusted odds ratio: 0.59, 95 % confidence interval [CI]: 0.36, 0.98, p = 0.042). This protective effect of CCA haplotype was consistent when we categorized the acute respiratory infection into upper respiratory infection and lower respiratory tract infection. In a similar dose dependent manner, infants with more copies of CCA haplotype were less likely to have lower respiratory tract infection compared with infants with less copies of CCA haplotype (p = 0.05). On the other hand, promoter haplotype TTG was associated with a higher BSS in African Americans (Additional file 1).\\nThe protective effect of CCA haplotype in African Americans was identified in all combined haplotypes with CCA combination (Figs. 5 and 6 ). However, due to the sample size, the effect was not statistically significant. In addition, all African American infants with a coding block haplotype GG had CCA combination in their promoter region; therefore, there was a similar additive protective effect of GG in African Americans (Additional file 2). There was no relationship between BSS and the other two coding block haplotypes in African Americans (Additional files 3 and 4). This was also true for the combined haplotypes (Additional files 5 and 6).\\nIn both Caucasian and African American infants, there were no significant relationships between BSS and coding block haplotypes except for GG haplotype among African Americans (Additional file 2). This was also true for the combined haplotypes. In general, there was no significant relationship between BSS and any haplotypes in Caucasian infants. There was no similar promoter haplotype CCA relationship with BSS in Caucasians as observed in the African Americans (Fig. 4 ).\\nIn this study we have identified a haplotype in the promoter region of the ADRB2 gene which is associated with a decreased BSS in African American infants. This promoter haplotype of the ADRB2 gene was not associated with the severity of respiratory infections in Caucasian infants. The common coding block polymorphisms, corresponding to amino acid changes at codons 16 and 27, were not associated with a protective effect on severity of respiratory infections in Caucasian or African American infants. Our results agree with the studied conducted by Janssen and colleagues, who found no relationship between the SNP rs1042713 and susceptibility to RSV bronchiolitis in a European population [17] . This observation that the CCA promoter region haplotype is associated with a protective effect on respiratory viral infection severity is similar to our observation in a cohort of adults hospitalized for an asthma exacerbation, where this haplotype was associated with spirometric improvement in response to asthma-specific therapy that included beta-agonists and corticosteroids [34] . In that study, we noted spirometric improvement only in Caucasian patients, although the small number of African American non-responders in that cohort may have limited our ability to detect a similar effect in African Americans [34] .\\nThe mechanism by which a haplotype in the promoter region of the ADRB2 gene is associated with a decreased BSS is not known. However, we can speculate that [46] . Moore et al. demonstrated that ADRB2 genetic variants are associated with differences in ADRB2 expression in cultured human airway smooth muscle (HASM) cells, an in vitro system used as a model for changes in the human airway [47] These studies were performed prior to identification of the complex promoter haplotypes, and we are not aware that these haplotypes have been studied in HASM. Small differences in airway tone could thus be important in the setting of acute viral respiratory infections.\\nAlthough beyond the scope of the current observational study, we also speculate that ADRB2 genotype might also influence response to bronchiolitis-specific therapy. The primary goal of TCRI was to establish a longitudinal, prospective investigation of infants and their biological mothers on the acute and the long-term health consequences of varying viral respiratory tract infections on early childhood outcomes [37] . Although a BSS was assigned for each infant at acute care visit or hospital admission, we did not study or quantify the effects of different treatments, including beta-agonist therapy. A number of clinical studies have suggested that some infants with bronchiolitis appear to improve with bronchodilator therapy, while on average it is not efficacious [48] [49] [50] . As ADRB2 is the primary target of bronchodilator therapy, variation in response to beta-agonists might reflect genetic variants in ADRB2.\\nThe diversity of SNPs of the ADRB2 gene was explored by Drysdale et al., using immortalized lymphocytes from 23 Caucasians, 19 African-Americans, 20 Asians, and 15 Hispanic-Latinos [27] . This report from (-2387, -2274 , and -1343) and in the coding block SNPs discussed in this manuscript [28] . Such frequency distributions observed in our study are consistent with the allele frequencies reported in the reference population of the International HapMap Project [51] and Hawkins et al. [28] . Our stratified analyses by race therefore allow us to detect the association between severity of a respiratory illness and haplotype frequency within each racial group, and to minimize the spurious associations due to such population substructure. We recognize that the significance of our findings is limited by our small sample size and in classifying infants in this study by a severity score rather than by measuring some direct response to bronchodilator therapy. However, BSS correlated with the severity of disease (classified as upper respiratory and lower respiratory illness) well. Our sensitivity analyses with the severity of disease (upper versus lower respiratory illness) showed consistent results indicated that BSS is a valid surrogate of disease severity. Lastly, although a p value of 0.042 is at the threshold of statistical significance, this p value reflects sample size of 113 African Americans in our study. Whether the association between BSS and CCA haplotype in African Americans is powerful or not depends upon not only p value, but also upon the effect size such as odds ratio and the clinical significance of the findings. As a difference of 0.5 in BSS is considered as clinically significant, a 65 % reduced odds of developing a more severe disease (higher BSS) comparing 2 copies of CCA with 0 copy of CCA might also be clinically significant and meaningful.\\nWe hope that this hypothesis-generating study will encourage the incorporation of pharmacokinetic data in future studies of bronchiolitis. Although validation studies in other populations are needed, this study underscores that the development of personalized approaches to the treatment of bronchiolitis may benefit from the analysis of specific genetic variants.\\nIn summary, we have observed that promoter region haplotype of ADRB2 CCA is associated with a protective effect on respiratory illness severity in African American infants. In this era of personalized medicine, ADRB2 genotype may be predictive of the severity of respiratory viral infection and potential response to therapy for the most common acute infant illness and one for which we currently have no therapy. Next steps would be to validate these findings, as well as evaluate available clinical trial data for which DNA might be available. Fig. 6 BSS distribution stratified by copy number of combined haplotype CCAAC in African Americans infants. The scatter plots and the box-and-whisker plots of BSS across 0, 1, and 2 copies of combined haplotype CCAAC for African American infants. P values were obtained from multivariable regression model adjusted for infant age at enrollment, gender, daycare exposure, secondhand smoke exposure, any prior history of breastfeeding, any siblings at home, and enrollment season\",\n          \"Acute bronchiolitis, a viral infection of the lower respiratory tract, is one of the most substantial health burdens for infants and young children worldwide. 1 Respiratory syncytial virus is the most prevalent viral cause of bronchiolitis in infants. Estimates suggest that about 34 million new cases of lower respiratory infection due to respiratory syncytial virus occur globally in children younger than 5 years, with 3\\u00b74 million admissions to hospitals and about 199 000 deaths per year, predominantly in the developing world. 2 In developed countries such as the USA, bronchiolitis is the most common reason for admission to hospital in the fi rst 12 months of life, 3 accounting for approximately 100 000 infant admissions annually. Although admissions to hospital have declined from 2000 to 2010, emergency department visits have increased, in addition to increased use of mechanical ventilation and hospital charges. 4, 5 The clinical management remains challenging despite the frequency, global reach, economic cost, and morbidity and mortality associated with bronchiolitis. Several treatment strategies (including bronchodilators and corticosteroids) showed no eff ect in pooled metaanalyses, making supportive care the hallmark of current therapy. In this Seminar, we aim to summarise the current evidence for the epidemiology, pathophysiology, diagnostic approach, and management of acute viral bronchiolitis.\\nBronchiolitis is a seasonal infection, with the season typically beginning in late October in the temperate northern hemisphere, peaking in January or February, and ending in April. 6 Globally, independent of region, respiratory syncytial virus infection peaks consistently during annual or biannual epidemics. Although the peak and duration of these epidemics vary worldwide, they are consistent year-to-year within a country. 7 Some data suggest that climate might also be associated with prevalence of respiratory syncytial virus infection, with global surveillance suggesting that infection peaks during wet months in areas with high precipitation and during cooler months in hot regions. 7 Indoor crowding in population-dense areas during rainy seasons or cooler months might be one factor that facilitates viral transmission. 8 Additionally, weather-related factors, such as inhalation of cold and dry air that might impair ciliary function, the airway mucosa, and inhibition of temperature-dependent antiviral responses, might infl uence both disease transmission and severity. 9, 10 Altitude, climate, and meteorological conditions (such as wind speed and dew point) have been shown to have a modest association with bronchiolitis. 11, 12 Furthermore, air pollutants, such as ozone and traffi c pollutants, have been associated with exacerbations of respiratory infections in children younger than 5 years. [13] [14] [15] Environmental tobacco smoke has been associated with increased risk for respiratory syncytial virus-attributable admission to hospital and disease severity in those admitted. 16, 17 As with other respiratory viral infections, the risk of severe respiratory syncytial virus bronchiolitis might be greater in boys than in girls. 18, 19 This diff erence might be due to diff erences in lung and airway development, and by genetic factors. 8, 20 Pathophysiology Bronchiolitis is characterised by extensive infl ammation and oedema of the airways, increased mucus production, and necrosis of airway epithelial cells. 21 Respiratory syncytial virus binds to epithelial cells and replicates, resulting in epithelial necrosis and ciliary destruction. 21, 22 The cell destruction triggers an infl ammatory response with proliferation of polymorphonuclear cells and lymphocytes. The submucosa and adventitial tissues become oedematous with increased mucus secretion. 22 Plugs composed of cellular debris and mucus form in the bronchiole lumens leading to bronchiolar obstruction, air trapping, and diff erent degrees of lobar collapse. 21 Microbiology Molecular testing has led to an improved understanding of the viruses associated with bronchiolitis. Respiratory syncytial virus remains the most commonly identifi ed virus, detected in 41-83% of patients. [23] [24] [25] [26] [27] [28] Other viruses associated with bronchiolitis include rhinovirus, metapneumovirus, coronavirus, human bocavirus, infl uenza virus, adenovirus, and parainfl uenza virus. 24, 25, 27, 29, 30 Studies have investigated whether severity of illness, as measured by need for hospital admission, length of hospital stay, intensive care unit admission, repeated emergency department visits, and apnoea, is associated with specifi c viral infections or co-infections, but the evidence is confl icting. Data from some studies have shown that in infections involving a single virus, respiratory syncytial virus is associated with a more severe course compared with other viruses. 31, 32 Up to 30% of children with bronchiolitis are found to have co-infections with two other viruses, 29 with the combination of respiratory syncytial virus and rhinovirus being the most commonly reported. 29 Some evidence suggests that co-infection in bronchiolitis, particularly respiratory syncytial virus in combination with rhinovirus or metapneumovirus, could be associated with a more severe disease course compared with infection by a single virus. 25, 30, 31, 33, 34 However, other studies do not confi rm this association. 24, 26 Furthermore, although use of nucleic acid amplifi cation tests has greatly improved our ability to detect viruses present in respiratory infections, studies using these technologies have also found at least one respiratory virus in up to 30% of children younger than 6 years with no respiratory symptoms. [35] [36] [37] These viruses might be detected because of asymptomatic colonisation, incubation before clinical infection, or prolonged viral shedding post-infection. The confl icting evidence and high prevalence of respiratory viruses in asymptomatic children suggest no indication at this time that management should vary based on presumed viral cause and presence, or absence of viral co-infections.\\nThe diagnosis of bronchiolitis is clinical and thus requires a clinician to recognise signs and symptoms of viral lower respiratory tract infection in young children. Peak incidence occurs between 3 months and 6 months of age. 38 Since the early clinical defi nition by Court, and as noted in recent practice guidelines, the most specifi c defi nition is in infants. [38] [39] [40] Although the same physiology can occur in toddlers older than 12 months, many clinical trials have excluded these children or have included them as a small subgroup of patients. Bronchiolitis in toddlers can overlap with other conditions such as viral-induced wheezing and asthma, and application of evidence from trials predominantly assessing infants might not be appropriate. Further eff orts to focus the defi nition might assist eff orts to standardise care.\\nThe classic clinical presentation of bronchiolitis starts with symptoms of a viral upper respiratory infection, such as nasal discharge, that progress to the lower respiratory tract over several days (fi gure 1). Timing of symptom progression can vary, and young infants can present with apnoea. Lower respiratory tract symptoms of bronchiolitis include persistent cough, tachypnoea, and increased work of breathing, as shown by intercostal or supraclavicular retractions, use of abdominal muscles, grunting, or nasal fl aring. Auscultatory fi ndings include crackles and wheeze. A hallmark characteristic of bronchiolitis is the minute-to-minute variation in clinical fi ndings, as mucus and debris in the airways are cleared by coughing or as the child's state changes from sleep to agitation. This variation can confound assessment, and often requires several examinations over a period of observation. Nasal congestion can also confound the clinical assessment. Nasal suctioning might help to ascertain which fi ndings are truly from the lower airways. Fever can be present in about a third of infants with bronchiolitis, but it is usually present early in the illness with a temperature less than 39\\u00b0C. 38 The median duration of symptoms is about 2 weeks, with 10-20% of infants still having symptoms at 3 weeks after onset. 38, 41 Various clinical scores have been shown in studies and clinical protocols to correlate with disease severity and improvement. 42, 43 Although documentation of a score can be useful as an objective measure, individual scores are not highly predictive, and they should be repeated and combined with other measures of severity for a universal assessment to guide decision making. The diff erential diagnosis for bronchiolitis includes considerations of various infectious and non-infectious causes. Absence of upper respiratory symptoms should raise suspicion of other causes of respiratory distress in young infants, including cardiac disease, congenital airway abnormalities such as a vascular ring, or foreign body aspiration. Other infections can resemble or complicate bronchiolitis. Pertussis should be considered in infants with severe or paroxysmal cough, or with known exposure. Bacterial infections complicating viral bronchiolitis, including otitis media or pneumonia, might present as a new fever or worsening status later in the course of illness.\\nVarious risk factors have been associated with progression to severe bronchiolitis. Those supported by the strongest evidence include presence of chronic lung disease of prematurity and haemodynamically important congenital heart disease, with immunodefi ciency and neuromuscular disorders also considered as high risk in practice guidelines. 38, 39 Young infants (aged <2-3 months) and those with a history of premature birth (especially <32 weeks' gestation) are also at high risk for progression and can present with apnoea without other clinical fi ndings. Studies assessing the risk for further apnoea in hospital have found it to be limited to infants less than 1 month for full-term infants, 48 weeks postconceptional age for preterm infants, or those with apnoea observed before admission. 44\\nBronchiolitis is a clinical diagnosis based on history and physical examination, according to consensus across national guidelines. For infants with a typical presentation of bronchiolitis, routine imaging or laboratory testing is not recommended, as they increase costs without evidence for benefi t (table).\\nAppropriate use of pulse oximetry monitoring and initiation of oxygen for bronchiolitis have received increasing attention in studies and practice guidelines. Findings suggest that arbitrary thresholds for oxygen administration might drive hospital admissions and prolong hospital length of stay. These outcomes represent only part of the morbidity of bronchiolitis, but developing evidence suggests that intermittent hypoxaemia might occur commonly in otherwise stable infants with bronchiolitis and raises questions as to whether this factor should be used as a sole indication for admission to hospitals. A Canadian randomised trial found reduced admissions to hospital from the emergency department without any increase in revisits when pulse oximeters displayed values 3% higher than the actual value, suggesting that arbitrary pulse oximetry thresholds result in unnecessary admissions. 51 A similar trial in the UK in the hospital setting found that reduction of the oxygen threshold from 94% to 90% resulted in earlier discharge from the hospital without any evidence of adverse outcomes. 52 A US trial comparing intermittent versus continuous pulse oximetry in non-hypoxaemic infants in hospitals found similar outcomes between the groups. 53 This US trial and other evidence supports recommendations in US practice guidelines that clinicians use a threshold of 90% for initiation of oxygen, whereas UK guidelines recommend 92%. 38, 39 As the child improves, reduction in intensity of monitoring to intermittent checks is appropriate. A recent study using blinded oximetry at home showed that a substantial proportion of infants with bronchiolitis who are otherwise doing well have oxygen desaturations less than 90%, particularly during sleep, further calling into question arbitrary thresholds for hospital admissions and initiation of oxygen. 54 This evidence will probably lead to eff orts to reduce continuous monitoring in children without other indications for monitoring.\\nThe majority of children with bronchiolitis have either normal radiographs or radiographic fi ndings consistent with simple bronchiolitis, 55-58 including peribronchial thickening, hyperinfl ation, and atelectasis. One prospective study 58 of routine radiographs as part of the assessment for bronchiolitis in the emergency department reported airspace disease in 17 (7%) of 246 patients. Despite the low prevalence of radiographic pneumonia, this and other studies have reported an increase in antibiotic prescription after radiographs are performed, because of non-specifi c fi ndings that infl uence clinicians' decisions. 58, 59 Factors that have been associated with defi nite focal infi ltrates consistent with pneumonia include hypoxia (oxygen saturation <92%), [56] [57] [58] 60 grunting, persistently focal crackles, and fever (especially >39\\u00b0C). 38 Chest radiographs should only be considered in patients when the presentation is not classic for bronchiolitis. These situations include when another diagnosis (such as foreign body aspiration) is high on the diff erential diagnosis, when a child is severely ill and respiratory failure is imminent, and when symptoms are progressing or not resolving according to the typical disease course expected for bronchiolitis. Lung ultrasound is increasingly used to assess cardiopulmonary conditions in adults and children. Several studies have investigated the use of lung ultrasound in the diagnosis of bronchiolitis. Two small studies found that ultrasound fi ndings in infants with bronchiolitis correlate with clinical fi ndings, and might be more specifi c than chest radiography, 61,62 but further studies are needed to establish whether there is a role for ultrasound in diagnosis or assessment of severity.\\nWith the development of PCR to detect respiratory viruses in the nasopharynx, interest in the use of viral testing for causative diagnosis in bronchiolitis has increased. Virological testing, however, does not generally assist in management and is insuffi cient to predict outcomes. 30, 63 Many national guidelines therefore recommend against routine virological testing in bronchiolitis (table). Recent studies suggest that higher respiratory syncytial virus genomic load, measured using quantitative PCR, might be associated with increased length of stay, use of respiratory support, and need for intensive care, in addition to recurrent wheezing, compared with lower viral loads. 28, 31, 32, 64 Further study is warranted to confi rm this association and clarify whether viral load measurement improves understanding of disease pathophysiology and severity. Several guidelines recommend using respiratory syncytial virus testing to guide cohorting of patients; however, the viruses most likely to cause bronchiolitis are all transmitted in a similar fashion (close contact with large-particle aerosols or direct contact with contaminated fomites). [65] [66] [67] Thus, infection control might not be dependent on the identifi cation of specifi c viruses, but rather on following strict precautions including hand hygiene, separating infants in shared hospital rooms by more than 1 m, and other infection control procedures. 65 Additionally, given the sensitivity of PCR testing, results should be interpreted with caution. Certain viruses, such as rhinovirus, might be detected because of viral shedding from an unrelated illness or colonisation; whereas certain other viruses, such as respiratory syncytial virus and metapneumovirus, are almost always associated with an acute infection.\\nBlood and urine testing is not routinely recommended as part of standard practice in the diagnostic work-up of bronchiolitis (table) . A blood gas measurement should not be routinely obtained in infants with bronchiolitis, unless there are signs of impending respiratory failure or severe distress. Proportions of serious bacterial infections, especially bacteraemia and meningitis, are very low in infants with bronchiolitis. 68 Abnormal white blood cell count is rarely useful in predicting serious bacterial infections in children infected with respiratory syncytial virus. 69 Guidelines universally do not recommend complete blood counts in infants with bronchiolitis unless blood count is part of assessment for a fever in infants younger than 1-2 months. Similarly, given that bacteraemia is exceedingly rare (with cited proportions of <0\\u00b71% in the post-pneumococcal vaccine era), blood cultures should not be routinely performed, except in the septic work-up of infants younger than 1-2 months, 70, 71 or in those with severe illness and signs of sepsis. Hydration status is an important consideration in infants with bronchiolitis and should be determined by clinical examination. Routine measurement of serum electrolytes is of little value in the majority of infants.\\nUrinary tract infections in infants with bronchiolitis occur with greater frequency than do bacteraemia and meningitis, with proportions ranging from 1% to 7%. 68, 71, 72 It is reasonable to obtain a urinalysis and urine culture for infants aged less than 60 days with fever and for older febrile infants who have risk factors for urinary tract infections, 73 but urine should not be routinely obtained in all infants with bronchiolitis.\\nCurrent recommendations for management of bronchiolitis focus on agents to treat the patho physiological eff ects of viral lower respiratory infection (eg, bronchodilators and hypertonic saline). Specifi c antivirals such as ribavirin to treat respiratory syncytial virus infection are not recommended in practice guidelines for typical cases of bronchiolitis because of challenging delivery methods, high cost, and potential health risks to caregivers. Multiple new agents for prevention and treatment are under investigation and might become available in the future, including immunoglobulins, small interfering RNA interference, fusion inhibitors, and small molecules. 74 Numerous studies have assessed the role of bronchodilators for the treatment of bronchiolitis, and systematic reviews have found no consistent benefi t. A 2014 Cochrane Collaboration systematic review identifi ed 30 studies assessing bronchodilators, predominantly salbutamol and excluding epinephrine, and 21 studies that looked specifi ally at clinical scores found no evidence of benefi t in any outcomes for infants admitted to hospitals. 75 In outpatients, oxygen saturation, admission to hospital, or time to resolution of symptoms did not improve with bronchodilator usage compared with placebo. For outpatient studies assessing short-term change in pooled clinical scores, the reviewers found a small signifi cant diff erence in mean score (Z=2\\u00b726; p=0\\u00b7024) that was of small eff ect with minimal clinical importance (fi gure 2). Outpatient studies were heterogeneous (I\\u00b2=81%; p<0\\u00b700001), and those showing benefi t in scores tended to include older children and children with recurrent wheezing.\\nNebulised epinephrine was assessed in another Cochrane Collaboration systematic review. 76 This review found no benefi t for epinephrine compared with placebo for inpatients in hospital length of stay or other outcomes. A multicentre Scandinavian study published after this Cochrane review found that inpatients receiving standing doses of epinephrine had longer length of stay compared with inpatients receiving as-needed epinephrine or placebo. 77 For outpatients, the Cochrane review found a diff erence in the numbers of admissions associated with epinephrine treatment during the time of an emergency department visit, but not during the overall course of illness when assessed at 1 week. Clinical practice guidelines including those from the USA, UK, and Canada do not recommend treatment with bronchodilators for bronchiolitis because of this evidence (table) . 38 \\nNebulised hypertonic saline is thought to reduce airway oedema, decrease mucus plugging, improve mucociliary clearance, and rehydrate the airway surface liquid in infants with bronchiolitis. 78 These physiological changes are extrapolated from the cystic fi brosis literature, 79, 80 and the pathophysiological processes in acute bronchiolitis are diff erent. Therefore, the theoretical benefi ts of hypertonic saline seen in cystic fi brosis might not be present in infants with acute viral bronchiolitis. Although initial trials demonstrated some ability of hypertonic saline to decrease hospital length of stay and transiently improve clinical severity score, 81-83 more recent trials demonstrated confl icting results. [84] [85] [86] [87] [88] [89] [90] The trials that showed the largest benefi t were done in hospitals with lengths of stay more than 72 h; thus, hypertonic saline for infants in countries and institutions in which the length of stay approaches or exceeds 72 h might be benefi cial at reducing length of stay. The confl icting results are \\nTest for heterogeneity in the inpatient studies demonstrated low inconsistency between the nine studies (I\\u00b2=36%; p=0\\u00b713) and the summary eff ect was not signifi cant (Z=1\\u00b706; p=0\\u00b729), the outpatient studies demonstrated very high inconsistency between the 12 studies (I\\u00b2=81%; p<0\\u00b700001) and the summary eff ect was signifi cant (Z=2\\u00b726; p=0\\u00b7024), and the overall heterogeneity of the meta-analysis demonstrated high inconsistency between the 21 studies (I\\u00b2=73%; p<0\\u00b700001) and the overall summary eff ect was signifi cant (Z=2\\u00b74; p=0\\u00b7016). IPR=ipratropium. SAL=salbutamol. neb=nebulised. Reproduced from Gadomski and Scribani, 75 refl ected in the diff erences in recommendations across national guidelines (table), with some countries not recommending hypertonic saline, some recommending use for all inpatients, and some recommending use only in moderate to severe illness. The largest systematic review and meta-analysis, published in 2015, examined 24 trials and 3209 patients. 91 Infants receiving hypertonic saline had a signifi cant diff erence in hospital length of stay of -0\\u00b745 days (95% CI -0\\u00b782 to -0\\u00b708; p=0\\u00b701) compared with those receiving 0\\u00b79% saline or standard care. In seven trials, hypertonic saline reduced the risk of admission to hospital by 20% (risk ratio 0\\u00b78; 95% CI 0\\u00b767-0\\u00b796) compared with 0\\u00b79% saline. No substantial adverse eff ects of hypertonic saline were noted in this systematic review. Although the results of this meta-analysis were signifi cant, attention should be paid to the outer limits of the confi dence intervals, which approach no diff erence, and to the clinical relevance of these diff erences. A second 2015 meta-analysis of 15 trials and 1922 patients found a smaller, but signifi cant, decrease in length of stay by -0\\u00b736 days (95% CI -0\\u00b75 to -0\\u00b722; p<0\\u00b7001) in those who received hypertonic saline (fi gure 3). 92 This study found a discrepancy between the overall combined eff ect of all studies on length of stay and the negative results from the largest trials, allowing the authors to conclude that neither individual trials nor pooled estimates provide a strong evidence-based foundation for the use of hypertonic saline. Both meta-analyses showed substantial heterogeneity across studies (I\\u00b2=82\\u00b71%, p<0\\u00b7001; 91 and I\\u00b2=77\\u00b78%, p=0\\u00b7029 92 ). A recent reanalysis of the fi rst 2015 meta-analysis removed two outlying Chinese studies and accounted for imbalances in day of illness at presentation. 93 These analyses resolved the heterogeneity and found that hypertonic saline does not reduce length of stay in infants admitted to hospitals with bronchiolitis (mean diff erence in length of stay removing outliers -0\\u00b721 days; 95% CI -0\\u00b743 to 0\\u00b702; mean diff erence in length of stay accounting for day of illness imbalance 0\\u00b702 days; 95% CI -0\\u00b714 to 0\\u00b717). Large trials have not demonstrated benefi t for hypertonic saline and this meta-analysis found no eff ect of hypertonic saline on length of stay after adjustment for outliers and imbalances. The decision to undertake future trials is controversial given the positive results of some metaanalyses and negative results of others.\\nMultiple studies have examined the role of corticosteroids in the management of children with bronchiolitis. Data from two large multicentre trials have shown no benefi t to corticosteroids alone in reducing admissions to hospital, 94, 95 and a 2013 Cochrane Collaboration review supports the results of these studies. 96 This review included only studies that enrolled children younger than 24 months with a fi rst episode of wheezing and signs of a viral illness. Among the included eight outpatient studies (1824 participants) comparing corticosteroids with placebo there was no reduction in admission at day 1 (Z=1\\u00b705; p=0\\u00b73) or day 7 (Z=1\\u00b738; p=0\\u00b717) after enrolment (fi gure 4), clinical scores, length of stay in the emergency department, or length of time to resolution of symptoms. Among the nine inpatient studies (772 participants), length of hospital stay was not reduced. 96 On the basis of this evidence, multiple clinical practice guidelines recommend against the use of corticosteroids for infants with bronchiolitis (table) . Although clinicians report considering a family or personal history of atopy when deciding whether to treat infants with bronchiolitis with corticosteroids, 41 there is no evidence that such infants receive any benefi t from corticosteroid treatment. [94] [95] [96] Evidence for the presence or absence of respiratory syncytial virus infection in these infants being associated with a response to corticosteroids is also unavailable. 94, 95 Authors from a large study with a factorial design have suggested, in an unadjusted analysis, that high-dose corticosteroids in combination with nebulised epinephrine might reduce admissions for outpatients with bronchiolitis by day 7, 95 but these results are considered exploratory. 39, 45 High-fl ow oxygen and respiratory support\\nNon-invasive technologies to improve oxygenation and ventilation for bronchiolitis include humidifi ed high-fl ow nasal cannula oxygen and continuous positive airway pressure. 97 High-fl ow nasal cannula allows delivery of high fl ows (usually 1-2 L/kg per min) with humidifi cation and a cannula designed to improve patient tolerance. It has been used widely in premature infants, but the mechanisms of action are unclear, in particular whether it might deliver positive end-expiratory pressure in some conditions. Evidence for effi cacy of high-fl ow nasal cannula is predominantly observational, with studies documenting Corneli (2007) Goebel (2000) Kuyucu (2004) Plint (2009) improved respiratory parameters and reduced intubation rates after implementation. 97 One small randomised trial compared high-fl ow nasal cannula with hypertonic saline and found no diff erence in change in respiratory score. 98 Concerns about high-fl ow nasal cannula include the potential for rapid deterioration if the infant is not closely monitored and costs associated with overuse. Continuous positive airway pressure has been studied in intensive care settings in observational studies and several small trials, with some evidence of improved respiratory parameters. 97 The UK guidelines recommend considering continuous positive airway pressure in children with impending respiratory failure from bronchiolitis.\\nAntibiotic overuse in children with bronchiolitis probably occurs because of concerns about the presence of fever, the young age of aff ected patients, diffi culty diff erentiating atelectasis from infectious consolidation on chest radiograph, and concern for undetected secondary bacterial infection. Bronchiolitis, however, has a clear viral cause and the occurrence of secondary bacterial infections is low, with a risk of bacteraemia or meningitis of less than 1%. 99 A detailed review of randomised clinical trials found that routine use of antibiotics did not improve duration of symptoms, length of hospital stay, need for oxygen therapy, or hospital admission. 38 Overuse of antibiotics is known to result in unnecessary adverse eff ects on the patient, and the development of antimicrobial resistance. Routine use should be avoided unless there is clear evidence of a secondary bacterial infection (table) . Acute otitis media has been documented in up to 60% of infants with bronchiolitis. 100, 101 Antibiotic use for acute otitis media in bronchiolitis should follow established evidence and guidelines for acute otitis media. 102 Macrolide antibiotics have anti-infl ammatory properties that might have potential benefi t in mitigating the infl ammation present in bronchiolitis. Two randomised trials found that there was no diff erence between azithromycin and placebo in hospital length of stay, need for oxygen, or hospital re-admission. 103, 104 Another randomised trial found that azithromycin lowered nasal interleukin-8 concentrations, prolonged time to subsequent wheezing episodes, and resulted in fewer days with respiratory symptoms in the year following the bronchiolitis episode compared with placebo. 105 Finally, a US multicentre study found that in children aged 12-72 months with a history of recurrent severe lower respiratory tract infections, early administration of azithromycin during a lower respiratory tract infection reduced the likelihood of progression to a severe infection, but it is not clear whether the underlying disease in these children was bronchiolitis or some other disease process. 106 Given the current evidence, routine use of macrolides is not recommended in bronchiolitis and more research is needed to clarify any potential role it might have in the future.\\nHydration, suctioning, and chest physiotherapy have been suggested as supportive therapies. Infants with bronchiolitis might have diffi culty feeding because of nasal congestion and increased work of breathing; thus, hydration remains a cornerstone of therapy. A multicentre study of 759 infants younger than 12 months admitted to hospital with bronchiolitis showed no benefi t of intravenous fl uids compared with administration of fl uids by nasogastric tube in mean length of stay, admission to the intensive care unit, need for ventilatory support, and adverse events. This trial also found that a nasogastric tube might be easier to place than an intravenous line in these infants. 107 Most guidelines recommend either nasogastric or intravenous fl uids to maintain hydration, with the UK and Scottish guidelines preferring nasogastric or orogastric hydration in those that can tolerate it compared with intravenous hydration (table). If intravenous fl uids are used, isotonic fl uids are preferred to avoid risk of hyponatraemia. 38 Because infants are obligate nasal breathers, nasal suctioning has been suggested to help with clearing of the nares, improve the work of breathing, and improve feeding; however, suctioning might irritate the nasal mucosa and result in oedema. No randomised controlled trials have examined the role of nasal suctioning in bronchiolitis. The insuffi cient available evidence includes a retrospective cohort study of 740 infants 108 and a small observational study of 40 infants. 109 These studies suggest that deep suctioning might increase length of stay for inpatients, 108 infrequent suctioning is associated with an increased length of stay, 108 and oxygen saturation might increase after suctioning. 109 To draw conclusions about causality from these observational studies is diffi cult, because the potential for confouding by indication exists (eg, sicker children might be more likely to receive deep suctioning). Evidence suggests that oxygen saturation increases after nasal irrigation even without suctioning. 110 Current guidelines give diff ering recommendations with regard to suctioning; those that support its use recommend only superfi cial suctioning rather than deep suctioning (table) . 38, 39, 45 Chest physiotherapy use in bronchiolitis appears to vary by country. 111, 112 A recent Cochrane Collaboration review of 12 studies (1249 participants) demonstrated no evidence of benefi t to any type of chest physiotherapy among inpatients in length of stay, oxygen saturation, or respiratory parameters. 113 No published guidelines routinely recommend chest physiotherapy for the management of uncomplicated bronchiolitis in otherwise healthy children without respiratory comorbidities (table) . 38, 39, 45 Prognosis Much work has been published about the risk of developing recurrent wheezing and asthma following bronchiolitis in infancy. [114] [115] [116] Studies have followed birth cohorts to determine the risk of subsequent wheezing after lower respiratory tract infection in young childhood, and cohorts of children admitted to hospital with bronchiolitis. [114] [115] [116] [117] Overall, admission to hospital with bronchiolitis at a young age is associated with an increased risk of recurrent wheezing. Studies report that 17-60% of children with bronchiolitis might develop recurrent wheezing in the years following their initial admission to hospital. A large study from Taiwan that followed up 1981 children admitted with bronchiolitis before age 3 years found that by age 10 years, 351 (17\\u00b77%) of 1981 children with bronchiolitis had a diagnosis of asthma compared with 2159 (11\\u00b77%) of 18 527 controls (hazard ratio 1\\u00b758; 95% CI 1\\u00b741-1\\u00b771). 118 One small cohort study of 138 patients has suggested that 18 (39%) of 46 children admitted with bronchiolitis before 12 months have asthma by 18 years compared with eight (9%) of 92 controls. 115 However, another study that followed a birth cohort of 1246 children found that although lower respiratory tract infection in childhood was associated with an increased risk of recurrent wheezing, this association decreased with age and was not signifi cant by age 13 years. 116 Most children in this cohort had mild illness not requiring hospital admission, and severity of illness might be associated with the increased risk of asthma. 119 The question remains whether respiratory infection at a young age itself predisposes children to asthma through damage or alteration of lung function, or whether children with severe bronchiolitis might have individual risk factors (such as altered immune response or airway function) that predisposes them to both severe bronchiolitis and recurrent wheezing. 8\\nSubstantial knowledge gaps and controversies exist in the management of acute bronchiolitis. The role of nebulised hypertonic saline in acute management is not clear, resulting in confl icting recommendations across clinical guidelines (table) . Although meta-analyses suggest a small reduction of length of stay, these analyses are limited by heterogeneity, not accounting for duration of illness, and not considering the role of outlying study populations. 91 Two large multicentre trials do not support a clinically signifi cant diff erence in length of stay for inpatients and there is no clear evidence of cost benefi t. 4, [89] [90] [91] 120 While the same meta-analysis suggests a possible reduction in admissions for outpatients, the confi dence intervals were wide and the studies were also heterogeneous. 91 No multicentre studies of nebulised hypertonic saline have been completed in outpatients. Although the evidence is increasing that hypertonic saline has little role in meaningfully reducing the length of stay, its role in outpatients is less clear. A large outpatient multicentre study could clarify whether there is any benefi t.\\nEvidence also suggests that combined therapy with nebulised epinephrine and corticosteroids might reduce admissions to hospital. 95 Synergy between corticosteroids and \\u03b22 agonists is well documented in clinical trials of asthma management. [121] [122] [123] [124] Basic science literature also shows that \\u03b22 agonists and corticosteroids enhance each other's eff ectiveness, particularly with regard to antiinfl ammatory gene expression. 125, 126 Because of the economic burden of bronchiolitis and the plausible basic and clinical evidence for synergy, a large and multicentre trial is needed to ascertain whether combined therapy with epinephrine and corticosteroids is benefi cial.\\nOxygen saturation and the use of pulse oximetry play an important role in the decision to admit infants with bronchiolitis to hospital and in the length of their hospital stay. [51] [52] [53] [54] [55] [56] [57] [58] [59] Clinical practice guidelines also give confl icting guidance on the level of oxygen saturation at which admission should be considered. Furthermore, a substantial proportion of discharged infants have episodes of transient desaturation. 54 Again, in view of the large health-care costs associated with hospital admission in bronchiolitis, further research is needed to clarify the level of oxygen saturation requiring admission, the role of continuous and spot measurements of oxygen saturation, and the clinical importance of transient desaturations in otherwise stable young infants.\\nTAF contributed to the design and coordinated the writing of this manuscript. All authors contributed to the literature search and writing of this manuscript.\\nWe declare no competing interests. TAF, ACP, and JJZ have participated in trials of bronchiolitis funded by unrestricted public or academic institutional funding sources that had no infl uence on the conception and writing of this manuscript.\",\n          \"In March and April 2009, health professionals warned the Mexican Ministry of Health about the increase in incidence of acute respiratory illness in young adults [1] . In April, a new viral subtype of influenza A(H1N1)pdm09, a combination of swine, avian and human virus, was identified in the United States and in samples originated from Mexico [2, 3] . On April 25 th , the The patients were identified in the SINAN database, using the following variables for selecting the cases and controls: RT PCR (positive for influenza A(H1N1)pdm2009), final classification (confirmed), criteria (laboratory), evolution (recovered or death), hospitalization (yes), date of hospitalization (28/06/2009 to 29/08/2009), municipalities of residence and hospitals (metropolitan areas of Sao Paulo and Campinas).\\nAt the time the patients were selected for the study, the SINAN included 193 reported and confirmed deaths of patients with influenza A(H1N1)pdm2009 and SARI, all of which were assessed in this study (cases) . For each case of death, two control patients were selected among the 1988 patients hospitalized during the study period with reported and confirmed influenza A(H1N1)pdm2009 and SARI who recovered. The controls were time-paired randomly selected from those hospitalized in the same epidemiological week (EW) or, if necessary, one week later than the date of hospitalization of cases to prevent possible differences in opportunity of treatment and clinical management protocol.\\nData of the study were collected by trained health professionals using two standardized questionnaires. The hospital questionnaire collected information from medical charts, evaluated in 177 hospitals were patients were hospitalized. Of these, 56.5% belonged to the Unified Health System-SUS and 43.5% were private hospitals. The following data were collected: previous pathological history, occurrence of pregnancy, health care, symptomatology during admission, laboratory and radiological tests, need for ICU admission, treatment, complications and disease progression. The International Classification of Diseases, 10 th revision-ICD-10, was used for coding comorbidities indicated in the medical charts. The Ministry of Health provided the antiviral drug known as Oseltamivir for treatment during the pandemic. The protocol for clinical management established that the treatment was for cases hospitalized with SARI and ambulatory patients with risk conditions for severity. Treatment should be started within 48hours after symptom onset. All patients studied had laboratory confirmations of influenza A(H1N1)pdm09 from respiratory secretion samples analyzed using the RT-PCR method, performed by the Adolfo Lutz Institute [16] . The definition of SARI included: fever associated with cough and dyspnea; pneumonia; respiratory insufficiency; tachypnea; radiological changes compatible with pneumonia; oxygen therapy or mechanical ventilation. SARI was confirmed in all cases and controls selected for the study by data their on medical charts.\\nHousehold interviews were conducted in 161 cases (83.4%), with 14 refusals (7.3%) and 18 (9.3%) addresses not found. A total of 337 (85.1%) controls were interviewed, with 6 (1.5%) refusals and 43 (11.1%) addresses not found. The household questionnaire included socio-demographic data, history of previous care related to the episode that led to hospitalization and presence of influenza vaccination. Occupations were grouped according to the occupational pyramid of influenza risk [17] . For cases, interviews were conducted with the closest family members; for controls, with the patients themselves, excluding children, whose parents/legal guardians responded for them.\\nAfter data collection, questionnaires were assessed by supervisors to verify the consistency of information and the fulfillment of criteria to include patients in this study. Hospital and household questionnaires were input to the Sphynx database, version 5.\\nThe present study began in order to assist with decisions on surveillance and control actions. The use of data was approved by the Research Ethics Committee, School of Public Health, University of S\\u00e3o Paulo-FSP-USP (protocol 2283, OF.COEP/312/11). Data collection was performed in accordance with the recommendations of the National Health Council for Human Research, including the signing of an informed consent form by patients, family members or legal guardians.\\nClinical and demographic characteristics are shown as median and inter-quartile intervals or percentages, and comparisons were made using the Mann-Whitney U or chi-square tests, as required. Odds ratio-OR and 95% confidence intervals-95%CI were calculated to assess the factors associated with death. Variables with p-value < 0.20 in the univariate analysis and adjustment variables were selected for the multiple logistic regression, which used forward stepwise strategy. The model including all patients assessed the following dichotomous categorical variables: health insurance, previous care, vaccination against influenza in 2009, sex (reference category = male); and the following risk conditions: diabetes mellitus, obesity, chronic kidney disease, chronic liver disease, immunosuppression, chronic cardiovascular disease, chronic neurological disease and delay in development, using the absence of a condition as a reference category. The model also assessed: age group [<18 years (reference), 18 to 59 years, 60 years], use of antiviral therapy [no use (reference category), 48 hours after the onset of symptoms, > 48 and 72 hours, > 72 hours], high-risk individuals or those with any of the risk conditions listed above adapted from the Centers for Disease Control and Prevention-CDC [18] [no condition (reference category), one condition, and two or more conditions], and time between onset of symptoms and admission in days as the continuous variable. The model was controlled for epidemiologic week of admission at hospital (categorized as EW26-28 (reference), EW29-31 and EW31-34), and income. A second model was developed with patients aged 18 years and over, which also assessed level of education [without formal education or incomplete primary education, complete primary education, complete secondary education, complete higher education (reference category)] and occupational risk [very high and high, average and low (reference category)]. Patients aged 18 to 22 years with incomplete higher education were included in reference category of education. For the modelling, patients with no occupation were included in the reference category of occupational risk. Significance of variables was assessed by Wald test and adjustment of models by Hosmer-Lemeshow test.\\nThe analyses were performed using SPSS software, version 17.0. P-values < 0.05 were considered to be significant. Table 1 presents the socioeconomic characteristics of patients and information associated with the disease obtained from the household interviews. Among patients aged 18 years and over, there was a higher risk of death for those without formal education or with incomplete primary education, OR 2.33(95% CI 1. 13-4.89 ) and complete primary education, OR 3.06 (95%CI 1.41-6.82), when compared to those with higher education. Distribution of ethnic groups, family income and history of smoking were not associated with death. The history of previous vaccination against seasonal influenza in 2009 showed protection against death. Health professionals, considered to be at a high risk of developing influenza, showed protection against death, OR 0.15(95%CI 0.02-0.61). Patients who had sought care prior to hospitalization had a risk of death 3.70 times (95%CI 2.25-6.30) that of patients without previous care, with a predominance of emergency care (45.34%). Table 2 presents the distribution of cases and controls according to variables assessed in the hospital investigation. Median age of cases was higher than that of controls (p<0.001) and the 18-to-59-year age group had an OR for death of 4.03(95%CI 2.62-6.31), compared to those aged less than 18 years. Elderly individuals were also at a significant risk of death.\\nThere were no differences in sex distribution in either group. With regard to risk conditions for severity, 73.6% of cases and 38.1% of controls reported having at least one risk factor, with an OR of 4.49 (95%CI 3.09-6.60). When individuals with more than one of these conditions were assessed, the risk of death increased, OR 10.29 (95%CI 6.04-17.85). The following medical conditions were risk factors for death: obesity, diabetes mellitus, chronic cardiovascular diseases, chronic kidney diseases, neurological diseases and delay in development, chronic liver diseases and immunosuppressive diseases. The immunosuppression-related diseases were: neoplasms (n = 18), HIV/AIDS (n = 8), organ transplants (n = 6), autoimmune diseases (n = 5) and use of immunosuppressants (n = 3).\\nOf the 212 women of reproductive age (10 to 49 years) who were studied, 33.5% were pregnant and there were no differences in the frequency of pregnancy in either group.\\nThe use of antiviral treatment during hospitalization was a protective factor against death, with an OR of 0.67 (95%CI 0.45-0.98). When time between onset of symptoms and beginning of antiviral treatment was assessed, there was an 81% protection among individuals who received treatment within the first 48 hours, with an OR of 0.19 (95%CI 0.11-0.35), and 65% between 48 and 72 hours, with an OR of 0.35 (95%CI 0.16-0.74). After 72 hours, there was no significant protection. Median of days between date of onset of symptoms and date of hospitalization was 5.0 among cases and 2.0 among controls (p<0.001). Among patients with chronic pulmonary disease, median was 4.0 days among cases and 2.0 days among controls; among those with asthma, 4.5 days and 2.0 days; and among pregnant women, 4.0 days and 2.0 days, respectively. With regard to health professionals, the median between the date of onset of symptoms and that of hospitalization was three days for both cases and controls. Median between date of hospitalization and of death or hospital discharge was seven days among cases and four days among controls (Table 2) .\\nAccording to medical charts, fever and cough occurred in 87.8% of cases and 79.3% of controls; and the proportions of dyspnea were 87% and 62.7%, respectively. The definition of cases of Severe Acute Respiratory Syndrome -SARS (fever+cough+dyspnea) based on information obtained from medical charts occurred in 68.9% of cases and 56.5% of controls. Vomit and diarrhea were present in 8% of cases and 18% of controls. Treatment with antibiotics was used in 95.8% of cases and 73.1% of controls. The mean number of antibiotics used was 4.0 and 1.8 respectively. Among cases, there was a higher proportion of hospitalizations in intensive care units (83.4%x16.8%), use of mechanical ventilation (94.8%x10.8%) and dialysis (25.0%x2.3%). The proportion of complications was higher among cases than controls, with an emphasis on: respiratory distress syndrome (64.7%vs4.9%), shock (69.4%vs1.0%), sepsis (63.2%vs2.3%) and kidney complications (44.6%vs1.3%), respectively. Table 3 shows the laboratory findings of entry tests of patients in the hospital. Cases had lower numbers of leukocytes and platelets and higher values of creatine phosphokinase-CPK, lactic dehydrogenase-LDH, glutamic-oxaloacetic transaminase, glutamic-pyruvic transaminase, urea and creatinine than controls, with statistical significance.\\nIn sub-group analysis, previous conditions of risk for severity and obesity were risk factors for death in univariate analysis for children younger than 18 years and women of reproductive age (10 to 49 years). Among children, presence of immunosuppression, chronic cardiovascular disease and neurological disease/delay in development were also risk factors, as well as diabetes mellitus among women. Treatment with antiviral administered within 72 hours after the onset of symptoms was a protective factor for women of reproductive age. Tables 4 and 5 show variables associated with death in the final models of multiple logistic regression, in all patients studied and in sub-group analysis of adults (aged 18 years and over). In patients in general, risk factors for death were: being aged between 18 and 59 years (reference: until 18 years), having one or more risk conditions for the severity of the disease, obesity, immunosuppression and having sought care prior to hospitalization. Although chronic neurological diseases/delay in development and chronic liver disease were highly associated with death, these variables were excluded from the final model because they had estimates with large confidence intervals. Private health insurance and antiviral treatment administered in the first 72 hours after the onset of symptoms were protective factors against death. Among adults aged 18 years and over, the risk factors for death were having more than one risk factor for severity of the disease, obesity and care prior to hospitalization. The protective factors were: antiviral drug administered in the first 48 hours after the onset of symptoms and being a health professional or health unit worker.\\nSensitivity analysis was performed to assess the severity of patients admitted during the study period. The proportion of ICU admissions and use of mechanical ventilation was estimated for epidemiological week of hospitalization, categorized as EW 26 to 28, EW 29 to 31 and EW 32 to 34. The results showed higher proportions of ICU admissions (50.0% x 16.8% x 7.7%) and mechanical ventilation (45.4% x 10.8% x 2.6%) in the three first weeks, among the controls. The cases showed high proportions in all periods analyzed: 89.7% x 84.0% x 76.9% for need of ICU and 89.7% x 97.6% x 89.7% for need of mechanical ventilation, in the 3 periods, respectively. The opportunity for antiviral treatment showed lower proportion of treatment in the first period of the study, in both cases (37.9% x 74.4% x 74.3%) and controls (59.1% x 74.1% x 89.7%). Sensitivity analysis to assess the opportunity for treatment of children and adults showed similar proportions for the groups: 73.9% and 74%, respectively.\\nIn the present study, risk factors for death were analyzed in patients infected with influenza A (H1N1)pdm09 and hospitalized with SARI. There was an increased risk of death in patients aged 18 to 59 years; those with obesity, immunosuppression, neurological and developmental diseases; or those who had received care prior to hospitalization. Additionally, presence of at least one risk factor for aggravation of the disease was also associated with occurrence of death. Use of antiviral drugs, especially when administered in the first 72 hours after onset of symptoms was protective factor against death.\\nThe design of this case-control study, including all deaths reported and confirmed in the region during the study period and the time-paired random selection of controls among patients who survived after hospitalization with SARI, with hospital and household data collection, enabled the analysis of the factors associated with death to be extended, including socioeconomic variables, access to health services, and data on hospitalizations.\\nFew studies have assessed risk factors for death in patients with SARI [15, 19] . Some studies have analyzed risk factors for severity of the disease and deaths, comparing patients hospitalized in ICUs and in wards [13, 14, [20] [21] [22] [23] , whereas one study used the rate of mortality from influenza A(H1N1)pdm09 in two epidemic waves [24] . A study conducted in Australia analyzed risk factors for hospitalization due to influenza, comparing cases with controls without infection in the community [25] . Other studies have analyzed epidemiological surveillance data, including mortality rates and comparisons among deaths, hospitalizations, treatment and other factors [9, 10, 12, [26] [27] [28] [29] [30] [31] [32] [33] [34] [35] . Some of the factors associated with death identified in this study have been described by other authors, such as the presence of comorbidities [31, 35] , age18years [13] , obesity [15, 19, 20] , neurological and developmental diseases [23, 24, 31] and immunosuppression [15, 21, 24, 34] . Children showed lower risk of death, despite a similar proportion of treatment than adults. A similar result was observed in a study of systematic review and metaanalysis, where the risk of death was 0.28 (95% CI 0.19-0.141), when compared to young adults [36] . In contrast, other factors associated with severity of infection that have been reported by different researchers, such as diabetes [19, 34] ; other chronic pulmonary diseases, excluding asthma and chronic obstructive pulmonary disease [20] ; chronic pulmonary disease, including asthma [31] ; and chronic cardiovascular disease [21, 24, 34] , were not associated with mortality in the present study. The population assessed, which was restricted to hospitalized patients with severe influenza could at least partly explain these differences. Diabetes, chronic cardiovascular and kidney disease, associated with death in univariate analysis, did not remain risk factors after multiple analysis. The design of this study, with data collected on medical charts and household interviews, allowed for adjustments in the analysis that can also explain part of these differences.\\nDeaths occurred mainly in adults aged 18 to 59 years, with a small proportion of cases and controls among elderly individuals. A possible preexisting immunity against the new influenza A(H1N1)pdm09 virus in this age group has been suggested, probably due to previous exposure, through other infections or vaccinations, to an influenza A(H1N1) virus which is genetically and antigenically more associated with the new virus than contemporary strains of seasonal H1N1 viruses [37] .\\nAll patients in the present study, whether cases or controls, had a laboratory-confirmed infection of influenza A/H1N1pdm09, which was [38] .\\nIt is possible that the probability of being hospitalized varied throughout the study. Treatment was available for hospitalized patients and for ambulatory patients with risk conditions for severity. This could have influenced hospitalization, but not much in our study population, consisting of patients with SARI. During the pandemic, criteria for hospitalization changed as well as the availability of treatment, especially in the first 3 weeks of the epidemic, as shown in sensitivity analysis, confirming the need for controlling period of hospitalization in the study.\\nThe socioeconomic profile was similar in both groups assessed, with no difference in income or having private health insurance. There was greater protection against death among patients with an occupation with higher risk of exposure to infection, such as health professionals. A greater opportunity of access to treatment among these workers could explain such findings, as the period between the date of onset of symptoms and the date of admission was shorter in this group. The low proportion of deaths among patients categorized as having a higher risk of exposure has also been observed in other studies [30, 33] .\\nPregnancy was not a risk factor for death in this study. In contrast, frequency of pregnant women among those of reproductive age hospitalized with severe manifestation of influenza A (H1N1)pdm09 was high, suggesting that pregnancy could be associated with development of SARI. An association between pregnancy and use of mechanical ventilation during hospitalization due to influenza A(H1N1)pdm2009 [25] and a higher admission rate were reported when compared to the general population [32] . In 2009, in the state of S\\u00e3o Paulo, 27.4% of all confirmed cases of influenza A(H1N1)pdm09 in women of reproductive age (10 to 49 years), hospitalized and recorded in the SINAN database, were found in pregnant women. In Brazil, only 4.27% of women of reproductive age had live newborns in the reference year according to the 2010 Census [39] .\\nPatients who died presented important laboratory changes when compared to controls, with lower levels of leukocytes and platelets and an increase in CPK, LDH, urea and creatinine. Studies revealed a similar trend for the laboratory results of leukocytes [23] , CPK [26, 27, 30] , LDH [26] , platelets [22] and creatinine [27, 30, 34] .\\nAlthough most patients who died received antiviral treatment, only 9.3% were treated in the first 48 hours. Early treatment was a protective factor against death, especially when provided in the first 72 hours after onset of symptoms. Other studies have also shown a more favorable prognostic with antiviral treatment [13, 21, 23, 27, 28, 29, 34] .\\nHistory of care prior to hospitalization was a risk factor for death from influenza A(H1N1) pdm09, which could reflect the difficulty in accessing health services. Emergency services were primarily sought for care prior to hospitalization, which could have contributed to greater delay in diagnosis and beginning of treatment. A study conducted in Mexico showed a similar result with regard to delay in hospitalization [29] .\\nIn one study, which assessed the overall standardized mortality rate from influenza A (H1N1)pdm09, the mortality among patients with preexisting chronic respiratory diseases was lower than those with other chronic diseases 24 . There might have been problems with recording information about previous diseases in the medical charts in the presence of acute pulmonary disease, especially among those with chronic pulmonary disease.\\nThe present study had some limitations. Underreporting of SARI by health professionals is likely to have occurred, as reporting this disease was not compulsory in Brazil before the pandemic. However, during the pandemic, professionals were more sensitive to reporting it. The system for notification, which was mandatory during the epidemic, was facilitated to allow for rapid notifications. The availability of samples from autopsies of cases (deaths) could have resulted in greater access to the laboratory diagnosis of this infection. Household data collection could also have caused information bias, as data on cases were collected through interviews with family members, while data on controls were obtained from the patients themselves, when they were adults. The proportion of refusals of household interviews was higher among cases (7.3%) than controls (1.6%), p = 0.001. Another possible limitation was memory bias, as a result of the time between the hospitalization and interview. However, the information analyzed from household interviews was about socio-demographic characteristics, which is less vulnerable to information and memory bias. Data on preexisting diseases were obtained from medical charts in hospitals, where they are completed at the moment of hospitalization, in the presence of the patients. The quality of completion of medical charts can differ between hospitals. Nonetheless, information about the presence of chronic diseases, clinical management of patients, laboratory results, radiological tests and treatment was available in these charts. The difficulties encountered during data collection were minimized by the use of a standardized questionnaire.\\nIn conclusion, the risk conditions for the severity of the disease, particularly obesity, immunosuppression and neurological/developmental diseases, were also important risk factors for death, emphasizing the need for good vaccination coverage during influenza vaccination campaigns in all patients with chronic diseases belonging to more vulnerable groups. Another relevant result was the antiviral treatment within the first 72 hours of onset of symptoms as a protective factor, also emphasizing the need to warn the population about the importance of seeking early medical care when an influenza-like illness appears, especially among high-risk groups and those with signs of disease aggravation. Training of health professionals, especially physicians, is required for adequate clinical management of patients and early antiviral treatment.\"\n        ],\n        \"semantic_type\": \"\",\n        \"description\": \"\"\n      }\n    },\n    {\n      \"column\": \"authors\",\n      \"properties\": {\n        \"dtype\": \"string\",\n        \"num_unique_values\": 1422,\n        \"samples\": [\n          \" Chen, Junbo.  Zhu, Huachen.  Horby, Peter W.. <br>Wang, Qianli.  Zhou, Jiaxin.  Jiang, Hui.  Liu, Liwei. <br>Zhang, Tianchen.  Zhang, Yongli.  Chen, Xinhua.  Deng,<br>Xiaowei.  Nikolay, Birgit.  Wang, Wei.  Cauchemez, Simon. <br>Guan, Yi.  Uyeki, Timothy M..  Yu, Hongjie\",\n          \" Chen, Yu.  Dong, Yuanji.  Cai, Shaozhe.  Ye, Cong. <br>Dong, Lingli\",\n          \" Arens, Amanda.  Scott, Cheryl.  Osburn, Bennie\"\n        ],\n        \"semantic_type\": \"\",\n        \"description\": \"\"\n      }\n    },\n    {\n      \"column\": \"title\",\n      \"properties\": {\n        \"dtype\": \"string\",\n        \"num_unique_values\": 1454,\n        \"samples\": [\n          \" Peptide Vaccine: Progress and Challenges\",\n          \" The Impact of Ebola Virus Disease on Government<br>Expenditure in Sierra Leone\",\n          \" The clinical implication of dynamic<br>neutrophil to lymphocyte ratio and D-dimer in COVID-19: A<br>retrospective study in Suzhou China\"\n        ],\n        \"semantic_type\": \"\",\n        \"description\": \"\"\n      }\n    },\n    {\n      \"column\": \"journal\",\n      \"properties\": {\n        \"dtype\": \"string\",\n        \"num_unique_values\": 785,\n        \"samples\": [\n          \"AMB Express\",\n          \"Dail and Hammar&#x02019;s Pulmonary Pathology\",\n          \"Benumof's Airway Management\"\n        ],\n        \"semantic_type\": \"\",\n        \"description\": \"\"\n      }\n    },\n    {\n      \"column\": \"abstract_summary\",\n      \"properties\": {\n        \"dtype\": \"string\",\n        \"num_unique_values\": 1044,\n        \"samples\": [\n          \" InnoMetrics is the system that aims at<br>collecting software development process metrics in a<br>non-invasive way to access and optimize the development<br>process and its efficiency. This paper demonstrates<br>the development and analysis of energy<br>consumption of MacOS systems based on the software process<br>measurement data. It represents the experience of the<br>development of MacOS system collector and Transfer, in<br>addition to the user interface and early analysis of<br>energy consumption metrics calculations.\",\n          \" During the global outbreak of severe acute<br>respiratory syndrome (SARS) in 2003, treatment was<br>empiric. We report the case history of the index patient<br>in a hospital outbreak of SARS in Hong Kong. The<br>patient recovered after conventional antimicrobial<br>therapy. Further studies are needed to address<br>treatment of SARS, which has high attack and death rates.\",\n          \" Intestinal transplantation (IT) is the least<br>common form of organ transplantation; however, it has<br>shown exceptional growth and improvement in graft<br>survival rates over the past two decades mainly due to<br>better outcomes achieved during the first year of<br>transplantation (76 % at 1 year), due to improvement in surgical<br>techniques and the development of better<br>immunosupressive therapies as we understand more about the<br>relationship between the recipient and host immune system.<br>There are still ongoing issues with chronic<br>rejection and long-term survival. Intestinal<br>transplantation is still an acceptable therapy for patients<br>with intestinal failure (IF), but it is generally...\"\n        ],\n        \"semantic_type\": \"\",\n        \"description\": \"\"\n      }\n    },\n    {\n      \"column\": \"cleaned_text\",\n      \"properties\": {\n        \"dtype\": \"object\",\n        \"semantic_type\": \"\",\n        \"description\": \"\"\n      }\n    },\n    {\n      \"column\": \"cleaned_text_str\",\n      \"properties\": {\n        \"dtype\": \"string\",\n        \"num_unique_values\": 1453,\n        \"samples\": [\n          \"respiratory tract infection common infants young children major public health problem age group 1 mainly caused infection seasonal viruses respiratory syncytial virus rsv rhinovirus 2 3 4 infections manifest upper respiratory infection 20 40 develop lower respiratory infection small number 2 -3 infected infants require hospitalization 5 6 particular concern infants lower respiratory viral infection increased risk developing recurrent wheezing childhood asthma 7 8 9 10 11 12 13 risk factors identified development severe lower respiratory viral infections premature birth young age born relation rsv season underlying chronic lung disease congenital heart disease high parity young maternal age poor socioeconomic factors 14 15 16 recently genetic polymorphisms identified associated severe viral infection 17 18 identified genetic variants associated severe lower respiratory tract infections associated pathways common asthma including innate immune genes involved cytokine chemokine signaling epithelial cell function 13 17 19 20 21 22 23 \\n clinical hallmark asthma wheezing therapy includes beta agonists number clinical studies past 2 decades examined relationship genetic variants \\u03b2 2 -adrenergic receptor adrb2 asthma 24 25 26 27 28 29 30 polymorphism resulted gly16 allele associated decreased bronchodilator responses early clinical studies larger clinical studies provided equivocal results 31 32 33 genetic variants promoter region adrb2 associated asthma severity 28 34 despite significant interest adrb2 polymorphisms related asthma relative contribution adrb2 genetic variants severity respiratory viral infections established wheezing clinical hallmark lower respiratory viral infections variable response bronchodilators demonstrated infants hospitalized rsv lower respiratory viral infections possible adrb2 genetic variants contribute severity acute respiratory viral infections 35 36 \\n aim present study explore associations genetic variants adrb2 associated response asthma therapy acute asthma exacerbation severity infant acute respiratory viral infections examined relationship 5 common polymorphisms promoter region coding block adrb2 gene infant severity respiratory infections prospective cohort caucasian african american infants \\n order examine association genetic variants regulatory regions adrb2 gene severity viral respiratory infections infancy conducted analysis infants enrolled tennessee children respiratory initiative tcri prospective cohort mother-infant dyads enrolled infant acute viral respiratory illness respiratory viral seasons 2004 2008 term \\u226537 weeks non-low birth weight \\u22652275 g infants healthy significant co-morbidities cardio-pulmonary disease eligible oversampling hospitalization infants eighty percent enrolled subjects respiratory virus identified polymerase chain reaction rsv rhinovirus influenza parainfluenza human metapneumovirus coronavirus 56 labconfirmed rsv infection rationale methods detailed inclusion exclusion criteria reported previously 37 630 infants enrolled tcri study limited 261 caucasian 113 african american infants dna available protocol approved institutional review board vanderbilt university mothers provided informed consent infants parents provided written informed consent child study participation \\n infants included study upper respiratory tract infections lower respiratory tract infections severity acute respiratory infection determined experienced research nurses medical record chart review ordinal bronchiolitis severity score bss score aggregate assigned values ranging 0 3 categories respiratory rate room air oxygen saturation presence extent wheezing presence extent flaring retractions 37 38 39 bss ranges 0 12 higher scores indicating severe illness difference 0.5 clinically significant difference 40 measured bss end outpatient visit discharge inpatient admission severe recorded value analysis \\n dna extracted vanderbilt dna core laboratory blood saliva samples oragene dna genotek standardized protocols oragene samples processed immediately collection blood samples processed immediately frozen processing genotyping pre-specified loci -2387 -2274 -1343 46 79 adrb2 gene taqman sm \\u00a9 genotyping assays applied biosystems foster city usa predetermined primers single-nucleotide polymorphisms (snp)s rs1432623 rs11168068 rs2400707 promoter region rs1042713 snp rs1042714 snp coding region adrb2 gene probes applied biosystems assays labeled 5\\u2032end vic \\u00ae 6-carboxy-fluorescine fam reporter dyes reaction components include 2.5 \\u03bcl taqman universal pcr master mix applied biosystems 0.125 \\u03bcl 0.250 \\u03bcl assay mix applied biosystems approximately 5 ng genomic dna total volume 10 \\u03bcl single tube reaction pcr amplification performed thermal cycler techne tc-412 initial step 95 \\u00b0 c 10 min followed 50 cycles 92 \\u00b0 c 15 60 \\u00b0 c 1 min amplification fluorescence sample read abi 7900ht dna resources core vanderbilt university analyzed sequence detection software applied biosystems percent samples run blind duplicates 100 concordance primers probes snps listed table 1 \\n demographics characteristics 374 infants included upper respiratory infection lower respiratory tract infection presented frequencies proportions specific categorical variables medians interquartile ranges calculated continuous variables analyses stratified race caucasian african american surrogate measure european african ancestry reduce known impact population substructure produce spurious associations 41 pairwise linkage disequilibrium ld estimated snps adrb2 standardized summary statistics r calculated haploview program whitehead institute biomedical research cambridge usa snps tested deviations hardy weinberg proportions haploview 42 promoter coding combined promoter coding snps assigned blocks confidence interval gabriel al. 43 individual haplotypes estimated program phase v2.1.1 44 45 haplotype frequencies block estimated expectation maximization em algorithm haploview program non-parametric kruskal-wallis test applied compare difference bss infants snp genotype haplotype genotype haplotype genotypes frequency 2 excluded haplotype analysis proportional odds model ordinal bss test additive effect haplotypes covariates included regression model included infant age enrollment gender daycare exposure secondhand smoke exposure prior history breastfeeding siblings home enrollment season analyses conducted separately according race analyses performed r-software version 2.11.1 www.r-project.org two-sided 5 significance level statistical inferences \\n total 374 infants 261 caucasians 113 african americans enrolled tcri cohort available dna adrb2 polymorphism tested table 3 lists allele frequencies 5 snps separately caucasian african americans caucasians african americans infants slightly g alleles promoter snp rs1432623 figs. 1 2 non statistical difference consistent caucasians african americans african americans decreasing trend severity acute respiratory infection increasing number copies c c allele promoter snps rs1432623 rs11168068 rs2400707 respectively fig. 1 analyses additive models indicated statistical significance p 0.05 \\n 2 5 32 possible haplotype combinations haplotypes 2 frequency identified table 4 fig. 3 promoter haplotypes coding block haplotypes statistically significant differences promoter coding block haplotype frequencies caucasian african americans 5 snps combined distribution haplotype frequency significantly different caucasians african americans genotype ccaac common african americans rare caucasians p 0.001 \\n promoter haplotype cca associated decreased bss african americans dose dependent manner p 0.037 univariate analysis fig. 4 infants cca haplotype highest bss median 5.5 interquartile range iqr 2.0 8.0 infants copy cca haplotype lower bronchiolitis severity score median 4.0 iqr 1.0 7.5 infants 2 copies cca haplotype comparatively lowest bss median 3.0 iqr 1.0 4.0 dose dependent relationship persisted adjusting infant age gender daycare exposure secondhand smoke exposure prior history breastfeeding siblings home enrollment season adjusted odds ratio 0.59 95 confidence interval ci 0.36 0.98 p 0.042 protective effect cca haplotype consistent categorized acute respiratory infection upper respiratory infection lower respiratory tract infection similar dose dependent manner infants copies cca haplotype likely lower respiratory tract infection compared infants copies cca haplotype p 0.05 hand promoter haplotype ttg associated higher bss african americans additional file 1 \\n protective effect cca haplotype african americans identified combined haplotypes cca combination figs. 5 6 sample size effect statistically significant addition african american infants coding block haplotype gg cca combination promoter region similar additive protective effect gg african americans additional file 2 relationship bss coding block haplotypes african americans additional files 3 4 true combined haplotypes additional files 5 6 \\n caucasian african american infants significant relationships bss coding block haplotypes gg haplotype african americans additional file 2 true combined haplotypes general significant relationship bss haplotypes caucasian infants similar promoter haplotype cca relationship bss caucasians observed african americans fig. 4 \\n study identified haplotype promoter region adrb2 gene associated decreased bss african american infants promoter haplotype adrb2 gene associated severity respiratory infections caucasian infants common coding block polymorphisms corresponding amino acid changes codons 16 27 associated protective effect severity respiratory infections caucasian african american infants results agree studied conducted janssen colleagues found relationship snp rs1042713 susceptibility rsv bronchiolitis european population 17 observation cca promoter region haplotype associated protective effect respiratory viral infection severity similar observation cohort adults hospitalized asthma exacerbation haplotype associated spirometric improvement response asthma-specific therapy included beta-agonists corticosteroids 34 study noted spirometric improvement caucasian patients small number african american non-responders cohort limited ability detect similar effect african americans 34 \\n mechanism haplotype promoter region adrb2 gene associated decreased bss known speculate 46 moore al. demonstrated adrb2 genetic variants associated differences adrb2 expression cultured human airway smooth muscle hasm cells vitro system model changes human airway 47 studies performed prior identification complex promoter haplotypes aware haplotypes studied hasm small differences airway tone important setting acute viral respiratory infections \\n scope current observational study speculate adrb2 genotype influence response bronchiolitis-specific therapy primary goal tcri establish longitudinal prospective investigation infants biological mothers acute long-term health consequences varying viral respiratory tract infections early childhood outcomes 37 bss assigned infant acute care visit hospital admission study quantify effects different treatments including beta-agonist therapy number clinical studies suggested infants bronchiolitis appear improve bronchodilator therapy average efficacious 48 49 50 adrb2 primary target bronchodilator therapy variation response beta-agonists reflect genetic variants adrb2 \\n diversity snps adrb2 gene explored drysdale al. immortalized lymphocytes 23 caucasians 19 african-americans 20 asians 15 hispanic-latinos 27 report -2387 -2274 -1343 coding block snps discussed manuscript 28 frequency distributions observed study consistent allele frequencies reported reference population international hapmap project 51 hawkins al. 28 stratified analyses race allow detect association severity respiratory illness haplotype frequency racial group minimize spurious associations population substructure recognize significance findings limited small sample size classifying infants study severity score measuring direct response bronchodilator therapy bss correlated severity disease classified upper respiratory lower respiratory illness sensitivity analyses severity disease upper versus lower respiratory illness showed consistent results indicated bss valid surrogate disease severity lastly p value 0.042 threshold statistical significance p value reflects sample size 113 african americans study association bss cca haplotype african americans powerful depends p value effect size odds ratio clinical significance findings difference 0.5 bss considered clinically significant 65 reduced odds developing severe disease higher bss comparing 2 copies cca 0 copy cca clinically significant meaningful \\n hope hypothesis-generating study encourage incorporation pharmacokinetic data future studies bronchiolitis validation studies populations needed study underscores development personalized approaches treatment bronchiolitis benefit analysis specific genetic variants \\n summary observed promoter region haplotype adrb2 cca associated protective effect respiratory illness severity african american infants era personalized medicine adrb2 genotype predictive severity respiratory viral infection potential response therapy common acute infant illness currently therapy steps validate findings evaluate available clinical trial data dna available fig. 6 bss distribution stratified copy number combined haplotype ccaac african americans infants scatter plots box-and-whisker plots bss 0 1 2 copies combined haplotype ccaac african american infants p values obtained multivariable regression model adjusted infant age enrollment gender daycare exposure secondhand smoke exposure prior history breastfeeding siblings home enrollment season\",\n          \"acute bronchiolitis viral infection lower respiratory tract substantial health burdens infants young children worldwide 1 respiratory syncytial virus prevalent viral cause bronchiolitis infants estimates suggest 34 million new cases lower respiratory infection respiratory syncytial virus occur globally children younger 5 years 3\\u00b74 million admissions hospitals 199 000 deaths year predominantly developing world 2 developed countries usa bronchiolitis common reason admission hospital fi rst 12 months life 3 accounting approximately 100 000 infant admissions annually admissions hospital declined 2000 2010 emergency department visits increased addition increased use mechanical ventilation hospital charges 4 5 clinical management remains challenging despite frequency global reach economic cost morbidity mortality associated bronchiolitis treatment strategies including bronchodilators corticosteroids showed eff ect pooled metaanalyses making supportive care hallmark current therapy seminar aim summarise current evidence epidemiology pathophysiology diagnostic approach management acute viral bronchiolitis \\n bronchiolitis seasonal infection season typically beginning late october temperate northern hemisphere peaking january february ending april 6 globally independent region respiratory syncytial virus infection peaks consistently annual biannual epidemics peak duration epidemics vary worldwide consistent year-to-year country 7 data suggest climate associated prevalence respiratory syncytial virus infection global surveillance suggesting infection peaks wet months areas high precipitation cooler months hot regions 7 indoor crowding population-dense areas rainy seasons cooler months factor facilitates viral transmission 8 additionally weather-related factors inhalation cold dry air impair ciliary function airway mucosa inhibition temperature-dependent antiviral responses infl uence disease transmission severity 9 10 altitude climate meteorological conditions wind speed dew point shown modest association bronchiolitis 11 12 furthermore air pollutants ozone traffi c pollutants associated exacerbations respiratory infections children younger 5 years 13 14 15 environmental tobacco smoke associated increased risk respiratory syncytial virus-attributable admission hospital disease severity admitted 16 17 respiratory viral infections risk severe respiratory syncytial virus bronchiolitis greater boys girls 18 19 diff erence diff erences lung airway development genetic factors 8 20 pathophysiology bronchiolitis characterised extensive infl ammation oedema airways increased mucus production necrosis airway epithelial cells 21 respiratory syncytial virus binds epithelial cells replicates resulting epithelial necrosis ciliary destruction 21 22 cell destruction triggers infl ammatory response proliferation polymorphonuclear cells lymphocytes submucosa adventitial tissues oedematous increased mucus secretion 22 plugs composed cellular debris mucus form bronchiole lumens leading bronchiolar obstruction air trapping diff erent degrees lobar collapse 21 microbiology molecular testing led improved understanding viruses associated bronchiolitis respiratory syncytial virus remains commonly identifi ed virus detected 41 83 patients 23 24 25 26 27 28 viruses associated bronchiolitis include rhinovirus metapneumovirus coronavirus human bocavirus infl uenza virus adenovirus parainfl uenza virus 24 25 27 29 30 studies investigated severity illness measured need hospital admission length hospital stay intensive care unit admission repeated emergency department visits apnoea associated specifi c viral infections co-infections evidence confl icting data studies shown infections involving single virus respiratory syncytial virus associated severe course compared viruses 31 32 30 children bronchiolitis found co-infections viruses 29 combination respiratory syncytial virus rhinovirus commonly reported 29 evidence suggests co-infection bronchiolitis particularly respiratory syncytial virus combination rhinovirus metapneumovirus associated severe disease course compared infection single virus 25 30 31 33 34 studies confi rm association 24 26 furthermore use nucleic acid amplifi cation tests greatly improved ability detect viruses present respiratory infections studies technologies found respiratory virus 30 children younger 6 years respiratory symptoms 35 36 37 viruses detected asymptomatic colonisation incubation clinical infection prolonged viral shedding post-infection confl icting evidence high prevalence respiratory viruses asymptomatic children suggest indication time management vary based presumed viral cause presence absence viral co-infections \\n diagnosis bronchiolitis clinical requires clinician recognise signs symptoms viral lower respiratory tract infection young children peak incidence occurs 3 months 6 months age 38 early clinical defi nition court noted recent practice guidelines specifi c defi nition infants 38 39 40 physiology occur toddlers older 12 months clinical trials excluded children included small subgroup patients bronchiolitis toddlers overlap conditions viral-induced wheezing asthma application evidence trials predominantly assessing infants appropriate eff orts focus defi nition assist eff orts standardise care \\n classic clinical presentation bronchiolitis starts symptoms viral upper respiratory infection nasal discharge progress lower respiratory tract days fi gure 1 timing symptom progression vary young infants present apnoea lower respiratory tract symptoms bronchiolitis include persistent cough tachypnoea increased work breathing shown intercostal supraclavicular retractions use abdominal muscles grunting nasal fl aring auscultatory fi ndings include crackles wheeze hallmark characteristic bronchiolitis minute-to-minute variation clinical fi ndings mucus debris airways cleared coughing child state changes sleep agitation variation confound assessment requires examinations period observation nasal congestion confound clinical assessment nasal suctioning help ascertain fi ndings truly lower airways fever present infants bronchiolitis usually present early illness temperature 39 \\u00b0 c 38 median duration symptoms 2 weeks 10 20 infants symptoms 3 weeks onset 38 41 clinical scores shown studies clinical protocols correlate disease severity improvement 42 43 documentation score useful objective measure individual scores highly predictive repeated combined measures severity universal assessment guide decision making diff erential diagnosis bronchiolitis includes considerations infectious non-infectious causes absence upper respiratory symptoms raise suspicion causes respiratory distress young infants including cardiac disease congenital airway abnormalities vascular ring foreign body aspiration infections resemble complicate bronchiolitis pertussis considered infants severe paroxysmal cough known exposure bacterial infections complicating viral bronchiolitis including otitis media pneumonia present new fever worsening status later course illness \\n risk factors associated progression severe bronchiolitis supported strongest evidence include presence chronic lung disease prematurity haemodynamically important congenital heart disease immunodefi ciency neuromuscular disorders considered high risk practice guidelines 38 39 young infants aged 2 3 months history premature birth especially 32 weeks gestation high risk progression present apnoea clinical fi ndings studies assessing risk apnoea hospital found limited infants 1 month full-term infants 48 weeks postconceptional age preterm infants apnoea observed admission 44 \\n bronchiolitis clinical diagnosis based history physical examination according consensus national guidelines infants typical presentation bronchiolitis routine imaging laboratory testing recommended increase costs evidence benefi table \\n appropriate use pulse oximetry monitoring initiation oxygen bronchiolitis received increasing attention studies practice guidelines findings suggest arbitrary thresholds oxygen administration drive hospital admissions prolong hospital length stay outcomes represent morbidity bronchiolitis developing evidence suggests intermittent hypoxaemia occur commonly stable infants bronchiolitis raises questions factor sole indication admission hospitals canadian randomised trial found reduced admissions hospital emergency department increase revisits pulse oximeters displayed values 3 higher actual value suggesting arbitrary pulse oximetry thresholds result unnecessary admissions 51 similar trial uk hospital setting found reduction oxygen threshold 94 90 resulted earlier discharge hospital evidence adverse outcomes 52 trial comparing intermittent versus continuous pulse oximetry non-hypoxaemic infants hospitals found similar outcomes groups 53 trial evidence supports recommendations practice guidelines clinicians use threshold 90 initiation oxygen uk guidelines recommend 92 38 39 child improves reduction intensity monitoring intermittent checks appropriate recent study blinded oximetry home showed substantial proportion infants bronchiolitis oxygen desaturations 90 particularly sleep calling question arbitrary thresholds hospital admissions initiation oxygen 54 evidence probably lead eff orts reduce continuous monitoring children indications monitoring \\n majority children bronchiolitis normal radiographs radiographic fi ndings consistent simple bronchiolitis 55 58 including peribronchial thickening hyperinfl ation atelectasis prospective study 58 routine radiographs assessment bronchiolitis emergency department reported airspace disease 17 7 246 patients despite low prevalence radiographic pneumonia studies reported increase antibiotic prescription radiographs performed non-specifi c fi ndings infl uence clinicians decisions 58 59 factors associated defi nite focal infi ltrates consistent pneumonia include hypoxia oxygen saturation 92 56 57 58 60 grunting persistently focal crackles fever especially 39 \\u00b0 c 38 chest radiographs considered patients presentation classic bronchiolitis situations include diagnosis foreign body aspiration high diff erential diagnosis child severely ill respiratory failure imminent symptoms progressing resolving according typical disease course expected bronchiolitis lung ultrasound increasingly assess cardiopulmonary conditions adults children studies investigated use lung ultrasound diagnosis bronchiolitis small studies found ultrasound fi ndings infants bronchiolitis correlate clinical fi ndings specifi c chest radiography 61,62 studies needed establish role ultrasound diagnosis assessment severity \\n development pcr detect respiratory viruses nasopharynx interest use viral testing causative diagnosis bronchiolitis increased virological testing generally assist management insuffi cient predict outcomes 30 63 national guidelines recommend routine virological testing bronchiolitis table recent studies suggest higher respiratory syncytial virus genomic load measured quantitative pcr associated increased length stay use respiratory support need intensive care addition recurrent wheezing compared lower viral loads 28 31 32 64 study warranted confi rm association clarify viral load measurement improves understanding disease pathophysiology severity guidelines recommend respiratory syncytial virus testing guide cohorting patients viruses likely cause bronchiolitis transmitted similar fashion close contact large-particle aerosols direct contact contaminated fomites 65 66 67 infection control dependent identifi cation specifi c viruses following strict precautions including hand hygiene separating infants shared hospital rooms 1 infection control procedures 65 additionally given sensitivity pcr testing results interpreted caution certain viruses rhinovirus detected viral shedding unrelated illness colonisation certain viruses respiratory syncytial virus metapneumovirus associated acute infection \\n blood urine testing routinely recommended standard practice diagnostic work-up bronchiolitis table blood gas measurement routinely obtained infants bronchiolitis signs impending respiratory failure severe distress proportions bacterial infections especially bacteraemia meningitis low infants bronchiolitis 68 abnormal white blood cell count rarely useful predicting bacterial infections children infected respiratory syncytial virus 69 guidelines universally recommend complete blood counts infants bronchiolitis blood count assessment fever infants younger 1 2 months similarly given bacteraemia exceedingly rare cited proportions 0\\u00b71 post-pneumococcal vaccine era blood cultures routinely performed septic work-up infants younger 1 2 months 70 71 severe illness signs sepsis hydration status important consideration infants bronchiolitis determined clinical examination routine measurement serum electrolytes little value majority infants \\n urinary tract infections infants bronchiolitis occur greater frequency bacteraemia meningitis proportions ranging 1 7 68 71 72 reasonable obtain urinalysis urine culture infants aged 60 days fever older febrile infants risk factors urinary tract infections 73 urine routinely obtained infants bronchiolitis \\n current recommendations management bronchiolitis focus agents treat patho physiological eff ects viral lower respiratory infection eg bronchodilators hypertonic saline specifi c antivirals ribavirin treat respiratory syncytial virus infection recommended practice guidelines typical cases bronchiolitis challenging delivery methods high cost potential health risks caregivers multiple new agents prevention treatment investigation available future including immunoglobulins small interfering rna interference fusion inhibitors small molecules 74 numerous studies assessed role bronchodilators treatment bronchiolitis systematic reviews found consistent benefi t. 2014 cochrane collaboration systematic review identifi ed 30 studies assessing bronchodilators predominantly salbutamol excluding epinephrine 21 studies looked specifi ally clinical scores found evidence benefi outcomes infants admitted hospitals 75 outpatients oxygen saturation admission hospital time resolution symptoms improve bronchodilator usage compared placebo outpatient studies assessing short-term change pooled clinical scores reviewers found small signifi nt diff erence mean score z=2\\u00b726 p=0\\u00b7024 small eff ect minimal clinical importance fi gure 2 outpatient studies heterogeneous i\\u00b2=81 p<0\\u00b700001 showing benefi scores tended include older children children recurrent wheezing \\n nebulised epinephrine assessed cochrane collaboration systematic review 76 review found benefi epinephrine compared placebo inpatients hospital length stay outcomes multicentre scandinavian study published cochrane review found inpatients receiving standing doses epinephrine longer length stay compared inpatients receiving as-needed epinephrine placebo 77 outpatients cochrane review found diff erence numbers admissions associated epinephrine treatment time emergency department visit overall course illness assessed 1 week clinical practice guidelines including usa uk canada recommend treatment bronchodilators bronchiolitis evidence table 38 \\n nebulised hypertonic saline thought reduce airway oedema decrease mucus plugging improve mucociliary clearance rehydrate airway surface liquid infants bronchiolitis 78 physiological changes extrapolated cystic fi brosis literature 79 80 pathophysiological processes acute bronchiolitis diff erent theoretical benefi ts hypertonic saline seen cystic fi brosis present infants acute viral bronchiolitis initial trials demonstrated ability hypertonic saline decrease hospital length stay transiently improve clinical severity score 81 83 recent trials demonstrated confl icting results 84 85 86 87 88 89 90 trials showed largest benefi hospitals lengths stay 72 h hypertonic saline infants countries institutions length stay approaches exceeds 72 h benefi cial reducing length stay confl icting results \\n test heterogeneity inpatient studies demonstrated low inconsistency studies i\\u00b2=36 p=0\\u00b713 summary eff ect signifi nt z=1\\u00b706 p=0\\u00b729 outpatient studies demonstrated high inconsistency 12 studies i\\u00b2=81 p<0\\u00b700001 summary eff ect signifi nt z=2\\u00b726 p=0\\u00b7024 overall heterogeneity meta-analysis demonstrated high inconsistency 21 studies i\\u00b2=73 p<0\\u00b700001 overall summary eff ect signifi nt z=2\\u00b74 p=0\\u00b7016 ipr ipratropium sal salbutamol neb nebulised reproduced gadomski scribani 75 refl ected diff erences recommendations national guidelines table countries recommending hypertonic saline recommending use inpatients recommending use moderate severe illness largest systematic review meta-analysis published 2015 examined 24 trials 3209 patients 91 infants receiving hypertonic saline signifi nt diff erence hospital length stay -0\\u00b745 days 95 ci -0\\u00b782 -0\\u00b708 p=0\\u00b701 compared receiving 0\\u00b79 saline standard care seven trials hypertonic saline reduced risk admission hospital 20 risk ratio 0\\u00b78 95 ci 0\\u00b767 0\\u00b796 compared 0\\u00b79 saline substantial adverse eff ects hypertonic saline noted systematic review results meta-analysis signifi nt attention paid outer limits confi dence intervals approach diff erence clinical relevance diff erences second 2015 meta-analysis 15 trials 1922 patients found smaller signifi nt decrease length stay -0\\u00b736 days 95 ci -0\\u00b75 -0\\u00b722 p<0\\u00b7001 received hypertonic saline fi gure 3 92 study found discrepancy overall combined eff ect studies length stay negative results largest trials allowing authors conclude individual trials pooled estimates provide strong evidence-based foundation use hypertonic saline meta-analyses showed substantial heterogeneity studies i\\u00b2=82\\u00b71 p<0\\u00b7001 91 i\\u00b2=77\\u00b78 p=0\\u00b7029 92 recent reanalysis fi rst 2015 meta-analysis removed outlying chinese studies accounted imbalances day illness presentation 93 analyses resolved heterogeneity found hypertonic saline reduce length stay infants admitted hospitals bronchiolitis mean diff erence length stay removing outliers -0\\u00b721 days 95 ci -0\\u00b743 0\\u00b702 mean diff erence length stay accounting day illness imbalance 0\\u00b702 days 95 ci -0\\u00b714 0\\u00b717 large trials demonstrated benefi hypertonic saline meta-analysis found eff ect hypertonic saline length stay adjustment outliers imbalances decision undertake future trials controversial given positive results metaanalyses negative results \\n multiple studies examined role corticosteroids management children bronchiolitis data large multicentre trials shown benefi corticosteroids reducing admissions hospital 94 95 2013 cochrane collaboration review supports results studies 96 review included studies enrolled children younger 24 months fi rst episode wheezing signs viral illness included outpatient studies 1824 participants comparing corticosteroids placebo reduction admission day 1 z=1\\u00b705 p=0\\u00b73 day 7 z=1\\u00b738 p=0\\u00b717 enrolment fi gure 4 clinical scores length stay emergency department length time resolution symptoms inpatient studies 772 participants length hospital stay reduced 96 basis evidence multiple clinical practice guidelines recommend use corticosteroids infants bronchiolitis table clinicians report considering family personal history atopy deciding treat infants bronchiolitis corticosteroids 41 evidence infants receive benefi corticosteroid treatment 94 95 96 evidence presence absence respiratory syncytial virus infection infants associated response corticosteroids unavailable 94 95 authors large study factorial design suggested unadjusted analysis high-dose corticosteroids combination nebulised epinephrine reduce admissions outpatients bronchiolitis day 7 95 results considered exploratory 39 45 high-fl ow oxygen respiratory support \\n non-invasive technologies improve oxygenation ventilation bronchiolitis include humidifi ed high-fl ow nasal cannula oxygen continuous positive airway pressure 97 high-fl ow nasal cannula allows delivery high fl ows usually 1 2 l/kg min humidifi cation cannula designed improve patient tolerance widely premature infants mechanisms action unclear particular deliver positive end-expiratory pressure conditions evidence effi cacy high-fl ow nasal cannula predominantly observational studies documenting corneli 2007 goebel 2000 kuyucu 2004 plint 2009 improved respiratory parameters reduced intubation rates implementation 97 small randomised trial compared high-fl ow nasal cannula hypertonic saline found diff erence change respiratory score 98 concerns high-fl ow nasal cannula include potential rapid deterioration infant closely monitored costs associated overuse continuous positive airway pressure studied intensive care settings observational studies small trials evidence improved respiratory parameters 97 uk guidelines recommend considering continuous positive airway pressure children impending respiratory failure bronchiolitis \\n antibiotic overuse children bronchiolitis probably occurs concerns presence fever young age aff ected patients diffi culty diff erentiating atelectasis infectious consolidation chest radiograph concern undetected secondary bacterial infection bronchiolitis clear viral cause occurrence secondary bacterial infections low risk bacteraemia meningitis 1 99 detailed review randomised clinical trials found routine use antibiotics improve duration symptoms length hospital stay need oxygen therapy hospital admission 38 overuse antibiotics known result unnecessary adverse eff ects patient development antimicrobial resistance routine use avoided clear evidence secondary bacterial infection table acute otitis media documented 60 infants bronchiolitis 100 101 antibiotic use acute otitis media bronchiolitis follow established evidence guidelines acute otitis media 102 macrolide antibiotics anti-infl ammatory properties potential benefi mitigating infl ammation present bronchiolitis randomised trials found diff erence azithromycin placebo hospital length stay need oxygen hospital re-admission 103 104 randomised trial found azithromycin lowered nasal interleukin-8 concentrations prolonged time subsequent wheezing episodes resulted fewer days respiratory symptoms year following bronchiolitis episode compared placebo 105 finally multicentre study found children aged 12 72 months history recurrent severe lower respiratory tract infections early administration azithromycin lower respiratory tract infection reduced likelihood progression severe infection clear underlying disease children bronchiolitis disease process 106 given current evidence routine use macrolides recommended bronchiolitis research needed clarify potential role future \\n hydration suctioning chest physiotherapy suggested supportive therapies infants bronchiolitis diffi culty feeding nasal congestion increased work breathing hydration remains cornerstone therapy multicentre study 759 infants younger 12 months admitted hospital bronchiolitis showed benefi intravenous fl uids compared administration fl uids nasogastric tube mean length stay admission intensive care unit need ventilatory support adverse events trial found nasogastric tube easier place intravenous line infants 107 guidelines recommend nasogastric intravenous fl uids maintain hydration uk scottish guidelines preferring nasogastric orogastric hydration tolerate compared intravenous hydration table intravenous fl uids isotonic fl uids preferred avoid risk hyponatraemia 38 infants obligate nasal breathers nasal suctioning suggested help clearing nares improve work breathing improve feeding suctioning irritate nasal mucosa result oedema randomised controlled trials examined role nasal suctioning bronchiolitis insuffi cient available evidence includes retrospective cohort study 740 infants 108 small observational study 40 infants 109 studies suggest deep suctioning increase length stay inpatients 108 infrequent suctioning associated increased length stay 108 oxygen saturation increase suctioning 109 draw conclusions causality observational studies diffi cult potential confouding indication exists eg sicker children likely receive deep suctioning evidence suggests oxygen saturation increases nasal irrigation suctioning 110 current guidelines diff ering recommendations regard suctioning support use recommend superfi cial suctioning deep suctioning table 38 39 45 chest physiotherapy use bronchiolitis appears vary country 111 112 recent cochrane collaboration review 12 studies 1249 participants demonstrated evidence benefi type chest physiotherapy inpatients length stay oxygen saturation respiratory parameters 113 published guidelines routinely recommend chest physiotherapy management uncomplicated bronchiolitis healthy children respiratory comorbidities table 38 39 45 prognosis work published risk developing recurrent wheezing asthma following bronchiolitis infancy 114 115 116 studies followed birth cohorts determine risk subsequent wheezing lower respiratory tract infection young childhood cohorts children admitted hospital bronchiolitis 114 115 116 117 overall admission hospital bronchiolitis young age associated increased risk recurrent wheezing studies report 17 60 children bronchiolitis develop recurrent wheezing years following initial admission hospital large study taiwan followed 1981 children admitted bronchiolitis age 3 years found age 10 years 351 17\\u00b77 1981 children bronchiolitis diagnosis asthma compared 2159 11\\u00b77 18 527 controls hazard ratio 1\\u00b758 95 ci 1\\u00b741 1\\u00b771 118 small cohort study 138 patients suggested 18 39 46 children admitted bronchiolitis 12 months asthma 18 years compared 9 92 controls 115 study followed birth cohort 1246 children found lower respiratory tract infection childhood associated increased risk recurrent wheezing association decreased age signifi nt age 13 years 116 children cohort mild illness requiring hospital admission severity illness associated increased risk asthma 119 question remains respiratory infection young age predisposes children asthma damage alteration lung function children severe bronchiolitis individual risk factors altered immune response airway function predisposes severe bronchiolitis recurrent wheezing 8 \\n substantial knowledge gaps controversies exist management acute bronchiolitis role nebulised hypertonic saline acute management clear resulting confl icting recommendations clinical guidelines table meta-analyses suggest small reduction length stay analyses limited heterogeneity accounting duration illness considering role outlying study populations 91 large multicentre trials support clinically signifi nt diff erence length stay inpatients clear evidence cost benefi t. 4 89 90 91 120 meta-analysis suggests possible reduction admissions outpatients confi dence intervals wide studies heterogeneous 91 multicentre studies nebulised hypertonic saline completed outpatients evidence increasing hypertonic saline little role meaningfully reducing length stay role outpatients clear large outpatient multicentre study clarify benefi t. \\n evidence suggests combined therapy nebulised epinephrine corticosteroids reduce admissions hospital 95 synergy corticosteroids \\u03b22 agonists documented clinical trials asthma management 121 122 123 124 basic science literature shows \\u03b22 agonists corticosteroids enhance eff ectiveness particularly regard antiinfl ammatory gene expression 125 126 economic burden bronchiolitis plausible basic clinical evidence synergy large multicentre trial needed ascertain combined therapy epinephrine corticosteroids benefi cial \\n oxygen saturation use pulse oximetry play important role decision admit infants bronchiolitis hospital length hospital stay 51 52 53 54 55 56 57 58 59 clinical practice guidelines confl icting guidance level oxygen saturation admission considered furthermore substantial proportion discharged infants episodes transient desaturation 54 view large health-care costs associated hospital admission bronchiolitis research needed clarify level oxygen saturation requiring admission role continuous spot measurements oxygen saturation clinical importance transient desaturations stable young infants \\n taf contributed design coordinated writing manuscript authors contributed literature search writing manuscript \\n declare competing interests taf acp jjz participated trials bronchiolitis funded unrestricted public academic institutional funding sources infl uence conception writing manuscript\",\n          \"march april 2009 health professionals warned mexican ministry health increase incidence acute respiratory illness young adults 1 april new viral subtype influenza a(h1n1)pdm09 combination swine avian human virus identified united states samples originated mexico 2 3 april 25 th patients identified sinan database following variables selecting cases controls pcr positive influenza a(h1n1)pdm2009 final classification confirmed criteria laboratory evolution recovered death hospitalization yes date hospitalization 28/06/2009 29/08/2009 municipalities residence hospitals metropolitan areas sao paulo campinas \\n time patients selected study sinan included 193 reported confirmed deaths patients influenza a(h1n1)pdm2009 sari assessed study cases case death control patients selected 1988 patients hospitalized study period reported confirmed influenza a(h1n1)pdm2009 sari recovered controls time-paired randomly selected hospitalized epidemiological week ew necessary week later date hospitalization cases prevent possible differences opportunity treatment clinical management protocol \\n data study collected trained health professionals standardized questionnaires hospital questionnaire collected information medical charts evaluated 177 hospitals patients hospitalized 56.5 belonged unified health system-sus 43.5 private hospitals following data collected previous pathological history occurrence pregnancy health care symptomatology admission laboratory radiological tests need icu admission treatment complications disease progression international classification diseases 10 th revision-icd-10 coding comorbidities indicated medical charts ministry health provided antiviral drug known oseltamivir treatment pandemic protocol clinical management established treatment cases hospitalized sari ambulatory patients risk conditions severity treatment started 48hours symptom onset patients studied laboratory confirmations influenza a(h1n1)pdm09 respiratory secretion samples analyzed rt-pcr method performed adolfo lutz institute 16 definition sari included fever associated cough dyspnea pneumonia respiratory insufficiency tachypnea radiological changes compatible pneumonia oxygen therapy mechanical ventilation sari confirmed cases controls selected study data medical charts \\n household interviews conducted 161 cases 83.4 14 refusals 7.3 18 9.3 addresses found total 337 85.1 controls interviewed 6 1.5 refusals 43 11.1 addresses found household questionnaire included socio-demographic data history previous care related episode led hospitalization presence influenza vaccination occupations grouped according occupational pyramid influenza risk 17 cases interviews conducted closest family members controls patients excluding children parents/legal guardians responded \\n data collection questionnaires assessed supervisors verify consistency information fulfillment criteria include patients study hospital household questionnaires input sphynx database version 5 \\n present study began order assist decisions surveillance control actions use data approved research ethics committee school public health university s\\u00e3o paulo-fsp-usp protocol 2283 of.coep/312/11 data collection performed accordance recommendations national health council human research including signing informed consent form patients family members legal guardians \\n clinical demographic characteristics shown median inter-quartile intervals percentages comparisons mann-whitney u chi-square tests required odds ratio-or 95 confidence intervals-95%ci calculated assess factors associated death variables p-value 0.20 univariate analysis adjustment variables selected multiple logistic regression forward stepwise strategy model including patients assessed following dichotomous categorical variables health insurance previous care vaccination influenza 2009 sex reference category male following risk conditions diabetes mellitus obesity chronic kidney disease chronic liver disease immunosuppression chronic cardiovascular disease chronic neurological disease delay development absence condition reference category model assessed age group 18 years reference 18 59 years 60 years use antiviral therapy use reference category 48 hours onset symptoms 48 72 hours 72 hours high-risk individuals risk conditions listed adapted centers disease control prevention-cdc 18 condition reference category condition conditions time onset symptoms admission days continuous variable model controlled epidemiologic week admission hospital categorized ew26 28 reference ew29 31 ew31 34 income second model developed patients aged 18 years assessed level education formal education incomplete primary education complete primary education complete secondary education complete higher education reference category occupational risk high high average low reference category patients aged 18 22 years incomplete higher education included reference category education modelling patients occupation included reference category occupational risk significance variables assessed wald test adjustment models hosmer-lemeshow test \\n analyses performed spss software version 17.0 p-values 0.05 considered significant table 1 presents socioeconomic characteristics patients information associated disease obtained household interviews patients aged 18 years higher risk death formal education incomplete primary education 2.33(95 ci 1 13 4.89 complete primary education 3.06 95%ci 1.41 6.82 compared higher education distribution ethnic groups family income history smoking associated death history previous vaccination seasonal influenza 2009 showed protection death health professionals considered high risk developing influenza showed protection death 0.15(95%ci 0.02 0.61 patients sought care prior hospitalization risk death 3.70 times 95%ci 2.25 6.30 patients previous care predominance emergency care 45.34 table 2 presents distribution cases controls according variables assessed hospital investigation median age cases higher controls p<0.001 18-to-59-year age group death 4.03(95%ci 2.62 6.31 compared aged 18 years elderly individuals significant risk death \\n differences sex distribution group regard risk conditions severity 73.6 cases 38.1 controls reported risk factor 4.49 95%ci 3.09 6.60 individuals conditions assessed risk death increased 10.29 95%ci 6.04 17.85 following medical conditions risk factors death obesity diabetes mellitus chronic cardiovascular diseases chronic kidney diseases neurological diseases delay development chronic liver diseases immunosuppressive diseases immunosuppression-related diseases neoplasms n 18 hiv/aids n 8) organ transplants n 6 autoimmune diseases n 5 use immunosuppressants n 3 \\n 212 women reproductive age 10 49 years studied 33.5 pregnant differences frequency pregnancy group \\n use antiviral treatment hospitalization protective factor death 0.67 95%ci 0.45 0.98 time onset symptoms beginning antiviral treatment assessed 81 protection individuals received treatment 48 hours 0.19 95%ci 0.11 0.35 65 48 72 hours 0.35 95%ci 0.16 0.74 72 hours significant protection median days date onset symptoms date hospitalization 5.0 cases 2.0 controls p<0.001 patients chronic pulmonary disease median 4.0 days cases 2.0 days controls asthma 4.5 days 2.0 days pregnant women 4.0 days 2.0 days respectively regard health professionals median date onset symptoms hospitalization days cases controls median date hospitalization death hospital discharge seven days cases days controls table 2 \\n according medical charts fever cough occurred 87.8 cases 79.3 controls proportions dyspnea 87 62.7 respectively definition cases severe acute respiratory syndrome -sars fever+cough+dyspnea based information obtained medical charts occurred 68.9 cases 56.5 controls vomit diarrhea present 8 cases 18 controls treatment antibiotics 95.8 cases 73.1 controls mean number antibiotics 4.0 1.8 respectively cases higher proportion hospitalizations intensive care units 83.4%x16.8 use mechanical ventilation 94.8%x10.8 dialysis 25.0%x2.3 proportion complications higher cases controls emphasis respiratory distress syndrome 64.7%vs4.9 shock 69.4%vs1.0 sepsis 63.2%vs2.3 kidney complications 44.6%vs1.3 respectively table 3 shows laboratory findings entry tests patients hospital cases lower numbers leukocytes platelets higher values creatine phosphokinase-cpk lactic dehydrogenase-ldh glutamic-oxaloacetic transaminase glutamic-pyruvic transaminase urea creatinine controls statistical significance \\n sub-group analysis previous conditions risk severity obesity risk factors death univariate analysis children younger 18 years women reproductive age 10 49 years children presence immunosuppression chronic cardiovascular disease neurological disease/delay development risk factors diabetes mellitus women treatment antiviral administered 72 hours onset symptoms protective factor women reproductive age tables 4 5 variables associated death final models multiple logistic regression patients studied sub-group analysis adults aged 18 years patients general risk factors death aged 18 59 years reference 18 years risk conditions severity disease obesity immunosuppression sought care prior hospitalization chronic neurological diseases/delay development chronic liver disease highly associated death variables excluded final model estimates large confidence intervals private health insurance antiviral treatment administered 72 hours onset symptoms protective factors death adults aged 18 years risk factors death risk factor severity disease obesity care prior hospitalization protective factors antiviral drug administered 48 hours onset symptoms health professional health unit worker \\n sensitivity analysis performed assess severity patients admitted study period proportion icu admissions use mechanical ventilation estimated epidemiological week hospitalization categorized ew 26 28 ew 29 31 ew 32 34 results showed higher proportions icu admissions 50.0 x 16.8 x 7.7 mechanical ventilation 45.4 x 10.8 x 2.6 weeks controls cases showed high proportions periods analyzed 89.7 x 84.0 x 76.9 need icu 89.7 x 97.6 x 89.7 need mechanical ventilation 3 periods respectively opportunity antiviral treatment showed lower proportion treatment period study cases 37.9 x 74.4 x 74.3 controls 59.1 x 74.1 x 89.7 sensitivity analysis assess opportunity treatment children adults showed similar proportions groups 73.9 74 respectively \\n present study risk factors death analyzed patients infected influenza (h1n1)pdm09 hospitalized sari increased risk death patients aged 18 59 years obesity immunosuppression neurological developmental diseases received care prior hospitalization additionally presence risk factor aggravation disease associated occurrence death use antiviral drugs especially administered 72 hours onset symptoms protective factor death \\n design case-control study including deaths reported confirmed region study period time-paired random selection controls patients survived hospitalization sari hospital household data collection enabled analysis factors associated death extended including socioeconomic variables access health services data hospitalizations \\n studies assessed risk factors death patients sari 15 19 studies analyzed risk factors severity disease deaths comparing patients hospitalized icus wards 13 14 20 21 22 23 study rate mortality influenza a(h1n1)pdm09 epidemic waves 24 study conducted australia analyzed risk factors hospitalization influenza comparing cases controls infection community 25 studies analyzed epidemiological surveillance data including mortality rates comparisons deaths hospitalizations treatment factors 9 10 12 26 27 28 29 30 31 32 33 34 35 factors associated death identified study described authors presence comorbidities 31 35 age18years 13 obesity 15 19 20 neurological developmental diseases 23 24 31 immunosuppression 15 21 24 34 children showed lower risk death despite similar proportion treatment adults similar result observed study systematic review metaanalysis risk death 0.28 95 ci 0.19 0.141 compared young adults 36 contrast factors associated severity infection reported different researchers diabetes 19 34 chronic pulmonary diseases excluding asthma chronic obstructive pulmonary disease 20 chronic pulmonary disease including asthma 31 chronic cardiovascular disease 21 24 34 associated mortality present study population assessed restricted hospitalized patients severe influenza partly explain differences diabetes chronic cardiovascular kidney disease associated death univariate analysis remain risk factors multiple analysis design study data collected medical charts household interviews allowed adjustments analysis explain differences \\n deaths occurred mainly adults aged 18 59 years small proportion cases controls elderly individuals possible preexisting immunity new influenza a(h1n1)pdm09 virus age group suggested probably previous exposure infections vaccinations influenza a(h1n1 virus genetically antigenically associated new virus contemporary strains seasonal h1n1 viruses 37 \\n patients present study cases controls laboratory-confirmed infection influenza a/h1n1pdm09 38 \\n possible probability hospitalized varied study treatment available hospitalized patients ambulatory patients risk conditions severity influenced hospitalization study population consisting patients sari pandemic criteria hospitalization changed availability treatment especially 3 weeks epidemic shown sensitivity analysis confirming need controlling period hospitalization study \\n socioeconomic profile similar groups assessed difference income private health insurance greater protection death patients occupation higher risk exposure infection health professionals greater opportunity access treatment workers explain findings period date onset symptoms date admission shorter group low proportion deaths patients categorized higher risk exposure observed studies 30 33 \\n pregnancy risk factor death study contrast frequency pregnant women reproductive age hospitalized severe manifestation influenza (h1n1)pdm09 high suggesting pregnancy associated development sari association pregnancy use mechanical ventilation hospitalization influenza a(h1n1)pdm2009 25 higher admission rate reported compared general population 32 2009 state s\\u00e3o paulo 27.4 confirmed cases influenza a(h1n1)pdm09 women reproductive age 10 49 years hospitalized recorded sinan database found pregnant women brazil 4.27 women reproductive age live newborns reference year according 2010 census 39 \\n patients died presented important laboratory changes compared controls lower levels leukocytes platelets increase cpk ldh urea creatinine studies revealed similar trend laboratory results leukocytes 23 cpk 26 27 30 ldh 26 platelets 22 creatinine 27 30 34 \\n patients died received antiviral treatment 9.3 treated 48 hours early treatment protective factor death especially provided 72 hours onset symptoms studies shown favorable prognostic antiviral treatment 13 21 23 27 28 29 34 \\n history care prior hospitalization risk factor death influenza a(h1n1 pdm09 reflect difficulty accessing health services emergency services primarily sought care prior hospitalization contributed greater delay diagnosis beginning treatment study conducted mexico showed similar result regard delay hospitalization 29 \\n study assessed overall standardized mortality rate influenza (h1n1)pdm09 mortality patients preexisting chronic respiratory diseases lower chronic diseases 24 problems recording information previous diseases medical charts presence acute pulmonary disease especially chronic pulmonary disease \\n present study limitations underreporting sari health professionals likely occurred reporting disease compulsory brazil pandemic pandemic professionals sensitive reporting system notification mandatory epidemic facilitated allow rapid notifications availability samples autopsies cases deaths resulted greater access laboratory diagnosis infection household data collection caused information bias data cases collected interviews family members data controls obtained patients adults proportion refusals household interviews higher cases 7.3 controls 1.6 p 0.001 possible limitation memory bias result time hospitalization interview information analyzed household interviews socio-demographic characteristics vulnerable information memory bias data preexisting diseases obtained medical charts hospitals completed moment hospitalization presence patients quality completion medical charts differ hospitals nonetheless information presence chronic diseases clinical management patients laboratory results radiological tests treatment available charts difficulties encountered data collection minimized use standardized questionnaire \\n conclusion risk conditions severity disease particularly obesity immunosuppression neurological/developmental diseases important risk factors death emphasizing need good vaccination coverage influenza vaccination campaigns patients chronic diseases belonging vulnerable groups relevant result antiviral treatment 72 hours onset symptoms protective factor emphasizing need warn population importance seeking early medical care influenza-like illness appears especially high-risk groups signs disease aggravation training health professionals especially physicians required adequate clinical management patients early antiviral treatment\"\n        ],\n        \"semantic_type\": \"\",\n        \"description\": \"\"\n      }\n    },\n    {\n      \"column\": \"cluster\",\n      \"properties\": {\n        \"dtype\": \"int32\",\n        \"num_unique_values\": 28,\n        \"samples\": [\n          24,\n          19,\n          6\n        ],\n        \"semantic_type\": \"\",\n        \"description\": \"\"\n      }\n    },\n    {\n      \"column\": \"hierarchical_cluster\",\n      \"properties\": {\n        \"dtype\": \"number\",\n        \"std\": 0,\n        \"min\": 0,\n        \"max\": 1,\n        \"num_unique_values\": 2,\n        \"samples\": [\n          1,\n          0\n        ],\n        \"semantic_type\": \"\",\n        \"description\": \"\"\n      }\n    }\n  ]\n}",
              "type": "dataframe",
              "variable_name": "df"
            },
            "text/html": [
              "\n",
              "  <div id=\"df-418ca4f1-47c3-4f40-90b6-4837c1f5ccf2\" class=\"colab-df-container\">\n",
              "    <div>\n",
              "<style scoped>\n",
              "    .dataframe tbody tr th:only-of-type {\n",
              "        vertical-align: middle;\n",
              "    }\n",
              "\n",
              "    .dataframe tbody tr th {\n",
              "        vertical-align: top;\n",
              "    }\n",
              "\n",
              "    .dataframe thead th {\n",
              "        text-align: right;\n",
              "    }\n",
              "</style>\n",
              "<table border=\"1\" class=\"dataframe\">\n",
              "  <thead>\n",
              "    <tr style=\"text-align: right;\">\n",
              "      <th></th>\n",
              "      <th>paper_id</th>\n",
              "      <th>doi</th>\n",
              "      <th>abstract</th>\n",
              "      <th>body_text</th>\n",
              "      <th>authors</th>\n",
              "      <th>title</th>\n",
              "      <th>journal</th>\n",
              "      <th>abstract_summary</th>\n",
              "      <th>cleaned_text</th>\n",
              "      <th>cleaned_text_str</th>\n",
              "      <th>cluster</th>\n",
              "      <th>hierarchical_cluster</th>\n",
              "    </tr>\n",
              "  </thead>\n",
              "  <tbody>\n",
              "    <tr>\n",
              "      <th>6252</th>\n",
              "      <td>fefd6cdcddcb00d1ba2ef1a0d3bba9f312010a1d</td>\n",
              "      <td>10.1111/j.1524-4733.2008.00379.x</td>\n",
              "      <td></td>\n",
              "      <td>China is the biggest country in the world that...</td>\n",
              "      <td>Li, Youing.  Sun, Xin.  Wang, Li</td>\n",
              "      <td>Evidence-Based Medicine in China</td>\n",
              "      <td>Value in Health</td>\n",
              "      <td>Not provided.</td>\n",
              "      <td>[china, biggest, country, world, 1.3, billion,...</td>\n",
              "      <td>china biggest country world 1.3 billion popula...</td>\n",
              "      <td>0</td>\n",
              "      <td>0</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>4684</th>\n",
              "      <td>2a3d080d5dcf6771db002366691f35e5d6f0e493</td>\n",
              "      <td>10.1007/s13238-010-0029-7</td>\n",
              "      <td></td>\n",
              "      <td>✉ Correspondence: zlshi@wh.iov.cn Bat, the onl...</td>\n",
              "      <td>Shi, Zhengli</td>\n",
              "      <td>Bat and virus</td>\n",
              "      <td>Protein &amp; Cell</td>\n",
              "      <td>Not provided.</td>\n",
              "      <td>[✉, correspondence, zlshi@wh.iov.cn, bat, flyi...</td>\n",
              "      <td>✉ correspondence zlshi@wh.iov.cn bat flying ma...</td>\n",
              "      <td>18</td>\n",
              "      <td>0</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>1731</th>\n",
              "      <td>2de657aabab331e75657bfc1443542fd345b6f26</td>\n",
              "      <td>10.1016/j.ccc.2019.06.001</td>\n",
              "      <td>KEYWORDS\\nPandemic Influenza Disaster prepared...</td>\n",
              "      <td>Outbreaks of infectious disease pose unique ch...</td>\n",
              "      <td>Maves, Ryan C..  Jamros, Christina M..  Smith...</td>\n",
              "      <td>Intensive Care Unit Preparedness During&lt;br&gt;Pa...</td>\n",
              "      <td>Crit Care Clin</td>\n",
              "      <td>KEYWORDS\\nPandemic Influenza Disaster&lt;br&gt;prep...</td>\n",
              "      <td>[outbreaks, infectious, disease, pose, unique,...</td>\n",
              "      <td>outbreaks infectious disease pose unique chall...</td>\n",
              "      <td>13</td>\n",
              "      <td>0</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>4742</th>\n",
              "      <td>bc22758234283f024f16e2dbe660bc909b512b92</td>\n",
              "      <td>10.1016/j.vetpar.2007.11.020</td>\n",
              "      <td>Cryptosporidium parvum is recognized as one of...</td>\n",
              "      <td>Diarrhoeal diseases of infectious aethiology r...</td>\n",
              "      <td>Klein, Pavel.  Kleinová, Tereza.  Volek,&lt;br&gt;Z...</td>\n",
              "      <td>Effect of Cryptosporidium parvum infection on...</td>\n",
              "      <td>Veterinary Parasitology</td>\n",
              "      <td>Cryptosporidium parvum is recognized as one o...</td>\n",
              "      <td>[diarrhoeal, diseases, infectious, aethiology,...</td>\n",
              "      <td>diarrhoeal diseases infectious aethiology repr...</td>\n",
              "      <td>12</td>\n",
              "      <td>0</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>4521</th>\n",
              "      <td>a204aafa38365dbcc0a26af3ca2c6d3313d7fab2</td>\n",
              "      <td>10.1111/irv.12518</td>\n",
              "      <td>, Brazil, in the period of Background: The res...</td>\n",
              "      <td>The inclusion criteria for asymptomatic adults...</td>\n",
              "      <td>Moreira, Luciana Peniche.  Watanabe, Aripuana...</td>\n",
              "      <td>Respiratory syncytial virus evaluation among&lt;...</td>\n",
              "      <td>Influenza Other Respir Viruses</td>\n",
              "      <td>, Brazil, in the period of Background: The&lt;br...</td>\n",
              "      <td>[inclusion, criteria, asymptomatic, adults, ab...</td>\n",
              "      <td>inclusion criteria asymptomatic adults absence...</td>\n",
              "      <td>15</td>\n",
              "      <td>0</td>\n",
              "    </tr>\n",
              "  </tbody>\n",
              "</table>\n",
              "</div>\n",
              "    <div class=\"colab-df-buttons\">\n",
              "\n",
              "  <div class=\"colab-df-container\">\n",
              "    <button class=\"colab-df-convert\" onclick=\"convertToInteractive('df-418ca4f1-47c3-4f40-90b6-4837c1f5ccf2')\"\n",
              "            title=\"Convert this dataframe to an interactive table.\"\n",
              "            style=\"display:none;\">\n",
              "\n",
              "  <svg xmlns=\"http://www.w3.org/2000/svg\" height=\"24px\" viewBox=\"0 -960 960 960\">\n",
              "    <path d=\"M120-120v-720h720v720H120Zm60-500h600v-160H180v160Zm220 220h160v-160H400v160Zm0 220h160v-160H400v160ZM180-400h160v-160H180v160Zm440 0h160v-160H620v160ZM180-180h160v-160H180v160Zm440 0h160v-160H620v160Z\"/>\n",
              "  </svg>\n",
              "    </button>\n",
              "\n",
              "  <style>\n",
              "    .colab-df-container {\n",
              "      display:flex;\n",
              "      gap: 12px;\n",
              "    }\n",
              "\n",
              "    .colab-df-convert {\n",
              "      background-color: #E8F0FE;\n",
              "      border: none;\n",
              "      border-radius: 50%;\n",
              "      cursor: pointer;\n",
              "      display: none;\n",
              "      fill: #1967D2;\n",
              "      height: 32px;\n",
              "      padding: 0 0 0 0;\n",
              "      width: 32px;\n",
              "    }\n",
              "\n",
              "    .colab-df-convert:hover {\n",
              "      background-color: #E2EBFA;\n",
              "      box-shadow: 0px 1px 2px rgba(60, 64, 67, 0.3), 0px 1px 3px 1px rgba(60, 64, 67, 0.15);\n",
              "      fill: #174EA6;\n",
              "    }\n",
              "\n",
              "    .colab-df-buttons div {\n",
              "      margin-bottom: 4px;\n",
              "    }\n",
              "\n",
              "    [theme=dark] .colab-df-convert {\n",
              "      background-color: #3B4455;\n",
              "      fill: #D2E3FC;\n",
              "    }\n",
              "\n",
              "    [theme=dark] .colab-df-convert:hover {\n",
              "      background-color: #434B5C;\n",
              "      box-shadow: 0px 1px 3px 1px rgba(0, 0, 0, 0.15);\n",
              "      filter: drop-shadow(0px 1px 2px rgba(0, 0, 0, 0.3));\n",
              "      fill: #FFFFFF;\n",
              "    }\n",
              "  </style>\n",
              "\n",
              "    <script>\n",
              "      const buttonEl =\n",
              "        document.querySelector('#df-418ca4f1-47c3-4f40-90b6-4837c1f5ccf2 button.colab-df-convert');\n",
              "      buttonEl.style.display =\n",
              "        google.colab.kernel.accessAllowed ? 'block' : 'none';\n",
              "\n",
              "      async function convertToInteractive(key) {\n",
              "        const element = document.querySelector('#df-418ca4f1-47c3-4f40-90b6-4837c1f5ccf2');\n",
              "        const dataTable =\n",
              "          await google.colab.kernel.invokeFunction('convertToInteractive',\n",
              "                                                    [key], {});\n",
              "        if (!dataTable) return;\n",
              "\n",
              "        const docLinkHtml = 'Like what you see? Visit the ' +\n",
              "          '<a target=\"_blank\" href=https://colab.research.google.com/notebooks/data_table.ipynb>data table notebook</a>'\n",
              "          + ' to learn more about interactive tables.';\n",
              "        element.innerHTML = '';\n",
              "        dataTable['output_type'] = 'display_data';\n",
              "        await google.colab.output.renderOutput(dataTable, element);\n",
              "        const docLink = document.createElement('div');\n",
              "        docLink.innerHTML = docLinkHtml;\n",
              "        element.appendChild(docLink);\n",
              "      }\n",
              "    </script>\n",
              "  </div>\n",
              "\n",
              "\n",
              "<div id=\"df-8947195a-686b-4c12-94fd-133851d7f81f\">\n",
              "  <button class=\"colab-df-quickchart\" onclick=\"quickchart('df-8947195a-686b-4c12-94fd-133851d7f81f')\"\n",
              "            title=\"Suggest charts\"\n",
              "            style=\"display:none;\">\n",
              "\n",
              "<svg xmlns=\"http://www.w3.org/2000/svg\" height=\"24px\"viewBox=\"0 0 24 24\"\n",
              "     width=\"24px\">\n",
              "    <g>\n",
              "        <path d=\"M19 3H5c-1.1 0-2 .9-2 2v14c0 1.1.9 2 2 2h14c1.1 0 2-.9 2-2V5c0-1.1-.9-2-2-2zM9 17H7v-7h2v7zm4 0h-2V7h2v10zm4 0h-2v-4h2v4z\"/>\n",
              "    </g>\n",
              "</svg>\n",
              "  </button>\n",
              "\n",
              "<style>\n",
              "  .colab-df-quickchart {\n",
              "      --bg-color: #E8F0FE;\n",
              "      --fill-color: #1967D2;\n",
              "      --hover-bg-color: #E2EBFA;\n",
              "      --hover-fill-color: #174EA6;\n",
              "      --disabled-fill-color: #AAA;\n",
              "      --disabled-bg-color: #DDD;\n",
              "  }\n",
              "\n",
              "  [theme=dark] .colab-df-quickchart {\n",
              "      --bg-color: #3B4455;\n",
              "      --fill-color: #D2E3FC;\n",
              "      --hover-bg-color: #434B5C;\n",
              "      --hover-fill-color: #FFFFFF;\n",
              "      --disabled-bg-color: #3B4455;\n",
              "      --disabled-fill-color: #666;\n",
              "  }\n",
              "\n",
              "  .colab-df-quickchart {\n",
              "    background-color: var(--bg-color);\n",
              "    border: none;\n",
              "    border-radius: 50%;\n",
              "    cursor: pointer;\n",
              "    display: none;\n",
              "    fill: var(--fill-color);\n",
              "    height: 32px;\n",
              "    padding: 0;\n",
              "    width: 32px;\n",
              "  }\n",
              "\n",
              "  .colab-df-quickchart:hover {\n",
              "    background-color: var(--hover-bg-color);\n",
              "    box-shadow: 0 1px 2px rgba(60, 64, 67, 0.3), 0 1px 3px 1px rgba(60, 64, 67, 0.15);\n",
              "    fill: var(--button-hover-fill-color);\n",
              "  }\n",
              "\n",
              "  .colab-df-quickchart-complete:disabled,\n",
              "  .colab-df-quickchart-complete:disabled:hover {\n",
              "    background-color: var(--disabled-bg-color);\n",
              "    fill: var(--disabled-fill-color);\n",
              "    box-shadow: none;\n",
              "  }\n",
              "\n",
              "  .colab-df-spinner {\n",
              "    border: 2px solid var(--fill-color);\n",
              "    border-color: transparent;\n",
              "    border-bottom-color: var(--fill-color);\n",
              "    animation:\n",
              "      spin 1s steps(1) infinite;\n",
              "  }\n",
              "\n",
              "  @keyframes spin {\n",
              "    0% {\n",
              "      border-color: transparent;\n",
              "      border-bottom-color: var(--fill-color);\n",
              "      border-left-color: var(--fill-color);\n",
              "    }\n",
              "    20% {\n",
              "      border-color: transparent;\n",
              "      border-left-color: var(--fill-color);\n",
              "      border-top-color: var(--fill-color);\n",
              "    }\n",
              "    30% {\n",
              "      border-color: transparent;\n",
              "      border-left-color: var(--fill-color);\n",
              "      border-top-color: var(--fill-color);\n",
              "      border-right-color: var(--fill-color);\n",
              "    }\n",
              "    40% {\n",
              "      border-color: transparent;\n",
              "      border-right-color: var(--fill-color);\n",
              "      border-top-color: var(--fill-color);\n",
              "    }\n",
              "    60% {\n",
              "      border-color: transparent;\n",
              "      border-right-color: var(--fill-color);\n",
              "    }\n",
              "    80% {\n",
              "      border-color: transparent;\n",
              "      border-right-color: var(--fill-color);\n",
              "      border-bottom-color: var(--fill-color);\n",
              "    }\n",
              "    90% {\n",
              "      border-color: transparent;\n",
              "      border-bottom-color: var(--fill-color);\n",
              "    }\n",
              "  }\n",
              "</style>\n",
              "\n",
              "  <script>\n",
              "    async function quickchart(key) {\n",
              "      const quickchartButtonEl =\n",
              "        document.querySelector('#' + key + ' button');\n",
              "      quickchartButtonEl.disabled = true;  // To prevent multiple clicks.\n",
              "      quickchartButtonEl.classList.add('colab-df-spinner');\n",
              "      try {\n",
              "        const charts = await google.colab.kernel.invokeFunction(\n",
              "            'suggestCharts', [key], {});\n",
              "      } catch (error) {\n",
              "        console.error('Error during call to suggestCharts:', error);\n",
              "      }\n",
              "      quickchartButtonEl.classList.remove('colab-df-spinner');\n",
              "      quickchartButtonEl.classList.add('colab-df-quickchart-complete');\n",
              "    }\n",
              "    (() => {\n",
              "      let quickchartButtonEl =\n",
              "        document.querySelector('#df-8947195a-686b-4c12-94fd-133851d7f81f button');\n",
              "      quickchartButtonEl.style.display =\n",
              "        google.colab.kernel.accessAllowed ? 'block' : 'none';\n",
              "    })();\n",
              "  </script>\n",
              "</div>\n",
              "\n",
              "    </div>\n",
              "  </div>\n"
            ],
            "text/plain": [
              "                                      paper_id  \\\n",
              "6252  fefd6cdcddcb00d1ba2ef1a0d3bba9f312010a1d   \n",
              "4684  2a3d080d5dcf6771db002366691f35e5d6f0e493   \n",
              "1731  2de657aabab331e75657bfc1443542fd345b6f26   \n",
              "4742  bc22758234283f024f16e2dbe660bc909b512b92   \n",
              "4521  a204aafa38365dbcc0a26af3ca2c6d3313d7fab2   \n",
              "\n",
              "                                   doi  \\\n",
              "6252  10.1111/j.1524-4733.2008.00379.x   \n",
              "4684         10.1007/s13238-010-0029-7   \n",
              "1731         10.1016/j.ccc.2019.06.001   \n",
              "4742      10.1016/j.vetpar.2007.11.020   \n",
              "4521                 10.1111/irv.12518   \n",
              "\n",
              "                                               abstract  \\\n",
              "6252                                                      \n",
              "4684                                                      \n",
              "1731  KEYWORDS\\nPandemic Influenza Disaster prepared...   \n",
              "4742  Cryptosporidium parvum is recognized as one of...   \n",
              "4521  , Brazil, in the period of Background: The res...   \n",
              "\n",
              "                                              body_text  \\\n",
              "6252  China is the biggest country in the world that...   \n",
              "4684  ✉ Correspondence: zlshi@wh.iov.cn Bat, the onl...   \n",
              "1731  Outbreaks of infectious disease pose unique ch...   \n",
              "4742  Diarrhoeal diseases of infectious aethiology r...   \n",
              "4521  The inclusion criteria for asymptomatic adults...   \n",
              "\n",
              "                                                authors  \\\n",
              "6252                   Li, Youing.  Sun, Xin.  Wang, Li   \n",
              "4684                                       Shi, Zhengli   \n",
              "1731   Maves, Ryan C..  Jamros, Christina M..  Smith...   \n",
              "4742   Klein, Pavel.  Kleinová, Tereza.  Volek,<br>Z...   \n",
              "4521   Moreira, Luciana Peniche.  Watanabe, Aripuana...   \n",
              "\n",
              "                                                  title  \\\n",
              "6252                   Evidence-Based Medicine in China   \n",
              "4684                                      Bat and virus   \n",
              "1731   Intensive Care Unit Preparedness During<br>Pa...   \n",
              "4742   Effect of Cryptosporidium parvum infection on...   \n",
              "4521   Respiratory syncytial virus evaluation among<...   \n",
              "\n",
              "                             journal  \\\n",
              "6252                 Value in Health   \n",
              "4684                  Protein & Cell   \n",
              "1731                  Crit Care Clin   \n",
              "4742         Veterinary Parasitology   \n",
              "4521  Influenza Other Respir Viruses   \n",
              "\n",
              "                                       abstract_summary  \\\n",
              "6252                                      Not provided.   \n",
              "4684                                      Not provided.   \n",
              "1731   KEYWORDS\\nPandemic Influenza Disaster<br>prep...   \n",
              "4742   Cryptosporidium parvum is recognized as one o...   \n",
              "4521   , Brazil, in the period of Background: The<br...   \n",
              "\n",
              "                                           cleaned_text  \\\n",
              "6252  [china, biggest, country, world, 1.3, billion,...   \n",
              "4684  [✉, correspondence, zlshi@wh.iov.cn, bat, flyi...   \n",
              "1731  [outbreaks, infectious, disease, pose, unique,...   \n",
              "4742  [diarrhoeal, diseases, infectious, aethiology,...   \n",
              "4521  [inclusion, criteria, asymptomatic, adults, ab...   \n",
              "\n",
              "                                       cleaned_text_str  cluster  \\\n",
              "6252  china biggest country world 1.3 billion popula...        0   \n",
              "4684  ✉ correspondence zlshi@wh.iov.cn bat flying ma...       18   \n",
              "1731  outbreaks infectious disease pose unique chall...       13   \n",
              "4742  diarrhoeal diseases infectious aethiology repr...       12   \n",
              "4521  inclusion criteria asymptomatic adults absence...       15   \n",
              "\n",
              "      hierarchical_cluster  \n",
              "6252                     0  \n",
              "4684                     0  \n",
              "1731                     0  \n",
              "4742                     0  \n",
              "4521                     0  "
            ]
          },
          "execution_count": 78,
          "metadata": {},
          "output_type": "execute_result"
        }
      ],
      "source": [
        "# Fit the K-Means model with the optimal number of clusters\n",
        "optimal_k = 28\n",
        "kmeans = KMeans(n_clusters=optimal_k, random_state=42)\n",
        "df['cluster'] = kmeans.fit_predict(tfidf_pca)\n",
        "clusters = kmeans.labels_\n",
        "\n",
        "# Display the first few rows of the DataFrame with cluster labels\n",
        "df.head()"
      ]
    },
    {
      "cell_type": "code",
      "execution_count": 82,
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/",
          "height": 624
        },
        "id": "vKlltL9vvh80",
        "outputId": "831cc7c3-8ae5-4c3a-92aa-ca593a2f3d7b"
      },
      "outputs": [
        {
          "data": {
            "image/svg+xml": [
              "<?xml version=\"1.0\" encoding=\"utf-8\" standalone=\"no\"?>\n",
              "<!DOCTYPE svg PUBLIC \"-//W3C//DTD SVG 1.1//EN\"\n",
              "  \"http://www.w3.org/Graphics/SVG/1.1/DTD/svg11.dtd\">\n",
              "<svg xmlns:xlink=\"http://www.w3.org/1999/xlink\" width=\"632.021563pt\" height=\"451.713625pt\" viewBox=\"0 0 632.021563 451.713625\" xmlns=\"http://www.w3.org/2000/svg\" version=\"1.1\">\n",
              " <metadata>\n",
              "  <rdf:RDF xmlns:dc=\"http://purl.org/dc/elements/1.1/\" xmlns:cc=\"http://creativecommons.org/ns#\" xmlns:rdf=\"http://www.w3.org/1999/02/22-rdf-syntax-ns#\">\n",
              "   <cc:Work>\n",
              "    <dc:type rdf:resource=\"http://purl.org/dc/dcmitype/StillImage\"/>\n",
              "    <dc:date>2024-06-05T19:11:15.735415</dc:date>\n",
              "    <dc:format>image/svg+xml</dc:format>\n",
              "    <dc:creator>\n",
              "     <cc:Agent>\n",
              "      <dc:title>Matplotlib v3.7.1, https://matplotlib.org/</dc:title>\n",
              "     </cc:Agent>\n",
              "    </dc:creator>\n",
              "   </cc:Work>\n",
              "  </rdf:RDF>\n",
              " </metadata>\n",
              " <defs>\n",
              "  <style type=\"text/css\">*{stroke-linejoin: round; stroke-linecap: butt}</style>\n",
              " </defs>\n",
              " <g id=\"figure_1\">\n",
              "  <g id=\"patch_1\">\n",
              "   <path d=\"M 0 451.713625 \n",
              "L 632.021563 451.713625 \n",
              "L 632.021563 0 \n",
              "L 0 0 \n",
              "z\n",
              "\" style=\"fill: #ffffff\"/>\n",
              "  </g>\n",
              "  <g id=\"axes_1\">\n",
              "   <g id=\"patch_2\">\n",
              "    <path d=\"M 66.821562 412.22175 \n",
              "L 624.821563 412.22175 \n",
              "L 624.821563 24.14175 \n",
              "L 66.821562 24.14175 \n",
              "z\n",
              "\" style=\"fill: #e5e5e5\"/>\n",
              "   </g>\n",
              "   <g id=\"matplotlib.axis_1\">\n",
              "    <g id=\"xtick_1\">\n",
              "     <g id=\"line2d_1\">\n",
              "      <path d=\"M 146.535848 412.22175 \n",
              "L 146.535848 24.14175 \n",
              "\" clip-path=\"url(#pd0ab672b7b)\" style=\"fill: none; stroke: #ffffff; stroke-width: 0.8; stroke-linecap: square\"/>\n",
              "     </g>\n",
              "     <g id=\"line2d_2\">\n",
              "      <defs>\n",
              "       <path id=\"maa2cf2715f\" d=\"M 0 0 \n",
              "L 0 3.5 \n",
              "\" style=\"stroke: #555555; stroke-width: 0.8\"/>\n",
              "      </defs>\n",
              "      <g>\n",
              "       <use xlink:href=\"#maa2cf2715f\" x=\"146.535848\" y=\"412.22175\" style=\"fill: #555555; stroke: #555555; stroke-width: 0.8\"/>\n",
              "      </g>\n",
              "     </g>\n",
              "     <g id=\"text_1\">\n",
              "      <!-- 5 -->\n",
              "      <g style=\"fill: #555555\" transform=\"translate(143.354598 426.820187) scale(0.1 -0.1)\">\n",
              "       <defs>\n",
              "        <path id=\"DejaVuSans-35\" d=\"M 691 4666 \n",
              "L 3169 4666 \n",
              "L 3169 4134 \n",
              "L 1269 4134 \n",
              "L 1269 2991 \n",
              "Q 1406 3038 1543 3061 \n",
              "Q 1681 3084 1819 3084 \n",
              "Q 2600 3084 3056 2656 \n",
              "Q 3513 2228 3513 1497 \n",
              "Q 3513 744 3044 326 \n",
              "Q 2575 -91 1722 -91 \n",
              "Q 1428 -91 1123 -41 \n",
              "Q 819 9 494 109 \n",
              "L 494 744 \n",
              "Q 775 591 1075 516 \n",
              "Q 1375 441 1709 441 \n",
              "Q 2250 441 2565 725 \n",
              "Q 2881 1009 2881 1497 \n",
              "Q 2881 1984 2565 2268 \n",
              "Q 2250 2553 1709 2553 \n",
              "Q 1456 2553 1204 2497 \n",
              "Q 953 2441 691 2322 \n",
              "L 691 4666 \n",
              "z\n",
              "\" transform=\"scale(0.015625)\"/>\n",
              "       </defs>\n",
              "       <use xlink:href=\"#DejaVuSans-35\"/>\n",
              "      </g>\n",
              "     </g>\n",
              "    </g>\n",
              "    <g id=\"xtick_2\">\n",
              "     <g id=\"line2d_3\">\n",
              "      <path d=\"M 237.120264 412.22175 \n",
              "L 237.120264 24.14175 \n",
              "\" clip-path=\"url(#pd0ab672b7b)\" style=\"fill: none; stroke: #ffffff; stroke-width: 0.8; stroke-linecap: square\"/>\n",
              "     </g>\n",
              "     <g id=\"line2d_4\">\n",
              "      <g>\n",
              "       <use xlink:href=\"#maa2cf2715f\" x=\"237.120264\" y=\"412.22175\" style=\"fill: #555555; stroke: #555555; stroke-width: 0.8\"/>\n",
              "      </g>\n",
              "     </g>\n",
              "     <g id=\"text_2\">\n",
              "      <!-- 10 -->\n",
              "      <g style=\"fill: #555555\" transform=\"translate(230.757764 426.820187) scale(0.1 -0.1)\">\n",
              "       <defs>\n",
              "        <path id=\"DejaVuSans-31\" d=\"M 794 531 \n",
              "L 1825 531 \n",
              "L 1825 4091 \n",
              "L 703 3866 \n",
              "L 703 4441 \n",
              "L 1819 4666 \n",
              "L 2450 4666 \n",
              "L 2450 531 \n",
              "L 3481 531 \n",
              "L 3481 0 \n",
              "L 794 0 \n",
              "L 794 531 \n",
              "z\n",
              "\" transform=\"scale(0.015625)\"/>\n",
              "        <path id=\"DejaVuSans-30\" d=\"M 2034 4250 \n",
              "Q 1547 4250 1301 3770 \n",
              "Q 1056 3291 1056 2328 \n",
              "Q 1056 1369 1301 889 \n",
              "Q 1547 409 2034 409 \n",
              "Q 2525 409 2770 889 \n",
              "Q 3016 1369 3016 2328 \n",
              "Q 3016 3291 2770 3770 \n",
              "Q 2525 4250 2034 4250 \n",
              "z\n",
              "M 2034 4750 \n",
              "Q 2819 4750 3233 4129 \n",
              "Q 3647 3509 3647 2328 \n",
              "Q 3647 1150 3233 529 \n",
              "Q 2819 -91 2034 -91 \n",
              "Q 1250 -91 836 529 \n",
              "Q 422 1150 422 2328 \n",
              "Q 422 3509 836 4129 \n",
              "Q 1250 4750 2034 4750 \n",
              "z\n",
              "\" transform=\"scale(0.015625)\"/>\n",
              "       </defs>\n",
              "       <use xlink:href=\"#DejaVuSans-31\"/>\n",
              "       <use xlink:href=\"#DejaVuSans-30\" x=\"63.623047\"/>\n",
              "      </g>\n",
              "     </g>\n",
              "    </g>\n",
              "    <g id=\"xtick_3\">\n",
              "     <g id=\"line2d_5\">\n",
              "      <path d=\"M 327.704679 412.22175 \n",
              "L 327.704679 24.14175 \n",
              "\" clip-path=\"url(#pd0ab672b7b)\" style=\"fill: none; stroke: #ffffff; stroke-width: 0.8; stroke-linecap: square\"/>\n",
              "     </g>\n",
              "     <g id=\"line2d_6\">\n",
              "      <g>\n",
              "       <use xlink:href=\"#maa2cf2715f\" x=\"327.704679\" y=\"412.22175\" style=\"fill: #555555; stroke: #555555; stroke-width: 0.8\"/>\n",
              "      </g>\n",
              "     </g>\n",
              "     <g id=\"text_3\">\n",
              "      <!-- 15 -->\n",
              "      <g style=\"fill: #555555\" transform=\"translate(321.342179 426.820187) scale(0.1 -0.1)\">\n",
              "       <use xlink:href=\"#DejaVuSans-31\"/>\n",
              "       <use xlink:href=\"#DejaVuSans-35\" x=\"63.623047\"/>\n",
              "      </g>\n",
              "     </g>\n",
              "    </g>\n",
              "    <g id=\"xtick_4\">\n",
              "     <g id=\"line2d_7\">\n",
              "      <path d=\"M 418.289095 412.22175 \n",
              "L 418.289095 24.14175 \n",
              "\" clip-path=\"url(#pd0ab672b7b)\" style=\"fill: none; stroke: #ffffff; stroke-width: 0.8; stroke-linecap: square\"/>\n",
              "     </g>\n",
              "     <g id=\"line2d_8\">\n",
              "      <g>\n",
              "       <use xlink:href=\"#maa2cf2715f\" x=\"418.289095\" y=\"412.22175\" style=\"fill: #555555; stroke: #555555; stroke-width: 0.8\"/>\n",
              "      </g>\n",
              "     </g>\n",
              "     <g id=\"text_4\">\n",
              "      <!-- 20 -->\n",
              "      <g style=\"fill: #555555\" transform=\"translate(411.926595 426.820187) scale(0.1 -0.1)\">\n",
              "       <defs>\n",
              "        <path id=\"DejaVuSans-32\" d=\"M 1228 531 \n",
              "L 3431 531 \n",
              "L 3431 0 \n",
              "L 469 0 \n",
              "L 469 531 \n",
              "Q 828 903 1448 1529 \n",
              "Q 2069 2156 2228 2338 \n",
              "Q 2531 2678 2651 2914 \n",
              "Q 2772 3150 2772 3378 \n",
              "Q 2772 3750 2511 3984 \n",
              "Q 2250 4219 1831 4219 \n",
              "Q 1534 4219 1204 4116 \n",
              "Q 875 4013 500 3803 \n",
              "L 500 4441 \n",
              "Q 881 4594 1212 4672 \n",
              "Q 1544 4750 1819 4750 \n",
              "Q 2544 4750 2975 4387 \n",
              "Q 3406 4025 3406 3419 \n",
              "Q 3406 3131 3298 2873 \n",
              "Q 3191 2616 2906 2266 \n",
              "Q 2828 2175 2409 1742 \n",
              "Q 1991 1309 1228 531 \n",
              "z\n",
              "\" transform=\"scale(0.015625)\"/>\n",
              "       </defs>\n",
              "       <use xlink:href=\"#DejaVuSans-32\"/>\n",
              "       <use xlink:href=\"#DejaVuSans-30\" x=\"63.623047\"/>\n",
              "      </g>\n",
              "     </g>\n",
              "    </g>\n",
              "    <g id=\"xtick_5\">\n",
              "     <g id=\"line2d_9\">\n",
              "      <path d=\"M 508.873511 412.22175 \n",
              "L 508.873511 24.14175 \n",
              "\" clip-path=\"url(#pd0ab672b7b)\" style=\"fill: none; stroke: #ffffff; stroke-width: 0.8; stroke-linecap: square\"/>\n",
              "     </g>\n",
              "     <g id=\"line2d_10\">\n",
              "      <g>\n",
              "       <use xlink:href=\"#maa2cf2715f\" x=\"508.873511\" y=\"412.22175\" style=\"fill: #555555; stroke: #555555; stroke-width: 0.8\"/>\n",
              "      </g>\n",
              "     </g>\n",
              "     <g id=\"text_5\">\n",
              "      <!-- 25 -->\n",
              "      <g style=\"fill: #555555\" transform=\"translate(502.511011 426.820187) scale(0.1 -0.1)\">\n",
              "       <use xlink:href=\"#DejaVuSans-32\"/>\n",
              "       <use xlink:href=\"#DejaVuSans-35\" x=\"63.623047\"/>\n",
              "      </g>\n",
              "     </g>\n",
              "    </g>\n",
              "    <g id=\"xtick_6\">\n",
              "     <g id=\"line2d_11\">\n",
              "      <path d=\"M 599.457926 412.22175 \n",
              "L 599.457926 24.14175 \n",
              "\" clip-path=\"url(#pd0ab672b7b)\" style=\"fill: none; stroke: #ffffff; stroke-width: 0.8; stroke-linecap: square\"/>\n",
              "     </g>\n",
              "     <g id=\"line2d_12\">\n",
              "      <g>\n",
              "       <use xlink:href=\"#maa2cf2715f\" x=\"599.457926\" y=\"412.22175\" style=\"fill: #555555; stroke: #555555; stroke-width: 0.8\"/>\n",
              "      </g>\n",
              "     </g>\n",
              "     <g id=\"text_6\">\n",
              "      <!-- 30 -->\n",
              "      <g style=\"fill: #555555\" transform=\"translate(593.095426 426.820187) scale(0.1 -0.1)\">\n",
              "       <defs>\n",
              "        <path id=\"DejaVuSans-33\" d=\"M 2597 2516 \n",
              "Q 3050 2419 3304 2112 \n",
              "Q 3559 1806 3559 1356 \n",
              "Q 3559 666 3084 287 \n",
              "Q 2609 -91 1734 -91 \n",
              "Q 1441 -91 1130 -33 \n",
              "Q 819 25 488 141 \n",
              "L 488 750 \n",
              "Q 750 597 1062 519 \n",
              "Q 1375 441 1716 441 \n",
              "Q 2309 441 2620 675 \n",
              "Q 2931 909 2931 1356 \n",
              "Q 2931 1769 2642 2001 \n",
              "Q 2353 2234 1838 2234 \n",
              "L 1294 2234 \n",
              "L 1294 2753 \n",
              "L 1863 2753 \n",
              "Q 2328 2753 2575 2939 \n",
              "Q 2822 3125 2822 3475 \n",
              "Q 2822 3834 2567 4026 \n",
              "Q 2313 4219 1838 4219 \n",
              "Q 1578 4219 1281 4162 \n",
              "Q 984 4106 628 3988 \n",
              "L 628 4550 \n",
              "Q 988 4650 1302 4700 \n",
              "Q 1616 4750 1894 4750 \n",
              "Q 2613 4750 3031 4423 \n",
              "Q 3450 4097 3450 3541 \n",
              "Q 3450 3153 3228 2886 \n",
              "Q 3006 2619 2597 2516 \n",
              "z\n",
              "\" transform=\"scale(0.015625)\"/>\n",
              "       </defs>\n",
              "       <use xlink:href=\"#DejaVuSans-33\"/>\n",
              "       <use xlink:href=\"#DejaVuSans-30\" x=\"63.623047\"/>\n",
              "      </g>\n",
              "     </g>\n",
              "    </g>\n",
              "    <g id=\"text_7\">\n",
              "     <!-- Number of clusters -->\n",
              "     <g style=\"fill: #555555\" transform=\"translate(288.590938 442.018) scale(0.12 -0.12)\">\n",
              "      <defs>\n",
              "       <path id=\"DejaVuSans-4e\" d=\"M 628 4666 \n",
              "L 1478 4666 \n",
              "L 3547 763 \n",
              "L 3547 4666 \n",
              "L 4159 4666 \n",
              "L 4159 0 \n",
              "L 3309 0 \n",
              "L 1241 3903 \n",
              "L 1241 0 \n",
              "L 628 0 \n",
              "L 628 4666 \n",
              "z\n",
              "\" transform=\"scale(0.015625)\"/>\n",
              "       <path id=\"DejaVuSans-75\" d=\"M 544 1381 \n",
              "L 544 3500 \n",
              "L 1119 3500 \n",
              "L 1119 1403 \n",
              "Q 1119 906 1312 657 \n",
              "Q 1506 409 1894 409 \n",
              "Q 2359 409 2629 706 \n",
              "Q 2900 1003 2900 1516 \n",
              "L 2900 3500 \n",
              "L 3475 3500 \n",
              "L 3475 0 \n",
              "L 2900 0 \n",
              "L 2900 538 \n",
              "Q 2691 219 2414 64 \n",
              "Q 2138 -91 1772 -91 \n",
              "Q 1169 -91 856 284 \n",
              "Q 544 659 544 1381 \n",
              "z\n",
              "M 1991 3584 \n",
              "L 1991 3584 \n",
              "z\n",
              "\" transform=\"scale(0.015625)\"/>\n",
              "       <path id=\"DejaVuSans-6d\" d=\"M 3328 2828 \n",
              "Q 3544 3216 3844 3400 \n",
              "Q 4144 3584 4550 3584 \n",
              "Q 5097 3584 5394 3201 \n",
              "Q 5691 2819 5691 2113 \n",
              "L 5691 0 \n",
              "L 5113 0 \n",
              "L 5113 2094 \n",
              "Q 5113 2597 4934 2840 \n",
              "Q 4756 3084 4391 3084 \n",
              "Q 3944 3084 3684 2787 \n",
              "Q 3425 2491 3425 1978 \n",
              "L 3425 0 \n",
              "L 2847 0 \n",
              "L 2847 2094 \n",
              "Q 2847 2600 2669 2842 \n",
              "Q 2491 3084 2119 3084 \n",
              "Q 1678 3084 1418 2786 \n",
              "Q 1159 2488 1159 1978 \n",
              "L 1159 0 \n",
              "L 581 0 \n",
              "L 581 3500 \n",
              "L 1159 3500 \n",
              "L 1159 2956 \n",
              "Q 1356 3278 1631 3431 \n",
              "Q 1906 3584 2284 3584 \n",
              "Q 2666 3584 2933 3390 \n",
              "Q 3200 3197 3328 2828 \n",
              "z\n",
              "\" transform=\"scale(0.015625)\"/>\n",
              "       <path id=\"DejaVuSans-62\" d=\"M 3116 1747 \n",
              "Q 3116 2381 2855 2742 \n",
              "Q 2594 3103 2138 3103 \n",
              "Q 1681 3103 1420 2742 \n",
              "Q 1159 2381 1159 1747 \n",
              "Q 1159 1113 1420 752 \n",
              "Q 1681 391 2138 391 \n",
              "Q 2594 391 2855 752 \n",
              "Q 3116 1113 3116 1747 \n",
              "z\n",
              "M 1159 2969 \n",
              "Q 1341 3281 1617 3432 \n",
              "Q 1894 3584 2278 3584 \n",
              "Q 2916 3584 3314 3078 \n",
              "Q 3713 2572 3713 1747 \n",
              "Q 3713 922 3314 415 \n",
              "Q 2916 -91 2278 -91 \n",
              "Q 1894 -91 1617 61 \n",
              "Q 1341 213 1159 525 \n",
              "L 1159 0 \n",
              "L 581 0 \n",
              "L 581 4863 \n",
              "L 1159 4863 \n",
              "L 1159 2969 \n",
              "z\n",
              "\" transform=\"scale(0.015625)\"/>\n",
              "       <path id=\"DejaVuSans-65\" d=\"M 3597 1894 \n",
              "L 3597 1613 \n",
              "L 953 1613 \n",
              "Q 991 1019 1311 708 \n",
              "Q 1631 397 2203 397 \n",
              "Q 2534 397 2845 478 \n",
              "Q 3156 559 3463 722 \n",
              "L 3463 178 \n",
              "Q 3153 47 2828 -22 \n",
              "Q 2503 -91 2169 -91 \n",
              "Q 1331 -91 842 396 \n",
              "Q 353 884 353 1716 \n",
              "Q 353 2575 817 3079 \n",
              "Q 1281 3584 2069 3584 \n",
              "Q 2775 3584 3186 3129 \n",
              "Q 3597 2675 3597 1894 \n",
              "z\n",
              "M 3022 2063 \n",
              "Q 3016 2534 2758 2815 \n",
              "Q 2500 3097 2075 3097 \n",
              "Q 1594 3097 1305 2825 \n",
              "Q 1016 2553 972 2059 \n",
              "L 3022 2063 \n",
              "z\n",
              "\" transform=\"scale(0.015625)\"/>\n",
              "       <path id=\"DejaVuSans-72\" d=\"M 2631 2963 \n",
              "Q 2534 3019 2420 3045 \n",
              "Q 2306 3072 2169 3072 \n",
              "Q 1681 3072 1420 2755 \n",
              "Q 1159 2438 1159 1844 \n",
              "L 1159 0 \n",
              "L 581 0 \n",
              "L 581 3500 \n",
              "L 1159 3500 \n",
              "L 1159 2956 \n",
              "Q 1341 3275 1631 3429 \n",
              "Q 1922 3584 2338 3584 \n",
              "Q 2397 3584 2469 3576 \n",
              "Q 2541 3569 2628 3553 \n",
              "L 2631 2963 \n",
              "z\n",
              "\" transform=\"scale(0.015625)\"/>\n",
              "       <path id=\"DejaVuSans-20\" transform=\"scale(0.015625)\"/>\n",
              "       <path id=\"DejaVuSans-6f\" d=\"M 1959 3097 \n",
              "Q 1497 3097 1228 2736 \n",
              "Q 959 2375 959 1747 \n",
              "Q 959 1119 1226 758 \n",
              "Q 1494 397 1959 397 \n",
              "Q 2419 397 2687 759 \n",
              "Q 2956 1122 2956 1747 \n",
              "Q 2956 2369 2687 2733 \n",
              "Q 2419 3097 1959 3097 \n",
              "z\n",
              "M 1959 3584 \n",
              "Q 2709 3584 3137 3096 \n",
              "Q 3566 2609 3566 1747 \n",
              "Q 3566 888 3137 398 \n",
              "Q 2709 -91 1959 -91 \n",
              "Q 1206 -91 779 398 \n",
              "Q 353 888 353 1747 \n",
              "Q 353 2609 779 3096 \n",
              "Q 1206 3584 1959 3584 \n",
              "z\n",
              "\" transform=\"scale(0.015625)\"/>\n",
              "       <path id=\"DejaVuSans-66\" d=\"M 2375 4863 \n",
              "L 2375 4384 \n",
              "L 1825 4384 \n",
              "Q 1516 4384 1395 4259 \n",
              "Q 1275 4134 1275 3809 \n",
              "L 1275 3500 \n",
              "L 2222 3500 \n",
              "L 2222 3053 \n",
              "L 1275 3053 \n",
              "L 1275 0 \n",
              "L 697 0 \n",
              "L 697 3053 \n",
              "L 147 3053 \n",
              "L 147 3500 \n",
              "L 697 3500 \n",
              "L 697 3744 \n",
              "Q 697 4328 969 4595 \n",
              "Q 1241 4863 1831 4863 \n",
              "L 2375 4863 \n",
              "z\n",
              "\" transform=\"scale(0.015625)\"/>\n",
              "       <path id=\"DejaVuSans-63\" d=\"M 3122 3366 \n",
              "L 3122 2828 \n",
              "Q 2878 2963 2633 3030 \n",
              "Q 2388 3097 2138 3097 \n",
              "Q 1578 3097 1268 2742 \n",
              "Q 959 2388 959 1747 \n",
              "Q 959 1106 1268 751 \n",
              "Q 1578 397 2138 397 \n",
              "Q 2388 397 2633 464 \n",
              "Q 2878 531 3122 666 \n",
              "L 3122 134 \n",
              "Q 2881 22 2623 -34 \n",
              "Q 2366 -91 2075 -91 \n",
              "Q 1284 -91 818 406 \n",
              "Q 353 903 353 1747 \n",
              "Q 353 2603 823 3093 \n",
              "Q 1294 3584 2113 3584 \n",
              "Q 2378 3584 2631 3529 \n",
              "Q 2884 3475 3122 3366 \n",
              "z\n",
              "\" transform=\"scale(0.015625)\"/>\n",
              "       <path id=\"DejaVuSans-6c\" d=\"M 603 4863 \n",
              "L 1178 4863 \n",
              "L 1178 0 \n",
              "L 603 0 \n",
              "L 603 4863 \n",
              "z\n",
              "\" transform=\"scale(0.015625)\"/>\n",
              "       <path id=\"DejaVuSans-73\" d=\"M 2834 3397 \n",
              "L 2834 2853 \n",
              "Q 2591 2978 2328 3040 \n",
              "Q 2066 3103 1784 3103 \n",
              "Q 1356 3103 1142 2972 \n",
              "Q 928 2841 928 2578 \n",
              "Q 928 2378 1081 2264 \n",
              "Q 1234 2150 1697 2047 \n",
              "L 1894 2003 \n",
              "Q 2506 1872 2764 1633 \n",
              "Q 3022 1394 3022 966 \n",
              "Q 3022 478 2636 193 \n",
              "Q 2250 -91 1575 -91 \n",
              "Q 1294 -91 989 -36 \n",
              "Q 684 19 347 128 \n",
              "L 347 722 \n",
              "Q 666 556 975 473 \n",
              "Q 1284 391 1588 391 \n",
              "Q 1994 391 2212 530 \n",
              "Q 2431 669 2431 922 \n",
              "Q 2431 1156 2273 1281 \n",
              "Q 2116 1406 1581 1522 \n",
              "L 1381 1569 \n",
              "Q 847 1681 609 1914 \n",
              "Q 372 2147 372 2553 \n",
              "Q 372 3047 722 3315 \n",
              "Q 1072 3584 1716 3584 \n",
              "Q 2034 3584 2315 3537 \n",
              "Q 2597 3491 2834 3397 \n",
              "z\n",
              "\" transform=\"scale(0.015625)\"/>\n",
              "       <path id=\"DejaVuSans-74\" d=\"M 1172 4494 \n",
              "L 1172 3500 \n",
              "L 2356 3500 \n",
              "L 2356 3053 \n",
              "L 1172 3053 \n",
              "L 1172 1153 \n",
              "Q 1172 725 1289 603 \n",
              "Q 1406 481 1766 481 \n",
              "L 2356 481 \n",
              "L 2356 0 \n",
              "L 1766 0 \n",
              "Q 1100 0 847 248 \n",
              "Q 594 497 594 1153 \n",
              "L 594 3053 \n",
              "L 172 3053 \n",
              "L 172 3500 \n",
              "L 594 3500 \n",
              "L 594 4494 \n",
              "L 1172 4494 \n",
              "z\n",
              "\" transform=\"scale(0.015625)\"/>\n",
              "      </defs>\n",
              "      <use xlink:href=\"#DejaVuSans-4e\"/>\n",
              "      <use xlink:href=\"#DejaVuSans-75\" x=\"74.804688\"/>\n",
              "      <use xlink:href=\"#DejaVuSans-6d\" x=\"138.183594\"/>\n",
              "      <use xlink:href=\"#DejaVuSans-62\" x=\"235.595703\"/>\n",
              "      <use xlink:href=\"#DejaVuSans-65\" x=\"299.072266\"/>\n",
              "      <use xlink:href=\"#DejaVuSans-72\" x=\"360.595703\"/>\n",
              "      <use xlink:href=\"#DejaVuSans-20\" x=\"401.708984\"/>\n",
              "      <use xlink:href=\"#DejaVuSans-6f\" x=\"433.496094\"/>\n",
              "      <use xlink:href=\"#DejaVuSans-66\" x=\"494.677734\"/>\n",
              "      <use xlink:href=\"#DejaVuSans-20\" x=\"529.882812\"/>\n",
              "      <use xlink:href=\"#DejaVuSans-63\" x=\"561.669922\"/>\n",
              "      <use xlink:href=\"#DejaVuSans-6c\" x=\"616.650391\"/>\n",
              "      <use xlink:href=\"#DejaVuSans-75\" x=\"644.433594\"/>\n",
              "      <use xlink:href=\"#DejaVuSans-73\" x=\"707.8125\"/>\n",
              "      <use xlink:href=\"#DejaVuSans-74\" x=\"759.912109\"/>\n",
              "      <use xlink:href=\"#DejaVuSans-65\" x=\"799.121094\"/>\n",
              "      <use xlink:href=\"#DejaVuSans-72\" x=\"860.644531\"/>\n",
              "      <use xlink:href=\"#DejaVuSans-73\" x=\"901.757812\"/>\n",
              "     </g>\n",
              "    </g>\n",
              "   </g>\n",
              "   <g id=\"matplotlib.axis_2\">\n",
              "    <g id=\"ytick_1\">\n",
              "     <g id=\"line2d_13\">\n",
              "      <path d=\"M 66.821562 372.703779 \n",
              "L 624.821563 372.703779 \n",
              "\" clip-path=\"url(#pd0ab672b7b)\" style=\"fill: none; stroke: #ffffff; stroke-width: 0.8; stroke-linecap: square\"/>\n",
              "     </g>\n",
              "     <g id=\"line2d_14\">\n",
              "      <defs>\n",
              "       <path id=\"mc41d8b09aa\" d=\"M 0 0 \n",
              "L -3.5 0 \n",
              "\" style=\"stroke: #555555; stroke-width: 0.8\"/>\n",
              "      </defs>\n",
              "      <g>\n",
              "       <use xlink:href=\"#mc41d8b09aa\" x=\"66.821562\" y=\"372.703779\" style=\"fill: #555555; stroke: #555555; stroke-width: 0.8\"/>\n",
              "      </g>\n",
              "     </g>\n",
              "     <g id=\"text_8\">\n",
              "      <!-- −0.075 -->\n",
              "      <g style=\"fill: #555555\" transform=\"translate(22.81375 376.502998) scale(0.1 -0.1)\">\n",
              "       <defs>\n",
              "        <path id=\"DejaVuSans-2212\" d=\"M 678 2272 \n",
              "L 4684 2272 \n",
              "L 4684 1741 \n",
              "L 678 1741 \n",
              "L 678 2272 \n",
              "z\n",
              "\" transform=\"scale(0.015625)\"/>\n",
              "        <path id=\"DejaVuSans-2e\" d=\"M 684 794 \n",
              "L 1344 794 \n",
              "L 1344 0 \n",
              "L 684 0 \n",
              "L 684 794 \n",
              "z\n",
              "\" transform=\"scale(0.015625)\"/>\n",
              "        <path id=\"DejaVuSans-37\" d=\"M 525 4666 \n",
              "L 3525 4666 \n",
              "L 3525 4397 \n",
              "L 1831 0 \n",
              "L 1172 0 \n",
              "L 2766 4134 \n",
              "L 525 4134 \n",
              "L 525 4666 \n",
              "z\n",
              "\" transform=\"scale(0.015625)\"/>\n",
              "       </defs>\n",
              "       <use xlink:href=\"#DejaVuSans-2212\"/>\n",
              "       <use xlink:href=\"#DejaVuSans-30\" x=\"83.789062\"/>\n",
              "       <use xlink:href=\"#DejaVuSans-2e\" x=\"147.412109\"/>\n",
              "       <use xlink:href=\"#DejaVuSans-30\" x=\"179.199219\"/>\n",
              "       <use xlink:href=\"#DejaVuSans-37\" x=\"242.822266\"/>\n",
              "       <use xlink:href=\"#DejaVuSans-35\" x=\"306.445312\"/>\n",
              "      </g>\n",
              "     </g>\n",
              "    </g>\n",
              "    <g id=\"ytick_2\">\n",
              "     <g id=\"line2d_15\">\n",
              "      <path d=\"M 66.821562 328.80075 \n",
              "L 624.821563 328.80075 \n",
              "\" clip-path=\"url(#pd0ab672b7b)\" style=\"fill: none; stroke: #ffffff; stroke-width: 0.8; stroke-linecap: square\"/>\n",
              "     </g>\n",
              "     <g id=\"line2d_16\">\n",
              "      <g>\n",
              "       <use xlink:href=\"#mc41d8b09aa\" x=\"66.821562\" y=\"328.80075\" style=\"fill: #555555; stroke: #555555; stroke-width: 0.8\"/>\n",
              "      </g>\n",
              "     </g>\n",
              "     <g id=\"text_9\">\n",
              "      <!-- −0.050 -->\n",
              "      <g style=\"fill: #555555\" transform=\"translate(22.81375 332.599969) scale(0.1 -0.1)\">\n",
              "       <use xlink:href=\"#DejaVuSans-2212\"/>\n",
              "       <use xlink:href=\"#DejaVuSans-30\" x=\"83.789062\"/>\n",
              "       <use xlink:href=\"#DejaVuSans-2e\" x=\"147.412109\"/>\n",
              "       <use xlink:href=\"#DejaVuSans-30\" x=\"179.199219\"/>\n",
              "       <use xlink:href=\"#DejaVuSans-35\" x=\"242.822266\"/>\n",
              "       <use xlink:href=\"#DejaVuSans-30\" x=\"306.445312\"/>\n",
              "      </g>\n",
              "     </g>\n",
              "    </g>\n",
              "    <g id=\"ytick_3\">\n",
              "     <g id=\"line2d_17\">\n",
              "      <path d=\"M 66.821562 284.897721 \n",
              "L 624.821563 284.897721 \n",
              "\" clip-path=\"url(#pd0ab672b7b)\" style=\"fill: none; stroke: #ffffff; stroke-width: 0.8; stroke-linecap: square\"/>\n",
              "     </g>\n",
              "     <g id=\"line2d_18\">\n",
              "      <g>\n",
              "       <use xlink:href=\"#mc41d8b09aa\" x=\"66.821562\" y=\"284.897721\" style=\"fill: #555555; stroke: #555555; stroke-width: 0.8\"/>\n",
              "      </g>\n",
              "     </g>\n",
              "     <g id=\"text_10\">\n",
              "      <!-- −0.025 -->\n",
              "      <g style=\"fill: #555555\" transform=\"translate(22.81375 288.69694) scale(0.1 -0.1)\">\n",
              "       <use xlink:href=\"#DejaVuSans-2212\"/>\n",
              "       <use xlink:href=\"#DejaVuSans-30\" x=\"83.789062\"/>\n",
              "       <use xlink:href=\"#DejaVuSans-2e\" x=\"147.412109\"/>\n",
              "       <use xlink:href=\"#DejaVuSans-30\" x=\"179.199219\"/>\n",
              "       <use xlink:href=\"#DejaVuSans-32\" x=\"242.822266\"/>\n",
              "       <use xlink:href=\"#DejaVuSans-35\" x=\"306.445312\"/>\n",
              "      </g>\n",
              "     </g>\n",
              "    </g>\n",
              "    <g id=\"ytick_4\">\n",
              "     <g id=\"line2d_19\">\n",
              "      <path d=\"M 66.821562 240.994692 \n",
              "L 624.821563 240.994692 \n",
              "\" clip-path=\"url(#pd0ab672b7b)\" style=\"fill: none; stroke: #ffffff; stroke-width: 0.8; stroke-linecap: square\"/>\n",
              "     </g>\n",
              "     <g id=\"line2d_20\">\n",
              "      <g>\n",
              "       <use xlink:href=\"#mc41d8b09aa\" x=\"66.821562\" y=\"240.994692\" style=\"fill: #555555; stroke: #555555; stroke-width: 0.8\"/>\n",
              "      </g>\n",
              "     </g>\n",
              "     <g id=\"text_11\">\n",
              "      <!-- 0.000 -->\n",
              "      <g style=\"fill: #555555\" transform=\"translate(31.193437 244.79391) scale(0.1 -0.1)\">\n",
              "       <use xlink:href=\"#DejaVuSans-30\"/>\n",
              "       <use xlink:href=\"#DejaVuSans-2e\" x=\"63.623047\"/>\n",
              "       <use xlink:href=\"#DejaVuSans-30\" x=\"95.410156\"/>\n",
              "       <use xlink:href=\"#DejaVuSans-30\" x=\"159.033203\"/>\n",
              "       <use xlink:href=\"#DejaVuSans-30\" x=\"222.65625\"/>\n",
              "      </g>\n",
              "     </g>\n",
              "    </g>\n",
              "    <g id=\"ytick_5\">\n",
              "     <g id=\"line2d_21\">\n",
              "      <path d=\"M 66.821562 197.091662 \n",
              "L 624.821563 197.091662 \n",
              "\" clip-path=\"url(#pd0ab672b7b)\" style=\"fill: none; stroke: #ffffff; stroke-width: 0.8; stroke-linecap: square\"/>\n",
              "     </g>\n",
              "     <g id=\"line2d_22\">\n",
              "      <g>\n",
              "       <use xlink:href=\"#mc41d8b09aa\" x=\"66.821562\" y=\"197.091662\" style=\"fill: #555555; stroke: #555555; stroke-width: 0.8\"/>\n",
              "      </g>\n",
              "     </g>\n",
              "     <g id=\"text_12\">\n",
              "      <!-- 0.025 -->\n",
              "      <g style=\"fill: #555555\" transform=\"translate(31.193437 200.890881) scale(0.1 -0.1)\">\n",
              "       <use xlink:href=\"#DejaVuSans-30\"/>\n",
              "       <use xlink:href=\"#DejaVuSans-2e\" x=\"63.623047\"/>\n",
              "       <use xlink:href=\"#DejaVuSans-30\" x=\"95.410156\"/>\n",
              "       <use xlink:href=\"#DejaVuSans-32\" x=\"159.033203\"/>\n",
              "       <use xlink:href=\"#DejaVuSans-35\" x=\"222.65625\"/>\n",
              "      </g>\n",
              "     </g>\n",
              "    </g>\n",
              "    <g id=\"ytick_6\">\n",
              "     <g id=\"line2d_23\">\n",
              "      <path d=\"M 66.821562 153.188633 \n",
              "L 624.821563 153.188633 \n",
              "\" clip-path=\"url(#pd0ab672b7b)\" style=\"fill: none; stroke: #ffffff; stroke-width: 0.8; stroke-linecap: square\"/>\n",
              "     </g>\n",
              "     <g id=\"line2d_24\">\n",
              "      <g>\n",
              "       <use xlink:href=\"#mc41d8b09aa\" x=\"66.821562\" y=\"153.188633\" style=\"fill: #555555; stroke: #555555; stroke-width: 0.8\"/>\n",
              "      </g>\n",
              "     </g>\n",
              "     <g id=\"text_13\">\n",
              "      <!-- 0.050 -->\n",
              "      <g style=\"fill: #555555\" transform=\"translate(31.193437 156.987852) scale(0.1 -0.1)\">\n",
              "       <use xlink:href=\"#DejaVuSans-30\"/>\n",
              "       <use xlink:href=\"#DejaVuSans-2e\" x=\"63.623047\"/>\n",
              "       <use xlink:href=\"#DejaVuSans-30\" x=\"95.410156\"/>\n",
              "       <use xlink:href=\"#DejaVuSans-35\" x=\"159.033203\"/>\n",
              "       <use xlink:href=\"#DejaVuSans-30\" x=\"222.65625\"/>\n",
              "      </g>\n",
              "     </g>\n",
              "    </g>\n",
              "    <g id=\"ytick_7\">\n",
              "     <g id=\"line2d_25\">\n",
              "      <path d=\"M 66.821562 109.285604 \n",
              "L 624.821563 109.285604 \n",
              "\" clip-path=\"url(#pd0ab672b7b)\" style=\"fill: none; stroke: #ffffff; stroke-width: 0.8; stroke-linecap: square\"/>\n",
              "     </g>\n",
              "     <g id=\"line2d_26\">\n",
              "      <g>\n",
              "       <use xlink:href=\"#mc41d8b09aa\" x=\"66.821562\" y=\"109.285604\" style=\"fill: #555555; stroke: #555555; stroke-width: 0.8\"/>\n",
              "      </g>\n",
              "     </g>\n",
              "     <g id=\"text_14\">\n",
              "      <!-- 0.075 -->\n",
              "      <g style=\"fill: #555555\" transform=\"translate(31.193437 113.084822) scale(0.1 -0.1)\">\n",
              "       <use xlink:href=\"#DejaVuSans-30\"/>\n",
              "       <use xlink:href=\"#DejaVuSans-2e\" x=\"63.623047\"/>\n",
              "       <use xlink:href=\"#DejaVuSans-30\" x=\"95.410156\"/>\n",
              "       <use xlink:href=\"#DejaVuSans-37\" x=\"159.033203\"/>\n",
              "       <use xlink:href=\"#DejaVuSans-35\" x=\"222.65625\"/>\n",
              "      </g>\n",
              "     </g>\n",
              "    </g>\n",
              "    <g id=\"ytick_8\">\n",
              "     <g id=\"line2d_27\">\n",
              "      <path d=\"M 66.821562 65.382574 \n",
              "L 624.821563 65.382574 \n",
              "\" clip-path=\"url(#pd0ab672b7b)\" style=\"fill: none; stroke: #ffffff; stroke-width: 0.8; stroke-linecap: square\"/>\n",
              "     </g>\n",
              "     <g id=\"line2d_28\">\n",
              "      <g>\n",
              "       <use xlink:href=\"#mc41d8b09aa\" x=\"66.821562\" y=\"65.382574\" style=\"fill: #555555; stroke: #555555; stroke-width: 0.8\"/>\n",
              "      </g>\n",
              "     </g>\n",
              "     <g id=\"text_15\">\n",
              "      <!-- 0.100 -->\n",
              "      <g style=\"fill: #555555\" transform=\"translate(31.193437 69.181793) scale(0.1 -0.1)\">\n",
              "       <use xlink:href=\"#DejaVuSans-30\"/>\n",
              "       <use xlink:href=\"#DejaVuSans-2e\" x=\"63.623047\"/>\n",
              "       <use xlink:href=\"#DejaVuSans-31\" x=\"95.410156\"/>\n",
              "       <use xlink:href=\"#DejaVuSans-30\" x=\"159.033203\"/>\n",
              "       <use xlink:href=\"#DejaVuSans-30\" x=\"222.65625\"/>\n",
              "      </g>\n",
              "     </g>\n",
              "    </g>\n",
              "    <g id=\"text_16\">\n",
              "     <!-- Silhouette Score -->\n",
              "     <g style=\"fill: #555555\" transform=\"translate(16.318125 267.398625) rotate(-90) scale(0.12 -0.12)\">\n",
              "      <defs>\n",
              "       <path id=\"DejaVuSans-53\" d=\"M 3425 4513 \n",
              "L 3425 3897 \n",
              "Q 3066 4069 2747 4153 \n",
              "Q 2428 4238 2131 4238 \n",
              "Q 1616 4238 1336 4038 \n",
              "Q 1056 3838 1056 3469 \n",
              "Q 1056 3159 1242 3001 \n",
              "Q 1428 2844 1947 2747 \n",
              "L 2328 2669 \n",
              "Q 3034 2534 3370 2195 \n",
              "Q 3706 1856 3706 1288 \n",
              "Q 3706 609 3251 259 \n",
              "Q 2797 -91 1919 -91 \n",
              "Q 1588 -91 1214 -16 \n",
              "Q 841 59 441 206 \n",
              "L 441 856 \n",
              "Q 825 641 1194 531 \n",
              "Q 1563 422 1919 422 \n",
              "Q 2459 422 2753 634 \n",
              "Q 3047 847 3047 1241 \n",
              "Q 3047 1584 2836 1778 \n",
              "Q 2625 1972 2144 2069 \n",
              "L 1759 2144 \n",
              "Q 1053 2284 737 2584 \n",
              "Q 422 2884 422 3419 \n",
              "Q 422 4038 858 4394 \n",
              "Q 1294 4750 2059 4750 \n",
              "Q 2388 4750 2728 4690 \n",
              "Q 3069 4631 3425 4513 \n",
              "z\n",
              "\" transform=\"scale(0.015625)\"/>\n",
              "       <path id=\"DejaVuSans-69\" d=\"M 603 3500 \n",
              "L 1178 3500 \n",
              "L 1178 0 \n",
              "L 603 0 \n",
              "L 603 3500 \n",
              "z\n",
              "M 603 4863 \n",
              "L 1178 4863 \n",
              "L 1178 4134 \n",
              "L 603 4134 \n",
              "L 603 4863 \n",
              "z\n",
              "\" transform=\"scale(0.015625)\"/>\n",
              "       <path id=\"DejaVuSans-68\" d=\"M 3513 2113 \n",
              "L 3513 0 \n",
              "L 2938 0 \n",
              "L 2938 2094 \n",
              "Q 2938 2591 2744 2837 \n",
              "Q 2550 3084 2163 3084 \n",
              "Q 1697 3084 1428 2787 \n",
              "Q 1159 2491 1159 1978 \n",
              "L 1159 0 \n",
              "L 581 0 \n",
              "L 581 4863 \n",
              "L 1159 4863 \n",
              "L 1159 2956 \n",
              "Q 1366 3272 1645 3428 \n",
              "Q 1925 3584 2291 3584 \n",
              "Q 2894 3584 3203 3211 \n",
              "Q 3513 2838 3513 2113 \n",
              "z\n",
              "\" transform=\"scale(0.015625)\"/>\n",
              "      </defs>\n",
              "      <use xlink:href=\"#DejaVuSans-53\"/>\n",
              "      <use xlink:href=\"#DejaVuSans-69\" x=\"63.476562\"/>\n",
              "      <use xlink:href=\"#DejaVuSans-6c\" x=\"91.259766\"/>\n",
              "      <use xlink:href=\"#DejaVuSans-68\" x=\"119.042969\"/>\n",
              "      <use xlink:href=\"#DejaVuSans-6f\" x=\"182.421875\"/>\n",
              "      <use xlink:href=\"#DejaVuSans-75\" x=\"243.603516\"/>\n",
              "      <use xlink:href=\"#DejaVuSans-65\" x=\"306.982422\"/>\n",
              "      <use xlink:href=\"#DejaVuSans-74\" x=\"368.505859\"/>\n",
              "      <use xlink:href=\"#DejaVuSans-74\" x=\"407.714844\"/>\n",
              "      <use xlink:href=\"#DejaVuSans-65\" x=\"446.923828\"/>\n",
              "      <use xlink:href=\"#DejaVuSans-20\" x=\"508.447266\"/>\n",
              "      <use xlink:href=\"#DejaVuSans-53\" x=\"540.234375\"/>\n",
              "      <use xlink:href=\"#DejaVuSans-63\" x=\"603.710938\"/>\n",
              "      <use xlink:href=\"#DejaVuSans-6f\" x=\"658.691406\"/>\n",
              "      <use xlink:href=\"#DejaVuSans-72\" x=\"719.873047\"/>\n",
              "      <use xlink:href=\"#DejaVuSans-65\" x=\"758.736328\"/>\n",
              "     </g>\n",
              "    </g>\n",
              "   </g>\n",
              "   <g id=\"line2d_29\">\n",
              "    <path d=\"M 92.185199 41.78175 \n",
              "L 110.302082 44.647933 \n",
              "L 128.418965 394.58175 \n",
              "L 146.535848 393.033892 \n",
              "L 164.652731 392.811157 \n",
              "L 182.769614 392.588368 \n",
              "L 200.886498 392.474378 \n",
              "L 219.003381 391.766769 \n",
              "L 237.120264 390.443545 \n",
              "L 255.237147 390.222035 \n",
              "L 273.35403 389.453488 \n",
              "L 291.470913 389.232299 \n",
              "L 309.587796 388.659706 \n",
              "L 327.704679 388.295288 \n",
              "L 345.821563 388.195622 \n",
              "L 363.938446 387.942485 \n",
              "L 382.055329 387.599879 \n",
              "L 400.172212 387.381734 \n",
              "L 418.289095 386.978061 \n",
              "L 436.405978 386.760561 \n",
              "L 454.522861 386.545764 \n",
              "L 472.639744 385.619816 \n",
              "L 490.756627 385.14438 \n",
              "L 508.873511 384.783206 \n",
              "L 526.990394 384.524972 \n",
              "L 545.107277 384.225203 \n",
              "L 563.22416 384.011206 \n",
              "L 581.341043 383.579834 \n",
              "L 599.457926 382.569582 \n",
              "\" clip-path=\"url(#pd0ab672b7b)\" style=\"fill: none; stroke: #0000ff; stroke-width: 1.5; stroke-linecap: square\"/>\n",
              "    <defs>\n",
              "     <path id=\"m8aedec9a16\" d=\"M -3 3 \n",
              "L 3 -3 \n",
              "M -3 -3 \n",
              "L 3 3 \n",
              "\" style=\"stroke: #0000ff\"/>\n",
              "    </defs>\n",
              "    <g clip-path=\"url(#pd0ab672b7b)\">\n",
              "     <use xlink:href=\"#m8aedec9a16\" x=\"92.185199\" y=\"41.78175\" style=\"fill: #0000ff; stroke: #0000ff\"/>\n",
              "     <use xlink:href=\"#m8aedec9a16\" x=\"110.302082\" y=\"44.647933\" style=\"fill: #0000ff; stroke: #0000ff\"/>\n",
              "     <use xlink:href=\"#m8aedec9a16\" x=\"128.418965\" y=\"394.58175\" style=\"fill: #0000ff; stroke: #0000ff\"/>\n",
              "     <use xlink:href=\"#m8aedec9a16\" x=\"146.535848\" y=\"393.033892\" style=\"fill: #0000ff; stroke: #0000ff\"/>\n",
              "     <use xlink:href=\"#m8aedec9a16\" x=\"164.652731\" y=\"392.811157\" style=\"fill: #0000ff; stroke: #0000ff\"/>\n",
              "     <use xlink:href=\"#m8aedec9a16\" x=\"182.769614\" y=\"392.588368\" style=\"fill: #0000ff; stroke: #0000ff\"/>\n",
              "     <use xlink:href=\"#m8aedec9a16\" x=\"200.886498\" y=\"392.474378\" style=\"fill: #0000ff; stroke: #0000ff\"/>\n",
              "     <use xlink:href=\"#m8aedec9a16\" x=\"219.003381\" y=\"391.766769\" style=\"fill: #0000ff; stroke: #0000ff\"/>\n",
              "     <use xlink:href=\"#m8aedec9a16\" x=\"237.120264\" y=\"390.443545\" style=\"fill: #0000ff; stroke: #0000ff\"/>\n",
              "     <use xlink:href=\"#m8aedec9a16\" x=\"255.237147\" y=\"390.222035\" style=\"fill: #0000ff; stroke: #0000ff\"/>\n",
              "     <use xlink:href=\"#m8aedec9a16\" x=\"273.35403\" y=\"389.453488\" style=\"fill: #0000ff; stroke: #0000ff\"/>\n",
              "     <use xlink:href=\"#m8aedec9a16\" x=\"291.470913\" y=\"389.232299\" style=\"fill: #0000ff; stroke: #0000ff\"/>\n",
              "     <use xlink:href=\"#m8aedec9a16\" x=\"309.587796\" y=\"388.659706\" style=\"fill: #0000ff; stroke: #0000ff\"/>\n",
              "     <use xlink:href=\"#m8aedec9a16\" x=\"327.704679\" y=\"388.295288\" style=\"fill: #0000ff; stroke: #0000ff\"/>\n",
              "     <use xlink:href=\"#m8aedec9a16\" x=\"345.821563\" y=\"388.195622\" style=\"fill: #0000ff; stroke: #0000ff\"/>\n",
              "     <use xlink:href=\"#m8aedec9a16\" x=\"363.938446\" y=\"387.942485\" style=\"fill: #0000ff; stroke: #0000ff\"/>\n",
              "     <use xlink:href=\"#m8aedec9a16\" x=\"382.055329\" y=\"387.599879\" style=\"fill: #0000ff; stroke: #0000ff\"/>\n",
              "     <use xlink:href=\"#m8aedec9a16\" x=\"400.172212\" y=\"387.381734\" style=\"fill: #0000ff; stroke: #0000ff\"/>\n",
              "     <use xlink:href=\"#m8aedec9a16\" x=\"418.289095\" y=\"386.978061\" style=\"fill: #0000ff; stroke: #0000ff\"/>\n",
              "     <use xlink:href=\"#m8aedec9a16\" x=\"436.405978\" y=\"386.760561\" style=\"fill: #0000ff; stroke: #0000ff\"/>\n",
              "     <use xlink:href=\"#m8aedec9a16\" x=\"454.522861\" y=\"386.545764\" style=\"fill: #0000ff; stroke: #0000ff\"/>\n",
              "     <use xlink:href=\"#m8aedec9a16\" x=\"472.639744\" y=\"385.619816\" style=\"fill: #0000ff; stroke: #0000ff\"/>\n",
              "     <use xlink:href=\"#m8aedec9a16\" x=\"490.756627\" y=\"385.14438\" style=\"fill: #0000ff; stroke: #0000ff\"/>\n",
              "     <use xlink:href=\"#m8aedec9a16\" x=\"508.873511\" y=\"384.783206\" style=\"fill: #0000ff; stroke: #0000ff\"/>\n",
              "     <use xlink:href=\"#m8aedec9a16\" x=\"526.990394\" y=\"384.524972\" style=\"fill: #0000ff; stroke: #0000ff\"/>\n",
              "     <use xlink:href=\"#m8aedec9a16\" x=\"545.107277\" y=\"384.225203\" style=\"fill: #0000ff; stroke: #0000ff\"/>\n",
              "     <use xlink:href=\"#m8aedec9a16\" x=\"563.22416\" y=\"384.011206\" style=\"fill: #0000ff; stroke: #0000ff\"/>\n",
              "     <use xlink:href=\"#m8aedec9a16\" x=\"581.341043\" y=\"383.579834\" style=\"fill: #0000ff; stroke: #0000ff\"/>\n",
              "     <use xlink:href=\"#m8aedec9a16\" x=\"599.457926\" y=\"382.569582\" style=\"fill: #0000ff; stroke: #0000ff\"/>\n",
              "    </g>\n",
              "   </g>\n",
              "   <g id=\"patch_3\">\n",
              "    <path d=\"M 66.821562 412.22175 \n",
              "L 66.821562 24.14175 \n",
              "\" style=\"fill: none; stroke: #ffffff; stroke-linejoin: miter; stroke-linecap: square\"/>\n",
              "   </g>\n",
              "   <g id=\"patch_4\">\n",
              "    <path d=\"M 624.821563 412.22175 \n",
              "L 624.821563 24.14175 \n",
              "\" style=\"fill: none; stroke: #ffffff; stroke-linejoin: miter; stroke-linecap: square\"/>\n",
              "   </g>\n",
              "   <g id=\"patch_5\">\n",
              "    <path d=\"M 66.821562 412.22175 \n",
              "L 624.821563 412.22175 \n",
              "\" style=\"fill: none; stroke: #ffffff; stroke-linejoin: miter; stroke-linecap: square\"/>\n",
              "   </g>\n",
              "   <g id=\"patch_6\">\n",
              "    <path d=\"M 66.821562 24.14175 \n",
              "L 624.821563 24.14175 \n",
              "\" style=\"fill: none; stroke: #ffffff; stroke-linejoin: miter; stroke-linecap: square\"/>\n",
              "   </g>\n",
              "   <g id=\"text_17\">\n",
              "    <!-- Silhouette Score for Optimal Number of Clusters -->\n",
              "    <g transform=\"translate(171.763813 18.14175) scale(0.144 -0.144)\">\n",
              "     <defs>\n",
              "      <path id=\"DejaVuSans-4f\" d=\"M 2522 4238 \n",
              "Q 1834 4238 1429 3725 \n",
              "Q 1025 3213 1025 2328 \n",
              "Q 1025 1447 1429 934 \n",
              "Q 1834 422 2522 422 \n",
              "Q 3209 422 3611 934 \n",
              "Q 4013 1447 4013 2328 \n",
              "Q 4013 3213 3611 3725 \n",
              "Q 3209 4238 2522 4238 \n",
              "z\n",
              "M 2522 4750 \n",
              "Q 3503 4750 4090 4092 \n",
              "Q 4678 3434 4678 2328 \n",
              "Q 4678 1225 4090 567 \n",
              "Q 3503 -91 2522 -91 \n",
              "Q 1538 -91 948 565 \n",
              "Q 359 1222 359 2328 \n",
              "Q 359 3434 948 4092 \n",
              "Q 1538 4750 2522 4750 \n",
              "z\n",
              "\" transform=\"scale(0.015625)\"/>\n",
              "      <path id=\"DejaVuSans-70\" d=\"M 1159 525 \n",
              "L 1159 -1331 \n",
              "L 581 -1331 \n",
              "L 581 3500 \n",
              "L 1159 3500 \n",
              "L 1159 2969 \n",
              "Q 1341 3281 1617 3432 \n",
              "Q 1894 3584 2278 3584 \n",
              "Q 2916 3584 3314 3078 \n",
              "Q 3713 2572 3713 1747 \n",
              "Q 3713 922 3314 415 \n",
              "Q 2916 -91 2278 -91 \n",
              "Q 1894 -91 1617 61 \n",
              "Q 1341 213 1159 525 \n",
              "z\n",
              "M 3116 1747 \n",
              "Q 3116 2381 2855 2742 \n",
              "Q 2594 3103 2138 3103 \n",
              "Q 1681 3103 1420 2742 \n",
              "Q 1159 2381 1159 1747 \n",
              "Q 1159 1113 1420 752 \n",
              "Q 1681 391 2138 391 \n",
              "Q 2594 391 2855 752 \n",
              "Q 3116 1113 3116 1747 \n",
              "z\n",
              "\" transform=\"scale(0.015625)\"/>\n",
              "      <path id=\"DejaVuSans-61\" d=\"M 2194 1759 \n",
              "Q 1497 1759 1228 1600 \n",
              "Q 959 1441 959 1056 \n",
              "Q 959 750 1161 570 \n",
              "Q 1363 391 1709 391 \n",
              "Q 2188 391 2477 730 \n",
              "Q 2766 1069 2766 1631 \n",
              "L 2766 1759 \n",
              "L 2194 1759 \n",
              "z\n",
              "M 3341 1997 \n",
              "L 3341 0 \n",
              "L 2766 0 \n",
              "L 2766 531 \n",
              "Q 2569 213 2275 61 \n",
              "Q 1981 -91 1556 -91 \n",
              "Q 1019 -91 701 211 \n",
              "Q 384 513 384 1019 \n",
              "Q 384 1609 779 1909 \n",
              "Q 1175 2209 1959 2209 \n",
              "L 2766 2209 \n",
              "L 2766 2266 \n",
              "Q 2766 2663 2505 2880 \n",
              "Q 2244 3097 1772 3097 \n",
              "Q 1472 3097 1187 3025 \n",
              "Q 903 2953 641 2809 \n",
              "L 641 3341 \n",
              "Q 956 3463 1253 3523 \n",
              "Q 1550 3584 1831 3584 \n",
              "Q 2591 3584 2966 3190 \n",
              "Q 3341 2797 3341 1997 \n",
              "z\n",
              "\" transform=\"scale(0.015625)\"/>\n",
              "      <path id=\"DejaVuSans-43\" d=\"M 4122 4306 \n",
              "L 4122 3641 \n",
              "Q 3803 3938 3442 4084 \n",
              "Q 3081 4231 2675 4231 \n",
              "Q 1875 4231 1450 3742 \n",
              "Q 1025 3253 1025 2328 \n",
              "Q 1025 1406 1450 917 \n",
              "Q 1875 428 2675 428 \n",
              "Q 3081 428 3442 575 \n",
              "Q 3803 722 4122 1019 \n",
              "L 4122 359 \n",
              "Q 3791 134 3420 21 \n",
              "Q 3050 -91 2638 -91 \n",
              "Q 1578 -91 968 557 \n",
              "Q 359 1206 359 2328 \n",
              "Q 359 3453 968 4101 \n",
              "Q 1578 4750 2638 4750 \n",
              "Q 3056 4750 3426 4639 \n",
              "Q 3797 4528 4122 4306 \n",
              "z\n",
              "\" transform=\"scale(0.015625)\"/>\n",
              "     </defs>\n",
              "     <use xlink:href=\"#DejaVuSans-53\"/>\n",
              "     <use xlink:href=\"#DejaVuSans-69\" x=\"63.476562\"/>\n",
              "     <use xlink:href=\"#DejaVuSans-6c\" x=\"91.259766\"/>\n",
              "     <use xlink:href=\"#DejaVuSans-68\" x=\"119.042969\"/>\n",
              "     <use xlink:href=\"#DejaVuSans-6f\" x=\"182.421875\"/>\n",
              "     <use xlink:href=\"#DejaVuSans-75\" x=\"243.603516\"/>\n",
              "     <use xlink:href=\"#DejaVuSans-65\" x=\"306.982422\"/>\n",
              "     <use xlink:href=\"#DejaVuSans-74\" x=\"368.505859\"/>\n",
              "     <use xlink:href=\"#DejaVuSans-74\" x=\"407.714844\"/>\n",
              "     <use xlink:href=\"#DejaVuSans-65\" x=\"446.923828\"/>\n",
              "     <use xlink:href=\"#DejaVuSans-20\" x=\"508.447266\"/>\n",
              "     <use xlink:href=\"#DejaVuSans-53\" x=\"540.234375\"/>\n",
              "     <use xlink:href=\"#DejaVuSans-63\" x=\"603.710938\"/>\n",
              "     <use xlink:href=\"#DejaVuSans-6f\" x=\"658.691406\"/>\n",
              "     <use xlink:href=\"#DejaVuSans-72\" x=\"719.873047\"/>\n",
              "     <use xlink:href=\"#DejaVuSans-65\" x=\"758.736328\"/>\n",
              "     <use xlink:href=\"#DejaVuSans-20\" x=\"820.259766\"/>\n",
              "     <use xlink:href=\"#DejaVuSans-66\" x=\"852.046875\"/>\n",
              "     <use xlink:href=\"#DejaVuSans-6f\" x=\"887.251953\"/>\n",
              "     <use xlink:href=\"#DejaVuSans-72\" x=\"948.433594\"/>\n",
              "     <use xlink:href=\"#DejaVuSans-20\" x=\"989.546875\"/>\n",
              "     <use xlink:href=\"#DejaVuSans-4f\" x=\"1021.333984\"/>\n",
              "     <use xlink:href=\"#DejaVuSans-70\" x=\"1100.044922\"/>\n",
              "     <use xlink:href=\"#DejaVuSans-74\" x=\"1163.521484\"/>\n",
              "     <use xlink:href=\"#DejaVuSans-69\" x=\"1202.730469\"/>\n",
              "     <use xlink:href=\"#DejaVuSans-6d\" x=\"1230.513672\"/>\n",
              "     <use xlink:href=\"#DejaVuSans-61\" x=\"1327.925781\"/>\n",
              "     <use xlink:href=\"#DejaVuSans-6c\" x=\"1389.205078\"/>\n",
              "     <use xlink:href=\"#DejaVuSans-20\" x=\"1416.988281\"/>\n",
              "     <use xlink:href=\"#DejaVuSans-4e\" x=\"1448.775391\"/>\n",
              "     <use xlink:href=\"#DejaVuSans-75\" x=\"1523.580078\"/>\n",
              "     <use xlink:href=\"#DejaVuSans-6d\" x=\"1586.958984\"/>\n",
              "     <use xlink:href=\"#DejaVuSans-62\" x=\"1684.371094\"/>\n",
              "     <use xlink:href=\"#DejaVuSans-65\" x=\"1747.847656\"/>\n",
              "     <use xlink:href=\"#DejaVuSans-72\" x=\"1809.371094\"/>\n",
              "     <use xlink:href=\"#DejaVuSans-20\" x=\"1850.484375\"/>\n",
              "     <use xlink:href=\"#DejaVuSans-6f\" x=\"1882.271484\"/>\n",
              "     <use xlink:href=\"#DejaVuSans-66\" x=\"1943.453125\"/>\n",
              "     <use xlink:href=\"#DejaVuSans-20\" x=\"1978.658203\"/>\n",
              "     <use xlink:href=\"#DejaVuSans-43\" x=\"2010.445312\"/>\n",
              "     <use xlink:href=\"#DejaVuSans-6c\" x=\"2080.269531\"/>\n",
              "     <use xlink:href=\"#DejaVuSans-75\" x=\"2108.052734\"/>\n",
              "     <use xlink:href=\"#DejaVuSans-73\" x=\"2171.431641\"/>\n",
              "     <use xlink:href=\"#DejaVuSans-74\" x=\"2223.53125\"/>\n",
              "     <use xlink:href=\"#DejaVuSans-65\" x=\"2262.740234\"/>\n",
              "     <use xlink:href=\"#DejaVuSans-72\" x=\"2324.263672\"/>\n",
              "     <use xlink:href=\"#DejaVuSans-73\" x=\"2365.376953\"/>\n",
              "    </g>\n",
              "   </g>\n",
              "  </g>\n",
              " </g>\n",
              " <defs>\n",
              "  <clipPath id=\"pd0ab672b7b\">\n",
              "   <rect x=\"66.821562\" y=\"24.14175\" width=\"558\" height=\"388.08\"/>\n",
              "  </clipPath>\n",
              " </defs>\n",
              "</svg>\n"
            ],
            "text/plain": [
              "<Figure size 1000x700 with 1 Axes>"
            ]
          },
          "metadata": {},
          "output_type": "display_data"
        }
      ],
      "source": [
        "from sklearn.cluster import AgglomerativeClustering\n",
        "from sklearn.metrics import silhouette_score\n",
        "import matplotlib.pyplot as plt\n",
        "\n",
        "# Define a range for the number of clusters to test\n",
        "range_n_clusters = range(2, 31)\n",
        "\n",
        "# List to store the average silhouette scores for each number of clusters\n",
        "silhouette_avg_scores = []\n",
        "\n",
        "# Loop through the range of cluster numbers\n",
        "for n_clusters in range_n_clusters:\n",
        "    # Create an AgglomerativeClustering model with the specified number of clusters\n",
        "    hierarchical = AgglomerativeClustering(n_clusters=n_clusters)\n",
        "\n",
        "    # Fit the model and predict the cluster for each sample\n",
        "    cluster_labels = hierarchical.fit_predict(data_scaled)\n",
        "\n",
        "    # Calculate the average silhouette score for the current clustering\n",
        "    silhouette_avg = silhouette_score(data_scaled, cluster_labels)\n",
        "\n",
        "    # Append the average silhouette score to the list\n",
        "    silhouette_avg_scores.append(silhouette_avg)\n",
        "\n",
        "# Plot the silhouette scores against the number of clusters\n",
        "plt.figure(figsize=(10, 7))\n",
        "plt.plot(range_n_clusters, silhouette_avg_scores, 'bx-')\n",
        "plt.xlabel('Number of clusters')\n",
        "plt.ylabel('Silhouette Score')\n",
        "plt.title('Silhouette Score for Optimal Number of Clusters')\n",
        "plt.show()"
      ]
    },
    {
      "cell_type": "code",
      "execution_count": 80,
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "id": "x_ZRXOTFv-6w",
        "outputId": "83d7396a-1dc1-4494-d545-818c5d9397aa"
      },
      "outputs": [
        {
          "name": "stdout",
          "output_type": "stream",
          "text": [
            "The optimal number of clusters is 2\n"
          ]
        }
      ],
      "source": [
        "# Find the optimal number of clusters with the highest silhouette score\n",
        "optimal_k = range_n_clusters[silhouette_avg_scores.index(max(silhouette_avg_scores))]\n",
        "print(f'The optimal number of clusters is {optimal_k}')"
      ]
    },
    {
      "cell_type": "code",
      "execution_count": 81,
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/",
          "height": 573
        },
        "id": "hS667e5Ltmc6",
        "outputId": "d83be835-c95b-4d20-d8fd-4678e0222cf3"
      },
      "outputs": [
        {
          "data": {
            "application/vnd.google.colaboratory.intrinsic+json": {
              "summary": "{\n  \"name\": \"df\",\n  \"rows\": 1454,\n  \"fields\": [\n    {\n      \"column\": \"paper_id\",\n      \"properties\": {\n        \"dtype\": \"string\",\n        \"num_unique_values\": 1454,\n        \"samples\": [\n          \"f78956e70c8bd1756ffb24ae47c35c3217bed384\",\n          \"51ad0f064355c4678f18225f7d231f9865f44a11\",\n          \"7400110e77de9824ff53797dfade054fa9a9cf80\"\n        ],\n        \"semantic_type\": \"\",\n        \"description\": \"\"\n      }\n    },\n    {\n      \"column\": \"doi\",\n      \"properties\": {\n        \"dtype\": \"string\",\n        \"num_unique_values\": 1411,\n        \"samples\": [\n          \"10.3201/eid1601.091376\",\n          \"10.1016/j.jcv.2011.12.020\",\n          \"10.1101/2020.04.14.040204\"\n        ],\n        \"semantic_type\": \"\",\n        \"description\": \"\"\n      }\n    },\n    {\n      \"column\": \"abstract\",\n      \"properties\": {\n        \"dtype\": \"string\",\n        \"num_unique_values\": 1044,\n        \"samples\": [\n          \"InnoMetrics is the system that aims at collecting software development process metrics in a non-invasive way to access and optimize the development process and its efficiency. This paper demonstrates the development and analysis of energy consumption of MacOS systems based on the software process measurement data. It represents the experience of the development of MacOS system collector and Transfer, in addition to the user interface and early analysis of energy consumption metrics calculations.\",\n          \"During the global outbreak of severe acute respiratory syndrome (SARS) in 2003, treatment was empiric. We report the case history of the index patient in a hospital outbreak of SARS in Hong Kong. The patient recovered after conventional antimicrobial therapy. Further studies are needed to address treatment of SARS, which has high attack and death rates.\",\n          \"Intestinal transplantation (IT) is the least common form of organ transplantation; however, it has shown exceptional growth and improvement in graft survival rates over the past two decades mainly due to better outcomes achieved during the first year of transplantation (76 % at 1 year), due to improvement in surgical techniques and the development of better immunosupressive therapies as we understand more about the relationship between the recipient and host immune system. There are still ongoing issues with chronic rejection and long-term survival. Intestinal transplantation is still an acceptable therapy for patients with intestinal failure (IF), but it is generally reserved for patients who develop severe and life-threatening complications despite standard therapies, or those who are not able to maintain a good quality of life. The purpose of this review is to describe the current status, indications, outcomes and advances in the field of intestinal transplantation.\"\n        ],\n        \"semantic_type\": \"\",\n        \"description\": \"\"\n      }\n    },\n    {\n      \"column\": \"body_text\",\n      \"properties\": {\n        \"dtype\": \"string\",\n        \"num_unique_values\": 1453,\n        \"samples\": [\n          \"Respiratory tract infection is common in infants and young children and is a major public health problem in this age group [1] . It is mainly caused by infection of seasonal viruses, such as respiratory syncytial virus (RSV) and rhinovirus [2] [3] [4] . Most infections manifest as an upper respiratory infection, while 20-40 % develop lower respiratory infection, and a small number (2 -3 %) of infected infants require hospitalization [5, 6] . This is of particular concern as infants with lower respiratory viral infection are at increased risk of developing recurrent wheezing or childhood asthma [7] [8] [9] [10] [11] [12] [13] . Several risk factors have been identified for the development of severe lower respiratory viral infections, such as premature birth, young age, being born in relation to the RSV season, underlying chronic lung disease, congenital heart disease, high parity, young maternal age and poor socioeconomic factors [14] [15] [16] . Recently, genetic polymorphisms have been identified which are associated with severe viral infection [17, 18] . Many of the identified genetic variants associated with severe lower respiratory tract infections are also associated with and pathways common to asthma, including innate immune genes involved in cytokine and chemokine signaling, and epithelial cell function [13, 17, [19] [20] [21] [22] [23] .\\nAs the clinical hallmark of asthma is wheezing, and therapy includes beta agonists, a number of clinical studies over the past 2 decades have examined the relationship between genetic variants in \\u03b2 2 -Adrenergic receptor (ADRB2) and asthma [24] [25] [26] [27] [28] [29] [30] . The polymorphism that resulted in the Gly16 allele was associated with decreased bronchodilator responses in early clinical studies; however, larger clinical studies have provided equivocal results [31] [32] [33] . Other genetic variants in the promoter region of ADRB2 have also been associated with asthma severity [28, 34] . Despite the significant interest in ADRB2 polymorphisms related to asthma, the relative contribution of ADRB2 genetic variants to severity of respiratory viral infections has not been established. As wheezing is also the clinical hallmark of lower respiratory viral infections, and variable response to bronchodilators has been demonstrated in infants hospitalized with RSV lower respiratory viral infections, it is possible that ADRB2 genetic variants also contribute to the severity of acute respiratory viral infections [35, 36] .\\nThe aim of the present study was to explore associations between the genetic variants in ADRB2 that have been associated with a response to asthma therapy during an acute asthma exacerbation and the severity of infant acute respiratory viral infections. We examined the relationship between 5 common polymorphisms in the promoter region and coding block of the ADRB2 gene and infant severity of respiratory infections in a prospective cohort of Caucasian and African American infants.\\nIn order to examine the association of genetic variants in the regulatory regions of the ADRB2 gene and the severity of viral respiratory infections during infancy, we conducted an analysis of infants enrolled in the Tennessee Children's Respiratory Initiative (TCRI), a prospective cohort of mother-infant dyads enrolled during an infant's acute viral respiratory illness over four respiratory viral seasons, 2004-2008. Term (\\u226537 weeks) and non-low birth weight (\\u22652275 g) infants who were otherwise healthy with no significant co-morbidities or cardio-pulmonary disease were eligible, with oversampling for hospitalization infants. Eighty five percent of enrolled subjects had at least one respiratory virus identified by polymerase chain reaction (RSV, rhinovirus, influenza, parainfluenza, Human Metapneumovirus, and coronavirus); 56 % had labconfirmed RSV infection. The rationale, methods, and detailed inclusion and exclusion criteria have been reported previously [37] . Although 630 infants were enrolled in TCRI, this study was limited to the 261 Caucasian and 113 African American infants for whom DNA was available. The protocol was approved by the Institutional Review Board of Vanderbilt University, and mothers provided informed consent for their infants. All parents provided written informed consent for both their and their child's study participation.\\nInfants included in this study had either upper respiratory tract infections or lower respiratory tract infections. The severity of acute respiratory infection was determined by experienced research nurses through medical record chart review using an ordinal bronchiolitis severity score (BSS). The score is an aggregate of assigned values ranging of 0-3 in categories of respiratory rate, room air oxygen saturation, the presence and extent of wheezing, and the presence and extent of flaring and retractions [37] [38] [39] . The BSS ranges from 0 to 12, with higher scores indicating more severe illness and a difference of 0.5 being a clinically significant difference [40] . We measured the BSS at the end of the outpatient visit or after discharge from an inpatient admission, and the most severe recorded value was used for analysis.\\nDNA was extracted by Vanderbilt's DNA core laboratory from whole blood or saliva samples (Oragene by DNA Genotek) by standardized protocols. All oragene samples were processed immediately after collection, while blood samples were either processed immediately or frozen before processing. Genotyping at five pre-specified loci (-2387, -2274, -1343, +46, and +79) of ADRB2 gene was done using Taqman SM\\u00a9 Genotyping Assays (Applied Biosystems, Foster City, CA, USA) with predetermined primers. The five single-nucleotide polymorphisms (SNP)s were rs1432623, rs11168068, and rs2400707 at the promoter region, and rs1042713 SNP and rs1042714 SNP at the coding region of ADRB2 gene. Probes for the Applied Biosystems assays used were labeled at their 5\\u2032end with either VIC\\u00ae or 6-carboxy-fluorescine (FAM) reporter dyes. The reaction components include: 2.5 \\u03bcl Taqman Universal PCR Master Mix (Applied Biosystems), 0.125 \\u03bcl or 0.250 \\u03bcl assay mix (Applied Biosystems) and approximately 5 ng of genomic DNA in a total volume of 10 \\u03bcl per single tube reaction. PCR amplification was performed in a thermal cycler (Techne TC-412) with an initial step of 95\\u00b0C for 10 min followed by 50 cycles of 92\\u00b0C for 15 s and 60\\u00b0C for 1 min. After amplification, the fluorescence of each sample was read on the ABI 7900HT (DNA Resources Core at Vanderbilt University) and analyzed with the Sequence Detection Software (Applied Biosystems). Five percent of samples were run as blind duplicates with 100 % concordance. Primers and probes for all SNPs are listed in Table 1 .\\nThe demographics and characteristics of the 374 infants included, with either upper respiratory infection or lower respiratory tract infection, are presented as frequencies and proportions for specific categorical variables and medians and interquartile ranges were calculated for continuous variables. All analyses were stratified by race (Caucasian or African American), as a surrogate measure of European or African ancestry, to reduce the known impact of population substructure which can produce spurious associations [41] . A pairwise linkage disequilibrium (LD) was estimated for the SNPs in ADRB2 using the standardized summary statistics D' and r, calculated by the HaploView program (Whitehead Institute for Biomedical Research, Cambridge, MA, USA). SNPs were tested for deviations from Hardy Weinberg proportions, also using Haploview [42] . The promoter, coding and combined promoter and coding SNPs were assigned to blocks using D' confidence interval of Gabriel et al. [43] . Individual haplotypes were estimated using the program PHASE v2.1.1 [44, 45] . The haplotype frequencies for each block were estimated using the expectation maximization (EM) algorithm, again using the HaploView program. Non-parametric Kruskal-Wallis test were applied to compare the difference in BSS among infants with each SNP genotype, as well as each haplotype genotype. Haplotype genotypes with frequency less than 2 % were excluded from haplotype analysis. We used a proportional odds model for the ordinal BSS to test the additive effect of haplotypes. Covariates included in the regression model included infant age at enrollment, gender, daycare exposure, secondhand smoke exposure, any prior history of breastfeeding, any siblings at home, and enrollment season. All analyses were conducted separately according to race. All analyses were performed using R-software version 2.11.1 (www.r-project.org). We used a two-sided 5 % significance level for all statistical inferences.\\nThere were in total 374 infants, 261 Caucasians and 113 African Americans, enrolled in the TCRI cohort with available DNA and at least one ADRB2 polymorphism tested ( Table 3 lists the allele frequencies of 5 SNPs separately for Caucasian and African Americans. In both Caucasians and African Americans, infants had slightly more T, T, and G alleles for the three promoter SNP rs1432623, (Figs. 1 and 2) . This non statistical difference was consistent within both Caucasians and African Americans. Among African Americans, there was a decreasing trend of the severity of acute respiratory infection with an increasing number of copies of C, C, and A allele of the three promoter SNPs rs1432623, rs11168068, and rs2400707, respectively (Fig. 1) . However, analyses with additive models indicated no statistical significance (p > 0.05).\\nAmong 2 5 (32) possible haplotype combinations, only four haplotypes with more than 2 % of frequency were identified (Table 4 and Fig. 3 ). There were two promoter haplotypes and three coding block haplotypes. There were no statistically significant differences in the promoter and coding block haplotype frequencies between Caucasian and African Americans. However, when all 5 SNPs were combined, the distribution of haplotype frequency was significantly different between Caucasians and African Americans, with genotype CCAAC more common in African Americans and rare in Caucasians (p < 0.001).\\nPromoter haplotype CCA was associated with a decreased BSS in African Americans in a dose dependent manner (p = 0.037 in the univariate analysis) (Fig. 4) . Infants without the CCA haplotype had the highest BSS (median 5.5, interquartile range [IQR]: 2.0, 8.0). Infants with one copy of CCA haplotype had a lower bronchiolitis severity score (median 4.0, IQR: 1.0, 7.5). Infants with 2 copies of the CCA haplotype had the comparatively lowest BSS (median 3.0, IQR: 1.0, 4.0). This dose dependent relationship persisted after adjusting for infant age, gender, daycare exposure, secondhand smoke exposure, prior history of breastfeeding, siblings at home, and enrollment season (adjusted odds ratio: 0.59, 95 % confidence interval [CI]: 0.36, 0.98, p = 0.042). This protective effect of CCA haplotype was consistent when we categorized the acute respiratory infection into upper respiratory infection and lower respiratory tract infection. In a similar dose dependent manner, infants with more copies of CCA haplotype were less likely to have lower respiratory tract infection compared with infants with less copies of CCA haplotype (p = 0.05). On the other hand, promoter haplotype TTG was associated with a higher BSS in African Americans (Additional file 1).\\nThe protective effect of CCA haplotype in African Americans was identified in all combined haplotypes with CCA combination (Figs. 5 and 6 ). However, due to the sample size, the effect was not statistically significant. In addition, all African American infants with a coding block haplotype GG had CCA combination in their promoter region; therefore, there was a similar additive protective effect of GG in African Americans (Additional file 2). There was no relationship between BSS and the other two coding block haplotypes in African Americans (Additional files 3 and 4). This was also true for the combined haplotypes (Additional files 5 and 6).\\nIn both Caucasian and African American infants, there were no significant relationships between BSS and coding block haplotypes except for GG haplotype among African Americans (Additional file 2). This was also true for the combined haplotypes. In general, there was no significant relationship between BSS and any haplotypes in Caucasian infants. There was no similar promoter haplotype CCA relationship with BSS in Caucasians as observed in the African Americans (Fig. 4 ).\\nIn this study we have identified a haplotype in the promoter region of the ADRB2 gene which is associated with a decreased BSS in African American infants. This promoter haplotype of the ADRB2 gene was not associated with the severity of respiratory infections in Caucasian infants. The common coding block polymorphisms, corresponding to amino acid changes at codons 16 and 27, were not associated with a protective effect on severity of respiratory infections in Caucasian or African American infants. Our results agree with the studied conducted by Janssen and colleagues, who found no relationship between the SNP rs1042713 and susceptibility to RSV bronchiolitis in a European population [17] . This observation that the CCA promoter region haplotype is associated with a protective effect on respiratory viral infection severity is similar to our observation in a cohort of adults hospitalized for an asthma exacerbation, where this haplotype was associated with spirometric improvement in response to asthma-specific therapy that included beta-agonists and corticosteroids [34] . In that study, we noted spirometric improvement only in Caucasian patients, although the small number of African American non-responders in that cohort may have limited our ability to detect a similar effect in African Americans [34] .\\nThe mechanism by which a haplotype in the promoter region of the ADRB2 gene is associated with a decreased BSS is not known. However, we can speculate that [46] . Moore et al. demonstrated that ADRB2 genetic variants are associated with differences in ADRB2 expression in cultured human airway smooth muscle (HASM) cells, an in vitro system used as a model for changes in the human airway [47] These studies were performed prior to identification of the complex promoter haplotypes, and we are not aware that these haplotypes have been studied in HASM. Small differences in airway tone could thus be important in the setting of acute viral respiratory infections.\\nAlthough beyond the scope of the current observational study, we also speculate that ADRB2 genotype might also influence response to bronchiolitis-specific therapy. The primary goal of TCRI was to establish a longitudinal, prospective investigation of infants and their biological mothers on the acute and the long-term health consequences of varying viral respiratory tract infections on early childhood outcomes [37] . Although a BSS was assigned for each infant at acute care visit or hospital admission, we did not study or quantify the effects of different treatments, including beta-agonist therapy. A number of clinical studies have suggested that some infants with bronchiolitis appear to improve with bronchodilator therapy, while on average it is not efficacious [48] [49] [50] . As ADRB2 is the primary target of bronchodilator therapy, variation in response to beta-agonists might reflect genetic variants in ADRB2.\\nThe diversity of SNPs of the ADRB2 gene was explored by Drysdale et al., using immortalized lymphocytes from 23 Caucasians, 19 African-Americans, 20 Asians, and 15 Hispanic-Latinos [27] . This report from (-2387, -2274 , and -1343) and in the coding block SNPs discussed in this manuscript [28] . Such frequency distributions observed in our study are consistent with the allele frequencies reported in the reference population of the International HapMap Project [51] and Hawkins et al. [28] . Our stratified analyses by race therefore allow us to detect the association between severity of a respiratory illness and haplotype frequency within each racial group, and to minimize the spurious associations due to such population substructure. We recognize that the significance of our findings is limited by our small sample size and in classifying infants in this study by a severity score rather than by measuring some direct response to bronchodilator therapy. However, BSS correlated with the severity of disease (classified as upper respiratory and lower respiratory illness) well. Our sensitivity analyses with the severity of disease (upper versus lower respiratory illness) showed consistent results indicated that BSS is a valid surrogate of disease severity. Lastly, although a p value of 0.042 is at the threshold of statistical significance, this p value reflects sample size of 113 African Americans in our study. Whether the association between BSS and CCA haplotype in African Americans is powerful or not depends upon not only p value, but also upon the effect size such as odds ratio and the clinical significance of the findings. As a difference of 0.5 in BSS is considered as clinically significant, a 65 % reduced odds of developing a more severe disease (higher BSS) comparing 2 copies of CCA with 0 copy of CCA might also be clinically significant and meaningful.\\nWe hope that this hypothesis-generating study will encourage the incorporation of pharmacokinetic data in future studies of bronchiolitis. Although validation studies in other populations are needed, this study underscores that the development of personalized approaches to the treatment of bronchiolitis may benefit from the analysis of specific genetic variants.\\nIn summary, we have observed that promoter region haplotype of ADRB2 CCA is associated with a protective effect on respiratory illness severity in African American infants. In this era of personalized medicine, ADRB2 genotype may be predictive of the severity of respiratory viral infection and potential response to therapy for the most common acute infant illness and one for which we currently have no therapy. Next steps would be to validate these findings, as well as evaluate available clinical trial data for which DNA might be available. Fig. 6 BSS distribution stratified by copy number of combined haplotype CCAAC in African Americans infants. The scatter plots and the box-and-whisker plots of BSS across 0, 1, and 2 copies of combined haplotype CCAAC for African American infants. P values were obtained from multivariable regression model adjusted for infant age at enrollment, gender, daycare exposure, secondhand smoke exposure, any prior history of breastfeeding, any siblings at home, and enrollment season\",\n          \"Acute bronchiolitis, a viral infection of the lower respiratory tract, is one of the most substantial health burdens for infants and young children worldwide. 1 Respiratory syncytial virus is the most prevalent viral cause of bronchiolitis in infants. Estimates suggest that about 34 million new cases of lower respiratory infection due to respiratory syncytial virus occur globally in children younger than 5 years, with 3\\u00b74 million admissions to hospitals and about 199 000 deaths per year, predominantly in the developing world. 2 In developed countries such as the USA, bronchiolitis is the most common reason for admission to hospital in the fi rst 12 months of life, 3 accounting for approximately 100 000 infant admissions annually. Although admissions to hospital have declined from 2000 to 2010, emergency department visits have increased, in addition to increased use of mechanical ventilation and hospital charges. 4, 5 The clinical management remains challenging despite the frequency, global reach, economic cost, and morbidity and mortality associated with bronchiolitis. Several treatment strategies (including bronchodilators and corticosteroids) showed no eff ect in pooled metaanalyses, making supportive care the hallmark of current therapy. In this Seminar, we aim to summarise the current evidence for the epidemiology, pathophysiology, diagnostic approach, and management of acute viral bronchiolitis.\\nBronchiolitis is a seasonal infection, with the season typically beginning in late October in the temperate northern hemisphere, peaking in January or February, and ending in April. 6 Globally, independent of region, respiratory syncytial virus infection peaks consistently during annual or biannual epidemics. Although the peak and duration of these epidemics vary worldwide, they are consistent year-to-year within a country. 7 Some data suggest that climate might also be associated with prevalence of respiratory syncytial virus infection, with global surveillance suggesting that infection peaks during wet months in areas with high precipitation and during cooler months in hot regions. 7 Indoor crowding in population-dense areas during rainy seasons or cooler months might be one factor that facilitates viral transmission. 8 Additionally, weather-related factors, such as inhalation of cold and dry air that might impair ciliary function, the airway mucosa, and inhibition of temperature-dependent antiviral responses, might infl uence both disease transmission and severity. 9, 10 Altitude, climate, and meteorological conditions (such as wind speed and dew point) have been shown to have a modest association with bronchiolitis. 11, 12 Furthermore, air pollutants, such as ozone and traffi c pollutants, have been associated with exacerbations of respiratory infections in children younger than 5 years. [13] [14] [15] Environmental tobacco smoke has been associated with increased risk for respiratory syncytial virus-attributable admission to hospital and disease severity in those admitted. 16, 17 As with other respiratory viral infections, the risk of severe respiratory syncytial virus bronchiolitis might be greater in boys than in girls. 18, 19 This diff erence might be due to diff erences in lung and airway development, and by genetic factors. 8, 20 Pathophysiology Bronchiolitis is characterised by extensive infl ammation and oedema of the airways, increased mucus production, and necrosis of airway epithelial cells. 21 Respiratory syncytial virus binds to epithelial cells and replicates, resulting in epithelial necrosis and ciliary destruction. 21, 22 The cell destruction triggers an infl ammatory response with proliferation of polymorphonuclear cells and lymphocytes. The submucosa and adventitial tissues become oedematous with increased mucus secretion. 22 Plugs composed of cellular debris and mucus form in the bronchiole lumens leading to bronchiolar obstruction, air trapping, and diff erent degrees of lobar collapse. 21 Microbiology Molecular testing has led to an improved understanding of the viruses associated with bronchiolitis. Respiratory syncytial virus remains the most commonly identifi ed virus, detected in 41-83% of patients. [23] [24] [25] [26] [27] [28] Other viruses associated with bronchiolitis include rhinovirus, metapneumovirus, coronavirus, human bocavirus, infl uenza virus, adenovirus, and parainfl uenza virus. 24, 25, 27, 29, 30 Studies have investigated whether severity of illness, as measured by need for hospital admission, length of hospital stay, intensive care unit admission, repeated emergency department visits, and apnoea, is associated with specifi c viral infections or co-infections, but the evidence is confl icting. Data from some studies have shown that in infections involving a single virus, respiratory syncytial virus is associated with a more severe course compared with other viruses. 31, 32 Up to 30% of children with bronchiolitis are found to have co-infections with two other viruses, 29 with the combination of respiratory syncytial virus and rhinovirus being the most commonly reported. 29 Some evidence suggests that co-infection in bronchiolitis, particularly respiratory syncytial virus in combination with rhinovirus or metapneumovirus, could be associated with a more severe disease course compared with infection by a single virus. 25, 30, 31, 33, 34 However, other studies do not confi rm this association. 24, 26 Furthermore, although use of nucleic acid amplifi cation tests has greatly improved our ability to detect viruses present in respiratory infections, studies using these technologies have also found at least one respiratory virus in up to 30% of children younger than 6 years with no respiratory symptoms. [35] [36] [37] These viruses might be detected because of asymptomatic colonisation, incubation before clinical infection, or prolonged viral shedding post-infection. The confl icting evidence and high prevalence of respiratory viruses in asymptomatic children suggest no indication at this time that management should vary based on presumed viral cause and presence, or absence of viral co-infections.\\nThe diagnosis of bronchiolitis is clinical and thus requires a clinician to recognise signs and symptoms of viral lower respiratory tract infection in young children. Peak incidence occurs between 3 months and 6 months of age. 38 Since the early clinical defi nition by Court, and as noted in recent practice guidelines, the most specifi c defi nition is in infants. [38] [39] [40] Although the same physiology can occur in toddlers older than 12 months, many clinical trials have excluded these children or have included them as a small subgroup of patients. Bronchiolitis in toddlers can overlap with other conditions such as viral-induced wheezing and asthma, and application of evidence from trials predominantly assessing infants might not be appropriate. Further eff orts to focus the defi nition might assist eff orts to standardise care.\\nThe classic clinical presentation of bronchiolitis starts with symptoms of a viral upper respiratory infection, such as nasal discharge, that progress to the lower respiratory tract over several days (fi gure 1). Timing of symptom progression can vary, and young infants can present with apnoea. Lower respiratory tract symptoms of bronchiolitis include persistent cough, tachypnoea, and increased work of breathing, as shown by intercostal or supraclavicular retractions, use of abdominal muscles, grunting, or nasal fl aring. Auscultatory fi ndings include crackles and wheeze. A hallmark characteristic of bronchiolitis is the minute-to-minute variation in clinical fi ndings, as mucus and debris in the airways are cleared by coughing or as the child's state changes from sleep to agitation. This variation can confound assessment, and often requires several examinations over a period of observation. Nasal congestion can also confound the clinical assessment. Nasal suctioning might help to ascertain which fi ndings are truly from the lower airways. Fever can be present in about a third of infants with bronchiolitis, but it is usually present early in the illness with a temperature less than 39\\u00b0C. 38 The median duration of symptoms is about 2 weeks, with 10-20% of infants still having symptoms at 3 weeks after onset. 38, 41 Various clinical scores have been shown in studies and clinical protocols to correlate with disease severity and improvement. 42, 43 Although documentation of a score can be useful as an objective measure, individual scores are not highly predictive, and they should be repeated and combined with other measures of severity for a universal assessment to guide decision making. The diff erential diagnosis for bronchiolitis includes considerations of various infectious and non-infectious causes. Absence of upper respiratory symptoms should raise suspicion of other causes of respiratory distress in young infants, including cardiac disease, congenital airway abnormalities such as a vascular ring, or foreign body aspiration. Other infections can resemble or complicate bronchiolitis. Pertussis should be considered in infants with severe or paroxysmal cough, or with known exposure. Bacterial infections complicating viral bronchiolitis, including otitis media or pneumonia, might present as a new fever or worsening status later in the course of illness.\\nVarious risk factors have been associated with progression to severe bronchiolitis. Those supported by the strongest evidence include presence of chronic lung disease of prematurity and haemodynamically important congenital heart disease, with immunodefi ciency and neuromuscular disorders also considered as high risk in practice guidelines. 38, 39 Young infants (aged <2-3 months) and those with a history of premature birth (especially <32 weeks' gestation) are also at high risk for progression and can present with apnoea without other clinical fi ndings. Studies assessing the risk for further apnoea in hospital have found it to be limited to infants less than 1 month for full-term infants, 48 weeks postconceptional age for preterm infants, or those with apnoea observed before admission. 44\\nBronchiolitis is a clinical diagnosis based on history and physical examination, according to consensus across national guidelines. For infants with a typical presentation of bronchiolitis, routine imaging or laboratory testing is not recommended, as they increase costs without evidence for benefi t (table).\\nAppropriate use of pulse oximetry monitoring and initiation of oxygen for bronchiolitis have received increasing attention in studies and practice guidelines. Findings suggest that arbitrary thresholds for oxygen administration might drive hospital admissions and prolong hospital length of stay. These outcomes represent only part of the morbidity of bronchiolitis, but developing evidence suggests that intermittent hypoxaemia might occur commonly in otherwise stable infants with bronchiolitis and raises questions as to whether this factor should be used as a sole indication for admission to hospitals. A Canadian randomised trial found reduced admissions to hospital from the emergency department without any increase in revisits when pulse oximeters displayed values 3% higher than the actual value, suggesting that arbitrary pulse oximetry thresholds result in unnecessary admissions. 51 A similar trial in the UK in the hospital setting found that reduction of the oxygen threshold from 94% to 90% resulted in earlier discharge from the hospital without any evidence of adverse outcomes. 52 A US trial comparing intermittent versus continuous pulse oximetry in non-hypoxaemic infants in hospitals found similar outcomes between the groups. 53 This US trial and other evidence supports recommendations in US practice guidelines that clinicians use a threshold of 90% for initiation of oxygen, whereas UK guidelines recommend 92%. 38, 39 As the child improves, reduction in intensity of monitoring to intermittent checks is appropriate. A recent study using blinded oximetry at home showed that a substantial proportion of infants with bronchiolitis who are otherwise doing well have oxygen desaturations less than 90%, particularly during sleep, further calling into question arbitrary thresholds for hospital admissions and initiation of oxygen. 54 This evidence will probably lead to eff orts to reduce continuous monitoring in children without other indications for monitoring.\\nThe majority of children with bronchiolitis have either normal radiographs or radiographic fi ndings consistent with simple bronchiolitis, 55-58 including peribronchial thickening, hyperinfl ation, and atelectasis. One prospective study 58 of routine radiographs as part of the assessment for bronchiolitis in the emergency department reported airspace disease in 17 (7%) of 246 patients. Despite the low prevalence of radiographic pneumonia, this and other studies have reported an increase in antibiotic prescription after radiographs are performed, because of non-specifi c fi ndings that infl uence clinicians' decisions. 58, 59 Factors that have been associated with defi nite focal infi ltrates consistent with pneumonia include hypoxia (oxygen saturation <92%), [56] [57] [58] 60 grunting, persistently focal crackles, and fever (especially >39\\u00b0C). 38 Chest radiographs should only be considered in patients when the presentation is not classic for bronchiolitis. These situations include when another diagnosis (such as foreign body aspiration) is high on the diff erential diagnosis, when a child is severely ill and respiratory failure is imminent, and when symptoms are progressing or not resolving according to the typical disease course expected for bronchiolitis. Lung ultrasound is increasingly used to assess cardiopulmonary conditions in adults and children. Several studies have investigated the use of lung ultrasound in the diagnosis of bronchiolitis. Two small studies found that ultrasound fi ndings in infants with bronchiolitis correlate with clinical fi ndings, and might be more specifi c than chest radiography, 61,62 but further studies are needed to establish whether there is a role for ultrasound in diagnosis or assessment of severity.\\nWith the development of PCR to detect respiratory viruses in the nasopharynx, interest in the use of viral testing for causative diagnosis in bronchiolitis has increased. Virological testing, however, does not generally assist in management and is insuffi cient to predict outcomes. 30, 63 Many national guidelines therefore recommend against routine virological testing in bronchiolitis (table). Recent studies suggest that higher respiratory syncytial virus genomic load, measured using quantitative PCR, might be associated with increased length of stay, use of respiratory support, and need for intensive care, in addition to recurrent wheezing, compared with lower viral loads. 28, 31, 32, 64 Further study is warranted to confi rm this association and clarify whether viral load measurement improves understanding of disease pathophysiology and severity. Several guidelines recommend using respiratory syncytial virus testing to guide cohorting of patients; however, the viruses most likely to cause bronchiolitis are all transmitted in a similar fashion (close contact with large-particle aerosols or direct contact with contaminated fomites). [65] [66] [67] Thus, infection control might not be dependent on the identifi cation of specifi c viruses, but rather on following strict precautions including hand hygiene, separating infants in shared hospital rooms by more than 1 m, and other infection control procedures. 65 Additionally, given the sensitivity of PCR testing, results should be interpreted with caution. Certain viruses, such as rhinovirus, might be detected because of viral shedding from an unrelated illness or colonisation; whereas certain other viruses, such as respiratory syncytial virus and metapneumovirus, are almost always associated with an acute infection.\\nBlood and urine testing is not routinely recommended as part of standard practice in the diagnostic work-up of bronchiolitis (table) . A blood gas measurement should not be routinely obtained in infants with bronchiolitis, unless there are signs of impending respiratory failure or severe distress. Proportions of serious bacterial infections, especially bacteraemia and meningitis, are very low in infants with bronchiolitis. 68 Abnormal white blood cell count is rarely useful in predicting serious bacterial infections in children infected with respiratory syncytial virus. 69 Guidelines universally do not recommend complete blood counts in infants with bronchiolitis unless blood count is part of assessment for a fever in infants younger than 1-2 months. Similarly, given that bacteraemia is exceedingly rare (with cited proportions of <0\\u00b71% in the post-pneumococcal vaccine era), blood cultures should not be routinely performed, except in the septic work-up of infants younger than 1-2 months, 70, 71 or in those with severe illness and signs of sepsis. Hydration status is an important consideration in infants with bronchiolitis and should be determined by clinical examination. Routine measurement of serum electrolytes is of little value in the majority of infants.\\nUrinary tract infections in infants with bronchiolitis occur with greater frequency than do bacteraemia and meningitis, with proportions ranging from 1% to 7%. 68, 71, 72 It is reasonable to obtain a urinalysis and urine culture for infants aged less than 60 days with fever and for older febrile infants who have risk factors for urinary tract infections, 73 but urine should not be routinely obtained in all infants with bronchiolitis.\\nCurrent recommendations for management of bronchiolitis focus on agents to treat the patho physiological eff ects of viral lower respiratory infection (eg, bronchodilators and hypertonic saline). Specifi c antivirals such as ribavirin to treat respiratory syncytial virus infection are not recommended in practice guidelines for typical cases of bronchiolitis because of challenging delivery methods, high cost, and potential health risks to caregivers. Multiple new agents for prevention and treatment are under investigation and might become available in the future, including immunoglobulins, small interfering RNA interference, fusion inhibitors, and small molecules. 74 Numerous studies have assessed the role of bronchodilators for the treatment of bronchiolitis, and systematic reviews have found no consistent benefi t. A 2014 Cochrane Collaboration systematic review identifi ed 30 studies assessing bronchodilators, predominantly salbutamol and excluding epinephrine, and 21 studies that looked specifi ally at clinical scores found no evidence of benefi t in any outcomes for infants admitted to hospitals. 75 In outpatients, oxygen saturation, admission to hospital, or time to resolution of symptoms did not improve with bronchodilator usage compared with placebo. For outpatient studies assessing short-term change in pooled clinical scores, the reviewers found a small signifi cant diff erence in mean score (Z=2\\u00b726; p=0\\u00b7024) that was of small eff ect with minimal clinical importance (fi gure 2). Outpatient studies were heterogeneous (I\\u00b2=81%; p<0\\u00b700001), and those showing benefi t in scores tended to include older children and children with recurrent wheezing.\\nNebulised epinephrine was assessed in another Cochrane Collaboration systematic review. 76 This review found no benefi t for epinephrine compared with placebo for inpatients in hospital length of stay or other outcomes. A multicentre Scandinavian study published after this Cochrane review found that inpatients receiving standing doses of epinephrine had longer length of stay compared with inpatients receiving as-needed epinephrine or placebo. 77 For outpatients, the Cochrane review found a diff erence in the numbers of admissions associated with epinephrine treatment during the time of an emergency department visit, but not during the overall course of illness when assessed at 1 week. Clinical practice guidelines including those from the USA, UK, and Canada do not recommend treatment with bronchodilators for bronchiolitis because of this evidence (table) . 38 \\nNebulised hypertonic saline is thought to reduce airway oedema, decrease mucus plugging, improve mucociliary clearance, and rehydrate the airway surface liquid in infants with bronchiolitis. 78 These physiological changes are extrapolated from the cystic fi brosis literature, 79, 80 and the pathophysiological processes in acute bronchiolitis are diff erent. Therefore, the theoretical benefi ts of hypertonic saline seen in cystic fi brosis might not be present in infants with acute viral bronchiolitis. Although initial trials demonstrated some ability of hypertonic saline to decrease hospital length of stay and transiently improve clinical severity score, 81-83 more recent trials demonstrated confl icting results. [84] [85] [86] [87] [88] [89] [90] The trials that showed the largest benefi t were done in hospitals with lengths of stay more than 72 h; thus, hypertonic saline for infants in countries and institutions in which the length of stay approaches or exceeds 72 h might be benefi cial at reducing length of stay. The confl icting results are \\nTest for heterogeneity in the inpatient studies demonstrated low inconsistency between the nine studies (I\\u00b2=36%; p=0\\u00b713) and the summary eff ect was not signifi cant (Z=1\\u00b706; p=0\\u00b729), the outpatient studies demonstrated very high inconsistency between the 12 studies (I\\u00b2=81%; p<0\\u00b700001) and the summary eff ect was signifi cant (Z=2\\u00b726; p=0\\u00b7024), and the overall heterogeneity of the meta-analysis demonstrated high inconsistency between the 21 studies (I\\u00b2=73%; p<0\\u00b700001) and the overall summary eff ect was signifi cant (Z=2\\u00b74; p=0\\u00b7016). IPR=ipratropium. SAL=salbutamol. neb=nebulised. Reproduced from Gadomski and Scribani, 75 refl ected in the diff erences in recommendations across national guidelines (table), with some countries not recommending hypertonic saline, some recommending use for all inpatients, and some recommending use only in moderate to severe illness. The largest systematic review and meta-analysis, published in 2015, examined 24 trials and 3209 patients. 91 Infants receiving hypertonic saline had a signifi cant diff erence in hospital length of stay of -0\\u00b745 days (95% CI -0\\u00b782 to -0\\u00b708; p=0\\u00b701) compared with those receiving 0\\u00b79% saline or standard care. In seven trials, hypertonic saline reduced the risk of admission to hospital by 20% (risk ratio 0\\u00b78; 95% CI 0\\u00b767-0\\u00b796) compared with 0\\u00b79% saline. No substantial adverse eff ects of hypertonic saline were noted in this systematic review. Although the results of this meta-analysis were signifi cant, attention should be paid to the outer limits of the confi dence intervals, which approach no diff erence, and to the clinical relevance of these diff erences. A second 2015 meta-analysis of 15 trials and 1922 patients found a smaller, but signifi cant, decrease in length of stay by -0\\u00b736 days (95% CI -0\\u00b75 to -0\\u00b722; p<0\\u00b7001) in those who received hypertonic saline (fi gure 3). 92 This study found a discrepancy between the overall combined eff ect of all studies on length of stay and the negative results from the largest trials, allowing the authors to conclude that neither individual trials nor pooled estimates provide a strong evidence-based foundation for the use of hypertonic saline. Both meta-analyses showed substantial heterogeneity across studies (I\\u00b2=82\\u00b71%, p<0\\u00b7001; 91 and I\\u00b2=77\\u00b78%, p=0\\u00b7029 92 ). A recent reanalysis of the fi rst 2015 meta-analysis removed two outlying Chinese studies and accounted for imbalances in day of illness at presentation. 93 These analyses resolved the heterogeneity and found that hypertonic saline does not reduce length of stay in infants admitted to hospitals with bronchiolitis (mean diff erence in length of stay removing outliers -0\\u00b721 days; 95% CI -0\\u00b743 to 0\\u00b702; mean diff erence in length of stay accounting for day of illness imbalance 0\\u00b702 days; 95% CI -0\\u00b714 to 0\\u00b717). Large trials have not demonstrated benefi t for hypertonic saline and this meta-analysis found no eff ect of hypertonic saline on length of stay after adjustment for outliers and imbalances. The decision to undertake future trials is controversial given the positive results of some metaanalyses and negative results of others.\\nMultiple studies have examined the role of corticosteroids in the management of children with bronchiolitis. Data from two large multicentre trials have shown no benefi t to corticosteroids alone in reducing admissions to hospital, 94, 95 and a 2013 Cochrane Collaboration review supports the results of these studies. 96 This review included only studies that enrolled children younger than 24 months with a fi rst episode of wheezing and signs of a viral illness. Among the included eight outpatient studies (1824 participants) comparing corticosteroids with placebo there was no reduction in admission at day 1 (Z=1\\u00b705; p=0\\u00b73) or day 7 (Z=1\\u00b738; p=0\\u00b717) after enrolment (fi gure 4), clinical scores, length of stay in the emergency department, or length of time to resolution of symptoms. Among the nine inpatient studies (772 participants), length of hospital stay was not reduced. 96 On the basis of this evidence, multiple clinical practice guidelines recommend against the use of corticosteroids for infants with bronchiolitis (table) . Although clinicians report considering a family or personal history of atopy when deciding whether to treat infants with bronchiolitis with corticosteroids, 41 there is no evidence that such infants receive any benefi t from corticosteroid treatment. [94] [95] [96] Evidence for the presence or absence of respiratory syncytial virus infection in these infants being associated with a response to corticosteroids is also unavailable. 94, 95 Authors from a large study with a factorial design have suggested, in an unadjusted analysis, that high-dose corticosteroids in combination with nebulised epinephrine might reduce admissions for outpatients with bronchiolitis by day 7, 95 but these results are considered exploratory. 39, 45 High-fl ow oxygen and respiratory support\\nNon-invasive technologies to improve oxygenation and ventilation for bronchiolitis include humidifi ed high-fl ow nasal cannula oxygen and continuous positive airway pressure. 97 High-fl ow nasal cannula allows delivery of high fl ows (usually 1-2 L/kg per min) with humidifi cation and a cannula designed to improve patient tolerance. It has been used widely in premature infants, but the mechanisms of action are unclear, in particular whether it might deliver positive end-expiratory pressure in some conditions. Evidence for effi cacy of high-fl ow nasal cannula is predominantly observational, with studies documenting Corneli (2007) Goebel (2000) Kuyucu (2004) Plint (2009) improved respiratory parameters and reduced intubation rates after implementation. 97 One small randomised trial compared high-fl ow nasal cannula with hypertonic saline and found no diff erence in change in respiratory score. 98 Concerns about high-fl ow nasal cannula include the potential for rapid deterioration if the infant is not closely monitored and costs associated with overuse. Continuous positive airway pressure has been studied in intensive care settings in observational studies and several small trials, with some evidence of improved respiratory parameters. 97 The UK guidelines recommend considering continuous positive airway pressure in children with impending respiratory failure from bronchiolitis.\\nAntibiotic overuse in children with bronchiolitis probably occurs because of concerns about the presence of fever, the young age of aff ected patients, diffi culty diff erentiating atelectasis from infectious consolidation on chest radiograph, and concern for undetected secondary bacterial infection. Bronchiolitis, however, has a clear viral cause and the occurrence of secondary bacterial infections is low, with a risk of bacteraemia or meningitis of less than 1%. 99 A detailed review of randomised clinical trials found that routine use of antibiotics did not improve duration of symptoms, length of hospital stay, need for oxygen therapy, or hospital admission. 38 Overuse of antibiotics is known to result in unnecessary adverse eff ects on the patient, and the development of antimicrobial resistance. Routine use should be avoided unless there is clear evidence of a secondary bacterial infection (table) . Acute otitis media has been documented in up to 60% of infants with bronchiolitis. 100, 101 Antibiotic use for acute otitis media in bronchiolitis should follow established evidence and guidelines for acute otitis media. 102 Macrolide antibiotics have anti-infl ammatory properties that might have potential benefi t in mitigating the infl ammation present in bronchiolitis. Two randomised trials found that there was no diff erence between azithromycin and placebo in hospital length of stay, need for oxygen, or hospital re-admission. 103, 104 Another randomised trial found that azithromycin lowered nasal interleukin-8 concentrations, prolonged time to subsequent wheezing episodes, and resulted in fewer days with respiratory symptoms in the year following the bronchiolitis episode compared with placebo. 105 Finally, a US multicentre study found that in children aged 12-72 months with a history of recurrent severe lower respiratory tract infections, early administration of azithromycin during a lower respiratory tract infection reduced the likelihood of progression to a severe infection, but it is not clear whether the underlying disease in these children was bronchiolitis or some other disease process. 106 Given the current evidence, routine use of macrolides is not recommended in bronchiolitis and more research is needed to clarify any potential role it might have in the future.\\nHydration, suctioning, and chest physiotherapy have been suggested as supportive therapies. Infants with bronchiolitis might have diffi culty feeding because of nasal congestion and increased work of breathing; thus, hydration remains a cornerstone of therapy. A multicentre study of 759 infants younger than 12 months admitted to hospital with bronchiolitis showed no benefi t of intravenous fl uids compared with administration of fl uids by nasogastric tube in mean length of stay, admission to the intensive care unit, need for ventilatory support, and adverse events. This trial also found that a nasogastric tube might be easier to place than an intravenous line in these infants. 107 Most guidelines recommend either nasogastric or intravenous fl uids to maintain hydration, with the UK and Scottish guidelines preferring nasogastric or orogastric hydration in those that can tolerate it compared with intravenous hydration (table). If intravenous fl uids are used, isotonic fl uids are preferred to avoid risk of hyponatraemia. 38 Because infants are obligate nasal breathers, nasal suctioning has been suggested to help with clearing of the nares, improve the work of breathing, and improve feeding; however, suctioning might irritate the nasal mucosa and result in oedema. No randomised controlled trials have examined the role of nasal suctioning in bronchiolitis. The insuffi cient available evidence includes a retrospective cohort study of 740 infants 108 and a small observational study of 40 infants. 109 These studies suggest that deep suctioning might increase length of stay for inpatients, 108 infrequent suctioning is associated with an increased length of stay, 108 and oxygen saturation might increase after suctioning. 109 To draw conclusions about causality from these observational studies is diffi cult, because the potential for confouding by indication exists (eg, sicker children might be more likely to receive deep suctioning). Evidence suggests that oxygen saturation increases after nasal irrigation even without suctioning. 110 Current guidelines give diff ering recommendations with regard to suctioning; those that support its use recommend only superfi cial suctioning rather than deep suctioning (table) . 38, 39, 45 Chest physiotherapy use in bronchiolitis appears to vary by country. 111, 112 A recent Cochrane Collaboration review of 12 studies (1249 participants) demonstrated no evidence of benefi t to any type of chest physiotherapy among inpatients in length of stay, oxygen saturation, or respiratory parameters. 113 No published guidelines routinely recommend chest physiotherapy for the management of uncomplicated bronchiolitis in otherwise healthy children without respiratory comorbidities (table) . 38, 39, 45 Prognosis Much work has been published about the risk of developing recurrent wheezing and asthma following bronchiolitis in infancy. [114] [115] [116] Studies have followed birth cohorts to determine the risk of subsequent wheezing after lower respiratory tract infection in young childhood, and cohorts of children admitted to hospital with bronchiolitis. [114] [115] [116] [117] Overall, admission to hospital with bronchiolitis at a young age is associated with an increased risk of recurrent wheezing. Studies report that 17-60% of children with bronchiolitis might develop recurrent wheezing in the years following their initial admission to hospital. A large study from Taiwan that followed up 1981 children admitted with bronchiolitis before age 3 years found that by age 10 years, 351 (17\\u00b77%) of 1981 children with bronchiolitis had a diagnosis of asthma compared with 2159 (11\\u00b77%) of 18 527 controls (hazard ratio 1\\u00b758; 95% CI 1\\u00b741-1\\u00b771). 118 One small cohort study of 138 patients has suggested that 18 (39%) of 46 children admitted with bronchiolitis before 12 months have asthma by 18 years compared with eight (9%) of 92 controls. 115 However, another study that followed a birth cohort of 1246 children found that although lower respiratory tract infection in childhood was associated with an increased risk of recurrent wheezing, this association decreased with age and was not signifi cant by age 13 years. 116 Most children in this cohort had mild illness not requiring hospital admission, and severity of illness might be associated with the increased risk of asthma. 119 The question remains whether respiratory infection at a young age itself predisposes children to asthma through damage or alteration of lung function, or whether children with severe bronchiolitis might have individual risk factors (such as altered immune response or airway function) that predisposes them to both severe bronchiolitis and recurrent wheezing. 8\\nSubstantial knowledge gaps and controversies exist in the management of acute bronchiolitis. The role of nebulised hypertonic saline in acute management is not clear, resulting in confl icting recommendations across clinical guidelines (table) . Although meta-analyses suggest a small reduction of length of stay, these analyses are limited by heterogeneity, not accounting for duration of illness, and not considering the role of outlying study populations. 91 Two large multicentre trials do not support a clinically signifi cant diff erence in length of stay for inpatients and there is no clear evidence of cost benefi t. 4, [89] [90] [91] 120 While the same meta-analysis suggests a possible reduction in admissions for outpatients, the confi dence intervals were wide and the studies were also heterogeneous. 91 No multicentre studies of nebulised hypertonic saline have been completed in outpatients. Although the evidence is increasing that hypertonic saline has little role in meaningfully reducing the length of stay, its role in outpatients is less clear. A large outpatient multicentre study could clarify whether there is any benefi t.\\nEvidence also suggests that combined therapy with nebulised epinephrine and corticosteroids might reduce admissions to hospital. 95 Synergy between corticosteroids and \\u03b22 agonists is well documented in clinical trials of asthma management. [121] [122] [123] [124] Basic science literature also shows that \\u03b22 agonists and corticosteroids enhance each other's eff ectiveness, particularly with regard to antiinfl ammatory gene expression. 125, 126 Because of the economic burden of bronchiolitis and the plausible basic and clinical evidence for synergy, a large and multicentre trial is needed to ascertain whether combined therapy with epinephrine and corticosteroids is benefi cial.\\nOxygen saturation and the use of pulse oximetry play an important role in the decision to admit infants with bronchiolitis to hospital and in the length of their hospital stay. [51] [52] [53] [54] [55] [56] [57] [58] [59] Clinical practice guidelines also give confl icting guidance on the level of oxygen saturation at which admission should be considered. Furthermore, a substantial proportion of discharged infants have episodes of transient desaturation. 54 Again, in view of the large health-care costs associated with hospital admission in bronchiolitis, further research is needed to clarify the level of oxygen saturation requiring admission, the role of continuous and spot measurements of oxygen saturation, and the clinical importance of transient desaturations in otherwise stable young infants.\\nTAF contributed to the design and coordinated the writing of this manuscript. All authors contributed to the literature search and writing of this manuscript.\\nWe declare no competing interests. TAF, ACP, and JJZ have participated in trials of bronchiolitis funded by unrestricted public or academic institutional funding sources that had no infl uence on the conception and writing of this manuscript.\",\n          \"In March and April 2009, health professionals warned the Mexican Ministry of Health about the increase in incidence of acute respiratory illness in young adults [1] . In April, a new viral subtype of influenza A(H1N1)pdm09, a combination of swine, avian and human virus, was identified in the United States and in samples originated from Mexico [2, 3] . On April 25 th , the The patients were identified in the SINAN database, using the following variables for selecting the cases and controls: RT PCR (positive for influenza A(H1N1)pdm2009), final classification (confirmed), criteria (laboratory), evolution (recovered or death), hospitalization (yes), date of hospitalization (28/06/2009 to 29/08/2009), municipalities of residence and hospitals (metropolitan areas of Sao Paulo and Campinas).\\nAt the time the patients were selected for the study, the SINAN included 193 reported and confirmed deaths of patients with influenza A(H1N1)pdm2009 and SARI, all of which were assessed in this study (cases) . For each case of death, two control patients were selected among the 1988 patients hospitalized during the study period with reported and confirmed influenza A(H1N1)pdm2009 and SARI who recovered. The controls were time-paired randomly selected from those hospitalized in the same epidemiological week (EW) or, if necessary, one week later than the date of hospitalization of cases to prevent possible differences in opportunity of treatment and clinical management protocol.\\nData of the study were collected by trained health professionals using two standardized questionnaires. The hospital questionnaire collected information from medical charts, evaluated in 177 hospitals were patients were hospitalized. Of these, 56.5% belonged to the Unified Health System-SUS and 43.5% were private hospitals. The following data were collected: previous pathological history, occurrence of pregnancy, health care, symptomatology during admission, laboratory and radiological tests, need for ICU admission, treatment, complications and disease progression. The International Classification of Diseases, 10 th revision-ICD-10, was used for coding comorbidities indicated in the medical charts. The Ministry of Health provided the antiviral drug known as Oseltamivir for treatment during the pandemic. The protocol for clinical management established that the treatment was for cases hospitalized with SARI and ambulatory patients with risk conditions for severity. Treatment should be started within 48hours after symptom onset. All patients studied had laboratory confirmations of influenza A(H1N1)pdm09 from respiratory secretion samples analyzed using the RT-PCR method, performed by the Adolfo Lutz Institute [16] . The definition of SARI included: fever associated with cough and dyspnea; pneumonia; respiratory insufficiency; tachypnea; radiological changes compatible with pneumonia; oxygen therapy or mechanical ventilation. SARI was confirmed in all cases and controls selected for the study by data their on medical charts.\\nHousehold interviews were conducted in 161 cases (83.4%), with 14 refusals (7.3%) and 18 (9.3%) addresses not found. A total of 337 (85.1%) controls were interviewed, with 6 (1.5%) refusals and 43 (11.1%) addresses not found. The household questionnaire included socio-demographic data, history of previous care related to the episode that led to hospitalization and presence of influenza vaccination. Occupations were grouped according to the occupational pyramid of influenza risk [17] . For cases, interviews were conducted with the closest family members; for controls, with the patients themselves, excluding children, whose parents/legal guardians responded for them.\\nAfter data collection, questionnaires were assessed by supervisors to verify the consistency of information and the fulfillment of criteria to include patients in this study. Hospital and household questionnaires were input to the Sphynx database, version 5.\\nThe present study began in order to assist with decisions on surveillance and control actions. The use of data was approved by the Research Ethics Committee, School of Public Health, University of S\\u00e3o Paulo-FSP-USP (protocol 2283, OF.COEP/312/11). Data collection was performed in accordance with the recommendations of the National Health Council for Human Research, including the signing of an informed consent form by patients, family members or legal guardians.\\nClinical and demographic characteristics are shown as median and inter-quartile intervals or percentages, and comparisons were made using the Mann-Whitney U or chi-square tests, as required. Odds ratio-OR and 95% confidence intervals-95%CI were calculated to assess the factors associated with death. Variables with p-value < 0.20 in the univariate analysis and adjustment variables were selected for the multiple logistic regression, which used forward stepwise strategy. The model including all patients assessed the following dichotomous categorical variables: health insurance, previous care, vaccination against influenza in 2009, sex (reference category = male); and the following risk conditions: diabetes mellitus, obesity, chronic kidney disease, chronic liver disease, immunosuppression, chronic cardiovascular disease, chronic neurological disease and delay in development, using the absence of a condition as a reference category. The model also assessed: age group [<18 years (reference), 18 to 59 years, 60 years], use of antiviral therapy [no use (reference category), 48 hours after the onset of symptoms, > 48 and 72 hours, > 72 hours], high-risk individuals or those with any of the risk conditions listed above adapted from the Centers for Disease Control and Prevention-CDC [18] [no condition (reference category), one condition, and two or more conditions], and time between onset of symptoms and admission in days as the continuous variable. The model was controlled for epidemiologic week of admission at hospital (categorized as EW26-28 (reference), EW29-31 and EW31-34), and income. A second model was developed with patients aged 18 years and over, which also assessed level of education [without formal education or incomplete primary education, complete primary education, complete secondary education, complete higher education (reference category)] and occupational risk [very high and high, average and low (reference category)]. Patients aged 18 to 22 years with incomplete higher education were included in reference category of education. For the modelling, patients with no occupation were included in the reference category of occupational risk. Significance of variables was assessed by Wald test and adjustment of models by Hosmer-Lemeshow test.\\nThe analyses were performed using SPSS software, version 17.0. P-values < 0.05 were considered to be significant. Table 1 presents the socioeconomic characteristics of patients and information associated with the disease obtained from the household interviews. Among patients aged 18 years and over, there was a higher risk of death for those without formal education or with incomplete primary education, OR 2.33(95% CI 1. 13-4.89 ) and complete primary education, OR 3.06 (95%CI 1.41-6.82), when compared to those with higher education. Distribution of ethnic groups, family income and history of smoking were not associated with death. The history of previous vaccination against seasonal influenza in 2009 showed protection against death. Health professionals, considered to be at a high risk of developing influenza, showed protection against death, OR 0.15(95%CI 0.02-0.61). Patients who had sought care prior to hospitalization had a risk of death 3.70 times (95%CI 2.25-6.30) that of patients without previous care, with a predominance of emergency care (45.34%). Table 2 presents the distribution of cases and controls according to variables assessed in the hospital investigation. Median age of cases was higher than that of controls (p<0.001) and the 18-to-59-year age group had an OR for death of 4.03(95%CI 2.62-6.31), compared to those aged less than 18 years. Elderly individuals were also at a significant risk of death.\\nThere were no differences in sex distribution in either group. With regard to risk conditions for severity, 73.6% of cases and 38.1% of controls reported having at least one risk factor, with an OR of 4.49 (95%CI 3.09-6.60). When individuals with more than one of these conditions were assessed, the risk of death increased, OR 10.29 (95%CI 6.04-17.85). The following medical conditions were risk factors for death: obesity, diabetes mellitus, chronic cardiovascular diseases, chronic kidney diseases, neurological diseases and delay in development, chronic liver diseases and immunosuppressive diseases. The immunosuppression-related diseases were: neoplasms (n = 18), HIV/AIDS (n = 8), organ transplants (n = 6), autoimmune diseases (n = 5) and use of immunosuppressants (n = 3).\\nOf the 212 women of reproductive age (10 to 49 years) who were studied, 33.5% were pregnant and there were no differences in the frequency of pregnancy in either group.\\nThe use of antiviral treatment during hospitalization was a protective factor against death, with an OR of 0.67 (95%CI 0.45-0.98). When time between onset of symptoms and beginning of antiviral treatment was assessed, there was an 81% protection among individuals who received treatment within the first 48 hours, with an OR of 0.19 (95%CI 0.11-0.35), and 65% between 48 and 72 hours, with an OR of 0.35 (95%CI 0.16-0.74). After 72 hours, there was no significant protection. Median of days between date of onset of symptoms and date of hospitalization was 5.0 among cases and 2.0 among controls (p<0.001). Among patients with chronic pulmonary disease, median was 4.0 days among cases and 2.0 days among controls; among those with asthma, 4.5 days and 2.0 days; and among pregnant women, 4.0 days and 2.0 days, respectively. With regard to health professionals, the median between the date of onset of symptoms and that of hospitalization was three days for both cases and controls. Median between date of hospitalization and of death or hospital discharge was seven days among cases and four days among controls (Table 2) .\\nAccording to medical charts, fever and cough occurred in 87.8% of cases and 79.3% of controls; and the proportions of dyspnea were 87% and 62.7%, respectively. The definition of cases of Severe Acute Respiratory Syndrome -SARS (fever+cough+dyspnea) based on information obtained from medical charts occurred in 68.9% of cases and 56.5% of controls. Vomit and diarrhea were present in 8% of cases and 18% of controls. Treatment with antibiotics was used in 95.8% of cases and 73.1% of controls. The mean number of antibiotics used was 4.0 and 1.8 respectively. Among cases, there was a higher proportion of hospitalizations in intensive care units (83.4%x16.8%), use of mechanical ventilation (94.8%x10.8%) and dialysis (25.0%x2.3%). The proportion of complications was higher among cases than controls, with an emphasis on: respiratory distress syndrome (64.7%vs4.9%), shock (69.4%vs1.0%), sepsis (63.2%vs2.3%) and kidney complications (44.6%vs1.3%), respectively. Table 3 shows the laboratory findings of entry tests of patients in the hospital. Cases had lower numbers of leukocytes and platelets and higher values of creatine phosphokinase-CPK, lactic dehydrogenase-LDH, glutamic-oxaloacetic transaminase, glutamic-pyruvic transaminase, urea and creatinine than controls, with statistical significance.\\nIn sub-group analysis, previous conditions of risk for severity and obesity were risk factors for death in univariate analysis for children younger than 18 years and women of reproductive age (10 to 49 years). Among children, presence of immunosuppression, chronic cardiovascular disease and neurological disease/delay in development were also risk factors, as well as diabetes mellitus among women. Treatment with antiviral administered within 72 hours after the onset of symptoms was a protective factor for women of reproductive age. Tables 4 and 5 show variables associated with death in the final models of multiple logistic regression, in all patients studied and in sub-group analysis of adults (aged 18 years and over). In patients in general, risk factors for death were: being aged between 18 and 59 years (reference: until 18 years), having one or more risk conditions for the severity of the disease, obesity, immunosuppression and having sought care prior to hospitalization. Although chronic neurological diseases/delay in development and chronic liver disease were highly associated with death, these variables were excluded from the final model because they had estimates with large confidence intervals. Private health insurance and antiviral treatment administered in the first 72 hours after the onset of symptoms were protective factors against death. Among adults aged 18 years and over, the risk factors for death were having more than one risk factor for severity of the disease, obesity and care prior to hospitalization. The protective factors were: antiviral drug administered in the first 48 hours after the onset of symptoms and being a health professional or health unit worker.\\nSensitivity analysis was performed to assess the severity of patients admitted during the study period. The proportion of ICU admissions and use of mechanical ventilation was estimated for epidemiological week of hospitalization, categorized as EW 26 to 28, EW 29 to 31 and EW 32 to 34. The results showed higher proportions of ICU admissions (50.0% x 16.8% x 7.7%) and mechanical ventilation (45.4% x 10.8% x 2.6%) in the three first weeks, among the controls. The cases showed high proportions in all periods analyzed: 89.7% x 84.0% x 76.9% for need of ICU and 89.7% x 97.6% x 89.7% for need of mechanical ventilation, in the 3 periods, respectively. The opportunity for antiviral treatment showed lower proportion of treatment in the first period of the study, in both cases (37.9% x 74.4% x 74.3%) and controls (59.1% x 74.1% x 89.7%). Sensitivity analysis to assess the opportunity for treatment of children and adults showed similar proportions for the groups: 73.9% and 74%, respectively.\\nIn the present study, risk factors for death were analyzed in patients infected with influenza A (H1N1)pdm09 and hospitalized with SARI. There was an increased risk of death in patients aged 18 to 59 years; those with obesity, immunosuppression, neurological and developmental diseases; or those who had received care prior to hospitalization. Additionally, presence of at least one risk factor for aggravation of the disease was also associated with occurrence of death. Use of antiviral drugs, especially when administered in the first 72 hours after onset of symptoms was protective factor against death.\\nThe design of this case-control study, including all deaths reported and confirmed in the region during the study period and the time-paired random selection of controls among patients who survived after hospitalization with SARI, with hospital and household data collection, enabled the analysis of the factors associated with death to be extended, including socioeconomic variables, access to health services, and data on hospitalizations.\\nFew studies have assessed risk factors for death in patients with SARI [15, 19] . Some studies have analyzed risk factors for severity of the disease and deaths, comparing patients hospitalized in ICUs and in wards [13, 14, [20] [21] [22] [23] , whereas one study used the rate of mortality from influenza A(H1N1)pdm09 in two epidemic waves [24] . A study conducted in Australia analyzed risk factors for hospitalization due to influenza, comparing cases with controls without infection in the community [25] . Other studies have analyzed epidemiological surveillance data, including mortality rates and comparisons among deaths, hospitalizations, treatment and other factors [9, 10, 12, [26] [27] [28] [29] [30] [31] [32] [33] [34] [35] . Some of the factors associated with death identified in this study have been described by other authors, such as the presence of comorbidities [31, 35] , age18years [13] , obesity [15, 19, 20] , neurological and developmental diseases [23, 24, 31] and immunosuppression [15, 21, 24, 34] . Children showed lower risk of death, despite a similar proportion of treatment than adults. A similar result was observed in a study of systematic review and metaanalysis, where the risk of death was 0.28 (95% CI 0.19-0.141), when compared to young adults [36] . In contrast, other factors associated with severity of infection that have been reported by different researchers, such as diabetes [19, 34] ; other chronic pulmonary diseases, excluding asthma and chronic obstructive pulmonary disease [20] ; chronic pulmonary disease, including asthma [31] ; and chronic cardiovascular disease [21, 24, 34] , were not associated with mortality in the present study. The population assessed, which was restricted to hospitalized patients with severe influenza could at least partly explain these differences. Diabetes, chronic cardiovascular and kidney disease, associated with death in univariate analysis, did not remain risk factors after multiple analysis. The design of this study, with data collected on medical charts and household interviews, allowed for adjustments in the analysis that can also explain part of these differences.\\nDeaths occurred mainly in adults aged 18 to 59 years, with a small proportion of cases and controls among elderly individuals. A possible preexisting immunity against the new influenza A(H1N1)pdm09 virus in this age group has been suggested, probably due to previous exposure, through other infections or vaccinations, to an influenza A(H1N1) virus which is genetically and antigenically more associated with the new virus than contemporary strains of seasonal H1N1 viruses [37] .\\nAll patients in the present study, whether cases or controls, had a laboratory-confirmed infection of influenza A/H1N1pdm09, which was [38] .\\nIt is possible that the probability of being hospitalized varied throughout the study. Treatment was available for hospitalized patients and for ambulatory patients with risk conditions for severity. This could have influenced hospitalization, but not much in our study population, consisting of patients with SARI. During the pandemic, criteria for hospitalization changed as well as the availability of treatment, especially in the first 3 weeks of the epidemic, as shown in sensitivity analysis, confirming the need for controlling period of hospitalization in the study.\\nThe socioeconomic profile was similar in both groups assessed, with no difference in income or having private health insurance. There was greater protection against death among patients with an occupation with higher risk of exposure to infection, such as health professionals. A greater opportunity of access to treatment among these workers could explain such findings, as the period between the date of onset of symptoms and the date of admission was shorter in this group. The low proportion of deaths among patients categorized as having a higher risk of exposure has also been observed in other studies [30, 33] .\\nPregnancy was not a risk factor for death in this study. In contrast, frequency of pregnant women among those of reproductive age hospitalized with severe manifestation of influenza A (H1N1)pdm09 was high, suggesting that pregnancy could be associated with development of SARI. An association between pregnancy and use of mechanical ventilation during hospitalization due to influenza A(H1N1)pdm2009 [25] and a higher admission rate were reported when compared to the general population [32] . In 2009, in the state of S\\u00e3o Paulo, 27.4% of all confirmed cases of influenza A(H1N1)pdm09 in women of reproductive age (10 to 49 years), hospitalized and recorded in the SINAN database, were found in pregnant women. In Brazil, only 4.27% of women of reproductive age had live newborns in the reference year according to the 2010 Census [39] .\\nPatients who died presented important laboratory changes when compared to controls, with lower levels of leukocytes and platelets and an increase in CPK, LDH, urea and creatinine. Studies revealed a similar trend for the laboratory results of leukocytes [23] , CPK [26, 27, 30] , LDH [26] , platelets [22] and creatinine [27, 30, 34] .\\nAlthough most patients who died received antiviral treatment, only 9.3% were treated in the first 48 hours. Early treatment was a protective factor against death, especially when provided in the first 72 hours after onset of symptoms. Other studies have also shown a more favorable prognostic with antiviral treatment [13, 21, 23, 27, 28, 29, 34] .\\nHistory of care prior to hospitalization was a risk factor for death from influenza A(H1N1) pdm09, which could reflect the difficulty in accessing health services. Emergency services were primarily sought for care prior to hospitalization, which could have contributed to greater delay in diagnosis and beginning of treatment. A study conducted in Mexico showed a similar result with regard to delay in hospitalization [29] .\\nIn one study, which assessed the overall standardized mortality rate from influenza A (H1N1)pdm09, the mortality among patients with preexisting chronic respiratory diseases was lower than those with other chronic diseases 24 . There might have been problems with recording information about previous diseases in the medical charts in the presence of acute pulmonary disease, especially among those with chronic pulmonary disease.\\nThe present study had some limitations. Underreporting of SARI by health professionals is likely to have occurred, as reporting this disease was not compulsory in Brazil before the pandemic. However, during the pandemic, professionals were more sensitive to reporting it. The system for notification, which was mandatory during the epidemic, was facilitated to allow for rapid notifications. The availability of samples from autopsies of cases (deaths) could have resulted in greater access to the laboratory diagnosis of this infection. Household data collection could also have caused information bias, as data on cases were collected through interviews with family members, while data on controls were obtained from the patients themselves, when they were adults. The proportion of refusals of household interviews was higher among cases (7.3%) than controls (1.6%), p = 0.001. Another possible limitation was memory bias, as a result of the time between the hospitalization and interview. However, the information analyzed from household interviews was about socio-demographic characteristics, which is less vulnerable to information and memory bias. Data on preexisting diseases were obtained from medical charts in hospitals, where they are completed at the moment of hospitalization, in the presence of the patients. The quality of completion of medical charts can differ between hospitals. Nonetheless, information about the presence of chronic diseases, clinical management of patients, laboratory results, radiological tests and treatment was available in these charts. The difficulties encountered during data collection were minimized by the use of a standardized questionnaire.\\nIn conclusion, the risk conditions for the severity of the disease, particularly obesity, immunosuppression and neurological/developmental diseases, were also important risk factors for death, emphasizing the need for good vaccination coverage during influenza vaccination campaigns in all patients with chronic diseases belonging to more vulnerable groups. Another relevant result was the antiviral treatment within the first 72 hours of onset of symptoms as a protective factor, also emphasizing the need to warn the population about the importance of seeking early medical care when an influenza-like illness appears, especially among high-risk groups and those with signs of disease aggravation. Training of health professionals, especially physicians, is required for adequate clinical management of patients and early antiviral treatment.\"\n        ],\n        \"semantic_type\": \"\",\n        \"description\": \"\"\n      }\n    },\n    {\n      \"column\": \"authors\",\n      \"properties\": {\n        \"dtype\": \"string\",\n        \"num_unique_values\": 1422,\n        \"samples\": [\n          \" Chen, Junbo.  Zhu, Huachen.  Horby, Peter W.. <br>Wang, Qianli.  Zhou, Jiaxin.  Jiang, Hui.  Liu, Liwei. <br>Zhang, Tianchen.  Zhang, Yongli.  Chen, Xinhua.  Deng,<br>Xiaowei.  Nikolay, Birgit.  Wang, Wei.  Cauchemez, Simon. <br>Guan, Yi.  Uyeki, Timothy M..  Yu, Hongjie\",\n          \" Chen, Yu.  Dong, Yuanji.  Cai, Shaozhe.  Ye, Cong. <br>Dong, Lingli\",\n          \" Arens, Amanda.  Scott, Cheryl.  Osburn, Bennie\"\n        ],\n        \"semantic_type\": \"\",\n        \"description\": \"\"\n      }\n    },\n    {\n      \"column\": \"title\",\n      \"properties\": {\n        \"dtype\": \"string\",\n        \"num_unique_values\": 1454,\n        \"samples\": [\n          \" Peptide Vaccine: Progress and Challenges\",\n          \" The Impact of Ebola Virus Disease on Government<br>Expenditure in Sierra Leone\",\n          \" The clinical implication of dynamic<br>neutrophil to lymphocyte ratio and D-dimer in COVID-19: A<br>retrospective study in Suzhou China\"\n        ],\n        \"semantic_type\": \"\",\n        \"description\": \"\"\n      }\n    },\n    {\n      \"column\": \"journal\",\n      \"properties\": {\n        \"dtype\": \"string\",\n        \"num_unique_values\": 785,\n        \"samples\": [\n          \"AMB Express\",\n          \"Dail and Hammar&#x02019;s Pulmonary Pathology\",\n          \"Benumof's Airway Management\"\n        ],\n        \"semantic_type\": \"\",\n        \"description\": \"\"\n      }\n    },\n    {\n      \"column\": \"abstract_summary\",\n      \"properties\": {\n        \"dtype\": \"string\",\n        \"num_unique_values\": 1044,\n        \"samples\": [\n          \" InnoMetrics is the system that aims at<br>collecting software development process metrics in a<br>non-invasive way to access and optimize the development<br>process and its efficiency. This paper demonstrates<br>the development and analysis of energy<br>consumption of MacOS systems based on the software process<br>measurement data. It represents the experience of the<br>development of MacOS system collector and Transfer, in<br>addition to the user interface and early analysis of<br>energy consumption metrics calculations.\",\n          \" During the global outbreak of severe acute<br>respiratory syndrome (SARS) in 2003, treatment was<br>empiric. We report the case history of the index patient<br>in a hospital outbreak of SARS in Hong Kong. The<br>patient recovered after conventional antimicrobial<br>therapy. Further studies are needed to address<br>treatment of SARS, which has high attack and death rates.\",\n          \" Intestinal transplantation (IT) is the least<br>common form of organ transplantation; however, it has<br>shown exceptional growth and improvement in graft<br>survival rates over the past two decades mainly due to<br>better outcomes achieved during the first year of<br>transplantation (76 % at 1 year), due to improvement in surgical<br>techniques and the development of better<br>immunosupressive therapies as we understand more about the<br>relationship between the recipient and host immune system.<br>There are still ongoing issues with chronic<br>rejection and long-term survival. Intestinal<br>transplantation is still an acceptable therapy for patients<br>with intestinal failure (IF), but it is generally...\"\n        ],\n        \"semantic_type\": \"\",\n        \"description\": \"\"\n      }\n    },\n    {\n      \"column\": \"cleaned_text\",\n      \"properties\": {\n        \"dtype\": \"object\",\n        \"semantic_type\": \"\",\n        \"description\": \"\"\n      }\n    },\n    {\n      \"column\": \"cleaned_text_str\",\n      \"properties\": {\n        \"dtype\": \"string\",\n        \"num_unique_values\": 1453,\n        \"samples\": [\n          \"respiratory tract infection common infants young children major public health problem age group 1 mainly caused infection seasonal viruses respiratory syncytial virus rsv rhinovirus 2 3 4 infections manifest upper respiratory infection 20 40 develop lower respiratory infection small number 2 -3 infected infants require hospitalization 5 6 particular concern infants lower respiratory viral infection increased risk developing recurrent wheezing childhood asthma 7 8 9 10 11 12 13 risk factors identified development severe lower respiratory viral infections premature birth young age born relation rsv season underlying chronic lung disease congenital heart disease high parity young maternal age poor socioeconomic factors 14 15 16 recently genetic polymorphisms identified associated severe viral infection 17 18 identified genetic variants associated severe lower respiratory tract infections associated pathways common asthma including innate immune genes involved cytokine chemokine signaling epithelial cell function 13 17 19 20 21 22 23 \\n clinical hallmark asthma wheezing therapy includes beta agonists number clinical studies past 2 decades examined relationship genetic variants \\u03b2 2 -adrenergic receptor adrb2 asthma 24 25 26 27 28 29 30 polymorphism resulted gly16 allele associated decreased bronchodilator responses early clinical studies larger clinical studies provided equivocal results 31 32 33 genetic variants promoter region adrb2 associated asthma severity 28 34 despite significant interest adrb2 polymorphisms related asthma relative contribution adrb2 genetic variants severity respiratory viral infections established wheezing clinical hallmark lower respiratory viral infections variable response bronchodilators demonstrated infants hospitalized rsv lower respiratory viral infections possible adrb2 genetic variants contribute severity acute respiratory viral infections 35 36 \\n aim present study explore associations genetic variants adrb2 associated response asthma therapy acute asthma exacerbation severity infant acute respiratory viral infections examined relationship 5 common polymorphisms promoter region coding block adrb2 gene infant severity respiratory infections prospective cohort caucasian african american infants \\n order examine association genetic variants regulatory regions adrb2 gene severity viral respiratory infections infancy conducted analysis infants enrolled tennessee children respiratory initiative tcri prospective cohort mother-infant dyads enrolled infant acute viral respiratory illness respiratory viral seasons 2004 2008 term \\u226537 weeks non-low birth weight \\u22652275 g infants healthy significant co-morbidities cardio-pulmonary disease eligible oversampling hospitalization infants eighty percent enrolled subjects respiratory virus identified polymerase chain reaction rsv rhinovirus influenza parainfluenza human metapneumovirus coronavirus 56 labconfirmed rsv infection rationale methods detailed inclusion exclusion criteria reported previously 37 630 infants enrolled tcri study limited 261 caucasian 113 african american infants dna available protocol approved institutional review board vanderbilt university mothers provided informed consent infants parents provided written informed consent child study participation \\n infants included study upper respiratory tract infections lower respiratory tract infections severity acute respiratory infection determined experienced research nurses medical record chart review ordinal bronchiolitis severity score bss score aggregate assigned values ranging 0 3 categories respiratory rate room air oxygen saturation presence extent wheezing presence extent flaring retractions 37 38 39 bss ranges 0 12 higher scores indicating severe illness difference 0.5 clinically significant difference 40 measured bss end outpatient visit discharge inpatient admission severe recorded value analysis \\n dna extracted vanderbilt dna core laboratory blood saliva samples oragene dna genotek standardized protocols oragene samples processed immediately collection blood samples processed immediately frozen processing genotyping pre-specified loci -2387 -2274 -1343 46 79 adrb2 gene taqman sm \\u00a9 genotyping assays applied biosystems foster city usa predetermined primers single-nucleotide polymorphisms (snp)s rs1432623 rs11168068 rs2400707 promoter region rs1042713 snp rs1042714 snp coding region adrb2 gene probes applied biosystems assays labeled 5\\u2032end vic \\u00ae 6-carboxy-fluorescine fam reporter dyes reaction components include 2.5 \\u03bcl taqman universal pcr master mix applied biosystems 0.125 \\u03bcl 0.250 \\u03bcl assay mix applied biosystems approximately 5 ng genomic dna total volume 10 \\u03bcl single tube reaction pcr amplification performed thermal cycler techne tc-412 initial step 95 \\u00b0 c 10 min followed 50 cycles 92 \\u00b0 c 15 60 \\u00b0 c 1 min amplification fluorescence sample read abi 7900ht dna resources core vanderbilt university analyzed sequence detection software applied biosystems percent samples run blind duplicates 100 concordance primers probes snps listed table 1 \\n demographics characteristics 374 infants included upper respiratory infection lower respiratory tract infection presented frequencies proportions specific categorical variables medians interquartile ranges calculated continuous variables analyses stratified race caucasian african american surrogate measure european african ancestry reduce known impact population substructure produce spurious associations 41 pairwise linkage disequilibrium ld estimated snps adrb2 standardized summary statistics r calculated haploview program whitehead institute biomedical research cambridge usa snps tested deviations hardy weinberg proportions haploview 42 promoter coding combined promoter coding snps assigned blocks confidence interval gabriel al. 43 individual haplotypes estimated program phase v2.1.1 44 45 haplotype frequencies block estimated expectation maximization em algorithm haploview program non-parametric kruskal-wallis test applied compare difference bss infants snp genotype haplotype genotype haplotype genotypes frequency 2 excluded haplotype analysis proportional odds model ordinal bss test additive effect haplotypes covariates included regression model included infant age enrollment gender daycare exposure secondhand smoke exposure prior history breastfeeding siblings home enrollment season analyses conducted separately according race analyses performed r-software version 2.11.1 www.r-project.org two-sided 5 significance level statistical inferences \\n total 374 infants 261 caucasians 113 african americans enrolled tcri cohort available dna adrb2 polymorphism tested table 3 lists allele frequencies 5 snps separately caucasian african americans caucasians african americans infants slightly g alleles promoter snp rs1432623 figs. 1 2 non statistical difference consistent caucasians african americans african americans decreasing trend severity acute respiratory infection increasing number copies c c allele promoter snps rs1432623 rs11168068 rs2400707 respectively fig. 1 analyses additive models indicated statistical significance p 0.05 \\n 2 5 32 possible haplotype combinations haplotypes 2 frequency identified table 4 fig. 3 promoter haplotypes coding block haplotypes statistically significant differences promoter coding block haplotype frequencies caucasian african americans 5 snps combined distribution haplotype frequency significantly different caucasians african americans genotype ccaac common african americans rare caucasians p 0.001 \\n promoter haplotype cca associated decreased bss african americans dose dependent manner p 0.037 univariate analysis fig. 4 infants cca haplotype highest bss median 5.5 interquartile range iqr 2.0 8.0 infants copy cca haplotype lower bronchiolitis severity score median 4.0 iqr 1.0 7.5 infants 2 copies cca haplotype comparatively lowest bss median 3.0 iqr 1.0 4.0 dose dependent relationship persisted adjusting infant age gender daycare exposure secondhand smoke exposure prior history breastfeeding siblings home enrollment season adjusted odds ratio 0.59 95 confidence interval ci 0.36 0.98 p 0.042 protective effect cca haplotype consistent categorized acute respiratory infection upper respiratory infection lower respiratory tract infection similar dose dependent manner infants copies cca haplotype likely lower respiratory tract infection compared infants copies cca haplotype p 0.05 hand promoter haplotype ttg associated higher bss african americans additional file 1 \\n protective effect cca haplotype african americans identified combined haplotypes cca combination figs. 5 6 sample size effect statistically significant addition african american infants coding block haplotype gg cca combination promoter region similar additive protective effect gg african americans additional file 2 relationship bss coding block haplotypes african americans additional files 3 4 true combined haplotypes additional files 5 6 \\n caucasian african american infants significant relationships bss coding block haplotypes gg haplotype african americans additional file 2 true combined haplotypes general significant relationship bss haplotypes caucasian infants similar promoter haplotype cca relationship bss caucasians observed african americans fig. 4 \\n study identified haplotype promoter region adrb2 gene associated decreased bss african american infants promoter haplotype adrb2 gene associated severity respiratory infections caucasian infants common coding block polymorphisms corresponding amino acid changes codons 16 27 associated protective effect severity respiratory infections caucasian african american infants results agree studied conducted janssen colleagues found relationship snp rs1042713 susceptibility rsv bronchiolitis european population 17 observation cca promoter region haplotype associated protective effect respiratory viral infection severity similar observation cohort adults hospitalized asthma exacerbation haplotype associated spirometric improvement response asthma-specific therapy included beta-agonists corticosteroids 34 study noted spirometric improvement caucasian patients small number african american non-responders cohort limited ability detect similar effect african americans 34 \\n mechanism haplotype promoter region adrb2 gene associated decreased bss known speculate 46 moore al. demonstrated adrb2 genetic variants associated differences adrb2 expression cultured human airway smooth muscle hasm cells vitro system model changes human airway 47 studies performed prior identification complex promoter haplotypes aware haplotypes studied hasm small differences airway tone important setting acute viral respiratory infections \\n scope current observational study speculate adrb2 genotype influence response bronchiolitis-specific therapy primary goal tcri establish longitudinal prospective investigation infants biological mothers acute long-term health consequences varying viral respiratory tract infections early childhood outcomes 37 bss assigned infant acute care visit hospital admission study quantify effects different treatments including beta-agonist therapy number clinical studies suggested infants bronchiolitis appear improve bronchodilator therapy average efficacious 48 49 50 adrb2 primary target bronchodilator therapy variation response beta-agonists reflect genetic variants adrb2 \\n diversity snps adrb2 gene explored drysdale al. immortalized lymphocytes 23 caucasians 19 african-americans 20 asians 15 hispanic-latinos 27 report -2387 -2274 -1343 coding block snps discussed manuscript 28 frequency distributions observed study consistent allele frequencies reported reference population international hapmap project 51 hawkins al. 28 stratified analyses race allow detect association severity respiratory illness haplotype frequency racial group minimize spurious associations population substructure recognize significance findings limited small sample size classifying infants study severity score measuring direct response bronchodilator therapy bss correlated severity disease classified upper respiratory lower respiratory illness sensitivity analyses severity disease upper versus lower respiratory illness showed consistent results indicated bss valid surrogate disease severity lastly p value 0.042 threshold statistical significance p value reflects sample size 113 african americans study association bss cca haplotype african americans powerful depends p value effect size odds ratio clinical significance findings difference 0.5 bss considered clinically significant 65 reduced odds developing severe disease higher bss comparing 2 copies cca 0 copy cca clinically significant meaningful \\n hope hypothesis-generating study encourage incorporation pharmacokinetic data future studies bronchiolitis validation studies populations needed study underscores development personalized approaches treatment bronchiolitis benefit analysis specific genetic variants \\n summary observed promoter region haplotype adrb2 cca associated protective effect respiratory illness severity african american infants era personalized medicine adrb2 genotype predictive severity respiratory viral infection potential response therapy common acute infant illness currently therapy steps validate findings evaluate available clinical trial data dna available fig. 6 bss distribution stratified copy number combined haplotype ccaac african americans infants scatter plots box-and-whisker plots bss 0 1 2 copies combined haplotype ccaac african american infants p values obtained multivariable regression model adjusted infant age enrollment gender daycare exposure secondhand smoke exposure prior history breastfeeding siblings home enrollment season\",\n          \"acute bronchiolitis viral infection lower respiratory tract substantial health burdens infants young children worldwide 1 respiratory syncytial virus prevalent viral cause bronchiolitis infants estimates suggest 34 million new cases lower respiratory infection respiratory syncytial virus occur globally children younger 5 years 3\\u00b74 million admissions hospitals 199 000 deaths year predominantly developing world 2 developed countries usa bronchiolitis common reason admission hospital fi rst 12 months life 3 accounting approximately 100 000 infant admissions annually admissions hospital declined 2000 2010 emergency department visits increased addition increased use mechanical ventilation hospital charges 4 5 clinical management remains challenging despite frequency global reach economic cost morbidity mortality associated bronchiolitis treatment strategies including bronchodilators corticosteroids showed eff ect pooled metaanalyses making supportive care hallmark current therapy seminar aim summarise current evidence epidemiology pathophysiology diagnostic approach management acute viral bronchiolitis \\n bronchiolitis seasonal infection season typically beginning late october temperate northern hemisphere peaking january february ending april 6 globally independent region respiratory syncytial virus infection peaks consistently annual biannual epidemics peak duration epidemics vary worldwide consistent year-to-year country 7 data suggest climate associated prevalence respiratory syncytial virus infection global surveillance suggesting infection peaks wet months areas high precipitation cooler months hot regions 7 indoor crowding population-dense areas rainy seasons cooler months factor facilitates viral transmission 8 additionally weather-related factors inhalation cold dry air impair ciliary function airway mucosa inhibition temperature-dependent antiviral responses infl uence disease transmission severity 9 10 altitude climate meteorological conditions wind speed dew point shown modest association bronchiolitis 11 12 furthermore air pollutants ozone traffi c pollutants associated exacerbations respiratory infections children younger 5 years 13 14 15 environmental tobacco smoke associated increased risk respiratory syncytial virus-attributable admission hospital disease severity admitted 16 17 respiratory viral infections risk severe respiratory syncytial virus bronchiolitis greater boys girls 18 19 diff erence diff erences lung airway development genetic factors 8 20 pathophysiology bronchiolitis characterised extensive infl ammation oedema airways increased mucus production necrosis airway epithelial cells 21 respiratory syncytial virus binds epithelial cells replicates resulting epithelial necrosis ciliary destruction 21 22 cell destruction triggers infl ammatory response proliferation polymorphonuclear cells lymphocytes submucosa adventitial tissues oedematous increased mucus secretion 22 plugs composed cellular debris mucus form bronchiole lumens leading bronchiolar obstruction air trapping diff erent degrees lobar collapse 21 microbiology molecular testing led improved understanding viruses associated bronchiolitis respiratory syncytial virus remains commonly identifi ed virus detected 41 83 patients 23 24 25 26 27 28 viruses associated bronchiolitis include rhinovirus metapneumovirus coronavirus human bocavirus infl uenza virus adenovirus parainfl uenza virus 24 25 27 29 30 studies investigated severity illness measured need hospital admission length hospital stay intensive care unit admission repeated emergency department visits apnoea associated specifi c viral infections co-infections evidence confl icting data studies shown infections involving single virus respiratory syncytial virus associated severe course compared viruses 31 32 30 children bronchiolitis found co-infections viruses 29 combination respiratory syncytial virus rhinovirus commonly reported 29 evidence suggests co-infection bronchiolitis particularly respiratory syncytial virus combination rhinovirus metapneumovirus associated severe disease course compared infection single virus 25 30 31 33 34 studies confi rm association 24 26 furthermore use nucleic acid amplifi cation tests greatly improved ability detect viruses present respiratory infections studies technologies found respiratory virus 30 children younger 6 years respiratory symptoms 35 36 37 viruses detected asymptomatic colonisation incubation clinical infection prolonged viral shedding post-infection confl icting evidence high prevalence respiratory viruses asymptomatic children suggest indication time management vary based presumed viral cause presence absence viral co-infections \\n diagnosis bronchiolitis clinical requires clinician recognise signs symptoms viral lower respiratory tract infection young children peak incidence occurs 3 months 6 months age 38 early clinical defi nition court noted recent practice guidelines specifi c defi nition infants 38 39 40 physiology occur toddlers older 12 months clinical trials excluded children included small subgroup patients bronchiolitis toddlers overlap conditions viral-induced wheezing asthma application evidence trials predominantly assessing infants appropriate eff orts focus defi nition assist eff orts standardise care \\n classic clinical presentation bronchiolitis starts symptoms viral upper respiratory infection nasal discharge progress lower respiratory tract days fi gure 1 timing symptom progression vary young infants present apnoea lower respiratory tract symptoms bronchiolitis include persistent cough tachypnoea increased work breathing shown intercostal supraclavicular retractions use abdominal muscles grunting nasal fl aring auscultatory fi ndings include crackles wheeze hallmark characteristic bronchiolitis minute-to-minute variation clinical fi ndings mucus debris airways cleared coughing child state changes sleep agitation variation confound assessment requires examinations period observation nasal congestion confound clinical assessment nasal suctioning help ascertain fi ndings truly lower airways fever present infants bronchiolitis usually present early illness temperature 39 \\u00b0 c 38 median duration symptoms 2 weeks 10 20 infants symptoms 3 weeks onset 38 41 clinical scores shown studies clinical protocols correlate disease severity improvement 42 43 documentation score useful objective measure individual scores highly predictive repeated combined measures severity universal assessment guide decision making diff erential diagnosis bronchiolitis includes considerations infectious non-infectious causes absence upper respiratory symptoms raise suspicion causes respiratory distress young infants including cardiac disease congenital airway abnormalities vascular ring foreign body aspiration infections resemble complicate bronchiolitis pertussis considered infants severe paroxysmal cough known exposure bacterial infections complicating viral bronchiolitis including otitis media pneumonia present new fever worsening status later course illness \\n risk factors associated progression severe bronchiolitis supported strongest evidence include presence chronic lung disease prematurity haemodynamically important congenital heart disease immunodefi ciency neuromuscular disorders considered high risk practice guidelines 38 39 young infants aged 2 3 months history premature birth especially 32 weeks gestation high risk progression present apnoea clinical fi ndings studies assessing risk apnoea hospital found limited infants 1 month full-term infants 48 weeks postconceptional age preterm infants apnoea observed admission 44 \\n bronchiolitis clinical diagnosis based history physical examination according consensus national guidelines infants typical presentation bronchiolitis routine imaging laboratory testing recommended increase costs evidence benefi table \\n appropriate use pulse oximetry monitoring initiation oxygen bronchiolitis received increasing attention studies practice guidelines findings suggest arbitrary thresholds oxygen administration drive hospital admissions prolong hospital length stay outcomes represent morbidity bronchiolitis developing evidence suggests intermittent hypoxaemia occur commonly stable infants bronchiolitis raises questions factor sole indication admission hospitals canadian randomised trial found reduced admissions hospital emergency department increase revisits pulse oximeters displayed values 3 higher actual value suggesting arbitrary pulse oximetry thresholds result unnecessary admissions 51 similar trial uk hospital setting found reduction oxygen threshold 94 90 resulted earlier discharge hospital evidence adverse outcomes 52 trial comparing intermittent versus continuous pulse oximetry non-hypoxaemic infants hospitals found similar outcomes groups 53 trial evidence supports recommendations practice guidelines clinicians use threshold 90 initiation oxygen uk guidelines recommend 92 38 39 child improves reduction intensity monitoring intermittent checks appropriate recent study blinded oximetry home showed substantial proportion infants bronchiolitis oxygen desaturations 90 particularly sleep calling question arbitrary thresholds hospital admissions initiation oxygen 54 evidence probably lead eff orts reduce continuous monitoring children indications monitoring \\n majority children bronchiolitis normal radiographs radiographic fi ndings consistent simple bronchiolitis 55 58 including peribronchial thickening hyperinfl ation atelectasis prospective study 58 routine radiographs assessment bronchiolitis emergency department reported airspace disease 17 7 246 patients despite low prevalence radiographic pneumonia studies reported increase antibiotic prescription radiographs performed non-specifi c fi ndings infl uence clinicians decisions 58 59 factors associated defi nite focal infi ltrates consistent pneumonia include hypoxia oxygen saturation 92 56 57 58 60 grunting persistently focal crackles fever especially 39 \\u00b0 c 38 chest radiographs considered patients presentation classic bronchiolitis situations include diagnosis foreign body aspiration high diff erential diagnosis child severely ill respiratory failure imminent symptoms progressing resolving according typical disease course expected bronchiolitis lung ultrasound increasingly assess cardiopulmonary conditions adults children studies investigated use lung ultrasound diagnosis bronchiolitis small studies found ultrasound fi ndings infants bronchiolitis correlate clinical fi ndings specifi c chest radiography 61,62 studies needed establish role ultrasound diagnosis assessment severity \\n development pcr detect respiratory viruses nasopharynx interest use viral testing causative diagnosis bronchiolitis increased virological testing generally assist management insuffi cient predict outcomes 30 63 national guidelines recommend routine virological testing bronchiolitis table recent studies suggest higher respiratory syncytial virus genomic load measured quantitative pcr associated increased length stay use respiratory support need intensive care addition recurrent wheezing compared lower viral loads 28 31 32 64 study warranted confi rm association clarify viral load measurement improves understanding disease pathophysiology severity guidelines recommend respiratory syncytial virus testing guide cohorting patients viruses likely cause bronchiolitis transmitted similar fashion close contact large-particle aerosols direct contact contaminated fomites 65 66 67 infection control dependent identifi cation specifi c viruses following strict precautions including hand hygiene separating infants shared hospital rooms 1 infection control procedures 65 additionally given sensitivity pcr testing results interpreted caution certain viruses rhinovirus detected viral shedding unrelated illness colonisation certain viruses respiratory syncytial virus metapneumovirus associated acute infection \\n blood urine testing routinely recommended standard practice diagnostic work-up bronchiolitis table blood gas measurement routinely obtained infants bronchiolitis signs impending respiratory failure severe distress proportions bacterial infections especially bacteraemia meningitis low infants bronchiolitis 68 abnormal white blood cell count rarely useful predicting bacterial infections children infected respiratory syncytial virus 69 guidelines universally recommend complete blood counts infants bronchiolitis blood count assessment fever infants younger 1 2 months similarly given bacteraemia exceedingly rare cited proportions 0\\u00b71 post-pneumococcal vaccine era blood cultures routinely performed septic work-up infants younger 1 2 months 70 71 severe illness signs sepsis hydration status important consideration infants bronchiolitis determined clinical examination routine measurement serum electrolytes little value majority infants \\n urinary tract infections infants bronchiolitis occur greater frequency bacteraemia meningitis proportions ranging 1 7 68 71 72 reasonable obtain urinalysis urine culture infants aged 60 days fever older febrile infants risk factors urinary tract infections 73 urine routinely obtained infants bronchiolitis \\n current recommendations management bronchiolitis focus agents treat patho physiological eff ects viral lower respiratory infection eg bronchodilators hypertonic saline specifi c antivirals ribavirin treat respiratory syncytial virus infection recommended practice guidelines typical cases bronchiolitis challenging delivery methods high cost potential health risks caregivers multiple new agents prevention treatment investigation available future including immunoglobulins small interfering rna interference fusion inhibitors small molecules 74 numerous studies assessed role bronchodilators treatment bronchiolitis systematic reviews found consistent benefi t. 2014 cochrane collaboration systematic review identifi ed 30 studies assessing bronchodilators predominantly salbutamol excluding epinephrine 21 studies looked specifi ally clinical scores found evidence benefi outcomes infants admitted hospitals 75 outpatients oxygen saturation admission hospital time resolution symptoms improve bronchodilator usage compared placebo outpatient studies assessing short-term change pooled clinical scores reviewers found small signifi nt diff erence mean score z=2\\u00b726 p=0\\u00b7024 small eff ect minimal clinical importance fi gure 2 outpatient studies heterogeneous i\\u00b2=81 p<0\\u00b700001 showing benefi scores tended include older children children recurrent wheezing \\n nebulised epinephrine assessed cochrane collaboration systematic review 76 review found benefi epinephrine compared placebo inpatients hospital length stay outcomes multicentre scandinavian study published cochrane review found inpatients receiving standing doses epinephrine longer length stay compared inpatients receiving as-needed epinephrine placebo 77 outpatients cochrane review found diff erence numbers admissions associated epinephrine treatment time emergency department visit overall course illness assessed 1 week clinical practice guidelines including usa uk canada recommend treatment bronchodilators bronchiolitis evidence table 38 \\n nebulised hypertonic saline thought reduce airway oedema decrease mucus plugging improve mucociliary clearance rehydrate airway surface liquid infants bronchiolitis 78 physiological changes extrapolated cystic fi brosis literature 79 80 pathophysiological processes acute bronchiolitis diff erent theoretical benefi ts hypertonic saline seen cystic fi brosis present infants acute viral bronchiolitis initial trials demonstrated ability hypertonic saline decrease hospital length stay transiently improve clinical severity score 81 83 recent trials demonstrated confl icting results 84 85 86 87 88 89 90 trials showed largest benefi hospitals lengths stay 72 h hypertonic saline infants countries institutions length stay approaches exceeds 72 h benefi cial reducing length stay confl icting results \\n test heterogeneity inpatient studies demonstrated low inconsistency studies i\\u00b2=36 p=0\\u00b713 summary eff ect signifi nt z=1\\u00b706 p=0\\u00b729 outpatient studies demonstrated high inconsistency 12 studies i\\u00b2=81 p<0\\u00b700001 summary eff ect signifi nt z=2\\u00b726 p=0\\u00b7024 overall heterogeneity meta-analysis demonstrated high inconsistency 21 studies i\\u00b2=73 p<0\\u00b700001 overall summary eff ect signifi nt z=2\\u00b74 p=0\\u00b7016 ipr ipratropium sal salbutamol neb nebulised reproduced gadomski scribani 75 refl ected diff erences recommendations national guidelines table countries recommending hypertonic saline recommending use inpatients recommending use moderate severe illness largest systematic review meta-analysis published 2015 examined 24 trials 3209 patients 91 infants receiving hypertonic saline signifi nt diff erence hospital length stay -0\\u00b745 days 95 ci -0\\u00b782 -0\\u00b708 p=0\\u00b701 compared receiving 0\\u00b79 saline standard care seven trials hypertonic saline reduced risk admission hospital 20 risk ratio 0\\u00b78 95 ci 0\\u00b767 0\\u00b796 compared 0\\u00b79 saline substantial adverse eff ects hypertonic saline noted systematic review results meta-analysis signifi nt attention paid outer limits confi dence intervals approach diff erence clinical relevance diff erences second 2015 meta-analysis 15 trials 1922 patients found smaller signifi nt decrease length stay -0\\u00b736 days 95 ci -0\\u00b75 -0\\u00b722 p<0\\u00b7001 received hypertonic saline fi gure 3 92 study found discrepancy overall combined eff ect studies length stay negative results largest trials allowing authors conclude individual trials pooled estimates provide strong evidence-based foundation use hypertonic saline meta-analyses showed substantial heterogeneity studies i\\u00b2=82\\u00b71 p<0\\u00b7001 91 i\\u00b2=77\\u00b78 p=0\\u00b7029 92 recent reanalysis fi rst 2015 meta-analysis removed outlying chinese studies accounted imbalances day illness presentation 93 analyses resolved heterogeneity found hypertonic saline reduce length stay infants admitted hospitals bronchiolitis mean diff erence length stay removing outliers -0\\u00b721 days 95 ci -0\\u00b743 0\\u00b702 mean diff erence length stay accounting day illness imbalance 0\\u00b702 days 95 ci -0\\u00b714 0\\u00b717 large trials demonstrated benefi hypertonic saline meta-analysis found eff ect hypertonic saline length stay adjustment outliers imbalances decision undertake future trials controversial given positive results metaanalyses negative results \\n multiple studies examined role corticosteroids management children bronchiolitis data large multicentre trials shown benefi corticosteroids reducing admissions hospital 94 95 2013 cochrane collaboration review supports results studies 96 review included studies enrolled children younger 24 months fi rst episode wheezing signs viral illness included outpatient studies 1824 participants comparing corticosteroids placebo reduction admission day 1 z=1\\u00b705 p=0\\u00b73 day 7 z=1\\u00b738 p=0\\u00b717 enrolment fi gure 4 clinical scores length stay emergency department length time resolution symptoms inpatient studies 772 participants length hospital stay reduced 96 basis evidence multiple clinical practice guidelines recommend use corticosteroids infants bronchiolitis table clinicians report considering family personal history atopy deciding treat infants bronchiolitis corticosteroids 41 evidence infants receive benefi corticosteroid treatment 94 95 96 evidence presence absence respiratory syncytial virus infection infants associated response corticosteroids unavailable 94 95 authors large study factorial design suggested unadjusted analysis high-dose corticosteroids combination nebulised epinephrine reduce admissions outpatients bronchiolitis day 7 95 results considered exploratory 39 45 high-fl ow oxygen respiratory support \\n non-invasive technologies improve oxygenation ventilation bronchiolitis include humidifi ed high-fl ow nasal cannula oxygen continuous positive airway pressure 97 high-fl ow nasal cannula allows delivery high fl ows usually 1 2 l/kg min humidifi cation cannula designed improve patient tolerance widely premature infants mechanisms action unclear particular deliver positive end-expiratory pressure conditions evidence effi cacy high-fl ow nasal cannula predominantly observational studies documenting corneli 2007 goebel 2000 kuyucu 2004 plint 2009 improved respiratory parameters reduced intubation rates implementation 97 small randomised trial compared high-fl ow nasal cannula hypertonic saline found diff erence change respiratory score 98 concerns high-fl ow nasal cannula include potential rapid deterioration infant closely monitored costs associated overuse continuous positive airway pressure studied intensive care settings observational studies small trials evidence improved respiratory parameters 97 uk guidelines recommend considering continuous positive airway pressure children impending respiratory failure bronchiolitis \\n antibiotic overuse children bronchiolitis probably occurs concerns presence fever young age aff ected patients diffi culty diff erentiating atelectasis infectious consolidation chest radiograph concern undetected secondary bacterial infection bronchiolitis clear viral cause occurrence secondary bacterial infections low risk bacteraemia meningitis 1 99 detailed review randomised clinical trials found routine use antibiotics improve duration symptoms length hospital stay need oxygen therapy hospital admission 38 overuse antibiotics known result unnecessary adverse eff ects patient development antimicrobial resistance routine use avoided clear evidence secondary bacterial infection table acute otitis media documented 60 infants bronchiolitis 100 101 antibiotic use acute otitis media bronchiolitis follow established evidence guidelines acute otitis media 102 macrolide antibiotics anti-infl ammatory properties potential benefi mitigating infl ammation present bronchiolitis randomised trials found diff erence azithromycin placebo hospital length stay need oxygen hospital re-admission 103 104 randomised trial found azithromycin lowered nasal interleukin-8 concentrations prolonged time subsequent wheezing episodes resulted fewer days respiratory symptoms year following bronchiolitis episode compared placebo 105 finally multicentre study found children aged 12 72 months history recurrent severe lower respiratory tract infections early administration azithromycin lower respiratory tract infection reduced likelihood progression severe infection clear underlying disease children bronchiolitis disease process 106 given current evidence routine use macrolides recommended bronchiolitis research needed clarify potential role future \\n hydration suctioning chest physiotherapy suggested supportive therapies infants bronchiolitis diffi culty feeding nasal congestion increased work breathing hydration remains cornerstone therapy multicentre study 759 infants younger 12 months admitted hospital bronchiolitis showed benefi intravenous fl uids compared administration fl uids nasogastric tube mean length stay admission intensive care unit need ventilatory support adverse events trial found nasogastric tube easier place intravenous line infants 107 guidelines recommend nasogastric intravenous fl uids maintain hydration uk scottish guidelines preferring nasogastric orogastric hydration tolerate compared intravenous hydration table intravenous fl uids isotonic fl uids preferred avoid risk hyponatraemia 38 infants obligate nasal breathers nasal suctioning suggested help clearing nares improve work breathing improve feeding suctioning irritate nasal mucosa result oedema randomised controlled trials examined role nasal suctioning bronchiolitis insuffi cient available evidence includes retrospective cohort study 740 infants 108 small observational study 40 infants 109 studies suggest deep suctioning increase length stay inpatients 108 infrequent suctioning associated increased length stay 108 oxygen saturation increase suctioning 109 draw conclusions causality observational studies diffi cult potential confouding indication exists eg sicker children likely receive deep suctioning evidence suggests oxygen saturation increases nasal irrigation suctioning 110 current guidelines diff ering recommendations regard suctioning support use recommend superfi cial suctioning deep suctioning table 38 39 45 chest physiotherapy use bronchiolitis appears vary country 111 112 recent cochrane collaboration review 12 studies 1249 participants demonstrated evidence benefi type chest physiotherapy inpatients length stay oxygen saturation respiratory parameters 113 published guidelines routinely recommend chest physiotherapy management uncomplicated bronchiolitis healthy children respiratory comorbidities table 38 39 45 prognosis work published risk developing recurrent wheezing asthma following bronchiolitis infancy 114 115 116 studies followed birth cohorts determine risk subsequent wheezing lower respiratory tract infection young childhood cohorts children admitted hospital bronchiolitis 114 115 116 117 overall admission hospital bronchiolitis young age associated increased risk recurrent wheezing studies report 17 60 children bronchiolitis develop recurrent wheezing years following initial admission hospital large study taiwan followed 1981 children admitted bronchiolitis age 3 years found age 10 years 351 17\\u00b77 1981 children bronchiolitis diagnosis asthma compared 2159 11\\u00b77 18 527 controls hazard ratio 1\\u00b758 95 ci 1\\u00b741 1\\u00b771 118 small cohort study 138 patients suggested 18 39 46 children admitted bronchiolitis 12 months asthma 18 years compared 9 92 controls 115 study followed birth cohort 1246 children found lower respiratory tract infection childhood associated increased risk recurrent wheezing association decreased age signifi nt age 13 years 116 children cohort mild illness requiring hospital admission severity illness associated increased risk asthma 119 question remains respiratory infection young age predisposes children asthma damage alteration lung function children severe bronchiolitis individual risk factors altered immune response airway function predisposes severe bronchiolitis recurrent wheezing 8 \\n substantial knowledge gaps controversies exist management acute bronchiolitis role nebulised hypertonic saline acute management clear resulting confl icting recommendations clinical guidelines table meta-analyses suggest small reduction length stay analyses limited heterogeneity accounting duration illness considering role outlying study populations 91 large multicentre trials support clinically signifi nt diff erence length stay inpatients clear evidence cost benefi t. 4 89 90 91 120 meta-analysis suggests possible reduction admissions outpatients confi dence intervals wide studies heterogeneous 91 multicentre studies nebulised hypertonic saline completed outpatients evidence increasing hypertonic saline little role meaningfully reducing length stay role outpatients clear large outpatient multicentre study clarify benefi t. \\n evidence suggests combined therapy nebulised epinephrine corticosteroids reduce admissions hospital 95 synergy corticosteroids \\u03b22 agonists documented clinical trials asthma management 121 122 123 124 basic science literature shows \\u03b22 agonists corticosteroids enhance eff ectiveness particularly regard antiinfl ammatory gene expression 125 126 economic burden bronchiolitis plausible basic clinical evidence synergy large multicentre trial needed ascertain combined therapy epinephrine corticosteroids benefi cial \\n oxygen saturation use pulse oximetry play important role decision admit infants bronchiolitis hospital length hospital stay 51 52 53 54 55 56 57 58 59 clinical practice guidelines confl icting guidance level oxygen saturation admission considered furthermore substantial proportion discharged infants episodes transient desaturation 54 view large health-care costs associated hospital admission bronchiolitis research needed clarify level oxygen saturation requiring admission role continuous spot measurements oxygen saturation clinical importance transient desaturations stable young infants \\n taf contributed design coordinated writing manuscript authors contributed literature search writing manuscript \\n declare competing interests taf acp jjz participated trials bronchiolitis funded unrestricted public academic institutional funding sources infl uence conception writing manuscript\",\n          \"march april 2009 health professionals warned mexican ministry health increase incidence acute respiratory illness young adults 1 april new viral subtype influenza a(h1n1)pdm09 combination swine avian human virus identified united states samples originated mexico 2 3 april 25 th patients identified sinan database following variables selecting cases controls pcr positive influenza a(h1n1)pdm2009 final classification confirmed criteria laboratory evolution recovered death hospitalization yes date hospitalization 28/06/2009 29/08/2009 municipalities residence hospitals metropolitan areas sao paulo campinas \\n time patients selected study sinan included 193 reported confirmed deaths patients influenza a(h1n1)pdm2009 sari assessed study cases case death control patients selected 1988 patients hospitalized study period reported confirmed influenza a(h1n1)pdm2009 sari recovered controls time-paired randomly selected hospitalized epidemiological week ew necessary week later date hospitalization cases prevent possible differences opportunity treatment clinical management protocol \\n data study collected trained health professionals standardized questionnaires hospital questionnaire collected information medical charts evaluated 177 hospitals patients hospitalized 56.5 belonged unified health system-sus 43.5 private hospitals following data collected previous pathological history occurrence pregnancy health care symptomatology admission laboratory radiological tests need icu admission treatment complications disease progression international classification diseases 10 th revision-icd-10 coding comorbidities indicated medical charts ministry health provided antiviral drug known oseltamivir treatment pandemic protocol clinical management established treatment cases hospitalized sari ambulatory patients risk conditions severity treatment started 48hours symptom onset patients studied laboratory confirmations influenza a(h1n1)pdm09 respiratory secretion samples analyzed rt-pcr method performed adolfo lutz institute 16 definition sari included fever associated cough dyspnea pneumonia respiratory insufficiency tachypnea radiological changes compatible pneumonia oxygen therapy mechanical ventilation sari confirmed cases controls selected study data medical charts \\n household interviews conducted 161 cases 83.4 14 refusals 7.3 18 9.3 addresses found total 337 85.1 controls interviewed 6 1.5 refusals 43 11.1 addresses found household questionnaire included socio-demographic data history previous care related episode led hospitalization presence influenza vaccination occupations grouped according occupational pyramid influenza risk 17 cases interviews conducted closest family members controls patients excluding children parents/legal guardians responded \\n data collection questionnaires assessed supervisors verify consistency information fulfillment criteria include patients study hospital household questionnaires input sphynx database version 5 \\n present study began order assist decisions surveillance control actions use data approved research ethics committee school public health university s\\u00e3o paulo-fsp-usp protocol 2283 of.coep/312/11 data collection performed accordance recommendations national health council human research including signing informed consent form patients family members legal guardians \\n clinical demographic characteristics shown median inter-quartile intervals percentages comparisons mann-whitney u chi-square tests required odds ratio-or 95 confidence intervals-95%ci calculated assess factors associated death variables p-value 0.20 univariate analysis adjustment variables selected multiple logistic regression forward stepwise strategy model including patients assessed following dichotomous categorical variables health insurance previous care vaccination influenza 2009 sex reference category male following risk conditions diabetes mellitus obesity chronic kidney disease chronic liver disease immunosuppression chronic cardiovascular disease chronic neurological disease delay development absence condition reference category model assessed age group 18 years reference 18 59 years 60 years use antiviral therapy use reference category 48 hours onset symptoms 48 72 hours 72 hours high-risk individuals risk conditions listed adapted centers disease control prevention-cdc 18 condition reference category condition conditions time onset symptoms admission days continuous variable model controlled epidemiologic week admission hospital categorized ew26 28 reference ew29 31 ew31 34 income second model developed patients aged 18 years assessed level education formal education incomplete primary education complete primary education complete secondary education complete higher education reference category occupational risk high high average low reference category patients aged 18 22 years incomplete higher education included reference category education modelling patients occupation included reference category occupational risk significance variables assessed wald test adjustment models hosmer-lemeshow test \\n analyses performed spss software version 17.0 p-values 0.05 considered significant table 1 presents socioeconomic characteristics patients information associated disease obtained household interviews patients aged 18 years higher risk death formal education incomplete primary education 2.33(95 ci 1 13 4.89 complete primary education 3.06 95%ci 1.41 6.82 compared higher education distribution ethnic groups family income history smoking associated death history previous vaccination seasonal influenza 2009 showed protection death health professionals considered high risk developing influenza showed protection death 0.15(95%ci 0.02 0.61 patients sought care prior hospitalization risk death 3.70 times 95%ci 2.25 6.30 patients previous care predominance emergency care 45.34 table 2 presents distribution cases controls according variables assessed hospital investigation median age cases higher controls p<0.001 18-to-59-year age group death 4.03(95%ci 2.62 6.31 compared aged 18 years elderly individuals significant risk death \\n differences sex distribution group regard risk conditions severity 73.6 cases 38.1 controls reported risk factor 4.49 95%ci 3.09 6.60 individuals conditions assessed risk death increased 10.29 95%ci 6.04 17.85 following medical conditions risk factors death obesity diabetes mellitus chronic cardiovascular diseases chronic kidney diseases neurological diseases delay development chronic liver diseases immunosuppressive diseases immunosuppression-related diseases neoplasms n 18 hiv/aids n 8) organ transplants n 6 autoimmune diseases n 5 use immunosuppressants n 3 \\n 212 women reproductive age 10 49 years studied 33.5 pregnant differences frequency pregnancy group \\n use antiviral treatment hospitalization protective factor death 0.67 95%ci 0.45 0.98 time onset symptoms beginning antiviral treatment assessed 81 protection individuals received treatment 48 hours 0.19 95%ci 0.11 0.35 65 48 72 hours 0.35 95%ci 0.16 0.74 72 hours significant protection median days date onset symptoms date hospitalization 5.0 cases 2.0 controls p<0.001 patients chronic pulmonary disease median 4.0 days cases 2.0 days controls asthma 4.5 days 2.0 days pregnant women 4.0 days 2.0 days respectively regard health professionals median date onset symptoms hospitalization days cases controls median date hospitalization death hospital discharge seven days cases days controls table 2 \\n according medical charts fever cough occurred 87.8 cases 79.3 controls proportions dyspnea 87 62.7 respectively definition cases severe acute respiratory syndrome -sars fever+cough+dyspnea based information obtained medical charts occurred 68.9 cases 56.5 controls vomit diarrhea present 8 cases 18 controls treatment antibiotics 95.8 cases 73.1 controls mean number antibiotics 4.0 1.8 respectively cases higher proportion hospitalizations intensive care units 83.4%x16.8 use mechanical ventilation 94.8%x10.8 dialysis 25.0%x2.3 proportion complications higher cases controls emphasis respiratory distress syndrome 64.7%vs4.9 shock 69.4%vs1.0 sepsis 63.2%vs2.3 kidney complications 44.6%vs1.3 respectively table 3 shows laboratory findings entry tests patients hospital cases lower numbers leukocytes platelets higher values creatine phosphokinase-cpk lactic dehydrogenase-ldh glutamic-oxaloacetic transaminase glutamic-pyruvic transaminase urea creatinine controls statistical significance \\n sub-group analysis previous conditions risk severity obesity risk factors death univariate analysis children younger 18 years women reproductive age 10 49 years children presence immunosuppression chronic cardiovascular disease neurological disease/delay development risk factors diabetes mellitus women treatment antiviral administered 72 hours onset symptoms protective factor women reproductive age tables 4 5 variables associated death final models multiple logistic regression patients studied sub-group analysis adults aged 18 years patients general risk factors death aged 18 59 years reference 18 years risk conditions severity disease obesity immunosuppression sought care prior hospitalization chronic neurological diseases/delay development chronic liver disease highly associated death variables excluded final model estimates large confidence intervals private health insurance antiviral treatment administered 72 hours onset symptoms protective factors death adults aged 18 years risk factors death risk factor severity disease obesity care prior hospitalization protective factors antiviral drug administered 48 hours onset symptoms health professional health unit worker \\n sensitivity analysis performed assess severity patients admitted study period proportion icu admissions use mechanical ventilation estimated epidemiological week hospitalization categorized ew 26 28 ew 29 31 ew 32 34 results showed higher proportions icu admissions 50.0 x 16.8 x 7.7 mechanical ventilation 45.4 x 10.8 x 2.6 weeks controls cases showed high proportions periods analyzed 89.7 x 84.0 x 76.9 need icu 89.7 x 97.6 x 89.7 need mechanical ventilation 3 periods respectively opportunity antiviral treatment showed lower proportion treatment period study cases 37.9 x 74.4 x 74.3 controls 59.1 x 74.1 x 89.7 sensitivity analysis assess opportunity treatment children adults showed similar proportions groups 73.9 74 respectively \\n present study risk factors death analyzed patients infected influenza (h1n1)pdm09 hospitalized sari increased risk death patients aged 18 59 years obesity immunosuppression neurological developmental diseases received care prior hospitalization additionally presence risk factor aggravation disease associated occurrence death use antiviral drugs especially administered 72 hours onset symptoms protective factor death \\n design case-control study including deaths reported confirmed region study period time-paired random selection controls patients survived hospitalization sari hospital household data collection enabled analysis factors associated death extended including socioeconomic variables access health services data hospitalizations \\n studies assessed risk factors death patients sari 15 19 studies analyzed risk factors severity disease deaths comparing patients hospitalized icus wards 13 14 20 21 22 23 study rate mortality influenza a(h1n1)pdm09 epidemic waves 24 study conducted australia analyzed risk factors hospitalization influenza comparing cases controls infection community 25 studies analyzed epidemiological surveillance data including mortality rates comparisons deaths hospitalizations treatment factors 9 10 12 26 27 28 29 30 31 32 33 34 35 factors associated death identified study described authors presence comorbidities 31 35 age18years 13 obesity 15 19 20 neurological developmental diseases 23 24 31 immunosuppression 15 21 24 34 children showed lower risk death despite similar proportion treatment adults similar result observed study systematic review metaanalysis risk death 0.28 95 ci 0.19 0.141 compared young adults 36 contrast factors associated severity infection reported different researchers diabetes 19 34 chronic pulmonary diseases excluding asthma chronic obstructive pulmonary disease 20 chronic pulmonary disease including asthma 31 chronic cardiovascular disease 21 24 34 associated mortality present study population assessed restricted hospitalized patients severe influenza partly explain differences diabetes chronic cardiovascular kidney disease associated death univariate analysis remain risk factors multiple analysis design study data collected medical charts household interviews allowed adjustments analysis explain differences \\n deaths occurred mainly adults aged 18 59 years small proportion cases controls elderly individuals possible preexisting immunity new influenza a(h1n1)pdm09 virus age group suggested probably previous exposure infections vaccinations influenza a(h1n1 virus genetically antigenically associated new virus contemporary strains seasonal h1n1 viruses 37 \\n patients present study cases controls laboratory-confirmed infection influenza a/h1n1pdm09 38 \\n possible probability hospitalized varied study treatment available hospitalized patients ambulatory patients risk conditions severity influenced hospitalization study population consisting patients sari pandemic criteria hospitalization changed availability treatment especially 3 weeks epidemic shown sensitivity analysis confirming need controlling period hospitalization study \\n socioeconomic profile similar groups assessed difference income private health insurance greater protection death patients occupation higher risk exposure infection health professionals greater opportunity access treatment workers explain findings period date onset symptoms date admission shorter group low proportion deaths patients categorized higher risk exposure observed studies 30 33 \\n pregnancy risk factor death study contrast frequency pregnant women reproductive age hospitalized severe manifestation influenza (h1n1)pdm09 high suggesting pregnancy associated development sari association pregnancy use mechanical ventilation hospitalization influenza a(h1n1)pdm2009 25 higher admission rate reported compared general population 32 2009 state s\\u00e3o paulo 27.4 confirmed cases influenza a(h1n1)pdm09 women reproductive age 10 49 years hospitalized recorded sinan database found pregnant women brazil 4.27 women reproductive age live newborns reference year according 2010 census 39 \\n patients died presented important laboratory changes compared controls lower levels leukocytes platelets increase cpk ldh urea creatinine studies revealed similar trend laboratory results leukocytes 23 cpk 26 27 30 ldh 26 platelets 22 creatinine 27 30 34 \\n patients died received antiviral treatment 9.3 treated 48 hours early treatment protective factor death especially provided 72 hours onset symptoms studies shown favorable prognostic antiviral treatment 13 21 23 27 28 29 34 \\n history care prior hospitalization risk factor death influenza a(h1n1 pdm09 reflect difficulty accessing health services emergency services primarily sought care prior hospitalization contributed greater delay diagnosis beginning treatment study conducted mexico showed similar result regard delay hospitalization 29 \\n study assessed overall standardized mortality rate influenza (h1n1)pdm09 mortality patients preexisting chronic respiratory diseases lower chronic diseases 24 problems recording information previous diseases medical charts presence acute pulmonary disease especially chronic pulmonary disease \\n present study limitations underreporting sari health professionals likely occurred reporting disease compulsory brazil pandemic pandemic professionals sensitive reporting system notification mandatory epidemic facilitated allow rapid notifications availability samples autopsies cases deaths resulted greater access laboratory diagnosis infection household data collection caused information bias data cases collected interviews family members data controls obtained patients adults proportion refusals household interviews higher cases 7.3 controls 1.6 p 0.001 possible limitation memory bias result time hospitalization interview information analyzed household interviews socio-demographic characteristics vulnerable information memory bias data preexisting diseases obtained medical charts hospitals completed moment hospitalization presence patients quality completion medical charts differ hospitals nonetheless information presence chronic diseases clinical management patients laboratory results radiological tests treatment available charts difficulties encountered data collection minimized use standardized questionnaire \\n conclusion risk conditions severity disease particularly obesity immunosuppression neurological/developmental diseases important risk factors death emphasizing need good vaccination coverage influenza vaccination campaigns patients chronic diseases belonging vulnerable groups relevant result antiviral treatment 72 hours onset symptoms protective factor emphasizing need warn population importance seeking early medical care influenza-like illness appears especially high-risk groups signs disease aggravation training health professionals especially physicians required adequate clinical management patients early antiviral treatment\"\n        ],\n        \"semantic_type\": \"\",\n        \"description\": \"\"\n      }\n    },\n    {\n      \"column\": \"cluster\",\n      \"properties\": {\n        \"dtype\": \"int32\",\n        \"num_unique_values\": 28,\n        \"samples\": [\n          24,\n          19,\n          6\n        ],\n        \"semantic_type\": \"\",\n        \"description\": \"\"\n      }\n    },\n    {\n      \"column\": \"hierarchical_cluster\",\n      \"properties\": {\n        \"dtype\": \"number\",\n        \"std\": 0,\n        \"min\": 0,\n        \"max\": 1,\n        \"num_unique_values\": 2,\n        \"samples\": [\n          1,\n          0\n        ],\n        \"semantic_type\": \"\",\n        \"description\": \"\"\n      }\n    }\n  ]\n}",
              "type": "dataframe",
              "variable_name": "df"
            },
            "text/html": [
              "\n",
              "  <div id=\"df-ffa395a7-2340-422a-91ee-95ab8c420d11\" class=\"colab-df-container\">\n",
              "    <div>\n",
              "<style scoped>\n",
              "    .dataframe tbody tr th:only-of-type {\n",
              "        vertical-align: middle;\n",
              "    }\n",
              "\n",
              "    .dataframe tbody tr th {\n",
              "        vertical-align: top;\n",
              "    }\n",
              "\n",
              "    .dataframe thead th {\n",
              "        text-align: right;\n",
              "    }\n",
              "</style>\n",
              "<table border=\"1\" class=\"dataframe\">\n",
              "  <thead>\n",
              "    <tr style=\"text-align: right;\">\n",
              "      <th></th>\n",
              "      <th>paper_id</th>\n",
              "      <th>doi</th>\n",
              "      <th>abstract</th>\n",
              "      <th>body_text</th>\n",
              "      <th>authors</th>\n",
              "      <th>title</th>\n",
              "      <th>journal</th>\n",
              "      <th>abstract_summary</th>\n",
              "      <th>cleaned_text</th>\n",
              "      <th>cleaned_text_str</th>\n",
              "      <th>cluster</th>\n",
              "      <th>hierarchical_cluster</th>\n",
              "    </tr>\n",
              "  </thead>\n",
              "  <tbody>\n",
              "    <tr>\n",
              "      <th>6252</th>\n",
              "      <td>fefd6cdcddcb00d1ba2ef1a0d3bba9f312010a1d</td>\n",
              "      <td>10.1111/j.1524-4733.2008.00379.x</td>\n",
              "      <td></td>\n",
              "      <td>China is the biggest country in the world that...</td>\n",
              "      <td>Li, Youing.  Sun, Xin.  Wang, Li</td>\n",
              "      <td>Evidence-Based Medicine in China</td>\n",
              "      <td>Value in Health</td>\n",
              "      <td>Not provided.</td>\n",
              "      <td>[china, biggest, country, world, 1.3, billion,...</td>\n",
              "      <td>china biggest country world 1.3 billion popula...</td>\n",
              "      <td>0</td>\n",
              "      <td>0</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>4684</th>\n",
              "      <td>2a3d080d5dcf6771db002366691f35e5d6f0e493</td>\n",
              "      <td>10.1007/s13238-010-0029-7</td>\n",
              "      <td></td>\n",
              "      <td>✉ Correspondence: zlshi@wh.iov.cn Bat, the onl...</td>\n",
              "      <td>Shi, Zhengli</td>\n",
              "      <td>Bat and virus</td>\n",
              "      <td>Protein &amp; Cell</td>\n",
              "      <td>Not provided.</td>\n",
              "      <td>[✉, correspondence, zlshi@wh.iov.cn, bat, flyi...</td>\n",
              "      <td>✉ correspondence zlshi@wh.iov.cn bat flying ma...</td>\n",
              "      <td>18</td>\n",
              "      <td>0</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>1731</th>\n",
              "      <td>2de657aabab331e75657bfc1443542fd345b6f26</td>\n",
              "      <td>10.1016/j.ccc.2019.06.001</td>\n",
              "      <td>KEYWORDS\\nPandemic Influenza Disaster prepared...</td>\n",
              "      <td>Outbreaks of infectious disease pose unique ch...</td>\n",
              "      <td>Maves, Ryan C..  Jamros, Christina M..  Smith...</td>\n",
              "      <td>Intensive Care Unit Preparedness During&lt;br&gt;Pa...</td>\n",
              "      <td>Crit Care Clin</td>\n",
              "      <td>KEYWORDS\\nPandemic Influenza Disaster&lt;br&gt;prep...</td>\n",
              "      <td>[outbreaks, infectious, disease, pose, unique,...</td>\n",
              "      <td>outbreaks infectious disease pose unique chall...</td>\n",
              "      <td>13</td>\n",
              "      <td>0</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>4742</th>\n",
              "      <td>bc22758234283f024f16e2dbe660bc909b512b92</td>\n",
              "      <td>10.1016/j.vetpar.2007.11.020</td>\n",
              "      <td>Cryptosporidium parvum is recognized as one of...</td>\n",
              "      <td>Diarrhoeal diseases of infectious aethiology r...</td>\n",
              "      <td>Klein, Pavel.  Kleinová, Tereza.  Volek,&lt;br&gt;Z...</td>\n",
              "      <td>Effect of Cryptosporidium parvum infection on...</td>\n",
              "      <td>Veterinary Parasitology</td>\n",
              "      <td>Cryptosporidium parvum is recognized as one o...</td>\n",
              "      <td>[diarrhoeal, diseases, infectious, aethiology,...</td>\n",
              "      <td>diarrhoeal diseases infectious aethiology repr...</td>\n",
              "      <td>12</td>\n",
              "      <td>0</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>4521</th>\n",
              "      <td>a204aafa38365dbcc0a26af3ca2c6d3313d7fab2</td>\n",
              "      <td>10.1111/irv.12518</td>\n",
              "      <td>, Brazil, in the period of Background: The res...</td>\n",
              "      <td>The inclusion criteria for asymptomatic adults...</td>\n",
              "      <td>Moreira, Luciana Peniche.  Watanabe, Aripuana...</td>\n",
              "      <td>Respiratory syncytial virus evaluation among&lt;...</td>\n",
              "      <td>Influenza Other Respir Viruses</td>\n",
              "      <td>, Brazil, in the period of Background: The&lt;br...</td>\n",
              "      <td>[inclusion, criteria, asymptomatic, adults, ab...</td>\n",
              "      <td>inclusion criteria asymptomatic adults absence...</td>\n",
              "      <td>15</td>\n",
              "      <td>0</td>\n",
              "    </tr>\n",
              "  </tbody>\n",
              "</table>\n",
              "</div>\n",
              "    <div class=\"colab-df-buttons\">\n",
              "\n",
              "  <div class=\"colab-df-container\">\n",
              "    <button class=\"colab-df-convert\" onclick=\"convertToInteractive('df-ffa395a7-2340-422a-91ee-95ab8c420d11')\"\n",
              "            title=\"Convert this dataframe to an interactive table.\"\n",
              "            style=\"display:none;\">\n",
              "\n",
              "  <svg xmlns=\"http://www.w3.org/2000/svg\" height=\"24px\" viewBox=\"0 -960 960 960\">\n",
              "    <path d=\"M120-120v-720h720v720H120Zm60-500h600v-160H180v160Zm220 220h160v-160H400v160Zm0 220h160v-160H400v160ZM180-400h160v-160H180v160Zm440 0h160v-160H620v160ZM180-180h160v-160H180v160Zm440 0h160v-160H620v160Z\"/>\n",
              "  </svg>\n",
              "    </button>\n",
              "\n",
              "  <style>\n",
              "    .colab-df-container {\n",
              "      display:flex;\n",
              "      gap: 12px;\n",
              "    }\n",
              "\n",
              "    .colab-df-convert {\n",
              "      background-color: #E8F0FE;\n",
              "      border: none;\n",
              "      border-radius: 50%;\n",
              "      cursor: pointer;\n",
              "      display: none;\n",
              "      fill: #1967D2;\n",
              "      height: 32px;\n",
              "      padding: 0 0 0 0;\n",
              "      width: 32px;\n",
              "    }\n",
              "\n",
              "    .colab-df-convert:hover {\n",
              "      background-color: #E2EBFA;\n",
              "      box-shadow: 0px 1px 2px rgba(60, 64, 67, 0.3), 0px 1px 3px 1px rgba(60, 64, 67, 0.15);\n",
              "      fill: #174EA6;\n",
              "    }\n",
              "\n",
              "    .colab-df-buttons div {\n",
              "      margin-bottom: 4px;\n",
              "    }\n",
              "\n",
              "    [theme=dark] .colab-df-convert {\n",
              "      background-color: #3B4455;\n",
              "      fill: #D2E3FC;\n",
              "    }\n",
              "\n",
              "    [theme=dark] .colab-df-convert:hover {\n",
              "      background-color: #434B5C;\n",
              "      box-shadow: 0px 1px 3px 1px rgba(0, 0, 0, 0.15);\n",
              "      filter: drop-shadow(0px 1px 2px rgba(0, 0, 0, 0.3));\n",
              "      fill: #FFFFFF;\n",
              "    }\n",
              "  </style>\n",
              "\n",
              "    <script>\n",
              "      const buttonEl =\n",
              "        document.querySelector('#df-ffa395a7-2340-422a-91ee-95ab8c420d11 button.colab-df-convert');\n",
              "      buttonEl.style.display =\n",
              "        google.colab.kernel.accessAllowed ? 'block' : 'none';\n",
              "\n",
              "      async function convertToInteractive(key) {\n",
              "        const element = document.querySelector('#df-ffa395a7-2340-422a-91ee-95ab8c420d11');\n",
              "        const dataTable =\n",
              "          await google.colab.kernel.invokeFunction('convertToInteractive',\n",
              "                                                    [key], {});\n",
              "        if (!dataTable) return;\n",
              "\n",
              "        const docLinkHtml = 'Like what you see? Visit the ' +\n",
              "          '<a target=\"_blank\" href=https://colab.research.google.com/notebooks/data_table.ipynb>data table notebook</a>'\n",
              "          + ' to learn more about interactive tables.';\n",
              "        element.innerHTML = '';\n",
              "        dataTable['output_type'] = 'display_data';\n",
              "        await google.colab.output.renderOutput(dataTable, element);\n",
              "        const docLink = document.createElement('div');\n",
              "        docLink.innerHTML = docLinkHtml;\n",
              "        element.appendChild(docLink);\n",
              "      }\n",
              "    </script>\n",
              "  </div>\n",
              "\n",
              "\n",
              "<div id=\"df-57f8a1e5-1fb1-4d5d-a12d-8e0be8030aad\">\n",
              "  <button class=\"colab-df-quickchart\" onclick=\"quickchart('df-57f8a1e5-1fb1-4d5d-a12d-8e0be8030aad')\"\n",
              "            title=\"Suggest charts\"\n",
              "            style=\"display:none;\">\n",
              "\n",
              "<svg xmlns=\"http://www.w3.org/2000/svg\" height=\"24px\"viewBox=\"0 0 24 24\"\n",
              "     width=\"24px\">\n",
              "    <g>\n",
              "        <path d=\"M19 3H5c-1.1 0-2 .9-2 2v14c0 1.1.9 2 2 2h14c1.1 0 2-.9 2-2V5c0-1.1-.9-2-2-2zM9 17H7v-7h2v7zm4 0h-2V7h2v10zm4 0h-2v-4h2v4z\"/>\n",
              "    </g>\n",
              "</svg>\n",
              "  </button>\n",
              "\n",
              "<style>\n",
              "  .colab-df-quickchart {\n",
              "      --bg-color: #E8F0FE;\n",
              "      --fill-color: #1967D2;\n",
              "      --hover-bg-color: #E2EBFA;\n",
              "      --hover-fill-color: #174EA6;\n",
              "      --disabled-fill-color: #AAA;\n",
              "      --disabled-bg-color: #DDD;\n",
              "  }\n",
              "\n",
              "  [theme=dark] .colab-df-quickchart {\n",
              "      --bg-color: #3B4455;\n",
              "      --fill-color: #D2E3FC;\n",
              "      --hover-bg-color: #434B5C;\n",
              "      --hover-fill-color: #FFFFFF;\n",
              "      --disabled-bg-color: #3B4455;\n",
              "      --disabled-fill-color: #666;\n",
              "  }\n",
              "\n",
              "  .colab-df-quickchart {\n",
              "    background-color: var(--bg-color);\n",
              "    border: none;\n",
              "    border-radius: 50%;\n",
              "    cursor: pointer;\n",
              "    display: none;\n",
              "    fill: var(--fill-color);\n",
              "    height: 32px;\n",
              "    padding: 0;\n",
              "    width: 32px;\n",
              "  }\n",
              "\n",
              "  .colab-df-quickchart:hover {\n",
              "    background-color: var(--hover-bg-color);\n",
              "    box-shadow: 0 1px 2px rgba(60, 64, 67, 0.3), 0 1px 3px 1px rgba(60, 64, 67, 0.15);\n",
              "    fill: var(--button-hover-fill-color);\n",
              "  }\n",
              "\n",
              "  .colab-df-quickchart-complete:disabled,\n",
              "  .colab-df-quickchart-complete:disabled:hover {\n",
              "    background-color: var(--disabled-bg-color);\n",
              "    fill: var(--disabled-fill-color);\n",
              "    box-shadow: none;\n",
              "  }\n",
              "\n",
              "  .colab-df-spinner {\n",
              "    border: 2px solid var(--fill-color);\n",
              "    border-color: transparent;\n",
              "    border-bottom-color: var(--fill-color);\n",
              "    animation:\n",
              "      spin 1s steps(1) infinite;\n",
              "  }\n",
              "\n",
              "  @keyframes spin {\n",
              "    0% {\n",
              "      border-color: transparent;\n",
              "      border-bottom-color: var(--fill-color);\n",
              "      border-left-color: var(--fill-color);\n",
              "    }\n",
              "    20% {\n",
              "      border-color: transparent;\n",
              "      border-left-color: var(--fill-color);\n",
              "      border-top-color: var(--fill-color);\n",
              "    }\n",
              "    30% {\n",
              "      border-color: transparent;\n",
              "      border-left-color: var(--fill-color);\n",
              "      border-top-color: var(--fill-color);\n",
              "      border-right-color: var(--fill-color);\n",
              "    }\n",
              "    40% {\n",
              "      border-color: transparent;\n",
              "      border-right-color: var(--fill-color);\n",
              "      border-top-color: var(--fill-color);\n",
              "    }\n",
              "    60% {\n",
              "      border-color: transparent;\n",
              "      border-right-color: var(--fill-color);\n",
              "    }\n",
              "    80% {\n",
              "      border-color: transparent;\n",
              "      border-right-color: var(--fill-color);\n",
              "      border-bottom-color: var(--fill-color);\n",
              "    }\n",
              "    90% {\n",
              "      border-color: transparent;\n",
              "      border-bottom-color: var(--fill-color);\n",
              "    }\n",
              "  }\n",
              "</style>\n",
              "\n",
              "  <script>\n",
              "    async function quickchart(key) {\n",
              "      const quickchartButtonEl =\n",
              "        document.querySelector('#' + key + ' button');\n",
              "      quickchartButtonEl.disabled = true;  // To prevent multiple clicks.\n",
              "      quickchartButtonEl.classList.add('colab-df-spinner');\n",
              "      try {\n",
              "        const charts = await google.colab.kernel.invokeFunction(\n",
              "            'suggestCharts', [key], {});\n",
              "      } catch (error) {\n",
              "        console.error('Error during call to suggestCharts:', error);\n",
              "      }\n",
              "      quickchartButtonEl.classList.remove('colab-df-spinner');\n",
              "      quickchartButtonEl.classList.add('colab-df-quickchart-complete');\n",
              "    }\n",
              "    (() => {\n",
              "      let quickchartButtonEl =\n",
              "        document.querySelector('#df-57f8a1e5-1fb1-4d5d-a12d-8e0be8030aad button');\n",
              "      quickchartButtonEl.style.display =\n",
              "        google.colab.kernel.accessAllowed ? 'block' : 'none';\n",
              "    })();\n",
              "  </script>\n",
              "</div>\n",
              "\n",
              "    </div>\n",
              "  </div>\n"
            ],
            "text/plain": [
              "                                      paper_id  \\\n",
              "6252  fefd6cdcddcb00d1ba2ef1a0d3bba9f312010a1d   \n",
              "4684  2a3d080d5dcf6771db002366691f35e5d6f0e493   \n",
              "1731  2de657aabab331e75657bfc1443542fd345b6f26   \n",
              "4742  bc22758234283f024f16e2dbe660bc909b512b92   \n",
              "4521  a204aafa38365dbcc0a26af3ca2c6d3313d7fab2   \n",
              "\n",
              "                                   doi  \\\n",
              "6252  10.1111/j.1524-4733.2008.00379.x   \n",
              "4684         10.1007/s13238-010-0029-7   \n",
              "1731         10.1016/j.ccc.2019.06.001   \n",
              "4742      10.1016/j.vetpar.2007.11.020   \n",
              "4521                 10.1111/irv.12518   \n",
              "\n",
              "                                               abstract  \\\n",
              "6252                                                      \n",
              "4684                                                      \n",
              "1731  KEYWORDS\\nPandemic Influenza Disaster prepared...   \n",
              "4742  Cryptosporidium parvum is recognized as one of...   \n",
              "4521  , Brazil, in the period of Background: The res...   \n",
              "\n",
              "                                              body_text  \\\n",
              "6252  China is the biggest country in the world that...   \n",
              "4684  ✉ Correspondence: zlshi@wh.iov.cn Bat, the onl...   \n",
              "1731  Outbreaks of infectious disease pose unique ch...   \n",
              "4742  Diarrhoeal diseases of infectious aethiology r...   \n",
              "4521  The inclusion criteria for asymptomatic adults...   \n",
              "\n",
              "                                                authors  \\\n",
              "6252                   Li, Youing.  Sun, Xin.  Wang, Li   \n",
              "4684                                       Shi, Zhengli   \n",
              "1731   Maves, Ryan C..  Jamros, Christina M..  Smith...   \n",
              "4742   Klein, Pavel.  Kleinová, Tereza.  Volek,<br>Z...   \n",
              "4521   Moreira, Luciana Peniche.  Watanabe, Aripuana...   \n",
              "\n",
              "                                                  title  \\\n",
              "6252                   Evidence-Based Medicine in China   \n",
              "4684                                      Bat and virus   \n",
              "1731   Intensive Care Unit Preparedness During<br>Pa...   \n",
              "4742   Effect of Cryptosporidium parvum infection on...   \n",
              "4521   Respiratory syncytial virus evaluation among<...   \n",
              "\n",
              "                             journal  \\\n",
              "6252                 Value in Health   \n",
              "4684                  Protein & Cell   \n",
              "1731                  Crit Care Clin   \n",
              "4742         Veterinary Parasitology   \n",
              "4521  Influenza Other Respir Viruses   \n",
              "\n",
              "                                       abstract_summary  \\\n",
              "6252                                      Not provided.   \n",
              "4684                                      Not provided.   \n",
              "1731   KEYWORDS\\nPandemic Influenza Disaster<br>prep...   \n",
              "4742   Cryptosporidium parvum is recognized as one o...   \n",
              "4521   , Brazil, in the period of Background: The<br...   \n",
              "\n",
              "                                           cleaned_text  \\\n",
              "6252  [china, biggest, country, world, 1.3, billion,...   \n",
              "4684  [✉, correspondence, zlshi@wh.iov.cn, bat, flyi...   \n",
              "1731  [outbreaks, infectious, disease, pose, unique,...   \n",
              "4742  [diarrhoeal, diseases, infectious, aethiology,...   \n",
              "4521  [inclusion, criteria, asymptomatic, adults, ab...   \n",
              "\n",
              "                                       cleaned_text_str  cluster  \\\n",
              "6252  china biggest country world 1.3 billion popula...        0   \n",
              "4684  ✉ correspondence zlshi@wh.iov.cn bat flying ma...       18   \n",
              "1731  outbreaks infectious disease pose unique chall...       13   \n",
              "4742  diarrhoeal diseases infectious aethiology repr...       12   \n",
              "4521  inclusion criteria asymptomatic adults absence...       15   \n",
              "\n",
              "      hierarchical_cluster  \n",
              "6252                     0  \n",
              "4684                     0  \n",
              "1731                     0  \n",
              "4742                     0  \n",
              "4521                     0  "
            ]
          },
          "execution_count": 81,
          "metadata": {},
          "output_type": "execute_result"
        }
      ],
      "source": [
        "from sklearn.cluster import AgglomerativeClustering\n",
        "\n",
        "# Fit the Hierarchical Clustering model\n",
        "hierarchical = AgglomerativeClustering(n_clusters=optimal_k)\n",
        "df['hierarchical_cluster'] = hierarchical.fit_predict(tfidf_pca)\n",
        "\n",
        "# Display the first few rows of the DataFrame with hierarchical cluster labels\n",
        "df.head()"
      ]
    },
    {
      "cell_type": "code",
      "execution_count": 71,
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "id": "Gt42-b9-uEzW",
        "outputId": "6cf90f13-d451-461e-8476-83ce9ea59873"
      },
      "outputs": [
        {
          "name": "stdout",
          "output_type": "stream",
          "text": [
            "K-Means Silhouette Score: 0.03723554387616387\n",
            "Hierarchical Silhouette Score: 0.012668738798288552\n"
          ]
        }
      ],
      "source": [
        "from sklearn.metrics import silhouette_score\n",
        "\n",
        "# Evaluate the K-Means clustering\n",
        "kmeans_silhouette = silhouette_score(tfidf_pca, df['cluster'])\n",
        "print(f'K-Means Silhouette Score: {kmeans_silhouette}')\n",
        "\n",
        "# Evaluate the Hierarchical clustering\n",
        "hierarchical_silhouette = silhouette_score(tfidf_pca, df['hierarchical_cluster'])\n",
        "print(f'Hierarchical Silhouette Score: {hierarchical_silhouette}')"
      ]
    },
    {
      "cell_type": "code",
      "execution_count": 58,
      "metadata": {
        "id": "itdhgYIyiP95"
      },
      "outputs": [],
      "source": [
        "# Reduce dimensionality using t-SNE\n",
        "tsne = TSNE(n_components=2, random_state=42)\n",
        "X_tsne = tsne.fit_transform(tfidf_matrix.toarray())"
      ]
    },
    {
      "cell_type": "code",
      "execution_count": 83,
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/",
          "height": 642
        },
        "id": "gzcOHYXHfLvv",
        "outputId": "596dd8e7-57bb-40e7-eb08-451ff6776aad"
      },
      "outputs": [
        {
          "data": {
            "image/svg+xml": [
              "<?xml version=\"1.0\" encoding=\"utf-8\" standalone=\"no\"?>\n",
              "<!DOCTYPE svg PUBLIC \"-//W3C//DTD SVG 1.1//EN\"\n",
              "  \"http://www.w3.org/Graphics/SVG/1.1/DTD/svg11.dtd\">\n",
              "<svg xmlns:xlink=\"http://www.w3.org/1999/xlink\" width=\"616.118438pt\" height=\"465.463281pt\" viewBox=\"0 0 616.118438 465.463281\" xmlns=\"http://www.w3.org/2000/svg\" version=\"1.1\">\n",
              " <metadata>\n",
              "  <rdf:RDF xmlns:dc=\"http://purl.org/dc/elements/1.1/\" xmlns:cc=\"http://creativecommons.org/ns#\" xmlns:rdf=\"http://www.w3.org/1999/02/22-rdf-syntax-ns#\">\n",
              "   <cc:Work>\n",
              "    <dc:type rdf:resource=\"http://purl.org/dc/dcmitype/StillImage\"/>\n",
              "    <dc:date>2024-06-05T19:29:54.978812</dc:date>\n",
              "    <dc:format>image/svg+xml</dc:format>\n",
              "    <dc:creator>\n",
              "     <cc:Agent>\n",
              "      <dc:title>Matplotlib v3.7.1, https://matplotlib.org/</dc:title>\n",
              "     </cc:Agent>\n",
              "    </dc:creator>\n",
              "   </cc:Work>\n",
              "  </rdf:RDF>\n",
              " </metadata>\n",
              " <defs>\n",
              "  <style type=\"text/css\">*{stroke-linejoin: round; stroke-linecap: butt}</style>\n",
              " </defs>\n",
              " <g id=\"figure_1\">\n",
              "  <g id=\"patch_1\">\n",
              "   <path d=\"M 0 465.463281 \n",
              "L 616.118438 465.463281 \n",
              "L 616.118438 -0 \n",
              "L 0 -0 \n",
              "z\n",
              "\" style=\"fill: #ffffff\"/>\n",
              "  </g>\n",
              "  <g id=\"axes_1\">\n",
              "   <g id=\"patch_2\">\n",
              "    <path d=\"M 50.918438 357.237656 \n",
              "L 608.918437 357.237656 \n",
              "L 608.918437 24.597656 \n",
              "L 50.918438 24.597656 \n",
              "z\n",
              "\" style=\"fill: #e5e5e5\"/>\n",
              "   </g>\n",
              "   <g id=\"matplotlib.axis_1\">\n",
              "    <g id=\"xtick_1\">\n",
              "     <g id=\"line2d_1\">\n",
              "      <path d=\"M 69.942516 357.237656 \n",
              "L 69.942516 24.597656 \n",
              "\" clip-path=\"url(#p596f914704)\" style=\"fill: none; stroke: #ffffff; stroke-width: 0.8; stroke-linecap: square\"/>\n",
              "     </g>\n",
              "     <g id=\"line2d_2\">\n",
              "      <defs>\n",
              "       <path id=\"m3acd92d0f0\" d=\"M 0 0 \n",
              "L 0 3.5 \n",
              "\" style=\"stroke: #555555; stroke-width: 0.8\"/>\n",
              "      </defs>\n",
              "      <g>\n",
              "       <use xlink:href=\"#m3acd92d0f0\" x=\"69.942516\" y=\"357.237656\" style=\"fill: #555555; stroke: #555555; stroke-width: 0.8\"/>\n",
              "      </g>\n",
              "     </g>\n",
              "     <g id=\"text_1\">\n",
              "      <!-- −60 -->\n",
              "      <g style=\"fill: #555555\" transform=\"translate(59.390172 371.836094) scale(0.1 -0.1)\">\n",
              "       <defs>\n",
              "        <path id=\"DejaVuSans-2212\" d=\"M 678 2272 \n",
              "L 4684 2272 \n",
              "L 4684 1741 \n",
              "L 678 1741 \n",
              "L 678 2272 \n",
              "z\n",
              "\" transform=\"scale(0.015625)\"/>\n",
              "        <path id=\"DejaVuSans-36\" d=\"M 2113 2584 \n",
              "Q 1688 2584 1439 2293 \n",
              "Q 1191 2003 1191 1497 \n",
              "Q 1191 994 1439 701 \n",
              "Q 1688 409 2113 409 \n",
              "Q 2538 409 2786 701 \n",
              "Q 3034 994 3034 1497 \n",
              "Q 3034 2003 2786 2293 \n",
              "Q 2538 2584 2113 2584 \n",
              "z\n",
              "M 3366 4563 \n",
              "L 3366 3988 \n",
              "Q 3128 4100 2886 4159 \n",
              "Q 2644 4219 2406 4219 \n",
              "Q 1781 4219 1451 3797 \n",
              "Q 1122 3375 1075 2522 \n",
              "Q 1259 2794 1537 2939 \n",
              "Q 1816 3084 2150 3084 \n",
              "Q 2853 3084 3261 2657 \n",
              "Q 3669 2231 3669 1497 \n",
              "Q 3669 778 3244 343 \n",
              "Q 2819 -91 2113 -91 \n",
              "Q 1303 -91 875 529 \n",
              "Q 447 1150 447 2328 \n",
              "Q 447 3434 972 4092 \n",
              "Q 1497 4750 2381 4750 \n",
              "Q 2619 4750 2861 4703 \n",
              "Q 3103 4656 3366 4563 \n",
              "z\n",
              "\" transform=\"scale(0.015625)\"/>\n",
              "        <path id=\"DejaVuSans-30\" d=\"M 2034 4250 \n",
              "Q 1547 4250 1301 3770 \n",
              "Q 1056 3291 1056 2328 \n",
              "Q 1056 1369 1301 889 \n",
              "Q 1547 409 2034 409 \n",
              "Q 2525 409 2770 889 \n",
              "Q 3016 1369 3016 2328 \n",
              "Q 3016 3291 2770 3770 \n",
              "Q 2525 4250 2034 4250 \n",
              "z\n",
              "M 2034 4750 \n",
              "Q 2819 4750 3233 4129 \n",
              "Q 3647 3509 3647 2328 \n",
              "Q 3647 1150 3233 529 \n",
              "Q 2819 -91 2034 -91 \n",
              "Q 1250 -91 836 529 \n",
              "Q 422 1150 422 2328 \n",
              "Q 422 3509 836 4129 \n",
              "Q 1250 4750 2034 4750 \n",
              "z\n",
              "\" transform=\"scale(0.015625)\"/>\n",
              "       </defs>\n",
              "       <use xlink:href=\"#DejaVuSans-2212\"/>\n",
              "       <use xlink:href=\"#DejaVuSans-36\" x=\"83.789062\"/>\n",
              "       <use xlink:href=\"#DejaVuSans-30\" x=\"147.412109\"/>\n",
              "      </g>\n",
              "     </g>\n",
              "    </g>\n",
              "    <g id=\"xtick_2\">\n",
              "     <g id=\"line2d_3\">\n",
              "      <path d=\"M 152.52949 357.237656 \n",
              "L 152.52949 24.597656 \n",
              "\" clip-path=\"url(#p596f914704)\" style=\"fill: none; stroke: #ffffff; stroke-width: 0.8; stroke-linecap: square\"/>\n",
              "     </g>\n",
              "     <g id=\"line2d_4\">\n",
              "      <g>\n",
              "       <use xlink:href=\"#m3acd92d0f0\" x=\"152.52949\" y=\"357.237656\" style=\"fill: #555555; stroke: #555555; stroke-width: 0.8\"/>\n",
              "      </g>\n",
              "     </g>\n",
              "     <g id=\"text_2\">\n",
              "      <!-- −40 -->\n",
              "      <g style=\"fill: #555555\" transform=\"translate(141.977146 371.836094) scale(0.1 -0.1)\">\n",
              "       <defs>\n",
              "        <path id=\"DejaVuSans-34\" d=\"M 2419 4116 \n",
              "L 825 1625 \n",
              "L 2419 1625 \n",
              "L 2419 4116 \n",
              "z\n",
              "M 2253 4666 \n",
              "L 3047 4666 \n",
              "L 3047 1625 \n",
              "L 3713 1625 \n",
              "L 3713 1100 \n",
              "L 3047 1100 \n",
              "L 3047 0 \n",
              "L 2419 0 \n",
              "L 2419 1100 \n",
              "L 313 1100 \n",
              "L 313 1709 \n",
              "L 2253 4666 \n",
              "z\n",
              "\" transform=\"scale(0.015625)\"/>\n",
              "       </defs>\n",
              "       <use xlink:href=\"#DejaVuSans-2212\"/>\n",
              "       <use xlink:href=\"#DejaVuSans-34\" x=\"83.789062\"/>\n",
              "       <use xlink:href=\"#DejaVuSans-30\" x=\"147.412109\"/>\n",
              "      </g>\n",
              "     </g>\n",
              "    </g>\n",
              "    <g id=\"xtick_3\">\n",
              "     <g id=\"line2d_5\">\n",
              "      <path d=\"M 235.116464 357.237656 \n",
              "L 235.116464 24.597656 \n",
              "\" clip-path=\"url(#p596f914704)\" style=\"fill: none; stroke: #ffffff; stroke-width: 0.8; stroke-linecap: square\"/>\n",
              "     </g>\n",
              "     <g id=\"line2d_6\">\n",
              "      <g>\n",
              "       <use xlink:href=\"#m3acd92d0f0\" x=\"235.116464\" y=\"357.237656\" style=\"fill: #555555; stroke: #555555; stroke-width: 0.8\"/>\n",
              "      </g>\n",
              "     </g>\n",
              "     <g id=\"text_3\">\n",
              "      <!-- −20 -->\n",
              "      <g style=\"fill: #555555\" transform=\"translate(224.56412 371.836094) scale(0.1 -0.1)\">\n",
              "       <defs>\n",
              "        <path id=\"DejaVuSans-32\" d=\"M 1228 531 \n",
              "L 3431 531 \n",
              "L 3431 0 \n",
              "L 469 0 \n",
              "L 469 531 \n",
              "Q 828 903 1448 1529 \n",
              "Q 2069 2156 2228 2338 \n",
              "Q 2531 2678 2651 2914 \n",
              "Q 2772 3150 2772 3378 \n",
              "Q 2772 3750 2511 3984 \n",
              "Q 2250 4219 1831 4219 \n",
              "Q 1534 4219 1204 4116 \n",
              "Q 875 4013 500 3803 \n",
              "L 500 4441 \n",
              "Q 881 4594 1212 4672 \n",
              "Q 1544 4750 1819 4750 \n",
              "Q 2544 4750 2975 4387 \n",
              "Q 3406 4025 3406 3419 \n",
              "Q 3406 3131 3298 2873 \n",
              "Q 3191 2616 2906 2266 \n",
              "Q 2828 2175 2409 1742 \n",
              "Q 1991 1309 1228 531 \n",
              "z\n",
              "\" transform=\"scale(0.015625)\"/>\n",
              "       </defs>\n",
              "       <use xlink:href=\"#DejaVuSans-2212\"/>\n",
              "       <use xlink:href=\"#DejaVuSans-32\" x=\"83.789062\"/>\n",
              "       <use xlink:href=\"#DejaVuSans-30\" x=\"147.412109\"/>\n",
              "      </g>\n",
              "     </g>\n",
              "    </g>\n",
              "    <g id=\"xtick_4\">\n",
              "     <g id=\"line2d_7\">\n",
              "      <path d=\"M 317.703437 357.237656 \n",
              "L 317.703437 24.597656 \n",
              "\" clip-path=\"url(#p596f914704)\" style=\"fill: none; stroke: #ffffff; stroke-width: 0.8; stroke-linecap: square\"/>\n",
              "     </g>\n",
              "     <g id=\"line2d_8\">\n",
              "      <g>\n",
              "       <use xlink:href=\"#m3acd92d0f0\" x=\"317.703437\" y=\"357.237656\" style=\"fill: #555555; stroke: #555555; stroke-width: 0.8\"/>\n",
              "      </g>\n",
              "     </g>\n",
              "     <g id=\"text_4\">\n",
              "      <!-- 0 -->\n",
              "      <g style=\"fill: #555555\" transform=\"translate(314.522187 371.836094) scale(0.1 -0.1)\">\n",
              "       <use xlink:href=\"#DejaVuSans-30\"/>\n",
              "      </g>\n",
              "     </g>\n",
              "    </g>\n",
              "    <g id=\"xtick_5\">\n",
              "     <g id=\"line2d_9\">\n",
              "      <path d=\"M 400.290411 357.237656 \n",
              "L 400.290411 24.597656 \n",
              "\" clip-path=\"url(#p596f914704)\" style=\"fill: none; stroke: #ffffff; stroke-width: 0.8; stroke-linecap: square\"/>\n",
              "     </g>\n",
              "     <g id=\"line2d_10\">\n",
              "      <g>\n",
              "       <use xlink:href=\"#m3acd92d0f0\" x=\"400.290411\" y=\"357.237656\" style=\"fill: #555555; stroke: #555555; stroke-width: 0.8\"/>\n",
              "      </g>\n",
              "     </g>\n",
              "     <g id=\"text_5\">\n",
              "      <!-- 20 -->\n",
              "      <g style=\"fill: #555555\" transform=\"translate(393.927911 371.836094) scale(0.1 -0.1)\">\n",
              "       <use xlink:href=\"#DejaVuSans-32\"/>\n",
              "       <use xlink:href=\"#DejaVuSans-30\" x=\"63.623047\"/>\n",
              "      </g>\n",
              "     </g>\n",
              "    </g>\n",
              "    <g id=\"xtick_6\">\n",
              "     <g id=\"line2d_11\">\n",
              "      <path d=\"M 482.877385 357.237656 \n",
              "L 482.877385 24.597656 \n",
              "\" clip-path=\"url(#p596f914704)\" style=\"fill: none; stroke: #ffffff; stroke-width: 0.8; stroke-linecap: square\"/>\n",
              "     </g>\n",
              "     <g id=\"line2d_12\">\n",
              "      <g>\n",
              "       <use xlink:href=\"#m3acd92d0f0\" x=\"482.877385\" y=\"357.237656\" style=\"fill: #555555; stroke: #555555; stroke-width: 0.8\"/>\n",
              "      </g>\n",
              "     </g>\n",
              "     <g id=\"text_6\">\n",
              "      <!-- 40 -->\n",
              "      <g style=\"fill: #555555\" transform=\"translate(476.514885 371.836094) scale(0.1 -0.1)\">\n",
              "       <use xlink:href=\"#DejaVuSans-34\"/>\n",
              "       <use xlink:href=\"#DejaVuSans-30\" x=\"63.623047\"/>\n",
              "      </g>\n",
              "     </g>\n",
              "    </g>\n",
              "    <g id=\"xtick_7\">\n",
              "     <g id=\"line2d_13\">\n",
              "      <path d=\"M 565.464359 357.237656 \n",
              "L 565.464359 24.597656 \n",
              "\" clip-path=\"url(#p596f914704)\" style=\"fill: none; stroke: #ffffff; stroke-width: 0.8; stroke-linecap: square\"/>\n",
              "     </g>\n",
              "     <g id=\"line2d_14\">\n",
              "      <g>\n",
              "       <use xlink:href=\"#m3acd92d0f0\" x=\"565.464359\" y=\"357.237656\" style=\"fill: #555555; stroke: #555555; stroke-width: 0.8\"/>\n",
              "      </g>\n",
              "     </g>\n",
              "     <g id=\"text_7\">\n",
              "      <!-- 60 -->\n",
              "      <g style=\"fill: #555555\" transform=\"translate(559.101859 371.836094) scale(0.1 -0.1)\">\n",
              "       <use xlink:href=\"#DejaVuSans-36\"/>\n",
              "       <use xlink:href=\"#DejaVuSans-30\" x=\"63.623047\"/>\n",
              "      </g>\n",
              "     </g>\n",
              "    </g>\n",
              "    <g id=\"text_8\">\n",
              "     <!-- t-SNE Component 1 -->\n",
              "     <g style=\"fill: #555555\" transform=\"translate(270.846563 387.033906) scale(0.12 -0.12)\">\n",
              "      <defs>\n",
              "       <path id=\"DejaVuSans-74\" d=\"M 1172 4494 \n",
              "L 1172 3500 \n",
              "L 2356 3500 \n",
              "L 2356 3053 \n",
              "L 1172 3053 \n",
              "L 1172 1153 \n",
              "Q 1172 725 1289 603 \n",
              "Q 1406 481 1766 481 \n",
              "L 2356 481 \n",
              "L 2356 0 \n",
              "L 1766 0 \n",
              "Q 1100 0 847 248 \n",
              "Q 594 497 594 1153 \n",
              "L 594 3053 \n",
              "L 172 3053 \n",
              "L 172 3500 \n",
              "L 594 3500 \n",
              "L 594 4494 \n",
              "L 1172 4494 \n",
              "z\n",
              "\" transform=\"scale(0.015625)\"/>\n",
              "       <path id=\"DejaVuSans-2d\" d=\"M 313 2009 \n",
              "L 1997 2009 \n",
              "L 1997 1497 \n",
              "L 313 1497 \n",
              "L 313 2009 \n",
              "z\n",
              "\" transform=\"scale(0.015625)\"/>\n",
              "       <path id=\"DejaVuSans-53\" d=\"M 3425 4513 \n",
              "L 3425 3897 \n",
              "Q 3066 4069 2747 4153 \n",
              "Q 2428 4238 2131 4238 \n",
              "Q 1616 4238 1336 4038 \n",
              "Q 1056 3838 1056 3469 \n",
              "Q 1056 3159 1242 3001 \n",
              "Q 1428 2844 1947 2747 \n",
              "L 2328 2669 \n",
              "Q 3034 2534 3370 2195 \n",
              "Q 3706 1856 3706 1288 \n",
              "Q 3706 609 3251 259 \n",
              "Q 2797 -91 1919 -91 \n",
              "Q 1588 -91 1214 -16 \n",
              "Q 841 59 441 206 \n",
              "L 441 856 \n",
              "Q 825 641 1194 531 \n",
              "Q 1563 422 1919 422 \n",
              "Q 2459 422 2753 634 \n",
              "Q 3047 847 3047 1241 \n",
              "Q 3047 1584 2836 1778 \n",
              "Q 2625 1972 2144 2069 \n",
              "L 1759 2144 \n",
              "Q 1053 2284 737 2584 \n",
              "Q 422 2884 422 3419 \n",
              "Q 422 4038 858 4394 \n",
              "Q 1294 4750 2059 4750 \n",
              "Q 2388 4750 2728 4690 \n",
              "Q 3069 4631 3425 4513 \n",
              "z\n",
              "\" transform=\"scale(0.015625)\"/>\n",
              "       <path id=\"DejaVuSans-4e\" d=\"M 628 4666 \n",
              "L 1478 4666 \n",
              "L 3547 763 \n",
              "L 3547 4666 \n",
              "L 4159 4666 \n",
              "L 4159 0 \n",
              "L 3309 0 \n",
              "L 1241 3903 \n",
              "L 1241 0 \n",
              "L 628 0 \n",
              "L 628 4666 \n",
              "z\n",
              "\" transform=\"scale(0.015625)\"/>\n",
              "       <path id=\"DejaVuSans-45\" d=\"M 628 4666 \n",
              "L 3578 4666 \n",
              "L 3578 4134 \n",
              "L 1259 4134 \n",
              "L 1259 2753 \n",
              "L 3481 2753 \n",
              "L 3481 2222 \n",
              "L 1259 2222 \n",
              "L 1259 531 \n",
              "L 3634 531 \n",
              "L 3634 0 \n",
              "L 628 0 \n",
              "L 628 4666 \n",
              "z\n",
              "\" transform=\"scale(0.015625)\"/>\n",
              "       <path id=\"DejaVuSans-20\" transform=\"scale(0.015625)\"/>\n",
              "       <path id=\"DejaVuSans-43\" d=\"M 4122 4306 \n",
              "L 4122 3641 \n",
              "Q 3803 3938 3442 4084 \n",
              "Q 3081 4231 2675 4231 \n",
              "Q 1875 4231 1450 3742 \n",
              "Q 1025 3253 1025 2328 \n",
              "Q 1025 1406 1450 917 \n",
              "Q 1875 428 2675 428 \n",
              "Q 3081 428 3442 575 \n",
              "Q 3803 722 4122 1019 \n",
              "L 4122 359 \n",
              "Q 3791 134 3420 21 \n",
              "Q 3050 -91 2638 -91 \n",
              "Q 1578 -91 968 557 \n",
              "Q 359 1206 359 2328 \n",
              "Q 359 3453 968 4101 \n",
              "Q 1578 4750 2638 4750 \n",
              "Q 3056 4750 3426 4639 \n",
              "Q 3797 4528 4122 4306 \n",
              "z\n",
              "\" transform=\"scale(0.015625)\"/>\n",
              "       <path id=\"DejaVuSans-6f\" d=\"M 1959 3097 \n",
              "Q 1497 3097 1228 2736 \n",
              "Q 959 2375 959 1747 \n",
              "Q 959 1119 1226 758 \n",
              "Q 1494 397 1959 397 \n",
              "Q 2419 397 2687 759 \n",
              "Q 2956 1122 2956 1747 \n",
              "Q 2956 2369 2687 2733 \n",
              "Q 2419 3097 1959 3097 \n",
              "z\n",
              "M 1959 3584 \n",
              "Q 2709 3584 3137 3096 \n",
              "Q 3566 2609 3566 1747 \n",
              "Q 3566 888 3137 398 \n",
              "Q 2709 -91 1959 -91 \n",
              "Q 1206 -91 779 398 \n",
              "Q 353 888 353 1747 \n",
              "Q 353 2609 779 3096 \n",
              "Q 1206 3584 1959 3584 \n",
              "z\n",
              "\" transform=\"scale(0.015625)\"/>\n",
              "       <path id=\"DejaVuSans-6d\" d=\"M 3328 2828 \n",
              "Q 3544 3216 3844 3400 \n",
              "Q 4144 3584 4550 3584 \n",
              "Q 5097 3584 5394 3201 \n",
              "Q 5691 2819 5691 2113 \n",
              "L 5691 0 \n",
              "L 5113 0 \n",
              "L 5113 2094 \n",
              "Q 5113 2597 4934 2840 \n",
              "Q 4756 3084 4391 3084 \n",
              "Q 3944 3084 3684 2787 \n",
              "Q 3425 2491 3425 1978 \n",
              "L 3425 0 \n",
              "L 2847 0 \n",
              "L 2847 2094 \n",
              "Q 2847 2600 2669 2842 \n",
              "Q 2491 3084 2119 3084 \n",
              "Q 1678 3084 1418 2786 \n",
              "Q 1159 2488 1159 1978 \n",
              "L 1159 0 \n",
              "L 581 0 \n",
              "L 581 3500 \n",
              "L 1159 3500 \n",
              "L 1159 2956 \n",
              "Q 1356 3278 1631 3431 \n",
              "Q 1906 3584 2284 3584 \n",
              "Q 2666 3584 2933 3390 \n",
              "Q 3200 3197 3328 2828 \n",
              "z\n",
              "\" transform=\"scale(0.015625)\"/>\n",
              "       <path id=\"DejaVuSans-70\" d=\"M 1159 525 \n",
              "L 1159 -1331 \n",
              "L 581 -1331 \n",
              "L 581 3500 \n",
              "L 1159 3500 \n",
              "L 1159 2969 \n",
              "Q 1341 3281 1617 3432 \n",
              "Q 1894 3584 2278 3584 \n",
              "Q 2916 3584 3314 3078 \n",
              "Q 3713 2572 3713 1747 \n",
              "Q 3713 922 3314 415 \n",
              "Q 2916 -91 2278 -91 \n",
              "Q 1894 -91 1617 61 \n",
              "Q 1341 213 1159 525 \n",
              "z\n",
              "M 3116 1747 \n",
              "Q 3116 2381 2855 2742 \n",
              "Q 2594 3103 2138 3103 \n",
              "Q 1681 3103 1420 2742 \n",
              "Q 1159 2381 1159 1747 \n",
              "Q 1159 1113 1420 752 \n",
              "Q 1681 391 2138 391 \n",
              "Q 2594 391 2855 752 \n",
              "Q 3116 1113 3116 1747 \n",
              "z\n",
              "\" transform=\"scale(0.015625)\"/>\n",
              "       <path id=\"DejaVuSans-6e\" d=\"M 3513 2113 \n",
              "L 3513 0 \n",
              "L 2938 0 \n",
              "L 2938 2094 \n",
              "Q 2938 2591 2744 2837 \n",
              "Q 2550 3084 2163 3084 \n",
              "Q 1697 3084 1428 2787 \n",
              "Q 1159 2491 1159 1978 \n",
              "L 1159 0 \n",
              "L 581 0 \n",
              "L 581 3500 \n",
              "L 1159 3500 \n",
              "L 1159 2956 \n",
              "Q 1366 3272 1645 3428 \n",
              "Q 1925 3584 2291 3584 \n",
              "Q 2894 3584 3203 3211 \n",
              "Q 3513 2838 3513 2113 \n",
              "z\n",
              "\" transform=\"scale(0.015625)\"/>\n",
              "       <path id=\"DejaVuSans-65\" d=\"M 3597 1894 \n",
              "L 3597 1613 \n",
              "L 953 1613 \n",
              "Q 991 1019 1311 708 \n",
              "Q 1631 397 2203 397 \n",
              "Q 2534 397 2845 478 \n",
              "Q 3156 559 3463 722 \n",
              "L 3463 178 \n",
              "Q 3153 47 2828 -22 \n",
              "Q 2503 -91 2169 -91 \n",
              "Q 1331 -91 842 396 \n",
              "Q 353 884 353 1716 \n",
              "Q 353 2575 817 3079 \n",
              "Q 1281 3584 2069 3584 \n",
              "Q 2775 3584 3186 3129 \n",
              "Q 3597 2675 3597 1894 \n",
              "z\n",
              "M 3022 2063 \n",
              "Q 3016 2534 2758 2815 \n",
              "Q 2500 3097 2075 3097 \n",
              "Q 1594 3097 1305 2825 \n",
              "Q 1016 2553 972 2059 \n",
              "L 3022 2063 \n",
              "z\n",
              "\" transform=\"scale(0.015625)\"/>\n",
              "       <path id=\"DejaVuSans-31\" d=\"M 794 531 \n",
              "L 1825 531 \n",
              "L 1825 4091 \n",
              "L 703 3866 \n",
              "L 703 4441 \n",
              "L 1819 4666 \n",
              "L 2450 4666 \n",
              "L 2450 531 \n",
              "L 3481 531 \n",
              "L 3481 0 \n",
              "L 794 0 \n",
              "L 794 531 \n",
              "z\n",
              "\" transform=\"scale(0.015625)\"/>\n",
              "      </defs>\n",
              "      <use xlink:href=\"#DejaVuSans-74\"/>\n",
              "      <use xlink:href=\"#DejaVuSans-2d\" x=\"39.208984\"/>\n",
              "      <use xlink:href=\"#DejaVuSans-53\" x=\"75.292969\"/>\n",
              "      <use xlink:href=\"#DejaVuSans-4e\" x=\"138.769531\"/>\n",
              "      <use xlink:href=\"#DejaVuSans-45\" x=\"213.574219\"/>\n",
              "      <use xlink:href=\"#DejaVuSans-20\" x=\"276.757812\"/>\n",
              "      <use xlink:href=\"#DejaVuSans-43\" x=\"308.544922\"/>\n",
              "      <use xlink:href=\"#DejaVuSans-6f\" x=\"378.369141\"/>\n",
              "      <use xlink:href=\"#DejaVuSans-6d\" x=\"439.550781\"/>\n",
              "      <use xlink:href=\"#DejaVuSans-70\" x=\"536.962891\"/>\n",
              "      <use xlink:href=\"#DejaVuSans-6f\" x=\"600.439453\"/>\n",
              "      <use xlink:href=\"#DejaVuSans-6e\" x=\"661.621094\"/>\n",
              "      <use xlink:href=\"#DejaVuSans-65\" x=\"725\"/>\n",
              "      <use xlink:href=\"#DejaVuSans-6e\" x=\"786.523438\"/>\n",
              "      <use xlink:href=\"#DejaVuSans-74\" x=\"849.902344\"/>\n",
              "      <use xlink:href=\"#DejaVuSans-20\" x=\"889.111328\"/>\n",
              "      <use xlink:href=\"#DejaVuSans-31\" x=\"920.898438\"/>\n",
              "     </g>\n",
              "    </g>\n",
              "   </g>\n",
              "   <g id=\"matplotlib.axis_2\">\n",
              "    <g id=\"ytick_1\">\n",
              "     <g id=\"line2d_15\">\n",
              "      <path d=\"M 50.918438 348.073589 \n",
              "L 608.918437 348.073589 \n",
              "\" clip-path=\"url(#p596f914704)\" style=\"fill: none; stroke: #ffffff; stroke-width: 0.8; stroke-linecap: square\"/>\n",
              "     </g>\n",
              "     <g id=\"line2d_16\">\n",
              "      <defs>\n",
              "       <path id=\"ma8318914ba\" d=\"M 0 0 \n",
              "L -3.5 0 \n",
              "\" style=\"stroke: #555555; stroke-width: 0.8\"/>\n",
              "      </defs>\n",
              "      <g>\n",
              "       <use xlink:href=\"#ma8318914ba\" x=\"50.918438\" y=\"348.073589\" style=\"fill: #555555; stroke: #555555; stroke-width: 0.8\"/>\n",
              "      </g>\n",
              "     </g>\n",
              "     <g id=\"text_9\">\n",
              "      <!-- −40 -->\n",
              "      <g style=\"fill: #555555\" transform=\"translate(22.81375 351.872808) scale(0.1 -0.1)\">\n",
              "       <use xlink:href=\"#DejaVuSans-2212\"/>\n",
              "       <use xlink:href=\"#DejaVuSans-34\" x=\"83.789062\"/>\n",
              "       <use xlink:href=\"#DejaVuSans-30\" x=\"147.412109\"/>\n",
              "      </g>\n",
              "     </g>\n",
              "    </g>\n",
              "    <g id=\"ytick_2\">\n",
              "     <g id=\"line2d_17\">\n",
              "      <path d=\"M 50.918438 308.690076 \n",
              "L 608.918437 308.690076 \n",
              "\" clip-path=\"url(#p596f914704)\" style=\"fill: none; stroke: #ffffff; stroke-width: 0.8; stroke-linecap: square\"/>\n",
              "     </g>\n",
              "     <g id=\"line2d_18\">\n",
              "      <g>\n",
              "       <use xlink:href=\"#ma8318914ba\" x=\"50.918438\" y=\"308.690076\" style=\"fill: #555555; stroke: #555555; stroke-width: 0.8\"/>\n",
              "      </g>\n",
              "     </g>\n",
              "     <g id=\"text_10\">\n",
              "      <!-- −30 -->\n",
              "      <g style=\"fill: #555555\" transform=\"translate(22.81375 312.489294) scale(0.1 -0.1)\">\n",
              "       <defs>\n",
              "        <path id=\"DejaVuSans-33\" d=\"M 2597 2516 \n",
              "Q 3050 2419 3304 2112 \n",
              "Q 3559 1806 3559 1356 \n",
              "Q 3559 666 3084 287 \n",
              "Q 2609 -91 1734 -91 \n",
              "Q 1441 -91 1130 -33 \n",
              "Q 819 25 488 141 \n",
              "L 488 750 \n",
              "Q 750 597 1062 519 \n",
              "Q 1375 441 1716 441 \n",
              "Q 2309 441 2620 675 \n",
              "Q 2931 909 2931 1356 \n",
              "Q 2931 1769 2642 2001 \n",
              "Q 2353 2234 1838 2234 \n",
              "L 1294 2234 \n",
              "L 1294 2753 \n",
              "L 1863 2753 \n",
              "Q 2328 2753 2575 2939 \n",
              "Q 2822 3125 2822 3475 \n",
              "Q 2822 3834 2567 4026 \n",
              "Q 2313 4219 1838 4219 \n",
              "Q 1578 4219 1281 4162 \n",
              "Q 984 4106 628 3988 \n",
              "L 628 4550 \n",
              "Q 988 4650 1302 4700 \n",
              "Q 1616 4750 1894 4750 \n",
              "Q 2613 4750 3031 4423 \n",
              "Q 3450 4097 3450 3541 \n",
              "Q 3450 3153 3228 2886 \n",
              "Q 3006 2619 2597 2516 \n",
              "z\n",
              "\" transform=\"scale(0.015625)\"/>\n",
              "       </defs>\n",
              "       <use xlink:href=\"#DejaVuSans-2212\"/>\n",
              "       <use xlink:href=\"#DejaVuSans-33\" x=\"83.789062\"/>\n",
              "       <use xlink:href=\"#DejaVuSans-30\" x=\"147.412109\"/>\n",
              "      </g>\n",
              "     </g>\n",
              "    </g>\n",
              "    <g id=\"ytick_3\">\n",
              "     <g id=\"line2d_19\">\n",
              "      <path d=\"M 50.918438 269.306562 \n",
              "L 608.918437 269.306562 \n",
              "\" clip-path=\"url(#p596f914704)\" style=\"fill: none; stroke: #ffffff; stroke-width: 0.8; stroke-linecap: square\"/>\n",
              "     </g>\n",
              "     <g id=\"line2d_20\">\n",
              "      <g>\n",
              "       <use xlink:href=\"#ma8318914ba\" x=\"50.918438\" y=\"269.306562\" style=\"fill: #555555; stroke: #555555; stroke-width: 0.8\"/>\n",
              "      </g>\n",
              "     </g>\n",
              "     <g id=\"text_11\">\n",
              "      <!-- −20 -->\n",
              "      <g style=\"fill: #555555\" transform=\"translate(22.81375 273.105781) scale(0.1 -0.1)\">\n",
              "       <use xlink:href=\"#DejaVuSans-2212\"/>\n",
              "       <use xlink:href=\"#DejaVuSans-32\" x=\"83.789062\"/>\n",
              "       <use xlink:href=\"#DejaVuSans-30\" x=\"147.412109\"/>\n",
              "      </g>\n",
              "     </g>\n",
              "    </g>\n",
              "    <g id=\"ytick_4\">\n",
              "     <g id=\"line2d_21\">\n",
              "      <path d=\"M 50.918438 229.923048 \n",
              "L 608.918437 229.923048 \n",
              "\" clip-path=\"url(#p596f914704)\" style=\"fill: none; stroke: #ffffff; stroke-width: 0.8; stroke-linecap: square\"/>\n",
              "     </g>\n",
              "     <g id=\"line2d_22\">\n",
              "      <g>\n",
              "       <use xlink:href=\"#ma8318914ba\" x=\"50.918438\" y=\"229.923048\" style=\"fill: #555555; stroke: #555555; stroke-width: 0.8\"/>\n",
              "      </g>\n",
              "     </g>\n",
              "     <g id=\"text_12\">\n",
              "      <!-- −10 -->\n",
              "      <g style=\"fill: #555555\" transform=\"translate(22.81375 233.722267) scale(0.1 -0.1)\">\n",
              "       <use xlink:href=\"#DejaVuSans-2212\"/>\n",
              "       <use xlink:href=\"#DejaVuSans-31\" x=\"83.789062\"/>\n",
              "       <use xlink:href=\"#DejaVuSans-30\" x=\"147.412109\"/>\n",
              "      </g>\n",
              "     </g>\n",
              "    </g>\n",
              "    <g id=\"ytick_5\">\n",
              "     <g id=\"line2d_23\">\n",
              "      <path d=\"M 50.918438 190.539534 \n",
              "L 608.918437 190.539534 \n",
              "\" clip-path=\"url(#p596f914704)\" style=\"fill: none; stroke: #ffffff; stroke-width: 0.8; stroke-linecap: square\"/>\n",
              "     </g>\n",
              "     <g id=\"line2d_24\">\n",
              "      <g>\n",
              "       <use xlink:href=\"#ma8318914ba\" x=\"50.918438\" y=\"190.539534\" style=\"fill: #555555; stroke: #555555; stroke-width: 0.8\"/>\n",
              "      </g>\n",
              "     </g>\n",
              "     <g id=\"text_13\">\n",
              "      <!-- 0 -->\n",
              "      <g style=\"fill: #555555\" transform=\"translate(37.555937 194.338753) scale(0.1 -0.1)\">\n",
              "       <use xlink:href=\"#DejaVuSans-30\"/>\n",
              "      </g>\n",
              "     </g>\n",
              "    </g>\n",
              "    <g id=\"ytick_6\">\n",
              "     <g id=\"line2d_25\">\n",
              "      <path d=\"M 50.918438 151.156021 \n",
              "L 608.918437 151.156021 \n",
              "\" clip-path=\"url(#p596f914704)\" style=\"fill: none; stroke: #ffffff; stroke-width: 0.8; stroke-linecap: square\"/>\n",
              "     </g>\n",
              "     <g id=\"line2d_26\">\n",
              "      <g>\n",
              "       <use xlink:href=\"#ma8318914ba\" x=\"50.918438\" y=\"151.156021\" style=\"fill: #555555; stroke: #555555; stroke-width: 0.8\"/>\n",
              "      </g>\n",
              "     </g>\n",
              "     <g id=\"text_14\">\n",
              "      <!-- 10 -->\n",
              "      <g style=\"fill: #555555\" transform=\"translate(31.193438 154.955239) scale(0.1 -0.1)\">\n",
              "       <use xlink:href=\"#DejaVuSans-31\"/>\n",
              "       <use xlink:href=\"#DejaVuSans-30\" x=\"63.623047\"/>\n",
              "      </g>\n",
              "     </g>\n",
              "    </g>\n",
              "    <g id=\"ytick_7\">\n",
              "     <g id=\"line2d_27\">\n",
              "      <path d=\"M 50.918438 111.772507 \n",
              "L 608.918437 111.772507 \n",
              "\" clip-path=\"url(#p596f914704)\" style=\"fill: none; stroke: #ffffff; stroke-width: 0.8; stroke-linecap: square\"/>\n",
              "     </g>\n",
              "     <g id=\"line2d_28\">\n",
              "      <g>\n",
              "       <use xlink:href=\"#ma8318914ba\" x=\"50.918438\" y=\"111.772507\" style=\"fill: #555555; stroke: #555555; stroke-width: 0.8\"/>\n",
              "      </g>\n",
              "     </g>\n",
              "     <g id=\"text_15\">\n",
              "      <!-- 20 -->\n",
              "      <g style=\"fill: #555555\" transform=\"translate(31.193438 115.571726) scale(0.1 -0.1)\">\n",
              "       <use xlink:href=\"#DejaVuSans-32\"/>\n",
              "       <use xlink:href=\"#DejaVuSans-30\" x=\"63.623047\"/>\n",
              "      </g>\n",
              "     </g>\n",
              "    </g>\n",
              "    <g id=\"ytick_8\">\n",
              "     <g id=\"line2d_29\">\n",
              "      <path d=\"M 50.918438 72.388993 \n",
              "L 608.918437 72.388993 \n",
              "\" clip-path=\"url(#p596f914704)\" style=\"fill: none; stroke: #ffffff; stroke-width: 0.8; stroke-linecap: square\"/>\n",
              "     </g>\n",
              "     <g id=\"line2d_30\">\n",
              "      <g>\n",
              "       <use xlink:href=\"#ma8318914ba\" x=\"50.918438\" y=\"72.388993\" style=\"fill: #555555; stroke: #555555; stroke-width: 0.8\"/>\n",
              "      </g>\n",
              "     </g>\n",
              "     <g id=\"text_16\">\n",
              "      <!-- 30 -->\n",
              "      <g style=\"fill: #555555\" transform=\"translate(31.193438 76.188212) scale(0.1 -0.1)\">\n",
              "       <use xlink:href=\"#DejaVuSans-33\"/>\n",
              "       <use xlink:href=\"#DejaVuSans-30\" x=\"63.623047\"/>\n",
              "      </g>\n",
              "     </g>\n",
              "    </g>\n",
              "    <g id=\"ytick_9\">\n",
              "     <g id=\"line2d_31\">\n",
              "      <path d=\"M 50.918438 33.005479 \n",
              "L 608.918437 33.005479 \n",
              "\" clip-path=\"url(#p596f914704)\" style=\"fill: none; stroke: #ffffff; stroke-width: 0.8; stroke-linecap: square\"/>\n",
              "     </g>\n",
              "     <g id=\"line2d_32\">\n",
              "      <g>\n",
              "       <use xlink:href=\"#ma8318914ba\" x=\"50.918438\" y=\"33.005479\" style=\"fill: #555555; stroke: #555555; stroke-width: 0.8\"/>\n",
              "      </g>\n",
              "     </g>\n",
              "     <g id=\"text_17\">\n",
              "      <!-- 40 -->\n",
              "      <g style=\"fill: #555555\" transform=\"translate(31.193438 36.804698) scale(0.1 -0.1)\">\n",
              "       <use xlink:href=\"#DejaVuSans-34\"/>\n",
              "       <use xlink:href=\"#DejaVuSans-30\" x=\"63.623047\"/>\n",
              "      </g>\n",
              "     </g>\n",
              "    </g>\n",
              "    <g id=\"text_18\">\n",
              "     <!-- t-SNE Component 2 -->\n",
              "     <g style=\"fill: #555555\" transform=\"translate(16.318125 249.989531) rotate(-90) scale(0.12 -0.12)\">\n",
              "      <use xlink:href=\"#DejaVuSans-74\"/>\n",
              "      <use xlink:href=\"#DejaVuSans-2d\" x=\"39.208984\"/>\n",
              "      <use xlink:href=\"#DejaVuSans-53\" x=\"75.292969\"/>\n",
              "      <use xlink:href=\"#DejaVuSans-4e\" x=\"138.769531\"/>\n",
              "      <use xlink:href=\"#DejaVuSans-45\" x=\"213.574219\"/>\n",
              "      <use xlink:href=\"#DejaVuSans-20\" x=\"276.757812\"/>\n",
              "      <use xlink:href=\"#DejaVuSans-43\" x=\"308.544922\"/>\n",
              "      <use xlink:href=\"#DejaVuSans-6f\" x=\"378.369141\"/>\n",
              "      <use xlink:href=\"#DejaVuSans-6d\" x=\"439.550781\"/>\n",
              "      <use xlink:href=\"#DejaVuSans-70\" x=\"536.962891\"/>\n",
              "      <use xlink:href=\"#DejaVuSans-6f\" x=\"600.439453\"/>\n",
              "      <use xlink:href=\"#DejaVuSans-6e\" x=\"661.621094\"/>\n",
              "      <use xlink:href=\"#DejaVuSans-65\" x=\"725\"/>\n",
              "      <use xlink:href=\"#DejaVuSans-6e\" x=\"786.523438\"/>\n",
              "      <use xlink:href=\"#DejaVuSans-74\" x=\"849.902344\"/>\n",
              "      <use xlink:href=\"#DejaVuSans-20\" x=\"889.111328\"/>\n",
              "      <use xlink:href=\"#DejaVuSans-32\" x=\"920.898438\"/>\n",
              "     </g>\n",
              "    </g>\n",
              "   </g>\n",
              "   <g id=\"PathCollection_1\">\n",
              "    <defs>\n",
              "     <path id=\"mc2a3f2c82a\" d=\"M 0 3 \n",
              "C 0.795609 3 1.55874 2.683901 2.12132 2.12132 \n",
              "C 2.683901 1.55874 3 0.795609 3 0 \n",
              "C 3 -0.795609 2.683901 -1.55874 2.12132 -2.12132 \n",
              "C 1.55874 -2.683901 0.795609 -3 0 -3 \n",
              "C -0.795609 -3 -1.55874 -2.683901 -2.12132 -2.12132 \n",
              "C -2.683901 -1.55874 -3 -0.795609 -3 0 \n",
              "C -3 0.795609 -2.683901 1.55874 -2.12132 2.12132 \n",
              "C -1.55874 2.683901 -0.795609 3 0 3 \n",
              "z\n",
              "\" style=\"stroke: #1f77b4; stroke-width: 0.5\"/>\n",
              "    </defs>\n",
              "    <g clip-path=\"url(#p596f914704)\">\n",
              "     <use xlink:href=\"#mc2a3f2c82a\" x=\"475.123971\" y=\"103.310544\" style=\"fill: #1f77b4; stroke: #1f77b4; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#mc2a3f2c82a\" x=\"500.936323\" y=\"103.721861\" style=\"fill: #1f77b4; stroke: #1f77b4; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#mc2a3f2c82a\" x=\"428.573214\" y=\"90.662047\" style=\"fill: #1f77b4; stroke: #1f77b4; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#mc2a3f2c82a\" x=\"435.67536\" y=\"146.836182\" style=\"fill: #1f77b4; stroke: #1f77b4; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#mc2a3f2c82a\" x=\"472.563811\" y=\"97.082909\" style=\"fill: #1f77b4; stroke: #1f77b4; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#mc2a3f2c82a\" x=\"466.995088\" y=\"94.045088\" style=\"fill: #1f77b4; stroke: #1f77b4; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#mc2a3f2c82a\" x=\"449.280139\" y=\"102.055373\" style=\"fill: #1f77b4; stroke: #1f77b4; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#mc2a3f2c82a\" x=\"474.772004\" y=\"87.808867\" style=\"fill: #1f77b4; stroke: #1f77b4; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#mc2a3f2c82a\" x=\"466.625068\" y=\"81.185802\" style=\"fill: #1f77b4; stroke: #1f77b4; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#mc2a3f2c82a\" x=\"457.283469\" y=\"101.903312\" style=\"fill: #1f77b4; stroke: #1f77b4; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#mc2a3f2c82a\" x=\"447.661174\" y=\"87.799101\" style=\"fill: #1f77b4; stroke: #1f77b4; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#mc2a3f2c82a\" x=\"497.399809\" y=\"122.620603\" style=\"fill: #1f77b4; stroke: #1f77b4; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#mc2a3f2c82a\" x=\"453.543531\" y=\"140.109552\" style=\"fill: #1f77b4; stroke: #1f77b4; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#mc2a3f2c82a\" x=\"454.035363\" y=\"88.080486\" style=\"fill: #1f77b4; stroke: #1f77b4; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#mc2a3f2c82a\" x=\"387.266488\" y=\"76.088972\" style=\"fill: #1f77b4; stroke: #1f77b4; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#mc2a3f2c82a\" x=\"472.5017\" y=\"125.884943\" style=\"fill: #1f77b4; stroke: #1f77b4; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#mc2a3f2c82a\" x=\"446.965508\" y=\"79.940795\" style=\"fill: #1f77b4; stroke: #1f77b4; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#mc2a3f2c82a\" x=\"473.661504\" y=\"95.318693\" style=\"fill: #1f77b4; stroke: #1f77b4; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#mc2a3f2c82a\" x=\"451.486465\" y=\"100.424334\" style=\"fill: #1f77b4; stroke: #1f77b4; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#mc2a3f2c82a\" x=\"471.636478\" y=\"126.456133\" style=\"fill: #1f77b4; stroke: #1f77b4; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#mc2a3f2c82a\" x=\"432.984575\" y=\"84.26484\" style=\"fill: #1f77b4; stroke: #1f77b4; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#mc2a3f2c82a\" x=\"405.942463\" y=\"95.715707\" style=\"fill: #1f77b4; stroke: #1f77b4; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#mc2a3f2c82a\" x=\"458.829281\" y=\"93.127896\" style=\"fill: #1f77b4; stroke: #1f77b4; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#mc2a3f2c82a\" x=\"462.706138\" y=\"136.928136\" style=\"fill: #1f77b4; stroke: #1f77b4; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#mc2a3f2c82a\" x=\"449.384458\" y=\"93.520275\" style=\"fill: #1f77b4; stroke: #1f77b4; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#mc2a3f2c82a\" x=\"464.842768\" y=\"100.619078\" style=\"fill: #1f77b4; stroke: #1f77b4; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#mc2a3f2c82a\" x=\"490.69951\" y=\"125.698447\" style=\"fill: #1f77b4; stroke: #1f77b4; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#mc2a3f2c82a\" x=\"448.026263\" y=\"82.154089\" style=\"fill: #1f77b4; stroke: #1f77b4; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#mc2a3f2c82a\" x=\"449.615511\" y=\"117.647222\" style=\"fill: #1f77b4; stroke: #1f77b4; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#mc2a3f2c82a\" x=\"454.395127\" y=\"73.124902\" style=\"fill: #1f77b4; stroke: #1f77b4; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#mc2a3f2c82a\" x=\"490.265694\" y=\"121.865689\" style=\"fill: #1f77b4; stroke: #1f77b4; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#mc2a3f2c82a\" x=\"467.549015\" y=\"106.432812\" style=\"fill: #1f77b4; stroke: #1f77b4; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#mc2a3f2c82a\" x=\"475.907896\" y=\"86.709739\" style=\"fill: #1f77b4; stroke: #1f77b4; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#mc2a3f2c82a\" x=\"428.239062\" y=\"140.529808\" style=\"fill: #1f77b4; stroke: #1f77b4; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#mc2a3f2c82a\" x=\"457.337184\" y=\"82.240941\" style=\"fill: #1f77b4; stroke: #1f77b4; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#mc2a3f2c82a\" x=\"468.340469\" y=\"120.104756\" style=\"fill: #1f77b4; stroke: #1f77b4; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#mc2a3f2c82a\" x=\"461.763888\" y=\"95.775644\" style=\"fill: #1f77b4; stroke: #1f77b4; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#mc2a3f2c82a\" x=\"431.676723\" y=\"82.87182\" style=\"fill: #1f77b4; stroke: #1f77b4; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#mc2a3f2c82a\" x=\"439.749742\" y=\"100.32001\" style=\"fill: #1f77b4; stroke: #1f77b4; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#mc2a3f2c82a\" x=\"440.524247\" y=\"85.73907\" style=\"fill: #1f77b4; stroke: #1f77b4; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#mc2a3f2c82a\" x=\"456.703488\" y=\"89.620009\" style=\"fill: #1f77b4; stroke: #1f77b4; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#mc2a3f2c82a\" x=\"476.349966\" y=\"173.692687\" style=\"fill: #1f77b4; stroke: #1f77b4; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#mc2a3f2c82a\" x=\"435.455436\" y=\"72.912739\" style=\"fill: #1f77b4; stroke: #1f77b4; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#mc2a3f2c82a\" x=\"460.471844\" y=\"106.395636\" style=\"fill: #1f77b4; stroke: #1f77b4; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#mc2a3f2c82a\" x=\"467.020307\" y=\"81.107086\" style=\"fill: #1f77b4; stroke: #1f77b4; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#mc2a3f2c82a\" x=\"453.912243\" y=\"102.134871\" style=\"fill: #1f77b4; stroke: #1f77b4; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#mc2a3f2c82a\" x=\"435.444496\" y=\"72.884156\" style=\"fill: #1f77b4; stroke: #1f77b4; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#mc2a3f2c82a\" x=\"455.740556\" y=\"106.047209\" style=\"fill: #1f77b4; stroke: #1f77b4; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#mc2a3f2c82a\" x=\"474.81789\" y=\"155.000579\" style=\"fill: #1f77b4; stroke: #1f77b4; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#mc2a3f2c82a\" x=\"455.197923\" y=\"151.89806\" style=\"fill: #1f77b4; stroke: #1f77b4; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#mc2a3f2c82a\" x=\"437.498293\" y=\"164.720566\" style=\"fill: #1f77b4; stroke: #1f77b4; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#mc2a3f2c82a\" x=\"446.939502\" y=\"113.838818\" style=\"fill: #1f77b4; stroke: #1f77b4; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#mc2a3f2c82a\" x=\"425.714234\" y=\"105.126516\" style=\"fill: #1f77b4; stroke: #1f77b4; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#mc2a3f2c82a\" x=\"504.538981\" y=\"117.627459\" style=\"fill: #1f77b4; stroke: #1f77b4; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#mc2a3f2c82a\" x=\"498.33486\" y=\"167.621692\" style=\"fill: #1f77b4; stroke: #1f77b4; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#mc2a3f2c82a\" x=\"496.246825\" y=\"127.799199\" style=\"fill: #1f77b4; stroke: #1f77b4; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#mc2a3f2c82a\" x=\"446.563795\" y=\"112.009196\" style=\"fill: #1f77b4; stroke: #1f77b4; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#mc2a3f2c82a\" x=\"463.309936\" y=\"90.029748\" style=\"fill: #1f77b4; stroke: #1f77b4; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#mc2a3f2c82a\" x=\"434.9555\" y=\"115.690216\" style=\"fill: #1f77b4; stroke: #1f77b4; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#mc2a3f2c82a\" x=\"422.588971\" y=\"90.985799\" style=\"fill: #1f77b4; stroke: #1f77b4; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#mc2a3f2c82a\" x=\"470.889744\" y=\"157.57843\" style=\"fill: #1f77b4; stroke: #1f77b4; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#mc2a3f2c82a\" x=\"484.877475\" y=\"119.602193\" style=\"fill: #1f77b4; stroke: #1f77b4; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#mc2a3f2c82a\" x=\"461.132618\" y=\"102.416038\" style=\"fill: #1f77b4; stroke: #1f77b4; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#mc2a3f2c82a\" x=\"465.302654\" y=\"95.457826\" style=\"fill: #1f77b4; stroke: #1f77b4; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#mc2a3f2c82a\" x=\"462.417037\" y=\"98.758921\" style=\"fill: #1f77b4; stroke: #1f77b4; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#mc2a3f2c82a\" x=\"454.890235\" y=\"95.645096\" style=\"fill: #1f77b4; stroke: #1f77b4; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#mc2a3f2c82a\" x=\"468.397445\" y=\"116.802534\" style=\"fill: #1f77b4; stroke: #1f77b4; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#mc2a3f2c82a\" x=\"454.464421\" y=\"83.320478\" style=\"fill: #1f77b4; stroke: #1f77b4; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#mc2a3f2c82a\" x=\"460.77008\" y=\"112.593871\" style=\"fill: #1f77b4; stroke: #1f77b4; stroke-width: 0.5\"/>\n",
              "    </g>\n",
              "   </g>\n",
              "   <g id=\"PathCollection_2\">\n",
              "    <defs>\n",
              "     <path id=\"mba7e5ca1c2\" d=\"M 0 3 \n",
              "C 0.795609 3 1.55874 2.683901 2.12132 2.12132 \n",
              "C 2.683901 1.55874 3 0.795609 3 0 \n",
              "C 3 -0.795609 2.683901 -1.55874 2.12132 -2.12132 \n",
              "C 1.55874 -2.683901 0.795609 -3 0 -3 \n",
              "C -0.795609 -3 -1.55874 -2.683901 -2.12132 -2.12132 \n",
              "C -2.683901 -1.55874 -3 -0.795609 -3 0 \n",
              "C -3 0.795609 -2.683901 1.55874 -2.12132 2.12132 \n",
              "C -1.55874 2.683901 -0.795609 3 0 3 \n",
              "z\n",
              "\" style=\"stroke: #aec7e8; stroke-width: 0.5\"/>\n",
              "    </defs>\n",
              "    <g clip-path=\"url(#p596f914704)\">\n",
              "     <use xlink:href=\"#mba7e5ca1c2\" x=\"179.524329\" y=\"211.613742\" style=\"fill: #aec7e8; stroke: #aec7e8; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#mba7e5ca1c2\" x=\"234.755958\" y=\"277.105855\" style=\"fill: #aec7e8; stroke: #aec7e8; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#mba7e5ca1c2\" x=\"123.457076\" y=\"267.063528\" style=\"fill: #aec7e8; stroke: #aec7e8; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#mba7e5ca1c2\" x=\"176.637814\" y=\"169.816457\" style=\"fill: #aec7e8; stroke: #aec7e8; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#mba7e5ca1c2\" x=\"195.820523\" y=\"229.96709\" style=\"fill: #aec7e8; stroke: #aec7e8; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#mba7e5ca1c2\" x=\"265.204352\" y=\"275.199749\" style=\"fill: #aec7e8; stroke: #aec7e8; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#mba7e5ca1c2\" x=\"120.107477\" y=\"266.629563\" style=\"fill: #aec7e8; stroke: #aec7e8; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#mba7e5ca1c2\" x=\"180.33697\" y=\"218.284268\" style=\"fill: #aec7e8; stroke: #aec7e8; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#mba7e5ca1c2\" x=\"233.409624\" y=\"212.062257\" style=\"fill: #aec7e8; stroke: #aec7e8; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#mba7e5ca1c2\" x=\"251.974614\" y=\"278.553982\" style=\"fill: #aec7e8; stroke: #aec7e8; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#mba7e5ca1c2\" x=\"193.978727\" y=\"217.228483\" style=\"fill: #aec7e8; stroke: #aec7e8; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#mba7e5ca1c2\" x=\"192.694598\" y=\"222.74254\" style=\"fill: #aec7e8; stroke: #aec7e8; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#mba7e5ca1c2\" x=\"180.507566\" y=\"209.473721\" style=\"fill: #aec7e8; stroke: #aec7e8; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#mba7e5ca1c2\" x=\"185.558546\" y=\"232.63077\" style=\"fill: #aec7e8; stroke: #aec7e8; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#mba7e5ca1c2\" x=\"116.533158\" y=\"266.470418\" style=\"fill: #aec7e8; stroke: #aec7e8; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#mba7e5ca1c2\" x=\"222.671836\" y=\"205.774417\" style=\"fill: #aec7e8; stroke: #aec7e8; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#mba7e5ca1c2\" x=\"191.651408\" y=\"204.385109\" style=\"fill: #aec7e8; stroke: #aec7e8; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#mba7e5ca1c2\" x=\"277.486992\" y=\"294.760012\" style=\"fill: #aec7e8; stroke: #aec7e8; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#mba7e5ca1c2\" x=\"256.98856\" y=\"282.820149\" style=\"fill: #aec7e8; stroke: #aec7e8; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#mba7e5ca1c2\" x=\"182.607541\" y=\"262.879218\" style=\"fill: #aec7e8; stroke: #aec7e8; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#mba7e5ca1c2\" x=\"192.538076\" y=\"211.68372\" style=\"fill: #aec7e8; stroke: #aec7e8; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#mba7e5ca1c2\" x=\"200.621247\" y=\"218.826544\" style=\"fill: #aec7e8; stroke: #aec7e8; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#mba7e5ca1c2\" x=\"265.296826\" y=\"295.253921\" style=\"fill: #aec7e8; stroke: #aec7e8; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#mba7e5ca1c2\" x=\"242.496463\" y=\"272.986943\" style=\"fill: #aec7e8; stroke: #aec7e8; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#mba7e5ca1c2\" x=\"187.342153\" y=\"227.859948\" style=\"fill: #aec7e8; stroke: #aec7e8; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#mba7e5ca1c2\" x=\"190.176583\" y=\"232.304247\" style=\"fill: #aec7e8; stroke: #aec7e8; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#mba7e5ca1c2\" x=\"185.138607\" y=\"221.062989\" style=\"fill: #aec7e8; stroke: #aec7e8; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#mba7e5ca1c2\" x=\"180.803298\" y=\"208.795056\" style=\"fill: #aec7e8; stroke: #aec7e8; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#mba7e5ca1c2\" x=\"242.687278\" y=\"229.071476\" style=\"fill: #aec7e8; stroke: #aec7e8; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#mba7e5ca1c2\" x=\"258.585106\" y=\"314.885816\" style=\"fill: #aec7e8; stroke: #aec7e8; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#mba7e5ca1c2\" x=\"184.882114\" y=\"236.591259\" style=\"fill: #aec7e8; stroke: #aec7e8; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#mba7e5ca1c2\" x=\"188.810669\" y=\"216.457479\" style=\"fill: #aec7e8; stroke: #aec7e8; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#mba7e5ca1c2\" x=\"192.258727\" y=\"217.60493\" style=\"fill: #aec7e8; stroke: #aec7e8; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#mba7e5ca1c2\" x=\"175.419144\" y=\"175.358711\" style=\"fill: #aec7e8; stroke: #aec7e8; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#mba7e5ca1c2\" x=\"220.734628\" y=\"217.756655\" style=\"fill: #aec7e8; stroke: #aec7e8; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#mba7e5ca1c2\" x=\"247.290579\" y=\"290.348604\" style=\"fill: #aec7e8; stroke: #aec7e8; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#mba7e5ca1c2\" x=\"202.03363\" y=\"212.227981\" style=\"fill: #aec7e8; stroke: #aec7e8; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#mba7e5ca1c2\" x=\"261.530152\" y=\"293.186182\" style=\"fill: #aec7e8; stroke: #aec7e8; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#mba7e5ca1c2\" x=\"244.008439\" y=\"304.368314\" style=\"fill: #aec7e8; stroke: #aec7e8; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#mba7e5ca1c2\" x=\"246.63468\" y=\"231.479025\" style=\"fill: #aec7e8; stroke: #aec7e8; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#mba7e5ca1c2\" x=\"232.745157\" y=\"278.351223\" style=\"fill: #aec7e8; stroke: #aec7e8; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#mba7e5ca1c2\" x=\"161.128671\" y=\"218.238112\" style=\"fill: #aec7e8; stroke: #aec7e8; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#mba7e5ca1c2\" x=\"254.766513\" y=\"284.122659\" style=\"fill: #aec7e8; stroke: #aec7e8; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#mba7e5ca1c2\" x=\"248.750061\" y=\"281.345843\" style=\"fill: #aec7e8; stroke: #aec7e8; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#mba7e5ca1c2\" x=\"220.18145\" y=\"207.376513\" style=\"fill: #aec7e8; stroke: #aec7e8; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#mba7e5ca1c2\" x=\"132.938413\" y=\"134.098336\" style=\"fill: #aec7e8; stroke: #aec7e8; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#mba7e5ca1c2\" x=\"194.679787\" y=\"242.320962\" style=\"fill: #aec7e8; stroke: #aec7e8; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#mba7e5ca1c2\" x=\"175.956421\" y=\"173.084916\" style=\"fill: #aec7e8; stroke: #aec7e8; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#mba7e5ca1c2\" x=\"206.035685\" y=\"211.301497\" style=\"fill: #aec7e8; stroke: #aec7e8; stroke-width: 0.5\"/>\n",
              "    </g>\n",
              "   </g>\n",
              "   <g id=\"PathCollection_3\">\n",
              "    <defs>\n",
              "     <path id=\"m8d03ac8010\" d=\"M 0 3 \n",
              "C 0.795609 3 1.55874 2.683901 2.12132 2.12132 \n",
              "C 2.683901 1.55874 3 0.795609 3 0 \n",
              "C 3 -0.795609 2.683901 -1.55874 2.12132 -2.12132 \n",
              "C 1.55874 -2.683901 0.795609 -3 0 -3 \n",
              "C -0.795609 -3 -1.55874 -2.683901 -2.12132 -2.12132 \n",
              "C -2.683901 -1.55874 -3 -0.795609 -3 0 \n",
              "C -3 0.795609 -2.683901 1.55874 -2.12132 2.12132 \n",
              "C -1.55874 2.683901 -0.795609 3 0 3 \n",
              "z\n",
              "\" style=\"stroke: #ff7f0e; stroke-width: 0.5\"/>\n",
              "    </defs>\n",
              "    <g clip-path=\"url(#p596f914704)\">\n",
              "     <use xlink:href=\"#m8d03ac8010\" x=\"351.111818\" y=\"222.311317\" style=\"fill: #ff7f0e; stroke: #ff7f0e; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m8d03ac8010\" x=\"377.686164\" y=\"52.256022\" style=\"fill: #ff7f0e; stroke: #ff7f0e; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m8d03ac8010\" x=\"375.442146\" y=\"54.884435\" style=\"fill: #ff7f0e; stroke: #ff7f0e; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m8d03ac8010\" x=\"369.125406\" y=\"65.766928\" style=\"fill: #ff7f0e; stroke: #ff7f0e; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m8d03ac8010\" x=\"370.130893\" y=\"70.685765\" style=\"fill: #ff7f0e; stroke: #ff7f0e; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m8d03ac8010\" x=\"388.44647\" y=\"212.598627\" style=\"fill: #ff7f0e; stroke: #ff7f0e; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m8d03ac8010\" x=\"390.376715\" y=\"47.596312\" style=\"fill: #ff7f0e; stroke: #ff7f0e; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m8d03ac8010\" x=\"390.821841\" y=\"201.907748\" style=\"fill: #ff7f0e; stroke: #ff7f0e; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m8d03ac8010\" x=\"374.037848\" y=\"51.236159\" style=\"fill: #ff7f0e; stroke: #ff7f0e; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m8d03ac8010\" x=\"365.082737\" y=\"44.130994\" style=\"fill: #ff7f0e; stroke: #ff7f0e; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m8d03ac8010\" x=\"398.931546\" y=\"191.529917\" style=\"fill: #ff7f0e; stroke: #ff7f0e; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m8d03ac8010\" x=\"372.609977\" y=\"58.671408\" style=\"fill: #ff7f0e; stroke: #ff7f0e; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m8d03ac8010\" x=\"367.108134\" y=\"221.879495\" style=\"fill: #ff7f0e; stroke: #ff7f0e; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m8d03ac8010\" x=\"365.819665\" y=\"214.051465\" style=\"fill: #ff7f0e; stroke: #ff7f0e; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m8d03ac8010\" x=\"366.111231\" y=\"199.845959\" style=\"fill: #ff7f0e; stroke: #ff7f0e; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m8d03ac8010\" x=\"376.847232\" y=\"51.844375\" style=\"fill: #ff7f0e; stroke: #ff7f0e; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m8d03ac8010\" x=\"380.275509\" y=\"216.783208\" style=\"fill: #ff7f0e; stroke: #ff7f0e; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m8d03ac8010\" x=\"364.933725\" y=\"43.938076\" style=\"fill: #ff7f0e; stroke: #ff7f0e; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m8d03ac8010\" x=\"390.37447\" y=\"47.587388\" style=\"fill: #ff7f0e; stroke: #ff7f0e; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m8d03ac8010\" x=\"376.696688\" y=\"215.306542\" style=\"fill: #ff7f0e; stroke: #ff7f0e; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m8d03ac8010\" x=\"351.566317\" y=\"222.173355\" style=\"fill: #ff7f0e; stroke: #ff7f0e; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m8d03ac8010\" x=\"367.314594\" y=\"46.193196\" style=\"fill: #ff7f0e; stroke: #ff7f0e; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m8d03ac8010\" x=\"206.452725\" y=\"141.966468\" style=\"fill: #ff7f0e; stroke: #ff7f0e; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m8d03ac8010\" x=\"394.833001\" y=\"201.348057\" style=\"fill: #ff7f0e; stroke: #ff7f0e; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m8d03ac8010\" x=\"388.002305\" y=\"73.634068\" style=\"fill: #ff7f0e; stroke: #ff7f0e; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m8d03ac8010\" x=\"382.767411\" y=\"59.158158\" style=\"fill: #ff7f0e; stroke: #ff7f0e; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m8d03ac8010\" x=\"352.315583\" y=\"222.038903\" style=\"fill: #ff7f0e; stroke: #ff7f0e; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m8d03ac8010\" x=\"369.224763\" y=\"70.4487\" style=\"fill: #ff7f0e; stroke: #ff7f0e; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m8d03ac8010\" x=\"379.477301\" y=\"210.253802\" style=\"fill: #ff7f0e; stroke: #ff7f0e; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m8d03ac8010\" x=\"205.849265\" y=\"143.11363\" style=\"fill: #ff7f0e; stroke: #ff7f0e; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m8d03ac8010\" x=\"384.735658\" y=\"204.68578\" style=\"fill: #ff7f0e; stroke: #ff7f0e; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m8d03ac8010\" x=\"400.587238\" y=\"190.443092\" style=\"fill: #ff7f0e; stroke: #ff7f0e; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m8d03ac8010\" x=\"422.352499\" y=\"195.933841\" style=\"fill: #ff7f0e; stroke: #ff7f0e; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m8d03ac8010\" x=\"361.891591\" y=\"42.234758\" style=\"fill: #ff7f0e; stroke: #ff7f0e; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m8d03ac8010\" x=\"201.067901\" y=\"146.224552\" style=\"fill: #ff7f0e; stroke: #ff7f0e; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m8d03ac8010\" x=\"416.559197\" y=\"132.443597\" style=\"fill: #ff7f0e; stroke: #ff7f0e; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m8d03ac8010\" x=\"387.309381\" y=\"68.929933\" style=\"fill: #ff7f0e; stroke: #ff7f0e; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m8d03ac8010\" x=\"369.484198\" y=\"212.032873\" style=\"fill: #ff7f0e; stroke: #ff7f0e; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m8d03ac8010\" x=\"379.114102\" y=\"210.961809\" style=\"fill: #ff7f0e; stroke: #ff7f0e; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m8d03ac8010\" x=\"410.075096\" y=\"165.656561\" style=\"fill: #ff7f0e; stroke: #ff7f0e; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m8d03ac8010\" x=\"383.817532\" y=\"60.567974\" style=\"fill: #ff7f0e; stroke: #ff7f0e; stroke-width: 0.5\"/>\n",
              "    </g>\n",
              "   </g>\n",
              "   <g id=\"PathCollection_4\">\n",
              "    <defs>\n",
              "     <path id=\"mba227bf8ff\" d=\"M 0 3 \n",
              "C 0.795609 3 1.55874 2.683901 2.12132 2.12132 \n",
              "C 2.683901 1.55874 3 0.795609 3 0 \n",
              "C 3 -0.795609 2.683901 -1.55874 2.12132 -2.12132 \n",
              "C 1.55874 -2.683901 0.795609 -3 0 -3 \n",
              "C -0.795609 -3 -1.55874 -2.683901 -2.12132 -2.12132 \n",
              "C -2.683901 -1.55874 -3 -0.795609 -3 0 \n",
              "C -3 0.795609 -2.683901 1.55874 -2.12132 2.12132 \n",
              "C -1.55874 2.683901 -0.795609 3 0 3 \n",
              "z\n",
              "\" style=\"stroke: #ffbb78; stroke-width: 0.5\"/>\n",
              "    </defs>\n",
              "    <g clip-path=\"url(#p596f914704)\">\n",
              "     <use xlink:href=\"#mba227bf8ff\" x=\"172.952725\" y=\"155.330314\" style=\"fill: #ffbb78; stroke: #ffbb78; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#mba227bf8ff\" x=\"159.448477\" y=\"200.599357\" style=\"fill: #ffbb78; stroke: #ffbb78; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#mba227bf8ff\" x=\"149.75636\" y=\"198.356852\" style=\"fill: #ffbb78; stroke: #ffbb78; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#mba227bf8ff\" x=\"211.482179\" y=\"177.693313\" style=\"fill: #ffbb78; stroke: #ffbb78; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#mba227bf8ff\" x=\"165.988246\" y=\"194.448063\" style=\"fill: #ffbb78; stroke: #ffbb78; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#mba227bf8ff\" x=\"132.289343\" y=\"234.896959\" style=\"fill: #ffbb78; stroke: #ffbb78; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#mba227bf8ff\" x=\"157.24386\" y=\"194.460379\" style=\"fill: #ffbb78; stroke: #ffbb78; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#mba227bf8ff\" x=\"178.120145\" y=\"279.881026\" style=\"fill: #ffbb78; stroke: #ffbb78; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#mba227bf8ff\" x=\"172.236614\" y=\"155.542786\" style=\"fill: #ffbb78; stroke: #ffbb78; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#mba227bf8ff\" x=\"150.76206\" y=\"150.50107\" style=\"fill: #ffbb78; stroke: #ffbb78; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#mba227bf8ff\" x=\"156.764394\" y=\"187.885231\" style=\"fill: #ffbb78; stroke: #ffbb78; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#mba227bf8ff\" x=\"155.091383\" y=\"202.123106\" style=\"fill: #ffbb78; stroke: #ffbb78; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#mba227bf8ff\" x=\"181.320066\" y=\"147.305636\" style=\"fill: #ffbb78; stroke: #ffbb78; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#mba227bf8ff\" x=\"163.008273\" y=\"198.209729\" style=\"fill: #ffbb78; stroke: #ffbb78; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#mba227bf8ff\" x=\"177.802297\" y=\"217.018919\" style=\"fill: #ffbb78; stroke: #ffbb78; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#mba227bf8ff\" x=\"346.757209\" y=\"190.912573\" style=\"fill: #ffbb78; stroke: #ffbb78; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#mba227bf8ff\" x=\"168.916315\" y=\"159.188017\" style=\"fill: #ffbb78; stroke: #ffbb78; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#mba227bf8ff\" x=\"172.39475\" y=\"104.834698\" style=\"fill: #ffbb78; stroke: #ffbb78; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#mba227bf8ff\" x=\"164.043698\" y=\"172.963326\" style=\"fill: #ffbb78; stroke: #ffbb78; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#mba227bf8ff\" x=\"148.412333\" y=\"215.593127\" style=\"fill: #ffbb78; stroke: #ffbb78; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#mba227bf8ff\" x=\"151.429072\" y=\"311.265146\" style=\"fill: #ffbb78; stroke: #ffbb78; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#mba227bf8ff\" x=\"146.469849\" y=\"219.807195\" style=\"fill: #ffbb78; stroke: #ffbb78; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#mba227bf8ff\" x=\"148.143821\" y=\"222.221566\" style=\"fill: #ffbb78; stroke: #ffbb78; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#mba227bf8ff\" x=\"143.94658\" y=\"220.587191\" style=\"fill: #ffbb78; stroke: #ffbb78; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#mba227bf8ff\" x=\"214.486158\" y=\"181.470223\" style=\"fill: #ffbb78; stroke: #ffbb78; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#mba227bf8ff\" x=\"226.454265\" y=\"115.35422\" style=\"fill: #ffbb78; stroke: #ffbb78; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#mba227bf8ff\" x=\"166.787214\" y=\"194.350211\" style=\"fill: #ffbb78; stroke: #ffbb78; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#mba227bf8ff\" x=\"149.102627\" y=\"310.563836\" style=\"fill: #ffbb78; stroke: #ffbb78; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#mba227bf8ff\" x=\"197.484729\" y=\"267.970647\" style=\"fill: #ffbb78; stroke: #ffbb78; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#mba227bf8ff\" x=\"305.439984\" y=\"93.304762\" style=\"fill: #ffbb78; stroke: #ffbb78; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#mba227bf8ff\" x=\"155.237674\" y=\"189.373417\" style=\"fill: #ffbb78; stroke: #ffbb78; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#mba227bf8ff\" x=\"170.106788\" y=\"188.642323\" style=\"fill: #ffbb78; stroke: #ffbb78; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#mba227bf8ff\" x=\"161.754554\" y=\"203.395681\" style=\"fill: #ffbb78; stroke: #ffbb78; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#mba227bf8ff\" x=\"130.353096\" y=\"235.455665\" style=\"fill: #ffbb78; stroke: #ffbb78; stroke-width: 0.5\"/>\n",
              "    </g>\n",
              "   </g>\n",
              "   <g id=\"PathCollection_5\">\n",
              "    <defs>\n",
              "     <path id=\"maf2cd97d8e\" d=\"M 0 3 \n",
              "C 0.795609 3 1.55874 2.683901 2.12132 2.12132 \n",
              "C 2.683901 1.55874 3 0.795609 3 0 \n",
              "C 3 -0.795609 2.683901 -1.55874 2.12132 -2.12132 \n",
              "C 1.55874 -2.683901 0.795609 -3 0 -3 \n",
              "C -0.795609 -3 -1.55874 -2.683901 -2.12132 -2.12132 \n",
              "C -2.683901 -1.55874 -3 -0.795609 -3 0 \n",
              "C -3 0.795609 -2.683901 1.55874 -2.12132 2.12132 \n",
              "C -1.55874 2.683901 -0.795609 3 0 3 \n",
              "z\n",
              "\" style=\"stroke: #2ca02c; stroke-width: 0.5\"/>\n",
              "    </defs>\n",
              "    <g clip-path=\"url(#p596f914704)\">\n",
              "     <use xlink:href=\"#maf2cd97d8e\" x=\"318.731275\" y=\"288.415425\" style=\"fill: #2ca02c; stroke: #2ca02c; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#maf2cd97d8e\" x=\"403.154731\" y=\"285.046897\" style=\"fill: #2ca02c; stroke: #2ca02c; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#maf2cd97d8e\" x=\"424.780545\" y=\"139.351479\" style=\"fill: #2ca02c; stroke: #2ca02c; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#maf2cd97d8e\" x=\"384.89563\" y=\"275.295224\" style=\"fill: #2ca02c; stroke: #2ca02c; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#maf2cd97d8e\" x=\"366.825311\" y=\"267.69289\" style=\"fill: #2ca02c; stroke: #2ca02c; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#maf2cd97d8e\" x=\"372.387162\" y=\"277.048593\" style=\"fill: #2ca02c; stroke: #2ca02c; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#maf2cd97d8e\" x=\"403.867763\" y=\"140.129252\" style=\"fill: #2ca02c; stroke: #2ca02c; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#maf2cd97d8e\" x=\"395.26432\" y=\"254.058581\" style=\"fill: #2ca02c; stroke: #2ca02c; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#maf2cd97d8e\" x=\"386.486879\" y=\"273.847804\" style=\"fill: #2ca02c; stroke: #2ca02c; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#maf2cd97d8e\" x=\"389.433701\" y=\"262.245965\" style=\"fill: #2ca02c; stroke: #2ca02c; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#maf2cd97d8e\" x=\"303.223001\" y=\"300.999516\" style=\"fill: #2ca02c; stroke: #2ca02c; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#maf2cd97d8e\" x=\"369.060967\" y=\"285.777961\" style=\"fill: #2ca02c; stroke: #2ca02c; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#maf2cd97d8e\" x=\"399.909814\" y=\"247.373837\" style=\"fill: #2ca02c; stroke: #2ca02c; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#maf2cd97d8e\" x=\"423.599948\" y=\"138.310695\" style=\"fill: #2ca02c; stroke: #2ca02c; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#maf2cd97d8e\" x=\"360.492995\" y=\"273.401001\" style=\"fill: #2ca02c; stroke: #2ca02c; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#maf2cd97d8e\" x=\"330.810191\" y=\"284.273691\" style=\"fill: #2ca02c; stroke: #2ca02c; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#maf2cd97d8e\" x=\"397.748177\" y=\"283.843685\" style=\"fill: #2ca02c; stroke: #2ca02c; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#maf2cd97d8e\" x=\"385.049749\" y=\"264.880251\" style=\"fill: #2ca02c; stroke: #2ca02c; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#maf2cd97d8e\" x=\"316.927738\" y=\"293.93203\" style=\"fill: #2ca02c; stroke: #2ca02c; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#maf2cd97d8e\" x=\"396.918626\" y=\"258.432813\" style=\"fill: #2ca02c; stroke: #2ca02c; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#maf2cd97d8e\" x=\"382.088545\" y=\"260.646483\" style=\"fill: #2ca02c; stroke: #2ca02c; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#maf2cd97d8e\" x=\"410.619926\" y=\"281.937166\" style=\"fill: #2ca02c; stroke: #2ca02c; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#maf2cd97d8e\" x=\"373.97111\" y=\"272.886675\" style=\"fill: #2ca02c; stroke: #2ca02c; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#maf2cd97d8e\" x=\"176.452143\" y=\"164.963548\" style=\"fill: #2ca02c; stroke: #2ca02c; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#maf2cd97d8e\" x=\"368.171526\" y=\"278.220112\" style=\"fill: #2ca02c; stroke: #2ca02c; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#maf2cd97d8e\" x=\"406.254893\" y=\"287.433098\" style=\"fill: #2ca02c; stroke: #2ca02c; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#maf2cd97d8e\" x=\"374.367857\" y=\"269.629472\" style=\"fill: #2ca02c; stroke: #2ca02c; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#maf2cd97d8e\" x=\"370.274132\" y=\"291.48309\" style=\"fill: #2ca02c; stroke: #2ca02c; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#maf2cd97d8e\" x=\"407.084901\" y=\"135.139056\" style=\"fill: #2ca02c; stroke: #2ca02c; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#maf2cd97d8e\" x=\"391.984079\" y=\"270.204463\" style=\"fill: #2ca02c; stroke: #2ca02c; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#maf2cd97d8e\" x=\"272.190633\" y=\"280.03012\" style=\"fill: #2ca02c; stroke: #2ca02c; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#maf2cd97d8e\" x=\"407.513613\" y=\"283.617565\" style=\"fill: #2ca02c; stroke: #2ca02c; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#maf2cd97d8e\" x=\"419.646316\" y=\"251.793853\" style=\"fill: #2ca02c; stroke: #2ca02c; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#maf2cd97d8e\" x=\"372.893702\" y=\"267.64963\" style=\"fill: #2ca02c; stroke: #2ca02c; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#maf2cd97d8e\" x=\"372.28812\" y=\"290.098258\" style=\"fill: #2ca02c; stroke: #2ca02c; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#maf2cd97d8e\" x=\"406.698341\" y=\"281.876132\" style=\"fill: #2ca02c; stroke: #2ca02c; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#maf2cd97d8e\" x=\"411.121713\" y=\"285.595875\" style=\"fill: #2ca02c; stroke: #2ca02c; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#maf2cd97d8e\" x=\"267.433125\" y=\"304.169784\" style=\"fill: #2ca02c; stroke: #2ca02c; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#maf2cd97d8e\" x=\"372.658336\" y=\"282.083188\" style=\"fill: #2ca02c; stroke: #2ca02c; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#maf2cd97d8e\" x=\"384.051909\" y=\"277.473694\" style=\"fill: #2ca02c; stroke: #2ca02c; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#maf2cd97d8e\" x=\"410.008142\" y=\"135.498481\" style=\"fill: #2ca02c; stroke: #2ca02c; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#maf2cd97d8e\" x=\"402.060188\" y=\"282.793121\" style=\"fill: #2ca02c; stroke: #2ca02c; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#maf2cd97d8e\" x=\"267.608203\" y=\"299.839399\" style=\"fill: #2ca02c; stroke: #2ca02c; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#maf2cd97d8e\" x=\"369.461526\" y=\"253.701101\" style=\"fill: #2ca02c; stroke: #2ca02c; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#maf2cd97d8e\" x=\"368.630888\" y=\"270.958461\" style=\"fill: #2ca02c; stroke: #2ca02c; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#maf2cd97d8e\" x=\"426.319072\" y=\"276.776763\" style=\"fill: #2ca02c; stroke: #2ca02c; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#maf2cd97d8e\" x=\"375.320252\" y=\"262.147019\" style=\"fill: #2ca02c; stroke: #2ca02c; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#maf2cd97d8e\" x=\"402.708124\" y=\"274.313205\" style=\"fill: #2ca02c; stroke: #2ca02c; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#maf2cd97d8e\" x=\"370.483373\" y=\"292.353078\" style=\"fill: #2ca02c; stroke: #2ca02c; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#maf2cd97d8e\" x=\"392.710965\" y=\"282.5094\" style=\"fill: #2ca02c; stroke: #2ca02c; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#maf2cd97d8e\" x=\"167.199843\" y=\"147.665422\" style=\"fill: #2ca02c; stroke: #2ca02c; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#maf2cd97d8e\" x=\"403.665812\" y=\"281.488327\" style=\"fill: #2ca02c; stroke: #2ca02c; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#maf2cd97d8e\" x=\"379.208532\" y=\"266.373771\" style=\"fill: #2ca02c; stroke: #2ca02c; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#maf2cd97d8e\" x=\"372.328642\" y=\"272.413146\" style=\"fill: #2ca02c; stroke: #2ca02c; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#maf2cd97d8e\" x=\"241.21947\" y=\"330.664807\" style=\"fill: #2ca02c; stroke: #2ca02c; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#maf2cd97d8e\" x=\"115.135432\" y=\"264.563906\" style=\"fill: #2ca02c; stroke: #2ca02c; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#maf2cd97d8e\" x=\"411.956525\" y=\"136.024101\" style=\"fill: #2ca02c; stroke: #2ca02c; stroke-width: 0.5\"/>\n",
              "    </g>\n",
              "   </g>\n",
              "   <g id=\"PathCollection_6\">\n",
              "    <defs>\n",
              "     <path id=\"m62c8c1ad36\" d=\"M 0 3 \n",
              "C 0.795609 3 1.55874 2.683901 2.12132 2.12132 \n",
              "C 2.683901 1.55874 3 0.795609 3 0 \n",
              "C 3 -0.795609 2.683901 -1.55874 2.12132 -2.12132 \n",
              "C 1.55874 -2.683901 0.795609 -3 0 -3 \n",
              "C -0.795609 -3 -1.55874 -2.683901 -2.12132 -2.12132 \n",
              "C -2.683901 -1.55874 -3 -0.795609 -3 0 \n",
              "C -3 0.795609 -2.683901 1.55874 -2.12132 2.12132 \n",
              "C -1.55874 2.683901 -0.795609 3 0 3 \n",
              "z\n",
              "\" style=\"stroke: #98df8a; stroke-width: 0.5\"/>\n",
              "    </defs>\n",
              "    <g clip-path=\"url(#p596f914704)\">\n",
              "     <use xlink:href=\"#m62c8c1ad36\" x=\"173.598692\" y=\"106.992735\" style=\"fill: #98df8a; stroke: #98df8a; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m62c8c1ad36\" x=\"103.441224\" y=\"192.555107\" style=\"fill: #98df8a; stroke: #98df8a; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m62c8c1ad36\" x=\"563.564232\" y=\"212.508457\" style=\"fill: #98df8a; stroke: #98df8a; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m62c8c1ad36\" x=\"207.307259\" y=\"81.145464\" style=\"fill: #98df8a; stroke: #98df8a; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m62c8c1ad36\" x=\"169.031873\" y=\"124.40438\" style=\"fill: #98df8a; stroke: #98df8a; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m62c8c1ad36\" x=\"174.092571\" y=\"111.513387\" style=\"fill: #98df8a; stroke: #98df8a; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m62c8c1ad36\" x=\"152.774925\" y=\"114.510486\" style=\"fill: #98df8a; stroke: #98df8a; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m62c8c1ad36\" x=\"233.438727\" y=\"91.48773\" style=\"fill: #98df8a; stroke: #98df8a; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m62c8c1ad36\" x=\"237.996457\" y=\"88.758044\" style=\"fill: #98df8a; stroke: #98df8a; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m62c8c1ad36\" x=\"196.662369\" y=\"101.256763\" style=\"fill: #98df8a; stroke: #98df8a; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m62c8c1ad36\" x=\"173.819727\" y=\"107.642649\" style=\"fill: #98df8a; stroke: #98df8a; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m62c8c1ad36\" x=\"101.32986\" y=\"188.874581\" style=\"fill: #98df8a; stroke: #98df8a; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m62c8c1ad36\" x=\"168.888606\" y=\"124.450735\" style=\"fill: #98df8a; stroke: #98df8a; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m62c8c1ad36\" x=\"221.325179\" y=\"70.079728\" style=\"fill: #98df8a; stroke: #98df8a; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m62c8c1ad36\" x=\"197.011138\" y=\"103.418782\" style=\"fill: #98df8a; stroke: #98df8a; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m62c8c1ad36\" x=\"563.524599\" y=\"209.629515\" style=\"fill: #98df8a; stroke: #98df8a; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m62c8c1ad36\" x=\"564.592458\" y=\"207.488448\" style=\"fill: #98df8a; stroke: #98df8a; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m62c8c1ad36\" x=\"230.935511\" y=\"92.987485\" style=\"fill: #98df8a; stroke: #98df8a; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m62c8c1ad36\" x=\"193.839713\" y=\"171.637793\" style=\"fill: #98df8a; stroke: #98df8a; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m62c8c1ad36\" x=\"221.674508\" y=\"69.905033\" style=\"fill: #98df8a; stroke: #98df8a; stroke-width: 0.5\"/>\n",
              "    </g>\n",
              "   </g>\n",
              "   <g id=\"PathCollection_7\">\n",
              "    <defs>\n",
              "     <path id=\"m92d6cab3c5\" d=\"M 0 3 \n",
              "C 0.795609 3 1.55874 2.683901 2.12132 2.12132 \n",
              "C 2.683901 1.55874 3 0.795609 3 0 \n",
              "C 3 -0.795609 2.683901 -1.55874 2.12132 -2.12132 \n",
              "C 1.55874 -2.683901 0.795609 -3 0 -3 \n",
              "C -0.795609 -3 -1.55874 -2.683901 -2.12132 -2.12132 \n",
              "C -2.683901 -1.55874 -3 -0.795609 -3 0 \n",
              "C -3 0.795609 -2.683901 1.55874 -2.12132 2.12132 \n",
              "C -1.55874 2.683901 -0.795609 3 0 3 \n",
              "z\n",
              "\" style=\"stroke: #d62728; stroke-width: 0.5\"/>\n",
              "    </defs>\n",
              "    <g clip-path=\"url(#p596f914704)\">\n",
              "     <use xlink:href=\"#m92d6cab3c5\" x=\"437.578858\" y=\"183.392795\" style=\"fill: #d62728; stroke: #d62728; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m92d6cab3c5\" x=\"435.85926\" y=\"192.472786\" style=\"fill: #d62728; stroke: #d62728; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m92d6cab3c5\" x=\"463.028853\" y=\"219.766438\" style=\"fill: #d62728; stroke: #d62728; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m92d6cab3c5\" x=\"440.097574\" y=\"252.309464\" style=\"fill: #d62728; stroke: #d62728; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m92d6cab3c5\" x=\"419.730362\" y=\"244.732189\" style=\"fill: #d62728; stroke: #d62728; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m92d6cab3c5\" x=\"392.43019\" y=\"240.745157\" style=\"fill: #d62728; stroke: #d62728; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m92d6cab3c5\" x=\"441.54866\" y=\"196.549274\" style=\"fill: #d62728; stroke: #d62728; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m92d6cab3c5\" x=\"483.520642\" y=\"194.671122\" style=\"fill: #d62728; stroke: #d62728; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m92d6cab3c5\" x=\"427.357734\" y=\"219.799792\" style=\"fill: #d62728; stroke: #d62728; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m92d6cab3c5\" x=\"473.493491\" y=\"194.108738\" style=\"fill: #d62728; stroke: #d62728; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m92d6cab3c5\" x=\"417.810301\" y=\"181.648385\" style=\"fill: #d62728; stroke: #d62728; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m92d6cab3c5\" x=\"479.381685\" y=\"163.323099\" style=\"fill: #d62728; stroke: #d62728; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m92d6cab3c5\" x=\"483.485168\" y=\"245.418611\" style=\"fill: #d62728; stroke: #d62728; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m92d6cab3c5\" x=\"498.676384\" y=\"180.985171\" style=\"fill: #d62728; stroke: #d62728; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m92d6cab3c5\" x=\"424.099845\" y=\"226.248883\" style=\"fill: #d62728; stroke: #d62728; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m92d6cab3c5\" x=\"421.841221\" y=\"223.126202\" style=\"fill: #d62728; stroke: #d62728; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m92d6cab3c5\" x=\"440.275133\" y=\"252.222252\" style=\"fill: #d62728; stroke: #d62728; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m92d6cab3c5\" x=\"434.327766\" y=\"242.108506\" style=\"fill: #d62728; stroke: #d62728; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m92d6cab3c5\" x=\"440.316404\" y=\"183.311383\" style=\"fill: #d62728; stroke: #d62728; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m92d6cab3c5\" x=\"474.350207\" y=\"174.860484\" style=\"fill: #d62728; stroke: #d62728; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m92d6cab3c5\" x=\"452.326893\" y=\"184.938226\" style=\"fill: #d62728; stroke: #d62728; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m92d6cab3c5\" x=\"427.710056\" y=\"216.120627\" style=\"fill: #d62728; stroke: #d62728; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m92d6cab3c5\" x=\"451.593186\" y=\"234.549079\" style=\"fill: #d62728; stroke: #d62728; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m92d6cab3c5\" x=\"438.179175\" y=\"227.610414\" style=\"fill: #d62728; stroke: #d62728; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m92d6cab3c5\" x=\"438.354568\" y=\"215.510683\" style=\"fill: #d62728; stroke: #d62728; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m92d6cab3c5\" x=\"439.111218\" y=\"179.282802\" style=\"fill: #d62728; stroke: #d62728; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m92d6cab3c5\" x=\"433.716446\" y=\"144.587454\" style=\"fill: #d62728; stroke: #d62728; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m92d6cab3c5\" x=\"500.299508\" y=\"186.433578\" style=\"fill: #d62728; stroke: #d62728; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m92d6cab3c5\" x=\"405.971542\" y=\"228.235336\" style=\"fill: #d62728; stroke: #d62728; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m92d6cab3c5\" x=\"417.320414\" y=\"183.507377\" style=\"fill: #d62728; stroke: #d62728; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m92d6cab3c5\" x=\"427.922482\" y=\"239.378955\" style=\"fill: #d62728; stroke: #d62728; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m92d6cab3c5\" x=\"412.340367\" y=\"243.542747\" style=\"fill: #d62728; stroke: #d62728; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m92d6cab3c5\" x=\"432.696672\" y=\"221.73284\" style=\"fill: #d62728; stroke: #d62728; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m92d6cab3c5\" x=\"416.582085\" y=\"218.271689\" style=\"fill: #d62728; stroke: #d62728; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m92d6cab3c5\" x=\"433.194347\" y=\"225.719383\" style=\"fill: #d62728; stroke: #d62728; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m92d6cab3c5\" x=\"415.074842\" y=\"224.046845\" style=\"fill: #d62728; stroke: #d62728; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m92d6cab3c5\" x=\"437.398353\" y=\"201.020963\" style=\"fill: #d62728; stroke: #d62728; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m92d6cab3c5\" x=\"418.372103\" y=\"231.727002\" style=\"fill: #d62728; stroke: #d62728; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m92d6cab3c5\" x=\"466.259129\" y=\"172.634642\" style=\"fill: #d62728; stroke: #d62728; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m92d6cab3c5\" x=\"450.608437\" y=\"222.801496\" style=\"fill: #d62728; stroke: #d62728; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m92d6cab3c5\" x=\"422.528428\" y=\"200.292347\" style=\"fill: #d62728; stroke: #d62728; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m92d6cab3c5\" x=\"446.59923\" y=\"214.477144\" style=\"fill: #d62728; stroke: #d62728; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m92d6cab3c5\" x=\"433.150682\" y=\"216.629294\" style=\"fill: #d62728; stroke: #d62728; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m92d6cab3c5\" x=\"417.367986\" y=\"279.643818\" style=\"fill: #d62728; stroke: #d62728; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m92d6cab3c5\" x=\"400.054514\" y=\"203.994417\" style=\"fill: #d62728; stroke: #d62728; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m92d6cab3c5\" x=\"528.538536\" y=\"180.479674\" style=\"fill: #d62728; stroke: #d62728; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m92d6cab3c5\" x=\"421.374121\" y=\"213.778061\" style=\"fill: #d62728; stroke: #d62728; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m92d6cab3c5\" x=\"445.861782\" y=\"204.438493\" style=\"fill: #d62728; stroke: #d62728; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m92d6cab3c5\" x=\"452.791662\" y=\"177.911656\" style=\"fill: #d62728; stroke: #d62728; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m92d6cab3c5\" x=\"470.730363\" y=\"156.401269\" style=\"fill: #d62728; stroke: #d62728; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m92d6cab3c5\" x=\"429.663835\" y=\"275.668704\" style=\"fill: #d62728; stroke: #d62728; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m92d6cab3c5\" x=\"441.014211\" y=\"186.30931\" style=\"fill: #d62728; stroke: #d62728; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m92d6cab3c5\" x=\"414.547623\" y=\"228.803229\" style=\"fill: #d62728; stroke: #d62728; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m92d6cab3c5\" x=\"450.459499\" y=\"222.598535\" style=\"fill: #d62728; stroke: #d62728; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m92d6cab3c5\" x=\"442.353142\" y=\"178.467229\" style=\"fill: #d62728; stroke: #d62728; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m92d6cab3c5\" x=\"458.927796\" y=\"186.140791\" style=\"fill: #d62728; stroke: #d62728; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m92d6cab3c5\" x=\"470.559704\" y=\"161.491344\" style=\"fill: #d62728; stroke: #d62728; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m92d6cab3c5\" x=\"421.584941\" y=\"211.735984\" style=\"fill: #d62728; stroke: #d62728; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m92d6cab3c5\" x=\"451.934631\" y=\"162.645096\" style=\"fill: #d62728; stroke: #d62728; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m92d6cab3c5\" x=\"451.994805\" y=\"234.668957\" style=\"fill: #d62728; stroke: #d62728; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m92d6cab3c5\" x=\"437.000405\" y=\"238.333589\" style=\"fill: #d62728; stroke: #d62728; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m92d6cab3c5\" x=\"453.901185\" y=\"208.605726\" style=\"fill: #d62728; stroke: #d62728; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m92d6cab3c5\" x=\"451.175705\" y=\"175.317547\" style=\"fill: #d62728; stroke: #d62728; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m92d6cab3c5\" x=\"478.626681\" y=\"245.607428\" style=\"fill: #d62728; stroke: #d62728; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m92d6cab3c5\" x=\"423.921317\" y=\"189.917633\" style=\"fill: #d62728; stroke: #d62728; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m92d6cab3c5\" x=\"359.32409\" y=\"238.873038\" style=\"fill: #d62728; stroke: #d62728; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m92d6cab3c5\" x=\"408.723753\" y=\"231.35499\" style=\"fill: #d62728; stroke: #d62728; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m92d6cab3c5\" x=\"487.23418\" y=\"181.877071\" style=\"fill: #d62728; stroke: #d62728; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m92d6cab3c5\" x=\"431.221516\" y=\"230.078265\" style=\"fill: #d62728; stroke: #d62728; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m92d6cab3c5\" x=\"436.37709\" y=\"237.229751\" style=\"fill: #d62728; stroke: #d62728; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m92d6cab3c5\" x=\"435.358741\" y=\"224.436753\" style=\"fill: #d62728; stroke: #d62728; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m92d6cab3c5\" x=\"408.111016\" y=\"227.785615\" style=\"fill: #d62728; stroke: #d62728; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m92d6cab3c5\" x=\"430.111835\" y=\"191.359242\" style=\"fill: #d62728; stroke: #d62728; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m92d6cab3c5\" x=\"434.874604\" y=\"178.775325\" style=\"fill: #d62728; stroke: #d62728; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m92d6cab3c5\" x=\"411.896588\" y=\"212.108502\" style=\"fill: #d62728; stroke: #d62728; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m92d6cab3c5\" x=\"403.488056\" y=\"256.506086\" style=\"fill: #d62728; stroke: #d62728; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m92d6cab3c5\" x=\"438.484571\" y=\"220.748526\" style=\"fill: #d62728; stroke: #d62728; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m92d6cab3c5\" x=\"428.203644\" y=\"177.714869\" style=\"fill: #d62728; stroke: #d62728; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m92d6cab3c5\" x=\"418.491111\" y=\"285.539401\" style=\"fill: #d62728; stroke: #d62728; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m92d6cab3c5\" x=\"388.866275\" y=\"229.959368\" style=\"fill: #d62728; stroke: #d62728; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m92d6cab3c5\" x=\"428.515845\" y=\"205.962536\" style=\"fill: #d62728; stroke: #d62728; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m92d6cab3c5\" x=\"415.362824\" y=\"198.454859\" style=\"fill: #d62728; stroke: #d62728; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m92d6cab3c5\" x=\"416.620748\" y=\"241.631596\" style=\"fill: #d62728; stroke: #d62728; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m92d6cab3c5\" x=\"438.968384\" y=\"173.774636\" style=\"fill: #d62728; stroke: #d62728; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m92d6cab3c5\" x=\"408.609274\" y=\"218.430314\" style=\"fill: #d62728; stroke: #d62728; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m92d6cab3c5\" x=\"424.010128\" y=\"220.113716\" style=\"fill: #d62728; stroke: #d62728; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m92d6cab3c5\" x=\"441.209421\" y=\"229.732308\" style=\"fill: #d62728; stroke: #d62728; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m92d6cab3c5\" x=\"421.848152\" y=\"231.617697\" style=\"fill: #d62728; stroke: #d62728; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m92d6cab3c5\" x=\"480.783112\" y=\"162.88359\" style=\"fill: #d62728; stroke: #d62728; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m92d6cab3c5\" x=\"429.223111\" y=\"212.593303\" style=\"fill: #d62728; stroke: #d62728; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m92d6cab3c5\" x=\"444.868873\" y=\"223.599235\" style=\"fill: #d62728; stroke: #d62728; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m92d6cab3c5\" x=\"441.660344\" y=\"254.916452\" style=\"fill: #d62728; stroke: #d62728; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m92d6cab3c5\" x=\"476.219695\" y=\"247.509177\" style=\"fill: #d62728; stroke: #d62728; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m92d6cab3c5\" x=\"403.448045\" y=\"242.836175\" style=\"fill: #d62728; stroke: #d62728; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m92d6cab3c5\" x=\"478.441341\" y=\"244.637015\" style=\"fill: #d62728; stroke: #d62728; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m92d6cab3c5\" x=\"427.600594\" y=\"276.346765\" style=\"fill: #d62728; stroke: #d62728; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m92d6cab3c5\" x=\"444.636811\" y=\"183.144274\" style=\"fill: #d62728; stroke: #d62728; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m92d6cab3c5\" x=\"509.150662\" y=\"160.762592\" style=\"fill: #d62728; stroke: #d62728; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m92d6cab3c5\" x=\"468.356457\" y=\"183.418334\" style=\"fill: #d62728; stroke: #d62728; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m92d6cab3c5\" x=\"433.745517\" y=\"234.637467\" style=\"fill: #d62728; stroke: #d62728; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m92d6cab3c5\" x=\"422.702064\" y=\"200.419654\" style=\"fill: #d62728; stroke: #d62728; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m92d6cab3c5\" x=\"475.739757\" y=\"182.629331\" style=\"fill: #d62728; stroke: #d62728; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m92d6cab3c5\" x=\"452.827939\" y=\"199.658765\" style=\"fill: #d62728; stroke: #d62728; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m92d6cab3c5\" x=\"402.816318\" y=\"210.696734\" style=\"fill: #d62728; stroke: #d62728; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m92d6cab3c5\" x=\"431.402666\" y=\"191.537463\" style=\"fill: #d62728; stroke: #d62728; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m92d6cab3c5\" x=\"422.253764\" y=\"221.221556\" style=\"fill: #d62728; stroke: #d62728; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m92d6cab3c5\" x=\"410.119242\" y=\"213.090541\" style=\"fill: #d62728; stroke: #d62728; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m92d6cab3c5\" x=\"419.352986\" y=\"176.706399\" style=\"fill: #d62728; stroke: #d62728; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m92d6cab3c5\" x=\"416.283541\" y=\"236.193264\" style=\"fill: #d62728; stroke: #d62728; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m92d6cab3c5\" x=\"470.898345\" y=\"181.808128\" style=\"fill: #d62728; stroke: #d62728; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m92d6cab3c5\" x=\"431.182624\" y=\"184.889039\" style=\"fill: #d62728; stroke: #d62728; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m92d6cab3c5\" x=\"445.209767\" y=\"249.363907\" style=\"fill: #d62728; stroke: #d62728; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m92d6cab3c5\" x=\"425.593131\" y=\"233.759964\" style=\"fill: #d62728; stroke: #d62728; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m92d6cab3c5\" x=\"439.501959\" y=\"255.848607\" style=\"fill: #d62728; stroke: #d62728; stroke-width: 0.5\"/>\n",
              "    </g>\n",
              "   </g>\n",
              "   <g id=\"PathCollection_8\">\n",
              "    <defs>\n",
              "     <path id=\"mefe40797ef\" d=\"M 0 3 \n",
              "C 0.795609 3 1.55874 2.683901 2.12132 2.12132 \n",
              "C 2.683901 1.55874 3 0.795609 3 0 \n",
              "C 3 -0.795609 2.683901 -1.55874 2.12132 -2.12132 \n",
              "C 1.55874 -2.683901 0.795609 -3 0 -3 \n",
              "C -0.795609 -3 -1.55874 -2.683901 -2.12132 -2.12132 \n",
              "C -2.683901 -1.55874 -3 -0.795609 -3 0 \n",
              "C -3 0.795609 -2.683901 1.55874 -2.12132 2.12132 \n",
              "C -1.55874 2.683901 -0.795609 3 0 3 \n",
              "z\n",
              "\" style=\"stroke: #ff9896; stroke-width: 0.5\"/>\n",
              "    </defs>\n",
              "    <g clip-path=\"url(#p596f914704)\">\n",
              "     <use xlink:href=\"#mefe40797ef\" x=\"99.317515\" y=\"172.323532\" style=\"fill: #ff9896; stroke: #ff9896; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#mefe40797ef\" x=\"86.659381\" y=\"162.044332\" style=\"fill: #ff9896; stroke: #ff9896; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#mefe40797ef\" x=\"87.248293\" y=\"175.810727\" style=\"fill: #ff9896; stroke: #ff9896; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#mefe40797ef\" x=\"85.454541\" y=\"167.890249\" style=\"fill: #ff9896; stroke: #ff9896; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#mefe40797ef\" x=\"81.292003\" y=\"169.499718\" style=\"fill: #ff9896; stroke: #ff9896; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#mefe40797ef\" x=\"78.505814\" y=\"170.322187\" style=\"fill: #ff9896; stroke: #ff9896; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#mefe40797ef\" x=\"87.43031\" y=\"161.182864\" style=\"fill: #ff9896; stroke: #ff9896; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#mefe40797ef\" x=\"81.416193\" y=\"176.560136\" style=\"fill: #ff9896; stroke: #ff9896; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#mefe40797ef\" x=\"76.545971\" y=\"173.287448\" style=\"fill: #ff9896; stroke: #ff9896; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#mefe40797ef\" x=\"99.265091\" y=\"172.395255\" style=\"fill: #ff9896; stroke: #ff9896; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#mefe40797ef\" x=\"81.261633\" y=\"173.069573\" style=\"fill: #ff9896; stroke: #ff9896; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#mefe40797ef\" x=\"88.824964\" y=\"159.578526\" style=\"fill: #ff9896; stroke: #ff9896; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#mefe40797ef\" x=\"76.885683\" y=\"163.039327\" style=\"fill: #ff9896; stroke: #ff9896; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#mefe40797ef\" x=\"77.96964\" y=\"185.045176\" style=\"fill: #ff9896; stroke: #ff9896; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#mefe40797ef\" x=\"214.488497\" y=\"314.853958\" style=\"fill: #ff9896; stroke: #ff9896; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#mefe40797ef\" x=\"78.029357\" y=\"167.149783\" style=\"fill: #ff9896; stroke: #ff9896; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#mefe40797ef\" x=\"82.587481\" y=\"173.481123\" style=\"fill: #ff9896; stroke: #ff9896; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#mefe40797ef\" x=\"81.053451\" y=\"167.673424\" style=\"fill: #ff9896; stroke: #ff9896; stroke-width: 0.5\"/>\n",
              "    </g>\n",
              "   </g>\n",
              "   <g id=\"PathCollection_9\">\n",
              "    <defs>\n",
              "     <path id=\"mc83ea64be4\" d=\"M 0 3 \n",
              "C 0.795609 3 1.55874 2.683901 2.12132 2.12132 \n",
              "C 2.683901 1.55874 3 0.795609 3 0 \n",
              "C 3 -0.795609 2.683901 -1.55874 2.12132 -2.12132 \n",
              "C 1.55874 -2.683901 0.795609 -3 0 -3 \n",
              "C -0.795609 -3 -1.55874 -2.683901 -2.12132 -2.12132 \n",
              "C -2.683901 -1.55874 -3 -0.795609 -3 0 \n",
              "C -3 0.795609 -2.683901 1.55874 -2.12132 2.12132 \n",
              "C -1.55874 2.683901 -0.795609 3 0 3 \n",
              "z\n",
              "\" style=\"stroke: #9467bd; stroke-width: 0.5\"/>\n",
              "    </defs>\n",
              "    <g clip-path=\"url(#p596f914704)\">\n",
              "     <use xlink:href=\"#mc83ea64be4\" x=\"287.361424\" y=\"193.792471\" style=\"fill: #9467bd; stroke: #9467bd; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#mc83ea64be4\" x=\"295.428958\" y=\"180.236895\" style=\"fill: #9467bd; stroke: #9467bd; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#mc83ea64be4\" x=\"294.607547\" y=\"192.007165\" style=\"fill: #9467bd; stroke: #9467bd; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#mc83ea64be4\" x=\"277.536029\" y=\"212.970771\" style=\"fill: #9467bd; stroke: #9467bd; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#mc83ea64be4\" x=\"316.82339\" y=\"178.6083\" style=\"fill: #9467bd; stroke: #9467bd; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#mc83ea64be4\" x=\"271.043832\" y=\"205.17044\" style=\"fill: #9467bd; stroke: #9467bd; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#mc83ea64be4\" x=\"279.654383\" y=\"180.933761\" style=\"fill: #9467bd; stroke: #9467bd; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#mc83ea64be4\" x=\"261.034442\" y=\"195.654968\" style=\"fill: #9467bd; stroke: #9467bd; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#mc83ea64be4\" x=\"305.935506\" y=\"169.317951\" style=\"fill: #9467bd; stroke: #9467bd; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#mc83ea64be4\" x=\"270.396928\" y=\"211.785702\" style=\"fill: #9467bd; stroke: #9467bd; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#mc83ea64be4\" x=\"324.923602\" y=\"164.667688\" style=\"fill: #9467bd; stroke: #9467bd; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#mc83ea64be4\" x=\"275.319318\" y=\"199.125508\" style=\"fill: #9467bd; stroke: #9467bd; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#mc83ea64be4\" x=\"279.073079\" y=\"206.912936\" style=\"fill: #9467bd; stroke: #9467bd; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#mc83ea64be4\" x=\"277.405652\" y=\"206.467024\" style=\"fill: #9467bd; stroke: #9467bd; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#mc83ea64be4\" x=\"301.311002\" y=\"199.39821\" style=\"fill: #9467bd; stroke: #9467bd; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#mc83ea64be4\" x=\"315.567959\" y=\"186.832061\" style=\"fill: #9467bd; stroke: #9467bd; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#mc83ea64be4\" x=\"282.339273\" y=\"209.475444\" style=\"fill: #9467bd; stroke: #9467bd; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#mc83ea64be4\" x=\"276.854549\" y=\"219.905795\" style=\"fill: #9467bd; stroke: #9467bd; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#mc83ea64be4\" x=\"264.629015\" y=\"207.902876\" style=\"fill: #9467bd; stroke: #9467bd; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#mc83ea64be4\" x=\"270.218046\" y=\"223.118036\" style=\"fill: #9467bd; stroke: #9467bd; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#mc83ea64be4\" x=\"305.97977\" y=\"190.259441\" style=\"fill: #9467bd; stroke: #9467bd; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#mc83ea64be4\" x=\"280.155201\" y=\"184.069329\" style=\"fill: #9467bd; stroke: #9467bd; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#mc83ea64be4\" x=\"267.36202\" y=\"230.626826\" style=\"fill: #9467bd; stroke: #9467bd; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#mc83ea64be4\" x=\"315.709802\" y=\"186.925182\" style=\"fill: #9467bd; stroke: #9467bd; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#mc83ea64be4\" x=\"307.969268\" y=\"207.030641\" style=\"fill: #9467bd; stroke: #9467bd; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#mc83ea64be4\" x=\"300.235032\" y=\"179.49924\" style=\"fill: #9467bd; stroke: #9467bd; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#mc83ea64be4\" x=\"288.040425\" y=\"187.176315\" style=\"fill: #9467bd; stroke: #9467bd; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#mc83ea64be4\" x=\"290.029542\" y=\"166.067589\" style=\"fill: #9467bd; stroke: #9467bd; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#mc83ea64be4\" x=\"283.858697\" y=\"216.564638\" style=\"fill: #9467bd; stroke: #9467bd; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#mc83ea64be4\" x=\"279.039027\" y=\"203.483574\" style=\"fill: #9467bd; stroke: #9467bd; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#mc83ea64be4\" x=\"290.366317\" y=\"196.125785\" style=\"fill: #9467bd; stroke: #9467bd; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#mc83ea64be4\" x=\"297.615866\" y=\"189.185308\" style=\"fill: #9467bd; stroke: #9467bd; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#mc83ea64be4\" x=\"299.291916\" y=\"179.800815\" style=\"fill: #9467bd; stroke: #9467bd; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#mc83ea64be4\" x=\"325.857979\" y=\"164.466928\" style=\"fill: #9467bd; stroke: #9467bd; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#mc83ea64be4\" x=\"289.575652\" y=\"190.97271\" style=\"fill: #9467bd; stroke: #9467bd; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#mc83ea64be4\" x=\"283.729024\" y=\"207.545033\" style=\"fill: #9467bd; stroke: #9467bd; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#mc83ea64be4\" x=\"287.706179\" y=\"198.587035\" style=\"fill: #9467bd; stroke: #9467bd; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#mc83ea64be4\" x=\"265.89289\" y=\"195.236784\" style=\"fill: #9467bd; stroke: #9467bd; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#mc83ea64be4\" x=\"282.426336\" y=\"194.870493\" style=\"fill: #9467bd; stroke: #9467bd; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#mc83ea64be4\" x=\"349.651096\" y=\"155.676778\" style=\"fill: #9467bd; stroke: #9467bd; stroke-width: 0.5\"/>\n",
              "    </g>\n",
              "   </g>\n",
              "   <g id=\"PathCollection_10\">\n",
              "    <defs>\n",
              "     <path id=\"mca817ee18d\" d=\"M 0 3 \n",
              "C 0.795609 3 1.55874 2.683901 2.12132 2.12132 \n",
              "C 2.683901 1.55874 3 0.795609 3 0 \n",
              "C 3 -0.795609 2.683901 -1.55874 2.12132 -2.12132 \n",
              "C 1.55874 -2.683901 0.795609 -3 0 -3 \n",
              "C -0.795609 -3 -1.55874 -2.683901 -2.12132 -2.12132 \n",
              "C -2.683901 -1.55874 -3 -0.795609 -3 0 \n",
              "C -3 0.795609 -2.683901 1.55874 -2.12132 2.12132 \n",
              "C -1.55874 2.683901 -0.795609 3 0 3 \n",
              "z\n",
              "\" style=\"stroke: #c5b0d5; stroke-width: 0.5\"/>\n",
              "    </defs>\n",
              "    <g clip-path=\"url(#p596f914704)\">\n",
              "     <use xlink:href=\"#mca817ee18d\" x=\"167.188297\" y=\"147.471155\" style=\"fill: #c5b0d5; stroke: #c5b0d5; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#mca817ee18d\" x=\"229.784567\" y=\"118.790144\" style=\"fill: #c5b0d5; stroke: #c5b0d5; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#mca817ee18d\" x=\"264.916382\" y=\"277.633425\" style=\"fill: #c5b0d5; stroke: #c5b0d5; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#mca817ee18d\" x=\"257.920284\" y=\"256.603582\" style=\"fill: #c5b0d5; stroke: #c5b0d5; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#mca817ee18d\" x=\"226.76894\" y=\"127.694293\" style=\"fill: #c5b0d5; stroke: #c5b0d5; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#mca817ee18d\" x=\"229.375025\" y=\"122.172306\" style=\"fill: #c5b0d5; stroke: #c5b0d5; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#mca817ee18d\" x=\"259.346356\" y=\"220.613652\" style=\"fill: #c5b0d5; stroke: #c5b0d5; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#mca817ee18d\" x=\"225.51259\" y=\"122.466228\" style=\"fill: #c5b0d5; stroke: #c5b0d5; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#mca817ee18d\" x=\"162.856894\" y=\"140.441953\" style=\"fill: #c5b0d5; stroke: #c5b0d5; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#mca817ee18d\" x=\"223.096704\" y=\"132.721673\" style=\"fill: #c5b0d5; stroke: #c5b0d5; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#mca817ee18d\" x=\"227.723138\" y=\"119.285142\" style=\"fill: #c5b0d5; stroke: #c5b0d5; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#mca817ee18d\" x=\"167.826435\" y=\"147.954172\" style=\"fill: #c5b0d5; stroke: #c5b0d5; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#mca817ee18d\" x=\"184.17224\" y=\"161.3391\" style=\"fill: #c5b0d5; stroke: #c5b0d5; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#mca817ee18d\" x=\"179.865821\" y=\"161.262487\" style=\"fill: #c5b0d5; stroke: #c5b0d5; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#mca817ee18d\" x=\"230.357081\" y=\"123.958299\" style=\"fill: #c5b0d5; stroke: #c5b0d5; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#mca817ee18d\" x=\"238.195061\" y=\"122.224273\" style=\"fill: #c5b0d5; stroke: #c5b0d5; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#mca817ee18d\" x=\"225.916107\" y=\"128.784024\" style=\"fill: #c5b0d5; stroke: #c5b0d5; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#mca817ee18d\" x=\"274.886471\" y=\"182.084474\" style=\"fill: #c5b0d5; stroke: #c5b0d5; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#mca817ee18d\" x=\"223.278619\" y=\"121.38754\" style=\"fill: #c5b0d5; stroke: #c5b0d5; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#mca817ee18d\" x=\"242.690735\" y=\"97.020463\" style=\"fill: #c5b0d5; stroke: #c5b0d5; stroke-width: 0.5\"/>\n",
              "    </g>\n",
              "   </g>\n",
              "   <g id=\"PathCollection_11\">\n",
              "    <defs>\n",
              "     <path id=\"mecdb7a7eca\" d=\"M 0 3 \n",
              "C 0.795609 3 1.55874 2.683901 2.12132 2.12132 \n",
              "C 2.683901 1.55874 3 0.795609 3 0 \n",
              "C 3 -0.795609 2.683901 -1.55874 2.12132 -2.12132 \n",
              "C 1.55874 -2.683901 0.795609 -3 0 -3 \n",
              "C -0.795609 -3 -1.55874 -2.683901 -2.12132 -2.12132 \n",
              "C -2.683901 -1.55874 -3 -0.795609 -3 0 \n",
              "C -3 0.795609 -2.683901 1.55874 -2.12132 2.12132 \n",
              "C -1.55874 2.683901 -0.795609 3 0 3 \n",
              "z\n",
              "\" style=\"stroke: #8c564b; stroke-width: 0.5\"/>\n",
              "    </defs>\n",
              "    <g clip-path=\"url(#p596f914704)\">\n",
              "     <use xlink:href=\"#mecdb7a7eca\" x=\"322.311236\" y=\"114.439717\" style=\"fill: #8c564b; stroke: #8c564b; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#mecdb7a7eca\" x=\"333.903939\" y=\"135.286092\" style=\"fill: #8c564b; stroke: #8c564b; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#mecdb7a7eca\" x=\"336.064946\" y=\"107.025667\" style=\"fill: #8c564b; stroke: #8c564b; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#mecdb7a7eca\" x=\"319.645161\" y=\"100.870333\" style=\"fill: #8c564b; stroke: #8c564b; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#mecdb7a7eca\" x=\"321.065879\" y=\"105.907376\" style=\"fill: #8c564b; stroke: #8c564b; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#mecdb7a7eca\" x=\"330.906092\" y=\"78.060108\" style=\"fill: #8c564b; stroke: #8c564b; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#mecdb7a7eca\" x=\"370.881797\" y=\"155.610024\" style=\"fill: #8c564b; stroke: #8c564b; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#mecdb7a7eca\" x=\"310.438681\" y=\"105.522824\" style=\"fill: #8c564b; stroke: #8c564b; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#mecdb7a7eca\" x=\"330.904535\" y=\"99.975954\" style=\"fill: #8c564b; stroke: #8c564b; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#mecdb7a7eca\" x=\"324.167457\" y=\"92.587436\" style=\"fill: #8c564b; stroke: #8c564b; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#mecdb7a7eca\" x=\"316.766846\" y=\"98.676133\" style=\"fill: #8c564b; stroke: #8c564b; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#mecdb7a7eca\" x=\"333.441553\" y=\"79.669821\" style=\"fill: #8c564b; stroke: #8c564b; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#mecdb7a7eca\" x=\"317.435182\" y=\"93.101808\" style=\"fill: #8c564b; stroke: #8c564b; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#mecdb7a7eca\" x=\"327.925605\" y=\"54.241709\" style=\"fill: #8c564b; stroke: #8c564b; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#mecdb7a7eca\" x=\"313.270249\" y=\"99.39107\" style=\"fill: #8c564b; stroke: #8c564b; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#mecdb7a7eca\" x=\"338.624866\" y=\"97.227301\" style=\"fill: #8c564b; stroke: #8c564b; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#mecdb7a7eca\" x=\"336.307495\" y=\"83.527654\" style=\"fill: #8c564b; stroke: #8c564b; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#mecdb7a7eca\" x=\"334.215551\" y=\"79.496734\" style=\"fill: #8c564b; stroke: #8c564b; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#mecdb7a7eca\" x=\"337.094871\" y=\"107.921292\" style=\"fill: #8c564b; stroke: #8c564b; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#mecdb7a7eca\" x=\"341.55249\" y=\"77.800688\" style=\"fill: #8c564b; stroke: #8c564b; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#mecdb7a7eca\" x=\"324.552617\" y=\"85.562693\" style=\"fill: #8c564b; stroke: #8c564b; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#mecdb7a7eca\" x=\"497.834428\" y=\"129.628618\" style=\"fill: #8c564b; stroke: #8c564b; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#mecdb7a7eca\" x=\"323.675377\" y=\"97.177362\" style=\"fill: #8c564b; stroke: #8c564b; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#mecdb7a7eca\" x=\"374.645679\" y=\"149.710335\" style=\"fill: #8c564b; stroke: #8c564b; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#mecdb7a7eca\" x=\"336.828828\" y=\"152.04783\" style=\"fill: #8c564b; stroke: #8c564b; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#mecdb7a7eca\" x=\"335.13059\" y=\"82.487471\" style=\"fill: #8c564b; stroke: #8c564b; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#mecdb7a7eca\" x=\"371.453646\" y=\"151.501819\" style=\"fill: #8c564b; stroke: #8c564b; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#mecdb7a7eca\" x=\"325.810829\" y=\"105.796855\" style=\"fill: #8c564b; stroke: #8c564b; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#mecdb7a7eca\" x=\"324.589906\" y=\"103.135864\" style=\"fill: #8c564b; stroke: #8c564b; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#mecdb7a7eca\" x=\"317.277904\" y=\"98.383383\" style=\"fill: #8c564b; stroke: #8c564b; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#mecdb7a7eca\" x=\"323.913574\" y=\"96.922006\" style=\"fill: #8c564b; stroke: #8c564b; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#mecdb7a7eca\" x=\"299.882665\" y=\"79.004996\" style=\"fill: #8c564b; stroke: #8c564b; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#mecdb7a7eca\" x=\"329.713933\" y=\"92.698153\" style=\"fill: #8c564b; stroke: #8c564b; stroke-width: 0.5\"/>\n",
              "    </g>\n",
              "   </g>\n",
              "   <g id=\"PathCollection_12\">\n",
              "    <defs>\n",
              "     <path id=\"m26fe20d470\" d=\"M 0 3 \n",
              "C 0.795609 3 1.55874 2.683901 2.12132 2.12132 \n",
              "C 2.683901 1.55874 3 0.795609 3 0 \n",
              "C 3 -0.795609 2.683901 -1.55874 2.12132 -2.12132 \n",
              "C 1.55874 -2.683901 0.795609 -3 0 -3 \n",
              "C -0.795609 -3 -1.55874 -2.683901 -2.12132 -2.12132 \n",
              "C -2.683901 -1.55874 -3 -0.795609 -3 0 \n",
              "C -3 0.795609 -2.683901 1.55874 -2.12132 2.12132 \n",
              "C -1.55874 2.683901 -0.795609 3 0 3 \n",
              "z\n",
              "\" style=\"stroke: #c49c94; stroke-width: 0.5\"/>\n",
              "    </defs>\n",
              "    <g clip-path=\"url(#p596f914704)\">\n",
              "     <use xlink:href=\"#m26fe20d470\" x=\"540.489789\" y=\"187.072928\" style=\"fill: #c49c94; stroke: #c49c94; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m26fe20d470\" x=\"519.927304\" y=\"185.18639\" style=\"fill: #c49c94; stroke: #c49c94; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m26fe20d470\" x=\"550.96624\" y=\"191.150468\" style=\"fill: #c49c94; stroke: #c49c94; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m26fe20d470\" x=\"505.117339\" y=\"175.07925\" style=\"fill: #c49c94; stroke: #c49c94; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m26fe20d470\" x=\"518.500326\" y=\"160.820526\" style=\"fill: #c49c94; stroke: #c49c94; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m26fe20d470\" x=\"510.407327\" y=\"182.835166\" style=\"fill: #c49c94; stroke: #c49c94; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m26fe20d470\" x=\"505.181435\" y=\"176.897673\" style=\"fill: #c49c94; stroke: #c49c94; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m26fe20d470\" x=\"500.457235\" y=\"170.244036\" style=\"fill: #c49c94; stroke: #c49c94; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m26fe20d470\" x=\"514.482062\" y=\"142.305385\" style=\"fill: #c49c94; stroke: #c49c94; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m26fe20d470\" x=\"532.188056\" y=\"169.567264\" style=\"fill: #c49c94; stroke: #c49c94; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m26fe20d470\" x=\"545.648749\" y=\"180.24426\" style=\"fill: #c49c94; stroke: #c49c94; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m26fe20d470\" x=\"551.488269\" y=\"157.326637\" style=\"fill: #c49c94; stroke: #c49c94; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m26fe20d470\" x=\"521.720851\" y=\"173.019004\" style=\"fill: #c49c94; stroke: #c49c94; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m26fe20d470\" x=\"531.276334\" y=\"166.859638\" style=\"fill: #c49c94; stroke: #c49c94; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m26fe20d470\" x=\"527.579273\" y=\"204.382574\" style=\"fill: #c49c94; stroke: #c49c94; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m26fe20d470\" x=\"547.553145\" y=\"186.855135\" style=\"fill: #c49c94; stroke: #c49c94; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m26fe20d470\" x=\"510.166538\" y=\"160.970783\" style=\"fill: #c49c94; stroke: #c49c94; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m26fe20d470\" x=\"509.640556\" y=\"179.088129\" style=\"fill: #c49c94; stroke: #c49c94; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m26fe20d470\" x=\"515.144411\" y=\"199.024941\" style=\"fill: #c49c94; stroke: #c49c94; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m26fe20d470\" x=\"514.873757\" y=\"176.679446\" style=\"fill: #c49c94; stroke: #c49c94; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m26fe20d470\" x=\"538.35037\" y=\"154.926299\" style=\"fill: #c49c94; stroke: #c49c94; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m26fe20d470\" x=\"515.22018\" y=\"143.104972\" style=\"fill: #c49c94; stroke: #c49c94; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m26fe20d470\" x=\"539.851257\" y=\"174.853766\" style=\"fill: #c49c94; stroke: #c49c94; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m26fe20d470\" x=\"576.704257\" y=\"191.807686\" style=\"fill: #c49c94; stroke: #c49c94; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m26fe20d470\" x=\"543.557753\" y=\"195.662368\" style=\"fill: #c49c94; stroke: #c49c94; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m26fe20d470\" x=\"540.655581\" y=\"180.088924\" style=\"fill: #c49c94; stroke: #c49c94; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m26fe20d470\" x=\"533.308968\" y=\"202.051953\" style=\"fill: #c49c94; stroke: #c49c94; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m26fe20d470\" x=\"509.72628\" y=\"171.788643\" style=\"fill: #c49c94; stroke: #c49c94; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m26fe20d470\" x=\"511.257285\" y=\"198.708247\" style=\"fill: #c49c94; stroke: #c49c94; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m26fe20d470\" x=\"509.447969\" y=\"199.442457\" style=\"fill: #c49c94; stroke: #c49c94; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m26fe20d470\" x=\"528.953166\" y=\"163.306246\" style=\"fill: #c49c94; stroke: #c49c94; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m26fe20d470\" x=\"557.033474\" y=\"146.292823\" style=\"fill: #c49c94; stroke: #c49c94; stroke-width: 0.5\"/>\n",
              "    </g>\n",
              "   </g>\n",
              "   <g id=\"PathCollection_13\">\n",
              "    <defs>\n",
              "     <path id=\"m89411d035a\" d=\"M 0 3 \n",
              "C 0.795609 3 1.55874 2.683901 2.12132 2.12132 \n",
              "C 2.683901 1.55874 3 0.795609 3 0 \n",
              "C 3 -0.795609 2.683901 -1.55874 2.12132 -2.12132 \n",
              "C 1.55874 -2.683901 0.795609 -3 0 -3 \n",
              "C -0.795609 -3 -1.55874 -2.683901 -2.12132 -2.12132 \n",
              "C -2.683901 -1.55874 -3 -0.795609 -3 0 \n",
              "C -3 0.795609 -2.683901 1.55874 -2.12132 2.12132 \n",
              "C -1.55874 2.683901 -0.795609 3 0 3 \n",
              "z\n",
              "\" style=\"stroke: #e377c2; stroke-width: 0.5\"/>\n",
              "    </defs>\n",
              "    <g clip-path=\"url(#p596f914704)\">\n",
              "     <use xlink:href=\"#m89411d035a\" x=\"243.780466\" y=\"87.599768\" style=\"fill: #e377c2; stroke: #e377c2; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m89411d035a\" x=\"112.909377\" y=\"120.982135\" style=\"fill: #e377c2; stroke: #e377c2; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m89411d035a\" x=\"106.383439\" y=\"154.786091\" style=\"fill: #e377c2; stroke: #e377c2; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m89411d035a\" x=\"146.278239\" y=\"292.079468\" style=\"fill: #e377c2; stroke: #e377c2; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m89411d035a\" x=\"290.419165\" y=\"51.835572\" style=\"fill: #e377c2; stroke: #e377c2; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m89411d035a\" x=\"136.622478\" y=\"310.301614\" style=\"fill: #e377c2; stroke: #e377c2; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m89411d035a\" x=\"120.117149\" y=\"155.959237\" style=\"fill: #e377c2; stroke: #e377c2; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m89411d035a\" x=\"212.406133\" y=\"155.784925\" style=\"fill: #e377c2; stroke: #e377c2; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m89411d035a\" x=\"226.661746\" y=\"231.354544\" style=\"fill: #e377c2; stroke: #e377c2; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m89411d035a\" x=\"250.570852\" y=\"119.976514\" style=\"fill: #e377c2; stroke: #e377c2; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m89411d035a\" x=\"318.802403\" y=\"50.892854\" style=\"fill: #e377c2; stroke: #e377c2; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m89411d035a\" x=\"107.445595\" y=\"192.437985\" style=\"fill: #e377c2; stroke: #e377c2; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m89411d035a\" x=\"226.834028\" y=\"235.42188\" style=\"fill: #e377c2; stroke: #e377c2; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m89411d035a\" x=\"184.778512\" y=\"139.17929\" style=\"fill: #e377c2; stroke: #e377c2; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m89411d035a\" x=\"267.740424\" y=\"131.288202\" style=\"fill: #e377c2; stroke: #e377c2; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m89411d035a\" x=\"202.14666\" y=\"228.559538\" style=\"fill: #e377c2; stroke: #e377c2; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m89411d035a\" x=\"112.394358\" y=\"155.045316\" style=\"fill: #e377c2; stroke: #e377c2; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m89411d035a\" x=\"251.45192\" y=\"228.623869\" style=\"fill: #e377c2; stroke: #e377c2; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m89411d035a\" x=\"243.245938\" y=\"93.540137\" style=\"fill: #e377c2; stroke: #e377c2; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m89411d035a\" x=\"123.352056\" y=\"171.97386\" style=\"fill: #e377c2; stroke: #e377c2; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m89411d035a\" x=\"190.377723\" y=\"175.229913\" style=\"fill: #e377c2; stroke: #e377c2; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m89411d035a\" x=\"162.13853\" y=\"139.566283\" style=\"fill: #e377c2; stroke: #e377c2; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m89411d035a\" x=\"325.824421\" y=\"231.82052\" style=\"fill: #e377c2; stroke: #e377c2; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m89411d035a\" x=\"287.338863\" y=\"101.351667\" style=\"fill: #e377c2; stroke: #e377c2; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m89411d035a\" x=\"235.938257\" y=\"178.350495\" style=\"fill: #e377c2; stroke: #e377c2; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m89411d035a\" x=\"246.212954\" y=\"85.789685\" style=\"fill: #e377c2; stroke: #e377c2; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m89411d035a\" x=\"339.326637\" y=\"244.649803\" style=\"fill: #e377c2; stroke: #e377c2; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m89411d035a\" x=\"240.218858\" y=\"87.829284\" style=\"fill: #e377c2; stroke: #e377c2; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m89411d035a\" x=\"188.088438\" y=\"147.235701\" style=\"fill: #e377c2; stroke: #e377c2; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m89411d035a\" x=\"214.129\" y=\"317.406584\" style=\"fill: #e377c2; stroke: #e377c2; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m89411d035a\" x=\"108.94352\" y=\"154.782714\" style=\"fill: #e377c2; stroke: #e377c2; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m89411d035a\" x=\"198.343217\" y=\"146.742006\" style=\"fill: #e377c2; stroke: #e377c2; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m89411d035a\" x=\"207.040527\" y=\"150.765805\" style=\"fill: #e377c2; stroke: #e377c2; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m89411d035a\" x=\"201.811169\" y=\"296.618035\" style=\"fill: #e377c2; stroke: #e377c2; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m89411d035a\" x=\"193.090333\" y=\"200.031546\" style=\"fill: #e377c2; stroke: #e377c2; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m89411d035a\" x=\"249.02804\" y=\"123.899849\" style=\"fill: #e377c2; stroke: #e377c2; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m89411d035a\" x=\"195.304284\" y=\"114.162907\" style=\"fill: #e377c2; stroke: #e377c2; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m89411d035a\" x=\"364.404892\" y=\"79.119026\" style=\"fill: #e377c2; stroke: #e377c2; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m89411d035a\" x=\"162.229468\" y=\"305.559304\" style=\"fill: #e377c2; stroke: #e377c2; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m89411d035a\" x=\"149.266529\" y=\"313.671119\" style=\"fill: #e377c2; stroke: #e377c2; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m89411d035a\" x=\"269.542985\" y=\"128.7506\" style=\"fill: #e377c2; stroke: #e377c2; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m89411d035a\" x=\"329.352627\" y=\"201.625496\" style=\"fill: #e377c2; stroke: #e377c2; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m89411d035a\" x=\"243.255035\" y=\"139.924356\" style=\"fill: #e377c2; stroke: #e377c2; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m89411d035a\" x=\"248.196812\" y=\"252.875546\" style=\"fill: #e377c2; stroke: #e377c2; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m89411d035a\" x=\"123.702196\" y=\"172.795096\" style=\"fill: #e377c2; stroke: #e377c2; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m89411d035a\" x=\"240.310985\" y=\"138.424477\" style=\"fill: #e377c2; stroke: #e377c2; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m89411d035a\" x=\"224.15823\" y=\"322.313087\" style=\"fill: #e377c2; stroke: #e377c2; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m89411d035a\" x=\"267.662757\" y=\"84.272119\" style=\"fill: #e377c2; stroke: #e377c2; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m89411d035a\" x=\"147.611664\" y=\"149.829165\" style=\"fill: #e377c2; stroke: #e377c2; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m89411d035a\" x=\"195.311585\" y=\"299.108433\" style=\"fill: #e377c2; stroke: #e377c2; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m89411d035a\" x=\"182.612204\" y=\"146.735704\" style=\"fill: #e377c2; stroke: #e377c2; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m89411d035a\" x=\"159.878103\" y=\"305.69825\" style=\"fill: #e377c2; stroke: #e377c2; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m89411d035a\" x=\"244.323303\" y=\"139.342022\" style=\"fill: #e377c2; stroke: #e377c2; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m89411d035a\" x=\"309.641774\" y=\"169.608607\" style=\"fill: #e377c2; stroke: #e377c2; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m89411d035a\" x=\"273.201472\" y=\"111.873323\" style=\"fill: #e377c2; stroke: #e377c2; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m89411d035a\" x=\"268.984348\" y=\"129.403414\" style=\"fill: #e377c2; stroke: #e377c2; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m89411d035a\" x=\"215.727936\" y=\"321.355632\" style=\"fill: #e377c2; stroke: #e377c2; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m89411d035a\" x=\"272.609496\" y=\"111.839362\" style=\"fill: #e377c2; stroke: #e377c2; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m89411d035a\" x=\"300.60442\" y=\"88.025147\" style=\"fill: #e377c2; stroke: #e377c2; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m89411d035a\" x=\"121.734382\" y=\"152.799383\" style=\"fill: #e377c2; stroke: #e377c2; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m89411d035a\" x=\"178.265018\" y=\"127.750106\" style=\"fill: #e377c2; stroke: #e377c2; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m89411d035a\" x=\"225.04786\" y=\"186.141089\" style=\"fill: #e377c2; stroke: #e377c2; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m89411d035a\" x=\"186.256629\" y=\"239.601195\" style=\"fill: #e377c2; stroke: #e377c2; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m89411d035a\" x=\"158.581916\" y=\"260.741274\" style=\"fill: #e377c2; stroke: #e377c2; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m89411d035a\" x=\"195.624061\" y=\"113.984442\" style=\"fill: #e377c2; stroke: #e377c2; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m89411d035a\" x=\"153.670927\" y=\"311.46949\" style=\"fill: #e377c2; stroke: #e377c2; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m89411d035a\" x=\"244.719211\" y=\"149.069296\" style=\"fill: #e377c2; stroke: #e377c2; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m89411d035a\" x=\"301.323975\" y=\"102.773863\" style=\"fill: #e377c2; stroke: #e377c2; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m89411d035a\" x=\"287.022149\" y=\"101.596732\" style=\"fill: #e377c2; stroke: #e377c2; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m89411d035a\" x=\"234.430147\" y=\"74.333936\" style=\"fill: #e377c2; stroke: #e377c2; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m89411d035a\" x=\"79.448442\" y=\"229.358998\" style=\"fill: #e377c2; stroke: #e377c2; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m89411d035a\" x=\"110.800943\" y=\"121.653841\" style=\"fill: #e377c2; stroke: #e377c2; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m89411d035a\" x=\"235.730233\" y=\"78.794996\" style=\"fill: #e377c2; stroke: #e377c2; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m89411d035a\" x=\"204.662801\" y=\"199.01365\" style=\"fill: #e377c2; stroke: #e377c2; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m89411d035a\" x=\"146.413944\" y=\"292.494495\" style=\"fill: #e377c2; stroke: #e377c2; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m89411d035a\" x=\"247.860037\" y=\"124.872764\" style=\"fill: #e377c2; stroke: #e377c2; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m89411d035a\" x=\"127.822849\" y=\"289.180247\" style=\"fill: #e377c2; stroke: #e377c2; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m89411d035a\" x=\"320.717469\" y=\"54.968972\" style=\"fill: #e377c2; stroke: #e377c2; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m89411d035a\" x=\"242.726658\" y=\"154.34306\" style=\"fill: #e377c2; stroke: #e377c2; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m89411d035a\" x=\"260.69584\" y=\"195.777861\" style=\"fill: #e377c2; stroke: #e377c2; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m89411d035a\" x=\"166.07493\" y=\"231.975823\" style=\"fill: #e377c2; stroke: #e377c2; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m89411d035a\" x=\"148.494576\" y=\"110.999151\" style=\"fill: #e377c2; stroke: #e377c2; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m89411d035a\" x=\"152.127934\" y=\"114.167985\" style=\"fill: #e377c2; stroke: #e377c2; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m89411d035a\" x=\"207.662377\" y=\"227.269892\" style=\"fill: #e377c2; stroke: #e377c2; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m89411d035a\" x=\"249.238821\" y=\"128.6075\" style=\"fill: #e377c2; stroke: #e377c2; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m89411d035a\" x=\"185.923036\" y=\"220.66199\" style=\"fill: #e377c2; stroke: #e377c2; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m89411d035a\" x=\"190.043083\" y=\"175.562699\" style=\"fill: #e377c2; stroke: #e377c2; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m89411d035a\" x=\"291.813898\" y=\"52.01587\" style=\"fill: #e377c2; stroke: #e377c2; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m89411d035a\" x=\"360.567109\" y=\"214.616721\" style=\"fill: #e377c2; stroke: #e377c2; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m89411d035a\" x=\"226.659485\" y=\"232.145717\" style=\"fill: #e377c2; stroke: #e377c2; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m89411d035a\" x=\"254.303497\" y=\"102.44441\" style=\"fill: #e377c2; stroke: #e377c2; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m89411d035a\" x=\"147.583877\" y=\"110.420854\" style=\"fill: #e377c2; stroke: #e377c2; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m89411d035a\" x=\"172.10743\" y=\"204.644376\" style=\"fill: #e377c2; stroke: #e377c2; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m89411d035a\" x=\"208.007177\" y=\"104.577546\" style=\"fill: #e377c2; stroke: #e377c2; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m89411d035a\" x=\"132.668372\" y=\"165.313645\" style=\"fill: #e377c2; stroke: #e377c2; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m89411d035a\" x=\"339.188652\" y=\"244.593874\" style=\"fill: #e377c2; stroke: #e377c2; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m89411d035a\" x=\"213.183064\" y=\"106.593911\" style=\"fill: #e377c2; stroke: #e377c2; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m89411d035a\" x=\"195.821413\" y=\"270.419489\" style=\"fill: #e377c2; stroke: #e377c2; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m89411d035a\" x=\"215.511154\" y=\"258.905456\" style=\"fill: #e377c2; stroke: #e377c2; stroke-width: 0.5\"/>\n",
              "    </g>\n",
              "   </g>\n",
              "   <g id=\"PathCollection_14\">\n",
              "    <defs>\n",
              "     <path id=\"m2d84f52954\" d=\"M 0 3 \n",
              "C 0.795609 3 1.55874 2.683901 2.12132 2.12132 \n",
              "C 2.683901 1.55874 3 0.795609 3 0 \n",
              "C 3 -0.795609 2.683901 -1.55874 2.12132 -2.12132 \n",
              "C 1.55874 -2.683901 0.795609 -3 0 -3 \n",
              "C -0.795609 -3 -1.55874 -2.683901 -2.12132 -2.12132 \n",
              "C -2.683901 -1.55874 -3 -0.795609 -3 0 \n",
              "C -3 0.795609 -2.683901 1.55874 -2.12132 2.12132 \n",
              "C -1.55874 2.683901 -0.795609 3 0 3 \n",
              "z\n",
              "\" style=\"stroke: #f7b6d2; stroke-width: 0.5\"/>\n",
              "    </defs>\n",
              "    <g clip-path=\"url(#p596f914704)\">\n",
              "     <use xlink:href=\"#m2d84f52954\" x=\"426.937622\" y=\"154.244343\" style=\"fill: #f7b6d2; stroke: #f7b6d2; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m2d84f52954\" x=\"373.855508\" y=\"174.662924\" style=\"fill: #f7b6d2; stroke: #f7b6d2; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m2d84f52954\" x=\"359.358182\" y=\"188.501392\" style=\"fill: #f7b6d2; stroke: #f7b6d2; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m2d84f52954\" x=\"359.513179\" y=\"205.871505\" style=\"fill: #f7b6d2; stroke: #f7b6d2; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m2d84f52954\" x=\"347.625235\" y=\"187.878227\" style=\"fill: #f7b6d2; stroke: #f7b6d2; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m2d84f52954\" x=\"351.63315\" y=\"161.837439\" style=\"fill: #f7b6d2; stroke: #f7b6d2; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m2d84f52954\" x=\"276.962897\" y=\"113.055456\" style=\"fill: #f7b6d2; stroke: #f7b6d2; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m2d84f52954\" x=\"340.811967\" y=\"168.36655\" style=\"fill: #f7b6d2; stroke: #f7b6d2; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m2d84f52954\" x=\"356.044435\" y=\"205.469359\" style=\"fill: #f7b6d2; stroke: #f7b6d2; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m2d84f52954\" x=\"347.095328\" y=\"171.323508\" style=\"fill: #f7b6d2; stroke: #f7b6d2; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m2d84f52954\" x=\"336.79106\" y=\"152.449708\" style=\"fill: #f7b6d2; stroke: #f7b6d2; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m2d84f52954\" x=\"392.07912\" y=\"158.521357\" style=\"fill: #f7b6d2; stroke: #f7b6d2; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m2d84f52954\" x=\"400.569989\" y=\"227.78231\" style=\"fill: #f7b6d2; stroke: #f7b6d2; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m2d84f52954\" x=\"404.55671\" y=\"188.500363\" style=\"fill: #f7b6d2; stroke: #f7b6d2; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m2d84f52954\" x=\"386.273956\" y=\"242.105054\" style=\"fill: #f7b6d2; stroke: #f7b6d2; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m2d84f52954\" x=\"212.111086\" y=\"243.98822\" style=\"fill: #f7b6d2; stroke: #f7b6d2; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m2d84f52954\" x=\"342.310719\" y=\"201.727727\" style=\"fill: #f7b6d2; stroke: #f7b6d2; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m2d84f52954\" x=\"333.123956\" y=\"185.749765\" style=\"fill: #f7b6d2; stroke: #f7b6d2; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m2d84f52954\" x=\"364.434987\" y=\"163.918154\" style=\"fill: #f7b6d2; stroke: #f7b6d2; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m2d84f52954\" x=\"363.609831\" y=\"142.755009\" style=\"fill: #f7b6d2; stroke: #f7b6d2; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m2d84f52954\" x=\"366.963142\" y=\"140.378084\" style=\"fill: #f7b6d2; stroke: #f7b6d2; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m2d84f52954\" x=\"410.310962\" y=\"178.502906\" style=\"fill: #f7b6d2; stroke: #f7b6d2; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m2d84f52954\" x=\"340.376642\" y=\"160.113224\" style=\"fill: #f7b6d2; stroke: #f7b6d2; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m2d84f52954\" x=\"364.332904\" y=\"142.222981\" style=\"fill: #f7b6d2; stroke: #f7b6d2; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m2d84f52954\" x=\"346.833314\" y=\"285.568675\" style=\"fill: #f7b6d2; stroke: #f7b6d2; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m2d84f52954\" x=\"370.216999\" y=\"159.900762\" style=\"fill: #f7b6d2; stroke: #f7b6d2; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m2d84f52954\" x=\"333.064095\" y=\"186.409159\" style=\"fill: #f7b6d2; stroke: #f7b6d2; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m2d84f52954\" x=\"389.866737\" y=\"178.308089\" style=\"fill: #f7b6d2; stroke: #f7b6d2; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m2d84f52954\" x=\"396.401741\" y=\"199.242595\" style=\"fill: #f7b6d2; stroke: #f7b6d2; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m2d84f52954\" x=\"397.754123\" y=\"263.475009\" style=\"fill: #f7b6d2; stroke: #f7b6d2; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m2d84f52954\" x=\"356.192207\" y=\"179.653455\" style=\"fill: #f7b6d2; stroke: #f7b6d2; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m2d84f52954\" x=\"346.752692\" y=\"188.666627\" style=\"fill: #f7b6d2; stroke: #f7b6d2; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m2d84f52954\" x=\"372.985852\" y=\"164.048608\" style=\"fill: #f7b6d2; stroke: #f7b6d2; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m2d84f52954\" x=\"335.536103\" y=\"199.688485\" style=\"fill: #f7b6d2; stroke: #f7b6d2; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m2d84f52954\" x=\"375.977911\" y=\"194.743113\" style=\"fill: #f7b6d2; stroke: #f7b6d2; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m2d84f52954\" x=\"389.726188\" y=\"258.97355\" style=\"fill: #f7b6d2; stroke: #f7b6d2; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m2d84f52954\" x=\"369.694482\" y=\"173.881292\" style=\"fill: #f7b6d2; stroke: #f7b6d2; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m2d84f52954\" x=\"357.352519\" y=\"236.816218\" style=\"fill: #f7b6d2; stroke: #f7b6d2; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m2d84f52954\" x=\"378.171655\" y=\"162.535285\" style=\"fill: #f7b6d2; stroke: #f7b6d2; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m2d84f52954\" x=\"361.085774\" y=\"104.320146\" style=\"fill: #f7b6d2; stroke: #f7b6d2; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m2d84f52954\" x=\"382.369687\" y=\"188.090074\" style=\"fill: #f7b6d2; stroke: #f7b6d2; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m2d84f52954\" x=\"354.581047\" y=\"154.81026\" style=\"fill: #f7b6d2; stroke: #f7b6d2; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m2d84f52954\" x=\"389.256496\" y=\"231.164821\" style=\"fill: #f7b6d2; stroke: #f7b6d2; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m2d84f52954\" x=\"374.995276\" y=\"122.35935\" style=\"fill: #f7b6d2; stroke: #f7b6d2; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m2d84f52954\" x=\"300.813973\" y=\"154.32431\" style=\"fill: #f7b6d2; stroke: #f7b6d2; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m2d84f52954\" x=\"361.645781\" y=\"164.296186\" style=\"fill: #f7b6d2; stroke: #f7b6d2; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m2d84f52954\" x=\"366.532422\" y=\"173.354258\" style=\"fill: #f7b6d2; stroke: #f7b6d2; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m2d84f52954\" x=\"368.204508\" y=\"224.007435\" style=\"fill: #f7b6d2; stroke: #f7b6d2; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m2d84f52954\" x=\"387.632648\" y=\"153.343745\" style=\"fill: #f7b6d2; stroke: #f7b6d2; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m2d84f52954\" x=\"400.754716\" y=\"226.222025\" style=\"fill: #f7b6d2; stroke: #f7b6d2; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m2d84f52954\" x=\"355.270348\" y=\"193.27028\" style=\"fill: #f7b6d2; stroke: #f7b6d2; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m2d84f52954\" x=\"342.05995\" y=\"205.748063\" style=\"fill: #f7b6d2; stroke: #f7b6d2; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m2d84f52954\" x=\"366.436042\" y=\"202.761157\" style=\"fill: #f7b6d2; stroke: #f7b6d2; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m2d84f52954\" x=\"401.635036\" y=\"107.957432\" style=\"fill: #f7b6d2; stroke: #f7b6d2; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m2d84f52954\" x=\"375.269435\" y=\"120.371553\" style=\"fill: #f7b6d2; stroke: #f7b6d2; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m2d84f52954\" x=\"341.025228\" y=\"195.811406\" style=\"fill: #f7b6d2; stroke: #f7b6d2; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m2d84f52954\" x=\"387.19554\" y=\"165.613342\" style=\"fill: #f7b6d2; stroke: #f7b6d2; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m2d84f52954\" x=\"367.301149\" y=\"181.410083\" style=\"fill: #f7b6d2; stroke: #f7b6d2; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m2d84f52954\" x=\"345.746943\" y=\"181.303678\" style=\"fill: #f7b6d2; stroke: #f7b6d2; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m2d84f52954\" x=\"413.691655\" y=\"149.119152\" style=\"fill: #f7b6d2; stroke: #f7b6d2; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m2d84f52954\" x=\"334.966201\" y=\"199.913305\" style=\"fill: #f7b6d2; stroke: #f7b6d2; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m2d84f52954\" x=\"374.485562\" y=\"218.826412\" style=\"fill: #f7b6d2; stroke: #f7b6d2; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m2d84f52954\" x=\"367.913521\" y=\"187.5151\" style=\"fill: #f7b6d2; stroke: #f7b6d2; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m2d84f52954\" x=\"386.14059\" y=\"174.54425\" style=\"fill: #f7b6d2; stroke: #f7b6d2; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m2d84f52954\" x=\"344.230057\" y=\"179.023006\" style=\"fill: #f7b6d2; stroke: #f7b6d2; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m2d84f52954\" x=\"330.793478\" y=\"160.187486\" style=\"fill: #f7b6d2; stroke: #f7b6d2; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m2d84f52954\" x=\"385.745792\" y=\"102.415948\" style=\"fill: #f7b6d2; stroke: #f7b6d2; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m2d84f52954\" x=\"362.798832\" y=\"154.820672\" style=\"fill: #f7b6d2; stroke: #f7b6d2; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m2d84f52954\" x=\"398.447662\" y=\"220.162802\" style=\"fill: #f7b6d2; stroke: #f7b6d2; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m2d84f52954\" x=\"375.853815\" y=\"124.233456\" style=\"fill: #f7b6d2; stroke: #f7b6d2; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m2d84f52954\" x=\"376.320785\" y=\"183.794841\" style=\"fill: #f7b6d2; stroke: #f7b6d2; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m2d84f52954\" x=\"353.392326\" y=\"159.696666\" style=\"fill: #f7b6d2; stroke: #f7b6d2; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m2d84f52954\" x=\"350.048695\" y=\"164.315897\" style=\"fill: #f7b6d2; stroke: #f7b6d2; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m2d84f52954\" x=\"327.510312\" y=\"146.578256\" style=\"fill: #f7b6d2; stroke: #f7b6d2; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m2d84f52954\" x=\"327.322599\" y=\"210.83975\" style=\"fill: #f7b6d2; stroke: #f7b6d2; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m2d84f52954\" x=\"463.180516\" y=\"205.905281\" style=\"fill: #f7b6d2; stroke: #f7b6d2; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m2d84f52954\" x=\"329.993389\" y=\"180.466886\" style=\"fill: #f7b6d2; stroke: #f7b6d2; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m2d84f52954\" x=\"389.808958\" y=\"178.328522\" style=\"fill: #f7b6d2; stroke: #f7b6d2; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m2d84f52954\" x=\"353.661881\" y=\"156.606608\" style=\"fill: #f7b6d2; stroke: #f7b6d2; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m2d84f52954\" x=\"210.396\" y=\"251.758277\" style=\"fill: #f7b6d2; stroke: #f7b6d2; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m2d84f52954\" x=\"369.791906\" y=\"186.499273\" style=\"fill: #f7b6d2; stroke: #f7b6d2; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m2d84f52954\" x=\"375.417061\" y=\"120.619239\" style=\"fill: #f7b6d2; stroke: #f7b6d2; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m2d84f52954\" x=\"364.89132\" y=\"105.972631\" style=\"fill: #f7b6d2; stroke: #f7b6d2; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m2d84f52954\" x=\"353.584869\" y=\"180.122151\" style=\"fill: #f7b6d2; stroke: #f7b6d2; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m2d84f52954\" x=\"369.152216\" y=\"133.331027\" style=\"fill: #f7b6d2; stroke: #f7b6d2; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m2d84f52954\" x=\"415.549629\" y=\"179.166185\" style=\"fill: #f7b6d2; stroke: #f7b6d2; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m2d84f52954\" x=\"417.899182\" y=\"208.504535\" style=\"fill: #f7b6d2; stroke: #f7b6d2; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m2d84f52954\" x=\"356.067398\" y=\"188.619853\" style=\"fill: #f7b6d2; stroke: #f7b6d2; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m2d84f52954\" x=\"424.483514\" y=\"162.888734\" style=\"fill: #f7b6d2; stroke: #f7b6d2; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m2d84f52954\" x=\"360.032778\" y=\"130.736786\" style=\"fill: #f7b6d2; stroke: #f7b6d2; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m2d84f52954\" x=\"455.472421\" y=\"236.360841\" style=\"fill: #f7b6d2; stroke: #f7b6d2; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m2d84f52954\" x=\"360.250111\" y=\"205.111058\" style=\"fill: #f7b6d2; stroke: #f7b6d2; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m2d84f52954\" x=\"373.447242\" y=\"156.793599\" style=\"fill: #f7b6d2; stroke: #f7b6d2; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m2d84f52954\" x=\"317.360287\" y=\"133.797879\" style=\"fill: #f7b6d2; stroke: #f7b6d2; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m2d84f52954\" x=\"379.961288\" y=\"105.374932\" style=\"fill: #f7b6d2; stroke: #f7b6d2; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m2d84f52954\" x=\"410.771084\" y=\"149.584685\" style=\"fill: #f7b6d2; stroke: #f7b6d2; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m2d84f52954\" x=\"368.941904\" y=\"165.826828\" style=\"fill: #f7b6d2; stroke: #f7b6d2; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m2d84f52954\" x=\"351.430998\" y=\"192.956368\" style=\"fill: #f7b6d2; stroke: #f7b6d2; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m2d84f52954\" x=\"341.756184\" y=\"185.650438\" style=\"fill: #f7b6d2; stroke: #f7b6d2; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m2d84f52954\" x=\"386.222667\" y=\"101.991365\" style=\"fill: #f7b6d2; stroke: #f7b6d2; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m2d84f52954\" x=\"381.4608\" y=\"247.847419\" style=\"fill: #f7b6d2; stroke: #f7b6d2; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m2d84f52954\" x=\"390.123128\" y=\"233.500987\" style=\"fill: #f7b6d2; stroke: #f7b6d2; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m2d84f52954\" x=\"371.724541\" y=\"202.691561\" style=\"fill: #f7b6d2; stroke: #f7b6d2; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m2d84f52954\" x=\"201.908179\" y=\"301.175743\" style=\"fill: #f7b6d2; stroke: #f7b6d2; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m2d84f52954\" x=\"413.26125\" y=\"269.971214\" style=\"fill: #f7b6d2; stroke: #f7b6d2; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m2d84f52954\" x=\"360.107227\" y=\"173.742671\" style=\"fill: #f7b6d2; stroke: #f7b6d2; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m2d84f52954\" x=\"373.525815\" y=\"192.513464\" style=\"fill: #f7b6d2; stroke: #f7b6d2; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m2d84f52954\" x=\"358.457312\" y=\"166.20814\" style=\"fill: #f7b6d2; stroke: #f7b6d2; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m2d84f52954\" x=\"400.623066\" y=\"127.894693\" style=\"fill: #f7b6d2; stroke: #f7b6d2; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m2d84f52954\" x=\"409.508387\" y=\"204.956011\" style=\"fill: #f7b6d2; stroke: #f7b6d2; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m2d84f52954\" x=\"362.07818\" y=\"170.848502\" style=\"fill: #f7b6d2; stroke: #f7b6d2; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m2d84f52954\" x=\"400.72381\" y=\"107.427879\" style=\"fill: #f7b6d2; stroke: #f7b6d2; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m2d84f52954\" x=\"386.434219\" y=\"135.930248\" style=\"fill: #f7b6d2; stroke: #f7b6d2; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m2d84f52954\" x=\"343.734979\" y=\"179.570207\" style=\"fill: #f7b6d2; stroke: #f7b6d2; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m2d84f52954\" x=\"365.160636\" y=\"187.76936\" style=\"fill: #f7b6d2; stroke: #f7b6d2; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m2d84f52954\" x=\"381.200853\" y=\"187.39762\" style=\"fill: #f7b6d2; stroke: #f7b6d2; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m2d84f52954\" x=\"403.40208\" y=\"233.163556\" style=\"fill: #f7b6d2; stroke: #f7b6d2; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m2d84f52954\" x=\"342.39924\" y=\"199.750724\" style=\"fill: #f7b6d2; stroke: #f7b6d2; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m2d84f52954\" x=\"360.388455\" y=\"160.468767\" style=\"fill: #f7b6d2; stroke: #f7b6d2; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m2d84f52954\" x=\"369.152338\" y=\"133.329874\" style=\"fill: #f7b6d2; stroke: #f7b6d2; stroke-width: 0.5\"/>\n",
              "    </g>\n",
              "   </g>\n",
              "   <g id=\"PathCollection_15\">\n",
              "    <defs>\n",
              "     <path id=\"me8168e8a62\" d=\"M 0 3 \n",
              "C 0.795609 3 1.55874 2.683901 2.12132 2.12132 \n",
              "C 2.683901 1.55874 3 0.795609 3 0 \n",
              "C 3 -0.795609 2.683901 -1.55874 2.12132 -2.12132 \n",
              "C 1.55874 -2.683901 0.795609 -3 0 -3 \n",
              "C -0.795609 -3 -1.55874 -2.683901 -2.12132 -2.12132 \n",
              "C -2.683901 -1.55874 -3 -0.795609 -3 0 \n",
              "C -3 0.795609 -2.683901 1.55874 -2.12132 2.12132 \n",
              "C -1.55874 2.683901 -0.795609 3 0 3 \n",
              "z\n",
              "\" style=\"stroke: #7f7f7f; stroke-width: 0.5\"/>\n",
              "    </defs>\n",
              "    <g clip-path=\"url(#p596f914704)\">\n",
              "     <use xlink:href=\"#me8168e8a62\" x=\"287.847961\" y=\"315.679183\" style=\"fill: #7f7f7f; stroke: #7f7f7f; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#me8168e8a62\" x=\"294.509405\" y=\"314.597062\" style=\"fill: #7f7f7f; stroke: #7f7f7f; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#me8168e8a62\" x=\"294.422534\" y=\"319.223946\" style=\"fill: #7f7f7f; stroke: #7f7f7f; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#me8168e8a62\" x=\"293.113643\" y=\"323.606035\" style=\"fill: #7f7f7f; stroke: #7f7f7f; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#me8168e8a62\" x=\"274.533047\" y=\"95.937808\" style=\"fill: #7f7f7f; stroke: #7f7f7f; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#me8168e8a62\" x=\"293.759302\" y=\"319.091332\" style=\"fill: #7f7f7f; stroke: #7f7f7f; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#me8168e8a62\" x=\"285.090727\" y=\"312.378584\" style=\"fill: #7f7f7f; stroke: #7f7f7f; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#me8168e8a62\" x=\"274.511253\" y=\"95.923739\" style=\"fill: #7f7f7f; stroke: #7f7f7f; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#me8168e8a62\" x=\"291.045903\" y=\"318.261773\" style=\"fill: #7f7f7f; stroke: #7f7f7f; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#me8168e8a62\" x=\"287.855648\" y=\"321.567089\" style=\"fill: #7f7f7f; stroke: #7f7f7f; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#me8168e8a62\" x=\"293.510482\" y=\"320.610956\" style=\"fill: #7f7f7f; stroke: #7f7f7f; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#me8168e8a62\" x=\"298.460624\" y=\"319.904876\" style=\"fill: #7f7f7f; stroke: #7f7f7f; stroke-width: 0.5\"/>\n",
              "    </g>\n",
              "   </g>\n",
              "   <g id=\"PathCollection_16\">\n",
              "    <defs>\n",
              "     <path id=\"m1c4d8747ba\" d=\"M 0 3 \n",
              "C 0.795609 3 1.55874 2.683901 2.12132 2.12132 \n",
              "C 2.683901 1.55874 3 0.795609 3 0 \n",
              "C 3 -0.795609 2.683901 -1.55874 2.12132 -2.12132 \n",
              "C 1.55874 -2.683901 0.795609 -3 0 -3 \n",
              "C -0.795609 -3 -1.55874 -2.683901 -2.12132 -2.12132 \n",
              "C -2.683901 -1.55874 -3 -0.795609 -3 0 \n",
              "C -3 0.795609 -2.683901 1.55874 -2.12132 2.12132 \n",
              "C -1.55874 2.683901 -0.795609 3 0 3 \n",
              "z\n",
              "\" style=\"stroke: #c7c7c7; stroke-width: 0.5\"/>\n",
              "    </defs>\n",
              "    <g clip-path=\"url(#p596f914704)\">\n",
              "     <use xlink:href=\"#m1c4d8747ba\" x=\"313.916678\" y=\"148.108078\" style=\"fill: #c7c7c7; stroke: #c7c7c7; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m1c4d8747ba\" x=\"306.095689\" y=\"136.653365\" style=\"fill: #c7c7c7; stroke: #c7c7c7; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m1c4d8747ba\" x=\"295.148289\" y=\"137.006525\" style=\"fill: #c7c7c7; stroke: #c7c7c7; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m1c4d8747ba\" x=\"290.366029\" y=\"162.18053\" style=\"fill: #c7c7c7; stroke: #c7c7c7; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m1c4d8747ba\" x=\"362.722654\" y=\"121.974535\" style=\"fill: #c7c7c7; stroke: #c7c7c7; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m1c4d8747ba\" x=\"351.706067\" y=\"149.617591\" style=\"fill: #c7c7c7; stroke: #c7c7c7; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m1c4d8747ba\" x=\"308.229341\" y=\"124.932475\" style=\"fill: #c7c7c7; stroke: #c7c7c7; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m1c4d8747ba\" x=\"310.557839\" y=\"148.261228\" style=\"fill: #c7c7c7; stroke: #c7c7c7; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m1c4d8747ba\" x=\"323.132245\" y=\"140.801893\" style=\"fill: #c7c7c7; stroke: #c7c7c7; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m1c4d8747ba\" x=\"356.404066\" y=\"123.485468\" style=\"fill: #c7c7c7; stroke: #c7c7c7; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m1c4d8747ba\" x=\"344.984019\" y=\"137.865249\" style=\"fill: #c7c7c7; stroke: #c7c7c7; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m1c4d8747ba\" x=\"290.43933\" y=\"162.247331\" style=\"fill: #c7c7c7; stroke: #c7c7c7; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m1c4d8747ba\" x=\"354.525406\" y=\"122.224821\" style=\"fill: #c7c7c7; stroke: #c7c7c7; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m1c4d8747ba\" x=\"342.488569\" y=\"134.482159\" style=\"fill: #c7c7c7; stroke: #c7c7c7; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m1c4d8747ba\" x=\"349.526299\" y=\"126.18881\" style=\"fill: #c7c7c7; stroke: #c7c7c7; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m1c4d8747ba\" x=\"322.297761\" y=\"146.725995\" style=\"fill: #c7c7c7; stroke: #c7c7c7; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m1c4d8747ba\" x=\"320.885089\" y=\"122.347279\" style=\"fill: #c7c7c7; stroke: #c7c7c7; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m1c4d8747ba\" x=\"300.087103\" y=\"137.617825\" style=\"fill: #c7c7c7; stroke: #c7c7c7; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m1c4d8747ba\" x=\"310.635611\" y=\"147.910014\" style=\"fill: #c7c7c7; stroke: #c7c7c7; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m1c4d8747ba\" x=\"347.386788\" y=\"140.129631\" style=\"fill: #c7c7c7; stroke: #c7c7c7; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m1c4d8747ba\" x=\"301.223005\" y=\"153.411986\" style=\"fill: #c7c7c7; stroke: #c7c7c7; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m1c4d8747ba\" x=\"337.231329\" y=\"122.845387\" style=\"fill: #c7c7c7; stroke: #c7c7c7; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m1c4d8747ba\" x=\"339.940341\" y=\"121.339907\" style=\"fill: #c7c7c7; stroke: #c7c7c7; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m1c4d8747ba\" x=\"307.554357\" y=\"146.261908\" style=\"fill: #c7c7c7; stroke: #c7c7c7; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m1c4d8747ba\" x=\"316.007793\" y=\"135.513737\" style=\"fill: #c7c7c7; stroke: #c7c7c7; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m1c4d8747ba\" x=\"312.427481\" y=\"130.749474\" style=\"fill: #c7c7c7; stroke: #c7c7c7; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m1c4d8747ba\" x=\"311.589266\" y=\"144.441474\" style=\"fill: #c7c7c7; stroke: #c7c7c7; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m1c4d8747ba\" x=\"323.574113\" y=\"155.209907\" style=\"fill: #c7c7c7; stroke: #c7c7c7; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m1c4d8747ba\" x=\"309.68622\" y=\"129.741426\" style=\"fill: #c7c7c7; stroke: #c7c7c7; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m1c4d8747ba\" x=\"319.97926\" y=\"138.813694\" style=\"fill: #c7c7c7; stroke: #c7c7c7; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m1c4d8747ba\" x=\"302.898051\" y=\"138.528485\" style=\"fill: #c7c7c7; stroke: #c7c7c7; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m1c4d8747ba\" x=\"295.201934\" y=\"134.629736\" style=\"fill: #c7c7c7; stroke: #c7c7c7; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m1c4d8747ba\" x=\"307.121835\" y=\"123.166584\" style=\"fill: #c7c7c7; stroke: #c7c7c7; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m1c4d8747ba\" x=\"340.244052\" y=\"125.471793\" style=\"fill: #c7c7c7; stroke: #c7c7c7; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m1c4d8747ba\" x=\"350.38632\" y=\"146.23991\" style=\"fill: #c7c7c7; stroke: #c7c7c7; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m1c4d8747ba\" x=\"351.846281\" y=\"133.353112\" style=\"fill: #c7c7c7; stroke: #c7c7c7; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m1c4d8747ba\" x=\"327.76363\" y=\"127.229128\" style=\"fill: #c7c7c7; stroke: #c7c7c7; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m1c4d8747ba\" x=\"357.90899\" y=\"119.276481\" style=\"fill: #c7c7c7; stroke: #c7c7c7; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m1c4d8747ba\" x=\"325.737243\" y=\"144.614024\" style=\"fill: #c7c7c7; stroke: #c7c7c7; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m1c4d8747ba\" x=\"359.460354\" y=\"103.358176\" style=\"fill: #c7c7c7; stroke: #c7c7c7; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m1c4d8747ba\" x=\"326.140765\" y=\"142.702155\" style=\"fill: #c7c7c7; stroke: #c7c7c7; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m1c4d8747ba\" x=\"307.279221\" y=\"153.4871\" style=\"fill: #c7c7c7; stroke: #c7c7c7; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m1c4d8747ba\" x=\"338.557752\" y=\"123.273718\" style=\"fill: #c7c7c7; stroke: #c7c7c7; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m1c4d8747ba\" x=\"327.313816\" y=\"133.228739\" style=\"fill: #c7c7c7; stroke: #c7c7c7; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m1c4d8747ba\" x=\"357.065754\" y=\"122.249032\" style=\"fill: #c7c7c7; stroke: #c7c7c7; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m1c4d8747ba\" x=\"322.608604\" y=\"155.287677\" style=\"fill: #c7c7c7; stroke: #c7c7c7; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m1c4d8747ba\" x=\"330.139598\" y=\"133.686831\" style=\"fill: #c7c7c7; stroke: #c7c7c7; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m1c4d8747ba\" x=\"329.746418\" y=\"142.557793\" style=\"fill: #c7c7c7; stroke: #c7c7c7; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m1c4d8747ba\" x=\"318.88905\" y=\"123.056145\" style=\"fill: #c7c7c7; stroke: #c7c7c7; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m1c4d8747ba\" x=\"341.688347\" y=\"143.340554\" style=\"fill: #c7c7c7; stroke: #c7c7c7; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m1c4d8747ba\" x=\"295.349075\" y=\"136.73382\" style=\"fill: #c7c7c7; stroke: #c7c7c7; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m1c4d8747ba\" x=\"303.937111\" y=\"168.920349\" style=\"fill: #c7c7c7; stroke: #c7c7c7; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m1c4d8747ba\" x=\"334.818597\" y=\"126.321544\" style=\"fill: #c7c7c7; stroke: #c7c7c7; stroke-width: 0.5\"/>\n",
              "    </g>\n",
              "   </g>\n",
              "   <g id=\"PathCollection_17\">\n",
              "    <defs>\n",
              "     <path id=\"mf30c6b51d6\" d=\"M 0 3 \n",
              "C 0.795609 3 1.55874 2.683901 2.12132 2.12132 \n",
              "C 2.683901 1.55874 3 0.795609 3 0 \n",
              "C 3 -0.795609 2.683901 -1.55874 2.12132 -2.12132 \n",
              "C 1.55874 -2.683901 0.795609 -3 0 -3 \n",
              "C -0.795609 -3 -1.55874 -2.683901 -2.12132 -2.12132 \n",
              "C -2.683901 -1.55874 -3 -0.795609 -3 0 \n",
              "C -3 0.795609 -2.683901 1.55874 -2.12132 2.12132 \n",
              "C -1.55874 2.683901 -0.795609 3 0 3 \n",
              "z\n",
              "\" style=\"stroke: #bcbd22; stroke-width: 0.5\"/>\n",
              "    </defs>\n",
              "    <g clip-path=\"url(#p596f914704)\">\n",
              "     <use xlink:href=\"#mf30c6b51d6\" x=\"127.323945\" y=\"208.149313\" style=\"fill: #bcbd22; stroke: #bcbd22; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#mf30c6b51d6\" x=\"133.182178\" y=\"199.29805\" style=\"fill: #bcbd22; stroke: #bcbd22; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#mf30c6b51d6\" x=\"133.016496\" y=\"199.648726\" style=\"fill: #bcbd22; stroke: #bcbd22; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#mf30c6b51d6\" x=\"129.55564\" y=\"199.54355\" style=\"fill: #bcbd22; stroke: #bcbd22; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#mf30c6b51d6\" x=\"142.005104\" y=\"204.025696\" style=\"fill: #bcbd22; stroke: #bcbd22; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#mf30c6b51d6\" x=\"135.539497\" y=\"201.097126\" style=\"fill: #bcbd22; stroke: #bcbd22; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#mf30c6b51d6\" x=\"201.488154\" y=\"302.135023\" style=\"fill: #bcbd22; stroke: #bcbd22; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#mf30c6b51d6\" x=\"132.039954\" y=\"194.467041\" style=\"fill: #bcbd22; stroke: #bcbd22; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#mf30c6b51d6\" x=\"137.479934\" y=\"195.74229\" style=\"fill: #bcbd22; stroke: #bcbd22; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#mf30c6b51d6\" x=\"138.337122\" y=\"200.562775\" style=\"fill: #bcbd22; stroke: #bcbd22; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#mf30c6b51d6\" x=\"132.995168\" y=\"202.019535\" style=\"fill: #bcbd22; stroke: #bcbd22; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#mf30c6b51d6\" x=\"125.95785\" y=\"199.174505\" style=\"fill: #bcbd22; stroke: #bcbd22; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#mf30c6b51d6\" x=\"141.35197\" y=\"203.900889\" style=\"fill: #bcbd22; stroke: #bcbd22; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#mf30c6b51d6\" x=\"131.100665\" y=\"206.752362\" style=\"fill: #bcbd22; stroke: #bcbd22; stroke-width: 0.5\"/>\n",
              "    </g>\n",
              "   </g>\n",
              "   <g id=\"PathCollection_18\">\n",
              "    <defs>\n",
              "     <path id=\"m8b31dfbfba\" d=\"M 0 3 \n",
              "C 0.795609 3 1.55874 2.683901 2.12132 2.12132 \n",
              "C 2.683901 1.55874 3 0.795609 3 0 \n",
              "C 3 -0.795609 2.683901 -1.55874 2.12132 -2.12132 \n",
              "C 1.55874 -2.683901 0.795609 -3 0 -3 \n",
              "C -0.795609 -3 -1.55874 -2.683901 -2.12132 -2.12132 \n",
              "C -2.683901 -1.55874 -3 -0.795609 -3 0 \n",
              "C -3 0.795609 -2.683901 1.55874 -2.12132 2.12132 \n",
              "C -1.55874 2.683901 -0.795609 3 0 3 \n",
              "z\n",
              "\" style=\"stroke: #dbdb8d; stroke-width: 0.5\"/>\n",
              "    </defs>\n",
              "    <g clip-path=\"url(#p596f914704)\">\n",
              "     <use xlink:href=\"#m8b31dfbfba\" x=\"572.750987\" y=\"174.413624\" style=\"fill: #dbdb8d; stroke: #dbdb8d; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m8b31dfbfba\" x=\"573.113666\" y=\"156.431635\" style=\"fill: #dbdb8d; stroke: #dbdb8d; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m8b31dfbfba\" x=\"547.448755\" y=\"176.000901\" style=\"fill: #dbdb8d; stroke: #dbdb8d; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m8b31dfbfba\" x=\"567.790677\" y=\"180.459004\" style=\"fill: #dbdb8d; stroke: #dbdb8d; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m8b31dfbfba\" x=\"563.100928\" y=\"166.268549\" style=\"fill: #dbdb8d; stroke: #dbdb8d; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m8b31dfbfba\" x=\"558.195152\" y=\"184.821885\" style=\"fill: #dbdb8d; stroke: #dbdb8d; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m8b31dfbfba\" x=\"561.142297\" y=\"173.219231\" style=\"fill: #dbdb8d; stroke: #dbdb8d; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m8b31dfbfba\" x=\"559.420643\" y=\"170.888664\" style=\"fill: #dbdb8d; stroke: #dbdb8d; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m8b31dfbfba\" x=\"574.107221\" y=\"179.659811\" style=\"fill: #dbdb8d; stroke: #dbdb8d; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m8b31dfbfba\" x=\"577.189898\" y=\"174.568955\" style=\"fill: #dbdb8d; stroke: #dbdb8d; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m8b31dfbfba\" x=\"569.6481\" y=\"185.200882\" style=\"fill: #dbdb8d; stroke: #dbdb8d; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m8b31dfbfba\" x=\"576.556375\" y=\"170.997197\" style=\"fill: #dbdb8d; stroke: #dbdb8d; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m8b31dfbfba\" x=\"560.21068\" y=\"182.26685\" style=\"fill: #dbdb8d; stroke: #dbdb8d; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m8b31dfbfba\" x=\"565.359559\" y=\"159.937089\" style=\"fill: #dbdb8d; stroke: #dbdb8d; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m8b31dfbfba\" x=\"555.220504\" y=\"188.226461\" style=\"fill: #dbdb8d; stroke: #dbdb8d; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m8b31dfbfba\" x=\"556.14215\" y=\"171.274187\" style=\"fill: #dbdb8d; stroke: #dbdb8d; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m8b31dfbfba\" x=\"582.074881\" y=\"163.626692\" style=\"fill: #dbdb8d; stroke: #dbdb8d; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m8b31dfbfba\" x=\"564.671912\" y=\"177.706174\" style=\"fill: #dbdb8d; stroke: #dbdb8d; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m8b31dfbfba\" x=\"566.089989\" y=\"169.556644\" style=\"fill: #dbdb8d; stroke: #dbdb8d; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m8b31dfbfba\" x=\"570.869054\" y=\"169.754206\" style=\"fill: #dbdb8d; stroke: #dbdb8d; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m8b31dfbfba\" x=\"564.390766\" y=\"174.810597\" style=\"fill: #dbdb8d; stroke: #dbdb8d; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m8b31dfbfba\" x=\"557.699919\" y=\"174.964438\" style=\"fill: #dbdb8d; stroke: #dbdb8d; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m8b31dfbfba\" x=\"565.80133\" y=\"176.513471\" style=\"fill: #dbdb8d; stroke: #dbdb8d; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m8b31dfbfba\" x=\"564.013202\" y=\"188.735026\" style=\"fill: #dbdb8d; stroke: #dbdb8d; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m8b31dfbfba\" x=\"559.64097\" y=\"191.787648\" style=\"fill: #dbdb8d; stroke: #dbdb8d; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m8b31dfbfba\" x=\"551.104844\" y=\"182.811694\" style=\"fill: #dbdb8d; stroke: #dbdb8d; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m8b31dfbfba\" x=\"570.613459\" y=\"164.38359\" style=\"fill: #dbdb8d; stroke: #dbdb8d; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m8b31dfbfba\" x=\"553.711127\" y=\"175.692265\" style=\"fill: #dbdb8d; stroke: #dbdb8d; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m8b31dfbfba\" x=\"559.5124\" y=\"160.347936\" style=\"fill: #dbdb8d; stroke: #dbdb8d; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m8b31dfbfba\" x=\"567.270791\" y=\"166.304519\" style=\"fill: #dbdb8d; stroke: #dbdb8d; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m8b31dfbfba\" x=\"556.801885\" y=\"180.328395\" style=\"fill: #dbdb8d; stroke: #dbdb8d; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m8b31dfbfba\" x=\"583.554801\" y=\"173.545935\" style=\"fill: #dbdb8d; stroke: #dbdb8d; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m8b31dfbfba\" x=\"550.013798\" y=\"178.649512\" style=\"fill: #dbdb8d; stroke: #dbdb8d; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m8b31dfbfba\" x=\"549.022244\" y=\"168.127584\" style=\"fill: #dbdb8d; stroke: #dbdb8d; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m8b31dfbfba\" x=\"563.838809\" y=\"171.523147\" style=\"fill: #dbdb8d; stroke: #dbdb8d; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m8b31dfbfba\" x=\"569.209369\" y=\"175.766492\" style=\"fill: #dbdb8d; stroke: #dbdb8d; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m8b31dfbfba\" x=\"562.533045\" y=\"176.592181\" style=\"fill: #dbdb8d; stroke: #dbdb8d; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m8b31dfbfba\" x=\"553.43718\" y=\"170.624455\" style=\"fill: #dbdb8d; stroke: #dbdb8d; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m8b31dfbfba\" x=\"559.706073\" y=\"177.451441\" style=\"fill: #dbdb8d; stroke: #dbdb8d; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m8b31dfbfba\" x=\"571.208152\" y=\"158.779744\" style=\"fill: #dbdb8d; stroke: #dbdb8d; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m8b31dfbfba\" x=\"546.71708\" y=\"170.736075\" style=\"fill: #dbdb8d; stroke: #dbdb8d; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m8b31dfbfba\" x=\"555.006495\" y=\"164.840199\" style=\"fill: #dbdb8d; stroke: #dbdb8d; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m8b31dfbfba\" x=\"564.34614\" y=\"184.759827\" style=\"fill: #dbdb8d; stroke: #dbdb8d; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m8b31dfbfba\" x=\"539.766982\" y=\"155.617393\" style=\"fill: #dbdb8d; stroke: #dbdb8d; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m8b31dfbfba\" x=\"559.414342\" y=\"150.53843\" style=\"fill: #dbdb8d; stroke: #dbdb8d; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m8b31dfbfba\" x=\"581.128755\" y=\"180.926853\" style=\"fill: #dbdb8d; stroke: #dbdb8d; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m8b31dfbfba\" x=\"566.452322\" y=\"174.034221\" style=\"fill: #dbdb8d; stroke: #dbdb8d; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m8b31dfbfba\" x=\"575.853779\" y=\"187.432608\" style=\"fill: #dbdb8d; stroke: #dbdb8d; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m8b31dfbfba\" x=\"546.873405\" y=\"162.534368\" style=\"fill: #dbdb8d; stroke: #dbdb8d; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m8b31dfbfba\" x=\"560.05415\" y=\"167.039091\" style=\"fill: #dbdb8d; stroke: #dbdb8d; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m8b31dfbfba\" x=\"578.065721\" y=\"164.66514\" style=\"fill: #dbdb8d; stroke: #dbdb8d; stroke-width: 0.5\"/>\n",
              "    </g>\n",
              "   </g>\n",
              "   <g id=\"PathCollection_19\">\n",
              "    <defs>\n",
              "     <path id=\"m35415c4a50\" d=\"M 0 3 \n",
              "C 0.795609 3 1.55874 2.683901 2.12132 2.12132 \n",
              "C 2.683901 1.55874 3 0.795609 3 0 \n",
              "C 3 -0.795609 2.683901 -1.55874 2.12132 -2.12132 \n",
              "C 1.55874 -2.683901 0.795609 -3 0 -3 \n",
              "C -0.795609 -3 -1.55874 -2.683901 -2.12132 -2.12132 \n",
              "C -2.683901 -1.55874 -3 -0.795609 -3 0 \n",
              "C -3 0.795609 -2.683901 1.55874 -2.12132 2.12132 \n",
              "C -1.55874 2.683901 -0.795609 3 0 3 \n",
              "z\n",
              "\" style=\"stroke: #17becf; stroke-width: 0.5\"/>\n",
              "    </defs>\n",
              "    <g clip-path=\"url(#p596f914704)\">\n",
              "     <use xlink:href=\"#m35415c4a50\" x=\"296.109879\" y=\"243.964197\" style=\"fill: #17becf; stroke: #17becf; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m35415c4a50\" x=\"250.293699\" y=\"216.501648\" style=\"fill: #17becf; stroke: #17becf; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m35415c4a50\" x=\"227.585377\" y=\"198.172672\" style=\"fill: #17becf; stroke: #17becf; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m35415c4a50\" x=\"265.858223\" y=\"172.181009\" style=\"fill: #17becf; stroke: #17becf; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m35415c4a50\" x=\"242.680354\" y=\"185.644356\" style=\"fill: #17becf; stroke: #17becf; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m35415c4a50\" x=\"229.945051\" y=\"159.429448\" style=\"fill: #17becf; stroke: #17becf; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m35415c4a50\" x=\"165.516183\" y=\"222.418991\" style=\"fill: #17becf; stroke: #17becf; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m35415c4a50\" x=\"308.093163\" y=\"125.150213\" style=\"fill: #17becf; stroke: #17becf; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m35415c4a50\" x=\"239.585202\" y=\"170.331309\" style=\"fill: #17becf; stroke: #17becf; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m35415c4a50\" x=\"237.58404\" y=\"193.444105\" style=\"fill: #17becf; stroke: #17becf; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m35415c4a50\" x=\"187.284335\" y=\"188.174211\" style=\"fill: #17becf; stroke: #17becf; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m35415c4a50\" x=\"218.175783\" y=\"162.335711\" style=\"fill: #17becf; stroke: #17becf; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m35415c4a50\" x=\"299.909631\" y=\"233.034011\" style=\"fill: #17becf; stroke: #17becf; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m35415c4a50\" x=\"245.88085\" y=\"169.931136\" style=\"fill: #17becf; stroke: #17becf; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m35415c4a50\" x=\"227.629759\" y=\"194.223383\" style=\"fill: #17becf; stroke: #17becf; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m35415c4a50\" x=\"230.436394\" y=\"176.1785\" style=\"fill: #17becf; stroke: #17becf; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m35415c4a50\" x=\"261.04652\" y=\"173.256321\" style=\"fill: #17becf; stroke: #17becf; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m35415c4a50\" x=\"231.496817\" y=\"201.624179\" style=\"fill: #17becf; stroke: #17becf; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m35415c4a50\" x=\"223.495377\" y=\"204.934845\" style=\"fill: #17becf; stroke: #17becf; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m35415c4a50\" x=\"209.219066\" y=\"203.530449\" style=\"fill: #17becf; stroke: #17becf; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m35415c4a50\" x=\"209.72713\" y=\"189.222168\" style=\"fill: #17becf; stroke: #17becf; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m35415c4a50\" x=\"222.552749\" y=\"213.936869\" style=\"fill: #17becf; stroke: #17becf; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m35415c4a50\" x=\"234.664588\" y=\"180.449777\" style=\"fill: #17becf; stroke: #17becf; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m35415c4a50\" x=\"266.53191\" y=\"156.68949\" style=\"fill: #17becf; stroke: #17becf; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m35415c4a50\" x=\"267.635317\" y=\"156.63738\" style=\"fill: #17becf; stroke: #17becf; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m35415c4a50\" x=\"240.982762\" y=\"300.279261\" style=\"fill: #17becf; stroke: #17becf; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m35415c4a50\" x=\"241.805319\" y=\"185.990364\" style=\"fill: #17becf; stroke: #17becf; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m35415c4a50\" x=\"245.301058\" y=\"222.680729\" style=\"fill: #17becf; stroke: #17becf; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m35415c4a50\" x=\"259.672816\" y=\"156.812627\" style=\"fill: #17becf; stroke: #17becf; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m35415c4a50\" x=\"219.804838\" y=\"190.777994\" style=\"fill: #17becf; stroke: #17becf; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m35415c4a50\" x=\"329.402847\" y=\"181.240957\" style=\"fill: #17becf; stroke: #17becf; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m35415c4a50\" x=\"243.793036\" y=\"205.586301\" style=\"fill: #17becf; stroke: #17becf; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m35415c4a50\" x=\"169.73009\" y=\"281.041352\" style=\"fill: #17becf; stroke: #17becf; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m35415c4a50\" x=\"126.61404\" y=\"290.389754\" style=\"fill: #17becf; stroke: #17becf; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m35415c4a50\" x=\"290.773155\" y=\"241.061397\" style=\"fill: #17becf; stroke: #17becf; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m35415c4a50\" x=\"262.931111\" y=\"167.005641\" style=\"fill: #17becf; stroke: #17becf; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m35415c4a50\" x=\"203.363456\" y=\"183.26972\" style=\"fill: #17becf; stroke: #17becf; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m35415c4a50\" x=\"181.245369\" y=\"313.953458\" style=\"fill: #17becf; stroke: #17becf; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m35415c4a50\" x=\"218.457126\" y=\"162.755028\" style=\"fill: #17becf; stroke: #17becf; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m35415c4a50\" x=\"250.996492\" y=\"161.429058\" style=\"fill: #17becf; stroke: #17becf; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m35415c4a50\" x=\"247.32751\" y=\"198.923206\" style=\"fill: #17becf; stroke: #17becf; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m35415c4a50\" x=\"233.26446\" y=\"210.348274\" style=\"fill: #17becf; stroke: #17becf; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m35415c4a50\" x=\"247.359266\" y=\"198.690614\" style=\"fill: #17becf; stroke: #17becf; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m35415c4a50\" x=\"229.084759\" y=\"185.81513\" style=\"fill: #17becf; stroke: #17becf; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m35415c4a50\" x=\"213.946243\" y=\"151.287609\" style=\"fill: #17becf; stroke: #17becf; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m35415c4a50\" x=\"204.70749\" y=\"199.552234\" style=\"fill: #17becf; stroke: #17becf; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m35415c4a50\" x=\"138.746632\" y=\"310.371331\" style=\"fill: #17becf; stroke: #17becf; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m35415c4a50\" x=\"229.73835\" y=\"159.571528\" style=\"fill: #17becf; stroke: #17becf; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m35415c4a50\" x=\"142.12312\" y=\"217.856063\" style=\"fill: #17becf; stroke: #17becf; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m35415c4a50\" x=\"228.316232\" y=\"183.365996\" style=\"fill: #17becf; stroke: #17becf; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m35415c4a50\" x=\"232.944477\" y=\"185.133982\" style=\"fill: #17becf; stroke: #17becf; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m35415c4a50\" x=\"229.566407\" y=\"191.148147\" style=\"fill: #17becf; stroke: #17becf; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m35415c4a50\" x=\"253.006223\" y=\"182.287603\" style=\"fill: #17becf; stroke: #17becf; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m35415c4a50\" x=\"249.992934\" y=\"189.305527\" style=\"fill: #17becf; stroke: #17becf; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m35415c4a50\" x=\"209.195036\" y=\"203.97164\" style=\"fill: #17becf; stroke: #17becf; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m35415c4a50\" x=\"433.795491\" y=\"115.934965\" style=\"fill: #17becf; stroke: #17becf; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m35415c4a50\" x=\"249.059726\" y=\"169.376494\" style=\"fill: #17becf; stroke: #17becf; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m35415c4a50\" x=\"240.909624\" y=\"90.634111\" style=\"fill: #17becf; stroke: #17becf; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m35415c4a50\" x=\"287.676695\" y=\"161.396524\" style=\"fill: #17becf; stroke: #17becf; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m35415c4a50\" x=\"212.71182\" y=\"148.698634\" style=\"fill: #17becf; stroke: #17becf; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m35415c4a50\" x=\"257.861595\" y=\"193.980563\" style=\"fill: #17becf; stroke: #17becf; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m35415c4a50\" x=\"209.61332\" y=\"213.937796\" style=\"fill: #17becf; stroke: #17becf; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m35415c4a50\" x=\"242.327158\" y=\"158.353186\" style=\"fill: #17becf; stroke: #17becf; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m35415c4a50\" x=\"229.832312\" y=\"195.869941\" style=\"fill: #17becf; stroke: #17becf; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m35415c4a50\" x=\"287.25957\" y=\"238.382919\" style=\"fill: #17becf; stroke: #17becf; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m35415c4a50\" x=\"268.946531\" y=\"213.317037\" style=\"fill: #17becf; stroke: #17becf; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m35415c4a50\" x=\"220.023959\" y=\"180.781875\" style=\"fill: #17becf; stroke: #17becf; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m35415c4a50\" x=\"239.571521\" y=\"167.071414\" style=\"fill: #17becf; stroke: #17becf; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m35415c4a50\" x=\"259.635799\" y=\"175.529573\" style=\"fill: #17becf; stroke: #17becf; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m35415c4a50\" x=\"243.824257\" y=\"220.83384\" style=\"fill: #17becf; stroke: #17becf; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m35415c4a50\" x=\"250.404382\" y=\"185.993635\" style=\"fill: #17becf; stroke: #17becf; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m35415c4a50\" x=\"187.258753\" y=\"188.184867\" style=\"fill: #17becf; stroke: #17becf; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m35415c4a50\" x=\"234.173772\" y=\"189.223241\" style=\"fill: #17becf; stroke: #17becf; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m35415c4a50\" x=\"182.420767\" y=\"322.501889\" style=\"fill: #17becf; stroke: #17becf; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m35415c4a50\" x=\"221.64246\" y=\"173.817881\" style=\"fill: #17becf; stroke: #17becf; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m35415c4a50\" x=\"242.140313\" y=\"177.902259\" style=\"fill: #17becf; stroke: #17becf; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m35415c4a50\" x=\"206.003543\" y=\"168.384195\" style=\"fill: #17becf; stroke: #17becf; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m35415c4a50\" x=\"294.854225\" y=\"115.01078\" style=\"fill: #17becf; stroke: #17becf; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m35415c4a50\" x=\"220.694373\" y=\"190.174916\" style=\"fill: #17becf; stroke: #17becf; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m35415c4a50\" x=\"244.262814\" y=\"142.972318\" style=\"fill: #17becf; stroke: #17becf; stroke-width: 0.5\"/>\n",
              "    </g>\n",
              "   </g>\n",
              "   <g id=\"PathCollection_20\">\n",
              "    <defs>\n",
              "     <path id=\"m4960ccf95d\" d=\"M 0 3 \n",
              "C 0.795609 3 1.55874 2.683901 2.12132 2.12132 \n",
              "C 2.683901 1.55874 3 0.795609 3 0 \n",
              "C 3 -0.795609 2.683901 -1.55874 2.12132 -2.12132 \n",
              "C 1.55874 -2.683901 0.795609 -3 0 -3 \n",
              "C -0.795609 -3 -1.55874 -2.683901 -2.12132 -2.12132 \n",
              "C -2.683901 -1.55874 -3 -0.795609 -3 0 \n",
              "C -3 0.795609 -2.683901 1.55874 -2.12132 2.12132 \n",
              "C -1.55874 2.683901 -0.795609 3 0 3 \n",
              "z\n",
              "\" style=\"stroke: #9edae5; stroke-width: 0.5\"/>\n",
              "    </defs>\n",
              "    <g clip-path=\"url(#p596f914704)\">\n",
              "     <use xlink:href=\"#m4960ccf95d\" x=\"346.299361\" y=\"289.787209\" style=\"fill: #9edae5; stroke: #9edae5; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m4960ccf95d\" x=\"345.636407\" y=\"306.863331\" style=\"fill: #9edae5; stroke: #9edae5; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m4960ccf95d\" x=\"338.113984\" y=\"299.659304\" style=\"fill: #9edae5; stroke: #9edae5; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m4960ccf95d\" x=\"345.690504\" y=\"293.507876\" style=\"fill: #9edae5; stroke: #9edae5; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m4960ccf95d\" x=\"338.751721\" y=\"296.68293\" style=\"fill: #9edae5; stroke: #9edae5; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m4960ccf95d\" x=\"336.700237\" y=\"303.801969\" style=\"fill: #9edae5; stroke: #9edae5; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m4960ccf95d\" x=\"333.138387\" y=\"298.937794\" style=\"fill: #9edae5; stroke: #9edae5; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m4960ccf95d\" x=\"340.866912\" y=\"302.861182\" style=\"fill: #9edae5; stroke: #9edae5; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m4960ccf95d\" x=\"340.55007\" y=\"314.930594\" style=\"fill: #9edae5; stroke: #9edae5; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m4960ccf95d\" x=\"340.380297\" y=\"307.780936\" style=\"fill: #9edae5; stroke: #9edae5; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m4960ccf95d\" x=\"363.070077\" y=\"293.068555\" style=\"fill: #9edae5; stroke: #9edae5; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m4960ccf95d\" x=\"332.110881\" y=\"298.985141\" style=\"fill: #9edae5; stroke: #9edae5; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m4960ccf95d\" x=\"355.618251\" y=\"309.156356\" style=\"fill: #9edae5; stroke: #9edae5; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m4960ccf95d\" x=\"333.683512\" y=\"291.583711\" style=\"fill: #9edae5; stroke: #9edae5; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m4960ccf95d\" x=\"350.700426\" y=\"302.748347\" style=\"fill: #9edae5; stroke: #9edae5; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m4960ccf95d\" x=\"346.026515\" y=\"304.573987\" style=\"fill: #9edae5; stroke: #9edae5; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m4960ccf95d\" x=\"343.821451\" y=\"308.508568\" style=\"fill: #9edae5; stroke: #9edae5; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m4960ccf95d\" x=\"333.159517\" y=\"307.192499\" style=\"fill: #9edae5; stroke: #9edae5; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m4960ccf95d\" x=\"346.78269\" y=\"311.995317\" style=\"fill: #9edae5; stroke: #9edae5; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m4960ccf95d\" x=\"339.01014\" y=\"309.46878\" style=\"fill: #9edae5; stroke: #9edae5; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m4960ccf95d\" x=\"346.412838\" y=\"300.879094\" style=\"fill: #9edae5; stroke: #9edae5; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m4960ccf95d\" x=\"349.192362\" y=\"315.609135\" style=\"fill: #9edae5; stroke: #9edae5; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m4960ccf95d\" x=\"341.471772\" y=\"294.314171\" style=\"fill: #9edae5; stroke: #9edae5; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m4960ccf95d\" x=\"342.531526\" y=\"301.046014\" style=\"fill: #9edae5; stroke: #9edae5; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m4960ccf95d\" x=\"342.760825\" y=\"304.011556\" style=\"fill: #9edae5; stroke: #9edae5; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m4960ccf95d\" x=\"350.391775\" y=\"309.385812\" style=\"fill: #9edae5; stroke: #9edae5; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m4960ccf95d\" x=\"354.287075\" y=\"304.018797\" style=\"fill: #9edae5; stroke: #9edae5; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m4960ccf95d\" x=\"431.957467\" y=\"138.864406\" style=\"fill: #9edae5; stroke: #9edae5; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m4960ccf95d\" x=\"319.027502\" y=\"294.418592\" style=\"fill: #9edae5; stroke: #9edae5; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m4960ccf95d\" x=\"363.249388\" y=\"292.799016\" style=\"fill: #9edae5; stroke: #9edae5; stroke-width: 0.5\"/>\n",
              "    </g>\n",
              "   </g>\n",
              "   <g id=\"PathCollection_21\">\n",
              "    <defs>\n",
              "     <path id=\"m6c29e73512\" d=\"M 0 3 \n",
              "C 0.795609 3 1.55874 2.683901 2.12132 2.12132 \n",
              "C 2.683901 1.55874 3 0.795609 3 0 \n",
              "C 3 -0.795609 2.683901 -1.55874 2.12132 -2.12132 \n",
              "C 1.55874 -2.683901 0.795609 -3 0 -3 \n",
              "C -0.795609 -3 -1.55874 -2.683901 -2.12132 -2.12132 \n",
              "C -2.683901 -1.55874 -3 -0.795609 -3 0 \n",
              "C -3 0.795609 -2.683901 1.55874 -2.12132 2.12132 \n",
              "C -1.55874 2.683901 -0.795609 3 0 3 \n",
              "z\n",
              "\" style=\"stroke: #393b79; stroke-width: 0.5\"/>\n",
              "    </defs>\n",
              "    <g clip-path=\"url(#p596f914704)\">\n",
              "     <use xlink:href=\"#m6c29e73512\" x=\"246.908241\" y=\"238.792774\" style=\"fill: #393b79; stroke: #393b79; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m6c29e73512\" x=\"265.806347\" y=\"266.622668\" style=\"fill: #393b79; stroke: #393b79; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m6c29e73512\" x=\"269.824481\" y=\"233.507616\" style=\"fill: #393b79; stroke: #393b79; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m6c29e73512\" x=\"268.160953\" y=\"251.953284\" style=\"fill: #393b79; stroke: #393b79; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m6c29e73512\" x=\"292.021888\" y=\"275.01293\" style=\"fill: #393b79; stroke: #393b79; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m6c29e73512\" x=\"244.324193\" y=\"331.584402\" style=\"fill: #393b79; stroke: #393b79; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m6c29e73512\" x=\"205.591008\" y=\"257.11902\" style=\"fill: #393b79; stroke: #393b79; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m6c29e73512\" x=\"223.93488\" y=\"295.212891\" style=\"fill: #393b79; stroke: #393b79; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m6c29e73512\" x=\"216.299946\" y=\"243.120647\" style=\"fill: #393b79; stroke: #393b79; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m6c29e73512\" x=\"200.549196\" y=\"342.106974\" style=\"fill: #393b79; stroke: #393b79; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m6c29e73512\" x=\"294.579975\" y=\"250.319022\" style=\"fill: #393b79; stroke: #393b79; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m6c29e73512\" x=\"245.401471\" y=\"242.344553\" style=\"fill: #393b79; stroke: #393b79; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m6c29e73512\" x=\"275.752382\" y=\"229.390536\" style=\"fill: #393b79; stroke: #393b79; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m6c29e73512\" x=\"267.568823\" y=\"237.761527\" style=\"fill: #393b79; stroke: #393b79; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m6c29e73512\" x=\"281.699639\" y=\"248.048619\" style=\"fill: #393b79; stroke: #393b79; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m6c29e73512\" x=\"264.900949\" y=\"231.537978\" style=\"fill: #393b79; stroke: #393b79; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m6c29e73512\" x=\"271.236805\" y=\"266.877836\" style=\"fill: #393b79; stroke: #393b79; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m6c29e73512\" x=\"300.510332\" y=\"229.42162\" style=\"fill: #393b79; stroke: #393b79; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m6c29e73512\" x=\"213.753711\" y=\"280.369879\" style=\"fill: #393b79; stroke: #393b79; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m6c29e73512\" x=\"267.440088\" y=\"235.660692\" style=\"fill: #393b79; stroke: #393b79; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m6c29e73512\" x=\"217.960687\" y=\"280.672124\" style=\"fill: #393b79; stroke: #393b79; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m6c29e73512\" x=\"267.731098\" y=\"272.159404\" style=\"fill: #393b79; stroke: #393b79; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m6c29e73512\" x=\"259.408687\" y=\"220.467316\" style=\"fill: #393b79; stroke: #393b79; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m6c29e73512\" x=\"301.063004\" y=\"245.098071\" style=\"fill: #393b79; stroke: #393b79; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m6c29e73512\" x=\"287.69062\" y=\"245.885526\" style=\"fill: #393b79; stroke: #393b79; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m6c29e73512\" x=\"200.55393\" y=\"342.117656\" style=\"fill: #393b79; stroke: #393b79; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m6c29e73512\" x=\"301.51382\" y=\"282.42345\" style=\"fill: #393b79; stroke: #393b79; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m6c29e73512\" x=\"247.034212\" y=\"235.815112\" style=\"fill: #393b79; stroke: #393b79; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m6c29e73512\" x=\"217.301134\" y=\"249.852527\" style=\"fill: #393b79; stroke: #393b79; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m6c29e73512\" x=\"214.947115\" y=\"280.028633\" style=\"fill: #393b79; stroke: #393b79; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m6c29e73512\" x=\"271.285562\" y=\"254.40157\" style=\"fill: #393b79; stroke: #393b79; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m6c29e73512\" x=\"285.908528\" y=\"220.663949\" style=\"fill: #393b79; stroke: #393b79; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m6c29e73512\" x=\"227.823086\" y=\"282.699997\" style=\"fill: #393b79; stroke: #393b79; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m6c29e73512\" x=\"201.127783\" y=\"254.717554\" style=\"fill: #393b79; stroke: #393b79; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m6c29e73512\" x=\"296.566863\" y=\"247.392834\" style=\"fill: #393b79; stroke: #393b79; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m6c29e73512\" x=\"268.044748\" y=\"252.209102\" style=\"fill: #393b79; stroke: #393b79; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m6c29e73512\" x=\"245.177766\" y=\"264.652696\" style=\"fill: #393b79; stroke: #393b79; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m6c29e73512\" x=\"228.905499\" y=\"250.761145\" style=\"fill: #393b79; stroke: #393b79; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m6c29e73512\" x=\"226.981776\" y=\"295.33471\" style=\"fill: #393b79; stroke: #393b79; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m6c29e73512\" x=\"205.851069\" y=\"251.765113\" style=\"fill: #393b79; stroke: #393b79; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m6c29e73512\" x=\"298.22818\" y=\"253.670686\" style=\"fill: #393b79; stroke: #393b79; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m6c29e73512\" x=\"203.744013\" y=\"254.920403\" style=\"fill: #393b79; stroke: #393b79; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m6c29e73512\" x=\"201.642471\" y=\"233.074444\" style=\"fill: #393b79; stroke: #393b79; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m6c29e73512\" x=\"224.473369\" y=\"295.087166\" style=\"fill: #393b79; stroke: #393b79; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m6c29e73512\" x=\"299.866064\" y=\"251.152469\" style=\"fill: #393b79; stroke: #393b79; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m6c29e73512\" x=\"262.288369\" y=\"170.9713\" style=\"fill: #393b79; stroke: #393b79; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m6c29e73512\" x=\"220.947212\" y=\"280.192661\" style=\"fill: #393b79; stroke: #393b79; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m6c29e73512\" x=\"264.762987\" y=\"170.065484\" style=\"fill: #393b79; stroke: #393b79; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m6c29e73512\" x=\"217.477976\" y=\"280.901625\" style=\"fill: #393b79; stroke: #393b79; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m6c29e73512\" x=\"246.211237\" y=\"264.366916\" style=\"fill: #393b79; stroke: #393b79; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m6c29e73512\" x=\"267.506137\" y=\"303.993129\" style=\"fill: #393b79; stroke: #393b79; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m6c29e73512\" x=\"294.608315\" y=\"243.377851\" style=\"fill: #393b79; stroke: #393b79; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m6c29e73512\" x=\"272.749407\" y=\"265.190241\" style=\"fill: #393b79; stroke: #393b79; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m6c29e73512\" x=\"294.119729\" y=\"274.691688\" style=\"fill: #393b79; stroke: #393b79; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m6c29e73512\" x=\"267.756345\" y=\"251.224281\" style=\"fill: #393b79; stroke: #393b79; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m6c29e73512\" x=\"258.727238\" y=\"232.811534\" style=\"fill: #393b79; stroke: #393b79; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m6c29e73512\" x=\"300.321975\" y=\"244.387472\" style=\"fill: #393b79; stroke: #393b79; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m6c29e73512\" x=\"237.425132\" y=\"262.968976\" style=\"fill: #393b79; stroke: #393b79; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m6c29e73512\" x=\"299.134896\" y=\"247.283248\" style=\"fill: #393b79; stroke: #393b79; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m6c29e73512\" x=\"205.434368\" y=\"205.254862\" style=\"fill: #393b79; stroke: #393b79; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m6c29e73512\" x=\"277.205028\" y=\"257.722286\" style=\"fill: #393b79; stroke: #393b79; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m6c29e73512\" x=\"273.961387\" y=\"257.130776\" style=\"fill: #393b79; stroke: #393b79; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m6c29e73512\" x=\"255.253761\" y=\"208.420633\" style=\"fill: #393b79; stroke: #393b79; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m6c29e73512\" x=\"213.183237\" y=\"233.206186\" style=\"fill: #393b79; stroke: #393b79; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m6c29e73512\" x=\"295.530064\" y=\"239.419947\" style=\"fill: #393b79; stroke: #393b79; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m6c29e73512\" x=\"300.571673\" y=\"228.527286\" style=\"fill: #393b79; stroke: #393b79; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m6c29e73512\" x=\"287.429336\" y=\"262.489114\" style=\"fill: #393b79; stroke: #393b79; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m6c29e73512\" x=\"309.279122\" y=\"169.685319\" style=\"fill: #393b79; stroke: #393b79; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m6c29e73512\" x=\"292.330921\" y=\"280.067176\" style=\"fill: #393b79; stroke: #393b79; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m6c29e73512\" x=\"250.127884\" y=\"229.943871\" style=\"fill: #393b79; stroke: #393b79; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m6c29e73512\" x=\"134.106361\" y=\"165.828405\" style=\"fill: #393b79; stroke: #393b79; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m6c29e73512\" x=\"265.165314\" y=\"291.661473\" style=\"fill: #393b79; stroke: #393b79; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m6c29e73512\" x=\"257.38973\" y=\"242.038943\" style=\"fill: #393b79; stroke: #393b79; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m6c29e73512\" x=\"208.944394\" y=\"245.753465\" style=\"fill: #393b79; stroke: #393b79; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m6c29e73512\" x=\"76.282074\" y=\"185.972388\" style=\"fill: #393b79; stroke: #393b79; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m6c29e73512\" x=\"292.37435\" y=\"246.274104\" style=\"fill: #393b79; stroke: #393b79; stroke-width: 0.5\"/>\n",
              "    </g>\n",
              "   </g>\n",
              "   <g id=\"PathCollection_22\">\n",
              "    <defs>\n",
              "     <path id=\"m71beae8f00\" d=\"M 0 3 \n",
              "C 0.795609 3 1.55874 2.683901 2.12132 2.12132 \n",
              "C 2.683901 1.55874 3 0.795609 3 0 \n",
              "C 3 -0.795609 2.683901 -1.55874 2.12132 -2.12132 \n",
              "C 1.55874 -2.683901 0.795609 -3 0 -3 \n",
              "C -0.795609 -3 -1.55874 -2.683901 -2.12132 -2.12132 \n",
              "C -2.683901 -1.55874 -3 -0.795609 -3 0 \n",
              "C -3 0.795609 -2.683901 1.55874 -2.12132 2.12132 \n",
              "C -1.55874 2.683901 -0.795609 3 0 3 \n",
              "z\n",
              "\" style=\"stroke: #5254a3; stroke-width: 0.5\"/>\n",
              "    </defs>\n",
              "    <g clip-path=\"url(#p596f914704)\">\n",
              "     <use xlink:href=\"#m71beae8f00\" x=\"296.071135\" y=\"204.220924\" style=\"fill: #5254a3; stroke: #5254a3; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m71beae8f00\" x=\"173.09878\" y=\"284.665087\" style=\"fill: #5254a3; stroke: #5254a3; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m71beae8f00\" x=\"176.962656\" y=\"294.652525\" style=\"fill: #5254a3; stroke: #5254a3; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m71beae8f00\" x=\"297.306002\" y=\"212.391573\" style=\"fill: #5254a3; stroke: #5254a3; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m71beae8f00\" x=\"194.060268\" y=\"299.016037\" style=\"fill: #5254a3; stroke: #5254a3; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m71beae8f00\" x=\"176.302213\" y=\"297.80587\" style=\"fill: #5254a3; stroke: #5254a3; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m71beae8f00\" x=\"292.767953\" y=\"202.906727\" style=\"fill: #5254a3; stroke: #5254a3; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m71beae8f00\" x=\"171.296758\" y=\"290.814562\" style=\"fill: #5254a3; stroke: #5254a3; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m71beae8f00\" x=\"175.837019\" y=\"291.895826\" style=\"fill: #5254a3; stroke: #5254a3; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m71beae8f00\" x=\"229.139175\" y=\"265.309205\" style=\"fill: #5254a3; stroke: #5254a3; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m71beae8f00\" x=\"177.034581\" y=\"293.674022\" style=\"fill: #5254a3; stroke: #5254a3; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m71beae8f00\" x=\"181.182643\" y=\"286.161965\" style=\"fill: #5254a3; stroke: #5254a3; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m71beae8f00\" x=\"295.290166\" y=\"205.657063\" style=\"fill: #5254a3; stroke: #5254a3; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m71beae8f00\" x=\"178.892334\" y=\"284.254942\" style=\"fill: #5254a3; stroke: #5254a3; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m71beae8f00\" x=\"295.842086\" y=\"202.057081\" style=\"fill: #5254a3; stroke: #5254a3; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m71beae8f00\" x=\"180.455332\" y=\"309.439206\" style=\"fill: #5254a3; stroke: #5254a3; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m71beae8f00\" x=\"187.026684\" y=\"284.856127\" style=\"fill: #5254a3; stroke: #5254a3; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m71beae8f00\" x=\"291.069698\" y=\"210.844555\" style=\"fill: #5254a3; stroke: #5254a3; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m71beae8f00\" x=\"275.715963\" y=\"185.153634\" style=\"fill: #5254a3; stroke: #5254a3; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m71beae8f00\" x=\"242.478183\" y=\"244.210167\" style=\"fill: #5254a3; stroke: #5254a3; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m71beae8f00\" x=\"294.138604\" y=\"210.676049\" style=\"fill: #5254a3; stroke: #5254a3; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m71beae8f00\" x=\"180.24849\" y=\"283.224216\" style=\"fill: #5254a3; stroke: #5254a3; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m71beae8f00\" x=\"297.338655\" y=\"204.389567\" style=\"fill: #5254a3; stroke: #5254a3; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m71beae8f00\" x=\"186.98053\" y=\"284.547174\" style=\"fill: #5254a3; stroke: #5254a3; stroke-width: 0.5\"/>\n",
              "    </g>\n",
              "   </g>\n",
              "   <g id=\"PathCollection_23\">\n",
              "    <defs>\n",
              "     <path id=\"m257374e8e1\" d=\"M 0 3 \n",
              "C 0.795609 3 1.55874 2.683901 2.12132 2.12132 \n",
              "C 2.683901 1.55874 3 0.795609 3 0 \n",
              "C 3 -0.795609 2.683901 -1.55874 2.12132 -2.12132 \n",
              "C 1.55874 -2.683901 0.795609 -3 0 -3 \n",
              "C -0.795609 -3 -1.55874 -2.683901 -2.12132 -2.12132 \n",
              "C -2.683901 -1.55874 -3 -0.795609 -3 0 \n",
              "C -3 0.795609 -2.683901 1.55874 -2.12132 2.12132 \n",
              "C -1.55874 2.683901 -0.795609 3 0 3 \n",
              "z\n",
              "\" style=\"stroke: #6b6ecf; stroke-width: 0.5\"/>\n",
              "    </defs>\n",
              "    <g clip-path=\"url(#p596f914704)\">\n",
              "     <use xlink:href=\"#m257374e8e1\" x=\"177.205272\" y=\"76.753692\" style=\"fill: #6b6ecf; stroke: #6b6ecf; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m257374e8e1\" x=\"183.591031\" y=\"73.479295\" style=\"fill: #6b6ecf; stroke: #6b6ecf; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m257374e8e1\" x=\"177.802045\" y=\"59.031794\" style=\"fill: #6b6ecf; stroke: #6b6ecf; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m257374e8e1\" x=\"189.010683\" y=\"68.484144\" style=\"fill: #6b6ecf; stroke: #6b6ecf; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m257374e8e1\" x=\"183.294432\" y=\"70.826657\" style=\"fill: #6b6ecf; stroke: #6b6ecf; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m257374e8e1\" x=\"175.539444\" y=\"66.163491\" style=\"fill: #6b6ecf; stroke: #6b6ecf; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m257374e8e1\" x=\"265.277565\" y=\"82.518007\" style=\"fill: #6b6ecf; stroke: #6b6ecf; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m257374e8e1\" x=\"179.79826\" y=\"69.573019\" style=\"fill: #6b6ecf; stroke: #6b6ecf; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m257374e8e1\" x=\"202.565204\" y=\"95.256127\" style=\"fill: #6b6ecf; stroke: #6b6ecf; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m257374e8e1\" x=\"204.58441\" y=\"86.045582\" style=\"fill: #6b6ecf; stroke: #6b6ecf; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m257374e8e1\" x=\"184.016482\" y=\"76.792904\" style=\"fill: #6b6ecf; stroke: #6b6ecf; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m257374e8e1\" x=\"176.293549\" y=\"72.074917\" style=\"fill: #6b6ecf; stroke: #6b6ecf; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m257374e8e1\" x=\"176.638806\" y=\"81.745747\" style=\"fill: #6b6ecf; stroke: #6b6ecf; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m257374e8e1\" x=\"180.655669\" y=\"72.922031\" style=\"fill: #6b6ecf; stroke: #6b6ecf; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m257374e8e1\" x=\"174.043739\" y=\"68.05818\" style=\"fill: #6b6ecf; stroke: #6b6ecf; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m257374e8e1\" x=\"199.857792\" y=\"92.482744\" style=\"fill: #6b6ecf; stroke: #6b6ecf; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m257374e8e1\" x=\"205.166564\" y=\"84.235567\" style=\"fill: #6b6ecf; stroke: #6b6ecf; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m257374e8e1\" x=\"170.147035\" y=\"67.000059\" style=\"fill: #6b6ecf; stroke: #6b6ecf; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m257374e8e1\" x=\"177.770194\" y=\"58.784055\" style=\"fill: #6b6ecf; stroke: #6b6ecf; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m257374e8e1\" x=\"145.082126\" y=\"313.329211\" style=\"fill: #6b6ecf; stroke: #6b6ecf; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m257374e8e1\" x=\"182.780863\" y=\"71.144436\" style=\"fill: #6b6ecf; stroke: #6b6ecf; stroke-width: 0.5\"/>\n",
              "    </g>\n",
              "   </g>\n",
              "   <g id=\"PathCollection_24\">\n",
              "    <defs>\n",
              "     <path id=\"ma20c134c12\" d=\"M 0 3 \n",
              "C 0.795609 3 1.55874 2.683901 2.12132 2.12132 \n",
              "C 2.683901 1.55874 3 0.795609 3 0 \n",
              "C 3 -0.795609 2.683901 -1.55874 2.12132 -2.12132 \n",
              "C 1.55874 -2.683901 0.795609 -3 0 -3 \n",
              "C -0.795609 -3 -1.55874 -2.683901 -2.12132 -2.12132 \n",
              "C -2.683901 -1.55874 -3 -0.795609 -3 0 \n",
              "C -3 0.795609 -2.683901 1.55874 -2.12132 2.12132 \n",
              "C -1.55874 2.683901 -0.795609 3 0 3 \n",
              "z\n",
              "\" style=\"stroke: #9c9ede; stroke-width: 0.5\"/>\n",
              "    </defs>\n",
              "    <g clip-path=\"url(#p596f914704)\">\n",
              "     <use xlink:href=\"#ma20c134c12\" x=\"160.058041\" y=\"160.065678\" style=\"fill: #9c9ede; stroke: #9c9ede; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#ma20c134c12\" x=\"337.672467\" y=\"221.128977\" style=\"fill: #9c9ede; stroke: #9c9ede; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#ma20c134c12\" x=\"144.73605\" y=\"185.169559\" style=\"fill: #9c9ede; stroke: #9c9ede; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#ma20c134c12\" x=\"360.205052\" y=\"130.723318\" style=\"fill: #9c9ede; stroke: #9c9ede; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#ma20c134c12\" x=\"152.694699\" y=\"187.196976\" style=\"fill: #9c9ede; stroke: #9c9ede; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#ma20c134c12\" x=\"191.773574\" y=\"119.951207\" style=\"fill: #9c9ede; stroke: #9c9ede; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#ma20c134c12\" x=\"148.848181\" y=\"185.851583\" style=\"fill: #9c9ede; stroke: #9c9ede; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#ma20c134c12\" x=\"159.583836\" y=\"161.529241\" style=\"fill: #9c9ede; stroke: #9c9ede; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#ma20c134c12\" x=\"150.450922\" y=\"192.167146\" style=\"fill: #9c9ede; stroke: #9c9ede; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#ma20c134c12\" x=\"325.899589\" y=\"122.587919\" style=\"fill: #9c9ede; stroke: #9c9ede; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#ma20c134c12\" x=\"169.12511\" y=\"202.564408\" style=\"fill: #9c9ede; stroke: #9c9ede; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#ma20c134c12\" x=\"161.873877\" y=\"162.124263\" style=\"fill: #9c9ede; stroke: #9c9ede; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#ma20c134c12\" x=\"82.916356\" y=\"206.195524\" style=\"fill: #9c9ede; stroke: #9c9ede; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#ma20c134c12\" x=\"421.401632\" y=\"259.484165\" style=\"fill: #9c9ede; stroke: #9c9ede; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#ma20c134c12\" x=\"126.352805\" y=\"219.753959\" style=\"fill: #9c9ede; stroke: #9c9ede; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#ma20c134c12\" x=\"201.814083\" y=\"98.175704\" style=\"fill: #9c9ede; stroke: #9c9ede; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#ma20c134c12\" x=\"413.260848\" y=\"274.566398\" style=\"fill: #9c9ede; stroke: #9c9ede; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#ma20c134c12\" x=\"160.585362\" y=\"185.9444\" style=\"fill: #9c9ede; stroke: #9c9ede; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#ma20c134c12\" x=\"201.438314\" y=\"101.260038\" style=\"fill: #9c9ede; stroke: #9c9ede; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#ma20c134c12\" x=\"364.655467\" y=\"218.951871\" style=\"fill: #9c9ede; stroke: #9c9ede; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#ma20c134c12\" x=\"132.863086\" y=\"134.348663\" style=\"fill: #9c9ede; stroke: #9c9ede; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#ma20c134c12\" x=\"155.860107\" y=\"162.525649\" style=\"fill: #9c9ede; stroke: #9c9ede; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#ma20c134c12\" x=\"116.221611\" y=\"229.171882\" style=\"fill: #9c9ede; stroke: #9c9ede; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#ma20c134c12\" x=\"368.724031\" y=\"125.721643\" style=\"fill: #9c9ede; stroke: #9c9ede; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#ma20c134c12\" x=\"152.845668\" y=\"279.867189\" style=\"fill: #9c9ede; stroke: #9c9ede; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#ma20c134c12\" x=\"142.746277\" y=\"284.364539\" style=\"fill: #9c9ede; stroke: #9c9ede; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#ma20c134c12\" x=\"142.096388\" y=\"284.271783\" style=\"fill: #9c9ede; stroke: #9c9ede; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#ma20c134c12\" x=\"153.851148\" y=\"283.909804\" style=\"fill: #9c9ede; stroke: #9c9ede; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#ma20c134c12\" x=\"157.973707\" y=\"299.879197\" style=\"fill: #9c9ede; stroke: #9c9ede; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#ma20c134c12\" x=\"200.618136\" y=\"196.929448\" style=\"fill: #9c9ede; stroke: #9c9ede; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#ma20c134c12\" x=\"147.197664\" y=\"277.669895\" style=\"fill: #9c9ede; stroke: #9c9ede; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#ma20c134c12\" x=\"153.593836\" y=\"271.531905\" style=\"fill: #9c9ede; stroke: #9c9ede; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#ma20c134c12\" x=\"199.507833\" y=\"105.401719\" style=\"fill: #9c9ede; stroke: #9c9ede; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#ma20c134c12\" x=\"201.367996\" y=\"183.980366\" style=\"fill: #9c9ede; stroke: #9c9ede; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#ma20c134c12\" x=\"151.143768\" y=\"186.987855\" style=\"fill: #9c9ede; stroke: #9c9ede; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#ma20c134c12\" x=\"178.551866\" y=\"196.177589\" style=\"fill: #9c9ede; stroke: #9c9ede; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#ma20c134c12\" x=\"122.10396\" y=\"214.285754\" style=\"fill: #9c9ede; stroke: #9c9ede; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#ma20c134c12\" x=\"178.647828\" y=\"196.017085\" style=\"fill: #9c9ede; stroke: #9c9ede; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#ma20c134c12\" x=\"153.215861\" y=\"184.080712\" style=\"fill: #9c9ede; stroke: #9c9ede; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#ma20c134c12\" x=\"155.315789\" y=\"278.644484\" style=\"fill: #9c9ede; stroke: #9c9ede; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#ma20c134c12\" x=\"125.941389\" y=\"210.90181\" style=\"fill: #9c9ede; stroke: #9c9ede; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#ma20c134c12\" x=\"372.617176\" y=\"236.449736\" style=\"fill: #9c9ede; stroke: #9c9ede; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#ma20c134c12\" x=\"159.61482\" y=\"276.643451\" style=\"fill: #9c9ede; stroke: #9c9ede; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#ma20c134c12\" x=\"200.577558\" y=\"99.943631\" style=\"fill: #9c9ede; stroke: #9c9ede; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#ma20c134c12\" x=\"153.754792\" y=\"178.348478\" style=\"fill: #9c9ede; stroke: #9c9ede; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#ma20c134c12\" x=\"132.907838\" y=\"134.087823\" style=\"fill: #9c9ede; stroke: #9c9ede; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#ma20c134c12\" x=\"151.868054\" y=\"174.316136\" style=\"fill: #9c9ede; stroke: #9c9ede; stroke-width: 0.5\"/>\n",
              "    </g>\n",
              "   </g>\n",
              "   <g id=\"PathCollection_25\">\n",
              "    <defs>\n",
              "     <path id=\"mbb1bfa27c1\" d=\"M 0 3 \n",
              "C 0.795609 3 1.55874 2.683901 2.12132 2.12132 \n",
              "C 2.683901 1.55874 3 0.795609 3 0 \n",
              "C 3 -0.795609 2.683901 -1.55874 2.12132 -2.12132 \n",
              "C 1.55874 -2.683901 0.795609 -3 0 -3 \n",
              "C -0.795609 -3 -1.55874 -2.683901 -2.12132 -2.12132 \n",
              "C -2.683901 -1.55874 -3 -0.795609 -3 0 \n",
              "C -3 0.795609 -2.683901 1.55874 -2.12132 2.12132 \n",
              "C -1.55874 2.683901 -0.795609 3 0 3 \n",
              "z\n",
              "\" style=\"stroke: #637939; stroke-width: 0.5\"/>\n",
              "    </defs>\n",
              "    <g clip-path=\"url(#p596f914704)\">\n",
              "     <use xlink:href=\"#mbb1bfa27c1\" x=\"217.625046\" y=\"222.682689\" style=\"fill: #637939; stroke: #637939; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#mbb1bfa27c1\" x=\"179.577461\" y=\"324.892732\" style=\"fill: #637939; stroke: #637939; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#mbb1bfa27c1\" x=\"110.370088\" y=\"121.8329\" style=\"fill: #637939; stroke: #637939; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#mbb1bfa27c1\" x=\"104.682262\" y=\"246.175606\" style=\"fill: #637939; stroke: #637939; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#mbb1bfa27c1\" x=\"141.436654\" y=\"262.086752\" style=\"fill: #637939; stroke: #637939; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#mbb1bfa27c1\" x=\"156.500954\" y=\"230.337861\" style=\"fill: #637939; stroke: #637939; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#mbb1bfa27c1\" x=\"181.710043\" y=\"322.013561\" style=\"fill: #637939; stroke: #637939; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#mbb1bfa27c1\" x=\"149.492542\" y=\"243.316912\" style=\"fill: #637939; stroke: #637939; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#mbb1bfa27c1\" x=\"229.934457\" y=\"326.209387\" style=\"fill: #637939; stroke: #637939; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#mbb1bfa27c1\" x=\"103.928629\" y=\"197.402269\" style=\"fill: #637939; stroke: #637939; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#mbb1bfa27c1\" x=\"116.551604\" y=\"227.999766\" style=\"fill: #637939; stroke: #637939; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#mbb1bfa27c1\" x=\"141.583732\" y=\"225.990616\" style=\"fill: #637939; stroke: #637939; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#mbb1bfa27c1\" x=\"111.087428\" y=\"209.053147\" style=\"fill: #637939; stroke: #637939; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#mbb1bfa27c1\" x=\"149.822866\" y=\"237.416307\" style=\"fill: #637939; stroke: #637939; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#mbb1bfa27c1\" x=\"136.783355\" y=\"247.127784\" style=\"fill: #637939; stroke: #637939; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#mbb1bfa27c1\" x=\"187.381077\" y=\"161.625054\" style=\"fill: #637939; stroke: #637939; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#mbb1bfa27c1\" x=\"166.844189\" y=\"253.616451\" style=\"fill: #637939; stroke: #637939; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#mbb1bfa27c1\" x=\"130.204584\" y=\"286.847583\" style=\"fill: #637939; stroke: #637939; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#mbb1bfa27c1\" x=\"251.230413\" y=\"271.852149\" style=\"fill: #637939; stroke: #637939; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#mbb1bfa27c1\" x=\"208.954948\" y=\"245.782975\" style=\"fill: #637939; stroke: #637939; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#mbb1bfa27c1\" x=\"97.719389\" y=\"234.700202\" style=\"fill: #637939; stroke: #637939; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#mbb1bfa27c1\" x=\"229.284679\" y=\"172.891596\" style=\"fill: #637939; stroke: #637939; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#mbb1bfa27c1\" x=\"243.654613\" y=\"305.512994\" style=\"fill: #637939; stroke: #637939; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#mbb1bfa27c1\" x=\"205.67974\" y=\"168.276985\" style=\"fill: #637939; stroke: #637939; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#mbb1bfa27c1\" x=\"148.691352\" y=\"228.800138\" style=\"fill: #637939; stroke: #637939; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#mbb1bfa27c1\" x=\"130.740616\" y=\"216.37824\" style=\"fill: #637939; stroke: #637939; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#mbb1bfa27c1\" x=\"176.920393\" y=\"234.450355\" style=\"fill: #637939; stroke: #637939; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#mbb1bfa27c1\" x=\"189.783833\" y=\"204.587653\" style=\"fill: #637939; stroke: #637939; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#mbb1bfa27c1\" x=\"169.593896\" y=\"237.380355\" style=\"fill: #637939; stroke: #637939; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#mbb1bfa27c1\" x=\"246.378179\" y=\"282.082482\" style=\"fill: #637939; stroke: #637939; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#mbb1bfa27c1\" x=\"224.790973\" y=\"322.851609\" style=\"fill: #637939; stroke: #637939; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#mbb1bfa27c1\" x=\"236.629999\" y=\"237.964406\" style=\"fill: #637939; stroke: #637939; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#mbb1bfa27c1\" x=\"100.228812\" y=\"238.108854\" style=\"fill: #637939; stroke: #637939; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#mbb1bfa27c1\" x=\"159.159566\" y=\"255.80899\" style=\"fill: #637939; stroke: #637939; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#mbb1bfa27c1\" x=\"222.932771\" y=\"324.245034\" style=\"fill: #637939; stroke: #637939; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#mbb1bfa27c1\" x=\"143.708533\" y=\"245.432816\" style=\"fill: #637939; stroke: #637939; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#mbb1bfa27c1\" x=\"79.416402\" y=\"192.184424\" style=\"fill: #637939; stroke: #637939; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#mbb1bfa27c1\" x=\"104.268672\" y=\"198.089323\" style=\"fill: #637939; stroke: #637939; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#mbb1bfa27c1\" x=\"158.690685\" y=\"253.363149\" style=\"fill: #637939; stroke: #637939; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#mbb1bfa27c1\" x=\"264.018687\" y=\"314.764372\" style=\"fill: #637939; stroke: #637939; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#mbb1bfa27c1\" x=\"155.385382\" y=\"246.984601\" style=\"fill: #637939; stroke: #637939; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#mbb1bfa27c1\" x=\"150.368286\" y=\"240.616698\" style=\"fill: #637939; stroke: #637939; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#mbb1bfa27c1\" x=\"145.076282\" y=\"251.601825\" style=\"fill: #637939; stroke: #637939; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#mbb1bfa27c1\" x=\"115.433212\" y=\"231.265728\" style=\"fill: #637939; stroke: #637939; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#mbb1bfa27c1\" x=\"170.170238\" y=\"245.903438\" style=\"fill: #637939; stroke: #637939; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#mbb1bfa27c1\" x=\"200.13522\" y=\"240.052028\" style=\"fill: #637939; stroke: #637939; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#mbb1bfa27c1\" x=\"130.395517\" y=\"223.965376\" style=\"fill: #637939; stroke: #637939; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#mbb1bfa27c1\" x=\"300.149169\" y=\"303.92393\" style=\"fill: #637939; stroke: #637939; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#mbb1bfa27c1\" x=\"219.482784\" y=\"219.797184\" style=\"fill: #637939; stroke: #637939; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#mbb1bfa27c1\" x=\"160.322126\" y=\"244.53772\" style=\"fill: #637939; stroke: #637939; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#mbb1bfa27c1\" x=\"126.297247\" y=\"176.811126\" style=\"fill: #637939; stroke: #637939; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#mbb1bfa27c1\" x=\"165.619108\" y=\"222.815566\" style=\"fill: #637939; stroke: #637939; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#mbb1bfa27c1\" x=\"180.713983\" y=\"236.021323\" style=\"fill: #637939; stroke: #637939; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#mbb1bfa27c1\" x=\"146.766006\" y=\"149.716472\" style=\"fill: #637939; stroke: #637939; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#mbb1bfa27c1\" x=\"133.888571\" y=\"234.651826\" style=\"fill: #637939; stroke: #637939; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#mbb1bfa27c1\" x=\"138.656278\" y=\"262.567598\" style=\"fill: #637939; stroke: #637939; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#mbb1bfa27c1\" x=\"202.918826\" y=\"245.570755\" style=\"fill: #637939; stroke: #637939; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#mbb1bfa27c1\" x=\"240.288451\" y=\"270.881105\" style=\"fill: #637939; stroke: #637939; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#mbb1bfa27c1\" x=\"141.129076\" y=\"235.334901\" style=\"fill: #637939; stroke: #637939; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#mbb1bfa27c1\" x=\"149.7126\" y=\"252.158078\" style=\"fill: #637939; stroke: #637939; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#mbb1bfa27c1\" x=\"143.26536\" y=\"274.730832\" style=\"fill: #637939; stroke: #637939; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#mbb1bfa27c1\" x=\"107.949886\" y=\"206.81601\" style=\"fill: #637939; stroke: #637939; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#mbb1bfa27c1\" x=\"128.876877\" y=\"179.825964\" style=\"fill: #637939; stroke: #637939; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#mbb1bfa27c1\" x=\"177.092202\" y=\"246.427597\" style=\"fill: #637939; stroke: #637939; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#mbb1bfa27c1\" x=\"117.735209\" y=\"249.493001\" style=\"fill: #637939; stroke: #637939; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#mbb1bfa27c1\" x=\"104.523354\" y=\"196.697988\" style=\"fill: #637939; stroke: #637939; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#mbb1bfa27c1\" x=\"129.02876\" y=\"161.342218\" style=\"fill: #637939; stroke: #637939; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#mbb1bfa27c1\" x=\"80.681305\" y=\"257.751018\" style=\"fill: #637939; stroke: #637939; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#mbb1bfa27c1\" x=\"104.165479\" y=\"245.649028\" style=\"fill: #637939; stroke: #637939; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#mbb1bfa27c1\" x=\"164.459194\" y=\"217.797824\" style=\"fill: #637939; stroke: #637939; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#mbb1bfa27c1\" x=\"142.485389\" y=\"262.173566\" style=\"fill: #637939; stroke: #637939; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#mbb1bfa27c1\" x=\"219.259607\" y=\"277.262349\" style=\"fill: #637939; stroke: #637939; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#mbb1bfa27c1\" x=\"162.263823\" y=\"249.346934\" style=\"fill: #637939; stroke: #637939; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#mbb1bfa27c1\" x=\"204.569366\" y=\"206.382007\" style=\"fill: #637939; stroke: #637939; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#mbb1bfa27c1\" x=\"179.144952\" y=\"246.535696\" style=\"fill: #637939; stroke: #637939; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#mbb1bfa27c1\" x=\"239.712921\" y=\"330.176118\" style=\"fill: #637939; stroke: #637939; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#mbb1bfa27c1\" x=\"115.641614\" y=\"180.74455\" style=\"fill: #637939; stroke: #637939; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#mbb1bfa27c1\" x=\"166.8351\" y=\"214.992488\" style=\"fill: #637939; stroke: #637939; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#mbb1bfa27c1\" x=\"99.481952\" y=\"237.222269\" style=\"fill: #637939; stroke: #637939; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#mbb1bfa27c1\" x=\"120.109683\" y=\"249.2467\" style=\"fill: #637939; stroke: #637939; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#mbb1bfa27c1\" x=\"162.311143\" y=\"234.695113\" style=\"fill: #637939; stroke: #637939; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#mbb1bfa27c1\" x=\"162.946382\" y=\"241.780138\" style=\"fill: #637939; stroke: #637939; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#mbb1bfa27c1\" x=\"79.503181\" y=\"229.350434\" style=\"fill: #637939; stroke: #637939; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#mbb1bfa27c1\" x=\"155.573448\" y=\"217.697537\" style=\"fill: #637939; stroke: #637939; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#mbb1bfa27c1\" x=\"259.295204\" y=\"314.93145\" style=\"fill: #637939; stroke: #637939; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#mbb1bfa27c1\" x=\"270.373241\" y=\"111.598804\" style=\"fill: #637939; stroke: #637939; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#mbb1bfa27c1\" x=\"246.614856\" y=\"253.115011\" style=\"fill: #637939; stroke: #637939; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#mbb1bfa27c1\" x=\"181.333502\" y=\"232.075508\" style=\"fill: #637939; stroke: #637939; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#mbb1bfa27c1\" x=\"121.440619\" y=\"246.402335\" style=\"fill: #637939; stroke: #637939; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#mbb1bfa27c1\" x=\"274.70282\" y=\"282.002075\" style=\"fill: #637939; stroke: #637939; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#mbb1bfa27c1\" x=\"181.431103\" y=\"316.838646\" style=\"fill: #637939; stroke: #637939; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#mbb1bfa27c1\" x=\"174.346733\" y=\"253.500946\" style=\"fill: #637939; stroke: #637939; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#mbb1bfa27c1\" x=\"171.772695\" y=\"247.779471\" style=\"fill: #637939; stroke: #637939; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#mbb1bfa27c1\" x=\"107.048686\" y=\"209.672411\" style=\"fill: #637939; stroke: #637939; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#mbb1bfa27c1\" x=\"99.282025\" y=\"200.023395\" style=\"fill: #637939; stroke: #637939; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#mbb1bfa27c1\" x=\"184.310135\" y=\"255.327566\" style=\"fill: #637939; stroke: #637939; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#mbb1bfa27c1\" x=\"163.041227\" y=\"234.002591\" style=\"fill: #637939; stroke: #637939; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#mbb1bfa27c1\" x=\"236.030714\" y=\"259.903212\" style=\"fill: #637939; stroke: #637939; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#mbb1bfa27c1\" x=\"139.212063\" y=\"251.819679\" style=\"fill: #637939; stroke: #637939; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#mbb1bfa27c1\" x=\"191.43854\" y=\"247.954504\" style=\"fill: #637939; stroke: #637939; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#mbb1bfa27c1\" x=\"175.635044\" y=\"224.501907\" style=\"fill: #637939; stroke: #637939; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#mbb1bfa27c1\" x=\"215.925312\" y=\"274.867855\" style=\"fill: #637939; stroke: #637939; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#mbb1bfa27c1\" x=\"182.283676\" y=\"261.730835\" style=\"fill: #637939; stroke: #637939; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#mbb1bfa27c1\" x=\"80.670609\" y=\"257.769475\" style=\"fill: #637939; stroke: #637939; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#mbb1bfa27c1\" x=\"158.202587\" y=\"209.015114\" style=\"fill: #637939; stroke: #637939; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#mbb1bfa27c1\" x=\"119.466472\" y=\"224.844701\" style=\"fill: #637939; stroke: #637939; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#mbb1bfa27c1\" x=\"115.06209\" y=\"181.139947\" style=\"fill: #637939; stroke: #637939; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#mbb1bfa27c1\" x=\"280.109024\" y=\"286.310774\" style=\"fill: #637939; stroke: #637939; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#mbb1bfa27c1\" x=\"156.760204\" y=\"230.198998\" style=\"fill: #637939; stroke: #637939; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#mbb1bfa27c1\" x=\"274.051765\" y=\"181.375293\" style=\"fill: #637939; stroke: #637939; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#mbb1bfa27c1\" x=\"142.215286\" y=\"248.915749\" style=\"fill: #637939; stroke: #637939; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#mbb1bfa27c1\" x=\"199.83199\" y=\"231.060899\" style=\"fill: #637939; stroke: #637939; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#mbb1bfa27c1\" x=\"122.221472\" y=\"251.306382\" style=\"fill: #637939; stroke: #637939; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#mbb1bfa27c1\" x=\"308.39446\" y=\"249.846788\" style=\"fill: #637939; stroke: #637939; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#mbb1bfa27c1\" x=\"183.040176\" y=\"258.798224\" style=\"fill: #637939; stroke: #637939; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#mbb1bfa27c1\" x=\"198.652795\" y=\"186.689288\" style=\"fill: #637939; stroke: #637939; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#mbb1bfa27c1\" x=\"249.314526\" y=\"302.002319\" style=\"fill: #637939; stroke: #637939; stroke-width: 0.5\"/>\n",
              "    </g>\n",
              "   </g>\n",
              "   <g id=\"PathCollection_26\">\n",
              "    <defs>\n",
              "     <path id=\"m0344f49d76\" d=\"M 0 3 \n",
              "C 0.795609 3 1.55874 2.683901 2.12132 2.12132 \n",
              "C 2.683901 1.55874 3 0.795609 3 0 \n",
              "C 3 -0.795609 2.683901 -1.55874 2.12132 -2.12132 \n",
              "C 1.55874 -2.683901 0.795609 -3 0 -3 \n",
              "C -0.795609 -3 -1.55874 -2.683901 -2.12132 -2.12132 \n",
              "C -2.683901 -1.55874 -3 -0.795609 -3 0 \n",
              "C -3 0.795609 -2.683901 1.55874 -2.12132 2.12132 \n",
              "C -1.55874 2.683901 -0.795609 3 0 3 \n",
              "z\n",
              "\" style=\"stroke: #8ca252; stroke-width: 0.5\"/>\n",
              "    </defs>\n",
              "    <g clip-path=\"url(#p596f914704)\">\n",
              "     <use xlink:href=\"#m0344f49d76\" x=\"292.394584\" y=\"52.093257\" style=\"fill: #8ca252; stroke: #8ca252; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m0344f49d76\" x=\"357.861083\" y=\"237.97499\" style=\"fill: #8ca252; stroke: #8ca252; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m0344f49d76\" x=\"448.310456\" y=\"53.487599\" style=\"fill: #8ca252; stroke: #8ca252; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m0344f49d76\" x=\"467.769404\" y=\"150.545085\" style=\"fill: #8ca252; stroke: #8ca252; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m0344f49d76\" x=\"483.954002\" y=\"208.61856\" style=\"fill: #8ca252; stroke: #8ca252; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m0344f49d76\" x=\"318.984166\" y=\"57.164419\" style=\"fill: #8ca252; stroke: #8ca252; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m0344f49d76\" x=\"485.982777\" y=\"177.620082\" style=\"fill: #8ca252; stroke: #8ca252; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m0344f49d76\" x=\"488.476289\" y=\"191.276989\" style=\"fill: #8ca252; stroke: #8ca252; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m0344f49d76\" x=\"302.402405\" y=\"273.383318\" style=\"fill: #8ca252; stroke: #8ca252; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m0344f49d76\" x=\"316.446573\" y=\"219.883692\" style=\"fill: #8ca252; stroke: #8ca252; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m0344f49d76\" x=\"443.731043\" y=\"231.375791\" style=\"fill: #8ca252; stroke: #8ca252; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m0344f49d76\" x=\"294.053759\" y=\"82.473199\" style=\"fill: #8ca252; stroke: #8ca252; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m0344f49d76\" x=\"356.296242\" y=\"238.035408\" style=\"fill: #8ca252; stroke: #8ca252; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m0344f49d76\" x=\"428.917321\" y=\"166.844965\" style=\"fill: #8ca252; stroke: #8ca252; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m0344f49d76\" x=\"441.329909\" y=\"132.087432\" style=\"fill: #8ca252; stroke: #8ca252; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m0344f49d76\" x=\"408.007862\" y=\"131.318711\" style=\"fill: #8ca252; stroke: #8ca252; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m0344f49d76\" x=\"481.838274\" y=\"236.324094\" style=\"fill: #8ca252; stroke: #8ca252; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m0344f49d76\" x=\"405.076044\" y=\"108.563011\" style=\"fill: #8ca252; stroke: #8ca252; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m0344f49d76\" x=\"484.855674\" y=\"176.145326\" style=\"fill: #8ca252; stroke: #8ca252; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m0344f49d76\" x=\"426.240461\" y=\"153.823629\" style=\"fill: #8ca252; stroke: #8ca252; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m0344f49d76\" x=\"488.804014\" y=\"142.775332\" style=\"fill: #8ca252; stroke: #8ca252; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m0344f49d76\" x=\"410.133262\" y=\"165.902939\" style=\"fill: #8ca252; stroke: #8ca252; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m0344f49d76\" x=\"388.397536\" y=\"153.310017\" style=\"fill: #8ca252; stroke: #8ca252; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m0344f49d76\" x=\"228.84149\" y=\"250.784104\" style=\"fill: #8ca252; stroke: #8ca252; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m0344f49d76\" x=\"279.189547\" y=\"295.37269\" style=\"fill: #8ca252; stroke: #8ca252; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m0344f49d76\" x=\"420.78198\" y=\"148.163361\" style=\"fill: #8ca252; stroke: #8ca252; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m0344f49d76\" x=\"399.330243\" y=\"126.847641\" style=\"fill: #8ca252; stroke: #8ca252; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m0344f49d76\" x=\"434.557591\" y=\"155.220983\" style=\"fill: #8ca252; stroke: #8ca252; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m0344f49d76\" x=\"436.126858\" y=\"100.393168\" style=\"fill: #8ca252; stroke: #8ca252; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m0344f49d76\" x=\"475.303468\" y=\"141.71092\" style=\"fill: #8ca252; stroke: #8ca252; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m0344f49d76\" x=\"340.504714\" y=\"143.515313\" style=\"fill: #8ca252; stroke: #8ca252; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m0344f49d76\" x=\"298.228803\" y=\"79.949404\" style=\"fill: #8ca252; stroke: #8ca252; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m0344f49d76\" x=\"479.177079\" y=\"231.142962\" style=\"fill: #8ca252; stroke: #8ca252; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m0344f49d76\" x=\"378.321576\" y=\"140.674399\" style=\"fill: #8ca252; stroke: #8ca252; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m0344f49d76\" x=\"438.030994\" y=\"209.320456\" style=\"fill: #8ca252; stroke: #8ca252; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m0344f49d76\" x=\"398.995587\" y=\"173.248508\" style=\"fill: #8ca252; stroke: #8ca252; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m0344f49d76\" x=\"487.757516\" y=\"142.640138\" style=\"fill: #8ca252; stroke: #8ca252; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m0344f49d76\" x=\"495.338789\" y=\"150.006447\" style=\"fill: #8ca252; stroke: #8ca252; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m0344f49d76\" x=\"425.082815\" y=\"105.701756\" style=\"fill: #8ca252; stroke: #8ca252; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m0344f49d76\" x=\"482.210594\" y=\"108.983815\" style=\"fill: #8ca252; stroke: #8ca252; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m0344f49d76\" x=\"444.8595\" y=\"151.754839\" style=\"fill: #8ca252; stroke: #8ca252; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m0344f49d76\" x=\"463.12658\" y=\"122.172727\" style=\"fill: #8ca252; stroke: #8ca252; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m0344f49d76\" x=\"211.043353\" y=\"93.839234\" style=\"fill: #8ca252; stroke: #8ca252; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m0344f49d76\" x=\"455.403994\" y=\"151.736526\" style=\"fill: #8ca252; stroke: #8ca252; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m0344f49d76\" x=\"410.339364\" y=\"110.191091\" style=\"fill: #8ca252; stroke: #8ca252; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m0344f49d76\" x=\"424.142888\" y=\"162.810709\" style=\"fill: #8ca252; stroke: #8ca252; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m0344f49d76\" x=\"445.898524\" y=\"115.32228\" style=\"fill: #8ca252; stroke: #8ca252; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m0344f49d76\" x=\"270.567206\" y=\"141.173345\" style=\"fill: #8ca252; stroke: #8ca252; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m0344f49d76\" x=\"411.31519\" y=\"110.525479\" style=\"fill: #8ca252; stroke: #8ca252; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m0344f49d76\" x=\"490.380921\" y=\"96.74653\" style=\"fill: #8ca252; stroke: #8ca252; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m0344f49d76\" x=\"521.149628\" y=\"118.549864\" style=\"fill: #8ca252; stroke: #8ca252; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m0344f49d76\" x=\"492.42789\" y=\"143.825764\" style=\"fill: #8ca252; stroke: #8ca252; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m0344f49d76\" x=\"487.775111\" y=\"249.923777\" style=\"fill: #8ca252; stroke: #8ca252; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m0344f49d76\" x=\"451.724071\" y=\"131.413743\" style=\"fill: #8ca252; stroke: #8ca252; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m0344f49d76\" x=\"454.486474\" y=\"69.578374\" style=\"fill: #8ca252; stroke: #8ca252; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m0344f49d76\" x=\"234.913433\" y=\"76.662364\" style=\"fill: #8ca252; stroke: #8ca252; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m0344f49d76\" x=\"494.076958\" y=\"215.696033\" style=\"fill: #8ca252; stroke: #8ca252; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m0344f49d76\" x=\"485.531302\" y=\"76.53473\" style=\"fill: #8ca252; stroke: #8ca252; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m0344f49d76\" x=\"438.853779\" y=\"130.493227\" style=\"fill: #8ca252; stroke: #8ca252; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m0344f49d76\" x=\"391.880233\" y=\"73.719605\" style=\"fill: #8ca252; stroke: #8ca252; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m0344f49d76\" x=\"327.711642\" y=\"258.022578\" style=\"fill: #8ca252; stroke: #8ca252; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m0344f49d76\" x=\"421.179912\" y=\"129.834975\" style=\"fill: #8ca252; stroke: #8ca252; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m0344f49d76\" x=\"482.223715\" y=\"107.560342\" style=\"fill: #8ca252; stroke: #8ca252; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m0344f49d76\" x=\"504.878504\" y=\"117.343272\" style=\"fill: #8ca252; stroke: #8ca252; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m0344f49d76\" x=\"451.636426\" y=\"172.929163\" style=\"fill: #8ca252; stroke: #8ca252; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m0344f49d76\" x=\"426.466993\" y=\"142.719553\" style=\"fill: #8ca252; stroke: #8ca252; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m0344f49d76\" x=\"447.017861\" y=\"60.437104\" style=\"fill: #8ca252; stroke: #8ca252; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m0344f49d76\" x=\"388.685116\" y=\"98.085405\" style=\"fill: #8ca252; stroke: #8ca252; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m0344f49d76\" x=\"446.791439\" y=\"160.406211\" style=\"fill: #8ca252; stroke: #8ca252; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m0344f49d76\" x=\"521.869095\" y=\"117.101655\" style=\"fill: #8ca252; stroke: #8ca252; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m0344f49d76\" x=\"502.551681\" y=\"101.810646\" style=\"fill: #8ca252; stroke: #8ca252; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m0344f49d76\" x=\"118.91316\" y=\"156.482997\" style=\"fill: #8ca252; stroke: #8ca252; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m0344f49d76\" x=\"424.201872\" y=\"39.728428\" style=\"fill: #8ca252; stroke: #8ca252; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m0344f49d76\" x=\"485.387359\" y=\"98.320134\" style=\"fill: #8ca252; stroke: #8ca252; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m0344f49d76\" x=\"466.667788\" y=\"222.572273\" style=\"fill: #8ca252; stroke: #8ca252; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m0344f49d76\" x=\"421.862133\" y=\"91.10311\" style=\"fill: #8ca252; stroke: #8ca252; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m0344f49d76\" x=\"457.82533\" y=\"189.607981\" style=\"fill: #8ca252; stroke: #8ca252; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m0344f49d76\" x=\"291.016452\" y=\"138.210544\" style=\"fill: #8ca252; stroke: #8ca252; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m0344f49d76\" x=\"483.046375\" y=\"149.838652\" style=\"fill: #8ca252; stroke: #8ca252; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m0344f49d76\" x=\"432.83114\" y=\"118.094423\" style=\"fill: #8ca252; stroke: #8ca252; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m0344f49d76\" x=\"270.335506\" y=\"140.389149\" style=\"fill: #8ca252; stroke: #8ca252; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m0344f49d76\" x=\"208.355852\" y=\"240.195102\" style=\"fill: #8ca252; stroke: #8ca252; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m0344f49d76\" x=\"477.341946\" y=\"137.858886\" style=\"fill: #8ca252; stroke: #8ca252; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m0344f49d76\" x=\"492.135592\" y=\"119.682292\" style=\"fill: #8ca252; stroke: #8ca252; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m0344f49d76\" x=\"465.447291\" y=\"159.98197\" style=\"fill: #8ca252; stroke: #8ca252; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m0344f49d76\" x=\"441.29323\" y=\"132.396821\" style=\"fill: #8ca252; stroke: #8ca252; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m0344f49d76\" x=\"414.785773\" y=\"156.071362\" style=\"fill: #8ca252; stroke: #8ca252; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m0344f49d76\" x=\"243.751915\" y=\"208.252692\" style=\"fill: #8ca252; stroke: #8ca252; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m0344f49d76\" x=\"408.611157\" y=\"85.945938\" style=\"fill: #8ca252; stroke: #8ca252; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m0344f49d76\" x=\"488.789837\" y=\"185.444039\" style=\"fill: #8ca252; stroke: #8ca252; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m0344f49d76\" x=\"466.699466\" y=\"204.632751\" style=\"fill: #8ca252; stroke: #8ca252; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m0344f49d76\" x=\"461.971187\" y=\"193.258836\" style=\"fill: #8ca252; stroke: #8ca252; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m0344f49d76\" x=\"483.906824\" y=\"160.020337\" style=\"fill: #8ca252; stroke: #8ca252; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m0344f49d76\" x=\"415.110528\" y=\"148.675201\" style=\"fill: #8ca252; stroke: #8ca252; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m0344f49d76\" x=\"449.236371\" y=\"142.918387\" style=\"fill: #8ca252; stroke: #8ca252; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m0344f49d76\" x=\"486.962139\" y=\"75.115156\" style=\"fill: #8ca252; stroke: #8ca252; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m0344f49d76\" x=\"454.895307\" y=\"130.586039\" style=\"fill: #8ca252; stroke: #8ca252; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m0344f49d76\" x=\"484.181007\" y=\"77.843904\" style=\"fill: #8ca252; stroke: #8ca252; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m0344f49d76\" x=\"310.452906\" y=\"73.417946\" style=\"fill: #8ca252; stroke: #8ca252; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m0344f49d76\" x=\"506.054832\" y=\"234.202391\" style=\"fill: #8ca252; stroke: #8ca252; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m0344f49d76\" x=\"303.736376\" y=\"275.086456\" style=\"fill: #8ca252; stroke: #8ca252; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m0344f49d76\" x=\"320.990263\" y=\"54.332377\" style=\"fill: #8ca252; stroke: #8ca252; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m0344f49d76\" x=\"446.732242\" y=\"159.630622\" style=\"fill: #8ca252; stroke: #8ca252; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m0344f49d76\" x=\"474.185565\" y=\"129.975472\" style=\"fill: #8ca252; stroke: #8ca252; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m0344f49d76\" x=\"484.087029\" y=\"208.802887\" style=\"fill: #8ca252; stroke: #8ca252; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m0344f49d76\" x=\"464.533064\" y=\"139.759258\" style=\"fill: #8ca252; stroke: #8ca252; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m0344f49d76\" x=\"472.839601\" y=\"78.584508\" style=\"fill: #8ca252; stroke: #8ca252; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m0344f49d76\" x=\"514.697679\" y=\"130.414218\" style=\"fill: #8ca252; stroke: #8ca252; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m0344f49d76\" x=\"215.062901\" y=\"92.757023\" style=\"fill: #8ca252; stroke: #8ca252; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m0344f49d76\" x=\"357.607894\" y=\"98.74265\" style=\"fill: #8ca252; stroke: #8ca252; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m0344f49d76\" x=\"445.931123\" y=\"99.522143\" style=\"fill: #8ca252; stroke: #8ca252; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m0344f49d76\" x=\"365.659253\" y=\"77.828985\" style=\"fill: #8ca252; stroke: #8ca252; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m0344f49d76\" x=\"448.355791\" y=\"53.077664\" style=\"fill: #8ca252; stroke: #8ca252; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m0344f49d76\" x=\"424.151307\" y=\"39.717656\" style=\"fill: #8ca252; stroke: #8ca252; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m0344f49d76\" x=\"158.590107\" y=\"261.817664\" style=\"fill: #8ca252; stroke: #8ca252; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m0344f49d76\" x=\"358.224688\" y=\"100.705524\" style=\"fill: #8ca252; stroke: #8ca252; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m0344f49d76\" x=\"293.388873\" y=\"82.637452\" style=\"fill: #8ca252; stroke: #8ca252; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m0344f49d76\" x=\"448.173231\" y=\"164.429631\" style=\"fill: #8ca252; stroke: #8ca252; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m0344f49d76\" x=\"398.020178\" y=\"161.244222\" style=\"fill: #8ca252; stroke: #8ca252; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m0344f49d76\" x=\"514.25282\" y=\"237.006842\" style=\"fill: #8ca252; stroke: #8ca252; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m0344f49d76\" x=\"417.421142\" y=\"74.107883\" style=\"fill: #8ca252; stroke: #8ca252; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m0344f49d76\" x=\"509.374863\" y=\"234.780041\" style=\"fill: #8ca252; stroke: #8ca252; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m0344f49d76\" x=\"304.970009\" y=\"270.116433\" style=\"fill: #8ca252; stroke: #8ca252; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m0344f49d76\" x=\"466.460647\" y=\"148.318622\" style=\"fill: #8ca252; stroke: #8ca252; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m0344f49d76\" x=\"326.012901\" y=\"231.957546\" style=\"fill: #8ca252; stroke: #8ca252; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m0344f49d76\" x=\"466.56039\" y=\"215.785384\" style=\"fill: #8ca252; stroke: #8ca252; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m0344f49d76\" x=\"480.085604\" y=\"152.483737\" style=\"fill: #8ca252; stroke: #8ca252; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m0344f49d76\" x=\"299.277242\" y=\"76.170986\" style=\"fill: #8ca252; stroke: #8ca252; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m0344f49d76\" x=\"459.796893\" y=\"179.250713\" style=\"fill: #8ca252; stroke: #8ca252; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#m0344f49d76\" x=\"342.518999\" y=\"51.616888\" style=\"fill: #8ca252; stroke: #8ca252; stroke-width: 0.5\"/>\n",
              "    </g>\n",
              "   </g>\n",
              "   <g id=\"PathCollection_27\">\n",
              "    <defs>\n",
              "     <path id=\"mac019e356a\" d=\"M 0 3 \n",
              "C 0.795609 3 1.55874 2.683901 2.12132 2.12132 \n",
              "C 2.683901 1.55874 3 0.795609 3 0 \n",
              "C 3 -0.795609 2.683901 -1.55874 2.12132 -2.12132 \n",
              "C 1.55874 -2.683901 0.795609 -3 0 -3 \n",
              "C -0.795609 -3 -1.55874 -2.683901 -2.12132 -2.12132 \n",
              "C -2.683901 -1.55874 -3 -0.795609 -3 0 \n",
              "C -3 0.795609 -2.683901 1.55874 -2.12132 2.12132 \n",
              "C -1.55874 2.683901 -0.795609 3 0 3 \n",
              "z\n",
              "\" style=\"stroke: #b5cf6b; stroke-width: 0.5\"/>\n",
              "    </defs>\n",
              "    <g clip-path=\"url(#p596f914704)\">\n",
              "     <use xlink:href=\"#mac019e356a\" x=\"102.62863\" y=\"279.295413\" style=\"fill: #b5cf6b; stroke: #b5cf6b; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#mac019e356a\" x=\"99.720803\" y=\"274.92562\" style=\"fill: #b5cf6b; stroke: #b5cf6b; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#mac019e356a\" x=\"113.950677\" y=\"269.399776\" style=\"fill: #b5cf6b; stroke: #b5cf6b; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#mac019e356a\" x=\"104.105495\" y=\"278.850571\" style=\"fill: #b5cf6b; stroke: #b5cf6b; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#mac019e356a\" x=\"102.436768\" y=\"275.711198\" style=\"fill: #b5cf6b; stroke: #b5cf6b; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#mac019e356a\" x=\"103.195868\" y=\"278.046191\" style=\"fill: #b5cf6b; stroke: #b5cf6b; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#mac019e356a\" x=\"103.840889\" y=\"284.176075\" style=\"fill: #b5cf6b; stroke: #b5cf6b; stroke-width: 0.5\"/>\n",
              "    </g>\n",
              "   </g>\n",
              "   <g id=\"PathCollection_28\">\n",
              "    <defs>\n",
              "     <path id=\"mc3413dad6b\" d=\"M 0 3 \n",
              "C 0.795609 3 1.55874 2.683901 2.12132 2.12132 \n",
              "C 2.683901 1.55874 3 0.795609 3 0 \n",
              "C 3 -0.795609 2.683901 -1.55874 2.12132 -2.12132 \n",
              "C 1.55874 -2.683901 0.795609 -3 0 -3 \n",
              "C -0.795609 -3 -1.55874 -2.683901 -2.12132 -2.12132 \n",
              "C -2.683901 -1.55874 -3 -0.795609 -3 0 \n",
              "C -3 0.795609 -2.683901 1.55874 -2.12132 2.12132 \n",
              "C -1.55874 2.683901 -0.795609 3 0 3 \n",
              "z\n",
              "\" style=\"stroke: #cedb9c; stroke-width: 0.5\"/>\n",
              "    </defs>\n",
              "    <g clip-path=\"url(#p596f914704)\">\n",
              "     <use xlink:href=\"#mc3413dad6b\" x=\"359.757705\" y=\"41.60907\" style=\"fill: #cedb9c; stroke: #cedb9c; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#mc3413dad6b\" x=\"520.979914\" y=\"214.526052\" style=\"fill: #cedb9c; stroke: #cedb9c; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#mc3413dad6b\" x=\"427.175449\" y=\"136.135016\" style=\"fill: #cedb9c; stroke: #cedb9c; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#mc3413dad6b\" x=\"256.558391\" y=\"157.380719\" style=\"fill: #cedb9c; stroke: #cedb9c; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#mc3413dad6b\" x=\"500.105441\" y=\"220.144582\" style=\"fill: #cedb9c; stroke: #cedb9c; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#mc3413dad6b\" x=\"490.400706\" y=\"241.880259\" style=\"fill: #cedb9c; stroke: #cedb9c; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#mc3413dad6b\" x=\"521.266274\" y=\"229.828681\" style=\"fill: #cedb9c; stroke: #cedb9c; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#mc3413dad6b\" x=\"517.058747\" y=\"215.264703\" style=\"fill: #cedb9c; stroke: #cedb9c; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#mc3413dad6b\" x=\"520.395964\" y=\"238.724132\" style=\"fill: #cedb9c; stroke: #cedb9c; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#mc3413dad6b\" x=\"489.04611\" y=\"246.357812\" style=\"fill: #cedb9c; stroke: #cedb9c; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#mc3413dad6b\" x=\"522.322208\" y=\"193.900383\" style=\"fill: #cedb9c; stroke: #cedb9c; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#mc3413dad6b\" x=\"514.709477\" y=\"216.644068\" style=\"fill: #cedb9c; stroke: #cedb9c; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#mc3413dad6b\" x=\"531.839082\" y=\"209.108283\" style=\"fill: #cedb9c; stroke: #cedb9c; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#mc3413dad6b\" x=\"327.927452\" y=\"54.030778\" style=\"fill: #cedb9c; stroke: #cedb9c; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#mc3413dad6b\" x=\"521.249308\" y=\"221.445018\" style=\"fill: #cedb9c; stroke: #cedb9c; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#mc3413dad6b\" x=\"548.946145\" y=\"228.735762\" style=\"fill: #cedb9c; stroke: #cedb9c; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#mc3413dad6b\" x=\"530.771822\" y=\"209.070033\" style=\"fill: #cedb9c; stroke: #cedb9c; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#mc3413dad6b\" x=\"527.475434\" y=\"195.41679\" style=\"fill: #cedb9c; stroke: #cedb9c; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#mc3413dad6b\" x=\"510.903458\" y=\"207.309438\" style=\"fill: #cedb9c; stroke: #cedb9c; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#mc3413dad6b\" x=\"509.518697\" y=\"217.491479\" style=\"fill: #cedb9c; stroke: #cedb9c; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#mc3413dad6b\" x=\"524.05848\" y=\"216.043963\" style=\"fill: #cedb9c; stroke: #cedb9c; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#mc3413dad6b\" x=\"497.280596\" y=\"247.691695\" style=\"fill: #cedb9c; stroke: #cedb9c; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#mc3413dad6b\" x=\"524.531928\" y=\"212.082173\" style=\"fill: #cedb9c; stroke: #cedb9c; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#mc3413dad6b\" x=\"515.398825\" y=\"219.77789\" style=\"fill: #cedb9c; stroke: #cedb9c; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#mc3413dad6b\" x=\"516.042934\" y=\"234.839091\" style=\"fill: #cedb9c; stroke: #cedb9c; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#mc3413dad6b\" x=\"511.862595\" y=\"215.084547\" style=\"fill: #cedb9c; stroke: #cedb9c; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#mc3413dad6b\" x=\"371.70513\" y=\"69.110374\" style=\"fill: #cedb9c; stroke: #cedb9c; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#mc3413dad6b\" x=\"511.993764\" y=\"220.552992\" style=\"fill: #cedb9c; stroke: #cedb9c; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#mc3413dad6b\" x=\"530.032822\" y=\"222.244702\" style=\"fill: #cedb9c; stroke: #cedb9c; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#mc3413dad6b\" x=\"522.640135\" y=\"234.21646\" style=\"fill: #cedb9c; stroke: #cedb9c; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#mc3413dad6b\" x=\"378.98252\" y=\"47.526753\" style=\"fill: #cedb9c; stroke: #cedb9c; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#mc3413dad6b\" x=\"530.335895\" y=\"221.420621\" style=\"fill: #cedb9c; stroke: #cedb9c; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#mc3413dad6b\" x=\"500.918255\" y=\"246.710341\" style=\"fill: #cedb9c; stroke: #cedb9c; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#mc3413dad6b\" x=\"533.457622\" y=\"212.466019\" style=\"fill: #cedb9c; stroke: #cedb9c; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#mc3413dad6b\" x=\"536.987898\" y=\"221.831603\" style=\"fill: #cedb9c; stroke: #cedb9c; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#mc3413dad6b\" x=\"533.843393\" y=\"231.514019\" style=\"fill: #cedb9c; stroke: #cedb9c; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#mc3413dad6b\" x=\"539.796927\" y=\"207.455094\" style=\"fill: #cedb9c; stroke: #cedb9c; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#mc3413dad6b\" x=\"492.920871\" y=\"214.628139\" style=\"fill: #cedb9c; stroke: #cedb9c; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#mc3413dad6b\" x=\"516.118198\" y=\"209.922377\" style=\"fill: #cedb9c; stroke: #cedb9c; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#mc3413dad6b\" x=\"259.128002\" y=\"155.004876\" style=\"fill: #cedb9c; stroke: #cedb9c; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#mc3413dad6b\" x=\"548.966339\" y=\"228.725223\" style=\"fill: #cedb9c; stroke: #cedb9c; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#mc3413dad6b\" x=\"206.495618\" y=\"104.384793\" style=\"fill: #cedb9c; stroke: #cedb9c; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#mc3413dad6b\" x=\"522.283756\" y=\"205.293714\" style=\"fill: #cedb9c; stroke: #cedb9c; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#mc3413dad6b\" x=\"497.42706\" y=\"251.286671\" style=\"fill: #cedb9c; stroke: #cedb9c; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#mc3413dad6b\" x=\"495.414967\" y=\"247.042777\" style=\"fill: #cedb9c; stroke: #cedb9c; stroke-width: 0.5\"/>\n",
              "     <use xlink:href=\"#mc3413dad6b\" x=\"516.482704\" y=\"212.314956\" style=\"fill: #cedb9c; stroke: #cedb9c; stroke-width: 0.5\"/>\n",
              "    </g>\n",
              "   </g>\n",
              "   <g id=\"patch_3\">\n",
              "    <path d=\"M 50.918438 357.237656 \n",
              "L 50.918438 24.597656 \n",
              "\" style=\"fill: none; stroke: #ffffff; stroke-linejoin: miter; stroke-linecap: square\"/>\n",
              "   </g>\n",
              "   <g id=\"patch_4\">\n",
              "    <path d=\"M 608.918437 357.237656 \n",
              "L 608.918437 24.597656 \n",
              "\" style=\"fill: none; stroke: #ffffff; stroke-linejoin: miter; stroke-linecap: square\"/>\n",
              "   </g>\n",
              "   <g id=\"patch_5\">\n",
              "    <path d=\"M 50.918437 357.237656 \n",
              "L 608.918437 357.237656 \n",
              "\" style=\"fill: none; stroke: #ffffff; stroke-linejoin: miter; stroke-linecap: square\"/>\n",
              "   </g>\n",
              "   <g id=\"patch_6\">\n",
              "    <path d=\"M 50.918437 24.597656 \n",
              "L 608.918437 24.597656 \n",
              "\" style=\"fill: none; stroke: #ffffff; stroke-linejoin: miter; stroke-linecap: square\"/>\n",
              "   </g>\n",
              "   <g id=\"text_19\">\n",
              "    <!-- t-SNE Visualization of Article Clusters -->\n",
              "    <g transform=\"translate(190.444219 18.597656) scale(0.15 -0.15)\">\n",
              "     <defs>\n",
              "      <path id=\"DejaVuSans-56\" d=\"M 1831 0 \n",
              "L 50 4666 \n",
              "L 709 4666 \n",
              "L 2188 738 \n",
              "L 3669 4666 \n",
              "L 4325 4666 \n",
              "L 2547 0 \n",
              "L 1831 0 \n",
              "z\n",
              "\" transform=\"scale(0.015625)\"/>\n",
              "      <path id=\"DejaVuSans-69\" d=\"M 603 3500 \n",
              "L 1178 3500 \n",
              "L 1178 0 \n",
              "L 603 0 \n",
              "L 603 3500 \n",
              "z\n",
              "M 603 4863 \n",
              "L 1178 4863 \n",
              "L 1178 4134 \n",
              "L 603 4134 \n",
              "L 603 4863 \n",
              "z\n",
              "\" transform=\"scale(0.015625)\"/>\n",
              "      <path id=\"DejaVuSans-73\" d=\"M 2834 3397 \n",
              "L 2834 2853 \n",
              "Q 2591 2978 2328 3040 \n",
              "Q 2066 3103 1784 3103 \n",
              "Q 1356 3103 1142 2972 \n",
              "Q 928 2841 928 2578 \n",
              "Q 928 2378 1081 2264 \n",
              "Q 1234 2150 1697 2047 \n",
              "L 1894 2003 \n",
              "Q 2506 1872 2764 1633 \n",
              "Q 3022 1394 3022 966 \n",
              "Q 3022 478 2636 193 \n",
              "Q 2250 -91 1575 -91 \n",
              "Q 1294 -91 989 -36 \n",
              "Q 684 19 347 128 \n",
              "L 347 722 \n",
              "Q 666 556 975 473 \n",
              "Q 1284 391 1588 391 \n",
              "Q 1994 391 2212 530 \n",
              "Q 2431 669 2431 922 \n",
              "Q 2431 1156 2273 1281 \n",
              "Q 2116 1406 1581 1522 \n",
              "L 1381 1569 \n",
              "Q 847 1681 609 1914 \n",
              "Q 372 2147 372 2553 \n",
              "Q 372 3047 722 3315 \n",
              "Q 1072 3584 1716 3584 \n",
              "Q 2034 3584 2315 3537 \n",
              "Q 2597 3491 2834 3397 \n",
              "z\n",
              "\" transform=\"scale(0.015625)\"/>\n",
              "      <path id=\"DejaVuSans-75\" d=\"M 544 1381 \n",
              "L 544 3500 \n",
              "L 1119 3500 \n",
              "L 1119 1403 \n",
              "Q 1119 906 1312 657 \n",
              "Q 1506 409 1894 409 \n",
              "Q 2359 409 2629 706 \n",
              "Q 2900 1003 2900 1516 \n",
              "L 2900 3500 \n",
              "L 3475 3500 \n",
              "L 3475 0 \n",
              "L 2900 0 \n",
              "L 2900 538 \n",
              "Q 2691 219 2414 64 \n",
              "Q 2138 -91 1772 -91 \n",
              "Q 1169 -91 856 284 \n",
              "Q 544 659 544 1381 \n",
              "z\n",
              "M 1991 3584 \n",
              "L 1991 3584 \n",
              "z\n",
              "\" transform=\"scale(0.015625)\"/>\n",
              "      <path id=\"DejaVuSans-61\" d=\"M 2194 1759 \n",
              "Q 1497 1759 1228 1600 \n",
              "Q 959 1441 959 1056 \n",
              "Q 959 750 1161 570 \n",
              "Q 1363 391 1709 391 \n",
              "Q 2188 391 2477 730 \n",
              "Q 2766 1069 2766 1631 \n",
              "L 2766 1759 \n",
              "L 2194 1759 \n",
              "z\n",
              "M 3341 1997 \n",
              "L 3341 0 \n",
              "L 2766 0 \n",
              "L 2766 531 \n",
              "Q 2569 213 2275 61 \n",
              "Q 1981 -91 1556 -91 \n",
              "Q 1019 -91 701 211 \n",
              "Q 384 513 384 1019 \n",
              "Q 384 1609 779 1909 \n",
              "Q 1175 2209 1959 2209 \n",
              "L 2766 2209 \n",
              "L 2766 2266 \n",
              "Q 2766 2663 2505 2880 \n",
              "Q 2244 3097 1772 3097 \n",
              "Q 1472 3097 1187 3025 \n",
              "Q 903 2953 641 2809 \n",
              "L 641 3341 \n",
              "Q 956 3463 1253 3523 \n",
              "Q 1550 3584 1831 3584 \n",
              "Q 2591 3584 2966 3190 \n",
              "Q 3341 2797 3341 1997 \n",
              "z\n",
              "\" transform=\"scale(0.015625)\"/>\n",
              "      <path id=\"DejaVuSans-6c\" d=\"M 603 4863 \n",
              "L 1178 4863 \n",
              "L 1178 0 \n",
              "L 603 0 \n",
              "L 603 4863 \n",
              "z\n",
              "\" transform=\"scale(0.015625)\"/>\n",
              "      <path id=\"DejaVuSans-7a\" d=\"M 353 3500 \n",
              "L 3084 3500 \n",
              "L 3084 2975 \n",
              "L 922 459 \n",
              "L 3084 459 \n",
              "L 3084 0 \n",
              "L 275 0 \n",
              "L 275 525 \n",
              "L 2438 3041 \n",
              "L 353 3041 \n",
              "L 353 3500 \n",
              "z\n",
              "\" transform=\"scale(0.015625)\"/>\n",
              "      <path id=\"DejaVuSans-66\" d=\"M 2375 4863 \n",
              "L 2375 4384 \n",
              "L 1825 4384 \n",
              "Q 1516 4384 1395 4259 \n",
              "Q 1275 4134 1275 3809 \n",
              "L 1275 3500 \n",
              "L 2222 3500 \n",
              "L 2222 3053 \n",
              "L 1275 3053 \n",
              "L 1275 0 \n",
              "L 697 0 \n",
              "L 697 3053 \n",
              "L 147 3053 \n",
              "L 147 3500 \n",
              "L 697 3500 \n",
              "L 697 3744 \n",
              "Q 697 4328 969 4595 \n",
              "Q 1241 4863 1831 4863 \n",
              "L 2375 4863 \n",
              "z\n",
              "\" transform=\"scale(0.015625)\"/>\n",
              "      <path id=\"DejaVuSans-41\" d=\"M 2188 4044 \n",
              "L 1331 1722 \n",
              "L 3047 1722 \n",
              "L 2188 4044 \n",
              "z\n",
              "M 1831 4666 \n",
              "L 2547 4666 \n",
              "L 4325 0 \n",
              "L 3669 0 \n",
              "L 3244 1197 \n",
              "L 1141 1197 \n",
              "L 716 0 \n",
              "L 50 0 \n",
              "L 1831 4666 \n",
              "z\n",
              "\" transform=\"scale(0.015625)\"/>\n",
              "      <path id=\"DejaVuSans-72\" d=\"M 2631 2963 \n",
              "Q 2534 3019 2420 3045 \n",
              "Q 2306 3072 2169 3072 \n",
              "Q 1681 3072 1420 2755 \n",
              "Q 1159 2438 1159 1844 \n",
              "L 1159 0 \n",
              "L 581 0 \n",
              "L 581 3500 \n",
              "L 1159 3500 \n",
              "L 1159 2956 \n",
              "Q 1341 3275 1631 3429 \n",
              "Q 1922 3584 2338 3584 \n",
              "Q 2397 3584 2469 3576 \n",
              "Q 2541 3569 2628 3553 \n",
              "L 2631 2963 \n",
              "z\n",
              "\" transform=\"scale(0.015625)\"/>\n",
              "      <path id=\"DejaVuSans-63\" d=\"M 3122 3366 \n",
              "L 3122 2828 \n",
              "Q 2878 2963 2633 3030 \n",
              "Q 2388 3097 2138 3097 \n",
              "Q 1578 3097 1268 2742 \n",
              "Q 959 2388 959 1747 \n",
              "Q 959 1106 1268 751 \n",
              "Q 1578 397 2138 397 \n",
              "Q 2388 397 2633 464 \n",
              "Q 2878 531 3122 666 \n",
              "L 3122 134 \n",
              "Q 2881 22 2623 -34 \n",
              "Q 2366 -91 2075 -91 \n",
              "Q 1284 -91 818 406 \n",
              "Q 353 903 353 1747 \n",
              "Q 353 2603 823 3093 \n",
              "Q 1294 3584 2113 3584 \n",
              "Q 2378 3584 2631 3529 \n",
              "Q 2884 3475 3122 3366 \n",
              "z\n",
              "\" transform=\"scale(0.015625)\"/>\n",
              "     </defs>\n",
              "     <use xlink:href=\"#DejaVuSans-74\"/>\n",
              "     <use xlink:href=\"#DejaVuSans-2d\" x=\"39.208984\"/>\n",
              "     <use xlink:href=\"#DejaVuSans-53\" x=\"75.292969\"/>\n",
              "     <use xlink:href=\"#DejaVuSans-4e\" x=\"138.769531\"/>\n",
              "     <use xlink:href=\"#DejaVuSans-45\" x=\"213.574219\"/>\n",
              "     <use xlink:href=\"#DejaVuSans-20\" x=\"276.757812\"/>\n",
              "     <use xlink:href=\"#DejaVuSans-56\" x=\"308.544922\"/>\n",
              "     <use xlink:href=\"#DejaVuSans-69\" x=\"374.703125\"/>\n",
              "     <use xlink:href=\"#DejaVuSans-73\" x=\"402.486328\"/>\n",
              "     <use xlink:href=\"#DejaVuSans-75\" x=\"454.585938\"/>\n",
              "     <use xlink:href=\"#DejaVuSans-61\" x=\"517.964844\"/>\n",
              "     <use xlink:href=\"#DejaVuSans-6c\" x=\"579.244141\"/>\n",
              "     <use xlink:href=\"#DejaVuSans-69\" x=\"607.027344\"/>\n",
              "     <use xlink:href=\"#DejaVuSans-7a\" x=\"634.810547\"/>\n",
              "     <use xlink:href=\"#DejaVuSans-61\" x=\"687.300781\"/>\n",
              "     <use xlink:href=\"#DejaVuSans-74\" x=\"748.580078\"/>\n",
              "     <use xlink:href=\"#DejaVuSans-69\" x=\"787.789062\"/>\n",
              "     <use xlink:href=\"#DejaVuSans-6f\" x=\"815.572266\"/>\n",
              "     <use xlink:href=\"#DejaVuSans-6e\" x=\"876.753906\"/>\n",
              "     <use xlink:href=\"#DejaVuSans-20\" x=\"940.132812\"/>\n",
              "     <use xlink:href=\"#DejaVuSans-6f\" x=\"971.919922\"/>\n",
              "     <use xlink:href=\"#DejaVuSans-66\" x=\"1033.101562\"/>\n",
              "     <use xlink:href=\"#DejaVuSans-20\" x=\"1068.306641\"/>\n",
              "     <use xlink:href=\"#DejaVuSans-41\" x=\"1100.09375\"/>\n",
              "     <use xlink:href=\"#DejaVuSans-72\" x=\"1168.501953\"/>\n",
              "     <use xlink:href=\"#DejaVuSans-74\" x=\"1209.615234\"/>\n",
              "     <use xlink:href=\"#DejaVuSans-69\" x=\"1248.824219\"/>\n",
              "     <use xlink:href=\"#DejaVuSans-63\" x=\"1276.607422\"/>\n",
              "     <use xlink:href=\"#DejaVuSans-6c\" x=\"1331.587891\"/>\n",
              "     <use xlink:href=\"#DejaVuSans-65\" x=\"1359.371094\"/>\n",
              "     <use xlink:href=\"#DejaVuSans-20\" x=\"1420.894531\"/>\n",
              "     <use xlink:href=\"#DejaVuSans-43\" x=\"1452.681641\"/>\n",
              "     <use xlink:href=\"#DejaVuSans-6c\" x=\"1522.505859\"/>\n",
              "     <use xlink:href=\"#DejaVuSans-75\" x=\"1550.289062\"/>\n",
              "     <use xlink:href=\"#DejaVuSans-73\" x=\"1613.667969\"/>\n",
              "     <use xlink:href=\"#DejaVuSans-74\" x=\"1665.767578\"/>\n",
              "     <use xlink:href=\"#DejaVuSans-65\" x=\"1704.976562\"/>\n",
              "     <use xlink:href=\"#DejaVuSans-72\" x=\"1766.5\"/>\n",
              "     <use xlink:href=\"#DejaVuSans-73\" x=\"1807.613281\"/>\n",
              "    </g>\n",
              "   </g>\n",
              "   <g id=\"legend_1\">\n",
              "    <g id=\"patch_7\">\n",
              "     <path d=\"M 518.523125 458.263281 \n",
              "L 601.918438 458.263281 \n",
              "Q 603.918438 458.263281 603.918438 456.263281 \n",
              "L 603.918438 31.597656 \n",
              "Q 603.918438 29.597656 601.918438 29.597656 \n",
              "L 518.523125 29.597656 \n",
              "Q 516.523125 29.597656 516.523125 31.597656 \n",
              "L 516.523125 456.263281 \n",
              "Q 516.523125 458.263281 518.523125 458.263281 \n",
              "z\n",
              "\" style=\"fill: #e5e5e5; opacity: 0.8; stroke: #cccccc; stroke-width: 0.5; stroke-linejoin: miter\"/>\n",
              "    </g>\n",
              "    <g id=\"text_20\">\n",
              "     <!-- Clusters -->\n",
              "     <g transform=\"translate(539.87 41.196094) scale(0.1 -0.1)\">\n",
              "      <use xlink:href=\"#DejaVuSans-43\"/>\n",
              "      <use xlink:href=\"#DejaVuSans-6c\" x=\"69.824219\"/>\n",
              "      <use xlink:href=\"#DejaVuSans-75\" x=\"97.607422\"/>\n",
              "      <use xlink:href=\"#DejaVuSans-73\" x=\"160.986328\"/>\n",
              "      <use xlink:href=\"#DejaVuSans-74\" x=\"213.085938\"/>\n",
              "      <use xlink:href=\"#DejaVuSans-65\" x=\"252.294922\"/>\n",
              "      <use xlink:href=\"#DejaVuSans-72\" x=\"313.818359\"/>\n",
              "      <use xlink:href=\"#DejaVuSans-73\" x=\"354.931641\"/>\n",
              "     </g>\n",
              "    </g>\n",
              "    <g id=\"PathCollection_29\">\n",
              "     <g>\n",
              "      <use xlink:href=\"#mc2a3f2c82a\" x=\"530.523125\" y=\"53.249219\" style=\"fill: #1f77b4; stroke: #1f77b4; stroke-width: 0.5\"/>\n",
              "     </g>\n",
              "    </g>\n",
              "    <g id=\"text_21\">\n",
              "     <!-- Cluster 0 -->\n",
              "     <g transform=\"translate(548.523125 55.874219) scale(0.1 -0.1)\">\n",
              "      <use xlink:href=\"#DejaVuSans-43\"/>\n",
              "      <use xlink:href=\"#DejaVuSans-6c\" x=\"69.824219\"/>\n",
              "      <use xlink:href=\"#DejaVuSans-75\" x=\"97.607422\"/>\n",
              "      <use xlink:href=\"#DejaVuSans-73\" x=\"160.986328\"/>\n",
              "      <use xlink:href=\"#DejaVuSans-74\" x=\"213.085938\"/>\n",
              "      <use xlink:href=\"#DejaVuSans-65\" x=\"252.294922\"/>\n",
              "      <use xlink:href=\"#DejaVuSans-72\" x=\"313.818359\"/>\n",
              "      <use xlink:href=\"#DejaVuSans-20\" x=\"354.931641\"/>\n",
              "      <use xlink:href=\"#DejaVuSans-30\" x=\"386.71875\"/>\n",
              "     </g>\n",
              "    </g>\n",
              "    <g id=\"PathCollection_30\">\n",
              "     <g>\n",
              "      <use xlink:href=\"#mba7e5ca1c2\" x=\"530.523125\" y=\"67.927344\" style=\"fill: #aec7e8; stroke: #aec7e8; stroke-width: 0.5\"/>\n",
              "     </g>\n",
              "    </g>\n",
              "    <g id=\"text_22\">\n",
              "     <!-- Cluster 1 -->\n",
              "     <g transform=\"translate(548.523125 70.552344) scale(0.1 -0.1)\">\n",
              "      <use xlink:href=\"#DejaVuSans-43\"/>\n",
              "      <use xlink:href=\"#DejaVuSans-6c\" x=\"69.824219\"/>\n",
              "      <use xlink:href=\"#DejaVuSans-75\" x=\"97.607422\"/>\n",
              "      <use xlink:href=\"#DejaVuSans-73\" x=\"160.986328\"/>\n",
              "      <use xlink:href=\"#DejaVuSans-74\" x=\"213.085938\"/>\n",
              "      <use xlink:href=\"#DejaVuSans-65\" x=\"252.294922\"/>\n",
              "      <use xlink:href=\"#DejaVuSans-72\" x=\"313.818359\"/>\n",
              "      <use xlink:href=\"#DejaVuSans-20\" x=\"354.931641\"/>\n",
              "      <use xlink:href=\"#DejaVuSans-31\" x=\"386.71875\"/>\n",
              "     </g>\n",
              "    </g>\n",
              "    <g id=\"PathCollection_31\">\n",
              "     <g>\n",
              "      <use xlink:href=\"#m8d03ac8010\" x=\"530.523125\" y=\"82.605469\" style=\"fill: #ff7f0e; stroke: #ff7f0e; stroke-width: 0.5\"/>\n",
              "     </g>\n",
              "    </g>\n",
              "    <g id=\"text_23\">\n",
              "     <!-- Cluster 2 -->\n",
              "     <g transform=\"translate(548.523125 85.230469) scale(0.1 -0.1)\">\n",
              "      <use xlink:href=\"#DejaVuSans-43\"/>\n",
              "      <use xlink:href=\"#DejaVuSans-6c\" x=\"69.824219\"/>\n",
              "      <use xlink:href=\"#DejaVuSans-75\" x=\"97.607422\"/>\n",
              "      <use xlink:href=\"#DejaVuSans-73\" x=\"160.986328\"/>\n",
              "      <use xlink:href=\"#DejaVuSans-74\" x=\"213.085938\"/>\n",
              "      <use xlink:href=\"#DejaVuSans-65\" x=\"252.294922\"/>\n",
              "      <use xlink:href=\"#DejaVuSans-72\" x=\"313.818359\"/>\n",
              "      <use xlink:href=\"#DejaVuSans-20\" x=\"354.931641\"/>\n",
              "      <use xlink:href=\"#DejaVuSans-32\" x=\"386.71875\"/>\n",
              "     </g>\n",
              "    </g>\n",
              "    <g id=\"PathCollection_32\">\n",
              "     <g>\n",
              "      <use xlink:href=\"#mba227bf8ff\" x=\"530.523125\" y=\"97.283594\" style=\"fill: #ffbb78; stroke: #ffbb78; stroke-width: 0.5\"/>\n",
              "     </g>\n",
              "    </g>\n",
              "    <g id=\"text_24\">\n",
              "     <!-- Cluster 3 -->\n",
              "     <g transform=\"translate(548.523125 99.908594) scale(0.1 -0.1)\">\n",
              "      <use xlink:href=\"#DejaVuSans-43\"/>\n",
              "      <use xlink:href=\"#DejaVuSans-6c\" x=\"69.824219\"/>\n",
              "      <use xlink:href=\"#DejaVuSans-75\" x=\"97.607422\"/>\n",
              "      <use xlink:href=\"#DejaVuSans-73\" x=\"160.986328\"/>\n",
              "      <use xlink:href=\"#DejaVuSans-74\" x=\"213.085938\"/>\n",
              "      <use xlink:href=\"#DejaVuSans-65\" x=\"252.294922\"/>\n",
              "      <use xlink:href=\"#DejaVuSans-72\" x=\"313.818359\"/>\n",
              "      <use xlink:href=\"#DejaVuSans-20\" x=\"354.931641\"/>\n",
              "      <use xlink:href=\"#DejaVuSans-33\" x=\"386.71875\"/>\n",
              "     </g>\n",
              "    </g>\n",
              "    <g id=\"PathCollection_33\">\n",
              "     <g>\n",
              "      <use xlink:href=\"#maf2cd97d8e\" x=\"530.523125\" y=\"111.961719\" style=\"fill: #2ca02c; stroke: #2ca02c; stroke-width: 0.5\"/>\n",
              "     </g>\n",
              "    </g>\n",
              "    <g id=\"text_25\">\n",
              "     <!-- Cluster 4 -->\n",
              "     <g transform=\"translate(548.523125 114.586719) scale(0.1 -0.1)\">\n",
              "      <use xlink:href=\"#DejaVuSans-43\"/>\n",
              "      <use xlink:href=\"#DejaVuSans-6c\" x=\"69.824219\"/>\n",
              "      <use xlink:href=\"#DejaVuSans-75\" x=\"97.607422\"/>\n",
              "      <use xlink:href=\"#DejaVuSans-73\" x=\"160.986328\"/>\n",
              "      <use xlink:href=\"#DejaVuSans-74\" x=\"213.085938\"/>\n",
              "      <use xlink:href=\"#DejaVuSans-65\" x=\"252.294922\"/>\n",
              "      <use xlink:href=\"#DejaVuSans-72\" x=\"313.818359\"/>\n",
              "      <use xlink:href=\"#DejaVuSans-20\" x=\"354.931641\"/>\n",
              "      <use xlink:href=\"#DejaVuSans-34\" x=\"386.71875\"/>\n",
              "     </g>\n",
              "    </g>\n",
              "    <g id=\"PathCollection_34\">\n",
              "     <g>\n",
              "      <use xlink:href=\"#m62c8c1ad36\" x=\"530.523125\" y=\"126.639844\" style=\"fill: #98df8a; stroke: #98df8a; stroke-width: 0.5\"/>\n",
              "     </g>\n",
              "    </g>\n",
              "    <g id=\"text_26\">\n",
              "     <!-- Cluster 5 -->\n",
              "     <g transform=\"translate(548.523125 129.264844) scale(0.1 -0.1)\">\n",
              "      <defs>\n",
              "       <path id=\"DejaVuSans-35\" d=\"M 691 4666 \n",
              "L 3169 4666 \n",
              "L 3169 4134 \n",
              "L 1269 4134 \n",
              "L 1269 2991 \n",
              "Q 1406 3038 1543 3061 \n",
              "Q 1681 3084 1819 3084 \n",
              "Q 2600 3084 3056 2656 \n",
              "Q 3513 2228 3513 1497 \n",
              "Q 3513 744 3044 326 \n",
              "Q 2575 -91 1722 -91 \n",
              "Q 1428 -91 1123 -41 \n",
              "Q 819 9 494 109 \n",
              "L 494 744 \n",
              "Q 775 591 1075 516 \n",
              "Q 1375 441 1709 441 \n",
              "Q 2250 441 2565 725 \n",
              "Q 2881 1009 2881 1497 \n",
              "Q 2881 1984 2565 2268 \n",
              "Q 2250 2553 1709 2553 \n",
              "Q 1456 2553 1204 2497 \n",
              "Q 953 2441 691 2322 \n",
              "L 691 4666 \n",
              "z\n",
              "\" transform=\"scale(0.015625)\"/>\n",
              "      </defs>\n",
              "      <use xlink:href=\"#DejaVuSans-43\"/>\n",
              "      <use xlink:href=\"#DejaVuSans-6c\" x=\"69.824219\"/>\n",
              "      <use xlink:href=\"#DejaVuSans-75\" x=\"97.607422\"/>\n",
              "      <use xlink:href=\"#DejaVuSans-73\" x=\"160.986328\"/>\n",
              "      <use xlink:href=\"#DejaVuSans-74\" x=\"213.085938\"/>\n",
              "      <use xlink:href=\"#DejaVuSans-65\" x=\"252.294922\"/>\n",
              "      <use xlink:href=\"#DejaVuSans-72\" x=\"313.818359\"/>\n",
              "      <use xlink:href=\"#DejaVuSans-20\" x=\"354.931641\"/>\n",
              "      <use xlink:href=\"#DejaVuSans-35\" x=\"386.71875\"/>\n",
              "     </g>\n",
              "    </g>\n",
              "    <g id=\"PathCollection_35\">\n",
              "     <g>\n",
              "      <use xlink:href=\"#m92d6cab3c5\" x=\"530.523125\" y=\"141.317969\" style=\"fill: #d62728; stroke: #d62728; stroke-width: 0.5\"/>\n",
              "     </g>\n",
              "    </g>\n",
              "    <g id=\"text_27\">\n",
              "     <!-- Cluster 6 -->\n",
              "     <g transform=\"translate(548.523125 143.942969) scale(0.1 -0.1)\">\n",
              "      <use xlink:href=\"#DejaVuSans-43\"/>\n",
              "      <use xlink:href=\"#DejaVuSans-6c\" x=\"69.824219\"/>\n",
              "      <use xlink:href=\"#DejaVuSans-75\" x=\"97.607422\"/>\n",
              "      <use xlink:href=\"#DejaVuSans-73\" x=\"160.986328\"/>\n",
              "      <use xlink:href=\"#DejaVuSans-74\" x=\"213.085938\"/>\n",
              "      <use xlink:href=\"#DejaVuSans-65\" x=\"252.294922\"/>\n",
              "      <use xlink:href=\"#DejaVuSans-72\" x=\"313.818359\"/>\n",
              "      <use xlink:href=\"#DejaVuSans-20\" x=\"354.931641\"/>\n",
              "      <use xlink:href=\"#DejaVuSans-36\" x=\"386.71875\"/>\n",
              "     </g>\n",
              "    </g>\n",
              "    <g id=\"PathCollection_36\">\n",
              "     <g>\n",
              "      <use xlink:href=\"#mefe40797ef\" x=\"530.523125\" y=\"155.996094\" style=\"fill: #ff9896; stroke: #ff9896; stroke-width: 0.5\"/>\n",
              "     </g>\n",
              "    </g>\n",
              "    <g id=\"text_28\">\n",
              "     <!-- Cluster 7 -->\n",
              "     <g transform=\"translate(548.523125 158.621094) scale(0.1 -0.1)\">\n",
              "      <defs>\n",
              "       <path id=\"DejaVuSans-37\" d=\"M 525 4666 \n",
              "L 3525 4666 \n",
              "L 3525 4397 \n",
              "L 1831 0 \n",
              "L 1172 0 \n",
              "L 2766 4134 \n",
              "L 525 4134 \n",
              "L 525 4666 \n",
              "z\n",
              "\" transform=\"scale(0.015625)\"/>\n",
              "      </defs>\n",
              "      <use xlink:href=\"#DejaVuSans-43\"/>\n",
              "      <use xlink:href=\"#DejaVuSans-6c\" x=\"69.824219\"/>\n",
              "      <use xlink:href=\"#DejaVuSans-75\" x=\"97.607422\"/>\n",
              "      <use xlink:href=\"#DejaVuSans-73\" x=\"160.986328\"/>\n",
              "      <use xlink:href=\"#DejaVuSans-74\" x=\"213.085938\"/>\n",
              "      <use xlink:href=\"#DejaVuSans-65\" x=\"252.294922\"/>\n",
              "      <use xlink:href=\"#DejaVuSans-72\" x=\"313.818359\"/>\n",
              "      <use xlink:href=\"#DejaVuSans-20\" x=\"354.931641\"/>\n",
              "      <use xlink:href=\"#DejaVuSans-37\" x=\"386.71875\"/>\n",
              "     </g>\n",
              "    </g>\n",
              "    <g id=\"PathCollection_37\">\n",
              "     <g>\n",
              "      <use xlink:href=\"#mc83ea64be4\" x=\"530.523125\" y=\"170.674219\" style=\"fill: #9467bd; stroke: #9467bd; stroke-width: 0.5\"/>\n",
              "     </g>\n",
              "    </g>\n",
              "    <g id=\"text_29\">\n",
              "     <!-- Cluster 8 -->\n",
              "     <g transform=\"translate(548.523125 173.299219) scale(0.1 -0.1)\">\n",
              "      <defs>\n",
              "       <path id=\"DejaVuSans-38\" d=\"M 2034 2216 \n",
              "Q 1584 2216 1326 1975 \n",
              "Q 1069 1734 1069 1313 \n",
              "Q 1069 891 1326 650 \n",
              "Q 1584 409 2034 409 \n",
              "Q 2484 409 2743 651 \n",
              "Q 3003 894 3003 1313 \n",
              "Q 3003 1734 2745 1975 \n",
              "Q 2488 2216 2034 2216 \n",
              "z\n",
              "M 1403 2484 \n",
              "Q 997 2584 770 2862 \n",
              "Q 544 3141 544 3541 \n",
              "Q 544 4100 942 4425 \n",
              "Q 1341 4750 2034 4750 \n",
              "Q 2731 4750 3128 4425 \n",
              "Q 3525 4100 3525 3541 \n",
              "Q 3525 3141 3298 2862 \n",
              "Q 3072 2584 2669 2484 \n",
              "Q 3125 2378 3379 2068 \n",
              "Q 3634 1759 3634 1313 \n",
              "Q 3634 634 3220 271 \n",
              "Q 2806 -91 2034 -91 \n",
              "Q 1263 -91 848 271 \n",
              "Q 434 634 434 1313 \n",
              "Q 434 1759 690 2068 \n",
              "Q 947 2378 1403 2484 \n",
              "z\n",
              "M 1172 3481 \n",
              "Q 1172 3119 1398 2916 \n",
              "Q 1625 2713 2034 2713 \n",
              "Q 2441 2713 2670 2916 \n",
              "Q 2900 3119 2900 3481 \n",
              "Q 2900 3844 2670 4047 \n",
              "Q 2441 4250 2034 4250 \n",
              "Q 1625 4250 1398 4047 \n",
              "Q 1172 3844 1172 3481 \n",
              "z\n",
              "\" transform=\"scale(0.015625)\"/>\n",
              "      </defs>\n",
              "      <use xlink:href=\"#DejaVuSans-43\"/>\n",
              "      <use xlink:href=\"#DejaVuSans-6c\" x=\"69.824219\"/>\n",
              "      <use xlink:href=\"#DejaVuSans-75\" x=\"97.607422\"/>\n",
              "      <use xlink:href=\"#DejaVuSans-73\" x=\"160.986328\"/>\n",
              "      <use xlink:href=\"#DejaVuSans-74\" x=\"213.085938\"/>\n",
              "      <use xlink:href=\"#DejaVuSans-65\" x=\"252.294922\"/>\n",
              "      <use xlink:href=\"#DejaVuSans-72\" x=\"313.818359\"/>\n",
              "      <use xlink:href=\"#DejaVuSans-20\" x=\"354.931641\"/>\n",
              "      <use xlink:href=\"#DejaVuSans-38\" x=\"386.71875\"/>\n",
              "     </g>\n",
              "    </g>\n",
              "    <g id=\"PathCollection_38\">\n",
              "     <g>\n",
              "      <use xlink:href=\"#mca817ee18d\" x=\"530.523125\" y=\"185.352344\" style=\"fill: #c5b0d5; stroke: #c5b0d5; stroke-width: 0.5\"/>\n",
              "     </g>\n",
              "    </g>\n",
              "    <g id=\"text_30\">\n",
              "     <!-- Cluster 9 -->\n",
              "     <g transform=\"translate(548.523125 187.977344) scale(0.1 -0.1)\">\n",
              "      <defs>\n",
              "       <path id=\"DejaVuSans-39\" d=\"M 703 97 \n",
              "L 703 672 \n",
              "Q 941 559 1184 500 \n",
              "Q 1428 441 1663 441 \n",
              "Q 2288 441 2617 861 \n",
              "Q 2947 1281 2994 2138 \n",
              "Q 2813 1869 2534 1725 \n",
              "Q 2256 1581 1919 1581 \n",
              "Q 1219 1581 811 2004 \n",
              "Q 403 2428 403 3163 \n",
              "Q 403 3881 828 4315 \n",
              "Q 1253 4750 1959 4750 \n",
              "Q 2769 4750 3195 4129 \n",
              "Q 3622 3509 3622 2328 \n",
              "Q 3622 1225 3098 567 \n",
              "Q 2575 -91 1691 -91 \n",
              "Q 1453 -91 1209 -44 \n",
              "Q 966 3 703 97 \n",
              "z\n",
              "M 1959 2075 \n",
              "Q 2384 2075 2632 2365 \n",
              "Q 2881 2656 2881 3163 \n",
              "Q 2881 3666 2632 3958 \n",
              "Q 2384 4250 1959 4250 \n",
              "Q 1534 4250 1286 3958 \n",
              "Q 1038 3666 1038 3163 \n",
              "Q 1038 2656 1286 2365 \n",
              "Q 1534 2075 1959 2075 \n",
              "z\n",
              "\" transform=\"scale(0.015625)\"/>\n",
              "      </defs>\n",
              "      <use xlink:href=\"#DejaVuSans-43\"/>\n",
              "      <use xlink:href=\"#DejaVuSans-6c\" x=\"69.824219\"/>\n",
              "      <use xlink:href=\"#DejaVuSans-75\" x=\"97.607422\"/>\n",
              "      <use xlink:href=\"#DejaVuSans-73\" x=\"160.986328\"/>\n",
              "      <use xlink:href=\"#DejaVuSans-74\" x=\"213.085938\"/>\n",
              "      <use xlink:href=\"#DejaVuSans-65\" x=\"252.294922\"/>\n",
              "      <use xlink:href=\"#DejaVuSans-72\" x=\"313.818359\"/>\n",
              "      <use xlink:href=\"#DejaVuSans-20\" x=\"354.931641\"/>\n",
              "      <use xlink:href=\"#DejaVuSans-39\" x=\"386.71875\"/>\n",
              "     </g>\n",
              "    </g>\n",
              "    <g id=\"PathCollection_39\">\n",
              "     <g>\n",
              "      <use xlink:href=\"#mecdb7a7eca\" x=\"530.523125\" y=\"200.030469\" style=\"fill: #8c564b; stroke: #8c564b; stroke-width: 0.5\"/>\n",
              "     </g>\n",
              "    </g>\n",
              "    <g id=\"text_31\">\n",
              "     <!-- Cluster 10 -->\n",
              "     <g transform=\"translate(548.523125 202.655469) scale(0.1 -0.1)\">\n",
              "      <use xlink:href=\"#DejaVuSans-43\"/>\n",
              "      <use xlink:href=\"#DejaVuSans-6c\" x=\"69.824219\"/>\n",
              "      <use xlink:href=\"#DejaVuSans-75\" x=\"97.607422\"/>\n",
              "      <use xlink:href=\"#DejaVuSans-73\" x=\"160.986328\"/>\n",
              "      <use xlink:href=\"#DejaVuSans-74\" x=\"213.085938\"/>\n",
              "      <use xlink:href=\"#DejaVuSans-65\" x=\"252.294922\"/>\n",
              "      <use xlink:href=\"#DejaVuSans-72\" x=\"313.818359\"/>\n",
              "      <use xlink:href=\"#DejaVuSans-20\" x=\"354.931641\"/>\n",
              "      <use xlink:href=\"#DejaVuSans-31\" x=\"386.71875\"/>\n",
              "      <use xlink:href=\"#DejaVuSans-30\" x=\"450.341797\"/>\n",
              "     </g>\n",
              "    </g>\n",
              "    <g id=\"PathCollection_40\">\n",
              "     <g>\n",
              "      <use xlink:href=\"#m26fe20d470\" x=\"530.523125\" y=\"214.708594\" style=\"fill: #c49c94; stroke: #c49c94; stroke-width: 0.5\"/>\n",
              "     </g>\n",
              "    </g>\n",
              "    <g id=\"text_32\">\n",
              "     <!-- Cluster 11 -->\n",
              "     <g transform=\"translate(548.523125 217.333594) scale(0.1 -0.1)\">\n",
              "      <use xlink:href=\"#DejaVuSans-43\"/>\n",
              "      <use xlink:href=\"#DejaVuSans-6c\" x=\"69.824219\"/>\n",
              "      <use xlink:href=\"#DejaVuSans-75\" x=\"97.607422\"/>\n",
              "      <use xlink:href=\"#DejaVuSans-73\" x=\"160.986328\"/>\n",
              "      <use xlink:href=\"#DejaVuSans-74\" x=\"213.085938\"/>\n",
              "      <use xlink:href=\"#DejaVuSans-65\" x=\"252.294922\"/>\n",
              "      <use xlink:href=\"#DejaVuSans-72\" x=\"313.818359\"/>\n",
              "      <use xlink:href=\"#DejaVuSans-20\" x=\"354.931641\"/>\n",
              "      <use xlink:href=\"#DejaVuSans-31\" x=\"386.71875\"/>\n",
              "      <use xlink:href=\"#DejaVuSans-31\" x=\"450.341797\"/>\n",
              "     </g>\n",
              "    </g>\n",
              "    <g id=\"PathCollection_41\">\n",
              "     <g>\n",
              "      <use xlink:href=\"#m89411d035a\" x=\"530.523125\" y=\"229.386719\" style=\"fill: #e377c2; stroke: #e377c2; stroke-width: 0.5\"/>\n",
              "     </g>\n",
              "    </g>\n",
              "    <g id=\"text_33\">\n",
              "     <!-- Cluster 12 -->\n",
              "     <g transform=\"translate(548.523125 232.011719) scale(0.1 -0.1)\">\n",
              "      <use xlink:href=\"#DejaVuSans-43\"/>\n",
              "      <use xlink:href=\"#DejaVuSans-6c\" x=\"69.824219\"/>\n",
              "      <use xlink:href=\"#DejaVuSans-75\" x=\"97.607422\"/>\n",
              "      <use xlink:href=\"#DejaVuSans-73\" x=\"160.986328\"/>\n",
              "      <use xlink:href=\"#DejaVuSans-74\" x=\"213.085938\"/>\n",
              "      <use xlink:href=\"#DejaVuSans-65\" x=\"252.294922\"/>\n",
              "      <use xlink:href=\"#DejaVuSans-72\" x=\"313.818359\"/>\n",
              "      <use xlink:href=\"#DejaVuSans-20\" x=\"354.931641\"/>\n",
              "      <use xlink:href=\"#DejaVuSans-31\" x=\"386.71875\"/>\n",
              "      <use xlink:href=\"#DejaVuSans-32\" x=\"450.341797\"/>\n",
              "     </g>\n",
              "    </g>\n",
              "    <g id=\"PathCollection_42\">\n",
              "     <g>\n",
              "      <use xlink:href=\"#m2d84f52954\" x=\"530.523125\" y=\"244.064844\" style=\"fill: #f7b6d2; stroke: #f7b6d2; stroke-width: 0.5\"/>\n",
              "     </g>\n",
              "    </g>\n",
              "    <g id=\"text_34\">\n",
              "     <!-- Cluster 13 -->\n",
              "     <g transform=\"translate(548.523125 246.689844) scale(0.1 -0.1)\">\n",
              "      <use xlink:href=\"#DejaVuSans-43\"/>\n",
              "      <use xlink:href=\"#DejaVuSans-6c\" x=\"69.824219\"/>\n",
              "      <use xlink:href=\"#DejaVuSans-75\" x=\"97.607422\"/>\n",
              "      <use xlink:href=\"#DejaVuSans-73\" x=\"160.986328\"/>\n",
              "      <use xlink:href=\"#DejaVuSans-74\" x=\"213.085938\"/>\n",
              "      <use xlink:href=\"#DejaVuSans-65\" x=\"252.294922\"/>\n",
              "      <use xlink:href=\"#DejaVuSans-72\" x=\"313.818359\"/>\n",
              "      <use xlink:href=\"#DejaVuSans-20\" x=\"354.931641\"/>\n",
              "      <use xlink:href=\"#DejaVuSans-31\" x=\"386.71875\"/>\n",
              "      <use xlink:href=\"#DejaVuSans-33\" x=\"450.341797\"/>\n",
              "     </g>\n",
              "    </g>\n",
              "    <g id=\"PathCollection_43\">\n",
              "     <g>\n",
              "      <use xlink:href=\"#me8168e8a62\" x=\"530.523125\" y=\"258.742969\" style=\"fill: #7f7f7f; stroke: #7f7f7f; stroke-width: 0.5\"/>\n",
              "     </g>\n",
              "    </g>\n",
              "    <g id=\"text_35\">\n",
              "     <!-- Cluster 14 -->\n",
              "     <g transform=\"translate(548.523125 261.367969) scale(0.1 -0.1)\">\n",
              "      <use xlink:href=\"#DejaVuSans-43\"/>\n",
              "      <use xlink:href=\"#DejaVuSans-6c\" x=\"69.824219\"/>\n",
              "      <use xlink:href=\"#DejaVuSans-75\" x=\"97.607422\"/>\n",
              "      <use xlink:href=\"#DejaVuSans-73\" x=\"160.986328\"/>\n",
              "      <use xlink:href=\"#DejaVuSans-74\" x=\"213.085938\"/>\n",
              "      <use xlink:href=\"#DejaVuSans-65\" x=\"252.294922\"/>\n",
              "      <use xlink:href=\"#DejaVuSans-72\" x=\"313.818359\"/>\n",
              "      <use xlink:href=\"#DejaVuSans-20\" x=\"354.931641\"/>\n",
              "      <use xlink:href=\"#DejaVuSans-31\" x=\"386.71875\"/>\n",
              "      <use xlink:href=\"#DejaVuSans-34\" x=\"450.341797\"/>\n",
              "     </g>\n",
              "    </g>\n",
              "    <g id=\"PathCollection_44\">\n",
              "     <g>\n",
              "      <use xlink:href=\"#m1c4d8747ba\" x=\"530.523125\" y=\"273.421094\" style=\"fill: #c7c7c7; stroke: #c7c7c7; stroke-width: 0.5\"/>\n",
              "     </g>\n",
              "    </g>\n",
              "    <g id=\"text_36\">\n",
              "     <!-- Cluster 15 -->\n",
              "     <g transform=\"translate(548.523125 276.046094) scale(0.1 -0.1)\">\n",
              "      <use xlink:href=\"#DejaVuSans-43\"/>\n",
              "      <use xlink:href=\"#DejaVuSans-6c\" x=\"69.824219\"/>\n",
              "      <use xlink:href=\"#DejaVuSans-75\" x=\"97.607422\"/>\n",
              "      <use xlink:href=\"#DejaVuSans-73\" x=\"160.986328\"/>\n",
              "      <use xlink:href=\"#DejaVuSans-74\" x=\"213.085938\"/>\n",
              "      <use xlink:href=\"#DejaVuSans-65\" x=\"252.294922\"/>\n",
              "      <use xlink:href=\"#DejaVuSans-72\" x=\"313.818359\"/>\n",
              "      <use xlink:href=\"#DejaVuSans-20\" x=\"354.931641\"/>\n",
              "      <use xlink:href=\"#DejaVuSans-31\" x=\"386.71875\"/>\n",
              "      <use xlink:href=\"#DejaVuSans-35\" x=\"450.341797\"/>\n",
              "     </g>\n",
              "    </g>\n",
              "    <g id=\"PathCollection_45\">\n",
              "     <g>\n",
              "      <use xlink:href=\"#mf30c6b51d6\" x=\"530.523125\" y=\"288.099219\" style=\"fill: #bcbd22; stroke: #bcbd22; stroke-width: 0.5\"/>\n",
              "     </g>\n",
              "    </g>\n",
              "    <g id=\"text_37\">\n",
              "     <!-- Cluster 16 -->\n",
              "     <g transform=\"translate(548.523125 290.724219) scale(0.1 -0.1)\">\n",
              "      <use xlink:href=\"#DejaVuSans-43\"/>\n",
              "      <use xlink:href=\"#DejaVuSans-6c\" x=\"69.824219\"/>\n",
              "      <use xlink:href=\"#DejaVuSans-75\" x=\"97.607422\"/>\n",
              "      <use xlink:href=\"#DejaVuSans-73\" x=\"160.986328\"/>\n",
              "      <use xlink:href=\"#DejaVuSans-74\" x=\"213.085938\"/>\n",
              "      <use xlink:href=\"#DejaVuSans-65\" x=\"252.294922\"/>\n",
              "      <use xlink:href=\"#DejaVuSans-72\" x=\"313.818359\"/>\n",
              "      <use xlink:href=\"#DejaVuSans-20\" x=\"354.931641\"/>\n",
              "      <use xlink:href=\"#DejaVuSans-31\" x=\"386.71875\"/>\n",
              "      <use xlink:href=\"#DejaVuSans-36\" x=\"450.341797\"/>\n",
              "     </g>\n",
              "    </g>\n",
              "    <g id=\"PathCollection_46\">\n",
              "     <g>\n",
              "      <use xlink:href=\"#m8b31dfbfba\" x=\"530.523125\" y=\"302.777344\" style=\"fill: #dbdb8d; stroke: #dbdb8d; stroke-width: 0.5\"/>\n",
              "     </g>\n",
              "    </g>\n",
              "    <g id=\"text_38\">\n",
              "     <!-- Cluster 17 -->\n",
              "     <g transform=\"translate(548.523125 305.402344) scale(0.1 -0.1)\">\n",
              "      <use xlink:href=\"#DejaVuSans-43\"/>\n",
              "      <use xlink:href=\"#DejaVuSans-6c\" x=\"69.824219\"/>\n",
              "      <use xlink:href=\"#DejaVuSans-75\" x=\"97.607422\"/>\n",
              "      <use xlink:href=\"#DejaVuSans-73\" x=\"160.986328\"/>\n",
              "      <use xlink:href=\"#DejaVuSans-74\" x=\"213.085938\"/>\n",
              "      <use xlink:href=\"#DejaVuSans-65\" x=\"252.294922\"/>\n",
              "      <use xlink:href=\"#DejaVuSans-72\" x=\"313.818359\"/>\n",
              "      <use xlink:href=\"#DejaVuSans-20\" x=\"354.931641\"/>\n",
              "      <use xlink:href=\"#DejaVuSans-31\" x=\"386.71875\"/>\n",
              "      <use xlink:href=\"#DejaVuSans-37\" x=\"450.341797\"/>\n",
              "     </g>\n",
              "    </g>\n",
              "    <g id=\"PathCollection_47\">\n",
              "     <g>\n",
              "      <use xlink:href=\"#m35415c4a50\" x=\"530.523125\" y=\"317.455469\" style=\"fill: #17becf; stroke: #17becf; stroke-width: 0.5\"/>\n",
              "     </g>\n",
              "    </g>\n",
              "    <g id=\"text_39\">\n",
              "     <!-- Cluster 18 -->\n",
              "     <g transform=\"translate(548.523125 320.080469) scale(0.1 -0.1)\">\n",
              "      <use xlink:href=\"#DejaVuSans-43\"/>\n",
              "      <use xlink:href=\"#DejaVuSans-6c\" x=\"69.824219\"/>\n",
              "      <use xlink:href=\"#DejaVuSans-75\" x=\"97.607422\"/>\n",
              "      <use xlink:href=\"#DejaVuSans-73\" x=\"160.986328\"/>\n",
              "      <use xlink:href=\"#DejaVuSans-74\" x=\"213.085938\"/>\n",
              "      <use xlink:href=\"#DejaVuSans-65\" x=\"252.294922\"/>\n",
              "      <use xlink:href=\"#DejaVuSans-72\" x=\"313.818359\"/>\n",
              "      <use xlink:href=\"#DejaVuSans-20\" x=\"354.931641\"/>\n",
              "      <use xlink:href=\"#DejaVuSans-31\" x=\"386.71875\"/>\n",
              "      <use xlink:href=\"#DejaVuSans-38\" x=\"450.341797\"/>\n",
              "     </g>\n",
              "    </g>\n",
              "    <g id=\"PathCollection_48\">\n",
              "     <g>\n",
              "      <use xlink:href=\"#m4960ccf95d\" x=\"530.523125\" y=\"332.133594\" style=\"fill: #9edae5; stroke: #9edae5; stroke-width: 0.5\"/>\n",
              "     </g>\n",
              "    </g>\n",
              "    <g id=\"text_40\">\n",
              "     <!-- Cluster 19 -->\n",
              "     <g transform=\"translate(548.523125 334.758594) scale(0.1 -0.1)\">\n",
              "      <use xlink:href=\"#DejaVuSans-43\"/>\n",
              "      <use xlink:href=\"#DejaVuSans-6c\" x=\"69.824219\"/>\n",
              "      <use xlink:href=\"#DejaVuSans-75\" x=\"97.607422\"/>\n",
              "      <use xlink:href=\"#DejaVuSans-73\" x=\"160.986328\"/>\n",
              "      <use xlink:href=\"#DejaVuSans-74\" x=\"213.085938\"/>\n",
              "      <use xlink:href=\"#DejaVuSans-65\" x=\"252.294922\"/>\n",
              "      <use xlink:href=\"#DejaVuSans-72\" x=\"313.818359\"/>\n",
              "      <use xlink:href=\"#DejaVuSans-20\" x=\"354.931641\"/>\n",
              "      <use xlink:href=\"#DejaVuSans-31\" x=\"386.71875\"/>\n",
              "      <use xlink:href=\"#DejaVuSans-39\" x=\"450.341797\"/>\n",
              "     </g>\n",
              "    </g>\n",
              "    <g id=\"PathCollection_49\">\n",
              "     <g>\n",
              "      <use xlink:href=\"#m6c29e73512\" x=\"530.523125\" y=\"346.811719\" style=\"fill: #393b79; stroke: #393b79; stroke-width: 0.5\"/>\n",
              "     </g>\n",
              "    </g>\n",
              "    <g id=\"text_41\">\n",
              "     <!-- Cluster 20 -->\n",
              "     <g transform=\"translate(548.523125 349.436719) scale(0.1 -0.1)\">\n",
              "      <use xlink:href=\"#DejaVuSans-43\"/>\n",
              "      <use xlink:href=\"#DejaVuSans-6c\" x=\"69.824219\"/>\n",
              "      <use xlink:href=\"#DejaVuSans-75\" x=\"97.607422\"/>\n",
              "      <use xlink:href=\"#DejaVuSans-73\" x=\"160.986328\"/>\n",
              "      <use xlink:href=\"#DejaVuSans-74\" x=\"213.085938\"/>\n",
              "      <use xlink:href=\"#DejaVuSans-65\" x=\"252.294922\"/>\n",
              "      <use xlink:href=\"#DejaVuSans-72\" x=\"313.818359\"/>\n",
              "      <use xlink:href=\"#DejaVuSans-20\" x=\"354.931641\"/>\n",
              "      <use xlink:href=\"#DejaVuSans-32\" x=\"386.71875\"/>\n",
              "      <use xlink:href=\"#DejaVuSans-30\" x=\"450.341797\"/>\n",
              "     </g>\n",
              "    </g>\n",
              "    <g id=\"PathCollection_50\">\n",
              "     <g>\n",
              "      <use xlink:href=\"#m71beae8f00\" x=\"530.523125\" y=\"361.489844\" style=\"fill: #5254a3; stroke: #5254a3; stroke-width: 0.5\"/>\n",
              "     </g>\n",
              "    </g>\n",
              "    <g id=\"text_42\">\n",
              "     <!-- Cluster 21 -->\n",
              "     <g transform=\"translate(548.523125 364.114844) scale(0.1 -0.1)\">\n",
              "      <use xlink:href=\"#DejaVuSans-43\"/>\n",
              "      <use xlink:href=\"#DejaVuSans-6c\" x=\"69.824219\"/>\n",
              "      <use xlink:href=\"#DejaVuSans-75\" x=\"97.607422\"/>\n",
              "      <use xlink:href=\"#DejaVuSans-73\" x=\"160.986328\"/>\n",
              "      <use xlink:href=\"#DejaVuSans-74\" x=\"213.085938\"/>\n",
              "      <use xlink:href=\"#DejaVuSans-65\" x=\"252.294922\"/>\n",
              "      <use xlink:href=\"#DejaVuSans-72\" x=\"313.818359\"/>\n",
              "      <use xlink:href=\"#DejaVuSans-20\" x=\"354.931641\"/>\n",
              "      <use xlink:href=\"#DejaVuSans-32\" x=\"386.71875\"/>\n",
              "      <use xlink:href=\"#DejaVuSans-31\" x=\"450.341797\"/>\n",
              "     </g>\n",
              "    </g>\n",
              "    <g id=\"PathCollection_51\">\n",
              "     <g>\n",
              "      <use xlink:href=\"#m257374e8e1\" x=\"530.523125\" y=\"376.167969\" style=\"fill: #6b6ecf; stroke: #6b6ecf; stroke-width: 0.5\"/>\n",
              "     </g>\n",
              "    </g>\n",
              "    <g id=\"text_43\">\n",
              "     <!-- Cluster 22 -->\n",
              "     <g transform=\"translate(548.523125 378.792969) scale(0.1 -0.1)\">\n",
              "      <use xlink:href=\"#DejaVuSans-43\"/>\n",
              "      <use xlink:href=\"#DejaVuSans-6c\" x=\"69.824219\"/>\n",
              "      <use xlink:href=\"#DejaVuSans-75\" x=\"97.607422\"/>\n",
              "      <use xlink:href=\"#DejaVuSans-73\" x=\"160.986328\"/>\n",
              "      <use xlink:href=\"#DejaVuSans-74\" x=\"213.085938\"/>\n",
              "      <use xlink:href=\"#DejaVuSans-65\" x=\"252.294922\"/>\n",
              "      <use xlink:href=\"#DejaVuSans-72\" x=\"313.818359\"/>\n",
              "      <use xlink:href=\"#DejaVuSans-20\" x=\"354.931641\"/>\n",
              "      <use xlink:href=\"#DejaVuSans-32\" x=\"386.71875\"/>\n",
              "      <use xlink:href=\"#DejaVuSans-32\" x=\"450.341797\"/>\n",
              "     </g>\n",
              "    </g>\n",
              "    <g id=\"PathCollection_52\">\n",
              "     <g>\n",
              "      <use xlink:href=\"#ma20c134c12\" x=\"530.523125\" y=\"390.846094\" style=\"fill: #9c9ede; stroke: #9c9ede; stroke-width: 0.5\"/>\n",
              "     </g>\n",
              "    </g>\n",
              "    <g id=\"text_44\">\n",
              "     <!-- Cluster 23 -->\n",
              "     <g transform=\"translate(548.523125 393.471094) scale(0.1 -0.1)\">\n",
              "      <use xlink:href=\"#DejaVuSans-43\"/>\n",
              "      <use xlink:href=\"#DejaVuSans-6c\" x=\"69.824219\"/>\n",
              "      <use xlink:href=\"#DejaVuSans-75\" x=\"97.607422\"/>\n",
              "      <use xlink:href=\"#DejaVuSans-73\" x=\"160.986328\"/>\n",
              "      <use xlink:href=\"#DejaVuSans-74\" x=\"213.085938\"/>\n",
              "      <use xlink:href=\"#DejaVuSans-65\" x=\"252.294922\"/>\n",
              "      <use xlink:href=\"#DejaVuSans-72\" x=\"313.818359\"/>\n",
              "      <use xlink:href=\"#DejaVuSans-20\" x=\"354.931641\"/>\n",
              "      <use xlink:href=\"#DejaVuSans-32\" x=\"386.71875\"/>\n",
              "      <use xlink:href=\"#DejaVuSans-33\" x=\"450.341797\"/>\n",
              "     </g>\n",
              "    </g>\n",
              "    <g id=\"PathCollection_53\">\n",
              "     <g>\n",
              "      <use xlink:href=\"#mbb1bfa27c1\" x=\"530.523125\" y=\"405.524219\" style=\"fill: #637939; stroke: #637939; stroke-width: 0.5\"/>\n",
              "     </g>\n",
              "    </g>\n",
              "    <g id=\"text_45\">\n",
              "     <!-- Cluster 24 -->\n",
              "     <g transform=\"translate(548.523125 408.149219) scale(0.1 -0.1)\">\n",
              "      <use xlink:href=\"#DejaVuSans-43\"/>\n",
              "      <use xlink:href=\"#DejaVuSans-6c\" x=\"69.824219\"/>\n",
              "      <use xlink:href=\"#DejaVuSans-75\" x=\"97.607422\"/>\n",
              "      <use xlink:href=\"#DejaVuSans-73\" x=\"160.986328\"/>\n",
              "      <use xlink:href=\"#DejaVuSans-74\" x=\"213.085938\"/>\n",
              "      <use xlink:href=\"#DejaVuSans-65\" x=\"252.294922\"/>\n",
              "      <use xlink:href=\"#DejaVuSans-72\" x=\"313.818359\"/>\n",
              "      <use xlink:href=\"#DejaVuSans-20\" x=\"354.931641\"/>\n",
              "      <use xlink:href=\"#DejaVuSans-32\" x=\"386.71875\"/>\n",
              "      <use xlink:href=\"#DejaVuSans-34\" x=\"450.341797\"/>\n",
              "     </g>\n",
              "    </g>\n",
              "    <g id=\"PathCollection_54\">\n",
              "     <g>\n",
              "      <use xlink:href=\"#m0344f49d76\" x=\"530.523125\" y=\"420.202344\" style=\"fill: #8ca252; stroke: #8ca252; stroke-width: 0.5\"/>\n",
              "     </g>\n",
              "    </g>\n",
              "    <g id=\"text_46\">\n",
              "     <!-- Cluster 25 -->\n",
              "     <g transform=\"translate(548.523125 422.827344) scale(0.1 -0.1)\">\n",
              "      <use xlink:href=\"#DejaVuSans-43\"/>\n",
              "      <use xlink:href=\"#DejaVuSans-6c\" x=\"69.824219\"/>\n",
              "      <use xlink:href=\"#DejaVuSans-75\" x=\"97.607422\"/>\n",
              "      <use xlink:href=\"#DejaVuSans-73\" x=\"160.986328\"/>\n",
              "      <use xlink:href=\"#DejaVuSans-74\" x=\"213.085938\"/>\n",
              "      <use xlink:href=\"#DejaVuSans-65\" x=\"252.294922\"/>\n",
              "      <use xlink:href=\"#DejaVuSans-72\" x=\"313.818359\"/>\n",
              "      <use xlink:href=\"#DejaVuSans-20\" x=\"354.931641\"/>\n",
              "      <use xlink:href=\"#DejaVuSans-32\" x=\"386.71875\"/>\n",
              "      <use xlink:href=\"#DejaVuSans-35\" x=\"450.341797\"/>\n",
              "     </g>\n",
              "    </g>\n",
              "    <g id=\"PathCollection_55\">\n",
              "     <g>\n",
              "      <use xlink:href=\"#mac019e356a\" x=\"530.523125\" y=\"434.880469\" style=\"fill: #b5cf6b; stroke: #b5cf6b; stroke-width: 0.5\"/>\n",
              "     </g>\n",
              "    </g>\n",
              "    <g id=\"text_47\">\n",
              "     <!-- Cluster 26 -->\n",
              "     <g transform=\"translate(548.523125 437.505469) scale(0.1 -0.1)\">\n",
              "      <use xlink:href=\"#DejaVuSans-43\"/>\n",
              "      <use xlink:href=\"#DejaVuSans-6c\" x=\"69.824219\"/>\n",
              "      <use xlink:href=\"#DejaVuSans-75\" x=\"97.607422\"/>\n",
              "      <use xlink:href=\"#DejaVuSans-73\" x=\"160.986328\"/>\n",
              "      <use xlink:href=\"#DejaVuSans-74\" x=\"213.085938\"/>\n",
              "      <use xlink:href=\"#DejaVuSans-65\" x=\"252.294922\"/>\n",
              "      <use xlink:href=\"#DejaVuSans-72\" x=\"313.818359\"/>\n",
              "      <use xlink:href=\"#DejaVuSans-20\" x=\"354.931641\"/>\n",
              "      <use xlink:href=\"#DejaVuSans-32\" x=\"386.71875\"/>\n",
              "      <use xlink:href=\"#DejaVuSans-36\" x=\"450.341797\"/>\n",
              "     </g>\n",
              "    </g>\n",
              "    <g id=\"PathCollection_56\">\n",
              "     <g>\n",
              "      <use xlink:href=\"#mc3413dad6b\" x=\"530.523125\" y=\"449.558594\" style=\"fill: #cedb9c; stroke: #cedb9c; stroke-width: 0.5\"/>\n",
              "     </g>\n",
              "    </g>\n",
              "    <g id=\"text_48\">\n",
              "     <!-- Cluster 27 -->\n",
              "     <g transform=\"translate(548.523125 452.183594) scale(0.1 -0.1)\">\n",
              "      <use xlink:href=\"#DejaVuSans-43\"/>\n",
              "      <use xlink:href=\"#DejaVuSans-6c\" x=\"69.824219\"/>\n",
              "      <use xlink:href=\"#DejaVuSans-75\" x=\"97.607422\"/>\n",
              "      <use xlink:href=\"#DejaVuSans-73\" x=\"160.986328\"/>\n",
              "      <use xlink:href=\"#DejaVuSans-74\" x=\"213.085938\"/>\n",
              "      <use xlink:href=\"#DejaVuSans-65\" x=\"252.294922\"/>\n",
              "      <use xlink:href=\"#DejaVuSans-72\" x=\"313.818359\"/>\n",
              "      <use xlink:href=\"#DejaVuSans-20\" x=\"354.931641\"/>\n",
              "      <use xlink:href=\"#DejaVuSans-32\" x=\"386.71875\"/>\n",
              "      <use xlink:href=\"#DejaVuSans-37\" x=\"450.341797\"/>\n",
              "     </g>\n",
              "    </g>\n",
              "   </g>\n",
              "  </g>\n",
              " </g>\n",
              " <defs>\n",
              "  <clipPath id=\"p596f914704\">\n",
              "   <rect x=\"50.918438\" y=\"24.597656\" width=\"558\" height=\"332.64\"/>\n",
              "  </clipPath>\n",
              " </defs>\n",
              "</svg>\n"
            ],
            "text/plain": [
              "<Figure size 1000x600 with 1 Axes>"
            ]
          },
          "metadata": {},
          "output_type": "display_data"
        }
      ],
      "source": [
        "# Define the number of clusters\n",
        "n_clusters = 28\n",
        "\n",
        "# Create a color palette with 28 colors\n",
        "colors = sns.color_palette('tab20', n_colors=min(20, n_clusters)) + sns.color_palette('tab20b', n_colors=max(0, n_clusters - 20))\n",
        "\n",
        "# Convert to a numpy array for easier access\n",
        "colors = np.array(colors)\n",
        "\n",
        "# Create a scatter plot with clusters\n",
        "plt.figure(figsize=(10, 6))\n",
        "\n",
        "# Plot each cluster with different colors\n",
        "for cluster_label in set(clusters):\n",
        "    plt.scatter(\n",
        "        X_tsne[clusters == cluster_label, 0],\n",
        "        X_tsne[clusters == cluster_label, 1],\n",
        "        label=f'Cluster {cluster_label}',\n",
        "        color=colors[cluster_label % len(colors)]\n",
        "    )\n",
        "\n",
        "# Add title and labels to the plot\n",
        "plt.title('t-SNE Visualization of Article Clusters', fontsize=15)\n",
        "plt.xlabel('t-SNE Component 1', fontsize=12)\n",
        "plt.ylabel('t-SNE Component 2', fontsize=12)\n",
        "\n",
        "# Show the legend\n",
        "plt.legend(title='Clusters')\n",
        "\n",
        "# Display the plot\n",
        "plt.show()"
      ]
    },
    {
      "cell_type": "code",
      "execution_count": 86,
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "id": "By4_qAM_C_p0",
        "outputId": "3db14d34-7ed9-454f-e383-512cb1166c2b"
      },
      "outputs": [
        {
          "name": "stdout",
          "output_type": "stream",
          "text": [
            "Cluster 0 Topics with LDA:\n",
            "control studies surveillance study dengue health interventions community intervention outbreak\n",
            "health public disease diseases national infectious response new biological surveillance\n",
            "health countries disease global people care human ebola international social\n",
            "pandemic public infection risk respiratory health influenza people outbreak healthcare\n",
            "health cases 2009 h1n1 diseases data disease population study china\n",
            "\n",
            "\n",
            "Cluster 1 Topics with LDA:\n",
            "virus viral rna protein viruses infection cells influenza replication proteins\n",
            "protein binding proteins domain structure al residues figure rna par\n",
            "proteins virus protein viral host viruses cell rna replication cells\n",
            "binding hiv peptide residues 3c peptides virus structure activity inhibitor\n",
            "cells cell proteins membrane protein viral er specific hcv type\n",
            "\n",
            "\n",
            "Cluster 2 Topics with LDA:\n",
            "ards lung surfactant injury syndrome respiratory pulmonary pressure clinical 10\n",
            "particles particle respiratory air transmission nanoparticles mm source concentration size\n",
            "wind air room door building al velocity airflow model pressure\n",
            "filter ventilation rate pathogen concentration efficiency filtration dispersion air infection\n",
            "patients airway ventilation patient high pressure respiratory tube intubation risk\n",
            "\n",
            "\n",
            "Cluster 3 Topics with LDA:\n",
            "il immune al response cells infection cytokines inflammatory levels cell\n",
            "cells mice fig infection cd8 cell specific rsv cd4 infected\n",
            "al cells cell immune antigen antigens responses balt peptide specific\n",
            "al cells cell virus response influenza immune older 1991 vaccine\n",
            "cells cell il ms mice cns patients expression eae disease\n",
            "\n",
            "\n",
            "Cluster 4 Topics with LDA:\n",
            "sars cov ace2 virus human cells protein infection al rna\n",
            "sars rbd cov s1 spike protein fig pdb residues structure\n",
            "sars al cov 2020 protein infection cell binding patients s1\n",
            "sars cov plpro nc fig isg15 domain structure terminal sequence\n",
            "patients 19 sars covid cov viral respiratory cases symptoms clinical\n",
            "\n",
            "\n",
            "Cluster 5 Topics with LDA:\n",
            "samples positive elisa serum al st 10 se 20 table\n",
            "samples tgev cancer negative sera serum antibodies sample infected results\n",
            "pcv2 samples positive virus 10 pcr study al elisa serum\n",
            "urgent variability constant 001 frequent especially marked paper passive mouse\n",
            "positive milk samples wd herds test herd sample production number\n",
            "\n",
            "\n",
            "Cluster 6 Topics with LDA:\n",
            "tb disease patients interferon 19 covid study severe number mg\n",
            "covid 19 patients care health patient risk pandemic staff infection\n",
            "patients 19 covid severe sars cov disease clinical diabetes treatment\n",
            "19 covid patients infection disease sars studies risk cov symptoms\n",
            "19 covid ct study data cases images age pneumonia patients\n",
            "\n",
            "\n",
            "Cluster 7 Topics with LDA:\n",
            "pedv piglets samples positive virus group al pcr gene dpi\n",
            "coe pedv egfp 10 recombinant strains vaccine mice tgev immunization\n",
            "al pedv ifn nsp1 cbp 2015 assays 2014 virus viral\n",
            "cholesterol prrsv viral virus porcine cellular entry mm depletion membrane\n",
            "cells pedv infection virus cell fig infected al vero 10\n",
            "\n",
            "\n",
            "Cluster 8 Topics with LDA:\n",
            "pcr dna viral viruses primers amplification human sequences sequence nucleic\n",
            "detection dna ml biosensor surface patients assay sensitivity specific identified\n",
            "pcr al rt samples rna virus 10 positive assay time\n",
            "pcr primer dna hiv array samples specific ml detection sets\n",
            "detection pcr dna water samples based time respiratory patients methods\n",
            "\n",
            "\n",
            "Cluster 9 Topics with LDA:\n",
            "motifs disease sera motif epitope panel peptide specificity specific antigens\n",
            "strains gene isolates strain ibv s1 al china sequences nucleotide\n",
            "ibv vaccine 10 strains protein epitopes virus strain chickens cell\n",
            "protein strains antigenic sites regions virulent phosphorylation analysis aa carried\n",
            "human malaria liver mouse parasite proteins antigens mice hepatocytes table\n",
            "\n",
            "\n",
            "Cluster 10 Topics with LDA:\n",
            "influenza virus h1n1 viruses viral infection pandemic respiratory patients countries\n",
            "influenza infections pandemic virus cases states state h1n1 time health\n",
            "influenza patients virus study infection illness travel respiratory case treatment\n",
            "out city border muscle correlations round utilized showing displayed helpful\n",
            "influenza patients study ili infection cases respiratory household associated disease\n",
            "\n",
            "\n",
            "Cluster 11 Topics with LDA:\n",
            "wuhan cities 2020 cases city hubei lockdown 19 january covid\n",
            "control number epidemic cases measures time model transmission population preprint\n",
            "2020 19 covid model time number lockdown infection population infected\n",
            "event seasonal scenarios sars interval size serial duration cov 2020\n",
            "cases number testing days reproduction data 19 case covid infected\n",
            "\n",
            "\n",
            "Cluster 12 Topics with LDA:\n",
            "virus infection viruses 10 viral al cells infected respiratory antibody\n",
            "phage vaccine antibodies virus dna vaccines cells viruses surface antibody\n",
            "resistance se drug al dmy resistant concentrations drugs mechanism cows\n",
            "infection virus treatment patients viral disease activity antiviral birds human\n",
            "rotavirus ms cells hape hsp90 patients fig disease tof small\n",
            "\n",
            "\n",
            "Cluster 13 Topics with LDA:\n",
            "patients cancer treatment study infection care patient risk disease therapy\n",
            "patients surgery laparoscopic patient performed surgical results cases approach postoperative\n",
            "blood patients imaging results clinical treatment patient transfusion positive anti\n",
            "patients cf study cells results cftr disease treatment age lung\n",
            "cases patients expression cells tumor cell results study carcinoma tumors\n",
            "\n",
            "\n",
            "Cluster 14 Topics with LDA:\n",
            "hcov oc43 sars 229e cov serum samples nucleocapsid patients cells\n",
            "cells hcov 229e virus coronavirus viral protein cell rna human\n",
            "hcov positive respiratory samples al 229e oc43 gene strains study\n",
            "hcov respiratory human viruses patients infection children 229e infections oc43\n",
            "oc43 cells infection virus hcov mrc cell treatment antibody culture\n",
            "\n",
            "\n",
            "Cluster 15 Topics with LDA:\n",
            "hrv virus infection children respiratory studies viral infections viruses ari\n",
            "rsv respiratory viral study virus patients viruses children influenza infection\n",
            "asthma children age years respiratory wheezing study symptoms diagnosis infection\n",
            "fever children infection respiratory common infants usually acute disease high\n",
            "children bronchiolitis study severe clinical treatment respiratory patients cases pneumonia\n",
            "\n",
            "\n",
            "Cluster 16 Topics with LDA:\n",
            "ifn type al cells viral ifns infection virus response cell\n",
            "summary guidelines content pharmaceutical aim authors conclusions information exhibited method\n",
            "summary guidelines content pharmaceutical aim authors conclusions information exhibited method\n",
            "ifn cells expression lps hcv treatment ml tnf treated il\n",
            "ifn cells type ifns treatment hcv infection expression treated cell\n",
            "\n",
            "\n",
            "Cluster 17 Topics with LDA:\n",
            "preprint 2020 cov sars license al 10 doi medrxiv viral\n",
            "number preprint 2020 cases license 19 rate medrxiv data covid\n",
            "data model preprint 2020 time license social number 19 covid\n",
            "preprint license 2020 patients medrxiv 19 covid doi display author\n",
            "cases confirmed 95 2020 ci covid preprint 19 beds active\n",
            "\n",
            "\n",
            "Cluster 18 Topics with LDA:\n",
            "al proteins protein viral virus cells membrane rna replication sirna\n",
            "al virus viruses infection respiratory infections patients viral dna clinical\n",
            "al cells cysteine pd activity conjugate cell studies golgi enzyme\n",
            "al species viruses human virus host studies disease strains population\n",
            "al emissions aircraft number 1999 model results 10 particles aviation\n",
            "\n",
            "\n",
            "Cluster 19 Topics with LDA:\n",
            "cov mers viral cells protein infection virus fig rbd mice\n",
            "mers cov plasma transmission virus study transfusion information blood level\n",
            "mers cov patients infection al cases respiratory transmission pcr reported\n",
            "mers cov transmission human cases infection risk control reported healthcare\n",
            "mers patients nursing study 119 risk days infection hospital students\n",
            "\n",
            "\n",
            "Cluster 20 Topics with LDA:\n",
            "rna viral sequence virus al viruses sequences genome pro dna\n",
            "genes human bat bats virus viruses gene identified nsp1 table\n",
            "sequences genome sequence species data viruses genomes virus based analysis\n",
            "species trna site frameshifting codon amino isg15 figure al acid\n",
            "patients gene analysis mutations genes expression mutation chromosome patient cells\n",
            "\n",
            "\n",
            "Cluster 21 Topics with LDA:\n",
            "lamp rt pcr al primer primers assay 10 reaction cdv\n",
            "mhv virus rna jhm al protein 10 cells a59 viruses\n",
            "al mhv infection virus mice rt jhm cells cell a59\n",
            "spike al fusion jhm virus viruses cell antigen substitutions recombinant\n",
            "mhv mice cells viral ribavirin plp peptide fig infected infection\n",
            "\n",
            "\n",
            "Cluster 22 Topics with LDA:\n",
            "cats fecv virus fip isolates rna cat 3c samples pcr\n",
            "cats cat animal population housing disease environment animals care health\n",
            "al dogs clinical cats signs fecal animals feces diagnosis diarrhea\n",
            "cats al infection animals anti infected 10 feline antibodies disease\n",
            "cells cats affected feline positive expression age fip figure animals\n",
            "\n",
            "\n",
            "Cluster 23 Topics with LDA:\n",
            "mice infection virus cells sars lung immune 10 infected response\n",
            "mice ccl5 figure group expression fig cells lung cd11b 10\n",
            "dogs barrier patients lung treatment disease cell increased protein levels\n",
            "cells lps cell lung iav induced angiotensin pulmonary ii activation\n",
            "il cells patients exosomes inflammatory treatment anti cytokines 10 specific\n",
            "\n",
            "\n",
            "Cluster 24 Topics with LDA:\n",
            "cells rna viral virus figure infection replication expression infected cell\n",
            "protein cells virus fig al antibody expression 10 tgev antibodies\n",
            "cells cell fig culture 10 proteins medium protein isg20 membrane\n",
            "cells ml activity 10 induced al 50 cell expression infected\n",
            "cells cell gene human expression protein genes patients analysis strains\n",
            "\n",
            "\n",
            "Cluster 25 Topics with LDA:\n",
            "time learning use group students al chinese mobile based stress\n",
            "al human waste vaccine risk potential system blood health copper\n",
            "ningbo qtl square effect rabies strains triage clinical animal power\n",
            "risk infl health surveillance uenza disease tb infection crisis china\n",
            "study research data care health studies participants sars patients people\n",
            "\n",
            "\n",
            "Cluster 26 Topics with LDA:\n",
            "denv al viral replication protein infection rna cells antiviral virus\n",
            "denv virus cells al infection dengue viral antibodies protein human\n",
            "cells autophagy pv viral fig denv pathway cell infection rna\n",
            "capsid able factor normally evolved molecule facilitate 56 require authors\n",
            "assay denv cells based 10 neutralizing plaque antibody enhancing 37\n",
            "\n",
            "\n",
            "Cluster 27 Topics with LDA:\n",
            "host models al time distribution model viral hosts rate infectious\n",
            "model time number individuals network disease epidemic infected infection models\n",
            "data infection transmission time spread influenza disease population based control\n",
            "exposure risk regression model variables response data models causal dose\n",
            "model image data task transmission results set method learning based\n",
            "\n",
            "\n"
          ]
        }
      ],
      "source": [
        "from sklearn.decomposition import LatentDirichletAllocation\n",
        "\n",
        "num_topics = 5\n",
        "\n",
        "def display_topics(model, feature_names, no_top_words):\n",
        "    topics = []\n",
        "    for topic_idx, topic in enumerate(model.components_):\n",
        "        topics.append(\" \".join([feature_names[i] for i in topic.argsort()[:-no_top_words - 1:-1]]))\n",
        "    return topics\n",
        "\n",
        "# Initialize the Count Vectorizer\n",
        "count_vectorizer = CountVectorizer(max_features=4096)\n",
        "\n",
        "# Fit and transform the cleaned text to extract Count features\n",
        "count_matrix = count_vectorizer.fit_transform(df['cleaned_text_str'])\n",
        "\n",
        "# Fit the LDA model on the entire dataset\n",
        "lda = LatentDirichletAllocation(n_components=num_topics, random_state=42)\n",
        "lda.fit(count_matrix)\n",
        "\n",
        "# Display topics for each cluster\n",
        "for cluster_num in sorted(df['cluster'].unique()):\n",
        "    cluster_data = df[df['cluster'] == cluster_num]\n",
        "    cluster_count_matrix = count_vectorizer.transform(cluster_data['cleaned_text_str'])\n",
        "    lda.fit(cluster_count_matrix)\n",
        "    print(f\"Cluster {cluster_num} Topics with LDA:\")\n",
        "    topics = display_topics(lda, count_vectorizer.get_feature_names_out(), 10)\n",
        "    for topic in topics:\n",
        "        print(topic)\n",
        "    print(\"\\n\")"
      ]
    },
    {
      "cell_type": "code",
      "execution_count": 61,
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/",
          "height": 0
        },
        "id": "G0nBbIF_vJCY",
        "outputId": "2fcee984-648d-4481-c60a-036d6f56f1ca"
      },
      "outputs": [
        {
          "name": "stdout",
          "output_type": "stream",
          "text": [
            "Cluster 0 Topics with NMF:\n",
            "health countries care ebola disease public global regional policy water\n",
            "control studies surveillance dengue study interventions intervention community health outbreak\n",
            "2009 h1n1 cases pandemic flu public diarrhea health infection respiratory\n",
            "china diseases aids hiv cases disease infectious population 000 government\n",
            "public health biological agents response research national local state program\n",
            "\n",
            "\n",
            "Cluster 1 Topics with NMF:\n",
            "rna viral virus dna viruses translation protein replication dsrna al\n",
            "binding 3c residues structure protein domain al site 1a figure\n",
            "cells virus hiv cell influenza viral infection type binding viruses\n",
            "proteins protein host cell virus cells cellular viral pathogen viruses\n",
            "autophagy viral virus protein role cell pkr proteins death replication\n",
            "\n",
            "\n",
            "Cluster 2 Topics with NMF:\n",
            "patients airway patient ventilation tube intubation respiratory obstruction risk described\n",
            "filter filtration efficiency filters aerosol pressure layer drop aerosols loading\n",
            "wind ventilation air room building pathogen model source airborne rate\n",
            "particles nanoparticles particle air surface concentration size membrane water fig\n",
            "ards syndrome path organ clinical respiratory different disease injury like\n",
            "\n",
            "\n",
            "Cluster 3 Topics with NMF:\n",
            "cells ms cell patients mice il expression eae disease cns\n",
            "il immune response al cytokines levels inflammatory infection activity increased\n",
            "al cells balt cell immune lymphoid responses cns antigen nasal\n",
            "cells mice nk cd8 cell fig rsv infection specific cd4\n",
            "vaccine response vaccines cell antigen dna responses immune antigens influenza\n",
            "\n",
            "\n",
            "Cluster 4 Topics with NMF:\n",
            "sars cov virus patients human infection 2003 viral study al\n",
            "cov sars plpro isg15 structure fig domain terminal mm binding\n",
            "ace2 cov expression sars 19 patients cells covid receptor viral\n",
            "rbd cov sars spike s1 protein pdb fig chain head\n",
            "al cov 2020 sars protein 19 covid virus coronavirus cases\n",
            "\n",
            "\n",
            "Cluster 5 Topics with NMF:\n",
            "samples positive sample test negative algorithm result tests pool results\n",
            "milk wd herds production herd outbreak study day al effect\n",
            "pcv2 cell al herds pigs virus herd samples cells sequence\n",
            "sera elisa serum tgev positive antibody antibodies collected negative samples\n",
            "positive infected procedure false true sub number group tests case\n",
            "\n",
            "\n",
            "Cluster 6 Topics with NMF:\n",
            "patients 19 covid sars disease infection cov severe treatment risk\n",
            "19 covid participants study health pandemic china reported increased mental\n",
            "19 covid images ct pneumonia learning based training set deep\n",
            "diabetes 19 covid patients severe individuals levels glucose ace2 blood\n",
            "public analysis covid 19 data research number positive new people\n",
            "\n",
            "\n",
            "Cluster 7 Topics with NMF:\n",
            "al pedv 2015 assays 2014 antibodies pcr detection virus detected\n",
            "cells pedv ifn al nsp1 cbp fig protein infection nuclear\n",
            "cells cholesterol pedv virus infection viral prrsv porcine cell cellular\n",
            "coe pedv egfp tgev recombinant mice immunization strains protein 10\n",
            "piglets pedv positive samples feed group dpi pcr inoculated 10\n",
            "\n",
            "\n"
          ]
        },
        {
          "name": "stderr",
          "output_type": "stream",
          "text": [
            "/usr/local/lib/python3.10/dist-packages/sklearn/decomposition/_nmf.py:1665: ConvergenceWarning: Maximum number of iterations 200 reached. Increase it to improve convergence.\n",
            "  warnings.warn(\n"
          ]
        },
        {
          "name": "stdout",
          "output_type": "stream",
          "text": [
            "Cluster 8 Topics with NMF:\n",
            "al pcr rna rt 10 virus samples real time viral\n",
            "water detection based method sensor technique samples analysis al monitoring\n",
            "pcr samples positive clinical respiratory detection patients results time assays\n",
            "dna viral pcr primer viruses hiv virus specific human rna\n",
            "detection surface biosensor dna sensitivity gold sensor ml disease measurement\n",
            "\n",
            "\n",
            "Cluster 9 Topics with NMF:\n",
            "ibv strains gene s1 strain al sequences 91 gi sequence\n",
            "epitopes protein cell vaccine predicted epitope score hla antigenic binding\n",
            "human liver malaria mouse parasite hepatocytes proteins stage infected host\n",
            "10 ibv strains vaccine chickens 09 strain group virus protection\n",
            "motifs sera disease epitope panel peptide motif antigens specific specificity\n",
            "\n",
            "\n",
            "Cluster 10 Topics with NMF:\n",
            "influenza virus viruses vaccines vaccine pandemic treatment years human viral\n",
            "patients flu disease days enrolled influenza h1n1 progression studies risk\n",
            "ili study influenza household cases children respiratory associated incidence age\n",
            "infection co influenza respiratory viral bacterial severe patients icu 2009\n",
            "h1n1 countries influenza time pandemic activity week peak weeks period\n",
            "\n",
            "\n",
            "Cluster 11 Topics with NMF:\n",
            "wuhan cities city cases lockdown hubei 2020 population preprint virus\n",
            "cases number control sars epidemic transmission outbreak measures days time\n",
            "2020 19 covid model feb time epidemic china period infectious\n",
            "region regional national regions model measures number lockdown parameters figure\n",
            "infection fig population peak lockdown preprint 2020 number containment susceptible\n",
            "\n",
            "\n",
            "Cluster 12 Topics with NMF:\n",
            "virus infection viruses viral cells disease human respiratory 10 infected\n",
            "resistance drug resistant mechanism drugs al gene like tolerance antibiotic\n",
            "se cows concentrations igg1 calves ha serum al control ml\n",
            "hiv treatment mg patients activity à1 al viral antiviral dna\n",
            "phage biosensor ms surface detection based tof al protein proteins\n",
            "\n",
            "\n",
            "Cluster 13 Topics with NMF:\n",
            "cases expression patients cells tumor cell results study carcinoma tumors\n",
            "cf patients cftr lung results study cells age treatment disease\n",
            "patients cancer study care treatment patient pain clinical symptoms chemotherapy\n",
            "patients laparoscopic surgery patient performed results surgical cases approach postoperative\n",
            "blood patients transfusion anti donors results positive platelet antibodies cells\n",
            "\n",
            "\n",
            "Cluster 14 Topics with NMF:\n",
            "cells hcov 229e virus coronavirus viral rna replication figure mm\n",
            "infection 229e virus oc43 antibody respiratory viruses coronaviruses human infections\n",
            "hcov respiratory patients human infection cns different associated viruses oc43\n",
            "cells hcov oc43 mrc treatment cell infection virus 10 figure\n",
            "hcov sars cov 229e serum oc43 nucleocapsid samples cells protein\n",
            "\n",
            "\n",
            "Cluster 15 Topics with NMF:\n",
            "fever children infection usually meningitis common il high pneumonia cause\n",
            "hrv virus infection viruses studies children viral types respiratory fi\n",
            "respiratory rsv viral virus infection infections influenza viruses study infants\n",
            "asthma age years children wheezing respiratory diagnosis airway copd adults\n",
            "children pneumonia pneumoniae study patients severe age years cases treatment\n",
            "\n",
            "\n",
            "Cluster 16 Topics with NMF:\n",
            "ifn type cells hcmv treated ifns viral expression replication ie\n",
            "al ifn viral cells type response virus activation proteins infection\n",
            "type ifns ifn cells infection signaling cell virus antiviral responses\n",
            "ifn expression lps tnf ml cells inflammatory liver hepatocytes signaling\n",
            "ifn hcv cells treatment il genotype production associated response expression\n",
            "\n",
            "\n",
            "Cluster 17 Topics with NMF:\n",
            "preprint 2020 license medrxiv available international by cc author funder\n",
            "data time number dynamics social deaths countries preprint model cases\n",
            "cases ci 95 confirmed active 2020 beds india preprint covid\n",
            "sars cov ct preprint model doi medrxiv chest clinical 10\n",
            "patients 19 covid preprint group medrxiv study severe license disease\n",
            "\n",
            "\n"
          ]
        },
        {
          "name": "stderr",
          "output_type": "stream",
          "text": [
            "/usr/local/lib/python3.10/dist-packages/sklearn/decomposition/_nmf.py:1665: ConvergenceWarning: Maximum number of iterations 200 reached. Increase it to improve convergence.\n",
            "  warnings.warn(\n"
          ]
        },
        {
          "name": "stdout",
          "output_type": "stream",
          "text": [
            "Cluster 18 Topics with NMF:\n",
            "al cells viral virus proteins membrane replication protein cell infection\n",
            "al emissions aircraft aviation rf ice particles 1999 climate model\n",
            "viruses al rna virus like plant host plants evolution fungi\n",
            "al cysteine conjugate enzyme conjugates reaction kidney acid amino human\n",
            "al species studies pd risk forest disease study pcr bats\n",
            "\n",
            "\n",
            "Cluster 19 Topics with NMF:\n",
            "cov al mers virus infection human 2014 animals reported transmission\n",
            "cov mers cells virus viral figure rna protein cell infection\n",
            "mers patients cov cases infection risk transmission saudi respiratory case\n",
            "rbd fc mers mice cov protein specific igg 10 serum\n",
            "cov mers infection lung viral mice cells treatment fig tissue\n",
            "\n",
            "\n",
            "Cluster 20 Topics with NMF:\n",
            "gene patients mutations analysis genes mutation chromosome expression patient syndrome\n",
            "viruses genome virus sequences viral species genomes human sequence genes\n",
            "mnmc trna figure nm growth strain mm pathway phase modification\n",
            "rna al strand figure sequence structure stem accumulation viral recombination\n",
            "isg15 nsp1 amino pro acid fig models species preferences protein\n",
            "\n",
            "\n",
            "Cluster 21 Topics with NMF:\n",
            "al mhv jhm virus cell cells infection spike viral mice\n",
            "gag cdv al mrna 10 assay rna virus vector day\n",
            "rt pcr lamp al assay primers influenza detection reaction samples\n",
            "mhv mice plp peptide cells ribavirin mimic hi viral fig\n",
            "lamp dna structure primer model loop primers structures size positive\n",
            "\n",
            "\n",
            "Cluster 22 Topics with NMF:\n",
            "cats cat animal population housing health disease 46 environment care\n",
            "al dogs csf cats clinical animals ventricular signs fig affected\n",
            "fecv cats virus fip 79 rna 10 serotype infection samples\n",
            "fecal clinical feces signs diarrhea dogs diagnosis animals young disease\n",
            "cells cats expression figure feline pbmcs stimulation viral mrna cell\n",
            "\n",
            "\n",
            "Cluster 23 Topics with NMF:\n",
            "il patients cells treatment inflammatory study therapy dogs anti 10\n",
            "mice day sars virus cov al mg kg lung 10\n",
            "infection cells lung iav viral epithelial host cell tolerance infections\n",
            "lung barrier ec cell actin protein vascular induced permeability lps\n",
            "mice cells antigen cell specific responses anti virus immune group\n",
            "\n",
            "\n",
            "Cluster 24 Topics with NMF:\n",
            "cells strains isolates infection cell results different aureus patients protein\n",
            "cells virus viral al infection rna replication figure infected egr\n",
            "gene genes expression analysis cancer study cell patients protein human\n",
            "cell cells line culture lines human schwann serum medium primary\n",
            "protein ul44 fig proteins sumo cells fusion fragment 10 binding\n",
            "\n",
            "\n",
            "Cluster 25 Topics with NMF:\n",
            "research study health al risk data studies use participants care\n",
            "infl uenza disease surveillance pandemic virus health risk severity 2009\n",
            "qtl effect strains cc power al strain mapping size number\n",
            "association year committee veterinary said dogs tumours animal bone years\n",
            "ningbo square government project urban fi area city local china\n",
            "\n",
            "\n",
            "Cluster 26 Topics with NMF:\n",
            "denv cells infection virus hr treatment assay titer viral 100\n",
            "cells autophagy pv fig viral denv pathway cell infection gfp\n",
            "antibodies denv donor neutralizing protein dengue antibody bound neutralization human\n",
            "denv al viral replication protein infection rna cells antiviral virus\n",
            "al denv genotypes codon urban sequences genotype strains human ae\n",
            "\n",
            "\n",
            "Cluster 27 Topics with NMF:\n",
            "exposure risk regression model response variables data models causal dose\n",
            "individuals network disease infection contact individual epidemic infected person number\n",
            "gr fractional control model function matrix problem differential optimal disease\n",
            "model time distribution user number data fig results eq network\n",
            "infection data influenza host transmission time spread infected wind population\n",
            "\n",
            "\n"
          ]
        }
      ],
      "source": [
        "from sklearn.decomposition import NMF\n",
        "\n",
        "# Fit the NMF model on the entire dataset\n",
        "nmf = NMF(n_components=num_topics, random_state=42)\n",
        "nmf.fit(count_matrix)\n",
        "\n",
        "# Display topics for each cluster\n",
        "for cluster_num in sorted(df['cluster'].unique()):\n",
        "    cluster_data = df[df['cluster'] == cluster_num]\n",
        "    cluster_count_matrix = count_vectorizer.transform(cluster_data['cleaned_text_str'])\n",
        "    nmf.fit(cluster_count_matrix)\n",
        "    print(f\"Cluster {cluster_num} Topics with NMF:\")\n",
        "    topics = display_topics(nmf, count_vectorizer.get_feature_names_out(), 10)\n",
        "    for topic in topics:\n",
        "        print(topic)\n",
        "    print(\"\\n\")"
      ]
    },
    {
      "cell_type": "code",
      "execution_count": 84,
      "metadata": {
        "id": "FboibFCYLKAW"
      },
      "outputs": [],
      "source": [
        "def interactive_plot(df):\n",
        "    clusters = sorted(df['y'].unique())\n",
        "\n",
        "    # Generate distinct colors for each cluster using Plotly's colors\n",
        "    colors = px.colors.qualitative.Plotly\n",
        "\n",
        "    # Create a scatter plot for each cluster with a unique color\n",
        "    data = []\n",
        "    for i, cluster in enumerate(clusters):\n",
        "        cluster_data = df[df['y'] == cluster]\n",
        "        scatter = go.Scatter(\n",
        "            x=cluster_data['tsne-2d-one'],\n",
        "            y=cluster_data['tsne-2d-two'],\n",
        "            mode='markers',\n",
        "            name=f'Cluster {cluster}',\n",
        "            text=cluster_data['title'],\n",
        "            hoverinfo='text',\n",
        "            marker=dict(size=10, color=colors[i % len(colors)]),  # Assign a unique color\n",
        "            visible=False  # Initially, make all traces invisible\n",
        "        )\n",
        "        data.append(scatter)\n",
        "\n",
        "    # Add a scatter plot for all clusters with different colors\n",
        "    scatter_all = go.Scatter(\n",
        "        x=df['tsne-2d-one'],\n",
        "        y=df['tsne-2d-two'],\n",
        "        mode='markers',\n",
        "        name='All Clusters',\n",
        "        text=df['title'],\n",
        "        hoverinfo='text',\n",
        "        marker=dict(size=10, color=df['y'].map(lambda x: colors[x % len(colors)])),  # Assign colors by cluster\n",
        "        visible=True  # Initially, show all clusters\n",
        "    )\n",
        "    data.append(scatter_all)\n",
        "\n",
        "    x_min = df['tsne-2d-one'].min()\n",
        "    x_max = df['tsne-2d-one'].max()\n",
        "    y_min = df['tsne-2d-two'].min()\n",
        "    y_max = df['tsne-2d-two'].max()\n",
        "\n",
        "    # Initialize the figure\n",
        "    fig = go.Figure(data=data)\n",
        "\n",
        "    # Ensure square aspect ratio\n",
        "    fig.update_layout(\n",
        "        title='Cluster Visualization',\n",
        "        xaxis=dict(title='t-SNE 1', range=[x_min, x_max]),  # Ensure x and y axes are equal\n",
        "        yaxis=dict(title='t-SNE 2', range=[y_min, y_max]),\n",
        "        width=1000,\n",
        "        height=700  # Ensure the figure is square-shaped\n",
        "    )\n",
        "\n",
        "    # Add hover functionality\n",
        "    fig.update_traces(\n",
        "        hoverinfo='text',\n",
        "        marker=dict(opacity=0.7, size=8),\n",
        "    )\n",
        "\n",
        "    # Add interactive slider for cluster selection\n",
        "    steps = []\n",
        "    for i, cluster in enumerate(clusters):\n",
        "        step = dict(\n",
        "            method='update',\n",
        "            args=[{'visible': [False] * len(clusters) + [False]},  # Hide all clusters\n",
        "                {'title': f'Cluster {cluster}'}],\n",
        "            label=f'Cluster {cluster}'\n",
        "        )\n",
        "        # Only make the current cluster visible\n",
        "        step['args'][0]['visible'][i] = True\n",
        "        steps.append(step)\n",
        "\n",
        "    # Add final step for all clusters\n",
        "    steps.append(dict(\n",
        "        method='update',\n",
        "        args=[{'visible': [False] * len(clusters) + [True]},  # Only show the 'all clusters' trace\n",
        "            {'title': 'All Clusters'}],\n",
        "        label='All Clusters'\n",
        "    ))\n",
        "\n",
        "    sliders = [dict(\n",
        "        active=len(steps) - 1,\n",
        "        currentvalue={\"prefix\": \"Cluster: \"},\n",
        "        pad={\"t\": 50},\n",
        "        steps=steps\n",
        "    )]\n",
        "\n",
        "    fig.update_layout(\n",
        "        sliders=sliders\n",
        "    )\n",
        "\n",
        "    # Display the plot\n",
        "    fig.show()"
      ]
    }
  ],
  "metadata": {
    "accelerator": "GPU",
    "colab": {
      "gpuType": "T4",
      "provenance": []
    },
    "kernelspec": {
      "display_name": "Python 3",
      "name": "python3"
    },
    "language_info": {
      "codemirror_mode": {
        "name": "ipython",
        "version": 3
      },
      "file_extension": ".py",
      "mimetype": "text/x-python",
      "name": "python",
      "nbconvert_exporter": "python",
      "pygments_lexer": "ipython3",
      "version": "3.12.2"
    }
  },
  "nbformat": 4,
  "nbformat_minor": 0
}
